<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    
  
    
  
    <link rel="alternate" type="application/rss+xml" href="/rss.xml" />
    
    <link rel="shortcut icon" type="image/ico" href="https://a0.awsstatic.com/libra-css/images/site/fav/favicon.ico"/>
    <link rel="apple-touch-icon" sizes="57x57" href="https://a0.awsstatic.com/libra-css/images/site/touch-icon-iphone-114-smile.png"/>
    <link rel="apple-touch-icon" sizes="72x72" href="https://a0.awsstatic.com/libra-css/images/site/touch-icon-ipad-144-smile.png"/>
    <link rel="apple-touch-icon" sizes="114x114" href="https://a0.awsstatic.com/libra-css/images/site/touch-icon-iphone-114-smile.png"/>
    <link rel="apple-touch-icon" sizes="144x144" href="https://a0.awsstatic.com/libra-css/images/site/touch-icon-ipad-144-smile.png"/>  
  
    <title>Registry of Open Data on AWS</title>
  
    <!-- Latest compiled and minified CSS -->
    <link rel="stylesheet" href="https://assets.opendata.aws/css/bootstrap/3.4.1/bootstrap.min.css">
  
    <!-- Our local CSS -->
    <link rel="stylesheet" href="/css/main.css">
  
  
  </head>
  <body>
    <!-- jQuery (necessary for Bootstrap's JavaScript plugins) -->
    <script src="https://assets.opendata.aws/js/jquery/3.5.1/jquery.min.js"></script>
    <!-- Latest compiled and minified JavaScript -->
    <script src="https://assets.opendata.aws/js/bootstrap/3.4.1/bootstrap.min.js"></script>
    <nav class="navbar navbar-default navbar-fixed-top">
      <div class="container">
        <div class="roda-header">
          <h2><a href="/" alt="Home">Registry of Open Data on AWS</a></h2>
          <a href="http://aws.amazon.com/what-is-cloud-computing">
            <img src="https://assets.opendata.aws/img/AWS-Logo_White-Color_300x180.png" alt="Powered by AWS Cloud Computing" id="aws_header_logo">
          </a>
        </div>
      </div>
    </nav>
    <div class="container" id="dataatwork">
  <p class="lede">Usage examples for all datasets listed in the <a href="/">Registry of Open Data on AWS</a> tagged with <strong>health</strong>.</p>
  <hr>

      <div id="foldingathome-covid19" class="dataset">
        <h4><a href="../foldingathome-covid19/">Foldingathome COVID-19 Datasets</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/FoldingAtHome/coronavirus/blob/master/README.md" class="" target="_none">Folding@home COVID-19 efforts</a> by Folding@home Consortium</a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://covid.molssi.org//simulations/#foldinghome-expanded-ensemble-absolute-free-energy-calculations-of-potential-small-molecule-inhibitors-of-the-sars-cov-2-main-protease-from-the-covid-moonshot-3clpro-3clpro-protease-activity" class="" target="_none">SARS-CoV-2 COVID Moonshot absolute free energy calculations</a> by <a href="http://www.voelzlab.org/" target="_none">The Voelz lab at Temple University</a></a>
            <br>
            
          </li>          <li>
            <a href="https://covid.molssi.org//simulations/#foldinghome-simulations-of-nsp5-3clpro-3clpro-protease-activity" class="" target="_none">SARS-CoV-2 main viral protease (Mpro, 3CLPro, nsp5) dimer simulations: A 2.9 ms dataset of the SARS-CoV-2 main viral protease (apo, dimer) in search of cryptic pockets</a> by <a href="https://bowmanlab.biochem.wustl.edu" target="_none">The Bowman lab at Washington University in St. Louis</a></a>
            <br>
            
          </li>          <li>
            <a href="https://covid.molssi.org//simulations/#foldinghome-sars-cov-2-main-protease-apo-monomer-simulations-3clpro-3clpro-protease-activity" class="" target="_none">SARS-CoV-2 main viral protease (Mpro, 3CLPro, nsp5) monomer simulations: A 2.6 ms equilibrium dataset of the SARS-CoV-2 main viral protease (apo, monomer)</a> by <a href="https://choderalab.org" target="_none">The Chodera lab at MSKCC</a></a>
            <br>
            
          </li>          <li>
            <a href="https://covid.molssi.org//simulations/#foldinghome-simulations-of-nsp5-3clpro-3clpro-protease-activity" class="" target="_none">SARS-CoV-2 main viral protease (Mpro, 3CLPro, nsp5) monomer simulations: A 6.4 ms dataset of the SARS-CoV-2 main viral protease (apo, monomer) in search of cryptic pockets</a> by <a href="https://bowmanlab.biochem.wustl.edu" target="_none">The Bowman lab at Washington University in St. Louis</a></a>
            <br>
            
          </li>          <li>
            <a href="https://covid.molssi.org//simulations/#foldinghome-simulations-of-nsp10-nsp10-nil" class="" target="_none">SARS-CoV-2 nsp10 dataset: A 6.1 ms dataset of the SARS-CoV-2 nsp10 protein in search of cryptic pockets</a> by <a href="https://bowmanlab.biochem.wustl.edu" target="_none">The Bowman lab at Washington University in St. Louis</a></a>
            <br>
            
          </li>          <li>
            <a href="https://covid.molssi.org//simulations/#foldinghome-simulations-of-nsp3-macrodomain-macrodomain-host-immune-response" class="" target="_none">SARS-CoV-2 nsp3 macrodomain dataset: An 11 ms dataset of the SARS-CoV-2 nsp3 macrodomain in search of cryptic pockets</a> by <a href="https://bowmanlab.biochem.wustl.edu" target="_none">The Bowman lab at Washington University in St. Louis</a></a>
            <br>
            
          </li>          <li>
            <a href="https://covid.molssi.org//simulations/#foldinghome-simulations-of-nsp3-pl2pro-domain-plpro-plpro-protease-activity" class="" target="_none">SARS-CoV-2 nsp3 pl2pro domain dataset: An 731 µs dataset of the SARS-CoV-2 nsp3 pl2pro domain in search of cryptic pockets</a> by <a href="https://bowmanlab.biochem.wustl.edu" target="_none">The Bowman lab at Washington University in St. Louis</a></a>
            <br>
            
          </li>          <li>
            <a href="https://covid.molssi.org//simulations/#foldinghome-simulations-of-nsp7-nsp7-nil" class="" target="_none">SARS-CoV-2 nsp7 simulations: A 3.7 ms dataset of the SARS-CoV-2 nsp7 protein in search of cryptic pockets</a> by <a href="https://bowmanlab.biochem.wustl.edu" target="_none">The Bowman lab at Washington University in St. Louis</a></a>
            <br>
            
          </li>          <li>
            <a href="https://covid.molssi.org//simulations/#foldinghome-simulations-of-nsp8-nsp8-nil" class="" target="_none">SARS-CoV-2 nsp8 simulations: A 1.8 ms dataset of the SARS-CoV-2 nsp8 protein in search of cryptic pockets</a> by <a href="https://bowmanlab.biochem.wustl.edu" target="_none">The Bowman lab at Washington University in St. Louis</a></a>
            <br>
            
          </li>          <li>
            <a href="https://covid.molssi.org//simulations/#foldinghome-simulations-of-nsp9-nsp9-nil" class="" target="_none">SARS-CoV-2 nsp9 simulations: A 9 ms dataset of the SARS-CoV-2 nsp9 protein in search of cryptic pockets</a> by <a href="https://bowmanlab.biochem.wustl.edu" target="_none">The Bowman lab at Washington University in St. Louis</a></a>
            <br>
            
          </li>          <li>
            <a href="https://covid.molssi.org//simulations/#foldinghome-simulations-of-nsp12-rdrp-nil" class="" target="_none">SARS-CoV-2 RNA polymerase (nsp12, RdRP) dataset: A 3.4 ms dataset of the SARS-CoV-2 nsp12 protein in search of cryptic pockets</a> by <a href="https://bowmanlab.biochem.wustl.edu" target="_none">The Bowman lab at Washington University in St. Louis</a></a>
            <br>
            
          </li>          <li>
            <a href="https://covid.molssi.org//simulations/#foldinghome-simulations-of-the-sars-cov-2-spike-protein-spike-spike-binding" class="" target="_none">SARS-CoV-2 spike protein dataset: A 1.2 ms dataset of the SARS-CoV-2 spike protein in search of cryptic pockets</a> by <a href="https://bowmanlab.biochem.wustl.edu" target="_none">The Bowman lab at Washington University in St. Louis</a></a>
            <br>
            
          </li>          <li>
            <a href="https://covid.molssi.org//simulations/#foldinghome-simulations-of-the-sars-cov-2-spike-rbd-bound-to-human-ace2" class="" target="_none">SARS-CoV-2 spike RBD bound to human ACE2 receptor (173.8 us): Wild-type and mutant simulations</a> by <a href="https://choderalab.org" target="_none">The Chodera lab at the Memorial Sloan Kettering Cancer Center</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1101/2020.06.27.175430" class="" target="_none">SARS-CoV-2 Simulations Go Exascale to Capture Spike Opening and Reveal Cryptic Pockets Across the Proteome</a> by Maxwell I. Zimmerman, Justin R. Porter, Michael D. Ward, Sukrit Singh, Neha Vithani, Artur Meller, Upasana L. Mallimadugula, Catherine E. Kuhn, Jonathan H. Borowsky,  View ORCID ProfileRafal P. Wiewiora, Matthew F. D. Hurley, Aoife M Harbison, Carl A Fogarty, Joseph E. Coffland, Elisa Fadda, Vincent A. Voelz, John D. Chodera, Gregory R. Bowman</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="nyu-fastmri" class="dataset">
        <h4><a href="../nyu-fastmri/">NYU Langone &amp; FAIR FastMRI Dataset</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/facebookresearch/fastMRI/blob/master/fastMRI_tutorial.ipynb" class="" target="_none">FastMRI Tutorial (Jupyter Notebook)</a> by <a href="https://scholar.google.com/citations?user&#x3D;LjjLn2YAAAAJ&amp;hl&#x3D;en" target="_none">Tullie Murrell</a></a>
            <br>
            
          </li>          <li>
            <a href="https://medium.com/@khodges42/how-to-get-started-diy-neurohacking-with-the-fastmri-dataset-and-why-you-should-6a02a6bb896e" class="" target="_none">How to Get Started Working with the FastMRI Dataset (and why you should!)</a> by Kevin Hodges</a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://arxiv.org/abs/2001.02518" class="" target="_none">Advancing machine learning for MR image reconstruction with an open competition:Overview of the 2019 fastMRI challenge</a> by <a href="https://arxiv.org/search/eess?searchtype&#x3D;author&amp;query&#x3D;Knoll%2C+F" target="_none">Knoll et al (2020)</a></a>
            <br>
            
          </li>          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/29774597" class="" target="_none">Assessment of the generalization of learned image reconstruction and the potential for transfer learning.</a> by <a href="https://arxiv.org/search/eess?searchtype&#x3D;author&amp;query&#x3D;Knoll%2C+F" target="_none">Knoll et al (2019)</a></a>
            <br>
            
          </li>          <li>
            <a href="https://arxiv.org/abs/1904.01112" class="" target="_none">Deep Learning Methods for Parallel Magnetic Resonance Image Reconstruction</a> by <a href="https://arxiv.org/search/eess?searchtype&#x3D;author&amp;query&#x3D;Knoll%2C+F" target="_none">Knoll et al (2019)</a></a>
            <br>
            
          </li>          <li>
            <a href="https://arxiv.org/abs/1811.08839" class="" target="_none">fastMRI:An Open Dataset and Benchmarks for Accelerated MRI</a> by <a href="https://arxiv.org/search/cs?searchtype&#x3D;author&amp;query&#x3D;Zbontar%2C+J" target="_none">Zbontar et al (2019)</a></a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="aws-covid19-lake" class="dataset">
        <h4><a href="../aws-covid19-lake/">COVID-19 Data Lake</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://aws.amazon.com/blogs/big-data/a-public-data-lake-for-analysis-of-covid-19-data/" class="aws-link" target="_none">A public data lake for analysis of COVID-19 data</a> by <a href="https://aws.amazon.com/big-data/datalakes-and-analytics/" target="_none">AWS Data Lake Team</a></a>
            <br>
            <span class="label label-info tag service-tag">Glue</span><span class="label label-info tag service-tag">Athena</span><span class="label label-info tag service-tag">Comprehend</span><span class="label label-info tag service-tag">Kendra</span><span class="label label-info tag service-tag">QuickSight</span>
          </li>          <li>
            <a href="https://aws.amazon.com/blogs/big-data/exploring-the-public-aws-covid-19-data-lake/" class="aws-link" target="_none">Exploring the public AWS COVID-19 data lake</a> by <a href="https://aws.amazon.com/big-data/datalakes-and-analytics/" target="_none">AWS Data Lake Team</a></a>
            <br>
            <span class="label label-info tag service-tag">Athena</span><span class="label label-info tag service-tag">Glue</span><span class="label label-info tag service-tag">SageMaker</span>
          </li>          <li>
            <a href="https://github.com/aws-samples/query-data-in-s3-with-amazon-athena-and-aws-sdk-for-dotnet" class="aws-link" target="_none">How to use SQL to query data in S3 Bucket with Amazon Athena and AWS SDK for .NET</a> by <a href="https://github.com/orgs/aws-samples/teams/aws-proserve-us-west-applications" target="_none">AWS ProServe US West Applications Team</a></a>
            <br>
            <span class="label label-info tag service-tag">Athena</span><span class="label label-info tag service-tag">Glue</span><span class="label label-info tag service-tag">S3</span><span class="label label-info tag service-tag">Cloud​Formation</span>
          </li>        </ul>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://us-east-2.console.aws.amazon.com/cloudformation/home?region&#x3D;us-east-2#/stacks/create/review?templateURL&#x3D;https://covid19-lake.s3.us-east-2.amazonaws.com/cfn/CovidLakeStack.template.json&amp;stackName&#x3D;CovidLakeStack" class="" target="_none">CloudFormation template for Glue Catalog table definitions</a> by <a href="https://aws.amazon.com/big-data/datalakes-and-analytics/" target="_none">AWS Data Lake Team</a></a>
            <br>
            
          </li>          <li>
            <a href="https://go.aws/covid-19-data-lake" class="aws-link" target="_none">Explore the COVID-19 data lake public S3 bucket</a> by <a href="https://aws.amazon.com/big-data/datalakes-and-analytics/" target="_none">AWS Data Lake Team</a></a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="ilmn-dragen-1kgp" class="dataset">
        <h4><a href="../ilmn-dragen-1kgp/">1000 Genomes Phase 3 Reanalysis with DRAGEN 3.5</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/aws-quickstart/quickstart-illumina-dragen" class="" target="_none">DRAGEN Quick Start on AWS</a> by <a href="https://github.com/aws-quickstart" target="_none">AWS Quick Start Team</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://support.illumina.com/content/dam/illumina-support/documents/downloads/software/dragen/DRAGEN-3.5.7b-Customer-Release-Notes-1000000131537-00.pdf" class="" target="_none">DRAGEN 3.7.5b Release</a> by <a href="https://www.illumina.com/" target="_none">Illumina Inc.</a></a>
            <br>
            
          </li>          <li>
            <a href="https://www.illumina.com/products/by-type/informatics-products/basespace-sequence-hub.html#key-features" class="" target="_none">DRAGEN on BaseSpace Sequence Hub</a> by <a href="https://www.illumina.com/" target="_none">Illumina Inc.</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://www.biorxiv.org/content/10.1101/2020.11.13.380741v1" class="" target="_none">precisionFDA Truth Challenge V2: Calling variants from short- and long-reads in difficult-to-map regions (Preprint)</a> by Olson et al (2020)</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="ncbi-blast-databases" class="dataset">
        <h4><a href="../ncbi-blast-databases/">Basic Local Alignment Sequences Tool (BLAST) Databases</a></h4>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/ncbi/blast_plus_docs" class="" target="_none">BLAST+ Docker</a> by NCBI BLAST</a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://pubmed.ncbi.nlm.nih.gov/20003500" class="" target="_none">BLAST+: Architecture and Applications</a> by Christiam Camacho  1 , George Coulouris, Vahram Avagyan, Ning Ma, Jason Papadopoulos, Kevin Bealer, Thomas L Madden</a>
            <br>
            
          </li>          <li>
            <a href="https://pubmed.ncbi.nlm.nih.gov/9254694/" class="" target="_none">Gapped BLAST and PSI-BLAST: A New Generation of Protein Database Search Programs</a> by S F Altschul, T L Madden, A A Schäffer, J Zhang, Z Zhang, W Miller, D J Lipman</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="msd" class="dataset">
        <h4><a href="../msd/">Medical Segmentation Decathlon</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/Project-MONAI/tutorials" class="" target="_none">Pytorch-Integrated MSD Data Loader</a> by <a href="https://github.com/Project-MONAI/MONAI" target="_none">MONAI Development Team</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="www.monai.io" class="" target="_none">MONAI: Getting Started</a> by <a href="https://monai.io/start.html" target="_none">MONAI Development Team</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://arxiv.org/pdf/1902.09063.pdf" class="" target="_none">A large annotated medical image dataset for the development and evaluation of segmentation algorithms</a> by Simpson A. L., Antonelli M., Bakas S., Bilello M., Farahana K., van Ginneken B., et al</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="mimiciii" class="dataset">
        <h4><a href="../mimiciii/">MIMIC-III (‘Medical Information Mart for Intensive Care’)</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://aws.amazon.com/blogs/big-data/perform-biomedical-informatics-without-a-database-using-mimic-iii-data-and-amazon-athena/" class="aws-link" target="_none">Perform biomedical informatics without a database using MIMIC-III data and Amazon Athena</a> by James Wiggins, Alistair Johnson</a>
            <br>
            <span class="label label-info tag service-tag">Athena</span><span class="label label-info tag service-tag">Glue</span>
          </li>        </ul>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/MIT-LCP/mimic-code" class="" target="_none">MIMIC-code GitHub repository</a> by <a href="https://github.com/alistairewj" target="_none">Alistair Johnson</a></a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="ncbi-covid-19" class="dataset">
        <h4><a href="../ncbi-covid-19/">COVID-19 Genome Sequence Dataset</a></h4>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/sra/docs/sra-aws-download/" class="" target="_none">Download SRA sequence data using Amazon Web Services (AWS)</a> by NCBI SRA</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="qiime2" class="dataset">
        <h4><a href="../qiime2/">QIIME 2 User Tutorial Datasets</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://docs.qiime2.org/2018.11/install/virtual/aws/" class="" target="_none">Installing QIIME 2 using Amazon Web Services</a> by <a href="https://github.com/qiime2/docs/graphs/contributors" target="_none">The QIIME 2 Development Team</a></a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="tabula-muris" class="dataset">
        <h4><a href="../tabula-muris/">Tabula Muris</a></h4>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/30283141" class="" target="_none">Single-cell transcriptomics of 20 mouse organs creates a Tabula Muris.</a> by <a href="https://tabula-muris.ds.czbiohub.org/" target="_none">Tabula Muris Consortium (2019)</a></a>
            <br>
            
          </li>        </ul>
      </div>

    <hr/>
    <p>If you want to add a dataset or usage example to this registry, please follow the instructions on the <a href="https://github.com/awslabs/open-data-registry/">Registry of Open Data on AWS GitHub repository</a>.</p>
    <p><a href="/"><b>Home</b></a></p>
  </div>