<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    
  
    
  
    <link rel="alternate" type="application/rss+xml" href="/rss.xml" />
    
    <link rel="shortcut icon" type="image/ico" href="https://a0.awsstatic.com/libra-css/images/site/fav/favicon.ico"/>
    <link rel="apple-touch-icon" sizes="57x57" href="https://a0.awsstatic.com/libra-css/images/site/touch-icon-iphone-114-smile.png"/>
    <link rel="apple-touch-icon" sizes="72x72" href="https://a0.awsstatic.com/libra-css/images/site/touch-icon-ipad-144-smile.png"/>
    <link rel="apple-touch-icon" sizes="114x114" href="https://a0.awsstatic.com/libra-css/images/site/touch-icon-iphone-114-smile.png"/>
    <link rel="apple-touch-icon" sizes="144x144" href="https://a0.awsstatic.com/libra-css/images/site/touch-icon-ipad-144-smile.png"/>  
  
    <title>Genome Aggregation Database (gnomAD) - Registry of Open Data on AWS</title>
  
    <!-- Latest compiled and minified CSS -->
    <link rel="stylesheet" href="https://assets.opendata.aws/css/bootstrap/3.4.1/bootstrap.min.css">
  
    <!-- Our local CSS -->
    <link rel="stylesheet" href="/css/main.css">
  
    <script type="application/ld+json">
    {
      "@context" : "http://schema.org",
      "@type" : "Dataset",
      "name" : "Genome Aggregation Database (gnomAD)",
      "description" : "The Genome Aggregation Database (gnomAD) is a resource developed by an international coalition of investigators that aggregates and harmonizes both exome and genome data from a wide range of large-scale human sequencing projects. The summary data provided here are released for the benefit of the wider scientific community without restriction on use.
  The v2 data set (GRCh37) spans 125,748 exome sequences and 15,708 whole-genome sequences from unrelated individuals. The v3 data set (GRCh38) spans 71,702 genomes, selected as in v2.
  Sign up for the gnomAD mailing list <a href=\"http://broad.io/gnomad_list\">here</a>.",
      "license" : "<a href=\"https://github.com/broadinstitute/gnomad_methods/blob/master/LICENSE\">MIT</a>; <a href=\"https://gnomad.broadinstitute.org/terms\">terms of use</a>",
      "sourceOrganization" : "gnomAD Production Team at the Broad Institute"
    }
    </script>
  
  </head>
  <body>
    <!-- jQuery (necessary for Bootstrap's JavaScript plugins) -->
    <script src="https://assets.opendata.aws/js/jquery/3.5.1/jquery.min.js"></script>
    <!-- Latest compiled and minified JavaScript -->
    <script src="https://assets.opendata.aws/js/bootstrap/3.4.1/bootstrap.min.js"></script>
    <nav class="navbar navbar-default navbar-fixed-top">
      <div class="container">
        <div class="roda-header">
          <h2><a href="/" alt="Home">Registry of Open Data on AWS</a></h2>
          <a href="http://aws.amazon.com/what-is-cloud-computing">
            <img src="https://assets.opendata.aws/img/AWS-Logo_White-Color_300x180.png" alt="Powered by AWS Cloud Computing" id="aws_header_logo">
          </a>
        </div>
      </div>
    </nav>
    <div class="container" >

    <h1>Genome Aggregation Database (gnomAD)</h1>
    <p>
      

      <span class="label label-info tag link-tag">bioinformatics</span>

      <span class="label label-info tag link-tag">genetic</span>

      <span class="label label-info tag link-tag">genomic</span>

      <span class="label label-info tag link-tag">life sciences</span>

      <span class="label label-info tag link-tag">population</span>

      <span class="label label-info tag link-tag">population genetics</span>

      <span class="label label-info tag link-tag">short read sequencing</span>

      <span class="label label-info tag link-tag">whole genome sequencing</span>
    </p>
    <div class="row">
      <div class="col-md-6">
      <h3>Description</h3>
      <p>The Genome Aggregation Database (gnomAD) is a resource developed by an international coalition of investigators that aggregates and harmonizes both exome and genome data from a wide range of large-scale human sequencing projects. The summary data provided here are released for the benefit of the wider scientific community without restriction on use.
The v2 data set (GRCh37) spans 125,748 exome sequences and 15,708 whole-genome sequences from unrelated individuals. The v3 data set (GRCh38) spans 71,702 genomes, selected as in v2.
Sign up for the gnomAD mailing list <a href="http://broad.io/gnomad_list">here</a>.</p>

      <h4>Update Frequency</h4>
      <p>Data from new releases are made public as soon as they are available. New releases, including both minor and major versions, have historically been issued on the order of once per year.</p>

      <h4>License</h4>
      <p><a href="https://github.com/broadinstitute/gnomad_methods/blob/master/LICENSE">MIT</a>; <a href="https://gnomad.broadinstitute.org/terms">terms of use</a></p>

      <h4>Documentation</h4>
      <p><a href="https://gnomad.broadinstitute.org/about">https://gnomad.broadinstitute.org/about</a></p>

      <h4>Managed By</h4>
      <p>gnomAD Production Team at the Broad Institute</p>
      <p>See all datasets managed by <a href="undefined?search&#x3D;managedBy:gnomad production team at the broad institute">gnomAD Production Team at the Broad Institute</a>.</p>

      <h4>Contact</h4>
      <p><a href="mailto:gnomad@broadinstitute.org">gnomad@broadinstitute.org</a></p>

      <h4 name="usageexamples">Usage Examples</h4>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/broadinstitute/gnomad_qc" class="" target="_none">gnomAD quality control GitHub repository</a> by <a href="https://github.com/orgs/broadinstitute/teams/gnomad/members" target="_none">gnomAD Production Team</a></a>
            <br>
            
          </li>          <li>
            <a href="https://hail.is/" class="" target="_none">Hail</a> by <a href="https://hail.is/" target="_none">Hail Team</a></a>
            <br>
            
          </li>          <li>
            <a href="https://aws.amazon.com/quickstart/architecture/hail/" class="aws-link" target="_none">Hail on AWS Quick Start</a> by Amazon Web Services and PrivoIT</a>
            <br>
            
          </li>          <li>
            <a href="https://pypi.org/project/gnomad/" class="" target="_none">Hail utilities for gnomAD</a> by <a href="https://github.com/orgs/broadinstitute/teams/gnomad/members" target="_none">gnomAD Production Team</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1038/s41586-020-2287-8" class="" target="_none">A structural variation reference for medical and population genetics. Nature 581, 444–451 (2020)</a> by Collins, R. L., Brand, H., Karczewski, K. J., Zhao, X., Alföldi, J., Francioli, L. C., Khera, A. V., Lowther, C., Gauthier, L. D., Wang, H., Watts, N. A., Solomonson, M., O’Donnell-Luria, A., Baumann, A., Munshi, R., Walker, M., Whelan, C., Huang, Y., Brookings, T., ... Talkowski, M. E.</a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1038/nature19057" class="" target="_none">Analysis of protein-coding genetic variation in 60,706 humans. Nature 536, 285–291 (2016)</a> by Lek, M., Karczewski, K., Minikel, E. et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1038/s41467-019-10717-9" class="" target="_none">Characterising the loss-of-function impact of 5’ untranslated region variants in 15,708 individuals. Nature Communications 11, 2523 (2020)</a> by Whiffin, N., Karczewski, K. J., Zhang, X., Chothani, S., Smith, M. J., Gareth Evans, D., Roberts, A. M., Quaife, N. M., Schafer, S., Rackham, O., Alföldi, J., O’Donnell-Luria, A. H., Francioli, L. C., Genome Aggregation Database (gnomAD) Production Team, Genome Aggregation Database (gnomAD) Consortium, Cook, S. A., Barton, P. J. R., MacArthur, D. G., &amp; Ware, J. S.</a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1038/s41586-020-2267-z" class="" target="_none">Evaluating potential drug targets through human loss-of-function genetic variation. Nature 581, 459–464 (2020)</a> by Minikel, E. V., Karczewski, K. J., Martin, H. C., Cummings, B. B., Whiffin, N., Rhodes, D., Alföldi, J., Trembath, R. C., van Heel, D. A., Daly, M. J., Genome Aggregation Database Production Team, Genome Aggregation Database Consortium, Schreiber, S. L., &amp; MacArthur, D. G.</a>
            <br>
            
          </li>          <li>
            <a href="https://gnomad.broadinstitute.org/blog/2018-10-gnomad-v2-1/" class="" target="_none">gnomAD v2.1</a> by Laurent Francioli, Grace Tiao, Konrad Karczewski, Matthew Solomonson, Nick Watts</a>
            <br>
            
          </li>          <li>
            <a href="https://gnomad.broadinstitute.org/blog/2019-10-gnomad-v3-0/" class="" target="_none">gnomAD v3.0</a> by Laurent Francioli, Daniel MacArthur</a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1038/s41467-019-12438-5" class="" target="_none">Landscape of multi-nucleotide variants in 125,748 human exomes and 15,708 genomes. Nature Communications 11, 2539 (2020)</a> by Wang, Q., Pierce-Hoffman, E., Cummings, B. B., Karczewski, K. J., Alföldi, J., Francioli, L. C., Gauthier, L. D., Hill, A. J., O’Donnell-Luria, A. H., Genome Aggregation Database (gnomAD) Production Team, Genome Aggregation Database (gnomAD) Consortium, &amp; MacArthur, D. G.</a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1101/784157" class="" target="_none">Technical artifact drives apparent deviation from Hardy-Weinberg equilibrium at CCR5-∆32 and other variants in gnomAD. bioRxiv (p. 784157)</a> by Karczewski, K. J., Gauthier, L. D., Daly, M. J.</a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1038/s41591-020-0893-5" class="" target="_none">The effect of LRRK2 loss-of-function variants in humans. Nature Medicine (2020)</a> by Whiffin, N., Armean, I. M., Kleinman, A., Marshall, J. L., Minikel, E. V., Goodrich, J. K., Quaife, N. M., Cole, J. B., Wang, Q., Karczewski, K. J., Cummings, B. B., Francioli, L., Laricchia, K., Guan, A., Alipanahi, B., Morrison, P., Baptista, M. A. S., Merchant, K. M., Genome Aggregation Database Production Team, ... MacArthur, D. G.</a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1038/s41586-020-2308-7" class="" target="_none">The mutational constraint spectrum quantified from variation in 141,456 humans. Nature 581, 434–443 (2020)</a> by Karczewski, K. J., Francioli, L. C., Tiao, G., Cummings, B. B., Alföldi, J., Wang, Q., Collins, R. L., Laricchia, K. M., Ganna, A., Birnbaum, D. P., Gauthier, L. D., Brand, H., Solomonson, M., Watts, N. A., Rhodes, D., Singer-Berk, M., England, E. M., Seaby, E. G., Kosmicki, J. A., ... MacArthur, D. G.</a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1038/s41586-020-2329-2" class="" target="_none">Transcript expression-aware annotation improves rare variant interpretation. Nature 581, 452–458 (2020)</a> by Cummings, B. B., Karczewski, K. J., Kosmicki, J. A., Seaby, E. G., Watts, N. A., Singer-Berk, M., Mudge, J. M., Karjalainen, J., Kyle Satterstrom, F., O’Donnell-Luria, A., Poterba, T., Seed, C., Solomonson, M., Alföldi, J., The Genome Aggregation Database Production Team, The Genome Aggregation Database Consortium, Daly, M. J., &amp; MacArthur, D. G.</a>
            <br>
            
          </li>        </ul>

    </div>
    <div class="col-md-5 col-md-offset-1">
      <h3>Resources on AWS</h3>
      <ul class="resource-list">
        <li>
          <dl class="resource-list">
            <dt class="resource-description">Description</dt>
            <dd class="resource-description">gnomAD summary data aggregated from large-scale human genome and exome sequencing projects.</dd>
            <dt class="resource-type">Resource type</dt>
            <dd class="resource-type s3-bucket">S3 Bucket</dd>
            <dt class="resource-arn">Amazon Resource Name (ARN)</dt>
            <dd><code>arn:aws:s3:::gnomad-public-us-east-1</code></dd>
            <dt class="resource-region">AWS Region</dt>
            <dd><code>us-east-1</code></dd>
            <dt class="resource-region"><a href="https://aws.amazon.com/cli/">AWS CLI</a> Access (No AWS account required)</dt>
            <dd><code>aws s3 ls s3://gnomad-public-us-east-1/ --no-sign-request</code></dd>
          </dl>
        </li>
      </ul>
    </div>
  </div>

    <hr/>
    <p><a href="https://github.com/awslabs/open-data-registry/blob/main/datasets/broad-gnomad.yaml">Edit this dataset entry on GitHub</a></p>
    <p><a href="/"><b>Home</b></a></p>
  </div>
  <script src="/js/tags.js"></script>
