<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    
  
    
  
    <link rel="alternate" type="application/rss+xml" href="/rss.xml" />
    
    <link rel="shortcut icon" type="image/ico" href="https://a0.awsstatic.com/libra-css/images/site/fav/favicon.ico"/>
    <link rel="apple-touch-icon" sizes="57x57" href="https://a0.awsstatic.com/libra-css/images/site/touch-icon-iphone-114-smile.png"/>
    <link rel="apple-touch-icon" sizes="72x72" href="https://a0.awsstatic.com/libra-css/images/site/touch-icon-ipad-144-smile.png"/>
    <link rel="apple-touch-icon" sizes="114x114" href="https://a0.awsstatic.com/libra-css/images/site/touch-icon-iphone-114-smile.png"/>
    <link rel="apple-touch-icon" sizes="144x144" href="https://a0.awsstatic.com/libra-css/images/site/touch-icon-ipad-144-smile.png"/>  
  
    <title>Registry of Open Data on AWS</title>
  
    <!-- Latest compiled and minified CSS -->
    <link rel="stylesheet" href="https://assets.opendata.aws/css/bootstrap/3.4.1/bootstrap.min.css">
  
    <!-- Our local CSS -->
    <link rel="stylesheet" href="/css/main.css">
  
  
  </head>
  <body>
    <!-- jQuery (necessary for Bootstrap's JavaScript plugins) -->
    <script src="https://assets.opendata.aws/js/jquery/3.5.1/jquery.min.js"></script>
    <!-- Latest compiled and minified JavaScript -->
    <script src="https://assets.opendata.aws/js/bootstrap/3.4.1/bootstrap.min.js"></script>
    <nav class="navbar navbar-default navbar-fixed-top">
      <div class="container">
        <div class="roda-header">
          <h2><a href="/" alt="Home">Registry of Open Data on AWS</a></h2>
          <a href="http://aws.amazon.com/what-is-cloud-computing">
            <img src="https://assets.opendata.aws/img/AWS-Logo_White-Color_300x180.png" alt="Powered by AWS Cloud Computing" id="aws_header_logo">
          </a>
        </div>
      </div>
    </nav>
    <div class="container" >

    <div class="about col-md-5">
      <div class="row aboutbox">
            <h3>About</h3>
            <p>This registry exists to help people discover and share datasets that are available via AWS resources. <a href="https://opendata.aws">Learn more about sharing data on AWS</a>.</p>
    
            <p>See <a href="/tag/genomic/usage-examples">all usage examples for datasets listed in this registry</a> tagged with <strong>genomic</strong>.</p>
    
        <hr>
    
        <h4>Search datasets (currently <span id="count-matching">13</span> matching <span id="count-matching-text">datasets</span>)</h4>
        <form>
          <div class="form-group">
            <input type="text" id="search-box" class="form-control" placeholder="Search datasets" spellcheck="false" autocorrect="off">
          </div>
        </form>
        <p>You are currently viewing a subset of data tagged with <strong>genomic</strong>.</p>
    
        <hr>
    
        <h4>Add to this registry</h4>
        <p>If you want to add a dataset or example of how to use a dataset to this registry, please follow the instructions on the <a href="https://github.com/awslabs/open-data-registry/">Registry of Open Data on AWS GitHub repository</a>.</p>
    
        <p>Unless specifically stated in the applicable dataset documentation, datasets available through the Registry of Open Data on AWS are not provided and maintained by AWS. Datasets are provided and maintained by a variety of third parties under a variety of licenses. Please check dataset licenses and related documentation to determine if a dataset may be used for your application.</p>
      </div>
    </div>
    <div class="col-md-6 col-md-offset-1 datasets">
      <div class="row">
        <div id="tcga" class="dataset">
          <h3><a href="/tcga/">The Cancer Genome Atlas</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">STRIDES</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>The Cancer Genome Atlas (TCGA), a collaboration between the National Cancer Institute (NCI) and National Human Genome Research Institute (NHGRI), aims to generate comprehensive, multi-dimensional maps of the key genomic changes in major types and subtypes of cancer. TCGA has analyzed matched tumor and normal tissues from 11,000 patients, allowing for the comprehensive characterization of 33 cancer types and subtypes, including 10 rare cancers.
The dataset contains open Clinical Supplement, Biospecimen Supplement, RNA-Seq Gene Expression Quantification, miRNA-Seq Isoform Expression Quantificati...</p>
          <p><a href="/tcga/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1016/j.celrep.2018.03.046" class="" target="_none">Machine Learning Detects Pan-Cancer Ras Pathway Activation in The Cancer Genome Atlas</a> by Gregory P. Way, Francisco Sanchez-Vega, et al.</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1016/j.ccell.2018.07.001" class="" target="_none">Comprehensive Analysis of Alternative Splicing Across Tumors from 8,705 Patients</a> by André Kahles, Kjong-Van Lehmann, et al.</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1016/j.ccell.2018.03.007" class="" target="_none">Genomic and Functional Approaches to Understanding Cancer Aneuploidy</a> by Alison M. Taylor, Juliann Shih, et al.</li></a>
          </li>
          <li>
            <a href="https://firecloud.terra.bio/" class="" target="_none">Broad Institute FireCloud</a> by <a href="https://www.broadinstitute.org/" target="_none">The Broad Institute of MIT &amp; Harvard</a></li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1016/j.cels.2019.06.006" class="" target="_none">&quot;Before and After: A Comparison of Legacy and Harmonized TCGA Data at the Genomic Data
Commons&quot;
</a> by Galen F. Gao, Joel S. Parker, et al.</li></a>
          </li>
        </ul>
        <p><a href="/tcga/#usageexamples">See 29 usage examples &rarr;</a></p>
        </div>
        <div id="target" class="dataset">
          <h3><a href="/target/">Therapeutically Applicable Research to Generate Effective Treatments (TARGET)</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">STRIDES</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>Therapeutically Applicable Research to Generate Effective Treatments (TARGET) is the collaborative effort of a large, diverse consortium of extramural and NCI investigators. The goal of the effort is to accelerate molecular discoveries that drive the initiation and progression of hard-to-treat childhood cancers and facilitate rapid translation of those findings into the clinic.
TARGET projects provide comprehensive molecular characterization to determine the genetic changes that drive the initiation and progression of childhood cancers.The dataset contains open Clinical Supplement, Biospecimen...</p>
          <p><a href="/target/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://portal.gdc.cancer.gov/projects/" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></li></a>
          </li>
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term&#x3D;29227476" class="" target="_none">The molecular landscape of pediatric acute myeloid leukemia reveals recurrent structural
alterations and age-specific mutational interactions
</a> by Bolouri H, Farrar JE, Triche T Jr, et al.</li></a>
          </li>
          <li>
            <a href="https://cgc-accounts.sbgenomics.com/" class="" target="_none">Cancer Genomics Cloud</a> by <a href="https://docs.sevenbridges.com/docs/target-data" target="_none">Seven Bridges</a></li></a>
          </li>
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term&#x3D;29315296" class="" target="_none">Genomics in childhood acute myeloid leukemia comes of age</a> by Brunner AM, Graubert TA.</li></a>
          </li>
          <li>
            <a href="https://firecloud.terra.bio/" class="" target="_none">Broad Institute FireCloud</a> by <a href="https://www.broadinstitute.org/" target="_none">The Broad Institute of MIT &amp; Harvard</a></li></a>
          </li>
        </ul>
        <p><a href="/target/#usageexamples">See 24 usage examples &rarr;</a></p>
        </div>
        <div id="kids-first" class="dataset">
          <h3><a href="/kids-first/">Gabriella Miller Kids First Pediatric Research Program (Kids First)</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">Homo sapiens</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">pediatric</span><span class="label label-info tag link-tag">STRIDES</span><span class="label label-info tag link-tag">structural birth defect</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>The NIH Common Fund&#39;s Gabriella Miller Kids First Pediatric Research Program’s (“Kids First”) vision is to “alleviate suffering from childhood cancer and structural birth defects by fostering collaborative research to uncover the etiology of these diseases and by supporting data sharing within the pediatric research community.” The program continues to generate and share whole genome sequence data from thousands of children affected by these conditions, ranging from rare pediatric cancers, such as osteosarcoma, to more prevalent diagnoses, such as congenital heart defects. In 2018, Kids Fi...</p>
          <p><a href="/kids-first/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/31397880" class="" target="_none">MAGEL2-Related Disorders: A study and case series.</a> by Jameson Patak, James Gilfert, et al.</li></a>
          </li>
          <li>
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/31562520" class="" target="_none">Germline microsatellite genotypes differentiate children with medulloblastoma.</a> by Samuel Rivero-Hinojosa, Nicholas Kinney, et al.</li></a>
          </li>
          <li>
            <a href="https://pubmed.ncbi.nlm.nih.gov/30098192/" class="" target="_none">Genome-Wide Association Study Identifies a Susceptibility Locus for Comitant Esotropia and Suggests a Parent-of-Origin Effect</a> by Sherin Shaaban, Sarah MacKinnon et al.</li></a>
          </li>
          <li>
            <a href="https://portal.kidsfirstdrc.org" class="" target="_none">Kids First DRC Portal</a> by <a href="https://kidsfirstdrc.org" target="_none">Kids First DRC</a></li></a>
          </li>
          <li>
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/31848685" class="" target="_none">Whole genome sequencing of orofacial cleft trios from the Gabriella Miller Kids First Pediatric Research Consortium identifies a new locus on chromosome 21.</a> by Nandita Mukhopadhyay, Madison Bishop, et al.</li></a>
          </li>
        </ul>
        <p><a href="/kids-first/#usageexamples">See 19 usage examples &rarr;</a></p>
        </div>
        <div id="broad-gnomad" class="dataset">
          <h3><a href="/broad-gnomad/">Genome Aggregation Database (gnomAD)</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">population</span><span class="label label-info tag link-tag">population genetics</span><span class="label label-info tag link-tag">short read sequencing</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>The Genome Aggregation Database (gnomAD) is a resource developed by an international coalition of investigators that aggregates and harmonizes both exome and genome data from a wide range of large-scale human sequencing projects. The summary data provided here are released for the benefit of the wider scientific community without restriction on use.
The v2 data set (GRCh37) spans 125,748 exome sequences and 15,708 whole-genome sequences from unrelated individuals. The v3 data set (GRCh38) spans 71,702 genomes, selected as in v2.
Sign up for the gnomAD mailing list <a href="http://broad.io/gnomad_list">here</a>.</p>
          <p><a href="/broad-gnomad/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://gnomad.broadinstitute.org/blog/2018-10-gnomad-v2-1/" class="" target="_none">gnomAD v2.1</a> by Laurent Francioli, Grace Tiao, Konrad Karczewski, Matthew Solomonson, Nick Watts</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1038/s41586-020-2267-z" class="" target="_none">Evaluating potential drug targets through human loss-of-function genetic variation. Nature 581, 459–464 (2020)</a> by Minikel, E. V., Karczewski, K. J., Martin, H. C., Cummings, B. B., Whiffin, N., Rhodes, D., Alföldi, J., Trembath, R. C., van Heel, D. A., Daly, M. J., Genome Aggregation Database Production Team, Genome Aggregation Database Consortium, Schreiber, S. L., &amp; MacArthur, D. G.</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1038/nature19057" class="" target="_none">Analysis of protein-coding genetic variation in 60,706 humans. Nature 536, 285–291 (2016)</a> by Lek, M., Karczewski, K., Minikel, E. et al.</li></a>
          </li>
          <li>
            <a href="https://aws.amazon.com/quickstart/architecture/hail/" class="aws-link" target="_none">Hail on AWS Quick Start</a> by Amazon Web Services and PrivoIT</li></a>
          </li>
          <li>
            <a href="https://pypi.org/project/gnomad/" class="" target="_none">Hail utilities for gnomAD</a> by <a href="https://github.com/orgs/broadinstitute/teams/gnomad/members" target="_none">gnomAD Production Team</a></li></a>
          </li>
        </ul>
        <p><a href="/broad-gnomad/#usageexamples">See 15 usage examples &rarr;</a></p>
        </div>
        <div id="ccle" class="dataset">
          <h3><a href="/ccle/">Cancer Cell Line Encyclopedia (CCLE)</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">Homo sapiens</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">STRIDES</span><span class="label label-info tag link-tag">transcriptomics</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>The Cancer Cell Line Encyclopedia (CCLE) project is an effort to conduct a detailed genetic
characterization of a large panel of human cancer cell lines. The CCLE provides public access to
genomic data, visualization and analysis for over 1100 cancer cell lines. This dataset contains
RNA-Seq Aligned Reads, WXS Aligned Reads, and WGS Aligned Reads data.</p>
          <p><a href="/ccle/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1038/s41586-019-1186-3" class="" target="_none">Next-generation characterization of the Cancer Cell Line Encyclopedia</a> by Ghandi, M., Huang F. et al.</li></a>
          </li>
          <li>
            <a href="https://portal.gdc.cancer.gov/legacy-archive/" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></li></a>
          </li>
          <li>
            <a href="https://cgc-accounts.sbgenomics.com/" class="" target="_none">Cancer Genomics Cloud</a> by <a href="https://docs.sevenbridges.com/docs/ccle-data" target="_none">Seven Bridges</a></li></a>
          </li>
          <li>
            <a href="https://console.cloud.google.com/bigquery?p&#x3D;isb-cgc&amp;d&#x3D;CCLE_bioclin_v0&amp;page&#x3D;dataset" class="" target="_none">ISB CGC BigQuery tables</a> by <a href="https://isb-cancer-genomics-cloud.readthedocs.io/en/latest/sections/data/CCLE_top.html
" target="_none">Institute for Systems Biology</a></li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1038/s41591-019-0404-8" class="" target="_none">The landscape of cancer cell line metabolism</a> by Li, H. et al.</li></a>
          </li>
        </ul>
        <p><a href="/ccle/#usageexamples">See 8 usage examples &rarr;</a></p>
        </div>
        <div id="cptac-2" class="dataset">
          <h3><a href="/cptac-2/">Clinical Proteomic Tumor Analysis Consortium 2 (CPTAC-2)</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">STRIDES</span><span class="label label-info tag link-tag">transcriptomics</span></p>
          <p>The Clinical Proteomic Tumor Analysis Consortium (CPTAC) is a national effort to accelerate the
understanding of the molecular basis of cancer through the application of large-scale proteome and
genome analysis, or proteogenomics. CPTAC-2 is the Phase II of the CPTAC Initiative (2011-2016).
Datasets contain open RNA-Seq Gene Expression Quantification, miRNA-Seq Isoform Expression
Quantification, and miRNA Expression Quantification data.</p>
          <p><a href="/cptac-2/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1038/sdata.2015.22" class="" target="_none">Proteomic analysis of colon and rectal carcinoma using standard and customized databases
</a> by Slebos RJ, Wang X, Wang X, Zhang B, Tabb DL, Liebler DC</li></a>
          </li>
          <li>
            <a href="https://pdc.esacinc.com/pdc/browse" class="" target="_none">Proteomic Data Commons</a> by <a href="https://pdc.esacinc.com/pdc" target="_none">National Cancer Institute</a></li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1016/j.cell.2019.03.030" class="" target="_none">Proteogenomic Analysis of Human Colon Cancer Reveals New Therapeutic Opportunities</a> by Suhas Vasaikar, Chen Huang, Xiaojing Wang. Vladislav A. Petyuk, Sara R. Savage, Bo Wen,
Yongchao Dou, Yun Zhang, Zhiao Shi, Osama A. Arshad, Marina A. Gritsenko, Lisa J. Zimmerman,
Jason E. McDermott, Therese R. Clauss, Ronald J. Moore, Rui Zhao, Matthew E. Monroe, Yi-Ting
Wang, Matthew C. Chambers, Robbert J.C. Slebos, Ken S. Lau, Qianxing Mo, Li Ding, Matthew
Ellis, Mathangi Thiagarajan, Christopher R. Kinsinger, Henry Rodriguez, Richard D. Smith,
Karin D. Rodland, Daniel C. Liebler, Tao Liu, Bing Zhang, Clinical Proteomic Tumor Analysis
Consortium
</li></a>
          </li>
          <li>
            <a href="https://cgc-accounts.sbgenomics.com/" class="" target="_none">Cancer Genomics Cloud</a> by <a href="http://www.cancergenomicscloud.org/" target="_none">Seven Bridges</a></li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1016/j.cell.2016.05.069" class="" target="_none">Integrated Proteogenomic Characterization of Human High-Grade Serous Ovarian Cancer</a> by Hui Zhang, Tao Liu, Zhen Zhang, Samuel H. Payne, Bai Zhang, Jason E. McDermott, Jian-Ying
Zhou, Vladislav A. Petyuk, Li Chen, Debjit Ray, Shisheng Sun, Feng Yang, Lijun Chen, Jing
Wang, Punit Shah, Seong Won Cha, Paul Aiyetan, Sunghee Woo, Yuan Tian, Marina A. Gritsenko,
Therese R. Clauss, Caitlin Choi, Matthew E. Monroe, Stefani Thomas, Song Nie, Chaochao Wu,
Ronald J. Moore, Kun-Hsing Yu, David L. Tabb, David Fenyö, Vineet Bafna, Yue Wang, Henry
Rodriguez, Emily S. Boja, Tara Hiltke, Robert C. Rivers, Lori Sokoll, Heng Zhu, Ie-Ming
Shih, Leslie Cope, Akhilesh Pandey, Bing Zhang, Michael P. Snyder, Douglas A. Levine,
Richard D. Smith, Daniel W. Chan, Karin D. Rodland, the CPTAC Investigators
</li></a>
          </li>
        </ul>
        <p><a href="/cptac-2/#usageexamples">See 7 usage examples &rarr;</a></p>
        </div>
        <div id="icgc" class="dataset">
          <h3><a href="/icgc/">ICGC on AWS</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span></p>
          <p>The International Cancer Genome Consortium (ICGC) coordinates projects with the common aim of accelerating research into the causes and control of cancer. The PanCancer Analysis of Whole Genomes (PCAWG) study is an international collaboration to identify common patterns of mutation in whole genomes from ICGC. More than 2,400 consistently analyzed genomes corresponding to over 1,100 unique ICGC donors are now freely available on Amazon S3 to credentialed researchers subject to ICGC data sharing policies.</p>
          <p><a href="/icgc/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.nature.com/articles/s41586-020-1943-3" class="" target="_none">The repertoire of mutational signatures in human cancer</a> by <a href="https://www.ncbi.nlm.nih.gov/pubmed?cmd&#x3D;search&amp;term&#x3D;Ludmil%20B.%20Alexandrov" target="_none">Alexandrov et al (2020)</a></li></a>
          </li>
          <li>
            <a href="https://www.nature.com/articles/s41586-020-1965-x" class="" target="_none">Analyses of non-coding somatic drivers in 2,693 cancer whole genomes</a> by <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term&#x3D;Rheinbay%20E%5BAuthor%5D&amp;cauthor&#x3D;true&amp;cauthor_uid&#x3D;28658208" target="_none">Rheinbay et al (2020)</a></li></a>
          </li>
          <li>
            <a href="https://github.com/ICGC-TCGA-PanCancer/cli/wiki" class="" target="_none">AWS Pancancer Workflow Launcher</a> by <a href="https://github.com/solomonshorser" target="_none">Solomon Shorser</a></li></a>
          </li>
          <li>
            <a href="https://www.nature.com/articles/s41586-019-1913-9" class="" target="_none">Patterns of somatic structural variation in human cancer genomes</a> by <a href="https://www.ncbi.nlm.nih.gov/pubmed?cmd&#x3D;search&amp;term&#x3D;Yilong%20Li" target="_none">Li et al. (2020)</a></li></a>
          </li>
          <li>
            <a href="https://www.nature.com/articles/s41586-019-1907-7" class="" target="_none">The evolutionary history of 2,658 cancers</a> by <a href="https://www.ncbi.nlm.nih.gov/pubmed?cmd&#x3D;search&amp;term&#x3D;Moritz%20Gerstung" target="_none">Gerstung et al (2020)</a></li></a>
          </li>
        </ul>
        <p><a href="/icgc/#usageexamples">See 7 usage examples &rarr;</a></p>
        </div>
        <div id="cptac-3" class="dataset">
          <h3><a href="/cptac-3/">Clinical Proteomic Tumor Analysis Consortium 3 (CPTAC-3)</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">STRIDES</span><span class="label label-info tag link-tag">transcriptomics</span></p>
          <p>The Clinical Proteomic Tumor Analysis Consortium (CPTAC) is a national effort to accelerate the
understanding of the molecular basis of cancer through the application of large-scale proteome and
genome analysis, or proteogenomics. CPTAC-3 is the Phase III of the CPTAC Initiative. The dataset
contains open RNA-Seq Gene Expression Quantification data.</p>
          <p><a href="/cptac-3/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1016/j.cell.2019.10.007" class="" target="_none">Integrated Proteogenomic Characterization of Clear Cell Renal Cell Carcinoma</a> by Clark DJ, Dhanasekaran SM, Petralia F, Pan J, Song X, Hu Y, da Veiga Leprevost F, Reva B,
Lih TM, Chang HY, Ma W, Huang C, Ricketts CJ, Chen L1, Krek A, Li Y, Rykunov D, Li QK, Chen
LS, Ozbek U, Vasaikar S, Wu Y, Yoo S, Chowdhury S, Wyczalkowski MA, Ji J, Schnaubelt M, Kong
A, Sethuraman S, Avtonomov DM, Ao M, Colaprico A, Cao S, Cho KC, Kalayci S, Ma S, Liu W,
Ruggles K, Calinawan A, Gümüş ZH, Geizler D, Kawaler E, Teo GC, Wen B, Zhang Y, Keegan S, Li
K, Chen F, Edwards N, Pierorazio PM, Chen XS, Pavlovich CP, Hakimi AA, Brominski G, Hsieh
JJ, Antczak A, Omelchenko T, Lubinski J, Wiznerowicz M, Linehan WM, Kinsinger CR,
Thiagarajan M, Boja ES, Mesri M, Hiltke T, Robles AI, Rodriguez H, Qian J, Fenyö D, Zhang B,
Ding L, Schadt E, Chinnaiyan AM, Zhang Z, Omenn GS, Cieslik M, Chan DW, Nesvizhskii AI, Wang
P, Zhang H; Clinical Proteomic Tumor Analysis Consortium
</li></a>
          </li>
          <li>
            <a href="https://cgc-accounts.sbgenomics.com/" class="" target="_none">Cancer Genomics Cloud</a> by <a href="http://www.cancergenomicscloud.org/" target="_none">Seven Bridges</a></li></a>
          </li>
          <li>
            <a href="https://proteomics.cancer.gov/data-portal" class="" target="_none">CPTAC Data Portal</a> by <a href="https://proteomics.cancer.gov/" target="_none">National Cancer Institute</a></li></a>
          </li>
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/29718670" class="" target="_none">Evaluation of NCI-7 Cell Line Panel as a Reference Material for Clinical Proteomics</a> by Clark DJ, Hu Y, Bocik W, Chen L, Schnaubelt M, Roberts R, Shah P, Whiteley G, Zhang H
</li></a>
          </li>
          <li>
            <a href="https://pdc.esacinc.com/pdc/browse" class="" target="_none">Proteomic Data Commons</a> by <a href="https://pdc.esacinc.com/pdc" target="_none">National Cancer Institute</a></li></a>
          </li>
        </ul>
        <p><a href="/cptac-3/#usageexamples">See 6 usage examples &rarr;</a></p>
        </div>
        <div id="mmrf-commpass" class="dataset">
          <h3><a href="/mmrf-commpass/">CoMMpass from the Multiple Myeloma Research Foundation</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">STRIDES</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>The Relating Clinical Outcomes in Multiple Myeloma to Personal Assessment of Genetic Profile study is the Multiple Myeloma Research Foundation (MMRF)’s landmark personalized medicine initiative.  CoMMpass is a
longitudinal observation study of around 1000 newly diagnosed myeloma patients receiving various
standard approved treatments. The MMRF’s vision is to track the treatment and results for each
CoMMpass patient so that someday the information can be used to guide decisions for newly
diagnosed patients. CoMMpass checked on patients every 6 months for 8 years, collecting tissue
samples, gene...</p>
          <p><a href="/mmrf-commpass/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1182/blood.V122.21.532.532" class="" target="_none">&quot;Interim Analysis Of The MMRF CoMMpass Trial: a Longitudinal Study In Multiple Myeloma
Relating Clinical Outcomes To Genomic and Immunophenotypic Profiles&quot;
</a> by Keats JJ, Craig DW, Liang W, Venkata Y, Kurdoglu A, Aldrich J, Auclair D, Allen K, Harrison
B, Jewell S, Kidd PG, Correll M, Jagannath S, Siegel DS, Vij R, Orloff G, Zimmerman TM, MMRF
CoMMpass Network, Capone W, Carpten J, Lonial S.
</li></a>
          </li>
          <li>
            <a href="https://portal.gdc.cancer.gov/projects/MMRF-COMMPASS" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1182/blood.V124.21.722.722" class="" target="_none">&quot;Interim Analysis of the Mmrf Commpass Trial: Identification of Novel Rearrangements
Potentially Associated with Disease Initiation and Progression&quot;
</a> by Sagar Lonial, MD, Venkata D Yellapantula, Winnie Liang, PhD, Ahmet Kurdoglu, BS, Jessica
Aldrich, MSc, Christophe M. Legendre, MD, Kristi Stephenson, Jonathan Adkins, Jackie
McDonald, Adrienne Helland, Megan Russell, Austin Christofferson, Lori Cuyugan, Dan Rohrer,
Alex Blanski, Meghan Hodges, Mmrf CoMMpass Network, Mary Derome, Daniel Auclair, PhD, Pamela
G. Kidd, MD, Scott Jewell, PhD, David Craig, PhD, John Carpten, PhD, Jonathan J. Keats, PhD
</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1182/blood.V126.23.722.722" class="" target="_none">&quot;Identification of Initiating Trunk Mutations and Distinct Molecular Subtypes: An Interim
Analysis of the Mmrf Commpass Study&quot;
</a> by Jonathan J Keats, PhD, Gil Speyer, Legendre Christophe, Christofferson Austin, Kristi
Stephenson, BS, Ahmet Kurdoglu, Megan Russell, Aldrich Jessica, Cuyugan Lori, Jonathan
Adkins, Jackie McDonald, Adrienne Helland, Alex Blanski, Meghan Hodges, Dan Rohrer, Sundar
Jagannath, MD, David Siegel, MD PhD, Ravi Vij, MD MBA, Gregory Orloff, MD, Todd Zimmerman,
MD, Ruben Niesvizky, MD, Darla Liles, MD, Joseph W. Fay, Jeffrey L. Wolf, MD PhD, Robert M.
Rifkin, Norma C Gutierrez, The MMRF CoMMpass Network, Jen Toups, Mary Derome, MS, Winnie
Liang, PhD, Seunchan Kim, Daniel Auclair, PhD, Pamela G. Kidd, MD, Scott Jewell, PhD, John
David Carpten, PhD, Sagar Lonial, MD
</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1182/blood.V128.22.194.194" class="" target="_none">&quot;Molecular Predictors of Outcome and Drug Response in Multiple Myeloma: An Interim Analysis
of the Mmrf CoMMpass Study&quot;
</a> by Jonathan J Keats, PhD, Gil Speyer, Austin Christofferson, Christophe Legendre, PhD, Jessica
Aldrich, Megan Russell, Lori Cuyugan, Jonathan Adkins, Alex Blanski, Meghan Hodges, Dan
Rohrer, Sundar Jagannath, MD, Ravi Vij, MD, Gregory Orloff, MD, Todd Zimmerman, MD, Ruben
Niesvizky, MD, Darla Liles, MD, Joseph W. Fay, Jeffrey L. Wolf, MD, Robert M Rifkin, Norma C
Gutierrez, MD PhD, Mmrf CoMMpass Network, Jennifer Yesil, MS, Mary Derome, MS, Seungchan
Kim, PhD, Winnie Liang, PhD, Pamela G. Kidd, MD, Scott Jewell, PhD, John David Carpten, PhD,
Daniel Auclair, PhD, Sagar Lonial, MD FACP
</li></a>
          </li>
        </ul>
        <p><a href="/mmrf-commpass/#usageexamples">See 5 usage examples &rarr;</a></p>
        </div>
        <div id="ilmn-dragen-1kgp" class="dataset">
          <h3><a href="/ilmn-dragen-1kgp/">1000 Genomes Phase 3 Reanalysis with DRAGEN 3.5</a></h3>
          <p><span class="label label-info tag link-tag">bam</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">health</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">vcf</span></p>
          <p>This dataset contains alignment files and short nucleotide, copy number, and structural variant call files from the newly revised (n=3202) 1000 Genomes Project Phase 3 using Illumina DRAGEN v3.5.7b</p>
          <p><a href="/ilmn-dragen-1kgp/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/aws-quickstart/quickstart-illumina-dragen" class="" target="_none">DRAGEN Quick Start on AWS</a> by <a href="https://github.com/aws-quickstart" target="_none">AWS Quick Start Team</a></li></a>
          </li>
          <li>
            <a href="https://support.illumina.com/content/dam/illumina-support/documents/downloads/software/dragen/DRAGEN-3.5.7b-Customer-Release-Notes-1000000131537-00.pdf" class="" target="_none">DRAGEN 3.7.5b Release</a> by <a href="https://www.illumina.com/" target="_none">Illumina Inc.</a></li></a>
          </li>
          <li>
            <a href="https://www.biorxiv.org/content/10.1101/2020.11.13.380741v1" class="" target="_none">precisionFDA Truth Challenge V2: Calling variants from short- and long-reads in difficult-to-map regions (Preprint)</a> by Olson et al (2020)</li></a>
          </li>
          <li>
            <a href="https://www.illumina.com/products/by-type/informatics-products/basespace-sequence-hub.html#key-features" class="" target="_none">DRAGEN on BaseSpace Sequence Hub</a> by <a href="https://www.illumina.com/" target="_none">Illumina Inc.</a></li></a>
          </li>
        </ul>
        <p><a href="/ilmn-dragen-1kgp/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="3kricegenome" class="dataset">
          <h3><a href="/3kricegenome/">3000 Rice Genomes Project</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">food security</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span></p>
          <p>The 3000 Rice Genome Project is an international effort to sequence the genomes of 3,024 rice varieties from 89 countries.</p>
          <p><a href="/3kricegenome/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://academic.oup.com/gigascience/article/8/5/giz028/5492234" class="" target="_none">Rice Galaxy: an open resource for plant science</a> by Juanillas V et al (2019)</li></a>
          </li>
          <li>
            <a href="https://github.com/InternationalRiceResearchInstitute/RiceGalaxy" class="" target="_none">RiceGalaxy</a> by <a href="https://www.irri.org/" target="_none">International Rice Research Institute</a></li></a>
          </li>
          <li>
            <a href="https://www.nature.com/articles/s41467-019-08451-3" class="" target="_none">Tracking the origin of two genetic components associated with transposable element bursts in domesticated rice</a> by Chen J et al (2019)</li></a>
          </li>
          <li>
            <a href="https://genome.cshlp.org/content/29/5/870.short" class="" target="_none">Structural variants in 3000 rice genomes</a> by Fuentes RR et al (2019)</li></a>
          </li>
        </ul>
        <p><a href="/3kricegenome/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="encode-project" class="dataset">
          <h3><a href="/encode-project/">Encyclopedia of DNA Elements (ENCODE)</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">deep learning</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">machine learning</span></p>
          <p>The Encyclopedia of DNA Elements (ENCODE) Consortium is an international collaboration of
research groups funded by the National Human Genome Research Institute (NHGRI). The goal
of ENCODE is to build a comprehensive parts list of functional elements in the human genome,
including elements that act at the protein and RNA levels, and regulatory elements that
control cells and circumstances in which a gene is active. ENCODE investigators employ a
variety of assays and methods to identify functional elements. The discovery and annotation
of gene elements is accomplished primarily by sequencing a ...</p>
          <p><a href="/encode-project/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/ENCODE-DCC/encode-data-usage-examples/blob/master/ingest_encode_data_tile_db_with_s3_backend.ipynb" class="" target="_none">Ingesting ENCODE data into TileDB with S3 backend</a> by <a href="https://github.com/ottojolanki" target="_none">Otto Jolanki</a></li></a>
          </li>
          <li>
            <a href="https://github.com/ENCODE-DCC/encode-data-usage-examples/blob/master/mount_s3_bucket_and_run_jupyter_on_ec2.ipynb" class="" target="_none">Exploring ENCODE data from EC2 with Jupyter notebook</a> by <a href="https://github.com/keenangraham" target="_none">Keenan Graham</a></li></a>
          </li>
          <li>
            <a href="https://academic.oup.com/nar/article/48/D1/D882/5622708#190992715" class="" target="_none">New developments on the Encyclopedia of DNA Elements (ENCODE) data portal</a> by <a href="http://orcid.org/0000-0001-6948-2042" target="_none">Luo et al 2020</a></li></a>
          </li>
          <li>
            <a href="https://github.com/ENCODE-DCC/encode-data-usage-examples/blob/master/ctcf_chip_seq_cross_cell_type_correlation.ipynb" class="" target="_none">ENCODE CTCF ChIP-seq data correlation across different cell types</a> by <a href="https://github.com/p-sud" target="_none">Paul Sud</a></li></a>
          </li>
        </ul>
        <p><a href="/encode-project/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="giab" class="dataset">
          <h3><a href="/giab/">Genome in a Bottle on AWS</a></h3>
          <p><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">reference index</span><span class="label label-info tag link-tag">vcf</span></p>
          <p>Several reference genomes to enable translation of whole human genome sequencing to clinical practice. On 11/12/2020 these data were updated to reflect the most <a href="https://www.nist.gov/programs-projects/genome-bottle">up to date GIAB release</a>.</p>
          <p><a href="/giab/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/ga4gh/benchmarking-tools" class="" target="_none">GA4GH Benchmarking Tools</a> by GA4GH Benchmarking Team</li></a>
          </li>
          <li>
            <a href="https://github.com/genome-in-a-bottle/about_GIAB" class="" target="_none">The Genome in a Bottle Github Project</a> by Genome In A Bottle Consortium</li></a>
          </li>
          <li>
            <a href="https://www.nature.com/articles/sdata201625" class="" target="_none">Extensive sequencing of seven human genomes to characterize benchmark reference materials</a> by Zook J et al (2016)</li></a>
          </li>
          <li>
            <a href="https://www.nature.com/articles/s41597-019-0098-2" class="" target="_none">High-coverage, long-read sequencing of Han Chinese trio reference samples</a> by Wang Y et al (2019)</li></a>
          </li>
        </ul>
        <p><a href="/giab/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="ncbi-sra" class="dataset">
          <h3><a href="/ncbi-sra/">NIH NCBI Sequence Read Archive (SRA) on AWS</a></h3>
          <p><span class="label label-info tag link-tag">bam</span><span class="label label-info tag link-tag">fastq</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">STRIDES</span><span class="label label-info tag link-tag">transcriptomics</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>The Sequence Read Archive (SRA), produced by the <a href="https://www.ncbi.nlm.nih.gov/">National Center for Biotechnology Information (NCBI)</a> at the <a href="http://nlm.nih.gov/">National Library of Medicine (NLM)</a> at the <a href="http://www.nih.gov/">National Institutes of Health (NIH)</a>, stores raw DNA sequencing data and alignment information from high-throughput sequencing platforms. The SRA provides open access to these biological sequence data to support the research community&#39;s efforts to enhance reproducibility and make new discoveries by comparing data s...</p>
          <p><a href="/ncbi-sra/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/sra/docs/sra-aws-download/" class="" target="_none">Access SRA data using Amazon Web Services (AWS)</a> by <a href="https://www.ncbi.nlm.nih.gov/sra/" target="_none">NCBI SRA</a></li></a>
          </li>
          <li>
            <a href="https://github.com/ncbi/sra-tools" class="" target="_none">SRA Toolkit</a> by <a href="https://www.ncbi.nlm.nih.gov/sra/" target="_none">NCBI SRA</a></li></a>
          </li>
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/sra/docs/sra-cloud/" class="" target="_none">SRA in the Cloud</a> by <a href="https://www.ncbi.nlm.nih.gov/sra/" target="_none">NCBI SRA</a></li></a>
          </li>
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3013647/" class="" target="_none">The Sequence Read Archive</a> by <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term&#x3D;Leinonen%20R%5BAuthor%5D&amp;cauthor&#x3D;true&amp;cauthor_uid&#x3D;21062823" target="_none">Leinonen et al (2011)</a></li></a>
          </li>
        </ul>
        <p><a href="/ncbi-sra/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="refgenie" class="dataset">
          <h3><a href="/refgenie/">Refgenie reference genome assets</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">infrastructure</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">single-cell transcriptomics</span><span class="label label-info tag link-tag">transcriptomics</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>Pre-built refgenie reference genome data assets used for aligning and analyzing DNA sequence data.</p>
          <p><a href="/refgenie/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="http://peppro.databio.org" class="" target="_none">Pipeline for PRO-seq data analysis</a> by <a href="http://databio.org" target="_none">Jason Smith and Nathan Sheffield</a></li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1093/gigascience/giz149" class="" target="_none">Refgenie: a reference genome resource manager</a> by <a href="http://www.databio.org" target="_none">Michał Stolarczyk, Vincent P Reuter, Jason P Smith, Neal E Magee, Nathan C Sheffield</a></li></a>
          </li>
          <li>
            <a href="http://pepatac.databio.org" class="" target="_none">Pipeline for ATAC-seq data analysis</a> by <a href="http://databio.org" target="_none">Jason Smith and Nathan Sheffield</a></li></a>
          </li>
          <li>
            <a href="http://refgenie.databio.org/en/latest/tutorial/" class="" target="_none">Basic Refgenie tutorial</a> by <a href="http://www.databio.org" target="_none">Nathan Sheffield</a></li></a>
          </li>
        </ul>
        <p><a href="/refgenie/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="broad-pan-ukb" class="dataset">
          <h3><a href="/broad-pan-ukb/">UK Biobank Pan-Ancestry Summary Statistics</a></h3>
          <p><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genome wide association study</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">population genetics</span></p>
          <p>A multi-ancestry analysis of 7,221 phenotypes using a generalized mixed model association testing framework, spanning 16,119 genome-wide association studies. We provide standard meta-analysis across all populations and with a leave-one-population-out approach for each trait. The data are provided in tsv format (per phenotype) and Hail MatrixTable (all phenotypes and variants). Metadata is provided in phenotype and variant manifests.</p>
          <p><a href="/broad-pan-ukb/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://aws.amazon.com/quickstart/architecture/hail/" class="aws-link" target="_none">Hail on AWS Quick Start</a> by Amazon Web Services and PrivoIT</li></a>
          </li>
          <li>
            <a href="https://hail.is" class="" target="_none">Hail</a> by <a href="https://hail.is/" target="_none">Hail Team</a></li></a>
          </li>
          <li>
            <a href="https://hail.is/docs/0.2/tutorials-landing.html" class="" target="_none">Hail Tutorials</a> by <a href="https://hail.is/" target="_none">Hail Team</a></li></a>
          </li>
          <li>
            <a href="https://pan.ukbb.broadinstitute.org/" class="" target="_none">Pan-ancestry genetic analysis of the UK Biobank</a> by Pan UKBB Team</li></a>
          </li>
        </ul>
        <p><a href="/broad-pan-ukb/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="ncbi-blast-databases" class="dataset">
          <h3><a href="/ncbi-blast-databases/">Basic Local Alignment Sequences Tool (BLAST) Databases</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">health</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">protein</span><span class="label label-info tag link-tag">reference index</span><span class="label label-info tag link-tag">transcriptomics</span></p>
          <p>A centralized repository of pre-formatted BLAST databases created by the National Center for Biotechnology Information (NCBI).</p>
          <p><a href="/ncbi-blast-databases/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://pubmed.ncbi.nlm.nih.gov/20003500" class="" target="_none">BLAST+: Architecture and Applications</a> by Christiam Camacho  1 , George Coulouris, Vahram Avagyan, Ning Ma, Jason Papadopoulos, Kevin Bealer, Thomas L Madden</li></a>
          </li>
          <li>
            <a href="https://pubmed.ncbi.nlm.nih.gov/9254694/" class="" target="_none">Gapped BLAST and PSI-BLAST: A New Generation of Protein Database Search Programs</a> by S F Altschul, T L Madden, A A Schäffer, J Zhang, Z Zhang, W Miller, D J Lipman</li></a>
          </li>
          <li>
            <a href="https://github.com/ncbi/blast_plus_docs" class="" target="_none">BLAST+ Docker</a> by NCBI BLAST</li></a>
          </li>
        </ul>
        <p><a href="/ncbi-blast-databases/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="beataml" class="dataset">
          <h3><a href="/beataml/">Beat Acute Myeloid Leukemia (AML) 1.0</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">Homo sapiens</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">STRIDES</span></p>
          <p>Beat AML 1.0 is a collaborative research program involving 11 academic medical centers who worked
collectively to better understand  drugs and drug combinations that should be prioritized for
further development within clinical and/or molecular subsets of acute myeloid leukemia (AML)
patients. Beat AML 1.0 provides the largest-to-date dataset on primary acute myeloid leukemia
samples offering genomic, clinical, and drug response.This dataset contains open Clinical Supplement and RNA-Seq Gene Expression Quantification data.This dataset also contains controlled Whole Exome Sequencing (WXS) and R...</p>
          <p><a href="/beataml/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://portal.gdc.cancer.gov/projects/" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></li></a>
          </li>
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/30333627" class="" target="_none">Functional Genomic Landscape of Acute Myeloid Leukemia</a> by Jeffrey W. Tyner, Cristina E. Tognon, Dan Bottomly et al.</li></a>
          </li>
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/30651561" class="" target="_none">Clinical resistance to crenolanib in acute myeloid leukemia due to diverse molecular
mechanisms
</a> by Zhang H, Savage S, Schultz AR, Bottomly D, White L, Segerdell E, et al.</li></a>
          </li>
        </ul>
        <p><a href="/beataml/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="ctsp-dlbcl" class="dataset">
          <h3><a href="/ctsp-dlbcl/">Clinical Trial Sequencing Project - Diffuse Large B-Cell Lymphoma</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">STRIDES</span><span class="label label-info tag link-tag">transcriptomics</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>The goal of the project is to identify recurrent genetic alterations (mutations, deletions,
amplifications, rearrangements) and/or gene expression signatures. National Cancer Institute (NCI)
utilized whole genome sequencing and/or whole exome sequencing in conjunction with transcriptome
sequencing. The samples were processed and submitted for genomic characterization using pipelines
and procedures established within The Cancer Genome Analysis (TCGA) project.</p>
          <p><a href="/ctsp-dlbcl/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed?cmd&#x3D;DetailsSearch&amp;term&#x3D;29925955[PMID]" class="" target="_none">A multiprotein supercomplex controlling oncogenic signalling in lymphoma</a> by Phelan JD, Young RM, Webster DE, Roulland S, Wright GW, Kasbekar M, Shaffer AL 3rd,
Ceribelli M, Wang JQ, Schmitz R, Nakagawa M, Bachy E, Huang DW, Ji Y, Chen L, Yang Y, Zhao
H, Yu X, Xu W, Palisoc MM, Valadez RR, Davies-Hill T, Wilson WH, Chan WC, Jaffe ES, Gascoyne
RD, Campo E, Rosenwald A, Ott G, Delabie J, Rimsza LM, Rodriguez FJ, Estephan F, Holdhoff M,
Kruhlak MJ, Hewitt SM, Thomas CJ, Pittaluga S, Oellerich T, Staudt LM
</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1056/NEJMoa1801445" class="" target="_none">Genetics and Pathogenesis of Diffuse Large B Cell Lymphoma</a> by Roland Schmitz, Ph.D., George W. Wright, Ph.D., Da Wei Huang, M.D., Calvin A.  Johnson,
Ph.D., James D. Phelan, Ph.D., James Q. Wang, Ph.D., Sandrine Roulland, Ph.D., Monica
Kasbekar, Ph.D., Ryan M. Young, Ph.D., Arthur L. Shaffer, Ph.D., Daniel J. Hodson, M.D.,
Ph.D., Wenming Xiao, Ph.D., et al.
</li></a>
          </li>
          <li>
            <a href="https://portal.gdc.cancer.gov/projects/CTSP-DLBCL1" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></li></a>
          </li>
        </ul>
        <p><a href="/ctsp-dlbcl/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="fm-ad" class="dataset">
          <h3><a href="/fm-ad/">Foundation Medicine Adult Cancer Clinical Dataset (FM-AD)</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genomic</span></p>
          <p>The Foundation Medicine Adult Cancer Clinical Dataset (FM-AD) is a study conducted by Foundation
Medicine Inc (FMI). Genomic profiling data for approximately 18,000 adult patients with a diverse
array of cancers was generated using FoundationeOne, FMI&#39;s commercially available, comprehensive
genomic profiling assay. This dataset contains open Clinical and Biospecimen data.</p>
          <p><a href="/fm-ad/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed?cmd&#x3D;DetailsSearch&amp;term&#x3D;28235761[PMID]" class="" target="_none">High-Throughput Genomic Profiling of Adult Solid Tumors Reveals Novel Insights into Cancer
Pathogenesis
</a> by Ryan J. Hartmaier, Lee A. Albacker, Juliann Chmielecki, Mark Bailey, Jie He, Michael E.
Goldberg, Shakti Ramkissoon, James Suh, Julia A. Elvin, Samuel Chiacchia, Garrett M.
Frampton, Jeffrey S. Ross, Vincent Miller, Philip J. Stephens and Doron Lipson
</li></a>
          </li>
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed?cmd&#x3D;DetailsSearch&amp;term&#x3D;22981675[PMID]" class="" target="_none">Targeted next-generation sequencing of advanced prostate cancer identifies potential
therapeutic targets and disease heterogeneity.
</a> by Beltran H, Yelensky R, Frampton GM, Park K, Downing SR, MacDonald TY, Jarosz M, Lipson D,
Tagawa ST, Nanus DM, Stephens PJ, Mosquera JM, Cronin MT, Rubin MA
</li></a>
          </li>
          <li>
            <a href="https://portal.gdc.cancer.gov/projects/FM-AD" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></li></a>
          </li>
        </ul>
        <p><a href="/fm-ad/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="human-microbiome-project" class="dataset">
          <h3><a href="/human-microbiome-project/">The Human Microbiome Project</a></h3>
          <p><span class="label label-info tag link-tag">amino acid</span><span class="label label-info tag link-tag">fasta</span><span class="label label-info tag link-tag">fastq</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">metagenomics</span><span class="label label-info tag link-tag">microbiome</span></p>
          <p>The NIH-funded Human Microbiome Project (HMP) is a collaborative effort of over 300 scientists from more than 80 organizations to comprehensively characterize the microbial communities inhabiting the human body and elucidate their role in human health and disease. To accomplish this task, microbial community samples were isolated from a cohort of 300 healthy adult human subjects at 18 specific sites within five regions of the body (oral cavity, airways, urogenital track, skin, and gut). Targeted sequencing of the 16S bacterial marker gene and/or whole metagenome shotgun sequencing was performe...</p>
          <p><a href="/human-microbiome-project/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.nature.com/articles/nature06244" class="" target="_none">The Human Microbiome Project</a> by Peter J. Turnbaugh, Ruth E. Ley, Micah Hamady, Claire M. Fraser-Liggett, Rob Knight &amp; Jeffrey I. Gordon</li></a>
          </li>
          <li>
            <a href="https://www.sciencedirect.com/science/article/pii/S2452231718300289" class="" target="_none">New microbe genomic variants in patients fecal community following surgical disruption of the upper human gastrointestinal tract</a> by Ranjit Kumar, Jayleen Grams, Daniel I. Chu, David K.Crossman, Richard Stahl, Peter Eipers, et al</li></a>
          </li>
          <li>
            <a href="https://www.nature.com/articles/nature23889" class="" target="_none">Strains, functions and dynamics in the expanded Human Microbiome Project</a> by Jason Lloyd-Price, Anup Mahurkar, Gholamali Rahnavard, Jonathan Crabtree, Joshua Orvis, A. Brantley Hall, et al.</li></a>
          </li>
        </ul>
        <p><a href="/human-microbiome-project/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="hail-vep-pipeline" class="dataset">
          <h3><a href="/hail-vep-pipeline/">Variant Effect Predictor (VEP) and the Loss-Of-Function Transcript Effect Estimator (LOFTEE) Plugin</a></h3>
          <p><span class="label label-info tag link-tag">genome wide association study</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">loftee</span><span class="label label-info tag link-tag">vep</span></p>
          <p>VEP determines the effect of genetic variants (SNPs, insertions, deletions, CNVs or structural variants) on genes, transcripts, and protein sequence, as well as regulatory regions. The European Bioinformatics Institute produces the VEP tool/db and releases updates every 1 - 6 months. The latest release contains 267 genomes from 232 species containing 5567663 protein coding genes. This dataset hosts the last 5 releases for human, rat, and zebrafish. Also, it hosts the required reference files for the Loss-Of-Function Transcript Effect Estimator (LOFTEE) plugin as it is commonly used with VEP.</p>
          <p><a href="/hail-vep-pipeline/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/konradjk/loftee" class="" target="_none">Loss-Of-Function Transcript Effect Estimator (LOFTEE)</a> by <a href="https://github.com/konradjk" target="_none">Konrad Karczewski</a></li></a>
          </li>
          <li>
            <a href="https://hail.is/" class="" target="_none">Hail</a> by <a href="https://www.nealelab.is/" target="_none">Neale Lab</a></li></a>
          </li>
          <li>
            <a href="http://uswest.ensembl.org/info/docs/tools/vep/index.html" class="" target="_none">Variant Effect Predictor (VEP)</a> by <a href="http://uswest.ensembl.org/info/docs/tools/vep/index.html" target="_none">Ensembl</a></li></a>
          </li>
        </ul>
        <p><a href="/hail-vep-pipeline/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="broad-references" class="dataset">
          <h3><a href="/broad-references/">Broad Genome References</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">Homo sapiens</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">reference index</span></p>
          <p>Broad maintained human genome reference builds hg19/hg38 and decoy references.</p>
          <p><a href="/broad-references/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://aws.amazon.com/blogs/storage/using-amazon-fsx-for-lustre-for-genomics-workflows-on-aws/" class="aws-link" target="_none">Using Amazon FSx for Lustre for Genomics Workflows on AWS</a> by <a href="https://www.linkedin.com/in/lee-pang-a039a26/" target="_none">W. Lee Pang</a></li></a>
          </li>
          <li>
            <a href="https://docs.opendata.aws/genomics-workflows/orchestration/cromwell/cromwell-examples/#real-world-example-haplotypecaller" class="aws-link" target="_none">Genomics Workflows on AWS - Cromwell on AWS</a> by <a href="https://www.linkedin.com/in/lee-pang-a039a26/" target="_none">W. Lee Pang</a></li></a>
          </li>
        </ul>
        <p><a href="/broad-references/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="cgci" class="dataset">
          <h3><a href="/cgci/">Cancer Genome Characterization Initiatives - Burkitt Lymphoma, HIV+ Cervical Cancer</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">STRIDES</span><span class="label label-info tag link-tag">transcriptomics</span></p>
          <p>The Cancer Genome Characterization Initiatives (CGCI) program supports cutting-edge genomics research of adult and pediatric cancers. CGCI investigators develop and apply advanced sequencing methods that examine genomes, exomes, and transcriptomes within various types of tumors. The program includes Burkitt Lymphoma Genome Sequencing Project (BLGSP) project and HIV+ Tumor Molecular Characterization Project - Cervical Cancer (HTMCP-CC) project.
The dataset contains open Clinical Supplement, Biospecimen Supplement, RNA-Seq Gene Expression Quantification, miRNA-Seq Isoform Expression Quantificati...</p>
          <p><a href="/cgci/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1182/blood-2018-09-871418" class="" target="_none">Genome-wide discovery of somatic coding and noncoding mutations in pediatric endemic and
sporadic Burkitt lymphoma
</a> by Grande B. M., Gerhard D. S., Jiang A., Griner N. B., Abramson J. S., Alexander T. B., et al.
</li></a>
          </li>
          <li>
            <a href="https://portal.gdc.cancer.gov/projects/CGCI-BLGSP" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></li></a>
          </li>
        </ul>
        <p><a href="/cgci/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="jhu-indexes" class="dataset">
          <h3><a href="/jhu-indexes/">Cloud Indexes for Genomic Analyses</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">mapping</span><span class="label label-info tag link-tag">medicine</span><span class="label label-info tag link-tag">reference index</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>Genomic tools use reference databases as indexes to operate quickly and efficiently, analogous to how web search engines use indexes for fast querying. Here, we aggregate genomic, pan-genomic and metagenomic indexes for analysis of sequencing data.</p>
          <p><a href="/jhu-indexes/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.biorxiv.org/content/10.1101/2020.03.03.975219v4.full.pdf" class="" target="_none">Reducing reference bias using multiple population reference genomes</a> by Chen et al (2020)</li></a>
          </li>
          <li>
            <a href="https://benlangmead.github.io/aws-indexes/tutorial" class="" target="_none">Table of contents for tutorials for constituent tools</a> by <a href="https://benlangmead.github.io/aws-indexes" target="_none">Ben Langmead</a></li></a>
          </li>
        </ul>
        <p><a href="/jhu-indexes/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="nciccr-dlbcl" class="dataset">
          <h3><a href="/nciccr-dlbcl/">National Cancer Institute Center for Cancer Research - Diffuse Large B Cell Lymphoma (DLBCL) Genomics and Expression</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genomic</span></p>
          <p>The study describes integrative analysis of genetic lesions in 574 diffuse large B cell lymphomas
(DLBCL) involving exome and transcriptome sequencing, array-based DNA copy number analysis and
targeted amplicon resequencing. The dataset contains open RNA-Seq Gene Expression Quantification
data.</p>
          <p><a href="/nciccr-dlbcl/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://portal.gdc.cancer.gov/projects/NCICCR-DLBCL" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1056/NEJMoa1801445" class="" target="_none">Genetics and Pathogenesis of Diffuse Large B Cell Lymphoma</a> by Roland Schmitz, Ph.D., George W. Wright, Ph.D., Da Wei Huang, M.D., et al.</li></a>
          </li>
        </ul>
        <p><a href="/nciccr-dlbcl/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="ohsu-cnl" class="dataset">
          <h3><a href="/ohsu-cnl/">Oregon Health &amp; Science University Chronic Neutrophilic Leukemia Dataset</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span></p>
          <p>The OHSU-CNL study offers the whole exome and RNA-sequencing on a cohort of 100 cases with rare
hematologic malignancies such as Chronic neutrophilic leukemia (CNL), atypical chronic myeloid
leukemia (aCML), and unclassified myelodysplastic syndrome/myeloproliferative neoplasms
(MDS/MPN-U). This dataset contains open RNA-Seq Gene Expression Quantification data.</p>
          <p><a href="/ohsu-cnl/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/31366621" class="" target="_none">Genomic landscape of neutrophilic leukemias of ambiguous diagnosis</a> by Zhang H, Wilmot B, Bottomly D et al.</li></a>
          </li>
          <li>
            <a href="https://portal.gdc.cancer.gov/projects/" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></li></a>
          </li>
        </ul>
        <p><a href="/ohsu-cnl/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="organoid-pancreatic" class="dataset">
          <h3><a href="/organoid-pancreatic/">Pancreatic Cancer Organoid Profiling</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">STRIDES</span><span class="label label-info tag link-tag">transcriptomics</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>This study generated a collection of patient-derived pancreatic normal and cancer organoids and it was sequenced using Whole Genome Sequencing (WGS), Whole Exome Sequencing (WXS) and RNA-Seq as well as matched tumor and normal tissue if available. The study provides a valuable resource for pancreatic cancer researchers.
The dataset contains open RNA-Seq Gene Expression Quantification data and controlled WGS/WXS/RNA-Seq Aligned Reads, WXS Annotated Somatic Mutation, WXS Raw Somatic Mutation, and RNA-Seq Splice Junction Quantification.</p>
          <p><a href="/organoid-pancreatic/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://portal.gdc.cancer.gov/projects/ORGANOID-PANCREATIC" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></li></a>
          </li>
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed?cmd&#x3D;DetailsSearch&amp;term&#x3D;29853643[PMID]" class="" target="_none">Organoid Profiling Identifies Common Responders to Chemotherapy in Pancreatic Cancer</a> by Tiriac H, Belleau P, Engle DD, Plenker D, Deschênes A, Somerville TD, et al.</li></a>
          </li>
        </ul>
        <p><a href="/organoid-pancreatic/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="serratus-lovelywater" class="dataset">
          <h3><a href="/serratus-lovelywater/">Serratus: Ultra-deep Search for Novel Viruses - Versioned Data Release</a></h3>
          <p><span class="label label-info tag link-tag">bam</span><span class="label label-info tag link-tag">COVID-19</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">MERS</span><span class="label label-info tag link-tag">SARS</span><span class="label label-info tag link-tag">SARS-CoV-2</span><span class="label label-info tag link-tag">virus</span></p>
          <p>Serratus is a collaborative open science project for ultra-rapid discovery of known and unknown coronaviruses in response to the COVID-19 pandemic through re-analysis of publicly available genomic data. Our resulting vertebrate viral alignment data is explorable via the <a href="https://serratus.io/explorer">Serratus Explorer</a> and directly accessible on Amazon S3.</p>
          <p><a href="/serratus-lovelywater/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.pnas.org/content/118/3/e2019907118" class="" target="_none">Diversification of mammalian deltaviruses by host shifting</a> by Bergner L.M., Orton R.J., et al (2021)</li></a>
          </li>
          <li>
            <a href="https://serratus.io/explorer" class="" target="_none">Serratus Explorer</a> by Serratus Team</li></a>
          </li>
          <li>
            <a href="https://www.biorxiv.org/content/10.1101/2020.07.28.224584v2" class="" target="_none">coronaSPAdes. From biosynthetic gene clusters to RNA viral assemblies</a> by Meleshko D., Hajirasouliha I., and Korobeynikov A. (2021)</li></a>
          </li>
          <li>
            <a href="https://github.com/serratus-bio/tantalus" class="" target="_none">Tantalus: An R Package for exploration of Serratus data</a> by Serratus Team</li></a>
          </li>
          <li>
            <a href="https://www.biorxiv.org/content/10.1101/2020.08.07.241729v1" class="" target="_none">Petabase-scale sequence alignment catalyses viral discovery</a> by Edgar R., Taylor J., Altman T., Barbera P., Meleshko D., Lin V., et al (2020)</li></a>
          </li>
        </ul>
        <p><a href="/serratus-lovelywater/#usageexamples">See 5 usage examples &rarr;</a></p>
        </div>
        <div id="1000-genomes" class="dataset">
          <h3><a href="/1000-genomes/">1000 Genomes</a></h3>
          <p><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span></p>
          <p>The 1000 Genomes Project is an international collaboration which has established the most detailed catalogue of human genetic variation, including SNPs, structural variants, and their haplotype context. The final phase of the project sequenced more than 2500 individuals from 26 different populations around the world and produced an integrated set of phased haplotypes with more than 80 million variants for these individuals.</p>
          <p><a href="/1000-genomes/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://aws.amazon.com/blogs/big-data/exploratory-data-analysis-of-genomic-datasets-using-adam-and-mango-with-apache-spark-on-amazon-emr/" class="aws-link" target="_none">Exploratory data analysis of genomic datasets using ADAM and Mango with Apache Spark on Amazon EMR</a> by <a href="https://research.eecs.berkeley.edu/~akmorrow/" target="_none">Alyssa Marrow</a></li></a>
          </li>
        </ul>
        <p><a href="/1000-genomes/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="aws-igenomes" class="dataset">
          <h3><a href="/aws-igenomes/">AWS iGenomes</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">Caenorhabditis elegans</span><span class="label label-info tag link-tag">Danio rerio</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">Homo sapiens</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">Mus musculus</span><span class="label label-info tag link-tag">Rattus norvegicus</span><span class="label label-info tag link-tag">reference index</span></p>
          <p>Common reference genomes hosted on AWS S3. Can be used when aligning and analysing raw DNA sequencing data.</p>
          <p><a href="/aws-igenomes/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="http://nf-co.re/" class="" target="_none">nf-core analysis pipelines</a> by <a href="http://phil.ewels.co.uk/" target="_none">Phil Ewels</a></li></a>
          </li>
        </ul>
        <p><a href="/aws-igenomes/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="ncbi-covid-19" class="dataset">
          <h3><a href="/ncbi-covid-19/">COVID-19 Genome Sequence Dataset</a></h3>
          <p><span class="label label-info tag link-tag">bam</span><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">coronavirus</span><span class="label label-info tag link-tag">COVID-19</span><span class="label label-info tag link-tag">cram</span><span class="label label-info tag link-tag">fastq</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">health</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">MERS</span><span class="label label-info tag link-tag">SARS</span><span class="label label-info tag link-tag">STRIDES</span><span class="label label-info tag link-tag">transcriptomics</span><span class="label label-info tag link-tag">virus</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>A centralized sequence repository for all records containing sequence associated with the novel corona virus (SARS-CoV-2) submitted to the National Center for Biotechnology Information (NCBI) Sequence Read Archive (SRA). Included are both the original sequences submitted by the principal investigator as well as SRA-processed sequences that require the SRA Toolkit for analysis. Additionally, submitter provided metadata included in associated BioSample and BioProject records is available alongside NCBI calculated data, such k-mer based taxonomy analysis results, contiguous assemblies (contigs) a...</p>
          <p><a href="/ncbi-covid-19/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/sra/docs/sra-aws-download/" class="" target="_none">Download SRA sequence data using Amazon Web Services (AWS)</a> by NCBI SRA</li></a>
          </li>
        </ul>
        <p><a href="/ncbi-covid-19/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="gatk-test-data" class="dataset">
          <h3><a href="/gatk-test-data/">GATK Test Data</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span></p>
          <p>The GATK test data resource bundle is a collection of files for resequencing human genomic data with the
Broad Institute&#39;s <a href="https://software.broadinstitute.org/gatk/">Genome Analysis Toolkit (GATK)</a>.</p>
          <p><a href="/gatk-test-data/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://docs.opendata.aws/genomics-workflows/orchestration/cromwell/cromwell-examples/#real-world-example-haplotypecaller" class="aws-link" target="_none">Genomics Workflows on AWS - Cromwell on AWS</a> by <a href="https://www.linkedin.com/in/lee-pang-a039a26/" target="_none">W. Lee Pang</a></li></a>
          </li>
        </ul>
        <p><a href="/gatk-test-data/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="hcmi-cmdc" class="dataset">
          <h3><a href="/hcmi-cmdc/">Human Cancer Models Initiative (HCMI) Cancer Model Development Center</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">STRIDES</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>The Human Cancer Models Initiative (HCMI) is an international consortium that is generating novel,
next-generation, tumor-derived culture models annotated with genomic and clinical data.
HCMI-developed models and related data are available as a community resource. The NCI is
contributing to the initiative by supporting four Cancer Model Development Centers (CMDCs).  CMDCs
are tasked with producing next-generation cancer models from clinical samples. The cancer models
include tumor types that are rare, originate from patients from underrepresented populations, lack
precision therapy, or lack ca...</p>
          <p><a href="/hcmi-cmdc/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://portal.gdc.cancer.gov/projects/HCMI-CMDC" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></li></a>
          </li>
        </ul>
        <p><a href="/hcmi-cmdc/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="hpgp-data" class="dataset">
          <h3><a href="/hpgp-data/">Human PanGenomics Project</a></h3>
          <p><span class="label label-info tag link-tag">cram</span><span class="label label-info tag link-tag">fast5</span><span class="label label-info tag link-tag">fastq</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span></p>
          <p>This dataset includes sequencing data, assemblies, and analyses for the offspring of ten parent-offspring trios.</p>
          <p><a href="/hpgp-data/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.nature.com/articles/s41587-020-0503-6" class="" target="_none">Nanopore sequencing and the Shasta toolkit enable efficient de novo assembly of eleven human genomes</a> by Shafin et al (2020)</li></a>
          </li>
        </ul>
        <p><a href="/hpgp-data/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="ont-open-data" class="dataset">
          <h3><a href="/ont-open-data/">Oxford Nanopore Technologies Benchmark Datasets</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">fast5</span><span class="label label-info tag link-tag">fastq</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">Homo sapiens</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>The ont-open-data registry provides reference sequencing data from Oxford Nanopore Technologies to support, 1) Exploration of the characteristics of nanopore sequence data. 2) Assessment and reproduction of performance benchmarks 3) Development of tools and methods. The data deposited showcases DNA sequences from a representative subset of sequencing chemistries. The datasets correspond to publicly-available reference samples (e.g. GM24385 as reference human). Raw data are provided with metadata and scripts to describe sample and data provenance.</p>
          <p><a href="/ont-open-data/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://nanoporetech.github.io/ont-open-datasets/tutorials" class="" target="_none">ONT Dataset Tutorials</a> by <a href="https://labs.epi2me.io/" target="_none">EPI2MELabs</a></li></a>
          </li>
        </ul>
        <p><a href="/ont-open-data/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="qiime2" class="dataset">
          <h3><a href="/qiime2/">QIIME 2 User Tutorial Datasets</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">denoising</span><span class="label label-info tag link-tag">ecosystems</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">health</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">microbiome</span><span class="label label-info tag link-tag">statistics</span></p>
          <p>QIIME 2 is a powerful, extensible, and decentralized microbiome analysis package with a focus on data and analysis transparency. QIIME 2 enables researchers to start an analysis with raw DNA sequence data and finish with publication-quality figures and statistical results. This dataset contains the user docs (and related datasets) for QIIME 2.</p>
          <p><a href="/qiime2/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://docs.qiime2.org/2018.11/install/virtual/aws/" class="" target="_none">Installing QIIME 2 using Amazon Web Services</a> by <a href="https://github.com/qiime2/docs/graphs/contributors" target="_none">The QIIME 2 Development Team</a></li></a>
          </li>
        </ul>
        <p><a href="/qiime2/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="tabula-muris" class="dataset">
          <h3><a href="/tabula-muris/">Tabula Muris</a></h3>
          <p><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">encyclopedic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">health</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">medicine</span></p>
          <p>Tabula Muris is a compendium of single cell transcriptomic data from the model organism <em>Mus musculus</em> comprising more than 100,000 cells from 20 organs and tissues. These data represent a new resource for cell biology, reveal gene expression in poorly characterized cell populations, and allow for direct and controlled comparison of gene expression in cell types shared between tissues, such as T-lymphocytes and endothelial cells from different anatomical locations. Two distinct technical approaches were used for most organs: one approach, microfluidic droplet-based 3’-end counting, enabled the s...</p>
          <p><a href="/tabula-muris/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/30283141" class="" target="_none">Single-cell transcriptomics of 20 mouse organs creates a Tabula Muris.</a> by <a href="https://tabula-muris.ds.czbiohub.org/" target="_none">Tabula Muris Consortium (2019)</a></li></a>
          </li>
        </ul>
        <p><a href="/tabula-muris/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="ihart" class="dataset">
          <h3><a href="/ihart/">iHART Whole Genome Sequencing Data Set</a></h3>
          <p><span class="label label-info tag link-tag">autism spectrum disorder</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>iHART is the <a href="http://www.thehartwellfoundation.org/">Hartwell Foundation</a>’s Autism Research and Technology Initiative. This release contains whole genome data from over 1000 families with 2 or more children with autism, of which biomaterials were provided by the Autism Genetic Resource Exchange (<a href="http://research.agre.org/">AGRE</a>).</p>
          <p><a href="/ihart/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.cell.com/cell/fulltext/S0092-8674(19)30780-9" class="" target="_none">Inherited and De Novo Genetic Risk for Autism Impacts Shared Networks</a> by Ruzzo et al. (2020)</li></a>
          </li>
        </ul>
        <p><a href="/ihart/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="genomeark" class="dataset">
          <h3><a href="/genomeark/">Genome Ark</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span></p>
          <p>The Genome Ark hosts genomic information for the Vertebrate Genomes Project (VGP) and other related projects. The VGP is an international collaboration that aims to generate complete and near error-free reference genomes for all extant vertebrate species. These genomes will be used to address fundamental questions in biology and disease, to identify species most genetically at risk for extinction, and to preserve genetic information of life.</p>
          <p><a href="/genomeark/">Details &rarr;</a></p>
        </div>
        <div id="nanopore" class="dataset">
          <h3><a href="/nanopore/">Nanopore Reference Human Genome</a></h3>
          <p><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span></p>
          <p>This dataset includes the sequencing and assembly of a reference standard human genome (GM12878) using the MinION nanopore sequencing instrument with the R9.4 1D chemistry.</p>
          <p><a href="/nanopore/">Details &rarr;</a></p>
        </div>
        <div id="tabula-muris-senis" class="dataset">
          <h3><a href="/tabula-muris-senis/">Tabula Muris Senis</a></h3>
          <p><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">encyclopedic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">health</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">medicine</span><span class="label label-info tag link-tag">single-cell transcriptomics</span></p>
          <p>Tabula Muris Senis is a comprehensive compendium of single cell transcriptomic data from the model organism <em>Mus musculus</em> comprising more than 500,000 cells from 18 organs and tissues across the mouse lifespan. We discovered cell-specific changes occurring across multiple cell types and organs, as well as age related changes in the cellular composition of different organs. Using single-cell transcriptomic data we were able to assess cell type specific manifestations of different hallmarks of aging, such as senescence, changes in the activity of metabolic pathways, depletion of stem-cell populat...</p>
          <p><a href="/tabula-muris-senis/">Details &rarr;</a></p>
        </div>
        <div id="gmsdata" class="dataset">
          <h3><a href="/gmsdata/">The Genome Modeling System</a></h3>
          <p><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span></p>
          <p>The Genome Institute at Washington University has developed a high-throughput, fault-tolerant analysis information management system called the Genome Modeling System (GMS), capable of executing complex, interdependent, and automated genome analysis pipelines at a massive scale. The GMS framework provides detailed tracking of samples and data coupled with reliable and repeatable analysis pipelines. GMS includes a full system image with software and services, expandable from one workstation to a large compute cluster.</p>
          <p><a href="/gmsdata/">Details &rarr;</a></p>
        </div>
        <div id="ubc-sunflower-genome" class="dataset">
          <h3><a href="/ubc-sunflower-genome/">University of British Columbia Sunflower Genome Dataset</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">biodiversity</span><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">food security</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>This dataset captures Sunflower&#39;s genetic diversity originating
from thousands of wild, cultivated, and landrace sunflower
individuals distributed across North America.The data consists of raw sequences and associated botanical metadata,
aligned sequences (to three different reference genomes), and sets of
SNPs computed across several cohorts.</p>
          <p><a href="/ubc-sunflower-genome/">Details &rarr;</a></p>
        </div>
        <div id="binding-db" class="dataset">
          <h3><a href="/binding-db/">Binding DB</a></h3>
          <p><span class="label label-info tag link-tag">biotech blueprint</span><span class="label label-info tag link-tag">chemistry</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">molecule</span><span class="label label-info tag link-tag">parquet</span></p>
          <p>This a parquet representation of The Binding Database&#39;s <a href="https://www.bindingdb.org/bind/chemsearch/marvin/SDFdownload.jsp?all_download=yes">Full BindingDB Database Dump</a> that you can query straight from Athena in under 60 seconds (no Oracle database required). The Binding Database projects aims to make experimental data on the noncovalent association of molecules in solution searchable via the WWW. The initial focus is on biomolecular systems, but data on host-guest and supramolecular systems are also important and being included over time. It is expected that the enhanced access to data provided by this resource will facilitate drug-discovery, the design of self-assemblin...</p>
          <p><a href="/binding-db/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://aws.amazon.com/blogs/startups/a-data-lake-as-code-featuring-chembl-and-opentargets/" class="aws-link" target="_none">Data Lake as Code, Featuring ChEMBL and Open Targets</a> by Paul Underwood</li></a>
          </li>
          <li>
            <a href="https://github.com/aws-samples/data-lake-as-code/blob/roda/docs/roda_install.md" class="aws-link" target="_none">Data Lake as Code Deployment Guide</a> by AWS Biotech Blueprints Team</li></a>
          </li>
        </ul>
        <p><a href="/binding-db/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="chembl" class="dataset">
          <h3><a href="/chembl/">ChEMBL</a></h3>
          <p><span class="label label-info tag link-tag">biotech blueprint</span><span class="label label-info tag link-tag">chemistry</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">molecule</span><span class="label label-info tag link-tag">parquet</span></p>
          <p>ChEMBL is a manually curated database of bioactive molecules with drug-like properties. It brings together chemical, bioactivity and genomic data to aid the translation of genomic information into effective new drugs. This representation of ChEMBL is stored in Parquet format and most easily utilized through Amazon Athena. Follow the documentation for install instructions (&lt; 2 minute install). New ChEMBL releases occur sporadically; the most up to date information on ChEMBL releases can be found <a href="https://chembl.gitbook.io/chembl-interface-documentation/downloads">here</a>.</p>
          <p><a href="/chembl/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/aws-samples/data-lake-as-code/blob/roda/docs/roda_install.md" class="aws-link" target="_none">Data Lake as Code Deployment Guide</a> by AWS Biotech Blueprints Team</li></a>
          </li>
          <li>
            <a href="https://aws.amazon.com/blogs/startups/a-data-lake-as-code-featuring-chembl-and-opentargets/" class="aws-link" target="_none">Data Lake as Code, Featuring ChEMBL and Open Targets</a> by Paul Underwood</li></a>
          </li>
        </ul>
        <p><a href="/chembl/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="clinvar" class="dataset">
          <h3><a href="/clinvar/">ClinVar</a></h3>
          <p><span class="label label-info tag link-tag">biotech blueprint</span><span class="label label-info tag link-tag">chemistry</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">parquet</span></p>
          <p>ClinVar is a freely accessible, public archive of reports of the relationships among human variations and phenotypes, with supporting evidence. ClinVar thus facilitates access to and communication about the relationships asserted between human variation and observed health status, and the history of that interpretation. ClinVar processes submissions reporting variants found in patient samples, assertions made regarding their clinical significance, information about the submitter, and other supporting data. The alleles described in submissions are mapped to reference sequences, and reported acc...</p>
          <p><a href="/clinvar/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://aws.amazon.com/blogs/startups/a-data-lake-as-code-featuring-chembl-and-opentargets/" class="aws-link" target="_none">Data Lake as Code, Featuring ChEMBL and Open Targets</a> by Paul Underwood</li></a>
          </li>
          <li>
            <a href="https://github.com/aws-samples/data-lake-as-code/blob/roda/docs/roda_install.md" class="aws-link" target="_none">Data Lake as Code Deployment Guide</a> by AWS Biotech Blueprints Team</li></a>
          </li>
        </ul>
        <p><a href="/clinvar/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="opentargets" class="dataset">
          <h3><a href="/opentargets/">Open Targets</a></h3>
          <p><span class="label label-info tag link-tag">biotech blueprint</span><span class="label label-info tag link-tag">chemistry</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">molecule</span><span class="label label-info tag link-tag">parquet</span></p>
          <p>This a Parquet representation of the Open Targets Platform&#39;s <a href="https://www.targetvalidation.org/downloads/data">2020-06 export</a>. The Open Targets Platform integrates evidence from genetics, genomics, transcriptomics, drugs, animal models and scientific literature to score and rank target-disease associations for drug target identification. The Open Targets Platform (<a href="https://www.targetvalidation.org">https://www.targetvalidation.org</a>) is a freely available resource for the integration of genetics, genomics, and chemical data to aid systematic drug target identification and prioritisation.</p>
          <p><a href="/opentargets/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/aws-samples/data-lake-as-code/blob/roda/docs/roda_install.md" class="aws-link" target="_none">Data Lake as Code Deployment Guide</a> by AWS Biotech Blueprints Team</li></a>
          </li>
          <li>
            <a href="https://aws.amazon.com/blogs/startups/a-data-lake-as-code-featuring-chembl-and-opentargets/" class="aws-link" target="_none">Data Lake as Code, Featuring ChEMBL and Open Targets</a> by Paul Underwood</li></a>
          </li>
        </ul>
        <p><a href="/opentargets/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
      </div>
    </div>


    <hr/>
  </div>  </body>
  <script src="/js/index.js"></script>
</html>
