<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    
  
    
  
    <link rel="alternate" type="application/rss+xml" href="/rss.xml" />
    
    <link rel="shortcut icon" type="image/ico" href="https://a0.awsstatic.com/libra-css/images/site/fav/favicon.ico"/>
    <link rel="apple-touch-icon" sizes="57x57" href="https://a0.awsstatic.com/libra-css/images/site/touch-icon-iphone-114-smile.png"/>
    <link rel="apple-touch-icon" sizes="72x72" href="https://a0.awsstatic.com/libra-css/images/site/touch-icon-ipad-144-smile.png"/>
    <link rel="apple-touch-icon" sizes="114x114" href="https://a0.awsstatic.com/libra-css/images/site/touch-icon-iphone-114-smile.png"/>
    <link rel="apple-touch-icon" sizes="144x144" href="https://a0.awsstatic.com/libra-css/images/site/touch-icon-ipad-144-smile.png"/>  
  
    <title>Registry of Open Data on AWS</title>
  
    <!-- Latest compiled and minified CSS -->
    <link rel="stylesheet" href="https://assets.opendata.aws/css/bootstrap/3.4.1/bootstrap.min.css">
  
    <!-- Our local CSS -->
    <link rel="stylesheet" href="/css/main.css">
  
  
  </head>
  <body>
    <!-- jQuery (necessary for Bootstrap's JavaScript plugins) -->
    <script src="https://assets.opendata.aws/js/jquery/3.5.1/jquery.min.js"></script>
    <!-- Latest compiled and minified JavaScript -->
    <script src="https://assets.opendata.aws/js/bootstrap/3.4.1/bootstrap.min.js"></script>
    <nav class="navbar navbar-default navbar-fixed-top">
      <div class="container">
        <div class="roda-header">
          <h2><a href="/" alt="Home">Registry of Open Data on AWS</a></h2>
          <a href="http://aws.amazon.com/what-is-cloud-computing">
            <img src="https://assets.opendata.aws/img/AWS-Logo_White-Color_300x180.png" alt="Powered by AWS Cloud Computing" id="aws_header_logo">
          </a>
        </div>
      </div>
    </nav>
    <div class="container" id="dataatwork">
  <p class="lede">Usage examples for all datasets listed in the <a href="/">Registry of Open Data on AWS</a> tagged with <strong>genomic</strong>.</p>
  <hr>

      <div id="tcga" class="dataset">
        <h4><a href="../tcga/">The Cancer Genome Atlas</a></h4>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://firecloud.terra.bio/" class="" target="_none">Broad Institute FireCloud</a> by <a href="https://www.broadinstitute.org/" target="_none">The Broad Institute of MIT &amp; Harvard</a></a>
            <br>
            
          </li>          <li>
            <a href="https://cgc-accounts.sbgenomics.com/" class="" target="_none">Cancer Genomics Cloud</a> by <a href="https://docs.sevenbridges.com/docs/tcga-data" target="_none">Seven Bridges</a></a>
            <br>
            
          </li>          <li>
            <a href="https://portal.gdc.cancer.gov/legacy-archive/" class="" target="_none">GDC Legacy Archive</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></a>
            <br>
            
          </li>          <li>
            <a href="https://portal.gdc.cancer.gov/projects/" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></a>
            <br>
            
          </li>          <li>
            <a href="https://isb-cgc.appspot.com/" class="" target="_none">ISB Cancer Genomics Cloud</a> by <a href="https://isb-cancer-genomics-cloud.readthedocs.io/en/latest/sections/data/TCGA_top.html
          " target="_none">Institute for Systems Biology</a></a>
            <br>
            
          </li>          <li>
            <a href="https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga/studied-cancers
          " class="" target="_none">TCGA Cancers Selected for Study</a> by <a href="https://ocg.cancer.gov/" target="_none">National Cancer Institute</a></a>
            <br>
            
          </li>          <li>
            <a href="https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga/using-tcga
          " class="" target="_none">Using TCGA Data, Resources, and Materials</a> by <a href="https://ocg.cancer.gov/" target="_none">National Cancer Institute</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1016/j.cels.2019.06.006" class="" target="_none">&quot;Before and After: A Comparison of Legacy and Harmonized TCGA Data at the Genomic Data
          Commons&quot;
          </a> by Galen F. Gao, Joel S. Parker, et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1016/j.cell.2018.03.027" class="" target="_none">A Pan-Cancer Analysis of Enhancer Expression in Nearly 9000 Patient Samples</a> by Han Chen, Chunyan Li, et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1016/j.cell.2018.02.052" class="" target="_none">An Integrated TCGA Pan-Cancer Clinical Data Resource to Drive High-Quality Survival Outcome
          Analytics
          </a> by Jianfang Liu, Tara Lichtenberg, et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1016/j.cell.2018.03.022" class="" target="_none">Cell-of-Origin Patterns Dominate the Molecular Classification of 10,000 Tumors from 33 Types
          of Cancer
          </a> by Katherine A. Hoadley, Christina Yau, et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1016/j.ccell.2018.03.010" class="" target="_none">Comparative Molecular Analysis of Gastrointestinal Adenocarcinomas</a> by Yang Liu, Nilay S. Sethi, et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1016/j.ccell.2018.07.001" class="" target="_none">Comprehensive Analysis of Alternative Splicing Across Tumors from 8,705 Patients</a> by André Kahles, Kjong-Van Lehmann, et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1016/j.cell.2018.02.060" class="" target="_none">Comprehensive Characterization of Cancer Driver Genes and Mutations</a> by Matthew H. Bailey, Collin Tokheim, et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1016/j.ccell.2018.03.007" class="" target="_none">Genomic and Functional Approaches to Understanding Cancer Aneuploidy</a> by Alison M. Taylor, Juliann Shih, et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1016/j.celrep.2018.03.076" class="" target="_none">Genomic and Molecular Landscape of DNA Damage Repair Deficiency across The Cancer Genome
          Atlas
          </a> by Theo A. Knijnenburg, Linghua Wang, et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1016/j.celrep.2018.03.063" class="" target="_none">Genomic, Pathway Network, and Immunologic Features Distinguishing Squamous Carcinomas</a> by Joshua D. Campbell, Christina Yau, et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1016/j.celrep.2018.03.047" class="" target="_none">Integrated Genomic Analysis of the Ubiquitin Pathway across Cancer Types</a> by Zhongqi Ge, Jake S. Leighton, et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1016/j.celrep.2018.03.046" class="" target="_none">Machine Learning Detects Pan-Cancer Ras Pathway Activation in The Cancer Genome Atlas</a> by Gregory P. Way, Francisco Sanchez-Vega, et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1016/j.cell.2018.03.034" class="" target="_none">Machine Learning Identifies Stemness Features Associated with Oncogenic Dedifferentiation
          </a> by Tathiane M. Malta, Artem Sokolov, et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1016/j.celrep.2018.03.077" class="" target="_none">Molecular Characterization and Clinical Relevance of Metabolic Expression Subtypes in Human
          Cancers
          </a> by Xinxin Peng, Zhongyuan Chen, et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1016/j.cell.2018.03.035" class="" target="_none">Oncogenic Signaling Pathways in The Cancer Genome Atlas</a> by Francisco Sanchez-Vega, Marco Mina, et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1016/j.cels.2018.03.003" class="" target="_none">Pan-Cancer Alterations of the MYC Oncogene and its Proximal Network Across The Cancer Genome
          Atlas
          </a> by Franz X. Schaub, Varsha Dhankani, et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1016/j.celrep.2018.03.064" class="" target="_none">Pan-Cancer Analysis of lncRNA Regulation Supports Their Targeting of Cancer Genes in Each
          Tumor Context
          </a> by Hua-Sheng Chiu, Sonal Somvanshi, et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1016/j.cell.2018.03.039" class="" target="_none">Pathogenic Germline Variants in 10,389 Adult Cancers</a> by Kuan-lin Huang, R. Jay Mashl, et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1016/j.cels.2018.03.002" class="" target="_none">Scalable Open Science Approach for Mutation Calling of Tumor Exomes Using Multiple Genomic
          Pipelines
          </a> by Kyle Ellrott, Matthew H. Bailey, et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1016/j.celrep.2018.03.086" class="" target="_none">Spatial Organization And Molecular Correlation Of Tumor-Infiltrating Lymphocytes Using Deep
          Learning On Pathology Images
          </a> by Joel Saltz, Rajarsi Gupta, et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1126/science.aav1898" class="" target="_none">The chromatin accessibility landscape of primary human cancers</a> by M. Ryan Corces, Jeffrey M. Granja, et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1016/j.immuni.2018.03.023" class="" target="_none">The Immune Landscape of Cancer</a> by Vésteinn Thorsson, David L. Gibbs, et al.</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="target" class="dataset">
        <h4><a href="../target/">Therapeutically Applicable Research to Generate Effective Treatments (TARGET)</a></h4>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://firecloud.terra.bio/" class="" target="_none">Broad Institute FireCloud</a> by <a href="https://www.broadinstitute.org/" target="_none">The Broad Institute of MIT &amp; Harvard</a></a>
            <br>
            
          </li>          <li>
            <a href="https://cgc-accounts.sbgenomics.com/" class="" target="_none">Cancer Genomics Cloud</a> by <a href="https://docs.sevenbridges.com/docs/target-data" target="_none">Seven Bridges</a></a>
            <br>
            
          </li>          <li>
            <a href="https://portal.gdc.cancer.gov/legacy-archive/" class="" target="_none">GDC Legacy Archive</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></a>
            <br>
            
          </li>          <li>
            <a href="https://portal.gdc.cancer.gov/projects/" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></a>
            <br>
            
          </li>          <li>
            <a href="https://isb-cgc.appspot.com/" class="" target="_none">ISB Cancer Genomics Cloud</a> by <a href="https://isb-cancer-genomics-cloud.readthedocs.io/en/latest/sections/data/TARGET_top.html
          " target="_none">Institute for Systems Biology</a></a>
            <br>
            
          </li>          <li>
            <a href="https://ocg.cancer.gov/programs/target/data-matrix" class="" target="_none">TARGET data matrix</a> by <a href="https://ocg.cancer.gov/" target="_none">National Cancer Institute</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1038/ng.3940" class="" target="_none">A Children&#x27;s Oncology Group and TARGET initiative exploring the genetic landscape of Wilms
          tumor
          </a> by Gadd S, Huff V, Walz AL, et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/21297632" class="" target="_none">Ancestry and pharmacogenomics of relapse in acute lymphoblastic leukemia</a> by Yang JJ, Cheng C, Devidas M, et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term&#x3D;29228567" class="" target="_none">Biomarker significance of plasma and tumor miR-21, miR-221, and miR-106a in osteosarcoma
          </a> by Nakka M, Allen-Rhoades W, Li Y, et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term&#x3D;27143256" class="" target="_none">CSF3R mutations have a high degree of overlap with CEBPA mutations in pediatric AM</a> by Maxson JE, Ries RE, Wang YC, et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term&#x3D;26560027" class="" target="_none">Genetic predisposition to neuroblastoma mediated by a LMO1 super-enhancer polymorphism</a> by Oldridge DA, Wood AC, Weichert-Leahey N, Crimmins I, Sussman R, Winter C, McDaniel LD,
          Diamond M, Hart LS, Zhu S, Durbin AD, Abraham BJ, et al.
          </a>
            <br>
            
          </li>          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term&#x3D;26977886" class="" target="_none">Genome-Wide Profiles of Extra-cranial Malignant Rhabdoid Tumors Reveal Heterogeneity and
          Dysregulated Developmental Pathways
          </a> by Chun HJ, Lim EL, Heravi-Moussavi A, et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1038/s41586-018-0436-0" class="" target="_none">Genomic classification and identification of the cell of origin of pediatric mixed phenotype
          acute leukemia
          </a> by Thomas B. Alexander, Zhaohui Gu, Ilaria Iacobucci, et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term&#x3D;29315296" class="" target="_none">Genomics in childhood acute myeloid leukemia comes of age</a> by Brunner AM, Graubert TA.</a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1016/j.celrep.2019.10.013" class="" target="_none">Identification and analyses of extra-cranial and cranial rhabdoid tumor molecular subgroups
          reveal tumors with cytotoxic T cell infiltration
          </a> by Hye-Jung E. Chun, Pascal D. Johann, Katy Milne et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term&#x3D;29068783" class="" target="_none">MicroRNA Expression-Based Model Indicates Event-Free Survival in Pediatric Acute Myeloid
          Leukemia
          </a> by Lim EL, Trinh DL, Ries RE, et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term&#x3D;26635203" class="" target="_none">MLLT1 YEATS domain mutations in clinically distinctive Favourable Histology Wilms tumours
          </a> by Perlman EJ, Gadd S, Arold ST, et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term&#x3D;25670082" class="" target="_none">Recurrent DGCR8, DROSHA, and SIX homeodomain mutations in favorable histology Wilms tumors
          </a> by Walz AL, Ooms A, Gadd S, et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term&#x3D;26121087" class="" target="_none">Relapsed neuroblastomas show frequent RAS-MAPK pathway mutations</a> by Eleveld TF, Oldridge DA, Bernard V, Koster J, et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term&#x3D;27702824" class="" target="_none">Significance of TP53 Mutation in Wilms Tumors with Diffuse Anaplasia: A Report from the
          Children&#x27;s Oncology Group
          </a> by Ooms AH, Gadd S, Gerhard DS, et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term&#x3D;26158413" class="" target="_none">TCF21 hypermethylation in genetically quiescent clear cell sarcoma of the kidney</a> by Gooskens SL, Gadd S, Guidry Auvil JM, et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term&#x3D;23334666" class="" target="_none">The genetic landscape of high-risk neuroblastoma</a> by Pugh TJ, Morozova O, Attiyeh EF, Asgharzadeh S, Wei JS, Auclair D, Carter SL, Cibulskis K,
          Hanna M, Kiezun A, Kim J, Lawrence MS, Lichenstein L, et al.
          </a>
            <br>
            
          </li>          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term&#x3D;28671688" class="" target="_none">The genomic landscape of pediatric and young adult T-lineage acute lymphoblastic leukemia
          </a> by Yu Liu, John Easton, Ying Shao, et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term&#x3D;29227476" class="" target="_none">The molecular landscape of pediatric acute myeloid leukemia reveals recurrent structural
          alterations and age-specific mutational interactions
          </a> by Bolouri H, Farrar JE, Triche T Jr, et al.</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="kids-first" class="dataset">
        <h4><a href="../kids-first/">Gabriella Miller Kids First Pediatric Research Program (Kids First)</a></h4>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="http://cavatica.org" class="" target="_none">CAVATICA</a> by <a href="http://www.sevenbridges.com" target="_none">Seven Bridges Genomics</a></a>
            <br>
            
          </li>          <li>
            <a href="https://portal.kidsfirstdrc.org" class="" target="_none">Kids First DRC Portal</a> by <a href="https://kidsfirstdrc.org" target="_none">Kids First DRC</a></a>
            <br>
            
          </li>          <li>
            <a href="https://github.com/kids-first/" class="" target="_none">Kids First DRC Source Code</a> by <a href="https://kidsfirstdrc.org" target="_none">Kids First DRC</a></a>
            <br>
            
          </li>          <li>
            <a href="https://pedcbioportal.kidsfirstdrc.org" class="" target="_none">PedcBioPortal</a> by <a href="https://www.cbioportal.org/" target="_none">cBioPortal</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://pubmed.ncbi.nlm.nih.gov/30322880/" class="" target="_none">Clinically Relevant and Minimally Invasive Tumor Surveillance of Pediatric Diffuse Midline Gliomas Using Patient-Derived Liquid Biopsy</a> by Eshini Panditharatna, Lindsay B Kilburn, et al.</a>
            <br>
            
          </li>          <li>
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/31211835" class="" target="_none">Decreased ACKR3 (CXCR7) function causes oculomotor synkinesis in mice and humans.</a> by Mary C Whitman, Noriko Miyake, et al.</a>
            <br>
            
          </li>          <li>
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/31206972" class="" target="_none">Deleterious de novo variants of X-linked ZC4H2 in females cause a variable phenotype with neurogenic arthrogryposis multiplex congenita.</a> by Suzanna G M Frints, Friederike Hennig, et al.</a>
            <br>
            
          </li>          <li>
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/31255796" class="" target="_none">Development and Clinical Validation of a Large Fusion Gene Panel for Pediatric Cancers.</a> by Fengqi Chang, Fumin Lin, et al.</a>
            <br>
            
          </li>          <li>
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/31898844" class="" target="_none">Elucidation of de novo small insertion/deletion biology with parent-of-origin phasing.</a> by Allison H Seiden, Felix Richter, et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://pubmed.ncbi.nlm.nih.gov/30098192/" class="" target="_none">Genome-Wide Association Study Identifies a Susceptibility Locus for Comitant Esotropia and Suggests a Parent-of-Origin Effect</a> by Sherin Shaaban, Sarah MacKinnon et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://pubmed.ncbi.nlm.nih.gov/32574564/" class="" target="_none">Genome-wide Enrichment of De Novo Coding Mutations in Orofacial Cleft Trios.</a> by Madison R Bishop, Kimberly K Diaz Perez, et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://pubmed.ncbi.nlm.nih.gov/32601476/" class="" target="_none">Genomic Analyses Implicate Noncoding De Novo Variants in Congenital Heart Disease.</a> by Felix Richter, Sarah U Morton, et al.</a>
            <br>
            
          </li>          <li>
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/31474320" class="" target="_none">Germline 16p11.2 Microdeletion Predisposes to Neuroblastoma.</a> by Laura Egolf, Zalman Vaksman, et al.</a>
            <br>
            
          </li>          <li>
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/31562520" class="" target="_none">Germline microsatellite genotypes differentiate children with medulloblastoma.</a> by Samuel Rivero-Hinojosa, Nicholas Kinney, et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://pubmed.ncbi.nlm.nih.gov/30471716/" class="" target="_none">MACF1 Mutations Encoding Highly Conserved Zinc-Binding Residues of the GAR Domain Cause Defects in Neuronal Migration and Axon Guidance</a> by William B Dobyns, Kimberly A Aldinger, et al.</a>
            <br>
            
          </li>          <li>
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/31397880" class="" target="_none">MAGEL2-Related Disorders: A study and case series.</a> by Jameson Patak, James Gilfert, et al.</a>
            <br>
            
          </li>          <li>
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/31361404" class="" target="_none">Phenotype delineation of ZNF462 related syndrome.</a> by Paul Kruszka, Tommy Hu, et al.</a>
            <br>
            
          </li>          <li>
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/30930166" class="" target="_none">The Pediatric Cell Atlas: Defining the Growth Phase of Human Development at Single-Cell Resolution.</a> by Deanne M Taylor, Bruce J Aronow, et al.</a>
            <br>
            
          </li>          <li>
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/31848685" class="" target="_none">Whole genome sequencing of orofacial cleft trios from the Gabriella Miller Kids First Pediatric Research Consortium identifies a new locus on chromosome 21.</a> by Nandita Mukhopadhyay, Madison Bishop, et al.</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="broad-gnomad" class="dataset">
        <h4><a href="../broad-gnomad/">Genome Aggregation Database (gnomAD)</a></h4>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/broadinstitute/gnomad_qc" class="" target="_none">gnomAD quality control GitHub repository</a> by <a href="https://github.com/orgs/broadinstitute/teams/gnomad/members" target="_none">gnomAD Production Team</a></a>
            <br>
            
          </li>          <li>
            <a href="https://hail.is/" class="" target="_none">Hail</a> by <a href="https://hail.is/" target="_none">Hail Team</a></a>
            <br>
            
          </li>          <li>
            <a href="https://aws.amazon.com/quickstart/architecture/hail/" class="aws-link" target="_none">Hail on AWS Quick Start</a> by Amazon Web Services and PrivoIT</a>
            <br>
            
          </li>          <li>
            <a href="https://pypi.org/project/gnomad/" class="" target="_none">Hail utilities for gnomAD</a> by <a href="https://github.com/orgs/broadinstitute/teams/gnomad/members" target="_none">gnomAD Production Team</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1038/s41586-020-2287-8" class="" target="_none">A structural variation reference for medical and population genetics. Nature 581, 444–451 (2020)</a> by Collins, R. L., Brand, H., Karczewski, K. J., Zhao, X., Alföldi, J., Francioli, L. C., Khera, A. V., Lowther, C., Gauthier, L. D., Wang, H., Watts, N. A., Solomonson, M., O’Donnell-Luria, A., Baumann, A., Munshi, R., Walker, M., Whelan, C., Huang, Y., Brookings, T., ... Talkowski, M. E.</a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1038/nature19057" class="" target="_none">Analysis of protein-coding genetic variation in 60,706 humans. Nature 536, 285–291 (2016)</a> by Lek, M., Karczewski, K., Minikel, E. et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1038/s41467-019-10717-9" class="" target="_none">Characterising the loss-of-function impact of 5’ untranslated region variants in 15,708 individuals. Nature Communications 11, 2523 (2020)</a> by Whiffin, N., Karczewski, K. J., Zhang, X., Chothani, S., Smith, M. J., Gareth Evans, D., Roberts, A. M., Quaife, N. M., Schafer, S., Rackham, O., Alföldi, J., O’Donnell-Luria, A. H., Francioli, L. C., Genome Aggregation Database (gnomAD) Production Team, Genome Aggregation Database (gnomAD) Consortium, Cook, S. A., Barton, P. J. R., MacArthur, D. G., &amp; Ware, J. S.</a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1038/s41586-020-2267-z" class="" target="_none">Evaluating potential drug targets through human loss-of-function genetic variation. Nature 581, 459–464 (2020)</a> by Minikel, E. V., Karczewski, K. J., Martin, H. C., Cummings, B. B., Whiffin, N., Rhodes, D., Alföldi, J., Trembath, R. C., van Heel, D. A., Daly, M. J., Genome Aggregation Database Production Team, Genome Aggregation Database Consortium, Schreiber, S. L., &amp; MacArthur, D. G.</a>
            <br>
            
          </li>          <li>
            <a href="https://gnomad.broadinstitute.org/blog/2018-10-gnomad-v2-1/" class="" target="_none">gnomAD v2.1</a> by Laurent Francioli, Grace Tiao, Konrad Karczewski, Matthew Solomonson, Nick Watts</a>
            <br>
            
          </li>          <li>
            <a href="https://gnomad.broadinstitute.org/blog/2019-10-gnomad-v3-0/" class="" target="_none">gnomAD v3.0</a> by Laurent Francioli, Daniel MacArthur</a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1038/s41467-019-12438-5" class="" target="_none">Landscape of multi-nucleotide variants in 125,748 human exomes and 15,708 genomes. Nature Communications 11, 2539 (2020)</a> by Wang, Q., Pierce-Hoffman, E., Cummings, B. B., Karczewski, K. J., Alföldi, J., Francioli, L. C., Gauthier, L. D., Hill, A. J., O’Donnell-Luria, A. H., Genome Aggregation Database (gnomAD) Production Team, Genome Aggregation Database (gnomAD) Consortium, &amp; MacArthur, D. G.</a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1101/784157" class="" target="_none">Technical artifact drives apparent deviation from Hardy-Weinberg equilibrium at CCR5-∆32 and other variants in gnomAD. bioRxiv (p. 784157)</a> by Karczewski, K. J., Gauthier, L. D., Daly, M. J.</a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1038/s41591-020-0893-5" class="" target="_none">The effect of LRRK2 loss-of-function variants in humans. Nature Medicine (2020)</a> by Whiffin, N., Armean, I. M., Kleinman, A., Marshall, J. L., Minikel, E. V., Goodrich, J. K., Quaife, N. M., Cole, J. B., Wang, Q., Karczewski, K. J., Cummings, B. B., Francioli, L., Laricchia, K., Guan, A., Alipanahi, B., Morrison, P., Baptista, M. A. S., Merchant, K. M., Genome Aggregation Database Production Team, ... MacArthur, D. G.</a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1038/s41586-020-2308-7" class="" target="_none">The mutational constraint spectrum quantified from variation in 141,456 humans. Nature 581, 434–443 (2020)</a> by Karczewski, K. J., Francioli, L. C., Tiao, G., Cummings, B. B., Alföldi, J., Wang, Q., Collins, R. L., Laricchia, K. M., Ganna, A., Birnbaum, D. P., Gauthier, L. D., Brand, H., Solomonson, M., Watts, N. A., Rhodes, D., Singer-Berk, M., England, E. M., Seaby, E. G., Kosmicki, J. A., ... MacArthur, D. G.</a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1038/s41586-020-2329-2" class="" target="_none">Transcript expression-aware annotation improves rare variant interpretation. Nature 581, 452–458 (2020)</a> by Cummings, B. B., Karczewski, K. J., Kosmicki, J. A., Seaby, E. G., Watts, N. A., Singer-Berk, M., Mudge, J. M., Karjalainen, J., Kyle Satterstrom, F., O’Donnell-Luria, A., Poterba, T., Seed, C., Solomonson, M., Alföldi, J., The Genome Aggregation Database Production Team, The Genome Aggregation Database Consortium, Daly, M. J., &amp; MacArthur, D. G.</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="ccle" class="dataset">
        <h4><a href="../ccle/">Cancer Cell Line Encyclopedia (CCLE)</a></h4>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://portals.broadinstitute.org/ccle" class="" target="_none">Broad Institute Cancer Cell Line Encyclopedia</a> by <a href="https://www.broadinstitute.org/" target="_none">The Broad Institute of MIT &amp; Harvard</a></a>
            <br>
            
          </li>          <li>
            <a href="https://cgc-accounts.sbgenomics.com/" class="" target="_none">Cancer Genomics Cloud</a> by <a href="https://docs.sevenbridges.com/docs/ccle-data" target="_none">Seven Bridges</a></a>
            <br>
            
          </li>          <li>
            <a href="https://portal.gdc.cancer.gov/legacy-archive/" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></a>
            <br>
            
          </li>          <li>
            <a href="https://console.cloud.google.com/bigquery?p&#x3D;isb-cgc&amp;d&#x3D;CCLE_bioclin_v0&amp;page&#x3D;dataset" class="" target="_none">ISB CGC BigQuery tables</a> by <a href="https://isb-cancer-genomics-cloud.readthedocs.io/en/latest/sections/data/CCLE_top.html
          " target="_none">Institute for Systems Biology</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1038/s41586-019-1186-3" class="" target="_none">Next-generation characterization of the Cancer Cell Line Encyclopedia</a> by Ghandi, M., Huang F. et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1038/nature15736" class="" target="_none">Pharmacogenomic agreement between two cancer cell line data sets</a> by The Cancer Cell Line Encyclopedia Consortium &amp; The Genomics of Drug Sensitivity in Cancer
          Consortium
          </a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1038/nature11003" class="" target="_none">The Cancer Cell Line Encyclopedia enables predictive modelling of anticancer drug
          sensitivity
          </a> by Barretina Caponigro Stransky et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1038/s41591-019-0404-8" class="" target="_none">The landscape of cancer cell line metabolism</a> by Li, H. et al.</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="cptac-2" class="dataset">
        <h4><a href="../cptac-2/">Clinical Proteomic Tumor Analysis Consortium 2 (CPTAC-2)</a></h4>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://cgc-accounts.sbgenomics.com/" class="" target="_none">Cancer Genomics Cloud</a> by <a href="http://www.cancergenomicscloud.org/" target="_none">Seven Bridges</a></a>
            <br>
            
          </li>          <li>
            <a href="https://proteomics.cancer.gov/data-portal" class="" target="_none">CPTAC Data Portal</a> by <a href="https://proteomics.cancer.gov/" target="_none">National Cancer Institute</a></a>
            <br>
            
          </li>          <li>
            <a href="https://portal.gdc.cancer.gov/projects/CPTAC-2" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></a>
            <br>
            
          </li>          <li>
            <a href="https://pdc.esacinc.com/pdc/browse" class="" target="_none">Proteomic Data Commons</a> by <a href="https://pdc.esacinc.com/pdc" target="_none">National Cancer Institute</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1016/j.cell.2016.05.069" class="" target="_none">Integrated Proteogenomic Characterization of Human High-Grade Serous Ovarian Cancer</a> by Hui Zhang, Tao Liu, Zhen Zhang, Samuel H. Payne, Bai Zhang, Jason E. McDermott, Jian-Ying
          Zhou, Vladislav A. Petyuk, Li Chen, Debjit Ray, Shisheng Sun, Feng Yang, Lijun Chen, Jing
          Wang, Punit Shah, Seong Won Cha, Paul Aiyetan, Sunghee Woo, Yuan Tian, Marina A. Gritsenko,
          Therese R. Clauss, Caitlin Choi, Matthew E. Monroe, Stefani Thomas, Song Nie, Chaochao Wu,
          Ronald J. Moore, Kun-Hsing Yu, David L. Tabb, David Fenyö, Vineet Bafna, Yue Wang, Henry
          Rodriguez, Emily S. Boja, Tara Hiltke, Robert C. Rivers, Lori Sokoll, Heng Zhu, Ie-Ming
          Shih, Leslie Cope, Akhilesh Pandey, Bing Zhang, Michael P. Snyder, Douglas A. Levine,
          Richard D. Smith, Daniel W. Chan, Karin D. Rodland, the CPTAC Investigators
          </a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1016/j.cell.2019.03.030" class="" target="_none">Proteogenomic Analysis of Human Colon Cancer Reveals New Therapeutic Opportunities</a> by Suhas Vasaikar, Chen Huang, Xiaojing Wang. Vladislav A. Petyuk, Sara R. Savage, Bo Wen,
          Yongchao Dou, Yun Zhang, Zhiao Shi, Osama A. Arshad, Marina A. Gritsenko, Lisa J. Zimmerman,
          Jason E. McDermott, Therese R. Clauss, Ronald J. Moore, Rui Zhao, Matthew E. Monroe, Yi-Ting
          Wang, Matthew C. Chambers, Robbert J.C. Slebos, Ken S. Lau, Qianxing Mo, Li Ding, Matthew
          Ellis, Mathangi Thiagarajan, Christopher R. Kinsinger, Henry Rodriguez, Richard D. Smith,
          Karin D. Rodland, Daniel C. Liebler, Tao Liu, Bing Zhang, Clinical Proteomic Tumor Analysis
          Consortium
          </a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1038/sdata.2015.22" class="" target="_none">Proteomic analysis of colon and rectal carcinoma using standard and customized databases
          </a> by Slebos RJ, Wang X, Wang X, Zhang B, Tabb DL, Liebler DC</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="icgc" class="dataset">
        <h4><a href="../icgc/">ICGC on AWS</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/ICGC-TCGA-PanCancer/cli/wiki" class="" target="_none">AWS Pancancer Workflow Launcher</a> by <a href="https://github.com/solomonshorser" target="_none">Solomon Shorser</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://www.nature.com/articles/s41586-020-1965-x" class="" target="_none">Analyses of non-coding somatic drivers in 2,693 cancer whole genomes</a> by <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term&#x3D;Rheinbay%20E%5BAuthor%5D&amp;cauthor&#x3D;true&amp;cauthor_uid&#x3D;28658208" target="_none">Rheinbay et al (2020)</a></a>
            <br>
            
          </li>          <li>
            <a href="https://www.nature.com/articles/s41586-020-1970-0" class="" target="_none">Genomic basis of RNA alterations in cancer</a> by <a href="https://icgc.org/committees-and-working-groups#71954" target="_none">PCAWG Transcriptome Core Group et al. (2020)</a></a>
            <br>
            
          </li>          <li>
            <a href="https://www.nature.com/articles/s41586-020-1969-6" class="" target="_none">Pan-cancer analysis of whole genomes</a> by <a href="https://dcc.icgc.org/pcawg" target="_none">The ICGC/TCGA Pan-Cancer Analysis of Whole Genomes Consortium (2020)</a></a>
            <br>
            
          </li>          <li>
            <a href="https://www.nature.com/articles/s41586-019-1913-9" class="" target="_none">Patterns of somatic structural variation in human cancer genomes</a> by <a href="https://www.ncbi.nlm.nih.gov/pubmed?cmd&#x3D;search&amp;term&#x3D;Yilong%20Li" target="_none">Li et al. (2020)</a></a>
            <br>
            
          </li>          <li>
            <a href="https://www.nature.com/articles/s41586-019-1907-7" class="" target="_none">The evolutionary history of 2,658 cancers</a> by <a href="https://www.ncbi.nlm.nih.gov/pubmed?cmd&#x3D;search&amp;term&#x3D;Moritz%20Gerstung" target="_none">Gerstung et al (2020)</a></a>
            <br>
            
          </li>          <li>
            <a href="https://www.nature.com/articles/s41586-020-1943-3" class="" target="_none">The repertoire of mutational signatures in human cancer</a> by <a href="https://www.ncbi.nlm.nih.gov/pubmed?cmd&#x3D;search&amp;term&#x3D;Ludmil%20B.%20Alexandrov" target="_none">Alexandrov et al (2020)</a></a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="cptac-3" class="dataset">
        <h4><a href="../cptac-3/">Clinical Proteomic Tumor Analysis Consortium 3 (CPTAC-3)</a></h4>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://cgc-accounts.sbgenomics.com/" class="" target="_none">Cancer Genomics Cloud</a> by <a href="http://www.cancergenomicscloud.org/" target="_none">Seven Bridges</a></a>
            <br>
            
          </li>          <li>
            <a href="https://proteomics.cancer.gov/data-portal" class="" target="_none">CPTAC Data Portal</a> by <a href="https://proteomics.cancer.gov/" target="_none">National Cancer Institute</a></a>
            <br>
            
          </li>          <li>
            <a href="https://portal.gdc.cancer.gov/projects/CPTAC-3" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></a>
            <br>
            
          </li>          <li>
            <a href="https://pdc.esacinc.com/pdc/browse" class="" target="_none">Proteomic Data Commons</a> by <a href="https://pdc.esacinc.com/pdc" target="_none">National Cancer Institute</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/29718670" class="" target="_none">Evaluation of NCI-7 Cell Line Panel as a Reference Material for Clinical Proteomics</a> by Clark DJ, Hu Y, Bocik W, Chen L, Schnaubelt M, Roberts R, Shah P, Whiteley G, Zhang H
          </a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1016/j.cell.2019.10.007" class="" target="_none">Integrated Proteogenomic Characterization of Clear Cell Renal Cell Carcinoma</a> by Clark DJ, Dhanasekaran SM, Petralia F, Pan J, Song X, Hu Y, da Veiga Leprevost F, Reva B,
          Lih TM, Chang HY, Ma W, Huang C, Ricketts CJ, Chen L1, Krek A, Li Y, Rykunov D, Li QK, Chen
          LS, Ozbek U, Vasaikar S, Wu Y, Yoo S, Chowdhury S, Wyczalkowski MA, Ji J, Schnaubelt M, Kong
          A, Sethuraman S, Avtonomov DM, Ao M, Colaprico A, Cao S, Cho KC, Kalayci S, Ma S, Liu W,
          Ruggles K, Calinawan A, Gümüş ZH, Geizler D, Kawaler E, Teo GC, Wen B, Zhang Y, Keegan S, Li
          K, Chen F, Edwards N, Pierorazio PM, Chen XS, Pavlovich CP, Hakimi AA, Brominski G, Hsieh
          JJ, Antczak A, Omelchenko T, Lubinski J, Wiznerowicz M, Linehan WM, Kinsinger CR,
          Thiagarajan M, Boja ES, Mesri M, Hiltke T, Robles AI, Rodriguez H, Qian J, Fenyö D, Zhang B,
          Ding L, Schadt E, Chinnaiyan AM, Zhang Z, Omenn GS, Cieslik M, Chan DW, Nesvizhskii AI, Wang
          P, Zhang H; Clinical Proteomic Tumor Analysis Consortium
          </a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="mmrf-commpass" class="dataset">
        <h4><a href="../mmrf-commpass/">CoMMpass from the Multiple Myeloma Research Foundation</a></h4>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://portal.gdc.cancer.gov/projects/MMRF-COMMPASS" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1182/blood.V126.23.722.722" class="" target="_none">&quot;Identification of Initiating Trunk Mutations and Distinct Molecular Subtypes: An Interim
          Analysis of the Mmrf Commpass Study&quot;
          </a> by Jonathan J Keats, PhD, Gil Speyer, Legendre Christophe, Christofferson Austin, Kristi
          Stephenson, BS, Ahmet Kurdoglu, Megan Russell, Aldrich Jessica, Cuyugan Lori, Jonathan
          Adkins, Jackie McDonald, Adrienne Helland, Alex Blanski, Meghan Hodges, Dan Rohrer, Sundar
          Jagannath, MD, David Siegel, MD PhD, Ravi Vij, MD MBA, Gregory Orloff, MD, Todd Zimmerman,
          MD, Ruben Niesvizky, MD, Darla Liles, MD, Joseph W. Fay, Jeffrey L. Wolf, MD PhD, Robert M.
          Rifkin, Norma C Gutierrez, The MMRF CoMMpass Network, Jen Toups, Mary Derome, MS, Winnie
          Liang, PhD, Seunchan Kim, Daniel Auclair, PhD, Pamela G. Kidd, MD, Scott Jewell, PhD, John
          David Carpten, PhD, Sagar Lonial, MD
          </a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1182/blood.V122.21.532.532" class="" target="_none">&quot;Interim Analysis Of The MMRF CoMMpass Trial: a Longitudinal Study In Multiple Myeloma
          Relating Clinical Outcomes To Genomic and Immunophenotypic Profiles&quot;
          </a> by Keats JJ, Craig DW, Liang W, Venkata Y, Kurdoglu A, Aldrich J, Auclair D, Allen K, Harrison
          B, Jewell S, Kidd PG, Correll M, Jagannath S, Siegel DS, Vij R, Orloff G, Zimmerman TM, MMRF
          CoMMpass Network, Capone W, Carpten J, Lonial S.
          </a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1182/blood.V124.21.722.722" class="" target="_none">&quot;Interim Analysis of the Mmrf Commpass Trial: Identification of Novel Rearrangements
          Potentially Associated with Disease Initiation and Progression&quot;
          </a> by Sagar Lonial, MD, Venkata D Yellapantula, Winnie Liang, PhD, Ahmet Kurdoglu, BS, Jessica
          Aldrich, MSc, Christophe M. Legendre, MD, Kristi Stephenson, Jonathan Adkins, Jackie
          McDonald, Adrienne Helland, Megan Russell, Austin Christofferson, Lori Cuyugan, Dan Rohrer,
          Alex Blanski, Meghan Hodges, Mmrf CoMMpass Network, Mary Derome, Daniel Auclair, PhD, Pamela
          G. Kidd, MD, Scott Jewell, PhD, David Craig, PhD, John Carpten, PhD, Jonathan J. Keats, PhD
          </a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1182/blood.V128.22.194.194" class="" target="_none">&quot;Molecular Predictors of Outcome and Drug Response in Multiple Myeloma: An Interim Analysis
          of the Mmrf CoMMpass Study&quot;
          </a> by Jonathan J Keats, PhD, Gil Speyer, Austin Christofferson, Christophe Legendre, PhD, Jessica
          Aldrich, Megan Russell, Lori Cuyugan, Jonathan Adkins, Alex Blanski, Meghan Hodges, Dan
          Rohrer, Sundar Jagannath, MD, Ravi Vij, MD, Gregory Orloff, MD, Todd Zimmerman, MD, Ruben
          Niesvizky, MD, Darla Liles, MD, Joseph W. Fay, Jeffrey L. Wolf, MD, Robert M Rifkin, Norma C
          Gutierrez, MD PhD, Mmrf CoMMpass Network, Jennifer Yesil, MS, Mary Derome, MS, Seungchan
          Kim, PhD, Winnie Liang, PhD, Pamela G. Kidd, MD, Scott Jewell, PhD, John David Carpten, PhD,
          Daniel Auclair, PhD, Sagar Lonial, MD FACP
          </a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="ilmn-dragen-1kgp" class="dataset">
        <h4><a href="../ilmn-dragen-1kgp/">1000 Genomes Phase 3 Reanalysis with DRAGEN 3.5</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/aws-quickstart/quickstart-illumina-dragen" class="" target="_none">DRAGEN Quick Start on AWS</a> by <a href="https://github.com/aws-quickstart" target="_none">AWS Quick Start Team</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://support.illumina.com/content/dam/illumina-support/documents/downloads/software/dragen/DRAGEN-3.5.7b-Customer-Release-Notes-1000000131537-00.pdf" class="" target="_none">DRAGEN 3.7.5b Release</a> by <a href="https://www.illumina.com/" target="_none">Illumina Inc.</a></a>
            <br>
            
          </li>          <li>
            <a href="https://www.illumina.com/products/by-type/informatics-products/basespace-sequence-hub.html#key-features" class="" target="_none">DRAGEN on BaseSpace Sequence Hub</a> by <a href="https://www.illumina.com/" target="_none">Illumina Inc.</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://www.biorxiv.org/content/10.1101/2020.11.13.380741v1" class="" target="_none">precisionFDA Truth Challenge V2: Calling variants from short- and long-reads in difficult-to-map regions (Preprint)</a> by Olson et al (2020)</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="3kricegenome" class="dataset">
        <h4><a href="../3kricegenome/">3000 Rice Genomes Project</a></h4>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/InternationalRiceResearchInstitute/RiceGalaxy" class="" target="_none">RiceGalaxy</a> by <a href="https://www.irri.org/" target="_none">International Rice Research Institute</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://academic.oup.com/gigascience/article/8/5/giz028/5492234" class="" target="_none">Rice Galaxy: an open resource for plant science</a> by Juanillas V et al (2019)</a>
            <br>
            
          </li>          <li>
            <a href="https://genome.cshlp.org/content/29/5/870.short" class="" target="_none">Structural variants in 3000 rice genomes</a> by Fuentes RR et al (2019)</a>
            <br>
            
          </li>          <li>
            <a href="https://www.nature.com/articles/s41467-019-08451-3" class="" target="_none">Tracking the origin of two genetic components associated with transposable element bursts in domesticated rice</a> by Chen J et al (2019)</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="encode-project" class="dataset">
        <h4><a href="../encode-project/">Encyclopedia of DNA Elements (ENCODE)</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/ENCODE-DCC/encode-data-usage-examples/blob/master/ctcf_chip_seq_cross_cell_type_correlation.ipynb" class="" target="_none">ENCODE CTCF ChIP-seq data correlation across different cell types</a> by <a href="https://github.com/p-sud" target="_none">Paul Sud</a></a>
            <br>
            
          </li>          <li>
            <a href="https://github.com/ENCODE-DCC/encode-data-usage-examples/blob/master/mount_s3_bucket_and_run_jupyter_on_ec2.ipynb" class="" target="_none">Exploring ENCODE data from EC2 with Jupyter notebook</a> by <a href="https://github.com/keenangraham" target="_none">Keenan Graham</a></a>
            <br>
            
          </li>          <li>
            <a href="https://github.com/ENCODE-DCC/encode-data-usage-examples/blob/master/ingest_encode_data_tile_db_with_s3_backend.ipynb" class="" target="_none">Ingesting ENCODE data into TileDB with S3 backend</a> by <a href="https://github.com/ottojolanki" target="_none">Otto Jolanki</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://academic.oup.com/nar/article/48/D1/D882/5622708#190992715" class="" target="_none">New developments on the Encyclopedia of DNA Elements (ENCODE) data portal</a> by <a href="http://orcid.org/0000-0001-6948-2042" target="_none">Luo et al 2020</a></a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="giab" class="dataset">
        <h4><a href="../giab/">Genome in a Bottle on AWS</a></h4>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/ga4gh/benchmarking-tools" class="" target="_none">GA4GH Benchmarking Tools</a> by GA4GH Benchmarking Team</a>
            <br>
            
          </li>          <li>
            <a href="https://github.com/genome-in-a-bottle/about_GIAB" class="" target="_none">The Genome in a Bottle Github Project</a> by Genome In A Bottle Consortium</a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://www.nature.com/articles/sdata201625" class="" target="_none">Extensive sequencing of seven human genomes to characterize benchmark reference materials</a> by Zook J et al (2016)</a>
            <br>
            
          </li>          <li>
            <a href="https://www.nature.com/articles/s41597-019-0098-2" class="" target="_none">High-coverage, long-read sequencing of Han Chinese trio reference samples</a> by Wang Y et al (2019)</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="ncbi-sra" class="dataset">
        <h4><a href="../ncbi-sra/">NIH NCBI Sequence Read Archive (SRA) on AWS</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/sra/docs/sra-aws-download/" class="" target="_none">Access SRA data using Amazon Web Services (AWS)</a> by <a href="https://www.ncbi.nlm.nih.gov/sra/" target="_none">NCBI SRA</a></a>
            <br>
            
          </li>          <li>
            <a href="https://www.ncbi.nlm.nih.gov/sra/docs/sra-cloud/" class="" target="_none">SRA in the Cloud</a> by <a href="https://www.ncbi.nlm.nih.gov/sra/" target="_none">NCBI SRA</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/ncbi/sra-tools" class="" target="_none">SRA Toolkit</a> by <a href="https://www.ncbi.nlm.nih.gov/sra/" target="_none">NCBI SRA</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3013647/" class="" target="_none">The Sequence Read Archive</a> by <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term&#x3D;Leinonen%20R%5BAuthor%5D&amp;cauthor&#x3D;true&amp;cauthor_uid&#x3D;21062823" target="_none">Leinonen et al (2011)</a></a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="refgenie" class="dataset">
        <h4><a href="../refgenie/">Refgenie reference genome assets</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="http://refgenie.databio.org/en/latest/tutorial/" class="" target="_none">Basic Refgenie tutorial</a> by <a href="http://www.databio.org" target="_none">Nathan Sheffield</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="http://pepatac.databio.org" class="" target="_none">Pipeline for ATAC-seq data analysis</a> by <a href="http://databio.org" target="_none">Jason Smith and Nathan Sheffield</a></a>
            <br>
            
          </li>          <li>
            <a href="http://peppro.databio.org" class="" target="_none">Pipeline for PRO-seq data analysis</a> by <a href="http://databio.org" target="_none">Jason Smith and Nathan Sheffield</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1093/gigascience/giz149" class="" target="_none">Refgenie: a reference genome resource manager</a> by <a href="http://www.databio.org" target="_none">Michał Stolarczyk, Vincent P Reuter, Jason P Smith, Neal E Magee, Nathan C Sheffield</a></a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="broad-pan-ukb" class="dataset">
        <h4><a href="../broad-pan-ukb/">UK Biobank Pan-Ancestry Summary Statistics</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://hail.is/docs/0.2/tutorials-landing.html" class="" target="_none">Hail Tutorials</a> by <a href="https://hail.is/" target="_none">Hail Team</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://hail.is" class="" target="_none">Hail</a> by <a href="https://hail.is/" target="_none">Hail Team</a></a>
            <br>
            
          </li>          <li>
            <a href="https://aws.amazon.com/quickstart/architecture/hail/" class="aws-link" target="_none">Hail on AWS Quick Start</a> by Amazon Web Services and PrivoIT</a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://pan.ukbb.broadinstitute.org/" class="" target="_none">Pan-ancestry genetic analysis of the UK Biobank</a> by Pan UKBB Team</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="ncbi-blast-databases" class="dataset">
        <h4><a href="../ncbi-blast-databases/">Basic Local Alignment Sequences Tool (BLAST) Databases</a></h4>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/ncbi/blast_plus_docs" class="" target="_none">BLAST+ Docker</a> by NCBI BLAST</a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://pubmed.ncbi.nlm.nih.gov/20003500" class="" target="_none">BLAST+: Architecture and Applications</a> by Christiam Camacho  1 , George Coulouris, Vahram Avagyan, Ning Ma, Jason Papadopoulos, Kevin Bealer, Thomas L Madden</a>
            <br>
            
          </li>          <li>
            <a href="https://pubmed.ncbi.nlm.nih.gov/9254694/" class="" target="_none">Gapped BLAST and PSI-BLAST: A New Generation of Protein Database Search Programs</a> by S F Altschul, T L Madden, A A Schäffer, J Zhang, Z Zhang, W Miller, D J Lipman</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="beataml" class="dataset">
        <h4><a href="../beataml/">Beat Acute Myeloid Leukemia (AML) 1.0</a></h4>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://portal.gdc.cancer.gov/projects/" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/30651561" class="" target="_none">Clinical resistance to crenolanib in acute myeloid leukemia due to diverse molecular
          mechanisms
          </a> by Zhang H, Savage S, Schultz AR, Bottomly D, White L, Segerdell E, et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/30333627" class="" target="_none">Functional Genomic Landscape of Acute Myeloid Leukemia</a> by Jeffrey W. Tyner, Cristina E. Tognon, Dan Bottomly et al.</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="ctsp-dlbcl" class="dataset">
        <h4><a href="../ctsp-dlbcl/">Clinical Trial Sequencing Project - Diffuse Large B-Cell Lymphoma</a></h4>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://portal.gdc.cancer.gov/projects/CTSP-DLBCL1" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed?cmd&#x3D;DetailsSearch&amp;term&#x3D;29925955[PMID]" class="" target="_none">A multiprotein supercomplex controlling oncogenic signalling in lymphoma</a> by Phelan JD, Young RM, Webster DE, Roulland S, Wright GW, Kasbekar M, Shaffer AL 3rd,
          Ceribelli M, Wang JQ, Schmitz R, Nakagawa M, Bachy E, Huang DW, Ji Y, Chen L, Yang Y, Zhao
          H, Yu X, Xu W, Palisoc MM, Valadez RR, Davies-Hill T, Wilson WH, Chan WC, Jaffe ES, Gascoyne
          RD, Campo E, Rosenwald A, Ott G, Delabie J, Rimsza LM, Rodriguez FJ, Estephan F, Holdhoff M,
          Kruhlak MJ, Hewitt SM, Thomas CJ, Pittaluga S, Oellerich T, Staudt LM
          </a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1056/NEJMoa1801445" class="" target="_none">Genetics and Pathogenesis of Diffuse Large B Cell Lymphoma</a> by Roland Schmitz, Ph.D., George W. Wright, Ph.D., Da Wei Huang, M.D., Calvin A.  Johnson,
          Ph.D., James D. Phelan, Ph.D., James Q. Wang, Ph.D., Sandrine Roulland, Ph.D., Monica
          Kasbekar, Ph.D., Ryan M. Young, Ph.D., Arthur L. Shaffer, Ph.D., Daniel J. Hodson, M.D.,
          Ph.D., Wenming Xiao, Ph.D., et al.
          </a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="fm-ad" class="dataset">
        <h4><a href="../fm-ad/">Foundation Medicine Adult Cancer Clinical Dataset (FM-AD)</a></h4>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://portal.gdc.cancer.gov/projects/FM-AD" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed?cmd&#x3D;DetailsSearch&amp;term&#x3D;28235761[PMID]" class="" target="_none">High-Throughput Genomic Profiling of Adult Solid Tumors Reveals Novel Insights into Cancer
          Pathogenesis
          </a> by Ryan J. Hartmaier, Lee A. Albacker, Juliann Chmielecki, Mark Bailey, Jie He, Michael E.
          Goldberg, Shakti Ramkissoon, James Suh, Julia A. Elvin, Samuel Chiacchia, Garrett M.
          Frampton, Jeffrey S. Ross, Vincent Miller, Philip J. Stephens and Doron Lipson
          </a>
            <br>
            
          </li>          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed?cmd&#x3D;DetailsSearch&amp;term&#x3D;22981675[PMID]" class="" target="_none">Targeted next-generation sequencing of advanced prostate cancer identifies potential
          therapeutic targets and disease heterogeneity.
          </a> by Beltran H, Yelensky R, Frampton GM, Park K, Downing SR, MacDonald TY, Jarosz M, Lipson D,
          Tagawa ST, Nanus DM, Stephens PJ, Mosquera JM, Cronin MT, Rubin MA
          </a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="human-microbiome-project" class="dataset">
        <h4><a href="../human-microbiome-project/">The Human Microbiome Project</a></h4>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://www.sciencedirect.com/science/article/pii/S2452231718300289" class="" target="_none">New microbe genomic variants in patients fecal community following surgical disruption of the upper human gastrointestinal tract</a> by Ranjit Kumar, Jayleen Grams, Daniel I. Chu, David K.Crossman, Richard Stahl, Peter Eipers, et al</a>
            <br>
            
          </li>          <li>
            <a href="https://www.nature.com/articles/nature23889" class="" target="_none">Strains, functions and dynamics in the expanded Human Microbiome Project</a> by Jason Lloyd-Price, Anup Mahurkar, Gholamali Rahnavard, Jonathan Crabtree, Joshua Orvis, A. Brantley Hall, et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://www.nature.com/articles/nature06244" class="" target="_none">The Human Microbiome Project</a> by Peter J. Turnbaugh, Ruth E. Ley, Micah Hamady, Claire M. Fraser-Liggett, Rob Knight &amp; Jeffrey I. Gordon</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="hail-vep-pipeline" class="dataset">
        <h4><a href="../hail-vep-pipeline/">Variant Effect Predictor (VEP) and the Loss-Of-Function Transcript Effect Estimator (LOFTEE) Plugin</a></h4>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://hail.is/" class="" target="_none">Hail</a> by <a href="https://www.nealelab.is/" target="_none">Neale Lab</a></a>
            <br>
            
          </li>          <li>
            <a href="https://github.com/konradjk/loftee" class="" target="_none">Loss-Of-Function Transcript Effect Estimator (LOFTEE)</a> by <a href="https://github.com/konradjk" target="_none">Konrad Karczewski</a></a>
            <br>
            
          </li>          <li>
            <a href="http://uswest.ensembl.org/info/docs/tools/vep/index.html" class="" target="_none">Variant Effect Predictor (VEP)</a> by <a href="http://uswest.ensembl.org/info/docs/tools/vep/index.html" target="_none">Ensembl</a></a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="broad-references" class="dataset">
        <h4><a href="../broad-references/">Broad Genome References</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://aws.amazon.com/blogs/storage/using-amazon-fsx-for-lustre-for-genomics-workflows-on-aws/" class="aws-link" target="_none">Using Amazon FSx for Lustre for Genomics Workflows on AWS</a> by <a href="https://www.linkedin.com/in/lee-pang-a039a26/" target="_none">W. Lee Pang</a></a>
            <br>
            <span class="label label-info tag service-tag">Batch</span><span class="label label-info tag service-tag">FSx</span>
          </li>        </ul>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://docs.opendata.aws/genomics-workflows/orchestration/cromwell/cromwell-examples/#real-world-example-haplotypecaller" class="aws-link" target="_none">Genomics Workflows on AWS - Cromwell on AWS</a> by <a href="https://www.linkedin.com/in/lee-pang-a039a26/" target="_none">W. Lee Pang</a></a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="cgci" class="dataset">
        <h4><a href="../cgci/">Cancer Genome Characterization Initiatives - Burkitt Lymphoma, HIV+ Cervical Cancer</a></h4>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://portal.gdc.cancer.gov/projects/CGCI-BLGSP" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1182/blood-2018-09-871418" class="" target="_none">Genome-wide discovery of somatic coding and noncoding mutations in pediatric endemic and
          sporadic Burkitt lymphoma
          </a> by Grande B. M., Gerhard D. S., Jiang A., Griner N. B., Abramson J. S., Alexander T. B., et al.
          </a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="jhu-indexes" class="dataset">
        <h4><a href="../jhu-indexes/">Cloud Indexes for Genomic Analyses</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://benlangmead.github.io/aws-indexes/tutorial" class="" target="_none">Table of contents for tutorials for constituent tools</a> by <a href="https://benlangmead.github.io/aws-indexes" target="_none">Ben Langmead</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://www.biorxiv.org/content/10.1101/2020.03.03.975219v4.full.pdf" class="" target="_none">Reducing reference bias using multiple population reference genomes</a> by Chen et al (2020)</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="nciccr-dlbcl" class="dataset">
        <h4><a href="../nciccr-dlbcl/">National Cancer Institute Center for Cancer Research - Diffuse Large B Cell Lymphoma (DLBCL) Genomics and Expression</a></h4>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://portal.gdc.cancer.gov/projects/NCICCR-DLBCL" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1056/NEJMoa1801445" class="" target="_none">Genetics and Pathogenesis of Diffuse Large B Cell Lymphoma</a> by Roland Schmitz, Ph.D., George W. Wright, Ph.D., Da Wei Huang, M.D., et al.</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="ohsu-cnl" class="dataset">
        <h4><a href="../ohsu-cnl/">Oregon Health &amp; Science University Chronic Neutrophilic Leukemia Dataset</a></h4>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://portal.gdc.cancer.gov/projects/" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/31366621" class="" target="_none">Genomic landscape of neutrophilic leukemias of ambiguous diagnosis</a> by Zhang H, Wilmot B, Bottomly D et al.</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="organoid-pancreatic" class="dataset">
        <h4><a href="../organoid-pancreatic/">Pancreatic Cancer Organoid Profiling</a></h4>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://portal.gdc.cancer.gov/projects/ORGANOID-PANCREATIC" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed?cmd&#x3D;DetailsSearch&amp;term&#x3D;29853643[PMID]" class="" target="_none">Organoid Profiling Identifies Common Responders to Chemotherapy in Pancreatic Cancer</a> by Tiriac H, Belleau P, Engle DD, Plenker D, Deschênes A, Somerville TD, et al.</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="serratus-lovelywater" class="dataset">
        <h4><a href="../serratus-lovelywater/">Serratus: Ultra-deep Search for Novel Viruses - Versioned Data Release</a></h4>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://serratus.io/explorer" class="" target="_none">Serratus Explorer</a> by Serratus Team</a>
            <br>
            
          </li>          <li>
            <a href="https://github.com/serratus-bio/tantalus" class="" target="_none">Tantalus: An R Package for exploration of Serratus data</a> by Serratus Team</a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://www.biorxiv.org/content/10.1101/2020.07.28.224584v2" class="" target="_none">coronaSPAdes. From biosynthetic gene clusters to RNA viral assemblies</a> by Meleshko D., Hajirasouliha I., and Korobeynikov A. (2021)</a>
            <br>
            
          </li>          <li>
            <a href="https://www.pnas.org/content/118/3/e2019907118" class="" target="_none">Diversification of mammalian deltaviruses by host shifting</a> by Bergner L.M., Orton R.J., et al (2021)</a>
            <br>
            
          </li>          <li>
            <a href="https://www.biorxiv.org/content/10.1101/2020.08.07.241729v1" class="" target="_none">Petabase-scale sequence alignment catalyses viral discovery</a> by Edgar R., Taylor J., Altman T., Barbera P., Meleshko D., Lin V., et al (2020)</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="1000-genomes" class="dataset">
        <h4><a href="../1000-genomes/">1000 Genomes</a></h4>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://aws.amazon.com/blogs/big-data/exploratory-data-analysis-of-genomic-datasets-using-adam-and-mango-with-apache-spark-on-amazon-emr/" class="aws-link" target="_none">Exploratory data analysis of genomic datasets using ADAM and Mango with Apache Spark on Amazon EMR</a> by <a href="https://research.eecs.berkeley.edu/~akmorrow/" target="_none">Alyssa Marrow</a></a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="aws-igenomes" class="dataset">
        <h4><a href="../aws-igenomes/">AWS iGenomes</a></h4>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="http://nf-co.re/" class="" target="_none">nf-core analysis pipelines</a> by <a href="http://phil.ewels.co.uk/" target="_none">Phil Ewels</a></a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="ncbi-covid-19" class="dataset">
        <h4><a href="../ncbi-covid-19/">COVID-19 Genome Sequence Dataset</a></h4>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/sra/docs/sra-aws-download/" class="" target="_none">Download SRA sequence data using Amazon Web Services (AWS)</a> by NCBI SRA</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="gatk-test-data" class="dataset">
        <h4><a href="../gatk-test-data/">GATK Test Data</a></h4>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://docs.opendata.aws/genomics-workflows/orchestration/cromwell/cromwell-examples/#real-world-example-haplotypecaller" class="aws-link" target="_none">Genomics Workflows on AWS - Cromwell on AWS</a> by <a href="https://www.linkedin.com/in/lee-pang-a039a26/" target="_none">W. Lee Pang</a></a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="hcmi-cmdc" class="dataset">
        <h4><a href="../hcmi-cmdc/">Human Cancer Models Initiative (HCMI) Cancer Model Development Center</a></h4>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://portal.gdc.cancer.gov/projects/HCMI-CMDC" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="hpgp-data" class="dataset">
        <h4><a href="../hpgp-data/">Human PanGenomics Project</a></h4>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://www.nature.com/articles/s41587-020-0503-6" class="" target="_none">Nanopore sequencing and the Shasta toolkit enable efficient de novo assembly of eleven human genomes</a> by Shafin et al (2020)</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="ont-open-data" class="dataset">
        <h4><a href="../ont-open-data/">Oxford Nanopore Technologies Benchmark Datasets</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://nanoporetech.github.io/ont-open-datasets/tutorials" class="" target="_none">ONT Dataset Tutorials</a> by <a href="https://labs.epi2me.io/" target="_none">EPI2MELabs</a></a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="qiime2" class="dataset">
        <h4><a href="../qiime2/">QIIME 2 User Tutorial Datasets</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://docs.qiime2.org/2018.11/install/virtual/aws/" class="" target="_none">Installing QIIME 2 using Amazon Web Services</a> by <a href="https://github.com/qiime2/docs/graphs/contributors" target="_none">The QIIME 2 Development Team</a></a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="tabula-muris" class="dataset">
        <h4><a href="../tabula-muris/">Tabula Muris</a></h4>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/30283141" class="" target="_none">Single-cell transcriptomics of 20 mouse organs creates a Tabula Muris.</a> by <a href="https://tabula-muris.ds.czbiohub.org/" target="_none">Tabula Muris Consortium (2019)</a></a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="ihart" class="dataset">
        <h4><a href="../ihart/">iHART Whole Genome Sequencing Data Set</a></h4>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://www.cell.com/cell/fulltext/S0092-8674(19)30780-9" class="" target="_none">Inherited and De Novo Genetic Risk for Autism Impacts Shared Networks</a> by Ruzzo et al. (2020)</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="binding-db" class="dataset">
        <h4><a href="../binding-db/">Binding DB</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/aws-samples/data-lake-as-code/blob/roda/docs/roda_install.md" class="aws-link" target="_none">Data Lake as Code Deployment Guide</a> by AWS Biotech Blueprints Team</a>
            <br>
            <span class="label label-info tag service-tag">Athena</span><span class="label label-info tag service-tag">Glue</span><span class="label label-info tag service-tag">Lake Formation</span>
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://aws.amazon.com/blogs/startups/a-data-lake-as-code-featuring-chembl-and-opentargets/" class="aws-link" target="_none">Data Lake as Code, Featuring ChEMBL and Open Targets</a> by Paul Underwood</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="chembl" class="dataset">
        <h4><a href="../chembl/">ChEMBL</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/aws-samples/data-lake-as-code/blob/roda/docs/roda_install.md" class="aws-link" target="_none">Data Lake as Code Deployment Guide</a> by AWS Biotech Blueprints Team</a>
            <br>
            <span class="label label-info tag service-tag">Athena</span><span class="label label-info tag service-tag">Glue</span><span class="label label-info tag service-tag">Lake Formation</span>
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://aws.amazon.com/blogs/startups/a-data-lake-as-code-featuring-chembl-and-opentargets/" class="aws-link" target="_none">Data Lake as Code, Featuring ChEMBL and Open Targets</a> by Paul Underwood</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="clinvar" class="dataset">
        <h4><a href="../clinvar/">ClinVar</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/aws-samples/data-lake-as-code/blob/roda/docs/roda_install.md" class="aws-link" target="_none">Data Lake as Code Deployment Guide</a> by AWS Biotech Blueprints Team</a>
            <br>
            <span class="label label-info tag service-tag">Athena</span><span class="label label-info tag service-tag">Glue</span><span class="label label-info tag service-tag">Lake Formation</span>
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://aws.amazon.com/blogs/startups/a-data-lake-as-code-featuring-chembl-and-opentargets/" class="aws-link" target="_none">Data Lake as Code, Featuring ChEMBL and Open Targets</a> by Paul Underwood</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="opentargets" class="dataset">
        <h4><a href="../opentargets/">Open Targets</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/aws-samples/data-lake-as-code/blob/roda/docs/roda_install.md" class="aws-link" target="_none">Data Lake as Code Deployment Guide</a> by AWS Biotech Blueprints Team</a>
            <br>
            <span class="label label-info tag service-tag">Athena</span><span class="label label-info tag service-tag">Glue</span><span class="label label-info tag service-tag">Lake Formation</span>
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://aws.amazon.com/blogs/startups/a-data-lake-as-code-featuring-chembl-and-opentargets/" class="aws-link" target="_none">Data Lake as Code, Featuring ChEMBL and Open Targets</a> by Paul Underwood</a>
            <br>
            
          </li>        </ul>
      </div>

    <hr/>
    <p>If you want to add a dataset or usage example to this registry, please follow the instructions on the <a href="https://github.com/awslabs/open-data-registry/">Registry of Open Data on AWS GitHub repository</a>.</p>
    <p><a href="/"><b>Home</b></a></p>
  </div>