<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="google-site-verification" content="d3RoBpynB-G21xjnmrqTE7iJfzqgS6DugRe7B6yUVb8" />
  
    
  
    <link rel="alternate" type="application/rss+xml" href="/rss.xml" />
    
    <link rel="shortcut icon" type="image/ico" href="https://a0.awsstatic.com/libra-css/images/site/fav/favicon.ico"/>
    <link rel="apple-touch-icon" sizes="57x57" href="https://a0.awsstatic.com/libra-css/images/site/touch-icon-iphone-114-smile.png"/>
    <link rel="apple-touch-icon" sizes="72x72" href="https://a0.awsstatic.com/libra-css/images/site/touch-icon-ipad-144-smile.png"/>
    <link rel="apple-touch-icon" sizes="114x114" href="https://a0.awsstatic.com/libra-css/images/site/touch-icon-iphone-114-smile.png"/>
    <link rel="apple-touch-icon" sizes="144x144" href="https://a0.awsstatic.com/libra-css/images/site/touch-icon-ipad-144-smile.png"/>  
  
    <title>Registry of Open Data on AWS</title>
  
    <!-- Latest compiled and minified CSS -->
    <link rel="stylesheet" href="https://assets.opendata.aws/css/bootstrap/3.4.1/bootstrap.min.css">
  
    <!-- Our local CSS -->
    <link rel="stylesheet" href="css/main.css">
  
  
  </head>
  <body>
    <!-- jQuery (necessary for Bootstrap's JavaScript plugins) -->
    <script src="https://assets.opendata.aws/js/jquery/3.5.1/jquery.min.js"></script>
    <!-- Latest compiled and minified JavaScript -->
    <script src="https://assets.opendata.aws/js/bootstrap/3.4.1/bootstrap.min.js"></script>
    <nav class="navbar navbar-default navbar-fixed-top">
      <div class="container">
        <div class="roda-header">
          <h1>Registry of Open Data on AWS</h1>
          <a href="http://aws.amazon.com/what-is-cloud-computing">
            <img src="https://assets.opendata.aws/img/AWS-Logo_White-Color_300x180.png" alt="Powered by AWS Cloud Computing" id="aws_header_logo">
          </a>
        </div>
      </div>
    </nav>
    <div class="container" >

    <div class="about col-md-5">
      <div class="row aboutbox">
            <h3>About</h3>
            <p>This registry exists to help people discover and share datasets that are available via AWS resources. <a href="https://opendata.aws">Learn more about sharing data on AWS</a>.</p>
    
            <p>See <a href="/usage-examples">all usage examples for datasets listed in this registry</a>.</p>
            <p>See datasets from <a href="/collab/facebook/">Facebook Data for Good</a>, <a href="/collab/nasa/">NASA Space Act Agreement</a>, <a href="/collab/nih/">NIH STRIDES</a>, <a href="/collab/noaa/">NOAA Big Data Program</a>, <a href="/collab/stsci/">Space Telescope Science Institute</a>, and <a href="/collab/asdi/">Amazon Sustainability Data Initiative</a>.
            </p>
    
        <hr>
    
        <h4>Search datasets (currently <span id="count-matching">13</span> matching <span id="count-matching-text">datasets</span>)</h4>
        <form>
          <div class="form-group">
            <input type="text" id="search-box" class="form-control" placeholder="Search datasets" spellcheck="false" autocorrect="off">
          </div>
        </form>
        
    
        <hr>
    
        <h4>Add to this registry</h4>
        <p>If you want to add a dataset or example of how to use a dataset to this registry, please follow the instructions on the <a href="https://github.com/awslabs/open-data-registry/">Registry of Open Data on AWS GitHub repository</a>.</p>
    
        <p>Unless specifically stated in the applicable dataset documentation, datasets available through the Registry of Open Data on AWS are not provided and maintained by AWS. Datasets are provided and maintained by a variety of third parties under a variety of licenses. Please check dataset licenses and related documentation to determine if a dataset may be used for your application.</p>
      </div>
    </div>
    <div class="col-md-6 col-md-offset-1 datasets">
      <div class="row">
        <div id="tcga" class="dataset">
          <h3><a href="/tcga/">The Cancer Genome Atlas</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">STRIDES</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>The Cancer Genome Atlas (TCGA), a collaboration between the National Cancer Institute (NCI) and National Human Genome Research Institute (NHGRI), aims to generate comprehensive, multi-dimensional maps of the key genomic changes in major types and subtypes of cancer. TCGA has analyzed matched tumor and normal tissues from 11,000 patients, allowing for the comprehensive characterization of 33 cancer types and subtypes, including 10 rare cancers.
The dataset contains open Clinical Supplement, Biospecimen Supplement, RNA-Seq Gene Expression Quantification, miRNA-Seq Isoform Expression Quantificati...</p>
          <p><a href="/tcga/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1016/j.celrep.2018.03.076" class="" target="_none">Genomic and Molecular Landscape of DNA Damage Repair Deficiency across The Cancer Genome
Atlas
</a> by Theo A. Knijnenburg, Linghua Wang, et al.</li></a>
          </li>
          <li>
            <a href="https://firecloud.terra.bio/" class="" target="_none">Broad Institute FireCloud</a> by <a href="https://www.broadinstitute.org/" target="_none">The Broad Institute of MIT &amp; Harvard</a></li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1016/j.celrep.2018.03.086" class="" target="_none">Spatial Organization And Molecular Correlation Of Tumor-Infiltrating Lymphocytes Using Deep
Learning On Pathology Images
</a> by Joel Saltz, Rajarsi Gupta, et al.</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1016/j.cell.2018.03.034" class="" target="_none">Machine Learning Identifies Stemness Features Associated with Oncogenic Dedifferentiation
</a> by Tathiane M. Malta, Artem Sokolov, et al.</li></a>
          </li>
          <li>
            <a href="https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga/using-tcga
" class="" target="_none">Using TCGA Data, Resources, and Materials</a> by <a href="https://ocg.cancer.gov/" target="_none">National Cancer Institute</a></li></a>
          </li>
        </ul>
        <p><a href="/tcga/#usageexamples">See 29 usage examples &rarr;</a></p>
        </div>
        <div id="target" class="dataset">
          <h3><a href="/target/">Therapeutically Applicable Research to Generate Effective Treatments (TARGET)</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">STRIDES</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>Therapeutically Applicable Research to Generate Effective Treatments (TARGET) is the collaborative effort of a large, diverse consortium of extramural and NCI investigators. The goal of the effort is to accelerate molecular discoveries that drive the initiation and progression of hard-to-treat childhood cancers and facilitate rapid translation of those findings into the clinic.
TARGET projects provide comprehensive molecular characterization to determine the genetic changes that drive the initiation and progression of childhood cancers.The dataset contains open Clinical Supplement, Biospecimen...</p>
          <p><a href="/target/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://ocg.cancer.gov/programs/target/data-matrix" class="" target="_none">TARGET data matrix</a> by <a href="https://ocg.cancer.gov/" target="_none">National Cancer Institute</a></li></a>
          </li>
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/21297632" class="" target="_none">Ancestry and pharmacogenomics of relapse in acute lymphoblastic leukemia</a> by Yang JJ, Cheng C, Devidas M, et al.</li></a>
          </li>
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term&#x3D;29068783" class="" target="_none">MicroRNA Expression-Based Model Indicates Event-Free Survival in Pediatric Acute Myeloid
Leukemia
</a> by Lim EL, Trinh DL, Ries RE, et al.</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1038/ng.3940" class="" target="_none">A Children&#x27;s Oncology Group and TARGET initiative exploring the genetic landscape of Wilms
tumor
</a> by Gadd S, Huff V, Walz AL, et al.</li></a>
          </li>
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term&#x3D;29227476" class="" target="_none">The molecular landscape of pediatric acute myeloid leukemia reveals recurrent structural
alterations and age-specific mutational interactions
</a> by Bolouri H, Farrar JE, Triche T Jr, et al.</li></a>
          </li>
        </ul>
        <p><a href="/target/#usageexamples">See 24 usage examples &rarr;</a></p>
        </div>
        <div id="commoncrawl" class="dataset">
          <h3><a href="/commoncrawl/">Common Crawl</a></h3>
          <p><span class="label label-info tag link-tag">encyclopedic</span><span class="label label-info tag link-tag">internet</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">natural language processing</span></p>
          <p>A corpus of web crawl data composed of over 50 billion web pages.</p>
          <p><a href="/commoncrawl/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://commoncrawl.org/2018/03/index-to-warc-files-and-urls-in-columnar-format/" class="" target="_none">Index to WARC Files and URLs in Columnar Format</a> by Sebastian Nagel</li></a>
          </li>
          <li>
            <a href="https://www.aclweb.org/anthology/L18-1550" class="" target="_none">Learning word vectors for 157 languages</a> by <a href="https://fasttext.cc/docs/en/crawl-vectors.html" target="_none">Facebook AI Research</a></li></a>
          </li>
          <li>
            <a href="https://arxiv.org/abs/2005.14165" class="" target="_none">Language models are few-shot learners</a> by Tom B. Brown, Benjamin Mann, Nick Ryder, Melanie Subbiah, Jared Kaplan, Prafulla Dhariwal, et al</li></a>
          </li>
          <li>
            <a href="https://www.aclweb.org/anthology/2020.emnlp-main.480" class="" target="_none">CCAligned: A Massive collection of cross-lingual web-document pairs</a> by <a href="http://www.statmt.org/cc-aligned/" target="_none">Ahmed El-Kishky, Vishrav Chaudhary, Francisco Guzmán, Philipp Koehn</a></li></a>
          </li>
          <li>
            <a href="https://psuter.net/2019/07/07/z-index" class="" target="_none">Index fun</a> by Philippe Suter</li></a>
          </li>
        </ul>
        <p><a href="/commoncrawl/#usageexamples">See 23 usage examples &rarr;</a></p>
        </div>
        <div id="sentinel-2" class="dataset">
          <h3><a href="/sentinel-2/">Sentinel-2</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">natural resource</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info-sustainability tag link-tag">sustainability</span></p>
          <p>The <a href="https://sentinel.esa.int/web/sentinel/missions/sentinel-2">Sentinel-2 mission</a> is
a land monitoring constellation of two satellites that provide high resolution
optical imagery and provide continuity for the current SPOT and Landsat missions.
The mission provides a global coverage of the Earth&#39;s land surface every 5 days,
making the data of great use in on-going studies. L1C data are available from
June 2015 globally. L2A data are available from September 2016 over wider Europe
region and globally since January 2017.</p>
          <p><a href="/sentinel-2/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="http://www.sentinel-hub.com/apps/wms" class="" target="_none">Sentinel Hub WMS/WMTS/WCS Service</a> by <a href="http://www.sinergise.com/" target="_none">Sinergise</a></li></a>
          </li>
          <li>
            <a href="https://www.sciencedirect.com/science/article/pii/S0034425720304284" class="" target="_none">Coral-spawn slicks: Reflectance spectra and detection using optical satellite data</a> by Hiroya Yamano, Asahi Sakuma, Saki Harii</li></a>
          </li>
          <li>
            <a href="https://github.com/sentinel-hub/sentinelhub-py" class="" target="_none">Python package for working with Sentinel-2 AWS data</a> by <a href="http://www.sinergise.com/" target="_none">Sinergise</a></li></a>
          </li>
          <li>
            <a href="https://eos.com/landviewer/" class="" target="_none">EOS Land Viewer</a> by <a href="https://eos.com/" target="_none">Earth Observing System</a></li></a>
          </li>
          <li>
            <a href="https://ieeexplore.ieee.org/abstract/document/9165649" class="" target="_none">Using Remote Sensing Images and Cloud Services on AWS to Improve Land Use and Cover Monitoring</a> by K. R. Ferreira, et al.</li></a>
          </li>
        </ul>
        <p><a href="/sentinel-2/#usageexamples">See 20 usage examples &rarr;</a></p>
        </div>
        <div id="kids-first" class="dataset">
          <h3><a href="/kids-first/">Gabriella Miller Kids First Pediatric Research Program (Kids First)</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">Homo sapiens</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">pediatric</span><span class="label label-info tag link-tag">STRIDES</span><span class="label label-info tag link-tag">structural birth defect</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>The NIH Common Fund&#39;s Gabriella Miller Kids First Pediatric Research Program’s (“Kids First”) vision is to “alleviate suffering from childhood cancer and structural birth defects by fostering collaborative research to uncover the etiology of these diseases and by supporting data sharing within the pediatric research community.” The program continues to generate and share whole genome sequence data from thousands of children affected by these conditions, ranging from rare pediatric cancers, such as osteosarcoma, to more prevalent diagnoses, such as congenital heart defects. In 2018, Kids Fi...</p>
          <p><a href="/kids-first/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://pubmed.ncbi.nlm.nih.gov/32574564/" class="" target="_none">Genome-wide Enrichment of De Novo Coding Mutations in Orofacial Cleft Trios.</a> by Madison R Bishop, Kimberly K Diaz Perez, et al.</li></a>
          </li>
          <li>
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/31397880" class="" target="_none">MAGEL2-Related Disorders: A study and case series.</a> by Jameson Patak, James Gilfert, et al.</li></a>
          </li>
          <li>
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/31255796" class="" target="_none">Development and Clinical Validation of a Large Fusion Gene Panel for Pediatric Cancers.</a> by Fengqi Chang, Fumin Lin, et al.</li></a>
          </li>
          <li>
            <a href="https://pubmed.ncbi.nlm.nih.gov/30322880/" class="" target="_none">Clinically Relevant and Minimally Invasive Tumor Surveillance of Pediatric Diffuse Midline Gliomas Using Patient-Derived Liquid Biopsy</a> by Eshini Panditharatna, Lindsay B Kilburn, et al.</li></a>
          </li>
          <li>
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/31211835" class="" target="_none">Decreased ACKR3 (CXCR7) function causes oculomotor synkinesis in mice and humans.</a> by Mary C Whitman, Noriko Miyake, et al.</li></a>
          </li>
        </ul>
        <p><a href="/kids-first/#usageexamples">See 19 usage examples &rarr;</a></p>
        </div>
        <div id="landsat-8" class="dataset">
          <h3><a href="/landsat-8/">Landsat 8</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">natural resource</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info-sustainability tag link-tag">sustainability</span></p>
          <p>An ongoing collection of satellite imagery of all land on Earth produced by
the Landsat 8 satellite.<strong>Note this dataset is deprecated in favor of the <a href="https://www.usgs.gov/core-science-systems/nli/landsat/landsat-commercial-cloud-data-access">USGS cloud access mechanism</a>. This bucket will be deleted on July 1, 2021.</strong></p>
          <p><a href="/landsat-8/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://docs.safe.com/fme/html/FME_Desktop_Documentation/FME_ReadersWriters/landsat8aws/landsat8aws.htm" class="" target="_none">FME Landsat-8 on AWS Reader</a> by <a href="https://www.safe.com/" target="_none">Safe Software</a></li></a>
          </li>
          <li>
            <a href="https://kylebarron.dev/blog/cog-mosaic/overview" class="" target="_none">Dynamic map tiling with Cloud-Optimized GeoTIFFs</a> by Kyle Barron</li></a>
          </li>
          <li>
            <a href="https://apps.sentinel-hub.com/sentinel-playground/?source&#x3D;L8" class="" target="_none">Sentinel Playground for Landsat</a> by <a href="http://www.sinergise.com/" target="_none">Sinergise</a></li></a>
          </li>
          <li>
            <a href="https://geotiffjs.github.io/cog-explorer/" class="" target="_none">COG-Explorer - View Cloud Optimized GeoTIFF images in the browser directly from object storage</a> by <a href="https://eox.at/" target="_none">EOX</a></li></a>
          </li>
          <li>
            <a href="https://landsatlive.live" class="" target="_none">landsatlive.live</a> by <a href="https://developmentseed.org/" target="_none">Development Seed</a></li></a>
          </li>
        </ul>
        <p><a href="/landsat-8/#usageexamples">See 18 usage examples &rarr;</a></p>
        </div>
        <div id="sudachi" class="dataset">
          <h3><a href="/sudachi/">Sudachi Language Resources</a></h3>
          <p><span class="label label-info tag link-tag">natural language processing</span></p>
          <p>Japanese dictionaries and word embeddings for natural language processing.
<a href="https://github.com/WorksApplications/SudachiDict">SudachiDict</a> is the dictionary for a Japanese tokenizer (morphological analyzer) <a href="https://github.com/WorksApplications/Sudachi">Sudachi</a>.
<a href="https://github.com/WorksApplications/chiVe">chiVe</a> is Japanese pretrained word embeddings (word vectors), trained using the ultra-large-scale web corpus NWJC by National Institute for Japanese Langauge and Linguistics, analyzed by Sudachi.</p>
          <p><a href="/sudachi/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.anlp.jp/proceedings/annual_meeting/2019/pdf_dir/P8-5.pdf" class="" target="_none">複数粒度の分割結果に基づく日本語単語分散表現</a> by 真鍋陽俊, 岡照晃, 海川祥毅, 髙岡一馬, 内田佳孝, 浅原正幸</li></a>
          </li>
          <li>
            <a href="https://www.anlp.jp/proceedings/annual_meeting/2020/pdf_dir/P4-14.pdf" class="" target="_none">詳細化した同義関係をもつ同義語辞書の作成</a> by 高岡一馬, 岡部裕子, 川原典子, 坂本美保, 内田佳孝</li></a>
          </li>
          <li>
            <a href="https://github.com/WorksApplications/Sudachi/blob/develop/docs/tutorial.md" class="" target="_none">Sudachi Tutorial</a> by Works Applications</li></a>
          </li>
          <li>
            <a href="https://github.com/WorksApplications/SudachiPy/blob/develop/docs/tutorial.md" class="" target="_none">SudachiPy Tutorial</a> by Works Applications</li></a>
          </li>
          <li>
            <a href="http://www.lrec-conf.org/proceedings/lrec2018/summaries/8884.html" class="" target="_none">Sudachi: a Japanese Tokenizer for Business</a> by Kazuma Takaoka, Sorami Hisamoto, Noriko Kawahara, Miho Sakamoto, Yoshitaka Uchida, Yuji Matsumoto</li></a>
          </li>
        </ul>
        <p><a href="/sudachi/#usageexamples">See 18 usage examples &rarr;</a></p>
        </div>
        <div id="foldingathome-covid19" class="dataset">
          <h3><a href="/foldingathome-covid19/">Foldingathome COVID-19 Datasets</a></h3>
          <p><span class="label label-info tag link-tag">alchemical free energy calculations</span><span class="label label-info tag link-tag">biomolecular modeling</span><span class="label label-info tag link-tag">coronavirus</span><span class="label label-info tag link-tag">COVID-19</span><span class="label label-info tag link-tag">foldingathome</span><span class="label label-info tag link-tag">health</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">molecular dynamics</span><span class="label label-info tag link-tag">protein</span><span class="label label-info tag link-tag">SARS-CoV-2</span><span class="label label-info tag link-tag">simulations</span><span class="label label-info tag link-tag">structural biology</span></p>
          <p><a href="http://foldingathome.org">Folding@home</a> is a massively distributed computing project that uses biomolecular simulations to investigate the <a href="https://foldingathome.org/diseases/">molecular origins of disease</a> and accelerate the discovery of new therapies. Run by the <a href="https://foldingathome.org/about/the-foldinghome-consortium/">Folding@home Consortium</a>, a worldwide network of research laboratories focusing on a variety of different diseases, Folding@home seeks to address problems in human health on a scale that is infeasible by another other means, sharing the results of these large-scale studies with the research community through <a href="https://foldingathome.org/papers-results/">peer-reviewed publications</a> and publicly shared datasets. During the <a href="https://en.wikipedia.org/wiki/Coronavirus_disease_2019">COVID-19 epidemic</a>, Folding@home focused its resources on understanding the vulernabilities in <a href="https://en.wikipedia.org/wiki/Severe_acute_respiratory_syndrome_coronavirus_2">SARS-CoV-2</a>, the virus that causes COVID-19 disease, and working closely with a number of experimental collaborators to accelerate progress toward effective therapies for treating COVID-19 and ending the pandemic. In the process, it created the world&#39;s first <a href="https://doi.org/10.1101/2020.06.27.175430">exascale distributed computing resource</a>, enabling it to generate valuable scientific datasets of unprecedented size. More information about Folding@home&#39;s COVID-19 research activities at the <a href="https://foldingathome.org/diseases/infectious-diseases/covid-19/">Folding@home COVID-19 page</a>. In addition to working directly with experimental collaborators and rapidly sharing new research findings through preprint servers, Folding@home has joined other researchers in committing to <a href="https://doi.org/10.1021/acs.jcim.0c00319">rapidly share all COVID-19 research data</a>, and has joined forces with <a href="https://aws.amazon.com/">AWS</a> and the <a href="http://molssi.org">Molecular Sciences Software Institute (MolSSI)</a> to share datasets of unprecented side through the <a href="https://registry.opendata.aws/">AWS Open Data Registry</a>, indexing these massive datsets via the <a href="https://covid.molssi.org/">MolSSI COVID-19 Molecular Structure and Therapeutics Hub</a>. The complete index of all Folding@home datasets can be found <a href="https://covid.molssi.org//org-contributions/#folding--home">here</a>. Th...</p>
          <p><a href="/foldingathome-covid19/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://covid.molssi.org//simulations/#foldinghome-simulations-of-nsp3-macrodomain-macrodomain-host-immune-response" class="" target="_none">SARS-CoV-2 nsp3 macrodomain dataset: An 11 ms dataset of the SARS-CoV-2 nsp3 macrodomain in search of cryptic pockets</a> by <a href="https://bowmanlab.biochem.wustl.edu" target="_none">The Bowman lab at Washington University in St. Louis</a></li></a>
          </li>
          <li>
            <a href="https://covid.molssi.org//simulations/#foldinghome-simulations-of-nsp7-nsp7-nil" class="" target="_none">SARS-CoV-2 nsp7 simulations: A 3.7 ms dataset of the SARS-CoV-2 nsp7 protein in search of cryptic pockets</a> by <a href="https://bowmanlab.biochem.wustl.edu" target="_none">The Bowman lab at Washington University in St. Louis</a></li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1101/2020.06.27.175430" class="" target="_none">SARS-CoV-2 Simulations Go Exascale to Capture Spike Opening and Reveal Cryptic Pockets Across the Proteome</a> by Maxwell I. Zimmerman, Justin R. Porter, Michael D. Ward, Sukrit Singh, Neha Vithani, Artur Meller, Upasana L. Mallimadugula, Catherine E. Kuhn, Jonathan H. Borowsky,  View ORCID ProfileRafal P. Wiewiora, Matthew F. D. Hurley, Aoife M Harbison, Carl A Fogarty, Joseph E. Coffland, Elisa Fadda, Vincent A. Voelz, John D. Chodera, Gregory R. Bowman</li></a>
          </li>
          <li>
            <a href="https://covid.molssi.org//simulations/#foldinghome-simulations-of-nsp10-nsp10-nil" class="" target="_none">SARS-CoV-2 nsp10 dataset: A 6.1 ms dataset of the SARS-CoV-2 nsp10 protein in search of cryptic pockets</a> by <a href="https://bowmanlab.biochem.wustl.edu" target="_none">The Bowman lab at Washington University in St. Louis</a></li></a>
          </li>
          <li>
            <a href="https://covid.molssi.org//simulations/#foldinghome-simulations-of-nsp5-3clpro-3clpro-protease-activity" class="" target="_none">SARS-CoV-2 main viral protease (Mpro, 3CLPro, nsp5) dimer simulations: A 2.9 ms dataset of the SARS-CoV-2 main viral protease (apo, dimer) in search of cryptic pockets</a> by <a href="https://bowmanlab.biochem.wustl.edu" target="_none">The Bowman lab at Washington University in St. Louis</a></li></a>
          </li>
        </ul>
        <p><a href="/foldingathome-covid19/#usageexamples">See 15 usage examples &rarr;</a></p>
        </div>
        <div id="broad-gnomad" class="dataset">
          <h3><a href="/broad-gnomad/">Genome Aggregation Database (gnomAD)</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">population</span><span class="label label-info tag link-tag">population genetics</span><span class="label label-info tag link-tag">short read sequencing</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>The Genome Aggregation Database (gnomAD) is a resource developed by an international coalition of investigators that aggregates and harmonizes both exome and genome data from a wide range of large-scale human sequencing projects. The summary data provided here are released for the benefit of the wider scientific community without restriction on use.
The v2 data set (GRCh37) spans 125,748 exome sequences and 15,708 whole-genome sequences from unrelated individuals. The v3 data set (GRCh38) spans 71,702 genomes, selected as in v2.
Sign up for the gnomAD mailing list <a href="http://broad.io/gnomad_list">here</a>.</p>
          <p><a href="/broad-gnomad/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://pypi.org/project/gnomad/" class="" target="_none">Hail utilities for gnomAD</a> by <a href="https://github.com/orgs/broadinstitute/teams/gnomad/members" target="_none">gnomAD Production Team</a></li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1038/s41467-019-12438-5" class="" target="_none">Landscape of multi-nucleotide variants in 125,748 human exomes and 15,708 genomes. Nature Communications 11, 2539 (2020)</a> by Wang, Q., Pierce-Hoffman, E., Cummings, B. B., Karczewski, K. J., Alföldi, J., Francioli, L. C., Gauthier, L. D., Hill, A. J., O’Donnell-Luria, A. H., Genome Aggregation Database (gnomAD) Production Team, Genome Aggregation Database (gnomAD) Consortium, &amp; MacArthur, D. G.</li></a>
          </li>
          <li>
            <a href="https://aws.amazon.com/quickstart/architecture/hail/" class="aws-link" target="_none">Hail on AWS Quick Start</a> by Amazon Web Services and PrivoIT</li></a>
          </li>
          <li>
            <a href="https://github.com/broadinstitute/gnomad_qc" class="" target="_none">gnomAD quality control GitHub repository</a> by <a href="https://github.com/orgs/broadinstitute/teams/gnomad/members" target="_none">gnomAD Production Team</a></li></a>
          </li>
          <li>
            <a href="https://gnomad.broadinstitute.org/blog/2019-10-gnomad-v3-0/" class="" target="_none">gnomAD v3.0</a> by Laurent Francioli, Daniel MacArthur</li></a>
          </li>
        </ul>
        <p><a href="/broad-gnomad/#usageexamples">See 15 usage examples &rarr;</a></p>
        </div>
        <div id="noaa-nexrad" class="dataset">
          <h3><a href="/noaa-nexrad/">NEXRAD on AWS</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">natural resource</span><span class="label label-info-sustainability tag link-tag">sustainability</span><span class="label label-info tag link-tag">weather</span></p>
          <p>Real-time and archival data from the Next Generation Weather Radar (NEXRAD) network.</p>
          <p><a href="/noaa-nexrad/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.roc.noaa.gov/wsr88d/PublicDocs/ICDs/2620010G.pdf" class="" target="_none">Level 2 Interface Control Document for Transfer: Build 18</a> by NOAA ROC</li></a>
          </li>
          <li>
            <a href="http://nbviewer.jupyter.org/gist/dopplershift/356f2e14832e9b676207" class="" target="_none">Using Python to Access NCEI Archived NEXRAD Level 2 Data (Jupyter notebook)</a> by <a href="http://dopplershift.github.io" target="_none">Ryan May</a></li></a>
          </li>
          <li>
            <a href="https://carto.com/blog/mapping-nexrad-radar-data/" class="" target="_none">Mapping Noaa Nexrad Radar Data With CARTO</a> by <a href="https://carto.com/blog/author/stuart-lynn/" target="_none">Stuart Lynn</a></li></a>
          </li>
          <li>
            <a href="https://agupubs.onlinelibrary.wiley.com/doi/abs/10.1029/2020GL089001" class="" target="_none">Extreme Pyroconvective Updrafts During a Megafire</a> by B. Rodriguez, N. P. Lareau, D. E. Kingsmill, &amp; C. B. Clements</li></a>
          </li>
          <li>
            <a href="https://github.com/stephenlienharrell/WeatherPipe" class="" target="_none">WeatherPipe - Amazon EMR based analysis tool for NEXRAD data stored on Amazon S3</a> by <a href="https://github.com/stephenlienharrell" target="_none">Stephen Lien Harrell</a></li></a>
          </li>
        </ul>
        <p><a href="/noaa-nexrad/#usageexamples">See 14 usage examples &rarr;</a></p>
        </div>
        <div id="terrain-tiles" class="dataset">
          <h3><a href="/terrain-tiles/">Terrain Tiles</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">elevation</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info-sustainability tag link-tag">sustainability</span></p>
          <p>A global dataset providing bare-earth terrain heights, tiled for easy usage and provided on S3.</p>
          <p><a href="/terrain-tiles/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://gis.stackexchange.com/questions/201831/how-to-efficiently-access-files-with-gdal-from-an-s3-bucket-using-vsis3/202608#202608" class="" target="_none">Using GDAL to Produce a Geotiff for an Area</a> by Chris Henrick</li></a>
          </li>
          <li>
            <a href="https://podpac.org/datasets.html#terraintiles" class="" target="_none">PODPAC: Python Library supporting TerrainTiles for analysis</a> by <a href="http://www.creare.com/" target="_none">Creare</a></li></a>
          </li>
          <li>
            <a href="https://terradactile.sparkgeo.com/" class="" target="_none">Terradactile: Generate DEM and hillshade Cloud Optimized GeoTIFFs for visualization and download based on terrain tiles</a> by <a href="https://sparkgeo.com/" target="_none">Sparkgeo</a></li></a>
          </li>
          <li>
            <a href="https://onthegomap.com/" class="" target="_none">On The Go Map</a> by <a href="https://onthegomap.com/" target="_none">On The Go Map</a></li></a>
          </li>
          <li>
            <a href="https://elevation.racemap.com" class="" target="_none">Open-Source Elevation Service</a> by <a href="https://racemap.com/" target="_none">Racemap</a></li></a>
          </li>
        </ul>
        <p><a href="/terrain-tiles/#usageexamples">See 12 usage examples &rarr;</a></p>
        </div>
        <div id="allen-cell-imaging-collections" class="dataset">
          <h3><a href="/allen-cell-imaging-collections/">Allen Cell Imaging Collections</a></h3>
          <p><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">cell biology</span><span class="label label-info tag link-tag">cell imaging</span><span class="label label-info tag link-tag">Homo sapiens</span><span class="label label-info tag link-tag">image processing</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">microscopy</span></p>
          <p>This bucket contains multiple datasets (as Quilt packages) created by the
Allen Institute for Cell Science (AICS). The imaging data in this bucket contains
either of the following:1) field of view images from glass plates
2) cell membrane, DNA, and structure segmentations
3) cell membrane, DNA and structure contours
4) machine learning imaging predictions of the previously listed modalities.In addition, many of the datasets include CSVs that contain feature sets
related to that data.</p>
          <p><a href="/allen-cell-imaging-collections/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.nature.com/articles/s41592-018-0111-2" class="" target="_none">Label-free prediction of three-dimensional fluorescence images from transmitted-light microscopy</a> by <a href="https://alleninstitute.org/about/team/" target="_none">Chawin Ounkomol, Sharmishtaa Seshamani, Mary M. Maleckar, Forrest Collman &amp; Gregory R. Johnson</a></li></a>
          </li>
          <li>
            <a href="https://imsc.allencell.org" class="" target="_none">Integrated Mitotic Stem Cell</a> by <a href="https://www.alleninstitute.org/what-we-do/cell-science/about/team/" target="_none">Allen Institute for Cell Science</a></li></a>
          </li>
          <li>
            <a href="https://www.biorxiv.org/content/10.1101/2020.12.08.415562v1" class="" target="_none">Robust integrated intracellular organization of the human iPS cell: where, how much, and how variable?</a> by <a href="https://www.alleninstitute.org/what-we-do/cell-science/about/team/" target="_none">Matheus P. Viana, Jianxu Chen*, Theo A. Knijnenburg*, Ritvik Vasan*, Calysta Yan*... Allen Institute for Cell Science... Graham T. Johnson, Ruwanthi N. Gunawardane, Nathalie Gaudreault, Julie A. Theriot, Susanne M. Rafelski</a></li></a>
          </li>
          <li>
            <a href="github.com/AllenCellModeling/pytorch_integrated_cell" class="" target="_none">Pytorch 3D Integrated Cell</a> by <a href="https://www.alleninstitute.org/what-we-do/cell-science/about/team/" target="_none">Gregory R. Johnson, Rory M. Donovan-Maiye, Mary M. Maleckar</a></li></a>
          </li>
          <li>
            <a href="https://github.com/AllenCellModeling/pytorch_fnet/blob/master/examples/download_and_train.py" class="" target="_none">Download and train label-free models</a> by <a href="https://www.alleninstitute.org/what-we-do/cell-science/about/team/staff-profiles/greg-johnson/" target="_none">Greg Johnson</a></li></a>
          </li>
        </ul>
        <p><a href="/allen-cell-imaging-collections/#usageexamples">See 11 usage examples &rarr;</a></p>
        </div>
        <div id="cbers" class="dataset">
          <h3><a href="/cbers/">CBERS on AWS</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">imaging</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info-sustainability tag link-tag">sustainability</span></p>
          <p>This project creates a S3 repository with imagery acquired
by the China-Brazil Earth Resources Satellite (CBERS). The
image files are recorded and processed by Instituto Nacional de Pesquisa
Espaciais (INPE) and are converted to Cloud Optimized Geotiff
format in order to optimize its use for cloud based applications.
The repository contains all CBERS-4 MUX, AWFI, PAN5M and
PAN10M scenes acquired since
the start of the satellite mission and is daily updated with
new scenes.</p>
          <p><a href="/cbers/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://stac.amskepler.com/v10/stac/search" class="" target="_none">STAC V1.0 search endpoint for archive</a> by <a href="https://github.com/fredliporace/cbers-2-stac" target="_none">AMS Kepler</a></li></a>
          </li>
          <li>
            <a href="https://github.com/fredliporace/cbersgif" class="" target="_none">CBERS timelapse GIF generator</a> by <a href="https://github.com/fredliporace" target="_none">Frederico Liporace</a></li></a>
          </li>
          <li>
            <a href="https://cbers.stac.cloud" class="" target="_none">CBERS static STAC catalog served by stac-browser</a> by <a href="https://github.com/radiantearth/stac-browser" target="_none">Radiant Earth</a></li></a>
          </li>
          <li>
            <a href="https://aws.amazon.com/blogs/publicsector/keeping-a-spatiotemporal-asset-catalog-stac-up-to-date-with-sns-sqs/" class="aws-link" target="_none">Keeping a SpatioTemporal Asset Catalog (STAC) Up To Date with SNS/SQS</a> by Frederico Liporace</li></a>
          </li>
          <li>
            <a href="https://eos.com/landviewer/" class="" target="_none">EOS Land Viewer</a> by <a href="https://eos.com/" target="_none">Earth Observing System</a></li></a>
          </li>
        </ul>
        <p><a href="/cbers/#usageexamples">See 11 usage examples &rarr;</a></p>
        </div>
        <div id="fcp-indi" class="dataset">
          <h3><a href="/fcp-indi/">International Neuroimaging Data-Sharing Initiative (INDI)</a></h3>
          <p><span class="label label-info tag link-tag">Homo sapiens</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">magnetic resonance imaging</span><span class="label label-info tag link-tag">neuroimaging</span><span class="label label-info tag link-tag">neuroscience</span></p>
          <p>This bucket contains multiple neuroimaging datasets that are part of the International Neuroimaging Data-Sharing Initiative. Raw human and non-human primate neuroimaging data include 1) Structural MRI; 2) Functional MRI; 3) Diffusion Tensor Imaging; 4) Electroencephalogram (EEG)
In addition to the raw data, preprocessed data is also included for some datasets.
A complete list of the available datasets can be seen in the documentation lonk provided below. </p>
          <p><a href="/fcp-indi/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.sciencedirect.com/science/article/pii/S089662731931089X" class="" target="_none">Accelerating the Evolution of Nonhuman Primate Neuroimaging</a> by M.P. Milham, C. Petkov</li></a>
          </li>
          <li>
            <a href="https://academic.oup.com/gigascience/article/6/2/giw011/2865212" class="" target="_none">The Healthy Brain Network Serial Scanning Initiative: a resource for evaluating inter-individual differences and their reliabilities across scan conditions and sessions</a> by D. O&#x27;Connor, N.V. Potler, ..., M.P. Milham</li></a>
          </li>
          <li>
            <a href="https://www.nature.com/articles/mp201378" class="" target="_none">The autism brain imaging data exchange: towards a large-scale evaluation of the intrinsic brain architecture in autism.</a> by A. Di Martino, C-G Yan, ..., M.P. Milham</li></a>
          </li>
          <li>
            <a href="https://www.nature.com/articles/sdata2017181" class="" target="_none">An open resource for transdiagnostic research in pediatric mental health and learning disorders</a> by L.M. Alexander, J. Escalera, ..., M.P. Milham</li></a>
          </li>
          <li>
            <a href="https://www.nature.com/articles/sdata201710" class="" target="_none">Enhancing studies of the connectome in autism using the autism brain imaging data exchange II.</a> by A. Di Martino, D. O&#x27;Connor, M.P. Milham</li></a>
          </li>
        </ul>
        <p><a href="/fcp-indi/#usageexamples">See 11 usage examples &rarr;</a></p>
        </div>
        <div id="spacenet" class="dataset">
          <h3><a href="/spacenet/">SpaceNet</a></h3>
          <p><span class="label label-info tag link-tag">computer vision</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">satellite imagery</span></p>
          <p>SpaceNet, launched in August 2016 as an open innovation project offering a repository of freely available
imagery with co-registered map features. Before SpaceNet, computer vision researchers had minimal options
to obtain free, precision-labeled, and high-resolution satellite imagery. Today, SpaceNet hosts datasets
developed by its own team, along with data sets from projects like IARPA’s Functional Map of the World (fMoW).</p>
          <p><a href="/spacenet/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://medium.com/the-downlinq/spacenet-winning-implementations-and-new-imagery-release-55f738e14253" class="" target="_none">SpaceNet: Winning Implementations and New Imagery Release</a> by <a href="https://medium.com/@toddstavish" target="_none">Todd Stavish</a></li></a>
          </li>
          <li>
            <a href="https://medium.com/the-downlinq/deploying-the-spacenet-6-baseline-on-aws-c811ad82da1" class="" target="_none">Deploying the SpaceNet 6 Baseline on AWS</a> by <a href="https://medium.com/@avanetten" target="_none">Adam Van Etten and Nick Weir</a></li></a>
          </li>
          <li>
            <a href="https://medium.com/the-downlinq/spacenet-6-dataset-release-66076c8fb79b" class="" target="_none">SpaceNet 6: Dataset Release</a> by <a href="https://medium.com/@jshermeyer" target="_none">Jake Shermeyer</a></li></a>
          </li>
          <li>
            <a href="https://medium.com/the-downlinq/introducing-the-spacenet-off-nadir-imagery-and-buildings-dataset-e4a3c1cb4ce3" class="" target="_none">Introducing the SpaceNet Off-Nadir Imagery Dataset</a> by <a href="https://medium.com/@david.lindenbaum" target="_none">David Lindenbaum</a></li></a>
          </li>
          <li>
            <a href="https://medium.com/the-downlinq/introducing-the-spacenet-road-detection-and-routing-challenge-and-dataset-7604de39b779" class="" target="_none">Introducing the SpaceNet Road Detection and Routing Challenge and Dataset</a> by <a href="https://medium.com/@david.lindenbaum" target="_none">David Lindenbaum</a></li></a>
          </li>
        </ul>
        <p><a href="/spacenet/#usageexamples">See 11 usage examples &rarr;</a></p>
        </div>
        <div id="janelia-flylight" class="dataset">
          <h3><a href="/janelia-flylight/">Fly Brain Anatomy: FlyLight Gen1 and Split-GAL4 Imagery</a></h3>
          <p><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">fluorescence imaging</span><span class="label label-info tag link-tag">image processing</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">microscopy</span><span class="label label-info tag link-tag">neurobiology</span><span class="label label-info tag link-tag">neuroimaging</span><span class="label label-info tag link-tag">neuroscience</span></p>
          <p>This data set, made available by Janelia&#39;s FlyLight project, consists of fluorescence images 
of Drosophila melanogaster driver lines, aligned to standard templates, and stored in formats 
suitable for rapid searching and visualization. Additional data will be added as it is published. 
A large release of Gen1 MCFO samples is coming at the beginning of May 2020. </p>
          <p><a href="/janelia-flylight/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1101/376384" class="" target="_none">An unbiased template of the Drosophila brain and ventral nerve cord</a> by Bogovic, J.A., Otsuna, H., Heinrich, L., Ito, M., Jeter, J., Meissner, G., Nern, A., Colonell, J., Malkesman, O., Ito, K., &amp; Saalfeld, S.</li></a>
          </li>
          <li>
            <a href="https://github.com/JaneliaSciComp/open-data-flylight/blob/master/tutorials/File_operations_on_AWS_S3.ipynb" class="" target="_none">File Operations on AWS S3</a> by Rob Svirskas</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1101/318006" class="" target="_none">Color depth MIP mask search: a new tool to expedite Split-GAL4 creation</a> by Otsuna, H., Ito, M., &amp; Kawase, T.</li></a>
          </li>
          <li>
            <a href="https://www.janelia.org/project-team/flylight" class="" target="_none">FlyLight Project Website</a> by Geoffrey Meissner</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.7554/eLife.04577" class="" target="_none">The neuronal architecture of the mushroom body provides a logic for associative learning</a> by Aso, Y., Hattori, D., Yu, Y., Johnston, R. M., Iyer, N. A., Ngo, T., Dionne, H., Abbott, L., Axel, R., Tanimoto, H., &amp; Rubin, G. M.</li></a>
          </li>
        </ul>
        <p><a href="/janelia-flylight/#usageexamples">See 10 usage examples &rarr;</a></p>
        </div>
        <div id="irs990" class="dataset">
          <h3><a href="/irs990/">IRS 990 Filings</a></h3>
          <p><span class="label label-info tag link-tag">regulatory</span><span class="label label-info tag link-tag">statistics</span></p>
          <p>Machine-readable data from certain electronic 990 forms filed with the IRS from 2013 to present.</p>
          <p><a href="/irs990/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.open990.com/" class="" target="_none">Open990</a> by <a href="https://www.990consulting.com/" target="_none">990 Consulting, LLC</a></li></a>
          </li>
          <li>
            <a href="https://github.com/ma-ji/npo_classifier" class="" target="_none">npo_classifier: Automated coding using machine-learning and remapping the U.S. nonprofit sector</a> by <a href="https://jima.me/" target="_none">Ji Ma</a></li></a>
          </li>
          <li>
            <a href="https://projects.propublica.org/nonprofits/" class="" target="_none">Nonprofit Explorer</a> by <a href="https://propublica.org" target="_none">ProPublica</a></li></a>
          </li>
          <li>
            <a href="https://github.com/CharityNavigator/990_long" class="" target="_none">990_long</a> by <a href="https://www.charitynavigator.org/" target="_none">Charity Navigator</a></li></a>
          </li>
          <li>
            <a href="https://lecy.github.io/Open-Data-for-Nonprofit-Research/" class="" target="_none">Guide to Open Data for Nonprofit Research</a> by <a href="https://github.com/lecy" target="_none">lecy</a></li></a>
          </li>
        </ul>
        <p><a href="/irs990/#usageexamples">See 10 usage examples &rarr;</a></p>
        </div>
        <div id="noaa-goes" class="dataset">
          <h3><a href="/noaa-goes/">NOAA Geostationary Operational Environmental Satellites (GOES) 16 &amp; 17</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info-sustainability tag link-tag">sustainability</span><span class="label label-info tag link-tag">weather</span></p>
          <p>GOES satellites (GOES-16 &amp; GOES-17) provide continuous weather imagery and
monitoring of meteorological and space environment data across North America.
GOES satellites provide the kind of continuous monitoring necessary for
intensive data analysis. They hover continuously over one position on the surface.
The satellites orbit high enough to allow for a full-disc view of the Earth. Because
they stay above a fixed spot on the surface, they provide a constant vigil for the
atmospheric &quot;triggers&quot; for severe weather conditions such as tornadoes, flash floods,
hailstorms, and hurrican...</p>
          <p><a href="/noaa-goes/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/HamedAlemo/visualize-goes16" class="" target="_none">Visualize GOES-16 in Python using Xarray</a> by <a href="https://www.linkedin.com/in/hamedalemohammad/" target="_none">Hamed Alemohammad</a></li></a>
          </li>
          <li>
            <a href="https://nbviewer.jupyter.org/github/oceanhackweek/ohw20-tutorials/blob/master/10-satellite-data-access/goes-cmp-netcdf-zarr.ipynb" class="" target="_none">Comparison reading GOES-R data from AWS S3 in netCDF versus zarr</a> by <a href="http://www.faralloninstitute.org/chelle" target="_none">Chelle Gentemann</a></li></a>
          </li>
          <li>
            <a href="https://github.com/blaylockbk/goes2go" class="" target="_none">GOES-2-go is a python package for downloading and plotting plotting GOES data.</a> by <a href="https://www.linkedin.com/in/blaylockbk/" target="_none">Brian K. Blaylock</a></li></a>
          </li>
          <li>
            <a href="https://github.com/awslabs/amazon-asdi/tree/master/examples/goes16-precip-julia" class="aws-link" target="_none">NOAA GOES16 Julia Jupyter Notebook Example</a> by Peter Schmiedeskamp</li></a>
          </li>
          <li>
            <a href="https://aws.amazon.com/blogs/publicsector/embracing-the-cloud-for-climate-research/" class="aws-link" target="_none">Embracing the cloud for climate research</a> by North Carolina State University’s North Carolina Institute for Climate Studies</li></a>
          </li>
        </ul>
        <p><a href="/noaa-goes/#usageexamples">See 10 usage examples &rarr;</a></p>
        </div>
        <div id="mur" class="dataset">
          <h3><a href="/mur/">Multi-Scale Ultra High Resolution (MUR) Sea Surface Temperature (SST)</a></h3>
          <p><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">natural resource</span><span class="label label-info tag link-tag">oceans</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info-sustainability tag link-tag">sustainability</span><span class="label label-info tag link-tag">water</span><span class="label label-info tag link-tag">weather</span></p>
          <p>A global, gap-free, gridded, daily 1 km Sea Surface Temperature (SST) dataset created by merging multiple Level-2 satellite SST datasets. Those input datasets include the NASA Advanced Microwave Scanning Radiometer-EOS (AMSR-E), the JAXA Advanced Microwave Scanning Radiometer 2 (AMSR-2) on GCOM-W1, the Moderate Resolution Imaging Spectroradiometers (MODIS) on the NASA Aqua and Terra platforms, the US Navy microwave WindSat radiometer, the Advanced Very High Resolution Radiometer (AVHRR) on several NOAA satellites, and in situ SST observations from the NOAA iQuam project. Data are available fro...</p>
          <p><a href="/mur/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://podaac-tools.jpl.nasa.gov/drive/files/allData/ghrsst/data/GDS2/L4/GLOB/JPL/MUR/v4.1" class="" target="_none">HTTPS server</a> by <a href="https://podaac.jpl.nasa.gov" target="_none">PO.DAAC</a></li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1016/j.rse.2017.07.029" class="" target="_none">A multi-scale high-resolution analysis of global sea surface temperature</a> by Chin, Toshio Michael, Jorge Vazquez-Cuervo, and Edward M. Armstrong</li></a>
          </li>
          <li>
            <a href="https://podaac.jpl.nasa.gov/ws/search/granule/?datasetId&#x3D;PODAAC-GHGMR-4FJ04" class="" target="_none">Web discovery service</a> by <a href="https://podaac.jpl.nasa.gov" target="_none">PO.DAAC</a></li></a>
          </li>
          <li>
            <a href="https://podaac-tools.jpl.nasa.gov/drive/files/allData/ghrsst/sw/generic_nc_readers/python/" class="" target="_none">Python Reader Software</a> by <a href="https://podaac.jpl.nasa.gov" target="_none">PO.DAAC</a></li></a>
          </li>
          <li>
            <a href="https://podaac-tools.jpl.nasa.gov/soto" class="" target="_none">State of the Ocean (SOTO) server</a> by <a href="https://podaac.jpl.nasa.gov" target="_none">PO.DAAC</a></li></a>
          </li>
        </ul>
        <p><a href="/mur/#usageexamples">See 9 usage examples &rarr;</a></p>
        </div>
        <div id="open-neurodata" class="dataset">
          <h3><a href="/open-neurodata/">Open NeuroData</a></h3>
          <p><span class="label label-info tag link-tag">array tomography</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">electron microscopy</span><span class="label label-info tag link-tag">image processing</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">lightsheet microscopy</span><span class="label label-info tag link-tag">magnetic resonance imaging</span><span class="label label-info tag link-tag">neuroimaging</span><span class="label label-info tag link-tag">neuroscience</span></p>
          <p>This bucket contains multiple neuroimaging datasets (as Neuroglancer Precomputed Volumes) across multiple modalities and scales, ranging from nanoscale (electron microscopy), to microscale (cleared lightsheet microscopy and array tomography), and mesoscale (structural and functional magnetic resonance imaging). Additionally, many of the datasets include segmentations and meshes.</p>
          <p><a href="/open-neurodata/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://neurodata.io/help/visualization/" class="" target="_none">Visualization using Neuroglancer</a> by <a href="https://github.com/falkben" target="_none">Benjamin Falk</a></li></a>
          </li>
          <li>
            <a href="https://github.com/google/neuroglancer" class="" target="_none">Neuroglancer</a> by <a href="https://github.com/jbms" target="_none">Jeremy Maitin-Shepard</a></li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1016/j.neuron.2016.10.033" class="" target="_none">To the Cloud! A Grassroots Proposal to Accelerate Brain Science Discovery</a> by J. T. Vogelstein, B. Mensh, M. Häusser, N. Spruston, A. C. Evans, K. Kording, K. Amunts, C. Ebell, J. Muller, M. Telefont, S. Hill, S. P. Koushika, C. Calì, P. A. Valdés-Sosa, P. B. Littlewood, C. Koch, S. Saalfeld, A. Kepecs, H. Peng, Y. O. Halchenko, G. Kiar, M. M. Poo, J. B. Poline, M. P. Milham, A. P. Schaffer, R. Gidron, H. Okano, V. D. Calhoun, M. Chun, D. M. Kleissas, R. J. Vogelstein, E. Perlman, R. Burns, R. Huganir, and M. I. Miller</li></a>
          </li>
          <li>
            <a href="https://neurodata.io/help/download/" class="" target="_none">Download</a> by <a href="https://github.com/falkben" target="_none">Benjamin Falk</a></li></a>
          </li>
          <li>
            <a href="https://www.nature.com/articles/s41592-018-0181-1" class="" target="_none">A Community-Developed Open-Source Computational Ecosystem for Big Neuro Data</a> by J. T. Vogelstein, E. Perlman, B. Falk, A. Baden, W. Gray Roncal, V. Chandrashekhar, F. Collman, S. Seshamani, J. L. Patsolic, K. Lillaney, M. Kazhdan, R. Hider, D. Pryor, J. Matelsky, T. Gion, P. Manavalan, B. Wester, M. Chevillet, E. T. Trautman, K. Khairy, E. Bridgeford, D. M. Kleissas, D. J. Tward, A. K. Crow, B. Hsueh, M. A. Wright, M. I. Miller, S. J. Smith, R. J. Vogelstein, K. Deisseroth, and R. Burns</li></a>
          </li>
        </ul>
        <p><a href="/open-neurodata/#usageexamples">See 9 usage examples &rarr;</a></p>
        </div>
        <div id="ccle" class="dataset">
          <h3><a href="/ccle/">Cancer Cell Line Encyclopedia (CCLE)</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">Homo sapiens</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">STRIDES</span><span class="label label-info tag link-tag">transcriptomics</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>The Cancer Cell Line Encyclopedia (CCLE) project is an effort to conduct a detailed genetic
characterization of a large panel of human cancer cell lines. The CCLE provides public access to
genomic data, visualization and analysis for over 1100 cancer cell lines. This dataset contains
RNA-Seq Aligned Reads, WXS Aligned Reads, and WGS Aligned Reads data.</p>
          <p><a href="/ccle/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://portal.gdc.cancer.gov/legacy-archive/" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1038/nature11003" class="" target="_none">The Cancer Cell Line Encyclopedia enables predictive modelling of anticancer drug
sensitivity
</a> by Barretina Caponigro Stransky et al.</li></a>
          </li>
          <li>
            <a href="https://cgc-accounts.sbgenomics.com/" class="" target="_none">Cancer Genomics Cloud</a> by <a href="https://docs.sevenbridges.com/docs/ccle-data" target="_none">Seven Bridges</a></li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1038/nature15736" class="" target="_none">Pharmacogenomic agreement between two cancer cell line data sets</a> by The Cancer Cell Line Encyclopedia Consortium &amp; The Genomics of Drug Sensitivity in Cancer
Consortium
</li></a>
          </li>
          <li>
            <a href="https://portals.broadinstitute.org/ccle" class="" target="_none">Broad Institute Cancer Cell Line Encyclopedia</a> by <a href="https://www.broadinstitute.org/" target="_none">The Broad Institute of MIT &amp; Harvard</a></li></a>
          </li>
        </ul>
        <p><a href="/ccle/#usageexamples">See 8 usage examples &rarr;</a></p>
        </div>
        <div id="ncei-wcsd-archive" class="dataset">
          <h3><a href="/ncei-wcsd-archive/">NOAA Water-Column Sonar Data Archive</a></h3>
          <p><span class="label label-info tag link-tag">biodiversity</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">ecosystems</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">mapping</span><span class="label label-info tag link-tag">oceans</span><span class="label label-info-sustainability tag link-tag">sustainability</span></p>
          <p>Water-column sonar data archived at the NOAA National Centers for Environmental Information.</p>
          <p><a href="/ncei-wcsd-archive/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/CI-CMG/pyEcholab/tree/master/examples/cloud%20tutorials/Reading_And_Plotting_Bottom_Data.ipynb" class="" target="_none">Reading and Plotting Bottom Data</a> by Carrie Wall</li></a>
          </li>
          <li>
            <a href="https://www.ngdc.noaa.gov/maps/water_column_sonar/index.html" class="" target="_none">Water Column Sonar Data Explorer - Map viewer for exploring sonar water column data</a> by NCEI, NOAA</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1093/icesjms/fsw014" class="" target="_none">Increasing the accessibility of acoustic data through global access and imagery</a> by Carrie Wall, Michael Jech, and Susan McLean</li></a>
          </li>
          <li>
            <a href="https://github.com/CI-CMG/pyEcholab" class="" target="_none">pyEcholab - an open-source, python-based toolkit for reading, processing, plotting, and exporting fisheries acoustic echosounder data</a> by Rick Towler, Chuck Anderson, Veronica Martinez, Pamme Crandell</li></a>
          </li>
          <li>
            <a href="https://github.com/CI-CMG/pyEcholab/tree/master/examples/cloud%20tutorials/Frequency_Differencing_With_Raw_Data.ipynb" class="" target="_none">Frequency Differencing with Raw Data</a> by Carrie Wall</li></a>
          </li>
        </ul>
        <p><a href="/ncei-wcsd-archive/#usageexamples">See 8 usage examples &rarr;</a></p>
        </div>
        <div id="sentinel-2-l2a-cogs" class="dataset">
          <h3><a href="/sentinel-2-l2a-cogs/">Sentinel-2 Cloud-Optimized GeoTIFFs</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">natural resource</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info-sustainability tag link-tag">sustainability</span></p>
          <p>The <a href="https://sentinel.esa.int/web/sentinel/missions/sentinel-2">Sentinel-2 mission</a> is
a land monitoring constellation of two satellites that provide high resolution
optical imagery and provide continuity for the current SPOT and Landsat missions.
The mission provides a global coverage of the Earth&#39;s land surface every 5 days,
making the data of great use in ongoing studies.
This dataset is the same as the <a href="https://registry.opendata.aws/sentinel-2/">Sentinel-2</a>
dataset, except the JP2K files were converted into Cloud-Optimized GeoTIFFs (COGs).
Additionally, SpatioTemporal Asset Catalog metadata has were in a JSON file
alongside the data, and a STAC API called <a href="https://earth-search.aws.element84.com/v0">Earth-search</a>
is freely available t...</p>
          <p><a href="/sentinel-2-l2a-cogs/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://docs.google.com/presentation/d/14NsKFZ3UF2Swwx_9L7sPMX9ccFUK1ruQyZXWK9Cz4L4/edit?usp&#x3D;sharing" class="" target="_none">STAC and Sentinel-2 COGs (ESIP Summer Meeting 2020)</a> by Matthew Hanson</li></a>
          </li>
          <li>
            <a href="https://github.com/sat-utils/sat-search" class="" target="_none">Sat-search</a> by Matthew Hanson</li></a>
          </li>
          <li>
            <a href="https://stacindex.org/" class="" target="_none">SpatioTemporal Asset Catalogs</a> by STAC Contriubutors</li></a>
          </li>
          <li>
            <a href="http://www.acgeospatial.co.uk/stac-cog-python-and-qgis/" class="" target="_none">STAC, COG, Python and QGIS</a> by Andrew Cutts</li></a>
          </li>
          <li>
            <a href="https://github.com/opendatacube/cube-in-a-box" class="" target="_none">Use Sentinel-2 data in the Open Data Cube</a> by Alex Leith</li></a>
          </li>
        </ul>
        <p><a href="/sentinel-2-l2a-cogs/#usageexamples">See 8 usage examples &rarr;</a></p>
        </div>
        <div id="wb-light-every-night" class="dataset">
          <h3><a href="/wb-light-every-night/">World Bank - Light Every Night</a></h3>
          <p><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">satellite imagery</span></p>
          <p>Light Every Night - World Bank Nightime Light Data – provides open access to all nightly imagery and data from the Visible Infrared Imaging Radiometer Suite Day-Night Band (VIIRS DNB) from 2012-2020 and the Defense Meteorological Satellite Program Operational Linescan System (DMSP-OLS) from 1992-2013. The underlying data are sourced from the NOAA National Centers for Environmental Information (NCEI) archive. Additional processing by the University of Michigan enables access in Cloud Optimized GeoTIFF format (COG) and search using the Spatial Temporal Asset Catalog (STAC) standard. The data is ...</p>
          <p><a href="/wb-light-every-night/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://worldbank.github.io/OpenNightLights/welcome.html" class="" target="_none">Open Nighttime Lights</a> by <a href="https://worldbank.github.io/OpenNightLights/welcome.html" target="_none">Daynan Crull, Trevor Monroe</a></li></a>
          </li>
          <li>
            <a href="http://nightlights.io" class="" target="_none">Twenty Years of India Lights</a> by <a href="http://nightlights.io" target="_none">Kwawu Mensan Gaba, Brian Min, Anand Thakker, Christopher Elvidge</a></li></a>
          </li>
          <li>
            <a href="http://documents.worldbank.org/curated/en/305771562750007469/Mainstreaming-Disruptive-Technologies-in-Energy" class="" target="_none">Mainstreaming Disruptive Technologies in Energy. World Bank Report. 2019</a> by Kwawu Mensan Gaba, Brian Min, Olaf Veerman, Kimberly Baugh</li></a>
          </li>
          <li>
            <a href="https://journals.sfu.ca/apan/index.php/apan/article/view/8/0" class="" target="_none">Nighttime lights compositing using the VIIRS day-night band - Preliminary results. Proceedings of the Asia-Pacific Advanced Network 35 (2013)70-86.</a> by Kimberly Baugh, Feng-Chi Hsu, Christopher D. Elvidge, and Mikhail Zhizhin.</li></a>
          </li>
          <li>
            <a href="http://www-personal.umich.edu/~brianmin/HREA/" class="" target="_none">High Resolution Electricity Access Indicators (HREA) - Settlement-level measures of electricity access, reliability, and usage.</a> by <a href="http://www-personal.umich.edu/~brianmin/" target="_none">Brian Min, Zachary O&#x27;Keeffe</a></li></a>
          </li>
        </ul>
        <p><a href="/wb-light-every-night/#usageexamples">See 8 usage examples &rarr;</a></p>
        </div>
        <div id="ebirdst" class="dataset">
          <h3><a href="/ebirdst/">eBird Status and Trends Model Results</a></h3>
          <p><span class="label label-info tag link-tag">biodiversity</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">ecosystems</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info-sustainability tag link-tag">sustainability</span></p>
          <p>The eBird Status and Trends project generates estimates of bird
occurrence and abundance at a high spatiotemporal resolution.
This dataset represents the primary modeled results from the
analysis workflow and are designed for further analysis,
synthesis, visualization, and exploration.</p>
          <p><a href="/ebirdst/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://cornelllabofornithology.github.io/ebirdst/articles/ebirdst-introduction.html" class="" target="_none">ebirdst - Background, Data Access and Data Structure</a> by Tom Auer and Daniel Fink</li></a>
          </li>
          <li>
            <a href="https://ebird.org/science/status-and-trends" class="" target="_none">eBird Status and Trends Visualizations</a> by Cornell Lab of Ornithology</li></a>
          </li>
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/31837058" class="" target="_none">Modeling avian full annual cycle distribution and population trends with citizen science data</a> by Fink et al (2019)</li></a>
          </li>
          <li>
            <a href="https://ebird.org/science/status-and-trends/faq" class="" target="_none">eBird Status and Trends FAQ</a> by Cornell Lab of Ornithology</li></a>
          </li>
          <li>
            <a href="https://cornelllabofornithology.github.io/ebirdst/articles/ebirdst-advanced-mapping.html" class="" target="_none">ebirdst - Generating Seasonal Abundance and Range Maps and Stats</a> by Matt Strimas-Mackey, Tom Auer, and Daniel Fink</li></a>
          </li>
        </ul>
        <p><a href="/ebirdst/#usageexamples">See 8 usage examples &rarr;</a></p>
        </div>
        <div id="cptac-2" class="dataset">
          <h3><a href="/cptac-2/">Clinical Proteomic Tumor Analysis Consortium 2 (CPTAC-2)</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">STRIDES</span><span class="label label-info tag link-tag">transcriptomics</span></p>
          <p>The Clinical Proteomic Tumor Analysis Consortium (CPTAC) is a national effort to accelerate the
understanding of the molecular basis of cancer through the application of large-scale proteome and
genome analysis, or proteogenomics. CPTAC-2 is the Phase II of the CPTAC Initiative (2011-2016).
Datasets contain open RNA-Seq Gene Expression Quantification, miRNA-Seq Isoform Expression
Quantification, and miRNA Expression Quantification data.</p>
          <p><a href="/cptac-2/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://portal.gdc.cancer.gov/projects/CPTAC-2" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></li></a>
          </li>
          <li>
            <a href="https://pdc.esacinc.com/pdc/browse" class="" target="_none">Proteomic Data Commons</a> by <a href="https://pdc.esacinc.com/pdc" target="_none">National Cancer Institute</a></li></a>
          </li>
          <li>
            <a href="https://cgc-accounts.sbgenomics.com/" class="" target="_none">Cancer Genomics Cloud</a> by <a href="http://www.cancergenomicscloud.org/" target="_none">Seven Bridges</a></li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1016/j.cell.2016.05.069" class="" target="_none">Integrated Proteogenomic Characterization of Human High-Grade Serous Ovarian Cancer</a> by Hui Zhang, Tao Liu, Zhen Zhang, Samuel H. Payne, Bai Zhang, Jason E. McDermott, Jian-Ying
Zhou, Vladislav A. Petyuk, Li Chen, Debjit Ray, Shisheng Sun, Feng Yang, Lijun Chen, Jing
Wang, Punit Shah, Seong Won Cha, Paul Aiyetan, Sunghee Woo, Yuan Tian, Marina A. Gritsenko,
Therese R. Clauss, Caitlin Choi, Matthew E. Monroe, Stefani Thomas, Song Nie, Chaochao Wu,
Ronald J. Moore, Kun-Hsing Yu, David L. Tabb, David Fenyö, Vineet Bafna, Yue Wang, Henry
Rodriguez, Emily S. Boja, Tara Hiltke, Robert C. Rivers, Lori Sokoll, Heng Zhu, Ie-Ming
Shih, Leslie Cope, Akhilesh Pandey, Bing Zhang, Michael P. Snyder, Douglas A. Levine,
Richard D. Smith, Daniel W. Chan, Karin D. Rodland, the CPTAC Investigators
</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1038/sdata.2015.22" class="" target="_none">Proteomic analysis of colon and rectal carcinoma using standard and customized databases
</a> by Slebos RJ, Wang X, Wang X, Zhang B, Tabb DL, Liebler DC</li></a>
          </li>
        </ul>
        <p><a href="/cptac-2/#usageexamples">See 7 usage examples &rarr;</a></p>
        </div>
        <div id="icgc" class="dataset">
          <h3><a href="/icgc/">ICGC on AWS</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span></p>
          <p>The International Cancer Genome Consortium (ICGC) coordinates projects with the common aim of accelerating research into the causes and control of cancer. The PanCancer Analysis of Whole Genomes (PCAWG) study is an international collaboration to identify common patterns of mutation in whole genomes from ICGC. More than 2,400 consistently analyzed genomes corresponding to over 1,100 unique ICGC donors are now freely available on Amazon S3 to credentialed researchers subject to ICGC data sharing policies.</p>
          <p><a href="/icgc/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.nature.com/articles/s41586-020-1970-0" class="" target="_none">Genomic basis of RNA alterations in cancer</a> by <a href="https://icgc.org/committees-and-working-groups#71954" target="_none">PCAWG Transcriptome Core Group et al. (2020)</a></li></a>
          </li>
          <li>
            <a href="https://www.nature.com/articles/s41586-020-1969-6" class="" target="_none">Pan-cancer analysis of whole genomes</a> by <a href="https://dcc.icgc.org/pcawg" target="_none">The ICGC/TCGA Pan-Cancer Analysis of Whole Genomes Consortium (2020)</a></li></a>
          </li>
          <li>
            <a href="https://www.nature.com/articles/s41586-019-1913-9" class="" target="_none">Patterns of somatic structural variation in human cancer genomes</a> by <a href="https://www.ncbi.nlm.nih.gov/pubmed?cmd&#x3D;search&amp;term&#x3D;Yilong%20Li" target="_none">Li et al. (2020)</a></li></a>
          </li>
          <li>
            <a href="https://www.nature.com/articles/s41586-019-1907-7" class="" target="_none">The evolutionary history of 2,658 cancers</a> by <a href="https://www.ncbi.nlm.nih.gov/pubmed?cmd&#x3D;search&amp;term&#x3D;Moritz%20Gerstung" target="_none">Gerstung et al (2020)</a></li></a>
          </li>
          <li>
            <a href="https://www.nature.com/articles/s41586-020-1943-3" class="" target="_none">The repertoire of mutational signatures in human cancer</a> by <a href="https://www.ncbi.nlm.nih.gov/pubmed?cmd&#x3D;search&amp;term&#x3D;Ludmil%20B.%20Alexandrov" target="_none">Alexandrov et al (2020)</a></li></a>
          </li>
        </ul>
        <p><a href="/icgc/#usageexamples">See 7 usage examples &rarr;</a></p>
        </div>
        <div id="nrel-pds-wtk" class="dataset">
          <h3><a href="/nrel-pds-wtk/">NREL Wind Integration National Dataset</a></h3>
          <p><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info-sustainability tag link-tag">sustainability</span></p>
          <p>Released to the public as part of the Department of Energy&#39;s Open Energy Data Initiative,
the <a href="https://www.nrel.gov/grid/wind-toolkit.html">Wind Integration National Dataset (WIND)</a>
is an update and expansion of the Eastern Wind Integration Data Set and
Western Wind Integration Data Set. It supports the next generation of wind
integration studies.</p>
          <p><a href="/nrel-pds-wtk/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.nrel.gov/docs/fy15osti/61740.pdf" class="" target="_none">Overview and Meteorological Validation of the Wind Integration National Dataset Toolkit</a> by Caroline Draxl, Bri-Mathias Hodge, Andrew Clifton, Jim McCaa</li></a>
          </li>
          <li>
            <a href="https://maps.nrel.gov/wind-prospector/" class="" target="_none">Wind Prospector</a> by Paul Edwards</li></a>
          </li>
          <li>
            <a href="https://aws.amazon.com/blogs/big-data/power-from-wind-open-data-on-aws/" class="aws-link" target="_none">Power from wind: Open data on AWS</a> by <a href="https://www.nrel.gov/research/caleb-phillips.html" target="_none">Caleb Phillips, Caroline Draxl, John Readey, Jordan Perr-Sauer</a></li></a>
          </li>
          <li>
            <a href="https://github.com/NREL/hsds-examples" class="" target="_none">HSDS Examples</a> by <a href="https://www.nrel.gov/research/caleb-phillips.html" target="_none">Caleb Phillips, Caroline Draxl, John Readey, Jordan Perr-Sauer, Michael Rossol</a></li></a>
          </li>
          <li>
            <a href="https://www.sciencedirect.com/science/article/pii/S0306261915004237?via%3Dihub/" class="" target="_none">The Wind Integration National Dataset (WIND) Toolkit</a> by Caroline Draxl, Andrew Clifton, Bri-Mathias Hodge, Jim McCaa</li></a>
          </li>
        </ul>
        <p><a href="/nrel-pds-wtk/#usageexamples">See 7 usage examples &rarr;</a></p>
        </div>
        <div id="cptac-3" class="dataset">
          <h3><a href="/cptac-3/">Clinical Proteomic Tumor Analysis Consortium 3 (CPTAC-3)</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">STRIDES</span><span class="label label-info tag link-tag">transcriptomics</span></p>
          <p>The Clinical Proteomic Tumor Analysis Consortium (CPTAC) is a national effort to accelerate the
understanding of the molecular basis of cancer through the application of large-scale proteome and
genome analysis, or proteogenomics. CPTAC-3 is the Phase III of the CPTAC Initiative. The dataset
contains open RNA-Seq Gene Expression Quantification data.</p>
          <p><a href="/cptac-3/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://portal.gdc.cancer.gov/projects/CPTAC-3" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1016/j.cell.2019.10.007" class="" target="_none">Integrated Proteogenomic Characterization of Clear Cell Renal Cell Carcinoma</a> by Clark DJ, Dhanasekaran SM, Petralia F, Pan J, Song X, Hu Y, da Veiga Leprevost F, Reva B,
Lih TM, Chang HY, Ma W, Huang C, Ricketts CJ, Chen L1, Krek A, Li Y, Rykunov D, Li QK, Chen
LS, Ozbek U, Vasaikar S, Wu Y, Yoo S, Chowdhury S, Wyczalkowski MA, Ji J, Schnaubelt M, Kong
A, Sethuraman S, Avtonomov DM, Ao M, Colaprico A, Cao S, Cho KC, Kalayci S, Ma S, Liu W,
Ruggles K, Calinawan A, Gümüş ZH, Geizler D, Kawaler E, Teo GC, Wen B, Zhang Y, Keegan S, Li
K, Chen F, Edwards N, Pierorazio PM, Chen XS, Pavlovich CP, Hakimi AA, Brominski G, Hsieh
JJ, Antczak A, Omelchenko T, Lubinski J, Wiznerowicz M, Linehan WM, Kinsinger CR,
Thiagarajan M, Boja ES, Mesri M, Hiltke T, Robles AI, Rodriguez H, Qian J, Fenyö D, Zhang B,
Ding L, Schadt E, Chinnaiyan AM, Zhang Z, Omenn GS, Cieslik M, Chan DW, Nesvizhskii AI, Wang
P, Zhang H; Clinical Proteomic Tumor Analysis Consortium
</li></a>
          </li>
          <li>
            <a href="https://cgc-accounts.sbgenomics.com/" class="" target="_none">Cancer Genomics Cloud</a> by <a href="http://www.cancergenomicscloud.org/" target="_none">Seven Bridges</a></li></a>
          </li>
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/29718670" class="" target="_none">Evaluation of NCI-7 Cell Line Panel as a Reference Material for Clinical Proteomics</a> by Clark DJ, Hu Y, Bocik W, Chen L, Schnaubelt M, Roberts R, Shah P, Whiteley G, Zhang H
</li></a>
          </li>
          <li>
            <a href="https://proteomics.cancer.gov/data-portal" class="" target="_none">CPTAC Data Portal</a> by <a href="https://proteomics.cancer.gov/" target="_none">National Cancer Institute</a></li></a>
          </li>
        </ul>
        <p><a href="/cptac-3/#usageexamples">See 6 usage examples &rarr;</a></p>
        </div>
        <div id="gdelt" class="dataset">
          <h3><a href="/gdelt/">Global Database of Events, Language and Tone (GDELT)</a></h3>
          <p><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">events</span></p>
          <p>This project monitors the world&#39;s broadcast, print,
and web news from nearly every corner of every country in
over 100 languages and identifies the people, locations,
organizations, counts, themes, sources, emotions,
quotes, images and events driving our global society every
second of every day.</p>
          <p><a href="/gdelt/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="http://www.geomesa.org/documentation/tutorials/geomesa-hbase-s3-on-aws.html" class="" target="_none">Bootstrapping GeoMesa HBase on AWS S3</a> by <a href="https://www.ccri.com" target="_none">Commonwealth Computer Research, Inc.</a></li></a>
          </li>
          <li>
            <a href="https://www.linkedin.com/pulse/media-brexit-story-so-far-may-corbyn-less-shambles-than-mark-chopping/" class="" target="_none">Analysing Brexit Coverage In The Media Over Time</a> by <a href="https://www.kognitio.com" target="_none">Mark Chopping</a></li></a>
          </li>
          <li>
            <a href="https://gist.github.com/jakechen/6955f2de51212163312b6430555b8e0b" class="" target="_none">Creating PySpark DataFrame from CSV in AWS S3 in EMR</a> by <a href="https://github.com/jakechen" target="_none">Jake Chen</a></li></a>
          </li>
          <li>
            <a href="http://live.thermobook.net/globe-events/" class="" target="_none">Globe Events</a> by <a href="http://www.thermobook.net" target="_none">thermobook</a></li></a>
          </li>
          <li>
            <a href="https://aws.amazon.com/blogs/big-data/running-r-on-amazon-athena/" class="aws-link" target="_none">Running R on Amazon Athena</a> by <a href="https://www.linkedin.com/in/gopal-wunnava-b11a77/" target="_none">Gopal Wunnava</a></li></a>
          </li>
        </ul>
        <p><a href="/gdelt/#usageexamples">See 6 usage examples &rarr;</a></p>
        </div>
        <div id="ladi" class="dataset">
          <h3><a href="/ladi/">Low Altitude Disaster Imagery (LADI) Dataset</a></h3>
          <p><span class="label label-info tag link-tag">aerial imagery</span><span class="label label-info tag link-tag">coastal</span><span class="label label-info tag link-tag">computer vision</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">earthquakes</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">image processing</span><span class="label label-info tag link-tag">imaging</span><span class="label label-info tag link-tag">infrastructure</span><span class="label label-info tag link-tag">land</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">mapping</span><span class="label label-info tag link-tag">natural resource</span><span class="label label-info tag link-tag">seismology</span><span class="label label-info tag link-tag">transportation</span><span class="label label-info tag link-tag">urban</span><span class="label label-info tag link-tag">water</span></p>
          <p>The Low Altitude Disaster Imagery (LADI) Dataset consists of human and machine annotated airborne images collected by the Civil Air Patrol in support of various disaster responses from 2015-2019. The initial release of LADI focuses on the Atlantic hurricane seasons and coastal states along the Atlantic Ocean and Gulf of Mexico. Annotations are included for major hurricanes of Harvey, Maria, and Florence. Two key distinctions are the low altitude, oblique perspective of the imagery and disaster-related features, which are rarely featured in computer vision benchmarks and datasets.</p>
          <p><a href="/ladi/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/bwsi-hadr" class="" target="_none">Remote Sensing for Disaster Response Course</a> by <a href="https://beaverworks.ll.mit.edu/CMS/bw/bwsi" target="_none">Beaver Works Summer Institute</a></li></a>
          </li>
          <li>
            <a href="https://www-nlpir.nist.gov/projects/tv2020/dsdi.html" class="" target="_none">NIST TRECVID 2020 - Disaster Scene Description and Indexing (DSDI)</a> by TREC Video Retrieval Evaluation (TRECVID)</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1109/HPEC.2019.8916437" class="" target="_none">Large Scale Organization and Inference of an Imagery Dataset for Public Safety</a> by Jeffrey Liu, David Strohschein, Siddharth Samsi, Andrew Weinert</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1109/LNET.2019.2955833" class="" target="_none">Video Testing at the FirstNet Innovation and Test Lab Using a Public Safety Dataset</a> by Chris Budny, Jeffrey Liu, Andrew Weinert</li></a>
          </li>
          <li>
            <a href="https://github.com/LADI-Dataset/ladi-tutorial" class="" target="_none">LADI Tutorials</a> by <a href="https://github.com/LADI-Dataset" target="_none">Andrew Weinert, Jianyu Mao, Kiana Harris, Nae-Rong Chang, Caleb Pennell, Yiming Ren, Ryan Earley, Nadia Dimitrova</a></li></a>
          </li>
        </ul>
        <p><a href="/ladi/#usageexamples">See 6 usage examples &rarr;</a></p>
        </div>
        <div id="nyu-fastmri" class="dataset">
          <h3><a href="/nyu-fastmri/">NYU Langone &amp; FAIR FastMRI Dataset</a></h3>
          <p><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">health</span><span class="label label-info tag link-tag">image processing</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">magnetic resonance imaging</span><span class="label label-info tag link-tag">neurobiology</span><span class="label label-info tag link-tag">neuroimaging</span></p>
          <p>This dataset contains deidentified raw k-space data and DICOM image files of over 1,500 knees and 6,970 brains.</p>
          <p><a href="/nyu-fastmri/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://arxiv.org/abs/1904.01112" class="" target="_none">Deep Learning Methods for Parallel Magnetic Resonance Image Reconstruction</a> by <a href="https://arxiv.org/search/eess?searchtype&#x3D;author&amp;query&#x3D;Knoll%2C+F" target="_none">Knoll et al (2019)</a></li></a>
          </li>
          <li>
            <a href="https://arxiv.org/abs/1811.08839" class="" target="_none">fastMRI:An Open Dataset and Benchmarks for Accelerated MRI</a> by <a href="https://arxiv.org/search/cs?searchtype&#x3D;author&amp;query&#x3D;Zbontar%2C+J" target="_none">Zbontar et al (2019)</a></li></a>
          </li>
          <li>
            <a href="https://medium.com/@khodges42/how-to-get-started-diy-neurohacking-with-the-fastmri-dataset-and-why-you-should-6a02a6bb896e" class="" target="_none">How to Get Started Working with the FastMRI Dataset (and why you should!)</a> by Kevin Hodges</li></a>
          </li>
          <li>
            <a href="https://arxiv.org/abs/2001.02518" class="" target="_none">Advancing machine learning for MR image reconstruction with an open competition:Overview of the 2019 fastMRI challenge</a> by <a href="https://arxiv.org/search/eess?searchtype&#x3D;author&amp;query&#x3D;Knoll%2C+F" target="_none">Knoll et al (2020)</a></li></a>
          </li>
          <li>
            <a href="https://github.com/facebookresearch/fastMRI/blob/master/fastMRI_tutorial.ipynb" class="" target="_none">FastMRI Tutorial (Jupyter Notebook)</a> by <a href="https://scholar.google.com/citations?user&#x3D;LjjLn2YAAAAJ&amp;hl&#x3D;en" target="_none">Tullie Murrell</a></li></a>
          </li>
        </ul>
        <p><a href="/nyu-fastmri/#usageexamples">See 6 usage examples &rarr;</a></p>
        </div>
        <div id="openaq" class="dataset">
          <h3><a href="/openaq/">OpenAQ</a></h3>
          <p><span class="label label-info tag link-tag">air quality</span><span class="label label-info tag link-tag">cities</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info-sustainability tag link-tag">sustainability</span></p>
          <p>Global, aggregated physical air quality data from public data sources provided by government, research-grade and other sources. These awesome groups do the hard work of measuring these data and publicly sharing them, and our community makes them more universally-accessible to both humans and machines.</p>
          <p><a href="/openaq/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/ksree/world-air-quality" class="" target="_none">OpenAQ Aggregator</a> by <a href="https://github.com/ksree" target="_none">Kapil Sreedharan</a></li></a>
          </li>
          <li>
            <a href="https://arisense.io/" class="" target="_none">ARISense</a> by <a href="http://www.aerodyne.com/" target="_none">Aerodyne Research, Inc.</a></li></a>
          </li>
          <li>
            <a href="https://aws.amazon.com/blogs/publicsector/enabling-rapid-covid-19-air-pollution-analysis-across-globe-openaq/" class="aws-link" target="_none">Enabling rapid COVID-19 and air pollution analysis across the globe with OpenAQ and AWS</a> by <a href="https://openaq.org" target="_none">Christa Hasenkopf</a></li></a>
          </li>
          <li>
            <a href="http://www.hackair.eu/" class="" target="_none">hackAIR</a> by hackAir</li></a>
          </li>
          <li>
            <a href="https://kapilsreed.medium.com/how-to-analyze-global-air-quality-using-apache-spark-bigquery-7743fd54e634" class="" target="_none">Analyze Global Air Pollution Using Apache Spark &amp; BigQuery</a> by <a href="https://kapilsreed.medium.com/" target="_none">Kapil Sreedharan</a></li></a>
          </li>
        </ul>
        <p><a href="/openaq/#usageexamples">See 9 usage examples &rarr;</a></p>
        </div>
        <div id="radiant-mlhub" class="dataset">
          <h3><a href="/radiant-mlhub/">Radiant MLHub</a></h3>
          <p><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">labeled</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info-sustainability tag link-tag">sustainability</span></p>
          <p>Radiant MLHub is an open library for geospatial training data that hosts datasets generated by <a href="https://www.radiant.earth/">Radiant Earth Foundation</a>&#39;s team as well as other training data catalogs contributed by Radiant Earth’s partners. Radiant MLHub is open to anyone to access, store, register and/or share their training datasets for high-quality Earth observations. All of the training datasets are stored using a <a href="https://stacspec.org/">SpatioTemporal Asset Catalog (STAC)</a> compliant catalog and exposed through a common API. Training datasets include pairs of imagery and labels for different types of machine learning problems including image ...</p>
          <p><a href="/radiant-mlhub/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/radiantearth/mlhub-tutorials/tree/master/notebooks" class="" target="_none">Radiant MLHub Tutorials with Jupyter Notebooks</a> by <a href="https://www.linkedin.com/in/kbgg/" target="_none">Kevin Booth</a></li></a>
          </li>
          <li>
            <a href="https://zindi.africa/competitions/iclr-workshop-challenge-2-radiant-earth-computer-vision-for-crop-recognition" class="" target="_none">Challenge on Computer Vision for Crop Detection from Satellite Imagery</a> by <a href="https://www.radiant.earth/" target="_none">Radiant Earth</a></li></a>
          </li>
          <li>
            <a href="https://medium.com/radiant-earth-insights/creating-a-machine-learning-commons-for-global-development-256ef3dd46aa" class="" target="_none">Creating a Machine Learning Commons for Global Development</a> by <a href="https://www.linkedin.com/in/hamedalemohammad/" target="_none">Hamed Alemohammad</a></li></a>
          </li>
          <li>
            <a href="https://aws.amazon.com/blogs/publicsector/geo-diverse-open-training-data-as-a-global-public-good/" class="aws-link" target="_none">Geo-Diverse Open Training Data as a Global Public Good</a> by <a href="https://www.linkedin.com/in/hamedalemohammad/" target="_none">Hamed Alemohammad</a></li></a>
          </li>
          <li>
            <a href="https://medium.com/radiant-earth-insights/a-guide-for-collecting-and-sharing-ground-reference-data-for-machine-learning-applications-90664930925e" class="" target="_none">A Guide for Collecting and Sharing Ground Reference Data for Machine Learning Applications</a> by <a href="https://www.linkedin.com/in/yonahbg/" target="_none">Yonah Bromberg Gaber</a></li></a>
          </li>
        </ul>
        <p><a href="/radiant-mlhub/#usageexamples">See 6 usage examples &rarr;</a></p>
        </div>
        <div id="usgs-lidar" class="dataset">
          <h3><a href="/usgs-lidar/">USGS 3DEP LiDAR Point Clouds</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">elevation</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">lidar</span><span class="label label-info-sustainability tag link-tag">sustainability</span></p>
          <p>The goal of the <a href="https://www.usgs.gov/core-science-systems/ngp/3dep">USGS 3D Elevation Program </a> (3DEP) is to collect elevation data in the form of light detection and ranging (LiDAR) data over the conterminous United States, Hawaii, and the U.S. territories, with data acquired over an 8-year period. This dataset provides two realizations of the 3DEP point cloud data. The first resource is a public access organization provided in <a href="https://entwine.io/entwine-point-tile.html">Entwine Point Tiles</a> format, which a lossless, full-density, streamable octree based on <a href="https://laszip.org">LASzip</a> (LAZ) encoding. The second resource is a <a href="https://docs.aws.amazon.com/AmazonS3/latest/dev/RequesterPaysBuckets.html">Requester Pays</a> of the same data in LAZ (Compressed LAS) format. Resource names in bot...</p>
          <p><a href="/usgs-lidar/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/hobu/usgs-lidar/tree/master/lambda" class="" target="_none">Using Lambda Layers with USGS 3DEP LiDAR Point Clouds</a> by <a href="https://twitter.com/howardbutler" target="_none">Howard Butler</a></li></a>
          </li>
          <li>
            <a href="https://aws.amazon.com/blogs/machine-learning/extracting-buildings-and-roads-from-aws-open-data-using-amazon-sagemaker/" class="aws-link" target="_none">Extracting buildings and roads from AWS Open Data using Amazon SageMaker</a> by Yunzhi Shi, Tianyu Zhang, and Xin Chen</li></a>
          </li>
          <li>
            <a href="https://www.usgs.gov/news/usgs-3dep-lidar-point-cloud-now-available-amazon-public-dataset" class="" target="_none">USGS 3DEP Lidar Point Cloud Now Available as Amazon Public Dataset</a> by <a href="https://www.usgs.gov" target="_none">Department of the Interior, U.S. Geological Survey</a></li></a>
          </li>
          <li>
            <a href="https://portal.opentopography.org/datasets" class="" target="_none">OpenTopography access to 3DEP lidar point cloud data</a> by <a href="https://opentopography.org/" target="_none">OpenTopography</a></li></a>
          </li>
          <li>
            <a href="https://www.facebook.com/isptoolbox/line-of-sight-check/" class="" target="_none">Facebook Line of Sight Check</a> by <a href="https://www.facebook.com/isptoolbox/" target="_none">Facebook</a></li></a>
          </li>
        </ul>
        <p><a href="/usgs-lidar/#usageexamples">See 6 usage examples &rarr;</a></p>
        </div>
        <div id="aws-covid19-lake" class="dataset">
          <h3><a href="/aws-covid19-lake/">COVID-19 Data Lake</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">coronavirus</span><span class="label label-info tag link-tag">COVID-19</span><span class="label label-info tag link-tag">health</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">medicine</span><span class="label label-info tag link-tag">MERS</span><span class="label label-info tag link-tag">SARS</span></p>
          <p>A centralized repository of up-to-date and curated datasets on or related to the spread and characteristics of the novel corona virus (SARS-CoV-2) and its associated illness, COVID-19. Globally, there are several efforts underway to gather this data, and we are working with partners to make this crucial data freely available and keep it up-to-date. Hosted on the AWS cloud, we have seeded our curated data lake with COVID-19 case tracking data from Johns Hopkins and The New York Times, hospital bed availability from Definitive Healthcare, and over 45,000 research articles about COVID-19 and rela...</p>
          <p><a href="/aws-covid19-lake/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://go.aws/covid-19-data-lake" class="aws-link" target="_none">Explore the COVID-19 data lake public S3 bucket</a> by <a href="https://aws.amazon.com/big-data/datalakes-and-analytics/" target="_none">AWS Data Lake Team</a></li></a>
          </li>
          <li>
            <a href="https://aws.amazon.com/blogs/big-data/a-public-data-lake-for-analysis-of-covid-19-data/" class="aws-link" target="_none">A public data lake for analysis of COVID-19 data</a> by <a href="https://aws.amazon.com/big-data/datalakes-and-analytics/" target="_none">AWS Data Lake Team</a></li></a>
          </li>
          <li>
            <a href="https://aws.amazon.com/blogs/big-data/exploring-the-public-aws-covid-19-data-lake/" class="aws-link" target="_none">Exploring the public AWS COVID-19 data lake</a> by <a href="https://aws.amazon.com/big-data/datalakes-and-analytics/" target="_none">AWS Data Lake Team</a></li></a>
          </li>
          <li>
            <a href="https://us-east-2.console.aws.amazon.com/cloudformation/home?region&#x3D;us-east-2#/stacks/create/review?templateURL&#x3D;https://covid19-lake.s3.us-east-2.amazonaws.com/cfn/CovidLakeStack.template.json&amp;stackName&#x3D;CovidLakeStack" class="" target="_none">CloudFormation template for Glue Catalog table definitions</a> by <a href="https://aws.amazon.com/big-data/datalakes-and-analytics/" target="_none">AWS Data Lake Team</a></li></a>
          </li>
          <li>
            <a href="https://github.com/aws-samples/query-data-in-s3-with-amazon-athena-and-aws-sdk-for-dotnet" class="aws-link" target="_none">How to use SQL to query data in S3 Bucket with Amazon Athena and AWS SDK for .NET</a> by <a href="https://github.com/orgs/aws-samples/teams/aws-proserve-us-west-applications" target="_none">AWS ProServe US West Applications Team</a></li></a>
          </li>
        </ul>
        <p><a href="/aws-covid19-lake/#usageexamples">See 5 usage examples &rarr;</a></p>
        </div>
        <div id="mmrf-commpass" class="dataset">
          <h3><a href="/mmrf-commpass/">CoMMpass from the Multiple Myeloma Research Foundation</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">STRIDES</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>The Relating Clinical Outcomes in Multiple Myeloma to Personal Assessment of Genetic Profile study is the Multiple Myeloma Research Foundation (MMRF)’s landmark personalized medicine initiative.  CoMMpass is a
longitudinal observation study of around 1000 newly diagnosed myeloma patients receiving various
standard approved treatments. The MMRF’s vision is to track the treatment and results for each
CoMMpass patient so that someday the information can be used to guide decisions for newly
diagnosed patients. CoMMpass checked on patients every 6 months for 8 years, collecting tissue
samples, gene...</p>
          <p><a href="/mmrf-commpass/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1182/blood.V128.22.194.194" class="" target="_none">&quot;Molecular Predictors of Outcome and Drug Response in Multiple Myeloma: An Interim Analysis
of the Mmrf CoMMpass Study&quot;
</a> by Jonathan J Keats, PhD, Gil Speyer, Austin Christofferson, Christophe Legendre, PhD, Jessica
Aldrich, Megan Russell, Lori Cuyugan, Jonathan Adkins, Alex Blanski, Meghan Hodges, Dan
Rohrer, Sundar Jagannath, MD, Ravi Vij, MD, Gregory Orloff, MD, Todd Zimmerman, MD, Ruben
Niesvizky, MD, Darla Liles, MD, Joseph W. Fay, Jeffrey L. Wolf, MD, Robert M Rifkin, Norma C
Gutierrez, MD PhD, Mmrf CoMMpass Network, Jennifer Yesil, MS, Mary Derome, MS, Seungchan
Kim, PhD, Winnie Liang, PhD, Pamela G. Kidd, MD, Scott Jewell, PhD, John David Carpten, PhD,
Daniel Auclair, PhD, Sagar Lonial, MD FACP
</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1182/blood.V122.21.532.532" class="" target="_none">&quot;Interim Analysis Of The MMRF CoMMpass Trial: a Longitudinal Study In Multiple Myeloma
Relating Clinical Outcomes To Genomic and Immunophenotypic Profiles&quot;
</a> by Keats JJ, Craig DW, Liang W, Venkata Y, Kurdoglu A, Aldrich J, Auclair D, Allen K, Harrison
B, Jewell S, Kidd PG, Correll M, Jagannath S, Siegel DS, Vij R, Orloff G, Zimmerman TM, MMRF
CoMMpass Network, Capone W, Carpten J, Lonial S.
</li></a>
          </li>
          <li>
            <a href="https://portal.gdc.cancer.gov/projects/MMRF-COMMPASS" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1182/blood.V126.23.722.722" class="" target="_none">&quot;Identification of Initiating Trunk Mutations and Distinct Molecular Subtypes: An Interim
Analysis of the Mmrf Commpass Study&quot;
</a> by Jonathan J Keats, PhD, Gil Speyer, Legendre Christophe, Christofferson Austin, Kristi
Stephenson, BS, Ahmet Kurdoglu, Megan Russell, Aldrich Jessica, Cuyugan Lori, Jonathan
Adkins, Jackie McDonald, Adrienne Helland, Alex Blanski, Meghan Hodges, Dan Rohrer, Sundar
Jagannath, MD, David Siegel, MD PhD, Ravi Vij, MD MBA, Gregory Orloff, MD, Todd Zimmerman,
MD, Ruben Niesvizky, MD, Darla Liles, MD, Joseph W. Fay, Jeffrey L. Wolf, MD PhD, Robert M.
Rifkin, Norma C Gutierrez, The MMRF CoMMpass Network, Jen Toups, Mary Derome, MS, Winnie
Liang, PhD, Seunchan Kim, Daniel Auclair, PhD, Pamela G. Kidd, MD, Scott Jewell, PhD, John
David Carpten, PhD, Sagar Lonial, MD
</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1182/blood.V124.21.722.722" class="" target="_none">&quot;Interim Analysis of the Mmrf Commpass Trial: Identification of Novel Rearrangements
Potentially Associated with Disease Initiation and Progression&quot;
</a> by Sagar Lonial, MD, Venkata D Yellapantula, Winnie Liang, PhD, Ahmet Kurdoglu, BS, Jessica
Aldrich, MSc, Christophe M. Legendre, MD, Kristi Stephenson, Jonathan Adkins, Jackie
McDonald, Adrienne Helland, Megan Russell, Austin Christofferson, Lori Cuyugan, Dan Rohrer,
Alex Blanski, Meghan Hodges, Mmrf CoMMpass Network, Mary Derome, Daniel Auclair, PhD, Pamela
G. Kidd, MD, Scott Jewell, PhD, David Craig, PhD, John Carpten, PhD, Jonathan J. Keats, PhD
</li></a>
          </li>
        </ul>
        <p><a href="/mmrf-commpass/#usageexamples">See 5 usage examples &rarr;</a></p>
        </div>
        <div id="noaa-hrrr-pds" class="dataset">
          <h3><a href="/noaa-hrrr-pds/">NOAA High-Resolution Rapid Refresh (HRRR) Model</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info-sustainability tag link-tag">sustainability</span><span class="label label-info tag link-tag">weather</span></p>
          <p>The HRRR is a NOAA real-time 3-km resolution, hourly updated, cloud-resolving, convection-allowing atmospheric model, initialized by 3km grids with 3km radar assimilation. Radar data is assimilated in the HRRR every 15 min over a 1-h period adding further detail to that provided by the hourly data assimilation from the 13km radar-enhanced Rapid Refresh.</p>
          <p><a href="/noaa-hrrr-pds/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://mesowest.utah.edu/html/hrrr/zarr_documentation/What_is_Zarr.html" class="" target="_none">What is Zarr?</a> by Taylor Gowan</li></a>
          </li>
          <li>
            <a href="https://mesowest.utah.edu/html/hrrr/zarr_documentation/conda_environment.html" class="" target="_none">Conda Enironment Setup Guide</a> by Zach Rieck</li></a>
          </li>
          <li>
            <a href="https://mesowest.utah.edu/html/hrrr/zarr_documentation/zarr_examples.html" class="" target="_none">Zarr Visualization Example</a> by Taylor Gowan, James Powell, Zach Rieck</li></a>
          </li>
          <li>
            <a href="https://mesowest.utah.edu/html/hrrr/zarr_documentation/zarrFileVariables.html" class="" target="_none">Zarr File Varable Definitions</a> by Taylor Gowan</li></a>
          </li>
          <li>
            <a href="https://github.com/blaylockbk/HRRR_archive_download" class="" target="_none">HRRR-B Python package: download and read HRRR grib2 files</a> by Brian Blaylock</li></a>
          </li>
        </ul>
        <p><a href="/noaa-hrrr-pds/#usageexamples">See 5 usage examples &rarr;</a></p>
        </div>
        <div id="nrel-oedi-pv-rooftops" class="dataset">
          <h3><a href="/nrel-oedi-pv-rooftops/">National Renewable Energy Laboratory&#x27;s (NREL) PV Rooftop Database</a></h3>
          <p><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">lidar</span><span class="label label-info tag link-tag">solar</span><span class="label label-info-sustainability tag link-tag">sustainability</span></p>
          <p>Released to the public as part of the Department of Energy&#39;s Open Energy Data Initiative, the National Renewable Energy Laboratory&#39;s (NREL) PV Rooftop Database (PVRDB) is a lidar-derived, geospatially-resolved dataset of suitable roof surfaces and their PV technical potential for 128 metropolitan regions in the United States. The source lidar data and building footprints were obtained by the U.S. Department of Homeland Security Homeland Security Infrastructure Program for 2006-2014. Using GIS methods, NREL identified suitable roof surfaces based on their size, orientation, and shading ...</p>
          <p><a href="/nrel-oedi-pv-rooftops/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.nrel.gov/docs/fy16osti/65298.pdf" class="" target="_none">Rooftop Solar Photovoltaic Technical Potential in the United States: A Detailed Assessment</a> by <a href="https://www.nrel.gov/" target="_none">Pieter Gagnon, Robert Margolis, Jennifer Melius, Caleb Phillips, and Ryan Elmore</a></li></a>
          </li>
          <li>
            <a href="https://www.nrel.gov/docs/fy18osti/70901.pdf" class="" target="_none">Rooftop Solar Technical Potential for Low-to-Moderate Income Households in the United States</a> by <a href="https://www.nrel.gov/" target="_none">Benjamin Sigrin and Meghan Mooney</a></li></a>
          </li>
          <li>
            <a href="https://iopscience.iop.org/article/10.1088/1748-9326/aa7225/pdf" class="" target="_none">Using GIS-based methods and lidar data to estimate rooftop solar technical potential in US cities</a> by Robert Margolis et al 2017 Environ. Res. Lett. 12 074013</li></a>
          </li>
          <li>
            <a href="https://maps.nrel.gov/nsrdb-viewer/?aL&#x3D;0&amp;bL&#x3D;groad&amp;cE&#x3D;0&amp;lR&#x3D;0&amp;mC&#x3D;31.653381399664%2C-3.1640625&amp;zL&#x3D;2" class="" target="_none">NSRDB Viewer</a> by <a href="https://www.nrel.gov/" target="_none">NREL</a></li></a>
          </li>
          <li>
            <a href="https://iopscience.iop.org/article/10.1088/1748-9326/aaa554/pdf" class="" target="_none">Estimating rooftop solar technical potential across the US using a combination of GIS-based methods, lidar data, and statistical modeling</a> by Pieter Gagnon et al 2018 Environ. Res. Lett. 13 024027</li></a>
          </li>
        </ul>
        <p><a href="/nrel-oedi-pv-rooftops/#usageexamples">See 5 usage examples &rarr;</a></p>
        </div>
        <div id="nyc-tlc-trip-records-pds" class="dataset">
          <h3><a href="/nyc-tlc-trip-records-pds/">New York City Taxi and Limousine Commission (TLC) Trip Record Data</a></h3>
          <p><span class="label label-info tag link-tag">cities</span><span class="label label-info tag link-tag">transportation</span><span class="label label-info tag link-tag">urban</span></p>
          <p>Data of trips taken by taxis and for-hire vehicles in New York City.</p>
          <p><a href="/nyc-tlc-trip-records-pds/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://aws.amazon.com/blogs/big-data/build-and-run-streaming-applications-with-apache-flink-and-amazon-kinesis-data-analytics-for-java-applications/" class="aws-link" target="_none">Build and run streaming applications with Apache Flink and Amazon Kinesis Data Analytics for Java Applications</a> by Steffen Hausmann</li></a>
          </li>
          <li>
            <a href="https://aws.amazon.com/blogs/big-data/build-a-real-time-stream-processing-pipeline-with-apache-flink-on-aws/" class="aws-link" target="_none">Build a Real-time Stream Processing Pipeline with Apache Flink on AWS</a> by Steffen Hausmann</li></a>
          </li>
          <li>
            <a href="https://github.com/aws-samples/aws-open-data-analytics-notebooks/tree/master/optimizing-data" class="aws-link" target="_none">Optimizing data for analysis with Amazon Athena and AWS Glue</a> by Manav Sehgal</li></a>
          </li>
          <li>
            <a href="https://www.youtube.com/watch?v&#x3D;MzTZp47Jy7E" class="" target="_none">Deep Dive on Flink &amp; Spark on Amazon EMR</a> by Keith Steward</li></a>
          </li>
          <li>
            <a href="https://github.com/aws-samples/aws-open-data-analytics-notebooks/tree/master/exploring-data" class="aws-link" target="_none">Exploring data with Python and Amazon S3 Select</a> by Manav Sehgal</li></a>
          </li>
        </ul>
        <p><a href="/nyc-tlc-trip-records-pds/#usageexamples">See 5 usage examples &rarr;</a></p>
        </div>
        <div id="omi-no2-nasa" class="dataset">
          <h3><a href="/omi-no2-nasa/">Ozone Monitoring Instrument (OMI) / Aura NO2 Tropospheric Column Density</a></h3>
          <p><span class="label label-info tag link-tag">air quality</span><span class="label label-info tag link-tag">atmosphere</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info-sustainability tag link-tag">sustainability</span></p>
          <p>NO2 tropospheric column density, screened for CloudFraction &lt; 30% global daily
 composite at 0.25 degree resolution for the temporal range of 2004 to May
2020. Original archive data in HDF5 has been processed into a <a href="https://www.cogeo.org/">Cloud-Optimized
GeoTiff (COG)</a> format. Quality Assurance - This data
has been validated by the NASA Science Team at Goddard Space Flight Center.Cautionary Note: <a href="https://airquality.gsfc.nasa.gov/caution-interpretation">https://airquality.gsfc.nasa.gov/caution-interpretation</a>.</p>
          <p><a href="/omi-no2-nasa/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://cogeo.xyz/?url&#x3D;s3://omi-no2-nasa/OMI-Aura_L3-OMNO2d_2020m0101_v003-2020m0330t173100.tif" class="" target="_none">COG Viewer</a> by <a href="https://developmentseed.org/" target="_none">Development Seed</a></li></a>
          </li>
          <li>
            <a href="https://rasterio.readthedocs.io/" class="" target="_none">Rasterio (Python Library) for access to geospatial raster data</a> by <a href="https://www.mapbox.com/" target="_none">Mapbox</a></li></a>
          </li>
          <li>
            <a href="https://pfg2k5mjei.execute-api.us-west-2.amazonaws.com/v1/metadata?url&#x3D;https%3A%2F%2Fomi-no2-nasa.s3-us-west-2.amazonaws.com%2FOMI-Aura_L3-OMNO2d_2018m0101_v003-2019m1123t055315.tif&amp;pmin&#x3D;2&amp;pmax&#x3D;98&amp;max_size&#x3D;1024&amp;histogram_bins&#x3D;20" class="" target="_none">COG metadata example </a> by <a href="https://developmentseed.org/" target="_none">Development Seed</a></li></a>
          </li>
          <li>
            <a href="https://medium.com/devseed/cog-talk-part-1-whats-new-941facbcd3d1" class="" target="_none">Medium Post: COG Talk — Part 1: What’s new?</a> by <a href="https://medium.com/@_VincentS_" target="_none">Vincent Sarago</a></li></a>
          </li>
          <li>
            <a href="https://pfg2k5mjei.execute-api.us-west-2.amazonaws.com/docs" class="" target="_none">COG Application Programming Interface (API)</a> by <a href="https://developmentseed.org/" target="_none">Development Seed</a></li></a>
          </li>
        </ul>
        <p><a href="/omi-no2-nasa/#usageexamples">See 5 usage examples &rarr;</a></p>
        </div>
        <div id="pmsp-lidar" class="dataset">
          <h3><a href="/pmsp-lidar/">Prefeitura Municipal de São Paulo (PMSP) LiDAR Point Cloud</a></h3>
          <p><span class="label label-info tag link-tag">cities</span><span class="label label-info tag link-tag">elevation</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">land</span><span class="label label-info tag link-tag">lidar</span><span class="label label-info tag link-tag">mapping</span><span class="label label-info tag link-tag">urban</span></p>
          <p>The objective of the Mapa 3D Digital da Cidade (M3DC) of the São Paulo City Hall is to publish LiDAR point cloud data. The initial data was acquired in 2017 by aerial surveying and future data will be added. This publicly accessible dataset is provided in the <a href="https://entwine.io/entwine-point-tile.html">Entwine Point Tiles</a> format as a lossless octree, full density, based on <a href="https://laszip.org">LASzip</a> (LAZ) encoding.</p>
          <p><a href="/pmsp-lidar/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://entwine.io/" class="" target="_none">Entwine</a> by <a href="https://hobu.co/" target="_none">Hobu, Inc.</a></li></a>
          </li>
          <li>
            <a href="https://rapidlasso.com/lastools/" class="" target="_none">LAStools</a> by <a href="https://rapidlasso.com" target="_none">rapidlasso GmbH, GERMANY</a></li></a>
          </li>
          <li>
            <a href="https://pdal.io/" class="" target="_none">PDAL - Point Data Abstraction Library</a> by <a href="https://doi.org/10.5281/zenodo.2556738" target="_none">PDAL Contributors</a></li></a>
          </li>
          <li>
            <a href="http://forsys.sefs.uw.edu/fusion/fusionlatest.html" class="" target="_none">Fusion</a> by <a href="https://www.usda.gov/" target="_none">US Department of Agriculture - Forest Service</a></li></a>
          </li>
          <li>
            <a href="https://doi.org/10.5194/isprs-annals-IV-2-W5-437-2019" class="" target="_none">Describing the Vertical Structure of Informal Settlements on the Basis of LiDAR Data – A Case Study for Favelas (Slums) in Sao Paulo City</a> by S. C. L. Ribeiro, M. Jarzabek-Rychard, J. P. Cintra, H.-G. Maas</li></a>
          </li>
        </ul>
        <p><a href="/pmsp-lidar/#usageexamples">See 5 usage examples &rarr;</a></p>
        </div>
        <div id="sondehub-telemetry" class="dataset">
          <h3><a href="/sondehub-telemetry/">SondeHub Radiosonde Telemetry</a></h3>
          <p><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">GPS</span><span class="label label-info tag link-tag">weather</span></p>
          <p>SondeHub Radiosonde telemetry contains global radiosonde (weather balloon) data captured by SondeHub from our participating radiosonde_auto_rx receiving stations. radiosonde_auto_rx is a open source project aimed at receiving and decoding telemetry from airborne radiosondes using software-defined-radio techniques, enabling study of the telemetry and sometimes recovery of the radiosonde itself.
Currently 313 receiver stations are providing data for an average of 384 radiosondes a day.  The data within this repository contains received telemetry frames, including radiosonde type, gps position, a...</p>
          <p><a href="/sondehub-telemetry/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.youtube.com/watch?v&#x3D;YBy-bXEWZeM" class="" target="_none">STM32 Development Boards (literally) Falling From The Sky (How to submit data)</a> by Mark Jessop &amp; Michaela Wheeler</li></a>
          </li>
          <li>
            <a href="https://github.com/projecthorus/pysondehub/blob/main/examples/README.md" class="" target="_none">Loading example notebooks into SageMaker</a> by Michaela Wheeler</li></a>
          </li>
          <li>
            <a href="https://github.com/projecthorus/pysondehub/blob/main/examples/sondehub.ipynb" class="" target="_none">Using pysondehub to read radiosonde data</a> by Michaela Wheeler</li></a>
          </li>
          <li>
            <a href="https://github.com/projecthorus/pysondehub/blob/main/examples/sondehub-athena.ipynb" class="" target="_none">Using Athena to read radiosonde data</a> by Michaela Wheeler</li></a>
          </li>
          <li>
            <a href="https://pypi.org/project/sondehub/" class="" target="_none">pysondehub</a> by <a href="https://github.com/projecthorus/pysondehub" target="_none">Sondehub</a></li></a>
          </li>
        </ul>
        <p><a href="/sondehub-telemetry/#usageexamples">See 5 usage examples &rarr;</a></p>
        </div>
        <div id="southern-california-earthquakes" class="dataset">
          <h3><a href="/southern-california-earthquakes/">Southern California Earthquake Data</a></h3>
          <p><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">earthquakes</span><span class="label label-info tag link-tag">seismology</span><span class="label label-info-sustainability tag link-tag">sustainability</span></p>
          <p>This dataset contains ground motion velocity and acceleration seismic waveforms recorded by the Southern California Seismic Network (SCSN) and archived at the Southern California Earthquake Data Center (SCEDC).</p>
          <p><a href="/southern-california-earthquakes/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://nextjournal.com/thclements/cactus-to-clouds/#accessing-the-scedc-dataset-on-aws" class="" target="_none">Cactus to Clouds: Processing The SCEDC Open Data Set on AWS</a> by <a href="https://tclements.github.io/" target="_none">Tim Clements</a></li></a>
          </li>
          <li>
            <a href="https://nextjournal.com/thclements/seisnoisejl-gpu-computing-tutorial" class="" target="_none">SeisNoise.jl GPU Computing Tutorial - Another example of accessing data s3://scedc-pds for ambient noise cross-correlation</a> by <a href="https://tclements.github.io/" target="_none">Tim Clements</a></li></a>
          </li>
          <li>
            <a href="https://github.com/SCEDC/cloud/tree/master/pds-fetch-continuous-data" class="" target="_none">Script to Download Seismic Waveforms from the SCEDC AWS Public Dataset</a> by <a href="https://scedc.caltech.edu" target="_none">Aparna Bhaskaran</a></li></a>
          </li>
          <li>
            <a href="https://github.com/SCEDC/cloud/tree/master/pds-lambda-example" class="" target="_none">Using Lambda to Process Seismograms</a> by <a href="https://scedc.caltech.edu" target="_none">Shang-Lin Chen</a></li></a>
          </li>
          <li>
            <a href="https://scedc.caltech.edu/data/getstarted-pds.html" class="" target="_none">Getting Started with SCEDC AWS Public Dataset</a> by <a href="https://scedc.caltech.edu" target="_none">Ellen Yu</a></li></a>
          </li>
        </ul>
        <p><a href="/southern-california-earthquakes/#usageexamples">See 5 usage examples &rarr;</a></p>
        </div>
        <div id="ilmn-dragen-1kgp" class="dataset">
          <h3><a href="/ilmn-dragen-1kgp/">1000 Genomes Phase 3 Reanalysis with DRAGEN 3.5</a></h3>
          <p><span class="label label-info tag link-tag">bam</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">health</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">vcf</span></p>
          <p>This dataset contains alignment files and short nucleotide, copy number, and structural variant call files from the newly revised (n=3202) 1000 Genomes Project Phase 3 using Illumina DRAGEN v3.5.7b</p>
          <p><a href="/ilmn-dragen-1kgp/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.biorxiv.org/content/10.1101/2020.11.13.380741v1" class="" target="_none">precisionFDA Truth Challenge V2: Calling variants from short- and long-reads in difficult-to-map regions (Preprint)</a> by Olson et al (2020)</li></a>
          </li>
          <li>
            <a href="https://github.com/aws-quickstart/quickstart-illumina-dragen" class="" target="_none">DRAGEN Quick Start on AWS</a> by <a href="https://github.com/aws-quickstart" target="_none">AWS Quick Start Team</a></li></a>
          </li>
          <li>
            <a href="https://support.illumina.com/content/dam/illumina-support/documents/downloads/software/dragen/DRAGEN-3.5.7b-Customer-Release-Notes-1000000131537-00.pdf" class="" target="_none">DRAGEN 3.7.5b Release</a> by <a href="https://www.illumina.com/" target="_none">Illumina Inc.</a></li></a>
          </li>
          <li>
            <a href="https://www.illumina.com/products/by-type/informatics-products/basespace-sequence-hub.html#key-features" class="" target="_none">DRAGEN on BaseSpace Sequence Hub</a> by <a href="https://www.illumina.com/" target="_none">Illumina Inc.</a></li></a>
          </li>
        </ul>
        <p><a href="/ilmn-dragen-1kgp/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="3kricegenome" class="dataset">
          <h3><a href="/3kricegenome/">3000 Rice Genomes Project</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">food security</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span></p>
          <p>The 3000 Rice Genome Project is an international effort to sequence the genomes of 3,024 rice varieties from 89 countries.</p>
          <p><a href="/3kricegenome/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://genome.cshlp.org/content/29/5/870.short" class="" target="_none">Structural variants in 3000 rice genomes</a> by Fuentes RR et al (2019)</li></a>
          </li>
          <li>
            <a href="https://github.com/InternationalRiceResearchInstitute/RiceGalaxy" class="" target="_none">RiceGalaxy</a> by <a href="https://www.irri.org/" target="_none">International Rice Research Institute</a></li></a>
          </li>
          <li>
            <a href="https://academic.oup.com/gigascience/article/8/5/giz028/5492234" class="" target="_none">Rice Galaxy: an open resource for plant science</a> by Juanillas V et al (2019)</li></a>
          </li>
          <li>
            <a href="https://www.nature.com/articles/s41467-019-08451-3" class="" target="_none">Tracking the origin of two genetic components associated with transposable element bursts in domesticated rice</a> by Chen J et al (2019)</li></a>
          </li>
        </ul>
        <p><a href="/3kricegenome/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="ncar-cesm-lens" class="dataset">
          <h3><a href="/ncar-cesm-lens/">Community Earth System Model Large Ensemble (CESM LENS)</a></h3>
          <p><span class="label label-info tag link-tag">atmosphere</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">ice</span><span class="label label-info tag link-tag">land</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">model</span><span class="label label-info tag link-tag">oceans</span><span class="label label-info-sustainability tag link-tag">sustainability</span></p>
          <p>The Community Earth System Model (CESM) Large Ensemble Numerical Simulation (LENS) dataset includes a 40-member ensemble of climate simulations for the period 1920-2100 using historical data (1920-2005) or assuming the RCP8.5 greenhouse gas concentration scenario (2006-2100), as well as longer control runs based on pre-industrial conditions. The data comprise both surface (2D) and volumetric (3D) variables in the atmosphere, ocean, land, and ice domains. The total data volume of the original dataset is ~500TB, which has traditionally been stored as ~150,000 individual CF/NetCDF files on disk o...</p>
          <p><a href="/ncar-cesm-lens/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/NCAR/cesm-lens-aws" class="" target="_none">Jupyter Notebook and other documentation and tools for CESM LENS on AWS</a> by NCAR Science at Scale team</li></a>
          </li>
          <li>
            <a href="http://ncar-aws-www.s3-website-us-west-2.amazonaws.com/kay-et-al-2015.v2.html" class="" target="_none">Rendered (static) version of Jupyter Notebook</a> by Anderson Banihirwe, NCAR</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.5065/d6j101d1" class="" target="_none">The Community Earth System Model (CESM) Large Ensemble Project: A Community Resource for Studying Climate Change in the Presence of Internal Climate Variability</a> by Kay et al. (2015), Bull. AMS, 96, 1333-1349</li></a>
          </li>
          <li>
            <a href="https://medium.com/pangeo/cesm-lens-on-aws-4e2a996397a1" class="" target="_none">Analyzing large climate model ensembles in the cloud</a> by Joe Hamman, NCAR</li></a>
          </li>
        </ul>
        <p><a href="/ncar-cesm-lens/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="wpto-pds-us-wave" class="dataset">
          <h3><a href="/wpto-pds-us-wave/">DOE&#x27;s Water Power Technology Office&#x27;s (WPTO) US Wave dataset</a></h3>
          <p><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">energy</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info-sustainability tag link-tag">sustainability</span><span class="label label-info tag link-tag">water</span></p>
          <p>Released to the public as part of the Department of Energy&#39;s Open Energy Data Initiative,
this is the highest resolution publicly available long-term wave hindcast
dataset that – when complete – will cover the entire U.S. Exclusive Economic
Zone (EEZ).</p>
          <p><a href="/wpto-pds-us-wave/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.osti.gov/biblio/1599105" class="" target="_none">Development and validation of a high-resolution regional wave hindcast model for U.S. West Coast wave   resource characterization</a> by Wu, Wei-Cheng; Wang, Taiping; Yang, Zhaoqing; Garcia Medina, Gabriel</li></a>
          </li>
          <li>
            <a href="https://github.com/NREL/hsds-examples" class="" target="_none">HSDS Examples</a> by Caleb Phillips, Caroline Draxl, John Readey, Jordan Perr-Sauer, Michael Rossol</li></a>
          </li>
          <li>
            <a href="http://swanmodel.sourceforge.net/download/zip/swanuse.pdf" class="" target="_none">SWAN Cycle III version 41.31A</a> by The SWAN team</li></a>
          </li>
          <li>
            <a href="https://www.osti.gov/biblio/1573061/" class="" target="_none">High-Resolution Regional Wave Hindcast for the U.S. West Coast</a> by Yang, Zhaoqing; Wu, Wei-Cheng; Wang, Taiping; Castrucci, Luca</li></a>
          </li>
        </ul>
        <p><a href="/wpto-pds-us-wave/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="oedi-data-lake" class="dataset">
          <h3><a href="/oedi-data-lake/">Department of Energy&#x27;s Open Energy Data Initiative (OEDI)</a></h3>
          <p><span class="label label-info tag link-tag">energy</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">model</span><span class="label label-info tag link-tag">solar</span><span class="label label-info-sustainability tag link-tag">sustainability</span></p>
          <p>Data released under the Department of Energy&#39;s Open Energy Data Initiative
(DOE). The Open Energy Data Initiative (OEDI) aims to improve and automate
access of high-value energy data sets across the U.S. Department of Energy’s
(DOE’s) programs, offices, and national laboratories. OEDI aims to make data
actionable and discoverable by researchers and industry to accelerate
analysis and advance innovation.</p>
          <p><a href="/oedi-data-lake/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://emp.lbl.gov/tracking-sun-tool" class="" target="_none">Tracking the Sun Tool</a> by <a href="https://emp.lbl.gov/tracking-the-sun" target="_none">Lawrence Berkeley National Laboratory (LBNL)</a></li></a>
          </li>
          <li>
            <a href="https://www.nrel.gov/docs/fy16osti/65283.pdf" class="" target="_none">On the Use of Coupled Wind, Wave, and Current Fields in the Simulation of Loads on BottomSupported Offshore Wind Turbines during Hurricanes</a> by E. Kim, L. Manuel, M. Curcic, S. S. Chen, C. Phillips, P. Veers</li></a>
          </li>
          <li>
            <a href="https://emp.lbl.gov/sites/default/files/tracking_the_sun_2019_report.pdf" class="" target="_none">Tracking the Sun Pricing and Design Trends for Distributed Photovoltaic Systems in the United States: 2019 Edition</a> by G. Barbose, N. Darghouth</li></a>
          </li>
          <li>
            <a href="https://www.nrel.gov/docs/fy16osti/65231.pdf" class="" target="_none">The Distributed Generation Market Demand Model (dGen):Documentation</a> by B. Sigrin, M. Gleason, R. Preus, I. Baring-Gould, R. Margolis</li></a>
          </li>
        </ul>
        <p><a href="/oedi-data-lake/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="ecmwf-era5" class="dataset">
          <h3><a href="/ecmwf-era5/">ECMWF ERA5 Reanalysis</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info-sustainability tag link-tag">sustainability</span><span class="label label-info tag link-tag">weather</span></p>
          <p>ERA5 is the fifth generation of ECMWF atmospheric reanalyses of the global climate, and the first reanalysis produced as an operational service. It utilizes the best available observation data from satellites and in-situ stations, which are assimilated and processed using ECMWF&#39;s Integrated Forecast System (IFS) Cycle 41r2.
The dataset provides all essential atmospheric meteorological parameters like, but not limited to, air temperature, pressure and wind at different altitudes, along with surface parameters like rainfall, soil moisture content and sea parameters like sea-surface temperatu...</p>
          <p><a href="/ecmwf-era5/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://nbviewer.jupyter.org/github/awslabs/amazon-asdi/blob/main/examples/dask/notebooks/era5.ipynb" class="aws-link" target="_none">Processing ERA5 data in NetCDF Format</a> by Zac Flamig</li></a>
          </li>
          <li>
            <a href="https://github.com/planet-os/notebooks/blob/master/aws/era5-s3-via-boto.ipynb" class="" target="_none">Accessing ERA5 Data on S3 Using Boto</a> by <a href="https://www.intertrust.com/" target="_none">Intertrust Technologies Corporation</a></li></a>
          </li>
          <li>
            <a href="https://github.com/planet-os/notebooks/blob/master/api-examples/ERA5_tutorial.ipynb" class="" target="_none">ERA5 tutorial using the Planet OS API</a> by <a href="https://www.intertrust.com/" target="_none">Intertrust Technologies Corporation</a></li></a>
          </li>
          <li>
            <a href="https://nbviewer.jupyter.org/github/awslabs/amazon-asdi/blob/main/examples/dask/notebooks/era5_zarr.ipynb" class="aws-link" target="_none">Processing ERA5 data in Zarr Format</a> by Zac Flamig</li></a>
          </li>
        </ul>
        <p><a href="/ecmwf-era5/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="encode-project" class="dataset">
          <h3><a href="/encode-project/">Encyclopedia of DNA Elements (ENCODE)</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">deep learning</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">machine learning</span></p>
          <p>The Encyclopedia of DNA Elements (ENCODE) Consortium is an international collaboration of
research groups funded by the National Human Genome Research Institute (NHGRI). The goal
of ENCODE is to build a comprehensive parts list of functional elements in the human genome,
including elements that act at the protein and RNA levels, and regulatory elements that
control cells and circumstances in which a gene is active. ENCODE investigators employ a
variety of assays and methods to identify functional elements. The discovery and annotation
of gene elements is accomplished primarily by sequencing a ...</p>
          <p><a href="/encode-project/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/ENCODE-DCC/encode-data-usage-examples/blob/master/ingest_encode_data_tile_db_with_s3_backend.ipynb" class="" target="_none">Ingesting ENCODE data into TileDB with S3 backend</a> by <a href="https://github.com/ottojolanki" target="_none">Otto Jolanki</a></li></a>
          </li>
          <li>
            <a href="https://academic.oup.com/nar/article/48/D1/D882/5622708#190992715" class="" target="_none">New developments on the Encyclopedia of DNA Elements (ENCODE) data portal</a> by <a href="http://orcid.org/0000-0001-6948-2042" target="_none">Luo et al 2020</a></li></a>
          </li>
          <li>
            <a href="https://github.com/ENCODE-DCC/encode-data-usage-examples/blob/master/mount_s3_bucket_and_run_jupyter_on_ec2.ipynb" class="" target="_none">Exploring ENCODE data from EC2 with Jupyter notebook</a> by <a href="https://github.com/keenangraham" target="_none">Keenan Graham</a></li></a>
          </li>
          <li>
            <a href="https://github.com/ENCODE-DCC/encode-data-usage-examples/blob/master/ctcf_chip_seq_cross_cell_type_correlation.ipynb" class="" target="_none">ENCODE CTCF ChIP-seq data correlation across different cell types</a> by <a href="https://github.com/p-sud" target="_none">Paul Sud</a></li></a>
          </li>
        </ul>
        <p><a href="/encode-project/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="fsf-flood-risk" class="dataset">
          <h3><a href="/fsf-flood-risk/">First Street Foundation (FSF) Flood Risk Summary Statistics</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">model</span><span class="label label-info tag link-tag">statistics</span><span class="label label-info-sustainability tag link-tag">sustainability</span><span class="label label-info tag link-tag">water</span><span class="label label-info tag link-tag">weather</span></p>
          <p>CSV files of flood statistics for the 48 contiguous states at the congressional district, county, and zip code level. The CSV for each of these geographical extents includes statistics on the amount of properties at risk according to FEMA, the number of properties at risk according to First Street Foundation, and the difference between the two.</p>
          <p><a href="/fsf-flood-risk/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://firststreet.org/flood-lab/" class="" target="_none">First Street Foundation Flood Lab</a> by First Street Foundation</li></a>
          </li>
          <li>
            <a href="https://link.springer.com/article/10.1007/s11113-018-9473-5" class="" target="_none">Estimating Recent Local Impacts of Sea-Level Rise on Current Real-Estate Losses: A Housing Market Case Study in Miami-Dade, Florida</a> by Steven A. McAlpine, Jeremy R. Porter</li></a>
          </li>
          <li>
            <a href="https://eos.org/opinions/do-you-know-your-homes-flood-risk" class="" target="_none">Do You Know Your Home’s Flood Risk?</a> by Edward Kearns, Jeremy Porter, Michael Amodeo</li></a>
          </li>
          <li>
            <a href="https://agupubs.onlinelibrary.wiley.com/doi/full/10.1002/2017WR020917" class="" target="_none">Validation of a 30 m resolution flood hazard model of the conterminous United States</a> by Oliver E. J. Wing, Paul D. Bates, Christopher C. Sampson, Andrew M. Smith, Kris A. Johnson, Tyler A. Erickson</li></a>
          </li>
        </ul>
        <p><a href="/fsf-flood-risk/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="geoschem-input-data" class="dataset">
          <h3><a href="/geoschem-input-data/">GEOS-Chem Input Data</a></h3>
          <p><span class="label label-info tag link-tag">air quality</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info-sustainability tag link-tag">sustainability</span><span class="label label-info tag link-tag">weather</span></p>
          <p>Input data for the GEOS-Chem Chemical Transport Model. Including the NASA/GMAO MERRA-2 and GEOS-FP <a href="http://wiki.seas.harvard.edu/geos-chem/index.php/Overview_of_GMAO_met_data_products">meteorological products</a>, the <a href="http://wiki.seas.harvard.edu/geos-chem/index.php/HEMCO_data_directories">HEMCO emission inventories</a>, and other small data such as <a href="http://wiki.seas.harvard.edu/geos-chem/index.php/GEOS-Chem_basics#Restart_files">model initial conditions</a>.</p>
          <p><a href="/geoschem-input-data/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="http://acmg.seas.harvard.edu/presentations/IGC8/posters/Mon_Posters_Zhuang_Jiawei.pdf" class="" target="_none">Running GEOS-Chem on Cloud Computing Platforms, presented at the 8th International GEOS-Chem Meeting</a> by <a href="https://github.com/JiaweiZhuang" target="_none">Jiawei Zhuang, et al.</a></li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1175/BAMS-D-18-0243.1" class="" target="_none">Enabling Immediate Access to Earth Science Models through Cloud Computing: Application to the GEOS-Chem Model</a> by Jiawei Zhuang, et al.</li></a>
          </li>
          <li>
            <a href="http://acmg.seas.harvard.edu/research.html#cloud" class="" target="_none">Overview of the GEOSChem-on-cloud project</a> by <a href="http://acmg.seas.harvard.edu/index.html" target="_none">Atmospheric Chemistry Modeling Group, Harvard University</a></li></a>
          </li>
          <li>
            <a href="http://cloud-gc.readthedocs.io/en/stable/chapter02_beginner-tutorial/use-s3.html#access-geos-chem-input-data-repository-in-s3" class="" target="_none">Tutorial on accessing GEOS-Chem data bucket in S3</a> by <a href="https://github.com/JiaweiZhuang" target="_none">Jiawei Zhuang</a></li></a>
          </li>
        </ul>
        <p><a href="/geoschem-input-data/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="giab" class="dataset">
          <h3><a href="/giab/">Genome in a Bottle on AWS</a></h3>
          <p><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">reference index</span><span class="label label-info tag link-tag">vcf</span></p>
          <p>Several reference genomes to enable translation of whole human genome sequencing to clinical practice. On 11/12/2020 these data were updated to reflect the most <a href="https://www.nist.gov/programs-projects/genome-bottle">up to date GIAB release</a>.</p>
          <p><a href="/giab/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.nature.com/articles/s41597-019-0098-2" class="" target="_none">High-coverage, long-read sequencing of Han Chinese trio reference samples</a> by Wang Y et al (2019)</li></a>
          </li>
          <li>
            <a href="https://www.nature.com/articles/sdata201625" class="" target="_none">Extensive sequencing of seven human genomes to characterize benchmark reference materials</a> by Zook J et al (2016)</li></a>
          </li>
          <li>
            <a href="https://github.com/genome-in-a-bottle/about_GIAB" class="" target="_none">The Genome in a Bottle Github Project</a> by Genome In A Bottle Consortium</li></a>
          </li>
          <li>
            <a href="https://github.com/ga4gh/benchmarking-tools" class="" target="_none">GA4GH Benchmarking Tools</a> by GA4GH Benchmarking Team</li></a>
          </li>
        </ul>
        <p><a href="/giab/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="ncbi-sra" class="dataset">
          <h3><a href="/ncbi-sra/">NIH NCBI Sequence Read Archive (SRA) on AWS</a></h3>
          <p><span class="label label-info tag link-tag">bam</span><span class="label label-info tag link-tag">fastq</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">STRIDES</span><span class="label label-info tag link-tag">transcriptomics</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>The Sequence Read Archive (SRA), produced by the <a href="https://www.ncbi.nlm.nih.gov/">National Center for Biotechnology Information (NCBI)</a> at the <a href="http://nlm.nih.gov/">National Library of Medicine (NLM)</a> at the <a href="http://www.nih.gov/">National Institutes of Health (NIH)</a>, stores raw DNA sequencing data and alignment information from high-throughput sequencing platforms. The SRA provides open access to these biological sequence data to support the research community&#39;s efforts to enhance reproducibility and make new discoveries by comparing data s...</p>
          <p><a href="/ncbi-sra/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/ncbi/sra-tools" class="" target="_none">SRA Toolkit</a> by <a href="https://www.ncbi.nlm.nih.gov/sra/" target="_none">NCBI SRA</a></li></a>
          </li>
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/sra/docs/sra-cloud/" class="" target="_none">SRA in the Cloud</a> by <a href="https://www.ncbi.nlm.nih.gov/sra/" target="_none">NCBI SRA</a></li></a>
          </li>
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3013647/" class="" target="_none">The Sequence Read Archive</a> by <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term&#x3D;Leinonen%20R%5BAuthor%5D&amp;cauthor&#x3D;true&amp;cauthor_uid&#x3D;21062823" target="_none">Leinonen et al (2011)</a></li></a>
          </li>
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/sra/docs/sra-aws-download/" class="" target="_none">Access SRA data using Amazon Web Services (AWS)</a> by <a href="https://www.ncbi.nlm.nih.gov/sra/" target="_none">NCBI SRA</a></li></a>
          </li>
        </ul>
        <p><a href="/ncbi-sra/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="osm" class="dataset">
          <h3><a href="/osm/">OpenStreetMap on AWS</a></h3>
          <p><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">mapping</span><span class="label label-info tag link-tag">osm</span><span class="label label-info-sustainability tag link-tag">sustainability</span></p>
          <p>OSM is a free, editable map of the world, created and maintained by volunteers. Regular OSM data archives are made available in Amazon S3.</p>
          <p><a href="/osm/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://aws.amazon.com/blogs/big-data/querying-openstreetmap-with-amazon-athena/" class="aws-link" target="_none">Querying OpenStreetMap with Amazon Athena</a> by <a href="https://github.com/mojodna" target="_none">Seth Fitzsimmons</a></li></a>
          </li>
          <li>
            <a href="https://aws.amazon.com/blogs/publicsector/develop-and-extract-value-from-open-data/" class="aws-link" target="_none">Develop and Extract Value from Open Data</a> by Daniel Bernao</li></a>
          </li>
          <li>
            <a href="https://github.com/interline-io/planetutils" class="" target="_none">PlanetUtils (GitHub): Scripts and a Docker container to maintain your own OpenStreetMap planet</a> by <a href="https://www.interline.io/" target="_none">Interline Technologies</a></li></a>
          </li>
          <li>
            <a href="https://github.com/developmentseed/mrworldwide/" class="" target="_none">OSM+Athena (GitHub)</a> by <a href="https://developmentseed.org/" target="_none">Development Seed</a></li></a>
          </li>
        </ul>
        <p><a href="/osm/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="refgenie" class="dataset">
          <h3><a href="/refgenie/">Refgenie reference genome assets</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">infrastructure</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">single-cell transcriptomics</span><span class="label label-info tag link-tag">transcriptomics</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>Pre-built refgenie reference genome data assets used for aligning and analyzing DNA sequence data.</p>
          <p><a href="/refgenie/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1093/gigascience/giz149" class="" target="_none">Refgenie: a reference genome resource manager</a> by <a href="http://www.databio.org" target="_none">Michał Stolarczyk, Vincent P Reuter, Jason P Smith, Neal E Magee, Nathan C Sheffield</a></li></a>
          </li>
          <li>
            <a href="http://peppro.databio.org" class="" target="_none">Pipeline for PRO-seq data analysis</a> by <a href="http://databio.org" target="_none">Jason Smith and Nathan Sheffield</a></li></a>
          </li>
          <li>
            <a href="http://pepatac.databio.org" class="" target="_none">Pipeline for ATAC-seq data analysis</a> by <a href="http://databio.org" target="_none">Jason Smith and Nathan Sheffield</a></li></a>
          </li>
          <li>
            <a href="http://refgenie.databio.org/en/latest/tutorial/" class="" target="_none">Basic Refgenie tutorial</a> by <a href="http://www.databio.org" target="_none">Nathan Sheffield</a></li></a>
          </li>
        </ul>
        <p><a href="/refgenie/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="sentinel-1" class="dataset">
          <h3><a href="/sentinel-1/">Sentinel-1</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info-sustainability tag link-tag">sustainability</span></p>
          <p><a href="https://sentinel.esa.int/web/sentinel/missions/sentinel-1">Sentinel-1</a> is a pair of European radar imaging (SAR) satellites launched in 2014 and 2016. Its 6 days revisit cycle and ability to observe through clouds makes it perfect for sea and land monitoring, emergency response due to environmental disasters, and economic applications. GRD data is available globally since January 2017.</p>
          <p><a href="/sentinel-1/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="http://www.sentinel-hub.com/apps/wms" class="" target="_none">Sentinel Hub WMS/WMTS/WCS Service by Sinergise</a> by <a href="http://www.sinergise.com/" target="_none">Sinergise</a></li></a>
          </li>
          <li>
            <a href="https://apps.sentinel-hub.com/sentinel-playground/?baseWmsUrl&#x3D;services.sentinel-hub.com&amp;instanceID&#x3D;a7c04225-b861-452d-ac89-7765352cd855&amp;source&#x3D;S2&amp;lat&#x3D;40.98676619345964&amp;lng&#x3D;28.89301300048828&amp;zoom&#x3D;13&amp;preset&#x3D;IW_VV&amp;layers&#x3D;B04,B03,B02&amp;maxcc&#x3D;20&amp;gain&#x3D;1.0&amp;gamma&#x3D;1.0&amp;time&#x3D;2015-01-01%7C2018-01-16&amp;atmFilter&#x3D;&amp;showDates&#x3D;false" class="" target="_none">Sentinel Playground</a> by <a href="http://www.sinergise.com/" target="_none">Sinergise</a></li></a>
          </li>
          <li>
            <a href="https://eos.com/landviewer/" class="" target="_none">EOS Land Viewer</a> by <a href="https://eos.com/" target="_none">Earth Observing System</a></li></a>
          </li>
          <li>
            <a href="http://apps.sentinel-hub.com/eo-browser/" class="" target="_none">EO Browser</a> by <a href="http://www.sinergise.com/" target="_none">Sinergise</a></li></a>
          </li>
        </ul>
        <p><a href="/sentinel-1/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="sentinel-3" class="dataset">
          <h3><a href="/sentinel-3/">Sentinel-3</a></h3>
          <p><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">land</span><span class="label label-info tag link-tag">oceans</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info-sustainability tag link-tag">sustainability</span></p>
          <p>This data set consists of observations from the Sentinel-3 satellite of the European Commission’s Copernicus Earth Observation Programme. Sentinel-3 is a polar orbiting satellite that completes 14 orbits of the Earth a day. It carries the Ocean and Land Colour Instrument (OLCI) for medium resolution marine and terrestrial optical measurements, the Sea and Land Surface Termperature Radiometer (SLSTR), the SAR Radar Altimeter (SRAL), the MicroWave Radiometer (MWR) and the Precise Orbit Determination (POD) instruments. The satellite was launched in 2016 and entered routine operational phase in 20...</p>
          <p><a href="/sentinel-3/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://sentinel.esa.int/web/sentinel/user-guides" class="" target="_none">Sentinel-3 Document Library</a> by <a href="https://www.esa.int/" target="_none">European Space Agency</a></li></a>
          </li>
          <li>
            <a href="https://github.com/Sentinel-5P/data-on-s3/blob/master/notebooks/Sentinel3_Tutorial.ipynb" class="" target="_none">Accessing Sentinel-3 Data on S3 by MEEO</a> by <a href="http://www.meeo.it/" target="_none">Meteorological Envionmental Earth Observation</a></li></a>
          </li>
          <li>
            <a href="https://meeo-s3.s3.amazonaws.com/index.html#/?t&#x3D;catalogs" class="" target="_none">Catalogue of data set</a> by <a href="https://www.meeo.it/" target="_none">Meteorological Envionmental Earth Observation</a></li></a>
          </li>
          <li>
            <a href="https://step.esa.int/main/toolboxes/sentinel-3-toolbox/" class="" target="_none">Sentinel-3 Toolbox</a> by <a href="https://www.esa.int/" target="_none">European Space Agency</a></li></a>
          </li>
        </ul>
        <p><a href="/sentinel-3/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="sentinel5p" class="dataset">
          <h3><a href="/sentinel5p/">Sentinel-5P Level 2</a></h3>
          <p><span class="label label-info tag link-tag">air quality</span><span class="label label-info tag link-tag">atmosphere</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info-sustainability tag link-tag">sustainability</span></p>
          <p>This data set consists of observations from the Sentinel-5 Precursor (Sentinel-5P) satellite of the European Commission’s Copernicus Earth Observation Programme. Sentinel-5P is a polar orbiting satellite that completes 14 orbits of the Earth a day. It carries the TROPOspheric Monitoring Instrument (TROPOMI) which is a spectrometer that senses ultraviolet (UV), visible (VIS), near (NIR) and short wave infrared (SWIR) to monitor ozone, methane, formaldehyde, aerosol, carbon monoxide, nitrogen dioxide and sulphur dioxide in the atmosphere. The satellite was launched in October 2017 and entered ro...</p>
          <p><a href="/sentinel5p/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://atmospherictoolbox.org/" class="" target="_none">The Atmospheric Toolbox</a> by <a href="https://www.esa.int/" target="_none">European Space Agency</a></li></a>
          </li>
          <li>
            <a href="https://meeo-s5p.s3.amazonaws.com/index.html#/?t&#x3D;catalogs" class="" target="_none">Catalogue of data set</a> by <a href="https://www.meeo.it/" target="_none">Meteorological Envionmental Earth Observation</a></li></a>
          </li>
          <li>
            <a href="https://github.com/Sentinel-5P/data-on-s3/blob/master/notebooks/Sentinel5P_Tutorial.ipynb" class="" target="_none">Accessing Sentinel-5P Data on S3 by MEEO</a> by <a href="http://www.meeo.it/" target="_none">Meteorological Envionmental Earth Observation</a></li></a>
          </li>
          <li>
            <a href="https://sentinel.esa.int/web/sentinel/user-guides/sentinel-5p-tropomi/document-library" class="" target="_none">Sentinel-5P TROPOMI Document Library</a> by <a href="https://www.esa.int/" target="_none">European Space Agency</a></li></a>
          </li>
        </ul>
        <p><a href="/sentinel5p/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="broad-pan-ukb" class="dataset">
          <h3><a href="/broad-pan-ukb/">UK Biobank Pan-Ancestry Summary Statistics</a></h3>
          <p><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genome wide association study</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">population genetics</span></p>
          <p>A multi-ancestry analysis of 7,221 phenotypes using a generalized mixed model association testing framework, spanning 16,119 genome-wide association studies. We provide standard meta-analysis across all populations and with a leave-one-population-out approach for each trait. The data are provided in tsv format (per phenotype) and Hail MatrixTable (all phenotypes and variants). Metadata is provided in phenotype and variant manifests.</p>
          <p><a href="/broad-pan-ukb/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://hail.is" class="" target="_none">Hail</a> by <a href="https://hail.is/" target="_none">Hail Team</a></li></a>
          </li>
          <li>
            <a href="https://aws.amazon.com/quickstart/architecture/hail/" class="aws-link" target="_none">Hail on AWS Quick Start</a> by Amazon Web Services and PrivoIT</li></a>
          </li>
          <li>
            <a href="https://pan.ukbb.broadinstitute.org/" class="" target="_none">Pan-ancestry genetic analysis of the UK Biobank</a> by Pan UKBB Team</li></a>
          </li>
          <li>
            <a href="https://hail.is/docs/0.2/tutorials-landing.html" class="" target="_none">Hail Tutorials</a> by <a href="https://hail.is/" target="_none">Hail Team</a></li></a>
          </li>
        </ul>
        <p><a href="/broad-pan-ukb/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="ycb-benchmarks" class="dataset">
          <h3><a href="/ycb-benchmarks/">Yale-CMU-Berkeley (YCB) Object and Model Set</a></h3>
          <p><span class="label label-info tag link-tag">robotics</span></p>
          <p>This project primarily aims to facilitate performance benchmarking in robotics research. The dataset provides mesh models, RGB, RGB-D and point cloud images of over 80 objects. The physical objects are also available via the <a href="http://www.ycbbenchmarks.com/">YCB benchmarking project</a>. The data are collected by two state of the art systems: UC Berkley&#39;s scanning rig and the Google scanner. The UC Berkley&#39;s scanning rig data provide meshes generated with Poisson reconstruction, meshes generated with volumetric range image integration, textured versions of both meshes, Kinbody files for using the meshes with OpenRAVE, 600 ...</p>
          <p><a href="/ycb-benchmarks/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://ieeexplore.ieee.org/document/7254318" class="" target="_none">Benchmarking in Manipulation Research: Using the Yale-CMU-Berkeley Object and Model Set</a> by <a href="https://www.wpi.edu/people/faculty/bcalli" target="_none">Berk Calli, Aaron Walsman, Arjun Singh, Siddhartha Srinivasa, Pieter Abbeel, Aaron M Dollar</a></li></a>
          </li>
          <li>
            <a href="https://ieeexplore.ieee.org/abstract/document/7989594" class="" target="_none">Pre-touch sensing for sequential manipulation</a> by <a href="https://sensor.cs.washington.edu/jrs.html" target="_none">Boling Yang, Patrick Lancaster, Joshua R. Smith</a></li></a>
          </li>
          <li>
            <a href="https://ieeexplore.ieee.org/abstract/document/8460950" class="" target="_none">Label Fusion: A Pipeline for Generating Ground Truth Labels for Real RGBD Data of Cluttered Scenes</a> by <a href="https://groups.csail.mit.edu/locomotion/russt.html" target="_none">Pat Marion, Peter R. Florence, Lucas Manuelli, Russ Tedrake</a></li></a>
          </li>
          <li>
            <a href="https://ieeexplore.ieee.org/abstract/document/8304743" class="" target="_none">The Closure Signature: A Functional Approach to Model Underactuated Compliant Robotic Hands</a> by <a href="http://www.dii.unisi.it/~domenico/" target="_none">Maria Pozzi, Gionata Salvietti, João Bimbo, Monica Malvezzi, Domenico Prattichizzo</a></li></a>
          </li>
        </ul>
        <p><a href="/ycb-benchmarks/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="allen-mouse-brain-atlas" class="dataset">
          <h3><a href="/allen-mouse-brain-atlas/">Allen Mouse Brain Atlas</a></h3>
          <p><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">gene expression</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">image processing</span><span class="label label-info tag link-tag">imaging</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">Mus musculus</span><span class="label label-info tag link-tag">neurobiology</span><span class="label label-info tag link-tag">transcriptomics</span></p>
          <p>The Allen Mouse Brain Atlas is a genome-scale collection of cellular resolution gene expression profiles using in situ hybridization (ISH). Highly methodical data production methods and comprehensive anatomical coverage via dense, uniformly spaced sampling facilitate data consistency and comparability across &gt;20,000 genes. The use of an inbred mouse strain with minimal animal-to-animal variance allows one to treat the brain essentially as a complex but highly reproducible three-dimensional tissue array. The entire Allen Mouse Brain Atlas dataset and associated tools are available through an...</p>
          <p><a href="/allen-mouse-brain-atlas/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/AllenInstitute/open_dataset_tools/blob/master/Visualizing_Images_from_Allen_Mouse_Brain_Atlas.ipynb" class="" target="_none">Visualizing Images from the Allen Mouse Brain Atlas</a> by <a href="www.alleninstitute.org" target="_none">Allen Institute for Brain Science</a></li></a>
          </li>
          <li>
            <a href="http://www.nature.com/articles/nature05453" class="" target="_none">Genome-wide atlas of gene expression in the adult mouse brain</a> by <a href="www.alleninstitute.org" target="_none">Ed Lein, et al.</a></li></a>
          </li>
          <li>
            <a href="https://mouse.brain-map.org" class="" target="_none">Allen Mouse Brain Atlas</a> by <a href="www.alleninstitute.org" target="_none">Allen Institute for Brain Science</a></li></a>
          </li>
        </ul>
        <p><a href="/allen-mouse-brain-atlas/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="ncbi-blast-databases" class="dataset">
          <h3><a href="/ncbi-blast-databases/">Basic Local Alignment Sequences Tool (BLAST) Databases</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">health</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">protein</span><span class="label label-info tag link-tag">reference index</span><span class="label label-info tag link-tag">transcriptomics</span></p>
          <p>A centralized repository of pre-formatted BLAST databases created by the National Center for Biotechnology Information (NCBI).</p>
          <p><a href="/ncbi-blast-databases/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://pubmed.ncbi.nlm.nih.gov/9254694/" class="" target="_none">Gapped BLAST and PSI-BLAST: A New Generation of Protein Database Search Programs</a> by S F Altschul, T L Madden, A A Schäffer, J Zhang, Z Zhang, W Miller, D J Lipman</li></a>
          </li>
          <li>
            <a href="https://pubmed.ncbi.nlm.nih.gov/20003500" class="" target="_none">BLAST+: Architecture and Applications</a> by Christiam Camacho  1 , George Coulouris, Vahram Avagyan, Ning Ma, Jason Papadopoulos, Kevin Bealer, Thomas L Madden</li></a>
          </li>
          <li>
            <a href="https://github.com/ncbi/blast_plus_docs" class="" target="_none">BLAST+ Docker</a> by NCBI BLAST</li></a>
          </li>
        </ul>
        <p><a href="/ncbi-blast-databases/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="beataml" class="dataset">
          <h3><a href="/beataml/">Beat Acute Myeloid Leukemia (AML) 1.0</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">Homo sapiens</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">STRIDES</span></p>
          <p>Beat AML 1.0 is a collaborative research program involving 11 academic medical centers who worked
collectively to better understand  drugs and drug combinations that should be prioritized for
further development within clinical and/or molecular subsets of acute myeloid leukemia (AML)
patients. Beat AML 1.0 provides the largest-to-date dataset on primary acute myeloid leukemia
samples offering genomic, clinical, and drug response.This dataset contains open Clinical Supplement and RNA-Seq Gene Expression Quantification data.This dataset also contains controlled Whole Exome Sequencing (WXS) and R...</p>
          <p><a href="/beataml/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://portal.gdc.cancer.gov/projects/" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></li></a>
          </li>
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/30333627" class="" target="_none">Functional Genomic Landscape of Acute Myeloid Leukemia</a> by Jeffrey W. Tyner, Cristina E. Tognon, Dan Bottomly et al.</li></a>
          </li>
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/30651561" class="" target="_none">Clinical resistance to crenolanib in acute myeloid leukemia due to diverse molecular
mechanisms
</a> by Zhang H, Savage S, Schultz AR, Bottomly D, White L, Segerdell E, et al.</li></a>
          </li>
        </ul>
        <p><a href="/beataml/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="talend-covid19" class="dataset">
          <h3><a href="/talend-covid19/">COVID-19 Harmonized Data</a></h3>
          <p><span class="label label-info tag link-tag">coronavirus</span><span class="label label-info tag link-tag">COVID-19</span><span class="label label-info tag link-tag">life sciences</span></p>
          <p>A harmonized collection of the core data pertaining to COVID-19 reported cases by geography, in a format prepared for analysis</p>
          <p><a href="/talend-covid19/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.stitchdata.com/integrations/covid-19/" class="" target="_none">Stitch COVID-19 Integration</a> by <a href="https://github.com/jeffhuth-bytecode" target="_none">Jeff Huth</a></li></a>
          </li>
          <li>
            <a href="https://www.talend.com/blog/2020/04/05/talend-joins-fight-against-covid-19-unlocking-data-for-health-researchers/" class="" target="_none">How Talend is joining the fight against COVID-19</a> by <a href="https://www.linkedin.com/in/thomasbennettw/" target="_none">Thomas Bennett</a></li></a>
          </li>
          <li>
            <a href="https://github.com/singer-io/tap-covid-19" class="" target="_none">Tap COVID-19 Python</a> by <a href="https://github.com/jeffhuth-bytecode" target="_none">Jeff Huth</a></li></a>
          </li>
        </ul>
        <p><a href="/talend-covid19/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="ctsp-dlbcl" class="dataset">
          <h3><a href="/ctsp-dlbcl/">Clinical Trial Sequencing Project - Diffuse Large B-Cell Lymphoma</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">STRIDES</span><span class="label label-info tag link-tag">transcriptomics</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>The goal of the project is to identify recurrent genetic alterations (mutations, deletions,
amplifications, rearrangements) and/or gene expression signatures. National Cancer Institute (NCI)
utilized whole genome sequencing and/or whole exome sequencing in conjunction with transcriptome
sequencing. The samples were processed and submitted for genomic characterization using pipelines
and procedures established within The Cancer Genome Analysis (TCGA) project.</p>
          <p><a href="/ctsp-dlbcl/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed?cmd&#x3D;DetailsSearch&amp;term&#x3D;29925955[PMID]" class="" target="_none">A multiprotein supercomplex controlling oncogenic signalling in lymphoma</a> by Phelan JD, Young RM, Webster DE, Roulland S, Wright GW, Kasbekar M, Shaffer AL 3rd,
Ceribelli M, Wang JQ, Schmitz R, Nakagawa M, Bachy E, Huang DW, Ji Y, Chen L, Yang Y, Zhao
H, Yu X, Xu W, Palisoc MM, Valadez RR, Davies-Hill T, Wilson WH, Chan WC, Jaffe ES, Gascoyne
RD, Campo E, Rosenwald A, Ott G, Delabie J, Rimsza LM, Rodriguez FJ, Estephan F, Holdhoff M,
Kruhlak MJ, Hewitt SM, Thomas CJ, Pittaluga S, Oellerich T, Staudt LM
</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1056/NEJMoa1801445" class="" target="_none">Genetics and Pathogenesis of Diffuse Large B Cell Lymphoma</a> by Roland Schmitz, Ph.D., George W. Wright, Ph.D., Da Wei Huang, M.D., Calvin A.  Johnson,
Ph.D., James D. Phelan, Ph.D., James Q. Wang, Ph.D., Sandrine Roulland, Ph.D., Monica
Kasbekar, Ph.D., Ryan M. Young, Ph.D., Arthur L. Shaffer, Ph.D., Daniel J. Hodson, M.D.,
Ph.D., Wenming Xiao, Ph.D., et al.
</li></a>
          </li>
          <li>
            <a href="https://portal.gdc.cancer.gov/projects/CTSP-DLBCL1" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></li></a>
          </li>
        </ul>
        <p><a href="/ctsp-dlbcl/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="deutsche-boerse-pds" class="dataset">
          <h3><a href="/deutsche-boerse-pds/">Deutsche Börse Public Dataset</a></h3>
          <p><span class="label label-info tag link-tag">financial markets</span><span class="label label-info tag link-tag">market data</span><span class="label label-info tag link-tag">trading</span></p>
          <p>The Deutsche Börse Public Data Set consists of trade data aggregated to one minute intervals from the Eurex and Xetra trading systems. It provides the initial price, lowest price, highest price, final price and volume for every minute of the trading day, and for every tradeable security.  If you need higher resolution data, including untraded price movements, please refer to our historical market data product <a href="http://www.eurexchange.com/exchange-en/market-data/historical-data">here</a>.  Also, be sure to check out our <a href="https://console.developer.deutsche-boerse.com/">developer&#39;s portal</a>.</p>
          <p><a href="/deutsche-boerse-pds/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="http://live.thermobook.net/dataStreamingWeb" class="" target="_none">Streaming XETRA Data Using Apache Spark</a> by <a href="http://www.thermobook.net" target="_none">Thermobook</a></li></a>
          </li>
          <li>
            <a href="https://github.com/Originate/dbg-pds-tensorflow-demo" class="" target="_none">Stock Price Movement Prediction Using The Deutsche Börse Public Dataset &amp; Machine Learning</a> by <a href="https://www.originate.com" target="_none">Originate</a></li></a>
          </li>
          <li>
            <a href="https://aws.amazon.com/blogs/big-data/10-visualizations-to-try-in-amazon-quicksight-with-sample-data/" class="aws-link" target="_none">10 visualizations to try in Amazon QuickSight with sample data</a> by <a href="https://aws.amazon.com/blogs/big-data/" target="_none">AWS Big Data Blog</a></li></a>
          </li>
        </ul>
        <p><a href="/deutsche-boerse-pds/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="dandiarchive" class="dataset">
          <h3><a href="/dandiarchive/">Distributed Archives for Neurophysiology Data Integration (DANDI)</a></h3>
          <p><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">cell imaging</span><span class="label label-info tag link-tag">electrophysiology</span><span class="label label-info tag link-tag">infrastructure</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">neuroimaging</span><span class="label label-info tag link-tag">neurophysiology</span><span class="label label-info tag link-tag">neuroscience</span></p>
          <p>DANDI is a public archive of neurophysiology datasets, including raw and processed data,  and associated software containers. Datasets are shared according to a Creative Commons  CC0 or CC-BY licenses. The data archive provides a broad range of cellular neurophysiology data.  This includes electrode and optical recordings, and associated imaging data using a  set of community standards: <a href="https://www.nwb.org/nwb-neurophysiology/">NWB:N - NWB:Neurophysiology</a>, <a href="https://bids.neuroimaging.io/">BIDS - Brain Imaging Data Structure</a>, and  <a href="htt...</p>
          <p><a href="/dandiarchive/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/dandi/dandi-cli" class="" target="_none">DANDI Shell Interface</a> by <a href="https://dandiarchive.org/" target="_none">DANDI Project</a></li></a>
          </li>
          <li>
            <a href="https://hub.dandiarchive.org/" class="" target="_none">DANDI JupyterHub Interface</a> by <a href="https://dandiarchive.org/" target="_none">DANDI Project</a></li></a>
          </li>
          <li>
            <a href="https://dandiarchive.org/" class="" target="_none">DANDI Web Interface</a> by <a href="https://dandiarchive.org/" target="_none">DANDI Project</a></li></a>
          </li>
        </ul>
        <p><a href="/dandiarchive/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="fmi-radar" class="dataset">
          <h3><a href="/fmi-radar/">Finnish Meteorological Institute Weather Radar Data</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info-sustainability tag link-tag">sustainability</span><span class="label label-info tag link-tag">weather</span></p>
          <p>The up-to-date weather radar from the FMI radar network is available as Open Data. The data contain both single radar data along with composites over Finland in GeoTIFF and HDF5-formats. Available composite parameters consist of radar reflectivity (DBZ), rainfall intensity (RR), and precipitation accumulation of 1, 12, and 24 hours. Single radar parameters consist of radar reflectivity (DBZ), radial velocity (VRAD), rain classification (HCLASS), and Cloud top height (ETOP 20). Raw volume data from singe radars are also provided in HDF5 format with ODIM 2.3 conventions. Radar data becomes avail...</p>
          <p><a href="/fmi-radar/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/fmidev/opendata-resources/tree/master/examples/qgis/radar-composite" class="" target="_none">Handling data with QGIS</a> by Markus Peura</li></a>
          </li>
          <li>
            <a href="https://github.com/fmidev/opendata-resources/blob/master/examples/python/radar-rhi-from-hdf5.ipynb" class="" target="_none">Processing HDF5 data with python</a> by Roope Tervo</li></a>
          </li>
          <li>
            <a href="https://github.com/fmidev/opendata-resources/blob/master/examples/python/Accumulated_prec_amount_from_radar_GeoTiff.ipynb" class="" target="_none">Processing GeoTIFF data with python</a> by Roope Tervo</li></a>
          </li>
        </ul>
        <p><a href="/fmi-radar/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="fm-ad" class="dataset">
          <h3><a href="/fm-ad/">Foundation Medicine Adult Cancer Clinical Dataset (FM-AD)</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genomic</span></p>
          <p>The Foundation Medicine Adult Cancer Clinical Dataset (FM-AD) is a study conducted by Foundation
Medicine Inc (FMI). Genomic profiling data for approximately 18,000 adult patients with a diverse
array of cancers was generated using FoundationeOne, FMI&#39;s commercially available, comprehensive
genomic profiling assay. This dataset contains open Clinical and Biospecimen data.</p>
          <p><a href="/fm-ad/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed?cmd&#x3D;DetailsSearch&amp;term&#x3D;22981675[PMID]" class="" target="_none">Targeted next-generation sequencing of advanced prostate cancer identifies potential
therapeutic targets and disease heterogeneity.
</a> by Beltran H, Yelensky R, Frampton GM, Park K, Downing SR, MacDonald TY, Jarosz M, Lipson D,
Tagawa ST, Nanus DM, Stephens PJ, Mosquera JM, Cronin MT, Rubin MA
</li></a>
          </li>
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed?cmd&#x3D;DetailsSearch&amp;term&#x3D;28235761[PMID]" class="" target="_none">High-Throughput Genomic Profiling of Adult Solid Tumors Reveals Novel Insights into Cancer
Pathogenesis
</a> by Ryan J. Hartmaier, Lee A. Albacker, Juliann Chmielecki, Mark Bailey, Jie He, Michael E.
Goldberg, Shakti Ramkissoon, James Suh, Julia A. Elvin, Samuel Chiacchia, Garrett M.
Frampton, Jeffrey S. Ross, Vincent Miller, Philip J. Stephens and Doron Lipson
</li></a>
          </li>
          <li>
            <a href="https://portal.gdc.cancer.gov/projects/FM-AD" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></li></a>
          </li>
        </ul>
        <p><a href="/fm-ad/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="noaa-himawari" class="dataset">
          <h3><a href="/noaa-himawari/">JMA Himawari-8</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info-sustainability tag link-tag">sustainability</span><span class="label label-info tag link-tag">weather</span></p>
          <p>Himawari-8, stationed at 140E, owned and operated by the Japan Meteorological Agency (JMA), is a geostationary meteorological satellite, with Himawari-9  as on-orbit back-up, that provides constant and uniform coverage of east Asia, and the west and central Pacific regions from around 35,800 km above the equator with an orbit corresponding to the period of the earth’s rotation. This allows JMA weather offices to perform uninterrupted observation of environmental phenomena such as typhoons, volcanoes, and general weather systems. For questions regarding Himawari-8 imagery specifications, visit ...</p>
          <p><a href="/noaa-himawari/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://docs.opendata.aws/noaa-himawari/2020July08_NOAA_Himawari.pdf" class="aws-link" target="_none">Himawari-8 Advanced Himawari Imager Data on AWS (pdf file)</a> by NOAA NESDIS</li></a>
          </li>
          <li>
            <a href="https://docs.opendata.aws/noaa-himawari/2020July08_Amazon_Himawari.pdf" class="aws-link" target="_none">Himawari-8 on AWS (pdf file)</a> by ASDI</li></a>
          </li>
          <li>
            <a href="https://docs.opendata.aws/noaa-himawari/2020July07_JMA_Himawari.pdf" class="aws-link" target="_none">Introduction of Himawari-8/9 (pdf file)</a> by JMA</li></a>
          </li>
        </ul>
        <p><a href="/noaa-himawari/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="cotonoha-dic" class="dataset">
          <h3><a href="/cotonoha-dic/">Japanese Tokenizer Dictionaries</a></h3>
          <p><span class="label label-info tag link-tag">csv</span><span class="label label-info tag link-tag">japanese</span><span class="label label-info tag link-tag">natural language processing</span></p>
          <p>Japanese Tokenizer Dictionaries for use with MeCab.</p>
          <p><a href="/cotonoha-dic/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.dampfkraft.com/nlp/how-to-tokenize-japanese.html" class="" target="_none">How to Tokenize Japanese in Python</a> by <a href="https://dampfkraft.com" target="_none">Paul O&#x27;Leary McCann</a></li></a>
          </li>
          <li>
            <a href="https://github.com/polm/unidic-py" class="" target="_none">unidic-py</a> by <a href="https://cotonoha.io" target="_none">Paul O&#x27;Leary McCann</a></li></a>
          </li>
          <li>
            <a href="https://github.com/polm/fugashi-sagemaker-demo/blob/master/fugashi%20wordcount.ipynb" class="" target="_none">Fugashi Word Count Tutorial</a> by <a href="https://cotonoha.io" target="_none">Paul O&#x27;Leary McCann</a></li></a>
          </li>
        </ul>
        <p><a href="/cotonoha-dic/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="msd" class="dataset">
          <h3><a href="/msd/">Medical Segmentation Decathlon</a></h3>
          <p><span class="label label-info tag link-tag">computed tomography</span><span class="label label-info tag link-tag">health</span><span class="label label-info tag link-tag">imaging</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">magnetic resonance imaging</span><span class="label label-info tag link-tag">medicine</span><span class="label label-info tag link-tag">nifti</span><span class="label label-info tag link-tag">segmentation</span></p>
          <p>With recent advances in machine learning, semantic segmentation algorithms are becoming increasingly general purpose and translatable to unseen tasks. Many key algorithmic advances in the field of medical imaging are commonly validated on a small number of tasks, limiting our understanding of the generalisability of the proposed contributions. A model which works out-of-the-box on many tasks, in the spirit of AutoML, would have a tremendous impact on healthcare. The field of medical imaging is also missing a fully open source and comprehensive benchmark for general purpose algorithmic validati...</p>
          <p><a href="/msd/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://arxiv.org/pdf/1902.09063.pdf" class="" target="_none">A large annotated medical image dataset for the development and evaluation of segmentation algorithms</a> by Simpson A. L., Antonelli M., Bakas S., Bilello M., Farahana K., van Ginneken B., et al</li></a>
          </li>
          <li>
            <a href="www.monai.io" class="" target="_none">MONAI: Getting Started</a> by <a href="https://monai.io/start.html" target="_none">MONAI Development Team</a></li></a>
          </li>
          <li>
            <a href="https://github.com/Project-MONAI/tutorials" class="" target="_none">Pytorch-Integrated MSD Data Loader</a> by <a href="https://github.com/Project-MONAI/MONAI" target="_none">MONAI Development Team</a></li></a>
          </li>
        </ul>
        <p><a href="/msd/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="nasanex" class="dataset">
          <h3><a href="/nasanex/">NASA NEX</a></h3>
          <p><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">natural resource</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info-sustainability tag link-tag">sustainability</span></p>
          <p>A collection of Earth science datasets maintained by NASA, including climate change projections and satellite images of the Earth&#39;s surface.</p>
          <p><a href="/nasanex/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://climate.azavea.com" class="" target="_none">Azavea Climate API</a> by <a href="https://www.azavea.com/" target="_none">Azavea</a></li></a>
          </li>
          <li>
            <a href="https://www.kdd.org/kdd2020/accepted-papers/view/climate-downscaling-using-ynet-a-deep-convolutional-network-with-skip-conne" class="" target="_none">Climate Downscaling Using YNet: a Deep Convolutional Network with Skip Connections and Fusion</a> by Yumin Liu, Auroop Ganguly, and Jennifer Dy</li></a>
          </li>
          <li>
            <a href="http://cloud-gc.readthedocs.io/en/stable/chapter02_beginner-tutorial/use-s3.html#access-nasa-nex-data-in-s3-optional-but-recommended" class="" target="_none">Accessing and plotting NASA-NEX data, from GEOSChem-on-cloud tutorial.</a> by <a href="https://github.com/JiaweiZhuang" target="_none">Jiawei Zhuang</a></li></a>
          </li>
        </ul>
        <p><a href="/nasanex/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="noaa-gefs-reforecast" class="dataset">
          <h3><a href="/noaa-gefs-reforecast/">NOAA Global Ensemble Forecast System (GEFS) Re-forecast</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info-sustainability tag link-tag">sustainability</span><span class="label label-info tag link-tag">weather</span></p>
          <p>NOAA has generated a multi-decadal reanalysis and reforecast data set to accompany the next-generation version of its ensemble prediction system, the Global Ensemble Forecast System, version 12 (GEFSv12). Accompanying the real-time forecasts are “reforecasts” of the weather, that is, retrospective forecasts spanning the period 2000-2019. These reforecasts are not as numerous as the real-time data; they were generated only once per day, from 00 UTC initial conditions, and only 5 members were provided, with the following exception. Once weekly, an 11-member reforecast was generated, and these ex...</p>
          <p><a href="/noaa-gefs-reforecast/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://mogismog.github.io/retro-diss-part-3.html#retro-diss-part-3" class="" target="_none">Retrospective Dissertation - Part 3: Reforecast Data</a> by <a href="https://github.com/mogismog" target="_none">Francisco Alvarez</a></li></a>
          </li>
          <li>
            <a href="https://drive.google.com/file/d/1lCIB3qz9S1nu2qAaetnLq0N5ReyTXuHd/view" class="" target="_none">The GEFS v12 and its reanalyses and reforecasts (Slides)</a> by Tom Hamill</li></a>
          </li>
          <li>
            <a href="https://github.com/vgensini/gefs_v12_example/blob/master/GEFS_v12_eample.ipynb" class="" target="_none">Example using GEFS Reforecast Data from AWS</a> by <a href="https://atlas.niu.edu" target="_none">Victor Gensini</a></li></a>
          </li>
        </ul>
        <p><a href="/noaa-gefs-reforecast/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="noaa-ghcn" class="dataset">
          <h3><a href="/noaa-ghcn/">NOAA Global Historical Climatology Network Daily (GHCN-D)</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info-sustainability tag link-tag">sustainability</span><span class="label label-info tag link-tag">weather</span></p>
          <p>Global Historical Climatology Network - Daily is a dataset from NOAA that contains daily observations over global land areas. It contains station-based measurements from land-based stations worldwide, about two thirds of which are for precipitation measurement only. Other meteorological elements include, but are not limited to, daily maximum and minimum temperature, temperature at the time of observation, snowfall and snow depth. It is a composite of climate records from numerous sources that were merged together and subjected to a common suite of quality assurance reviews.  Some data are more...</p>
          <p><a href="/noaa-ghcn/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://medium.com/swlh/explore-visualize-200-years-of-global-temperature-using-apache-spark-bigquery-and-google-data-699ed8b3c67?sk&#x3D;3721ca3220a75af80293ac3e4bf3f4d1" class="" target="_none">Explore &amp; Visualize 200+ Years of Global Temperature</a> by Kapil Sreedharan</li></a>
          </li>
          <li>
            <a href="https://aws.amazon.com/blogs/industries/calculating-growing-degree-days-using-aws-registry-of-open-data/" class="aws-link" target="_none">Calculating growing degree days using AWS Registry of Open Data</a> by Karen Hildebrand and Zac Flamig</li></a>
          </li>
          <li>
            <a href="https://aws.amazon.com/blogs/big-data/visualize-over-200-years-of-global-climate-data-using-amazon-athena-and-amazon-quicksight/" class="aws-link" target="_none">Visualize over 200 years of global climate data using Amazon Athena and Amazon QuickSight</a> by Conor Delaney</li></a>
          </li>
        </ul>
        <p><a href="/noaa-ghcn/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="nrel-pds-nsrdb" class="dataset">
          <h3><a href="/nrel-pds-nsrdb/">NREL National Solar Radiation Database</a></h3>
          <p><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">energy</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">solar</span><span class="label label-info-sustainability tag link-tag">sustainability</span></p>
          <p>Released to the public as part of the Department of Energy&#39;s Open Energy Data Initiative,
the <a href="https://nsrdb.nrel.gov/">National Solar Radiation Database (NSRDB)</a> is
a serially complete collection of hourly and half-hourly values of the three
most common measurements of solar radiation – global horizontal, direct
normal, and diffuse horizontal irradiance — and meteorological data. These
data have been collected at a sufficient number of locations and temporal and
spatial scales to accurately represent regional solar radiation climates.</p>
          <p><a href="/nrel-pds-nsrdb/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/NREL/hsds-examples" class="" target="_none">HSDS Examples</a> by Caleb Phillips, Caroline Draxl, John Readey, Jordan Perr-Sauer, Michael Rossol</li></a>
          </li>
          <li>
            <a href="https://www.sciencedirect.com/science/article/pii/S136403211830087X" class="" target="_none">The National Solar Radiation Data Base (NSRDB)</a> by Manajit Sengupta, Yu Xe, Anthony Lopez, Aron Habte, Galen Maclaurin, James Shelby</li></a>
          </li>
          <li>
            <a href="https://nsrdb.nrel.gov/nsrdb-viewer" class="" target="_none">NSRDB Viewer</a> by Manajit Sengupta, Yu Xe, Anthony Lopez, Aron Habte, Galen Maclaurin, James Shelby, Paul Edwards</li></a>
          </li>
        </ul>
        <p><a href="/nrel-pds-nsrdb/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="nsw-herbarium" class="dataset">
          <h3><a href="/nsw-herbarium/">National Herbarium of NSW</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">biodiversity</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">digital preservation</span><span class="label label-info tag link-tag">ecosystems</span><span class="label label-info tag link-tag">environmental</span></p>
          <p>The National Herbarium of New South Wales is one of the most significant scientific, cultural and historical botanical resources in the Southern hemisphere. The 1.43 million preserved plant specimens have been captured as high-resolution images and the biodiversity metadata associated with each of the images captured in digital form. Botanical specimens date from year 1770 to today, and form voucher collections that document the distribution and diversity of the world&#39;s flora through time, particularly that of NSW, Austalia and the Pacific.The data is used in biodiversity assessment, syste...</p>
          <p><a href="/nsw-herbarium/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://avh.ala.org.au/occurrences/search?q&#x3D;collection_uid%3Aco54#tab_mapView" class="" target="_none">The Australasian Virtual Herbarium</a> by <a href="https://avh.chah.org.au/index.php/about/" target="_none">avh@chah.org.au</a></li></a>
          </li>
          <li>
            <a href="https://www.ala.org.au/" class="" target="_none">Atlas of Living Australia</a> by <a href="https://www.ala.org.au/about-ala/" target="_none">CSIRO</a></li></a>
          </li>
          <li>
            <a href="https://www.rbgsyd.nsw.gov.au/science/national-herbarium-of-new-south-wales" class="" target="_none">Accessing the National Herbarium of NSW on AWS</a> by <a href="https://www.rbgsyd.nsw.gov.au/Science/Our-Science-Staff" target="_none">Dr Shelley James</a></li></a>
          </li>
        </ul>
        <p><a href="/nsw-herbarium/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="grillo-openeew" class="dataset">
          <h3><a href="/grillo-openeew/">OpenEEW</a></h3>
          <p><span class="label label-info tag link-tag">deep learning</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">earthquakes</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info-sustainability tag link-tag">sustainability</span></p>
          <p>Grillo has developed an IoT-based earthquake early-warning system,
with sensors currently deployed in Mexico, Chile, Puerto Rico and Costa Rica,
and is now opening its entire archive of unprocessed accelerometer
data to the world to encourage the development of new algorithms
capable of rapidly detecting and characterizing earthquakes in
real time.</p>
          <p><a href="/grillo-openeew/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://openeew.com/docs/machine-learning" class="" target="_none">Developing a machine learning model for better earthquake detection</a> by <a href="https://grillo.io/" target="_none">Grillo</a></li></a>
          </li>
          <li>
            <a href="https://github.com/openeew/openeew-python" class="" target="_none">OpenEEW library for Python</a> by <a href="https://grillo.io/" target="_none">Grillo</a></li></a>
          </li>
          <li>
            <a href="https://openeew.com/docs/analyze-historic" class="" target="_none">Analyzing a magnitude 7.2 earthquake in Mexico using Python</a> by <a href="https://grillo.io/" target="_none">Grillo</a></li></a>
          </li>
        </ul>
        <p><a href="/grillo-openeew/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="nrel-pds-porotomo" class="dataset">
          <h3><a href="/nrel-pds-porotomo/">PoroTomo</a></h3>
          <p><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">geothermal</span><span class="label label-info tag link-tag">image processing</span><span class="label label-info tag link-tag">seismology</span></p>
          <p>Released to the public as part of the Department of Energy&#39;s Open Energy Data
Initiative, these data represent vertical and horizontal distributed acoustic
sensing (DAS) data collected as part of the Poroelastic Tomography (PoroTomo)
project funded in part by the Office of Energy Efficiency and Renewable
Energy (EERE), U.S. Department of Energy.</p>
          <p><a href="/nrel-pds-porotomo/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://pangea.stanford.edu/ERE/pdf/IGAstandard/SGW/2018/Miller.pdf" class="" target="_none">DAS and DTS at Brady Hot Springs: Observations about Coupling and Coupled Interpretations</a> by Douglas E. Miller, Thomas Coleman, Xiangfang Zeng, Jeremy R. Patterson , Elena C. Reinnisch, Michael A. Cardiff, Herbert F. Wang, Dante Fratta, Whitney Trainor-Guitton, Clifford H. Thurber, Michelle ROBERTSON, Kurt FEIGL, and The PoroTomo Team</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1093/gji/ggy182" class="" target="_none">Ground motion response to an ML 4.3 earthquake using co-located distributed acoustic sensing and seismometer arrays</a> by Herbert F Wang, Xiangfang Zeng, Douglas E Miller, Dante Fratta, Kurt L Feigl, Clifford H Thurber, Robert J Mellors</li></a>
          </li>
          <li>
            <a href="https://github.com/openEDI/documentation/blob/master/PoroTomo_Distributed_Acoustic_Sensing_(DAS)_Data.ipynb" class="" target="_none">PoroTomo DAS Data Processing Tutorial</a> by <a href="https://orcid.org/0000-0001-7776-2028" target="_none">Nicole Taverna and Ross Ring-Jarvi</a></li></a>
          </li>
        </ul>
        <p><a href="/nrel-pds-porotomo/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="silo" class="dataset">
          <h3><a href="/silo/">SILO climate data on AWS</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">model</span><span class="label label-info-sustainability tag link-tag">sustainability</span><span class="label label-info tag link-tag">water</span><span class="label label-info tag link-tag">weather</span></p>
          <p><a href="https://www.longpaddock.qld.gov.au/silo">SILO</a> is a database of Australian <a href="https://www.longpaddock.qld.gov.au/silo/about/climate-variables">climate data</a> from 1889 to the present. It provides continuous, daily time-step <a href="https://www.longpaddock.qld.gov.au/silo/about/data-products">data products</a> in ready-to-use <a href="https://www.longpaddock.qld.gov.au/silo/about/file-formats-and-samples">formats</a> for research and operational applications.
SIL...</p>
          <p><a href="/silo/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.longpaddock.qld.gov.au/silo/gridded-data/nco" class="" target="_none">NetCDF Operators to calculate seasonal means</a> by SILO</li></a>
          </li>
          <li>
            <a href="https://www.longpaddock.qld.gov.au/silo/gridded-data/conversion" class="" target="_none">Convert NetCDF to ESRI ArcASCII or GeoTIFF</a> by SILO</li></a>
          </li>
          <li>
            <a href="https://www.longpaddock.qld.gov.au/silo/gridded-data/python" class="" target="_none">Python script to calculate a regional mean</a> by SILO</li></a>
          </li>
        </ul>
        <p><a href="/silo/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="sorel-20m" class="dataset">
          <h3><a href="/sorel-20m/">Sophos/ReversingLabs 20 Million malware detection dataset</a></h3>
          <p><span class="label label-info tag link-tag">cyber security</span><span class="label label-info tag link-tag">deep learning</span><span class="label label-info tag link-tag">labeled</span><span class="label label-info tag link-tag">machine learning</span></p>
          <p>A dataset intended to support research on machine learning
techniques for detecting malware.  It includes metadata and EMBER-v2
features for approximately 10 million benign and 10 million malicous
Portable Executable files, with disarmed but otherwise complete
files for all malware samples.  All samples are labeled using Sophos
in-house labeling methods, have features extracted using the
EMBER-v2 feature set, well as metadata obtained via the pefile
python library, detection counts obtained via ReversingLabs
telemetry, and additional behavioral tags that indicate the rough
behavior of the samp...</p>
          <p><a href="/sorel-20m/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/sophos-ai/SOREL-20M" class="" target="_none">SOREL-20M dataset interface code</a> by Richard Harang and Ethan M Rudd</li></a>
          </li>
          <li>
            <a href="https://arxiv.org/abs/2012.07634" class="" target="_none">SOREL-20M: A Large Scale Benchmark Dataset for Malicious PE Detection
</a> by Richard Harang and Ethan M Rudd</li></a>
          </li>
          <li>
            <a href="https://github.com/sophos-ai/SOREL-20M#quickstart" class="" target="_none">SOREL-20M quickstart</a> by Richard Harang</li></a>
          </li>
        </ul>
        <p><a href="/sorel-20m/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="sevir" class="dataset">
          <h3><a href="/sevir/">Storm EVent ImageRy (SEVIR)</a></h3>
          <p><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info tag link-tag">weather</span></p>
          <p>Collection of spatially and temporally aligned GOES-16 ABI satellite imagery, NEXRAD radar mosaics, and GOES-16 GLM lightning detections.</p>
          <p><a href="/sevir/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://nbviewer.jupyter.org/github/MIT-AI-Accelerator/eie-sevir/blob/master/examples/Generators.ipynb" class="" target="_none">Using Generators for SEVIR data</a> by <a href="https://github.com/markveillette" target="_none">Mark Veillette</a></li></a>
          </li>
          <li>
            <a href="https://github.com/MIT-AI-Accelerator/eie-sevir" class="" target="_none">sevir -- python utilities for working with SEVIR dataset</a> by <a href="https://github.com/markveillette" target="_none">Mark Veillette</a></li></a>
          </li>
          <li>
            <a href="https://nbviewer.jupyter.org/github/MIT-AI-Accelerator/eie-sevir/blob/master/examples/SEVIR_Tutorial.ipynb" class="" target="_none">Introduction to SEVIR</a> by <a href="https://github.com/markveillette" target="_none">Mark Veillette</a></li></a>
          </li>
        </ul>
        <p><a href="/sevir/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="human-microbiome-project" class="dataset">
          <h3><a href="/human-microbiome-project/">The Human Microbiome Project</a></h3>
          <p><span class="label label-info tag link-tag">amino acid</span><span class="label label-info tag link-tag">fasta</span><span class="label label-info tag link-tag">fastq</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">metagenomics</span><span class="label label-info tag link-tag">microbiome</span></p>
          <p>The NIH-funded Human Microbiome Project (HMP) is a collaborative effort of over 300 scientists from more than 80 organizations to comprehensively characterize the microbial communities inhabiting the human body and elucidate their role in human health and disease. To accomplish this task, microbial community samples were isolated from a cohort of 300 healthy adult human subjects at 18 specific sites within five regions of the body (oral cavity, airways, urogenital track, skin, and gut). Targeted sequencing of the 16S bacterial marker gene and/or whole metagenome shotgun sequencing was performe...</p>
          <p><a href="/human-microbiome-project/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.sciencedirect.com/science/article/pii/S2452231718300289" class="" target="_none">New microbe genomic variants in patients fecal community following surgical disruption of the upper human gastrointestinal tract</a> by Ranjit Kumar, Jayleen Grams, Daniel I. Chu, David K.Crossman, Richard Stahl, Peter Eipers, et al</li></a>
          </li>
          <li>
            <a href="https://www.nature.com/articles/nature06244" class="" target="_none">The Human Microbiome Project</a> by Peter J. Turnbaugh, Ruth E. Ley, Micah Hamady, Claire M. Fraser-Liggett, Rob Knight &amp; Jeffrey I. Gordon</li></a>
          </li>
          <li>
            <a href="https://www.nature.com/articles/nature23889" class="" target="_none">Strains, functions and dynamics in the expanded Human Microbiome Project</a> by Jason Lloyd-Price, Anup Mahurkar, Gholamali Rahnavard, Jonathan Crabtree, Joshua Orvis, A. Brantley Hall, et al.</li></a>
          </li>
        </ul>
        <p><a href="/human-microbiome-project/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="hail-vep-pipeline" class="dataset">
          <h3><a href="/hail-vep-pipeline/">Variant Effect Predictor (VEP) and the Loss-Of-Function Transcript Effect Estimator (LOFTEE) Plugin</a></h3>
          <p><span class="label label-info tag link-tag">genome wide association study</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">loftee</span><span class="label label-info tag link-tag">vep</span></p>
          <p>VEP determines the effect of genetic variants (SNPs, insertions, deletions, CNVs or structural variants) on genes, transcripts, and protein sequence, as well as regulatory regions. The European Bioinformatics Institute produces the VEP tool/db and releases updates every 1 - 6 months. The latest release contains 267 genomes from 232 species containing 5567663 protein coding genes. This dataset hosts the last 5 releases for human, rat, and zebrafish. Also, it hosts the required reference files for the Loss-Of-Function Transcript Effect Estimator (LOFTEE) plugin as it is commonly used with VEP.</p>
          <p><a href="/hail-vep-pipeline/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="http://uswest.ensembl.org/info/docs/tools/vep/index.html" class="" target="_none">Variant Effect Predictor (VEP)</a> by <a href="http://uswest.ensembl.org/info/docs/tools/vep/index.html" target="_none">Ensembl</a></li></a>
          </li>
          <li>
            <a href="https://hail.is/" class="" target="_none">Hail</a> by <a href="https://www.nealelab.is/" target="_none">Neale Lab</a></li></a>
          </li>
          <li>
            <a href="https://github.com/konradjk/loftee" class="" target="_none">Loss-Of-Function Transcript Effect Estimator (LOFTEE)</a> by <a href="https://github.com/konradjk" target="_none">Konrad Karczewski</a></li></a>
          </li>
        </ul>
        <p><a href="/hail-vep-pipeline/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="afsis" class="dataset">
          <h3><a href="/afsis/">Africa Soil Information Service (AfSIS) Soil Chemistry</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">food security</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info-sustainability tag link-tag">sustainability</span></p>
          <p>This dataset contains soil infrared spectral data and paired soil property
reference measurements for georeferenced soil samples that were collected
through the Africa Soil Information Service (AfSIS) project, which lasted
from 2009 through 2018. In this release, we include data collected during
Phase I (2009-2013.) Georeferenced samples were collected from 19 countries
in Sub-Saharan African using a statistically sound sampling scheme,
and their soil properties were analyzed using <em>both</em> conventional soil
testing methods and spectral methods (infrared diffuse reflectance
spectroscopy). The two ...</p>
          <p><a href="/afsis/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.youtube.com/watch?v&#x3D;Fb9R0CnPMkc" class="" target="_none">Goalkeepers 2018, Soil - The Big Data Beneath Your Feet</a> by <a href="https://qed.ai" target="_none">QED</a></li></a>
          </li>
          <li>
            <a href="https://github.com/qedsoftware/afsis-soil-chem-tutorial/" class="" target="_none">AfSIS Soil Chemistry - Usage Tutorial</a> by <a href="https://qed.ai" target="_none">QED</a></li></a>
          </li>
        </ul>
        <p><a href="/afsis/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="amazon-bin-imagery" class="dataset">
          <h3><a href="/amazon-bin-imagery/">Amazon Bin Image Dataset</a></h3>
          <p><span class="label label-info tag link-tag">computer vision</span><span class="label label-info tag link-tag">machine learning</span></p>
          <p>The Amazon Bin Image Dataset contains over 500,000 images and metadata from bins of a pod in an operating Amazon Fulfillment Center. The bin images in this dataset are captured as robot units carry pods as part of normal Amazon Fulfillment Center operations.</p>
          <p><a href="/amazon-bin-imagery/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/OneNow/AI-Inventory-Reconciliation" class="" target="_none">Amazon Inventory Reconciliation using AI</a> by <a href="https://github.com/pablo-tech" target="_none">Pablo Rodriguez Bertorello, Sravan Sripada, Nutchapol Dendumrongsup</a></li></a>
          </li>
          <li>
            <a href="https://github.com/silverbottlep/abid_challenge" class="" target="_none">Amazon Bin Image Dataset Challenge</a> by <a href="https://github.com/silverbottlep" target="_none">silverbottlep</a></li></a>
          </li>
        </ul>
        <p><a href="/amazon-bin-imagery/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="meteo-france-models" class="dataset">
          <h3><a href="/meteo-france-models/">Atmospheric Models from Météo-France</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">model</span><span class="label label-info-sustainability tag link-tag">sustainability</span><span class="label label-info tag link-tag">weather</span></p>
          <p>Global and high-resolution regional atmospheric models from Météo-France.<ul>
<li>ARPEGE World covers the entire world at a base horizontal resolution of 0.5° (~55km) between grid points, it predicts weather out up to 114 hours in the future.</li>
<li>ARPEGE Europe covers Europe and North-Africa at a base horizontal resolution of 0.1° (~11km) between grid points, it predicts weather out up to 114 hours in the future.</li>
<li>AROME France covers France at a base horizontal resolution of 0.025° (~2.5km) between grid points, it predicts weather out up to 42 hours in the future.</li>
<li>AROME France HD covers France and neigborhood at a base horizontal resolution of 0.01° (~1.5km) between grid points, it predicts weather out up to 42 hours in the future.</li>
</ul>
Dozens of atmospheric variables are avail...</p>
          <p><a href="/meteo-france-models/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://windguru.cz" class="" target="_none">Windguru.cz</a> by <a href="https://windguru.cz" target="_none">Windguru</a></li></a>
          </li>
          <li>
            <a href="https://windy.com" class="" target="_none">Windy.com</a> by <a href="https://windy.com" target="_none">Windy</a></li></a>
          </li>
        </ul>
        <p><a href="/meteo-france-models/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="broad-references" class="dataset">
          <h3><a href="/broad-references/">Broad Genome References</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">Homo sapiens</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">reference index</span></p>
          <p>Broad maintained human genome reference builds hg19/hg38 and decoy references.</p>
          <p><a href="/broad-references/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://aws.amazon.com/blogs/storage/using-amazon-fsx-for-lustre-for-genomics-workflows-on-aws/" class="aws-link" target="_none">Using Amazon FSx for Lustre for Genomics Workflows on AWS</a> by <a href="https://www.linkedin.com/in/lee-pang-a039a26/" target="_none">W. Lee Pang</a></li></a>
          </li>
          <li>
            <a href="https://docs.opendata.aws/genomics-workflows/orchestration/cromwell/cromwell-examples/#real-world-example-haplotypecaller" class="aws-link" target="_none">Genomics Workflows on AWS - Cromwell on AWS</a> by <a href="https://www.linkedin.com/in/lee-pang-a039a26/" target="_none">W. Lee Pang</a></li></a>
          </li>
        </ul>
        <p><a href="/broad-references/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="cgci" class="dataset">
          <h3><a href="/cgci/">Cancer Genome Characterization Initiatives - Burkitt Lymphoma, HIV+ Cervical Cancer</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">STRIDES</span><span class="label label-info tag link-tag">transcriptomics</span></p>
          <p>The Cancer Genome Characterization Initiatives (CGCI) program supports cutting-edge genomics research of adult and pediatric cancers. CGCI investigators develop and apply advanced sequencing methods that examine genomes, exomes, and transcriptomes within various types of tumors. The program includes Burkitt Lymphoma Genome Sequencing Project (BLGSP) project and HIV+ Tumor Molecular Characterization Project - Cervical Cancer (HTMCP-CC) project.
The dataset contains open Clinical Supplement, Biospecimen Supplement, RNA-Seq Gene Expression Quantification, miRNA-Seq Isoform Expression Quantificati...</p>
          <p><a href="/cgci/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://portal.gdc.cancer.gov/projects/CGCI-BLGSP" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1182/blood-2018-09-871418" class="" target="_none">Genome-wide discovery of somatic coding and noncoding mutations in pediatric endemic and
sporadic Burkitt lymphoma
</a> by Grande B. M., Gerhard D. S., Jiang A., Griner N. B., Abramson J. S., Alexander T. B., et al.
</li></a>
          </li>
        </ul>
        <p><a href="/cgci/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="janelia-cosem" class="dataset">
          <h3><a href="/janelia-cosem/">Cell Organelle Segmentation in Electron Microscopy (COSEM) on AWS</a></h3>
          <p><span class="label label-info tag link-tag">cell biology</span><span class="label label-info tag link-tag">computer vision</span><span class="label label-info tag link-tag">electron microscopy</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">organelle</span></p>
          <p>High resolution images of subcellular structures.</p>
          <p><a href="/janelia-cosem/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://elifesciences.org/articles/25916" class="" target="_none">Enhanced FIB-SEM systems for large-volume 3D imaging</a> by C. Shan Xu, Kenneth J. Hayworth, Zhiyuan Lu, Patricia Grob, Ahmed M. Hassan, José G. García-Cerdán, Krishna K. Niyogi, Eva Nogales, Richard J. Weinberg, Harald F. Hess.</li></a>
          </li>
          <li>
            <a href="https://science.sciencemag.org/content/367/6475/eaaz5357" class="" target="_none">Correlative three-dimensional super-resolution and block-face electron microscopy of whole vitreously frozen cells.</a> by David P. Hoffman1, Gleb Shtengel, C. Shan Xu, Kirby R. Campbell, Melanie Freeman, Lei Wang, Daniel E. Milkie, H. Amalia Pasolli, Nirmala Iyer, John A. Bogovic, Daniel R. Stabley, Abbas Shirinifard, Song Pang, David Peale, Kathy Schaefer, Wim Pomp, Chi-Lun Chang, Jennifer Lippincott-Schwartz, Tom Kirchhausen1, David J. Solecki, Eric Betzig, Harald F. Hess.</li></a>
          </li>
        </ul>
        <p><a href="/janelia-cosem/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="jhu-indexes" class="dataset">
          <h3><a href="/jhu-indexes/">Cloud Indexes for Genomic Analyses</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">mapping</span><span class="label label-info tag link-tag">medicine</span><span class="label label-info tag link-tag">reference index</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>Genomic tools use reference databases as indexes to operate quickly and efficiently, analogous to how web search engines use indexes for fast querying. Here, we aggregate genomic, pan-genomic and metagenomic indexes for analysis of sequencing data.</p>
          <p><a href="/jhu-indexes/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.biorxiv.org/content/10.1101/2020.03.03.975219v4.full.pdf" class="" target="_none">Reducing reference bias using multiple population reference genomes</a> by Chen et al (2020)</li></a>
          </li>
          <li>
            <a href="https://benlangmead.github.io/aws-indexes/tutorial" class="" target="_none">Table of contents for tutorials for constituent tools</a> by <a href="https://benlangmead.github.io/aws-indexes" target="_none">Ben Langmead</a></li></a>
          </li>
        </ul>
        <p><a href="/jhu-indexes/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="copernicus-dem" class="dataset">
          <h3><a href="/copernicus-dem/">Copernicus Digital Elevation Model (DEM)</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">elevation</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info-sustainability tag link-tag">sustainability</span></p>
          <p>The Copernicus DEM is a Digital Surface Model (DSM) which represents the surface of the Earth including buildings, infrastructure and vegetation. We provide two instances of Copernicus DEM named GLO-30 Public and GLO-90. GLO-90 provides worldwide coverage at 90 meters. GLO-30 Public provides limited worldwide coverage at 30 meters because a small subset of tiles covering specific countries are not yet released to the public by the Copernicus Programme. Note that in both cases ocean areas do not have tiles, there one can assume height values equal to zero. Data is provided as Cloud Optimized Ge...</p>
          <p><a href="/copernicus-dem/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="http://www.sentinel-hub.com/" class="" target="_none">Sentinel Hub WMS/WMTS/WCS Service and Process API</a> by <a href="http://www.sinergise.com/" target="_none">Sinergise</a></li></a>
          </li>
          <li>
            <a href="http://apps.sentinel-hub.com/eo-browser/" class="" target="_none">EO Browser</a> by <a href="http://www.sinergise.com/" target="_none">Sinergise</a></li></a>
          </li>
        </ul>
        <p><a href="/copernicus-dem/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="cmip6" class="dataset">
          <h3><a href="/cmip6/">Coupled Model Intercomparison Project 6</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">atmosphere</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">model</span><span class="label label-info tag link-tag">oceans</span><span class="label label-info tag link-tag">simulations</span><span class="label label-info tag link-tag">weather</span></p>
          <p>The sixth phase of global coupled ocean-atmosphere general circulation model ensemble. Additional data can be requested via <a href="https://docs.google.com/forms/d/e/1FAIpQLScFjA5IddqBs2Rc0xGbzn32NPq12TKr-b-8KGtXWyNDK5sJCg/viewform">Google Form</a></p>
          <p><a href="/cmip6/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://nbviewer.jupyter.org/github/awslabs/amazon-asdi/blob/main/examples/dask/notebooks/cmip6_zarr.ipynb" class="aws-link" target="_none">Processing CMIP6 data in Zarr format with Dask &amp; AWS Fargate</a> by Zac Flamig</li></a>
          </li>
          <li>
            <a href="https://gmd.copernicus.org/articles/special_issue590.html" class="" target="_none">Special issue | Coupled Model Intercomparison Project Phase 6 (CMIP6) Experimental Design and Organization</a> by V. Eyring</li></a>
          </li>
        </ul>
        <p><a href="/cmip6/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="digitalcorpora" class="dataset">
          <h3><a href="/digitalcorpora/">DigitalCorpora</a></h3>
          <p><span class="label label-info tag link-tag">computer forensics</span><span class="label label-info tag link-tag">computer security</span><span class="label label-info tag link-tag">CSI</span><span class="label label-info tag link-tag">cyber security</span><span class="label label-info tag link-tag">digital forensics</span><span class="label label-info tag link-tag">image processing</span><span class="label label-info tag link-tag">imaging</span><span class="label label-info tag link-tag">information retrieval</span><span class="label label-info tag link-tag">internet</span><span class="label label-info tag link-tag">intrusion detection</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">machine translation</span><span class="label label-info tag link-tag">text analysis</span></p>
          <p>Disk images, memory dumps, network packet captures, and files for use in digital forensics research and education. All of this information is accessible through the digitalcorpora.org website, and made available at s3://digitalcorpora/. Some of these datasets implement scenarios that were performed by students, faculty, and others acting <em>in persona</em>. As such, the information is synthetic and may be used without prior authorization or IRB approval. Details of these datasets can be found at <a href="http://www.simson.net/clips/academic/2009...</p>
          <p><a href="/digitalcorpora/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="http://www.simson.net/clips/academic/2009.DFRWS.Corpora.pdf" class="" target="_none">Bringing Science to Digital Forensics with Standardized Forensic Corpora</a> by <a href="https://simson.net/" target="_none">Garfinkel, Farrell, Roussev and Dinolt</a></li></a>
          </li>
          <li>
            <a href="http://simson.net/clips/academic/2011.ADFSL.Corpora.pdf" class="" target="_none">Creating Realistic Corpora for Forensic and Security Education</a> by <a href="https://simson.net/" target="_none">Woods, K., Christopher Lee, Simson Garfinkel, David Dittrich, Adam Russel, Kris Kearton</a></li></a>
          </li>
        </ul>
        <p><a href="/digitalcorpora/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="hst" class="dataset">
          <h3><a href="/hst/">Hubble Space Telescope Public Data</a></h3>
          <p><span class="label label-info tag link-tag">astronomy</span></p>
          <p>The Hubble Space Telescope (HST) is one of the most productive scientific instruments ever created. This dataset contains calibrated and raw data for all of the currently active instruments on HST: ACS, COS, STIS and WFC3.</p>
          <p><a href="/hst/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://mast-labs.stsci.io/2018/06/exploring-aws-lambda-with-hst-public-data" class="" target="_none">Exploring AWS Lambda with cloud-hosted Hubble public data</a> by <a href="https://twitter.com/arfon" target="_none">Arfon Smith</a></li></a>
          </li>
          <li>
            <a href="https://mast-labs.stsci.io/2018/06/hst-public-data-on-aws" class="" target="_none">Making HST Public Data Available on AWS</a> by <a href="https://twitter.com/arfon" target="_none">Arfon Smith</a></li></a>
          </li>
        </ul>
        <p><a href="/hst/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="mimiciii" class="dataset">
          <h3><a href="/mimiciii/">MIMIC-III (‘Medical Information Mart for Intensive Care’)</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">health</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">natural language processing</span><span class="label label-info tag link-tag">us</span></p>
          <p>MIMIC-III (‘Medical Information Mart for Intensive Care’) is a large, 
single-center database comprising information relating to patients 
admitted to critical care units at a large tertiary care hospital. 
Data includes vital signs, medications, laboratory measurements, 
observations and notes charted by care providers, fluid balance, 
procedure codes, diagnostic codes, imaging reports, hospital length 
of stay, survival data, and more. The database supports applications 
including academic and industrial research, quality improvement initiatives, 
and higher education coursework. The MIMIC-I...</p>
          <p><a href="/mimiciii/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/MIT-LCP/mimic-code" class="" target="_none">MIMIC-code GitHub repository</a> by <a href="https://github.com/alistairewj" target="_none">Alistair Johnson</a></li></a>
          </li>
          <li>
            <a href="https://aws.amazon.com/blogs/big-data/perform-biomedical-informatics-without-a-database-using-mimic-iii-data-and-amazon-athena/" class="aws-link" target="_none">Perform biomedical informatics without a database using MIMIC-III data and Amazon Athena</a> by James Wiggins, Alistair Johnson</li></a>
          </li>
        </ul>
        <p><a href="/mimiciii/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="naip" class="dataset">
          <h3><a href="/naip/">NAIP on AWS</a></h3>
          <p><span class="label label-info tag link-tag">aerial imagery</span><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">natural resource</span><span class="label label-info tag link-tag">regulatory</span><span class="label label-info-sustainability tag link-tag">sustainability</span></p>
          <p>The National Agriculture Imagery Program (NAIP) acquires aerial imagery during the agricultural growing seasons in the continental U.S. This &quot;leaf-on&quot; imagery andtypically ranges from 60 centimeters to 100 centimeters in resolution and is available from the naip-analytic Amazon S3 bucket as 4-band (RGB + NIR) imagery in MRF format, on naip-source Amazon S3 bucket as 4-band (RGB + NIR) in uncompressed Raw GeoTiff format and naip-visualization as 3-band (RGB) Cloud Optimized GeotTiff format. NAIP data is delivered at the state level; every year, a number of states receive updates, with...</p>
          <p><a href="/naip/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://eos.com/landviewer/" class="" target="_none">EOS Land Viewer</a> by <a href="https://eos.com/" target="_none">Earth Observing System</a></li></a>
          </li>
          <li>
            <a href="https://www.voyagersearch.com/using-aws-batch-to-generate-image-thumbnails-for-voyager" class="" target="_none">VoyagerSearch showing off Batch + NAIP</a> by <a href="https://www.voyagersearch.com/" target="_none">Voyager</a></li></a>
          </li>
        </ul>
        <p><a href="/naip/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="noaa-cfs" class="dataset">
          <h3><a href="/noaa-cfs/">NOAA Climate Forecast System (CFS)</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info-sustainability tag link-tag">sustainability</span><span class="label label-info tag link-tag">weather</span></p>
          <p>The Climate Forecast System (CFS) is a model representing the global interaction between Earth&#39;s oceans, land, and atmosphere. Produced by several dozen scientists under guidance from the National Centers for Environmental Prediction (NCEP), this model offers hourly data with a horizontal resolution down to one-half of a degree (approximately 56 km) around Earth for many variables. CFS uses the latest scientific approaches for taking in, or assimilating, observations from data sources including surface observations, upper air balloon observations, aircraft observations, and satellite obser...</p>
          <p><a href="/noaa-cfs/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://journals.ametsoc.org/view/journals/bams/91/8/2010bams3001_1.xml" class="" target="_none">The NCEP Climate Forecast System Reanalysis</a> by Saha, Suranjana, and Coauthors</li></a>
          </li>
          <li>
            <a href="https://journals.ametsoc.org/view/journals/clim/27/6/jcli-d-12-00823.1.xml" class="" target="_none">The NCEP Climate Forecast System Version 2</a> by Saha, Suranjana, and Coauthors</li></a>
          </li>
        </ul>
        <p><a href="/noaa-cfs/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="nwm-archive" class="dataset">
          <h3><a href="/nwm-archive/">NOAA National Water Model Reanalysis</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info-sustainability tag link-tag">sustainability</span><span class="label label-info tag link-tag">transportation</span><span class="label label-info tag link-tag">weather</span></p>
          <p>The NOAA National Water Model Reanalysis dataset contains output
from multi-decade retrospective simulations. These
simulations used observed rainfall as input and ingested other
required meteorological input fields from a weather reanalysis
dataset. The output frequency and fields available in this
historical NWM dataset differ from those contained in the
real-time forecast model. One application of this dataset is
to provide historical context to current real-time streamflow,
soil moisture and snowpack NWM conditions. The reanalysis data
can be used to infer flow frequencies and perform temp...</p>
          <p><a href="/nwm-archive/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1016/j.ocemod.2019.101526" class="" target="_none">Simulating storm surge and compound flooding events with a creek-to-ocean model: Importance of baroclinic effects</a> by Fei Ye, et al.</li></a>
          </li>
          <li>
            <a href="https://nbviewer.jupyter.org/gist/rsignell-usgs/d3dfaf3cd3d8b39894a69b22127dfe38" class="" target="_none">Explore the National Water Model Reanalysis in Zarr</a> by <a href="https://www.usgs.gov/staff-profiles/richard-p-signell" target="_none">Rich Signell</a></li></a>
          </li>
        </ul>
        <p><a href="/nwm-archive/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="noaa-ofs" class="dataset">
          <h3><a href="/noaa-ofs/">NOAA Operational Forecast System (OFS)</a></h3>
          <p><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">coastal</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">oceans</span><span class="label label-info-sustainability tag link-tag">sustainability</span><span class="label label-info tag link-tag">water</span><span class="label label-info tag link-tag">weather</span></p>
          <p>The Operational Forecast System (OFS) has been developed to serve the maritime user community. OFS was developed in a joint project of the NOAA/National Ocean Service (NOS)/Office of Coast Survey, the NOAA/NOS/Center for Operational Oceanographic Products and Services (CO-OPS), and the NOAA/National Weather Service (NWS)/National Centers for Environmental Prediction (NCEP) Central Operations (NCO). OFS generates water level, water current, water temperature, water salinity (except for the Great Lakes) and nowcast and forecast guidance four times per day.</p>
          <p><a href="/noaa-ofs/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://opendap.co-ops.nos.noaa.gov/ofsdataagg/index.html" class="" target="_none">OFS Data Aggregation and Sub-Setting</a> by NOAA</li></a>
          </li>
          <li>
            <a href="https://www.weather.gov/media/notification/pdf2/scn21-14nos_wcofs_ngofs2aaa.pdf" class="" target="_none">Implementation of new Oceanographic Forecast Modeling System for the U.S. West Coast (WCOFS) and the Upgraded Northern Gulf of Mexico (NGOFS2)</a> by NOAA</li></a>
          </li>
        </ul>
        <p><a href="/noaa-ofs/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="noaa-wod" class="dataset">
          <h3><a href="/noaa-wod/">NOAA World Ocean Database (WOD)</a></h3>
          <p><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">oceans</span><span class="label label-info-sustainability tag link-tag">sustainability</span></p>
          <p>The World Ocean Database (WOD) is the largest uniformly formatted, quality-controlled, publicly available historical subsurface ocean profile database. From Captain Cook&#39;s second voyage in 1772 to today&#39;s automated Argo floats, global aggregation of ocean variable information including temperature, salinity, oxygen, nutrients, and others vs. depth allow for study and understanding of the changing physical, chemical, and to some extent biological state of the World&#39;s Oceans. Browse the bucket via the AWS S3 explorer: <a href="https://noaa-wod-pds.s3.amazonaws.com/index.html">https://noaa-wod-pds.s3.amazonaws.com/index.html</a></p>
          <p><a href="/noaa-wod/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://data.nodc.noaa.gov/woa/WOD/DOC/wod_intro.pdf" class="" target="_none">The World Ocean Database Introduction</a> by Tim P. Boyer, Olga K. Baranova, Carla Coleman, Hernan E. Garcia, Alexandra Grodsky, Ricardo A. Locarnini, Alexey V. Mishonov, Christopher R. Paver, James R. Reagan, Dan Seidov, Igor V. Smolyar, Katharine W. Weathers, Melissa M. Zweng</li></a>
          </li>
          <li>
            <a href="https://data.nodc.noaa.gov/woa/WOD/DOC/wodreadme.pdf" class="" target="_none">The World Ocean Database User&#x27;s Manual</a> by Hernan E. Garcia, Tim P. Boyer, Ricardo A. Locarnini, Olga K. Baranova, Melissa M. Zweng</li></a>
          </li>
        </ul>
        <p><a href="/noaa-wod/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="nciccr-dlbcl" class="dataset">
          <h3><a href="/nciccr-dlbcl/">National Cancer Institute Center for Cancer Research - Diffuse Large B Cell Lymphoma (DLBCL) Genomics and Expression</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genomic</span></p>
          <p>The study describes integrative analysis of genetic lesions in 574 diffuse large B cell lymphomas
(DLBCL) involving exome and transcriptome sequencing, array-based DNA copy number analysis and
targeted amplicon resequencing. The dataset contains open RNA-Seq Gene Expression Quantification
data.</p>
          <p><a href="/nciccr-dlbcl/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://portal.gdc.cancer.gov/projects/NCICCR-DLBCL" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1056/NEJMoa1801445" class="" target="_none">Genetics and Pathogenesis of Diffuse Large B Cell Lymphoma</a> by Roland Schmitz, Ph.D., George W. Wright, Ph.D., Da Wei Huang, M.D., et al.</li></a>
          </li>
        </ul>
        <p><a href="/nciccr-dlbcl/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="opencitymodel" class="dataset">
          <h3><a href="/opencitymodel/">Open City Model (OCM)</a></h3>
          <p><span class="label label-info tag link-tag">cities</span><span class="label label-info tag link-tag">events</span><span class="label label-info tag link-tag">geospatial</span></p>
          <p>Open City Model is an initiative to provide cityGML data for all the buildings in the United States.
By using other open datasets in conjunction with our own code and algorithms it is our goal to provide 3D geometries for every US building.</p>
          <p><a href="/opencitymodel/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/opencitymodel/opencitymodel/blob/master/examples/Query-OpenCityModel-using-AWS-Athena.md" class="" target="_none">Running queries on Open City Model using AWS Athena</a> by <a href="https://github.com/agilliland" target="_none">Allen Gilliland</a></li></a>
          </li>
          <li>
            <a href="https://github.com/opencitymodel/opencitymodel/blob/master/examples/3dCityDB-to-GoogleEarth.md" class="" target="_none">Using Open City Model with the 3dCityDB</a> by <a href="https://github.com/agilliland" target="_none">Allen Gilliland</a></li></a>
          </li>
        </ul>
        <p><a href="/opencitymodel/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="ohsu-cnl" class="dataset">
          <h3><a href="/ohsu-cnl/">Oregon Health &amp; Science University Chronic Neutrophilic Leukemia Dataset</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span></p>
          <p>The OHSU-CNL study offers the whole exome and RNA-sequencing on a cohort of 100 cases with rare
hematologic malignancies such as Chronic neutrophilic leukemia (CNL), atypical chronic myeloid
leukemia (aCML), and unclassified myelodysplastic syndrome/myeloproliferative neoplasms
(MDS/MPN-U). This dataset contains open RNA-Seq Gene Expression Quantification data.</p>
          <p><a href="/ohsu-cnl/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://portal.gdc.cancer.gov/projects/" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></li></a>
          </li>
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/31366621" class="" target="_none">Genomic landscape of neutrophilic leukemias of ambiguous diagnosis</a> by Zhang H, Wilmot B, Bottomly D et al.</li></a>
          </li>
        </ul>
        <p><a href="/ohsu-cnl/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="organoid-pancreatic" class="dataset">
          <h3><a href="/organoid-pancreatic/">Pancreatic Cancer Organoid Profiling</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">STRIDES</span><span class="label label-info tag link-tag">transcriptomics</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>This study generated a collection of patient-derived pancreatic normal and cancer organoids and it was sequenced using Whole Genome Sequencing (WGS), Whole Exome Sequencing (WXS) and RNA-Seq as well as matched tumor and normal tissue if available. The study provides a valuable resource for pancreatic cancer researchers.
The dataset contains open RNA-Seq Gene Expression Quantification data and controlled WGS/WXS/RNA-Seq Aligned Reads, WXS Annotated Somatic Mutation, WXS Raw Somatic Mutation, and RNA-Seq Splice Junction Quantification.</p>
          <p><a href="/organoid-pancreatic/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://portal.gdc.cancer.gov/projects/ORGANOID-PANCREATIC" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></li></a>
          </li>
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed?cmd&#x3D;DetailsSearch&amp;term&#x3D;29853643[PMID]" class="" target="_none">Organoid Profiling Identifies Common Responders to Chemotherapy in Pancreatic Cancer</a> by Tiriac H, Belleau P, Engle DD, Plenker D, Deschênes A, Somerville TD, et al.</li></a>
          </li>
        </ul>
        <p><a href="/organoid-pancreatic/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="rapid-nrt-flood-maps" class="dataset">
          <h3><a href="/rapid-nrt-flood-maps/">RAPID NRT Flood Maps</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">water</span></p>
          <p>Near Real-time and archival data of High-resolution (10 m) flood inundation dataset over the Contiguous United States, developed based on the Sentinel-1 SAR imagery (2016-current) archive, using an automated Radar Produced Inundation Diary (RAPID) algorithm.</p>
          <p><a href="/rapid-nrt-flood-maps/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1016/j.rse.2018.11.008" class="" target="_none">Near Real-Time Nonobstructed Flood Inundation Mapping by Synthetic Aperture Radar</a> by Xinyi Shen, Emmanouil N. Anagnostou, George H. Allen, G. Robert Brakenridge, Albert J. Kettner</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.3390/rs11070879" class="" target="_none">Inundation Extent Mapping by Synthetic Aperture Radar: A Review</a> by Xinyi Shen, Dacheng Wang, Kebiao Mao, Emmanouil Anagnostou, and Yang Hong</li></a>
          </li>
        </ul>
        <p><a href="/rapid-nrt-flood-maps/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="rapid7-fdns-any" class="dataset">
          <h3><a href="/rapid7-fdns-any/">Rapid7 FDNS ANY Dataset</a></h3>
          <p><span class="label label-info tag link-tag">analytics</span><span class="label label-info tag link-tag">computer security</span><span class="label label-info tag link-tag">cyber security</span><span class="label label-info tag link-tag">internet</span></p>
          <p>Subset of FDNS ANY queries against domain names produced by <a href="https://www.rapid7.com">Rapid7</a> Project Sonar, made available in s3.
More information on the schema can be found at Rapid7&#39;s <a href="https://opendata.rapid7.com/sonar.fdns_v2/">Open Data website</a>.</p>
          <p><a href="/rapid7-fdns-any/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://blog.rapid7.com/2018/10/16/how-to-conduct-dns-reconnaissance-for-02-using-rapid7-open-data-and-aws/" class="" target="_none">How to Conduct DNS Reconnaissance for $.02 Using Rapid7 Open Data and AWS</a> by <a href="https://blog.rapid7.com/author/shan/" target="_none">Shan Sikdar at Rapid7</a></li></a>
          </li>
          <li>
            <a href="http://securityriskadvisors.com/blog/creating-a-project-sonar-fdns-api-with-aws/" class="" target="_none">Creating a Project Sonar FDNS API with AWS</a> by <a href="https://twitter.com/2xxeformyshirt" target="_none">Evan Perotti at SecurityRiskAdvisors</a></li></a>
          </li>
        </ul>
        <p><a href="/rapid7-fdns-any/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="rareplanes" class="dataset">
          <h3><a href="/rareplanes/">RarePlanes</a></h3>
          <p><span class="label label-info tag link-tag">computer vision</span><span class="label label-info tag link-tag">deep learning</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">labeled</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">satellite imagery</span></p>
          <p>RarePlanes is a unique open-source machine learning dataset from CosmiQ Works and AI.Reverie that incorporates both real and synthetically generated satellite imagery. The RarePlanes dataset specifically focuses on the value of AI.Reverie synthetic data to aid computer vision algorithms in their ability to automatically detect aircraft and their attributes in satellite imagery. Although other synthetic/real combination datasets exist, RarePlanes is the largest openly-available very high resolution dataset built to test the value of synthetic data from an overhead perspective. The real portion ...</p>
          <p><a href="/rareplanes/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://arxiv.org/abs/2006.02963" class="" target="_none">RarePlanes: Synthetic Data Takes Flight</a> by Jacob Shermeyer, Thomas Hossler, Adam Van Etten, Daniel Hogan, Ryan Lewis, Daeil Kim</li></a>
          </li>
          <li>
            <a href="https://github.com/aireveries/RarePlanes" class="" target="_none">RarePlanes Codebase</a> by Thomas Hossler and Jacob Shermeyer</li></a>
          </li>
        </ul>
        <p><a href="/rareplanes/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="sentinel1-slc-seasia-pds" class="dataset">
          <h3><a href="/sentinel1-slc-seasia-pds/">Sentinel-1 SLC dataset for South and Southeast Asia, Taiwan, Korea and Japan</a></h3>
          <p><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info-sustainability tag link-tag">sustainability</span></p>
          <p>The S1 Single Look Complex (SLC) dataset contains Synthetic Aperture Radar (SAR) data in the C-Band wavelength. The SAR sensors are installed on a two-satellite (Sentinel-1A and Sentinel-1B) constellation orbiting the Earth with a combined revisit time of six days, operated by the European Space Agency. The S1 SLC data are a Level-1 product that collects radar amplitude and phase information in all-weather, day or night conditions, which is ideal for studying natural hazards and emergency response, land applications, oil spill monitoring, sea-ice conditions, and associated climate change effec...</p>
          <p><a href="/sentinel1-slc-seasia-pds/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/earthobservatory/sentinel1-opds/wiki" class="" target="_none">Sentinel-1 Opendataset Wiki and Tutorials</a> by <a href="https://earthobservatory.sg/research-group/geodesy-group-emma-hill/people" target="_none">Earth Observatory of Singapore</a></li></a>
          </li>
          <li>
            <a href="https://www.nature.com/articles/s41597-020-0443-5" class="" target="_none">Rapid flood and damage mapping using synthetic aperture radar in response to Typhoon Hagibis, Japan</a> by Cheryl W. J. Tay, Sang-Ho Yun, Shi Tong Chin, Alok Bhardwaj, Jungkyo Jung &amp; Emma M. Hill</li></a>
          </li>
        </ul>
        <p><a href="/sentinel1-slc-seasia-pds/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="serratus-lovelywater" class="dataset">
          <h3><a href="/serratus-lovelywater/">Serratus: Ultra-deep Search for Novel Viruses - Versioned Data Release</a></h3>
          <p><span class="label label-info tag link-tag">bam</span><span class="label label-info tag link-tag">COVID-19</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">MERS</span><span class="label label-info tag link-tag">SARS</span><span class="label label-info tag link-tag">SARS-CoV-2</span><span class="label label-info tag link-tag">virus</span></p>
          <p>Serratus is a collaborative open science project for ultra-rapid discovery of known and unknown coronaviruses in response to the COVID-19 pandemic through re-analysis of publicly available genomic data. Our resulting vertebrate viral alignment data is explorable via the <a href="https://serratus.io/explorer">Serratus Explorer</a> and directly accessible on Amazon S3.</p>
          <p><a href="/serratus-lovelywater/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.biorxiv.org/content/10.1101/2020.07.28.224584v2" class="" target="_none">coronaSPAdes. From biosynthetic gene clusters to RNA viral assemblies</a> by Meleshko D., Hajirasouliha I., and Korobeynikov A. (2021)</li></a>
          </li>
          <li>
            <a href="https://www.biorxiv.org/content/10.1101/2020.08.07.241729v1" class="" target="_none">Petabase-scale sequence alignment catalyses viral discovery</a> by Edgar R., Taylor J., Altman T., Barbera P., Meleshko D., Lin V., et al (2020)</li></a>
          </li>
          <li>
            <a href="https://serratus.io/explorer" class="" target="_none">Serratus Explorer</a> by Serratus Team</li></a>
          </li>
          <li>
            <a href="https://github.com/serratus-bio/tantalus" class="" target="_none">Tantalus: An R Package for exploration of Serratus data</a> by Serratus Team</li></a>
          </li>
          <li>
            <a href="https://www.pnas.org/content/118/3/e2019907118" class="" target="_none">Diversification of mammalian deltaviruses by host shifting</a> by Bergner L.M., Orton R.J., et al (2021)</li></a>
          </li>
        </ul>
        <p><a href="/serratus-lovelywater/#usageexamples">See 5 usage examples &rarr;</a></p>
        </div>
        <div id="terrafusion" class="dataset">
          <h3><a href="/terrafusion/">Terra Fusion Data Sampler</a></h3>
          <p><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info-sustainability tag link-tag">sustainability</span></p>
          <p>The Terra Basic Fusion dataset is a fused dataset of the original Level 1 radiances
from the five Terra instruments. They have been fully validate to contain the original
Terra instrument Level 1 data. Each Level 1 Terra Basic Fusion file contains one full
Terra orbit of data and is typically 15 – 40 GB in size, depending on how much data was
collected for that orbit. It contains instrument radiance in physical units; radiance
quality indicator; geolocation for each IFOV at its native resolution; sun-view geometry;
bservation time; and other attributes/metadata. It is stored in HDF5, conformed to CF
conventions, and accessible by netCDF-4 enhanced models. It’s naming convention
follows: TERRA_BF_L1B_OXXXX_YYYYMMDDHHMMSS_F000_V000.h5. A concise description of the
dataset, along with links to complete documentation and available software tools, can
be found on the Terra Fusion project page: <a href="https://terrafusion.web.illinois.edu">https://terrafusion.web.illinois.edu</a>.</br></br>Terra is the flagship satellite of NASA’s Earth Observing System (EOS). It was launched
into orbit on December 18, 1999 and carries five instruments. These are the
Moderate-resolution Imaging Spectroradiometer (MODIS), the Multi-angle Imaging
SpectroRadiometer (MISR), the Advanced Spaceborne Thermal Emission and Reflection
Radiometer (ASTER), the Clouds and Earth’s Radiant Energy System (CERES), and the
Measurements of Pollution in the Troposphere (MOPITT).</br></br>The Terra Basic Fusion dataset is an easy-to-access record of the Level 1 radiances
for instruments on...</p>
          <p><a href="/terrafusion/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/TerraFusion" class="" target="_none">TerraFusion GitHub</a> by University of Illinois</li></a>
          </li>
          <li>
            <a href="https://go.illinois.edu/terra-fusion-doc" class="" target="_none">Basic Terra fusion product algorithm theoretical basis and data specifications</a> by <a href="http://hdl.handle.net/2142/105967" target="_none">Zhao, Guangu; Yang, Muqun; Clipp, Landon; Gao, Yizhao; Lee, Joe H.</a></li></a>
          </li>
        </ul>
        <p><a href="/terrafusion/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="uk-met-office" class="dataset">
          <h3><a href="/uk-met-office/">UK Met Office Atmospheric Deterministic and Probabilistic Forecasts</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info-sustainability tag link-tag">sustainability</span><span class="label label-info tag link-tag">weather</span></p>
          <p>Meteorological data reusers now have an exciting opportunity to sample, experiment and evaluate
Met Office atmospheric model data, whilst also experiencing a transformative method of requesting
data via Restful APIs on AWS.For information about the data see the <a href="https://www.metoffice.gov.uk/services/data/met-office-data-for-reuse/discovery">Met Office website</a>.
For examples of using the data check out the <a href="https://github.com/MetOffice/aws-earth-examples">examples repository</a>.
If you need help and support using the data please raise an issue on the examples repository. <strong>Please note:</strong> Met Office continuously improves and updates its operational forecast models.
Our last update became effective 04/12/2019. Please find the detail...</p>
          <p><a href="/uk-met-office/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/MetOffice/aws-earth-examples/blob/master/examples/1.%20Getting%20Started.ipynb" class="" target="_none">Met Office AWS Earth data - Getting Started</a> by Jacob Tomlinson</li></a>
          </li>
          <li>
            <a href="https://github.com/MetOffice/aws-earth-examples/blob/master/examples/2.%20Subscribing%20to%20data.ipynb" class="" target="_none">Met Office AWS Earth data - Subscribing to data</a> by Jacob Tomlinson</li></a>
          </li>
        </ul>
        <p><a href="/uk-met-office/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="1000-genomes" class="dataset">
          <h3><a href="/1000-genomes/">1000 Genomes</a></h3>
          <p><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span></p>
          <p>The 1000 Genomes Project is an international collaboration which has established the most detailed catalogue of human genetic variation, including SNPs, structural variants, and their haplotype context. The final phase of the project sequenced more than 2500 individuals from 26 different populations around the world and produced an integrated set of phased haplotypes with more than 80 million variants for these individuals.</p>
          <p><a href="/1000-genomes/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://aws.amazon.com/blogs/big-data/exploratory-data-analysis-of-genomic-datasets-using-adam-and-mango-with-apache-spark-on-amazon-emr/" class="aws-link" target="_none">Exploratory data analysis of genomic datasets using ADAM and Mango with Apache Spark on Amazon EMR</a> by <a href="https://research.eecs.berkeley.edu/~akmorrow/" target="_none">Alyssa Marrow</a></li></a>
          </li>
        </ul>
        <p><a href="/1000-genomes/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="aws-igenomes" class="dataset">
          <h3><a href="/aws-igenomes/">AWS iGenomes</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">Caenorhabditis elegans</span><span class="label label-info tag link-tag">Danio rerio</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">Homo sapiens</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">Mus musculus</span><span class="label label-info tag link-tag">Rattus norvegicus</span><span class="label label-info tag link-tag">reference index</span></p>
          <p>Common reference genomes hosted on AWS S3. Can be used when aligning and analysing raw DNA sequencing data.</p>
          <p><a href="/aws-igenomes/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="http://nf-co.re/" class="" target="_none">nf-core analysis pipelines</a> by <a href="http://phil.ewels.co.uk/" target="_none">Phil Ewels</a></li></a>
          </li>
        </ul>
        <p><a href="/aws-igenomes/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="allen-brain-observatory" class="dataset">
          <h3><a href="/allen-brain-observatory/">Allen Brain Observatory - Visual Coding AWS Public Data Set</a></h3>
          <p><span class="label label-info tag link-tag">electrophysiology</span><span class="label label-info tag link-tag">image processing</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">Mus musculus</span><span class="label label-info tag link-tag">neurobiology</span><span class="label label-info tag link-tag">neuroimaging</span><span class="label label-info tag link-tag">signal processing</span></p>
          <p>The Allen Brain Observatory – Visual Coding is a large-scale, standardized survey of physiological activity across the mouse visual cortex, hippocampus, and thalamus. It includes datasets collected with both two-photon imaging and Neuropixels probes, two complementary techniques for measuring the activity of neurons in vivo. The two-photon imaging dataset features visually evoked calcium responses from GCaMP6-expressing neurons in a range of cortical layers, visual areas, and Cre lines. The Neuropixels dataset features spiking activity from distributed cortical and subcortical brain regions, c...</p>
          <p><a href="/allen-brain-observatory/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/AllenInstitute/AllenSDK/wiki/Use-the-Allen-Brain-Observatory-%E2%80%93-Visual-Coding-on-AWS" class="" target="_none">Use the Allen Brain Observatory – Visual Coding on AWS</a> by <a href="https://twitter.com/AllenInstitute" target="_none">Nika Keller, David Feng</a></li></a>
          </li>
        </ul>
        <p><a href="/allen-brain-observatory/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="answer-reformulation" class="dataset">
          <h3><a href="/answer-reformulation/">Answer Reformulation</a></h3>
          <p><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">natural language processing</span></p>
          <p>Original StackExchange answers and their voice-friendly Reformulation.</p>
          <p><a href="/answer-reformulation/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www2020.thewebconf.org/" class="" target="_none">Voice-based Reformulation of Community Answers</a> by Simone Filice, Nachshon Cohen &amp; David Carmel</li></a>
          </li>
        </ul>
        <p><a href="/answer-reformulation/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="asr-error-robustness" class="dataset">
          <h3><a href="/asr-error-robustness/">Automatic Speech Recognition (ASR) Error Robustness</a></h3>
          <p><span class="label label-info tag link-tag">deep learning</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">natural language processing</span><span class="label label-info tag link-tag">speech recognition</span></p>
          <p>Sentence classification datatasets with ASR Errors.</p>
          <p><a href="/asr-error-robustness/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://assets.amazon.science/64/94/639ae0c44890837b0f1fbf11ef77/using-phoneme-representations-to-build-predictive-models-robust-to-asr-errors.pdf" class="" target="_none">Using Phoneme Representations to Build Predictive Models Robust to ASR Errors</a> by <a href="https://anjiefang.github.io" target="_none">Anjie Fang, Simone Filice, Nut Limsopatham and Oleg Rokhlenko</a></li></a>
          </li>
        </ul>
        <p><a href="/asr-error-robustness/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="wrf-cmip6" class="dataset">
          <h3><a href="/wrf-cmip6/">CMIP6 GCMs downscaled using WRF</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">atmosphere</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">model</span><span class="label label-info tag link-tag">oceans</span><span class="label label-info tag link-tag">simulations</span><span class="label label-info tag link-tag">weather</span></p>
          <p>High-resolution historical and future climate simulations from 1980-2100</p>
          <p><a href="/wrf-cmip6/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://nbviewer.jupyter.org/urls/wrf-cmip6-noversioning.s3.amazonaws.com/Example_notebook/Notebook_example.ipynb" class="" target="_none">Jupyter Notebook Example</a> by Stefan Rahimi</li></a>
          </li>
        </ul>
        <p><a href="/wrf-cmip6/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="ncbi-covid-19" class="dataset">
          <h3><a href="/ncbi-covid-19/">COVID-19 Genome Sequence Dataset</a></h3>
          <p><span class="label label-info tag link-tag">bam</span><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">coronavirus</span><span class="label label-info tag link-tag">COVID-19</span><span class="label label-info tag link-tag">cram</span><span class="label label-info tag link-tag">fastq</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">health</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">MERS</span><span class="label label-info tag link-tag">SARS</span><span class="label label-info tag link-tag">STRIDES</span><span class="label label-info tag link-tag">transcriptomics</span><span class="label label-info tag link-tag">virus</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>A centralized sequence repository for all records containing sequence associated with the novel corona virus (SARS-CoV-2) submitted to the National Center for Biotechnology Information (NCBI) Sequence Read Archive (SRA). Included are both the original sequences submitted by the principal investigator as well as SRA-processed sequences that require the SRA Toolkit for analysis. Additionally, submitter provided metadata included in associated BioSample and BioProject records is available alongside NCBI calculated data, such k-mer based taxonomy analysis results, contiguous assemblies (contigs) a...</p>
          <p><a href="/ncbi-covid-19/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/sra/docs/sra-aws-download/" class="" target="_none">Download SRA sequence data using Amazon Web Services (AWS)</a> by NCBI SRA</li></a>
          </li>
        </ul>
        <p><a href="/ncbi-covid-19/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="cell-painting-image-collection" class="dataset">
          <h3><a href="/cell-painting-image-collection/">Cell Painting Image Collection</a></h3>
          <p><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">cell imaging</span><span class="label label-info tag link-tag">cell painting</span><span class="label label-info tag link-tag">fluorescence imaging</span><span class="label label-info tag link-tag">high-throughput imaging</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">microscopy</span></p>
          <p>The Cell Painting Image Collection is a collection of freely
downloadable microscopy image sets. Cell Painting is an
unbiased high throughput imaging assay used to analyze
perturbations in cell models. In addition to the images
themselves, each set includes a description of the biological
application and some type of &quot;ground truth&quot; (expected results).
Researchers are encouraged to use these image sets as reference
points when developing, testing, and publishing new image
analysis algorithms for the life sciences. We hope that the
this data set will lead to a better understanding of w...</p>
          <p><a href="/cell-painting-image-collection/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/cytodata/cytodata-hackathon-2018/tree/master/cytodata-toolkit/" class="" target="_none">Example submission for the 2018 CytoData Hackathon (in R and Python)</a> by <a href="broadinstitute.org" target="_none">Juan Caicedo, Tim Becker</a></li></a>
          </li>
        </ul>
        <p><a href="/cell-painting-image-collection/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="covers-br" class="dataset">
          <h3><a href="/covers-br/">CoversBR</a></h3>
          <p><span class="label label-info tag link-tag">copyright monitoring</span><span class="label label-info tag link-tag">cover song identification</span><span class="label label-info tag link-tag">live song identification</span><span class="label label-info tag link-tag">music</span><span class="label label-info tag link-tag">music features dataset</span><span class="label label-info tag link-tag">music information retrieval</span><span class="label label-info tag link-tag">music recognition</span></p>
          <p>CoversBR is the first large audio database with, predominantly, Brazilian music for the tasks of Covers Song
Identification (CSI) and Live Song Identifications (LSI). Due to copyright restrictions audios of
the songs cannot be made available, however metadata and files of features have public access. Audio
streamings captured from radio and TV channels for the live song identification task will be made public.
CoversBR is composed of metadata and features extracted from 102298 songs, distributed in 26366
groups of covers/versions, with an average of 3.88 versions per group. The entire collecti...</p>
          <p><a href="/covers-br/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/SPLab-IT/CoversBR#using-the-dataset---tutorial" class="" target="_none">Using the (CoversBR) dataset</a> by <a href="https://github.com/SPLab-IT/CoversBR" target="_none">Dirceu Silva, Atila Xavier, Edgard Moraes, Marco Grivet and Fernando Perdigão</a></li></a>
          </li>
        </ul>
        <p><a href="/covers-br/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="odp-noaa-nesdis-ncei-csb" class="dataset">
          <h3><a href="/odp-noaa-nesdis-ncei-csb/">Crowdsourced Bathymetry</a></h3>
          <p><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">oceans</span><span class="label label-info-sustainability tag link-tag">sustainability</span></p>
          <p>Community provided bathymetry data collected in collaboration with the International Hydrographic Organization.</p>
          <p><a href="/odp-noaa-nesdis-ncei-csb/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/dneufeldcu/notebooks/blob/main/esipCSBfinal.ipynb" class="" target="_none">Crowdsourced Bathymetry Data (CSB) Visualization</a> by David Neufeld</li></a>
          </li>
        </ul>
        <p><a href="/odp-noaa-nesdis-ncei-csb/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="dialoglue" class="dataset">
          <h3><a href="/dialoglue/">DialoGLUE: A Natural Language Understanding Benchmark for Task-Oriented Dialogue</a></h3>
          <p><span class="label label-info tag link-tag">conversation data</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">natural language processing</span></p>
          <p>This bucket contains the checkpoints used to reproduce the baseline results reported in the DialoGLUE benchmark hosted
on EvalAI (<a href="https://evalai.cloudcv.org/web/challenges/challenge-page/708/overview">https://evalai.cloudcv.org/web/challenges/challenge-page/708/overview</a>). The associated scripts for using the checkpoints are located here:
<a href="https://github.com/alexa/dialoglue">https://github.com/alexa/dialoglue</a>. The associated paper describing the benchmark and checkpoints is here: <a href="https://arxiv.org/abs/2009.13570">https://arxiv.org/abs/2009.13570</a>.
The provided checkpoints include the CONVBERT model, a BERT-esque model trained on a large open-domain conversational
dataset. It also includes the CONVBERT-DG and BERT-DG checkpoints descri...</p>
          <p><a href="/dialoglue/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://arxiv.org/abs/2009.13570" class="" target="_none">DialoGLUE: A Natural Language Understanding Benchmark for Task-Oriented Dialogue</a> by Shikib Mehri, Mihail Eric, Dilek Hakkani-Tur</li></a>
          </li>
        </ul>
        <p><a href="/dialoglue/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="topical-chat-enriched" class="dataset">
          <h3><a href="/topical-chat-enriched/">Enriched Topical-Chat Dataset for Knowledge-Grounded Dialogue Systems</a></h3>
          <p><span class="label label-info tag link-tag">conversation data</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">natural language processing</span></p>
          <p>This dataset provides extra annotations on top of the publicly released
Topical-Chat dataset(<a href="https://github.com/alexa/Topical-Chat">https://github.com/alexa/Topical-Chat</a>) which will help in reproducing the results in our paper
&quot;Policy-Driven Neural Response Generation for Knowledge-Grounded Dialogue Systems&quot; (<a href="https://arxiv.org/abs/2005.12529?context=cs.CL">https://arxiv.org/abs/2005.12529?context=cs.CL</a>). 
The dataset contains 5 files: train.json, valid_freq.json, valid_rare.json, test_freq.json and test_rare.json. 
Each of these files will have additional annotations on top of the original Topical-Chat dataset.
These specific annotations are: dialogue act annotations a...</p>
          <p><a href="/topical-chat-enriched/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://arxiv.org/abs/2005.12529?context&#x3D;cs.CL" class="" target="_none">Policy-Driven Neural Response Generation for Knowledge-Grounded Dialogue Systems</a> by Behnam Hedayatnia, Karthik Gopalakrishnan, Seokhwan Kim, Yang Liu, Mihail Eric &amp; Dilek Hakkani-Tur</li></a>
          </li>
        </ul>
        <p><a href="/topical-chat-enriched/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="ford-multi-av-seasonal" class="dataset">
          <h3><a href="/ford-multi-av-seasonal/">Ford Multi-AV Seasonal Dataset</a></h3>
          <p><span class="label label-info tag link-tag">autonomous vehicles</span><span class="label label-info tag link-tag">computer vision</span><span class="label label-info tag link-tag">lidar</span><span class="label label-info tag link-tag">mapping</span><span class="label label-info tag link-tag">robotics</span><span class="label label-info tag link-tag">transportation</span><span class="label label-info tag link-tag">urban</span><span class="label label-info tag link-tag">weather</span></p>
          <p>This research presents a challenging multi-agent seasonal dataset collected by a fleet of Ford autonomous vehicles at different days and times during 2017-18. The vehicles The vehicles were manually driven on an average route of 66 km in Michigan that included a mix of driving scenarios like the Detroit Airport, freeways, city-centres, university campus and suburban neighbourhood, etc. Each vehicle used in this data collection  is a Ford Fusion outfitted with an Applanix POS-LV inertial measurement unit (IMU), four HDL-32E Velodyne 3D-lidar scanners, 6 Point Grey 1.3 MP Cameras arranged on the...</p>
          <p><a href="/ford-multi-av-seasonal/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/Ford/AVData" class="" target="_none">Ford AV Dataset Tutorial</a> by Ford Motor Company</li></a>
          </li>
        </ul>
        <p><a href="/ford-multi-av-seasonal/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="gatk-test-data" class="dataset">
          <h3><a href="/gatk-test-data/">GATK Test Data</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span></p>
          <p>The GATK test data resource bundle is a collection of files for resequencing human genomic data with the
Broad Institute&#39;s <a href="https://software.broadinstitute.org/gatk/">Genome Analysis Toolkit (GATK)</a>.</p>
          <p><a href="/gatk-test-data/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://docs.opendata.aws/genomics-workflows/orchestration/cromwell/cromwell-examples/#real-world-example-haplotypecaller" class="aws-link" target="_none">Genomics Workflows on AWS - Cromwell on AWS</a> by <a href="https://www.linkedin.com/in/lee-pang-a039a26/" target="_none">W. Lee Pang</a></li></a>
          </li>
        </ul>
        <p><a href="/gatk-test-data/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="spatial-ucr" class="dataset">
          <h3><a href="/spatial-ucr/">Geosnap Data, Center for Geospatial Sciences</a></h3>
          <p><span class="label label-info tag link-tag">demographics</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">urban</span></p>
          <p>This bucket contains multiple datasets (as Quilt packages) created by the
Center for Geospatial Sciences (CGS) at the University of California-Riverside.
The data in this bucket contains the following:1) Tabular and geographic data from the US Census
2) Land Cover imagery collected from <a href="https://www.mrlc.gov/">Multi-Resolution Land Characteristics Consortium</a>
3) Road network data processed from OpenStreetMap</p>
          <p><a href="/spatial-ucr/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://spatialucr.github.io/geosnap-guide/home" class="" target="_none">Geosnap User Guide</a> by <a href="https://knaaptime.com" target="_none">Eli Knaap</a></li></a>
          </li>
        </ul>
        <p><a href="/spatial-ucr/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="hcmi-cmdc" class="dataset">
          <h3><a href="/hcmi-cmdc/">Human Cancer Models Initiative (HCMI) Cancer Model Development Center</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">STRIDES</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>The Human Cancer Models Initiative (HCMI) is an international consortium that is generating novel,
next-generation, tumor-derived culture models annotated with genomic and clinical data.
HCMI-developed models and related data are available as a community resource. The NCI is
contributing to the initiative by supporting four Cancer Model Development Centers (CMDCs).  CMDCs
are tasked with producing next-generation cancer models from clinical samples. The cancer models
include tumor types that are rare, originate from patients from underrepresented populations, lack
precision therapy, or lack ca...</p>
          <p><a href="/hcmi-cmdc/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://portal.gdc.cancer.gov/projects/HCMI-CMDC" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></li></a>
          </li>
        </ul>
        <p><a href="/hcmi-cmdc/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="hpgp-data" class="dataset">
          <h3><a href="/hpgp-data/">Human PanGenomics Project</a></h3>
          <p><span class="label label-info tag link-tag">cram</span><span class="label label-info tag link-tag">fast5</span><span class="label label-info tag link-tag">fastq</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span></p>
          <p>This dataset includes sequencing data, assemblies, and analyses for the offspring of ten parent-offspring trios.</p>
          <p><a href="/hpgp-data/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.nature.com/articles/s41587-020-0503-6" class="" target="_none">Nanopore sequencing and the Shasta toolkit enable efficient de novo assembly of eleven human genomes</a> by Shafin et al (2020)</li></a>
          </li>
        </ul>
        <p><a href="/hpgp-data/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="humor-detection" class="dataset">
          <h3><a href="/humor-detection/">Humor Detection from Product Question Answering Systems</a></h3>
          <p><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">natural language processing</span></p>
          <p>This dataset provides labeled humor detection from product question answering systems.
The dataset contains 3 csv files: <a href="https://humor-detection-pds.s3-us-west-2.amazonaws.com/Humorous.csv">Humorous.csv</a> 
containing the humorous product questions, 
<a href="https://humor-detection-pds.s3-us-west-2.amazonaws.com/Non-humorous-unbiased.csv">Non-humorous-unbiased.csv</a> containing 
the non-humorous prodcut questions from the same products as the humorous one, and, 
<a href="https://humor-detection-pds.s3-us-west-2.a...</p>
          <p><a href="/humor-detection/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://sigir.org/sigir2020/" class="" target="_none">Humor Detection in Product Question Answering Systems.</a> by Yftah Ziser, Elad Kravi &amp; David Carmel</li></a>
          </li>
        </ul>
        <p><a href="/humor-detection/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="ideam-radares" class="dataset">
          <h3><a href="/ideam-radares/">IDEAM - Colombian Radar Network</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">natural resource</span><span class="label label-info-sustainability tag link-tag">sustainability</span><span class="label label-info tag link-tag">weather</span></p>
          <p>Historical and one-day delay data from the IDEAM radar network.</p>
          <p><a href="/ideam-radares/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="http://www.pronosticosyalertas.gov.co/documents/78690/112207077/AWS_RADARESCOL.pdf/aafc181d-0b6a-47e0-97e9-9b6290ac84d6" class="" target="_none">Guia de como explorar y plotear los archivos de radar utilizando el lenguaje de programación Python</a> by IDEAM</li></a>
          </li>
        </ul>
        <p><a href="/ideam-radares/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="lofar-elais-n1" class="dataset">
          <h3><a href="/lofar-elais-n1/">LOFAR ELAIS-N1 cycle 2 observations on AWS</a></h3>
          <p><span class="label label-info tag link-tag">astronomy</span><span class="label label-info tag link-tag">imaging</span><span class="label label-info tag link-tag">survey</span></p>
          <p>These data correspond to the <a href="http://www.lofar.org/">International LOFAR Telescope</a> observations of the sky field ELAIS-N1 (16:10:01 +54:30:36) during the cycle 2 of observations. There are 11 runs of about 8 hours each plus the corresponding observation of the calibration targets before and after the target field. The data are measurement sets (<a href="https://casa.nrao.edu/Memos/229.html">MS</a>) containing the cross-correlated data and metadata divided in 371 frequency sub-bands per target centred at ~150 MHz.</p>
          <p><a href="/lofar-elais-n1/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.lofarcloud.uk" class="" target="_none">Calibration of LOFAR ELAIS-N1 data in the Amazon cloud</a> by <a href="https://github.com/nudomarinero" target="_none">J. Sabater</a></li></a>
          </li>
        </ul>
        <p><a href="/lofar-elais-n1/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="noaa-gfs-bdp-pds" class="dataset">
          <h3><a href="/noaa-gfs-bdp-pds/">NOAA Global Forecast System (GFS)</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info-sustainability tag link-tag">sustainability</span><span class="label label-info tag link-tag">weather</span></p>
          <p>The Global Forecast System (GFS) is a weather forecast model produced
by the National Centers for Environmental Prediction (NCEP). Dozens of
atmospheric and land-soil variables are available through this dataset,
from temperatures, winds, and precipitation to soil moisture and
atmospheric ozone concentration. The entire globe is covered by the GFS
at a base horizontal resolution of 18 miles (28 kilometers) between grid
points, which is used by the operational forecasters who predict weather
out to 16 days in the future. Horizontal resolution drops to 44 miles
(70 kilometers) between grid point for forecasts between one week and two
weeks.
<br/>
<br/>
The NOAA Global Forecast Systems (GFS) Warm Start Initial Conditions are 
produced by the National Centers for Environmental Prediction Center (NCEP) 
to run operational deterministic medium-range numerical weather predictions.<br>The GFS is built with the GFDL Finite-Volume Cubed-Sphere Dynamical Core (FV3) 
and the Grid-Point Statistical Interpolation (GSI) data assimilation system.<br>Please visit the links below in the Documentation section to find more details 
about the model and the data assimilation systems.   The current operational 
GFS is run at 64 layers in the vertical extending from th</br/>...</p>
          <p><a href="/noaa-gfs-bdp-pds/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://noaa-gfs-warmstart-pds.s3.amazonaws.com/GFS+Warm+Restart+Files.v1.pdf" class="" target="_none">GFS Warm Restart Files Additional Information</a> by Fanglin Yang</li></a>
          </li>
        </ul>
        <p><a href="/noaa-gfs-bdp-pds/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="noaa-ndfd" class="dataset">
          <h3><a href="/noaa-ndfd/">NOAA National Digital Forecast Database (NDFD)</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info-sustainability tag link-tag">sustainability</span><span class="label label-info tag link-tag">weather</span></p>
          <p>The National Digital Forecast Database (NDFD) is a suite of gridded forecasts of sensible weather elements (e.g., cloud cover, maximum temperature).  Forecasts prepared by NWS field offices working in collaboration with the National Centers for Environmental Prediction (NCEP) are combined in the NDFD to create a seamless mosaic of digital forecasts from which operational NWS products are generated. The most recent data is under the opnl and expr prefixes. A copy is also placed under the wmo prefix. The wmo prefix is structured like so: wmo/&lt;parameter&gt;/&lt;year&gt;/&lt;month&gt;/&lt;day&g...</p>
          <p><a href="/noaa-ndfd/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://noaa-ndfd-pds.s3.amazonaws.com/NDFDelem_fullres_201906.xls" class="" target="_none">NDFD Product Spreadsheet (excel file)</a> by NOAA MDL</li></a>
          </li>
        </ul>
        <p><a href="/noaa-ndfd/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="nj-imagery" class="dataset">
          <h3><a href="/nj-imagery/">New Jersey Statewide Digital Aerial Imagery Catalog</a></h3>
          <p><span class="label label-info tag link-tag">aerial imagery</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">imaging</span><span class="label label-info tag link-tag">mapping</span></p>
          <p>The New Jersey Office of GIS, NJ Office of Information Technology manages a series of 11 digital orthophotography and scanned aerial photo maps collected at various years ranging from 1930 to 2017. Each year’s worth of imagery are available as Cloud Optimized GeoTIFF (COG) files and some years are available as compressed MrSID and/or JP2 files.  Additionally, each year of imagery is organized into a tile grid scheme covering the entire geography of New Jersey.  Many years share the same tiling grid while others have unique grids as defined by the project at the time.</p>
          <p><a href="/nj-imagery/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://njogis-imagery.s3-us-west-2.amazonaws.com/NJ+Imagery+Change.pdf" class="" target="_none">Visualize Imagery Changes</a> by <a href="https://njgin.nj.gov/njgin/edata/imagery/index.html" target="_none">stephanie.bosits@tech.nj.gov</a></li></a>
          </li>
        </ul>
        <p><a href="/nj-imagery/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="nj-lidar" class="dataset">
          <h3><a href="/nj-lidar/">New Jersey Statewide LiDAR</a></h3>
          <p><span class="label label-info tag link-tag">elevation</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">lidar</span><span class="label label-info tag link-tag">mapping</span></p>
          <p>Elevation datasets in New Jersey have been collected over several years as several
discrete projects.  Each project covers a geographic area, which is a subsection of
the entire state, and has differing specifications based on the available technology
at the time and project budget.  The geographic extent of one project may overlap that
of a neighboring project. Each of the 18 projects contains deliverable products such
as LAS (Lidar point cloud) files, unclassified/classified, tiled to cover project area;
relevant metadata records or documents, most adhering to the Federal Geographic Data
Com...</p>
          <p><a href="/nj-lidar/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://njogis-elevation.s3-us-west-2.amazonaws.com/NJ+3D+Visualization.pdf" class="" target="_none">3D Visualization</a> by <a href="https://njgin.nj.gov/njgin/edata/elevation/index.html" target="_none">stephanie.bosits@tech.nj.us</a></li></a>
          </li>
        </ul>
        <p><a href="/nj-lidar/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="ocmr_data" class="dataset">
          <h3><a href="/ocmr_data/">Ohio State Cardiac MRI Raw Data (OCMR)</a></h3>
          <p><span class="label label-info tag link-tag">Homo sapiens</span><span class="label label-info tag link-tag">image processing</span><span class="label label-info tag link-tag">imaging</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">magnetic resonance imaging</span><span class="label label-info tag link-tag">signal processing</span></p>
          <p>OCMR is an open-access repository that provides multi-coil k-space data for cardiac cine.  The fully sampled MRI datasets are intended for quantitative comparison and evaluation of image reconstruction methods. The free-breathing, prospectively undersampled datasets are intended to evaluate their performance and generalizability qualitatively.</p>
          <p><a href="/ocmr_data/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/MRIOSU/OCMR/blob/master/Python/example_ocmr.ipynb" class="" target="_none">OCMR Tutorial</a> by <a href="https://cmr.engineering.osu.edu/people/trainees" target="_none">Chong Chen</a></li></a>
          </li>
        </ul>
        <p><a href="/ocmr_data/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="ont-open-data" class="dataset">
          <h3><a href="/ont-open-data/">Oxford Nanopore Technologies Benchmark Datasets</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">fast5</span><span class="label label-info tag link-tag">fastq</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">Homo sapiens</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>The ont-open-data registry provides reference sequencing data from Oxford Nanopore Technologies to support, 1) Exploration of the characteristics of nanopore sequence data. 2) Assessment and reproduction of performance benchmarks 3) Development of tools and methods. The data deposited showcases DNA sequences from a representative subset of sequencing chemistries. The datasets correspond to publicly-available reference samples (e.g. GM24385 as reference human). Raw data are provided with metadata and scripts to describe sample and data provenance.</p>
          <p><a href="/ont-open-data/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://nanoporetech.github.io/ont-open-datasets/tutorials" class="" target="_none">ONT Dataset Tutorials</a> by <a href="https://labs.epi2me.io/" target="_none">EPI2MELabs</a></li></a>
          </li>
        </ul>
        <p><a href="/ont-open-data/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="qiime2" class="dataset">
          <h3><a href="/qiime2/">QIIME 2 User Tutorial Datasets</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">denoising</span><span class="label label-info tag link-tag">ecosystems</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">health</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">microbiome</span><span class="label label-info tag link-tag">statistics</span></p>
          <p>QIIME 2 is a powerful, extensible, and decentralized microbiome analysis package with a focus on data and analysis transparency. QIIME 2 enables researchers to start an analysis with raw DNA sequence data and finish with publication-quality figures and statistical results. This dataset contains the user docs (and related datasets) for QIIME 2.</p>
          <p><a href="/qiime2/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://docs.qiime2.org/2018.11/install/virtual/aws/" class="" target="_none">Installing QIIME 2 using Amazon Web Services</a> by <a href="https://github.com/qiime2/docs/graphs/contributors" target="_none">The QIIME 2 Development Team</a></li></a>
          </li>
        </ul>
        <p><a href="/qiime2/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="silam" class="dataset">
          <h3><a href="/silam/">SILAM Air Quality</a></h3>
          <p><span class="label label-info tag link-tag">air quality</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info-sustainability tag link-tag">sustainability</span><span class="label label-info tag link-tag">weather</span></p>
          <p>Air Quality is a global SILAM atmospheric composition and air quality forecast performed on a daily basis for &gt; 100 species and covering the troposphere and the stratosphere. The output produces 3D concentration fields and aerosol optical thickness. The data are unique: 20km resolution for global AQ models is unseen worldwide.</p>
          <p><a href="/silam/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/fmidev/opendata-resources/tree/master/examples/python" class="" target="_none">Simple examples</a> by Roope Tervo</li></a>
          </li>
        </ul>
        <p><a href="/silam/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="safecast" class="dataset">
          <h3><a href="/safecast/">Safecast</a></h3>
          <p><span class="label label-info tag link-tag">air quality</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">radiation</span><span class="label label-info-sustainability tag link-tag">sustainability</span></p>
          <p>An ongoing collection of radiation and air quality measurements taken by devices involved in the Safecast project.</p>
          <p><a href="/safecast/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="http://safecast.org/tilemap/" class="" target="_none">Safecast Map</a> by <a href="https://github.com/frangible" target="_none">Nick Dolezal</a></li></a>
          </li>
        </ul>
        <p><a href="/safecast/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="sentinel-s2-l2a-mosaic-120" class="dataset">
          <h3><a href="/sentinel-s2-l2a-mosaic-120/">Sentinel-2 L2A 120m Mosaic</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">natural resource</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info-sustainability tag link-tag">sustainability</span></p>
          <p>Sentinel-2 L2A 120m mosaic is a derived product, which contains best pixel values for 10-daily periods, modelled by removing the cloudy pixels and then performing interpolation among remaining values. As there are some parts of the world, which have lengthy cloudy periods, clouds might be remaining in some parts. The actual modelling script is available <a href="https://sentinel-hub.github.io/custom-scripts/sentinel-2/interpolated_time_series/">here</a>.</p>
          <p><a href="/sentinel-s2-l2a-mosaic-120/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/sentinel-hub/public-collections/blob/main/collections/sentinel-s2-l2a-mosaic-120.md" class="" target="_none">Sentinel Hub WMS/WMTS/WCS Service and Process API</a> by <a href="http://www.sinergise.com/" target="_none">Sinergise</a></li></a>
          </li>
        </ul>
        <p><a href="/sentinel-s2-l2a-mosaic-120/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="speedtest-global-performance" class="dataset">
          <h3><a href="/speedtest-global-performance/">Speedtest by Ookla Global Fixed and Mobile Network Performance Maps</a></h3>
          <p><span class="label label-info tag link-tag">analytics</span><span class="label label-info tag link-tag">broadband</span><span class="label label-info tag link-tag">cities</span><span class="label label-info tag link-tag">civic</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">global</span><span class="label label-info tag link-tag">government spending</span><span class="label label-info tag link-tag">infrastructure</span><span class="label label-info tag link-tag">internet</span><span class="label label-info tag link-tag">mapping</span><span class="label label-info tag link-tag">network traffic</span><span class="label label-info tag link-tag">parquet</span><span class="label label-info tag link-tag">regulatory</span><span class="label label-info tag link-tag">telecommunications</span><span class="label label-info tag link-tag">tiles</span></p>
          <p>Global fixed broadband and mobile (cellular) network performance, allocated to zoom level 16 web mercator tiles (approximately 610.8 meters by 610.8 meters at the equator). Data is provided in both Shapefile format as well as Apache Parquet with geometries represented in Well Known Text (WKT) projected in EPSG:4326. Download speed, upload speed, and latency are collected via the Speedtest by Ookla applications for Android and iOS and averaged for each tile. Measurements are filtered to results containing GPS-quality location accuracy.</p>
          <p><a href="/speedtest-global-performance/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.speedtest.net/insights/blog/best-ookla-open-data-projects-2021/" class="" target="_none">New Year, Great Data: The Best Ookla Open Data Projects We’ve Seen So Far</a> by Katie Jolly</li></a>
          </li>
        </ul>
        <p><a href="/speedtest-global-performance/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="tabula-muris" class="dataset">
          <h3><a href="/tabula-muris/">Tabula Muris</a></h3>
          <p><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">encyclopedic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">health</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">medicine</span></p>
          <p>Tabula Muris is a compendium of single cell transcriptomic data from the model organism <em>Mus musculus</em> comprising more than 100,000 cells from 20 organs and tissues. These data represent a new resource for cell biology, reveal gene expression in poorly characterized cell populations, and allow for direct and controlled comparison of gene expression in cell types shared between tissues, such as T-lymphocytes and endothelial cells from different anatomical locations. Two distinct technical approaches were used for most organs: one approach, microfluidic droplet-based 3’-end counting, enabled the s...</p>
          <p><a href="/tabula-muris/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/30283141" class="" target="_none">Single-cell transcriptomics of 20 mouse organs creates a Tabula Muris.</a> by <a href="https://tabula-muris.ds.czbiohub.org/" target="_none">Tabula Muris Consortium (2019)</a></li></a>
          </li>
        </ul>
        <p><a href="/tabula-muris/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="amazon-reviews-ml" class="dataset">
          <h3><a href="/amazon-reviews-ml/">The Multilingual Amazon Reviews Corpus</a></h3>
          <p><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">natural language processing</span></p>
          <p>We present a collection of Amazon reviews specifically designed to aid research in multilingual text classification. The dataset contains reviews in English, Japanese, German, French, Chinese and Spanish, collected between November 1, 2015 and November 1, 2019. Each record in the dataset contains the review text, the review title, the star rating, an anonymized reviewer ID, an anonymized product ID and the coarse-grained product category (e.g. &#39;books&#39;, &#39;appliances&#39;, etc.)</p>
          <p><a href="/amazon-reviews-ml/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://arxiv.org/abs/2010.02573" class="" target="_none">The Multilingual Amazon Reviews Corpus</a> by Phillip Keung, Yichao Lu, György Szarvas, Noah A. Smith</li></a>
          </li>
        </ul>
        <p><a href="/amazon-reviews-ml/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="tess" class="dataset">
          <h3><a href="/tess/">Transiting Exoplanet Survey Satellite (TESS)</a></h3>
          <p><span class="label label-info tag link-tag">astronomy</span></p>
          <p>The Transiting Exoplanet Survey Satellite (TESS) is a multi-year survey that will discover exoplanets in orbit around bright stars across the entire sky using high-precision photometry.  The survey will also enable a wide variety of stellar astrophysics, solar system science, and extragalactic variability studies. More information about TESS is available at <a href="https://archive.stsci.edu/tess/">MAST</a> and the <a href="https://heasarc.gsfc.nasa.gov/docs/tess/">TESS Science Support Center</a>.</p>
          <p><a href="/tess/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://mast-labs.stsci.io/2018/12/tess-data-available-on-aws" class="" target="_none">TESS data available on AWS</a> by <a href="https://twitter.com/arfon" target="_none">Arfon Smith</a></li></a>
          </li>
        </ul>
        <p><a href="/tess/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="census-dataworld-pums" class="dataset">
          <h3><a href="/census-dataworld-pums/">U.S. Census ACS PUMS</a></h3>
          <p><span class="label label-info tag link-tag">census</span><span class="label label-info tag link-tag">statistics</span><span class="label label-info tag link-tag">survey</span><span class="label label-info-sustainability tag link-tag">sustainability</span></p>
          <p>U.S. Census Bureau American Community Survey (ACS) Public Use Microdata Sample (PUMS) available in a linked data format using the Resource Description Framework (RDF) data model.</p>
          <p><a href="/census-dataworld-pums/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://docs.data.world/uscensus/#50---getting-started" class="" target="_none">Setting up Blazegraph on EC2</a> by <a href="https://data.world/" target="_none">data.world</a></li></a>
          </li>
        </ul>
        <p><a href="/census-dataworld-pums/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="lab41-sri-voices" class="dataset">
          <h3><a href="/lab41-sri-voices/">Voices Obscured in Complex Environmental Settings (VOiCES)</a></h3>
          <p><span class="label label-info tag link-tag">automatic speech recognition</span><span class="label label-info tag link-tag">denoising</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">speaker identification</span><span class="label label-info tag link-tag">speech processing</span></p>
          <p>VOiCES is a speech corpus recorded in acoustically challenging settings,
using distant microphone recording. Speech was recorded in real rooms with various
acoustic features (reverb, echo, HVAC systems, outside noise, etc.). Adversarial noise,
either television, music, or babble, was concurrently played with clean speech.
Data was recorded using multiple microphones strategically placed
throughout the room. The corpus includes audio recordings, orthographic transcriptions,
and speaker labels.</p>
          <p><a href="/lab41-sri-voices/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/voices18/utilities/blob/master/Using%20VOiCES%20corpus%20tutorial.ipynb" class="" target="_none">Getting started with VOiCES data</a> by M.A. Barrios</li></a>
          </li>
        </ul>
        <p><a href="/lab41-sri-voices/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="xiph-media" class="dataset">
          <h3><a href="/xiph-media/">Xiph.Org Test Media</a></h3>
          <p><span class="label label-info tag link-tag">computer vision</span><span class="label label-info tag link-tag">image processing</span><span class="label label-info tag link-tag">imaging</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">media</span><span class="label label-info tag link-tag">movies</span><span class="label label-info tag link-tag">multimedia</span></p>
          <p>Uncompressed video used for video compression and video processing research.</p>
          <p><a href="/xiph-media/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://media.xiph.org/aws.html" class="" target="_none">Encoding video with AV1 on EC2</a> by <a href="https://www.xiph.org/" target="_none">Thomas Daede</a></li></a>
          </li>
        </ul>
        <p><a href="/xiph-media/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="zinc15" class="dataset">
          <h3><a href="/zinc15/">ZINC Database</a></h3>
          <p><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">chemical biology</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">molecular docking</span><span class="label label-info tag link-tag">pharmaceutical</span><span class="label label-info tag link-tag">protein</span></p>
          <p>3D models for molecular docking screens.</p>
          <p><a href="/zinc15/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="http://zinc15.docking.org" class="" target="_none">ZINC Database</a> by <a href="http://irwinlab.compbio.ucsf.edu" target="_none">John Irwin</a></li></a>
          </li>
        </ul>
        <p><a href="/zinc15/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="ihart" class="dataset">
          <h3><a href="/ihart/">iHART Whole Genome Sequencing Data Set</a></h3>
          <p><span class="label label-info tag link-tag">autism spectrum disorder</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>iHART is the <a href="http://www.thehartwellfoundation.org/">Hartwell Foundation</a>’s Autism Research and Technology Initiative. This release contains whole genome data from over 1000 families with 2 or more children with autism, of which biomaterials were provided by the Autism Genetic Resource Exchange (<a href="http://research.agre.org/">AGRE</a>).</p>
          <p><a href="/ihart/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.cell.com/cell/fulltext/S0092-8674(19)30780-9" class="" target="_none">Inherited and De Novo Genetic Risk for Autism Impacts Shared Networks</a> by Ruzzo et al. (2020)</li></a>
          </li>
        </ul>
        <p><a href="/ihart/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="cse-cic-ids2018" class="dataset">
          <h3><a href="/cse-cic-ids2018/">A Realistic Cyber Defense Dataset (CSE-CIC-IDS2018)</a></h3>
          <p><span class="label label-info tag link-tag">cyber security</span><span class="label label-info tag link-tag">internet</span><span class="label label-info tag link-tag">intrusion detection</span><span class="label label-info tag link-tag">network traffic</span></p>
          <p>This dataset is the result of a collaborative project between the Communications Security Establishment (CSE) and The Canadian Institute for Cybersecurity (CIC) that use the notion of profiles to generate cybersecurity dataset in a systematic manner. It incluides a detailed description of intrusions along with abstract distribution models for applications, protocols, or lower level network entities. The dataset includes seven different attack scenarios, namely Brute-force, Heartbleed, Botnet, DoS, DDoS, Web attacks, and infiltration of the network from inside. The attacking infrastructure incl...</p>
          <p><a href="/cse-cic-ids2018/">Details &rarr;</a></p>
        </div>
        <div id="aev-a2d2" class="dataset">
          <h3><a href="/aev-a2d2/">A2D2: Audi Autonomous Driving Dataset</a></h3>
          <p><span class="label label-info tag link-tag">autonomous vehicles</span><span class="label label-info tag link-tag">computer vision</span><span class="label label-info tag link-tag">deep learning</span><span class="label label-info tag link-tag">lidar</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">mapping</span><span class="label label-info tag link-tag">robotics</span></p>
          <p>An open multi-sensor dataset for autonomous driving research. This dataset comprises semantically segmented images, semantic point clouds, and 3D bounding boxes. In addition, it contains unlabelled 360 degree camera images, lidar, and bus data for three sequences. We hope this dataset will further facilitate active research and development in AI, computer vision, and robotics for autonomous driving.</p>
          <p><a href="/aev-a2d2/">Details &rarr;</a></p>
        </div>
        <div id="sentinel-1-rtc-indigo" class="dataset">
          <h3><a href="/sentinel-1-rtc-indigo/">Analysis Ready Sentinel-1 Backscatter Imagery</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info-sustainability tag link-tag">sustainability</span></p>
          <p>The <a href="https://sentinel.esa.int/web/sentinel/missions/sentinel-1">Sentinel-1 mission</a> is a constellation of
C-band Synthetic Aperature Radar (SAR) satellites from the European Space Agency launched since 2014.
These satellites collect observations of radar backscatter intensity day or night, regardless of the
weather conditions, making them enormously valuable for environmental monitoring.
These radar data have been processed from original Ground Range Detected (GRD) scenes into a Radiometrically
Terrain Corrected, tiled product suitable for analysis. This product is available over the Contiguous United States (CONUS)
since 2017 when Sentinel-1 data becam...</p>
          <p><a href="/sentinel-1-rtc-indigo/">Details &rarr;</a></p>
        </div>
        <div id="cgiardata" class="dataset">
          <h3><a href="/cgiardata/">CCAFS-Climate Data</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">food security</span><span class="label label-info-sustainability tag link-tag">sustainability</span></p>
          <p>High resolution climate data to help assess the impacts of climate change primarily on agriculture. These open access datasets of climate projections will help researchers make climate change impact assessments.</p>
          <p><a href="/cgiardata/">Details &rarr;</a></p>
        </div>
        <div id="fast-ai-coco" class="dataset">
          <h3><a href="/fast-ai-coco/">COCO - Common Objects in Context - fast.ai datasets</a></h3>
          <p><span class="label label-info tag link-tag">computer vision</span><span class="label label-info tag link-tag">deep learning</span><span class="label label-info tag link-tag">machine learning</span></p>
          <p>COCO is a large-scale object detection, segmentation, and captioning dataset.
This is part of the fast.ai datasets collection hosted by AWS for convenience
of fast.ai students. If you use this dataset in your research please cite
arXiv:1405.0312 [cs.CV].</p>
          <p><a href="/fast-ai-coco/">Details &rarr;</a></p>
        </div>
        <div id="molssi-covid19-hub" class="dataset">
          <h3><a href="/molssi-covid19-hub/">COVID-19 Molecular Structure and Therapeutics Hub</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">coronavirus</span><span class="label label-info tag link-tag">COVID-19</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">molecular docking</span><span class="label label-info tag link-tag">pharmaceutical</span></p>
          <p>Aggregating critical information to accelerate drug discovery for the molecular modeling and simulation community.
A community-driven data repository and curation service for molecular structures, models, therapeutics, and
simulations related to computational research related to therapeutic opportunities for COVID-19
(caused by the SARS-CoV-2 coronavirus).</p>
          <p><a href="/molssi-covid19-hub/">Details &rarr;</a></p>
        </div>
        <div id="dc-lidar" class="dataset">
          <h3><a href="/dc-lidar/">District of Columbia - Classified Point Cloud LiDAR</a></h3>
          <p><span class="label label-info tag link-tag">cities</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">us-dc</span></p>
          <p>LiDAR point cloud data for Washington, DC is available for anyone to use on Amazon S3.
This dataset, managed by the Office of the Chief Technology Officer (OCTO), through the
direction of the District of Columbia GIS program, contains tiled point cloud data for
the entire District along with associated metadata.</p>
          <p><a href="/dc-lidar/">Details &rarr;</a></p>
        </div>
        <div id="wrf-alaska-snap" class="dataset">
          <h3><a href="/wrf-alaska-snap/">Downscaled Climate Data for Alaska</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">coastal</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info-sustainability tag link-tag">sustainability</span><span class="label label-info tag link-tag">weather</span></p>
          <p>Warning to data users - Errors have been discovered in the following variables in the previously-available versions of this dataset: u, u10, ubot, v, v10 and vbot. The SNAP team is working to address the issue and those variables have been removed from the currently-available version of the dataset. If you have previously downloaded this dataset which included those variables please download the new version of the dataset and refrain from utilizing the problematic variables for analysis. The dataset is being corrected and will be available including the variables at-issue when possible. We apo...</p>
          <p><a href="/wrf-alaska-snap/">Details &rarr;</a></p>
        </div>
        <div id="epa-rsei-pds" class="dataset">
          <h3><a href="/epa-rsei-pds/">EPA Risk-Screening Environmental Indicators</a></h3>
          <p><span class="label label-info tag link-tag">environmental</span><span class="label label-info-sustainability tag link-tag">sustainability</span></p>
          <p>Detailed air model results from EPA’s Risk-Screening Environmental Indicators (RSEI) model.</p>
          <p><a href="/epa-rsei-pds/">Details &rarr;</a></p>
        </div>
        <div id="epoch-of-reionization" class="dataset">
          <h3><a href="/epoch-of-reionization/">Epoch of Reionization Dataset</a></h3>
          <p><span class="label label-info tag link-tag">astronomy</span></p>
          <p>The data are from observations with the Murchison Widefield Array (MWA) which is a
Square Kilometer Array (SKA) precursor in Western Australia.  This particular
dataset is from the Epoch of Reionization project which is a key science driver
of the SKA. Nearly 2PB of such observations have been recorded to date, this is
a small subset of that which has been exported from the MWA data archive in
Perth and made available to the public on AWS.  The data were taken to detect
signatures of the first stars and galaxies forming and the effect of these early
stars and galaxies on the evolution of the u...</p>
          <p><a href="/epoch-of-reionization/">Details &rarr;</a></p>
        </div>
        <div id="genomeark" class="dataset">
          <h3><a href="/genomeark/">Genome Ark</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span></p>
          <p>The Genome Ark hosts genomic information for the Vertebrate Genomes Project (VGP) and other related projects. The VGP is an international collaboration that aims to generate complete and near error-free reference genomes for all extant vertebrate species. These genomes will be used to address fundamental questions in biology and disease, to identify species most genetically at risk for extinction, and to preserve genetic information of life.</p>
          <p><a href="/genomeark/">Details &rarr;</a></p>
        </div>
        <div id="noaa-gsod" class="dataset">
          <h3><a href="/noaa-gsod/">Global Surface Summary of Day</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">natural resource</span><span class="label label-info tag link-tag">regulatory</span><span class="label label-info-sustainability tag link-tag">sustainability</span><span class="label label-info tag link-tag">weather</span></p>
          <p>GSOD is a collection of daily weather measurements (temperature, wind speed, humidity, pressure, and more) from 9000+ weather stations around the world.</p>
          <p><a href="/noaa-gsod/">Details &rarr;</a></p>
        </div>
        <div id="google-ngrams" class="dataset">
          <h3><a href="/google-ngrams/">Google Books Ngrams</a></h3>
          <p><span class="label label-info tag link-tag">natural language processing</span></p>
          <p>N-grams are fixed size tuples of items. In this case the items are words extracted from the Google Books corpus. The n specifies the number of elements in the tuple, so a 5-gram contains five words or characters. The n-grams in this dataset were produced by passing a sliding window of the text of books and outputting a record for each new token.</p>
          <p><a href="/google-ngrams/">Details &rarr;</a></p>
        </div>
        <div id="hirlam" class="dataset">
          <h3><a href="/hirlam/">HIRLAM Weather Model</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info-sustainability tag link-tag">sustainability</span><span class="label label-info tag link-tag">weather</span></p>
          <p>HIRLAM (High Resolution Limited Area Model) is an operational synoptic and mesoscale weather prediction model managed by the Finnish Meteorological Institute.</p>
          <p><a href="/hirlam/">Details &rarr;</a></p>
        </div>
        <div id="wrf-se-alaska-snap" class="dataset">
          <h3><a href="/wrf-se-alaska-snap/">High Resolution Downscaled Climate Data for Southeast Alaska</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">coastal</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info-sustainability tag link-tag">sustainability</span><span class="label label-info tag link-tag">weather</span></p>
          <p>This dataset contains historical and projected dynamically downscaled climate data for the Southeast region of the State of Alaska at 1 and 4km spatial resolution and hourly temporal resolution. Select variables are also summarized into daily resolutions. This data was produced using the Weather Research and Forecasting (WRF) model (Version 4.0). We downscaled both Climate Forecast System Reanalysis (CFSR) historical reanalysis data (1980-2019) and both historical and projected runs from two GCM’s from the Coupled Model Inter-comparison Project 5 (CMIP5): GFDL-CM3 and NCAR-CCSM4 (historical ru...</p>
          <p><a href="/wrf-se-alaska-snap/">Details &rarr;</a></p>
        </div>
        <div id="dataforgood-fb-hrsl" class="dataset">
          <h3><a href="/dataforgood-fb-hrsl/">High Resolution Population Density Maps + Demographic Estimates by CIESIN and Facebook</a></h3>
          <p><span class="label label-info tag link-tag">aerial imagery</span><span class="label label-info tag link-tag">demographics</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">image processing</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">population</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info-sustainability tag link-tag">sustainability</span></p>
          <p>Population data for a selection of countries, allocated to 1 arcsecond blocks and provided in a combination of CSV
and Cloud-optimized GeoTIFF files. This refines <a href="https://sedac.ciesin.columbia.edu/data/collection/gpw-v4">CIESIN’s Gridded Population of the World</a>
using machine learning models on high-resolution worldwide Digital Globe
satellite imagery. CIESIN population counts aggregated from worldwide census
data are allocated to blocks where imagery appears to contain buildings.</p>
          <p><a href="/dataforgood-fb-hrsl/">Details &rarr;</a></p>
        </div>
        <div id="ichangemycity" class="dataset">
          <h3><a href="/ichangemycity/">IChangeMyCity Complaints Data from Janaagraha</a></h3>
          <p><span class="label label-info tag link-tag">cities</span><span class="label label-info tag link-tag">civic</span><span class="label label-info tag link-tag">complaints</span><span class="label label-info tag link-tag">machine learning</span></p>
          <p>The <a href="https://www.ichangemycity.com">IChangeMyCity</a> project provides insight into the complaints raised by citizens from diffent cities of India related to the issues in their neighbourhoods and the resolution of the same by the civic bodies.</p>
          <p><a href="/ichangemycity/">Details &rarr;</a></p>
        </div>
        <div id="990-spreadsheets" class="dataset">
          <h3><a href="/990-spreadsheets/">IRS 990 Filings (Spreadsheets)</a></h3>
          <p><span class="label label-info tag link-tag">economics</span><span class="label label-info tag link-tag">regulatory</span><span class="label label-info tag link-tag">statistics</span><span class="label label-info tag link-tag">us</span></p>
          <p>Excerpts of electronic Form 990 and 990-EZ filings, converted to spreadsheet form. Additional fields being added regularly.</p>
          <p><a href="/990-spreadsheets/">Details &rarr;</a></p>
        </div>
        <div id="iserv" class="dataset">
          <h3><a href="/iserv/">ISERV</a></h3>
          <p><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info-sustainability tag link-tag">sustainability</span></p>
          <p>ISS SERVIR Environmental Research and Visualization System (ISERV) was a fully-automated prototype camera aboard the International Space Station that was tasked to capture high-resolution Earth imagery of specific locations at 3-7 frames per second. In the course of its regular operations during 2013 and 2014, ISERV&#39;s camera acquired images that can be used primaliry in use is environmental and disaster management.</p>
          <p><a href="/iserv/">Details &rarr;</a></p>
        </div>
        <div id="fast-ai-imageclas" class="dataset">
          <h3><a href="/fast-ai-imageclas/">Image classification - fast.ai datasets</a></h3>
          <p><span class="label label-info tag link-tag">computer vision</span><span class="label label-info tag link-tag">deep learning</span><span class="label label-info tag link-tag">machine learning</span></p>
          <p>Some of the most important datasets for image classification research, including
CIFAR 10 and 100, Caltech 101, MNIST, Food-101, Oxford-102-Flowers, Oxford-IIIT-Pets,
and Stanford-Cars.  This is part of the fast.ai datasets collection hosted by
AWS for convenience of fast.ai students. See documentation link for citation and
license details for each dataset.</p>
          <p><a href="/fast-ai-imageclas/">Details &rarr;</a></p>
        </div>
        <div id="fast-ai-imagelocal" class="dataset">
          <h3><a href="/fast-ai-imagelocal/">Image localization  - fast.ai datasets</a></h3>
          <p><span class="label label-info tag link-tag">computer vision</span><span class="label label-info tag link-tag">deep learning</span><span class="label label-info tag link-tag">machine learning</span></p>
          <p>Some of the most important datasets for image localization  research, including
Camvid and PASCAL VOC (2007 and 2012). This is part of the fast.ai datasets
collection hosted by AWS for convenience of fast.ai students. See
documentation link for citation and license details for each dataset.</p>
          <p><a href="/fast-ai-imagelocal/">Details &rarr;</a></p>
        </div>
        <div id="k2" class="dataset">
          <h3><a href="/k2/">K2 Mission Data</a></h3>
          <p><span class="label label-info tag link-tag">astronomy</span></p>
          <p>The K2 mission observed 100 square degrees for 80 days each across 20 different pointings along the ecliptic, collecting high-precision photometry for a selection of targets within each field. The mission began when the original Kepler mission ended due to loss of the second reaction wheel in 2011. More information about the K2 mission is available at <a href="https://archive.stsci.edu/k2/">MAST</a>.</p>
          <p><a href="/k2/">Details &rarr;</a></p>
        </div>
        <div id="kitti" class="dataset">
          <h3><a href="/kitti/">KITTI Vision Benchmark Suite</a></h3>
          <p><span class="label label-info tag link-tag">autonomous vehicles</span><span class="label label-info tag link-tag">computer vision</span><span class="label label-info tag link-tag">deep learning</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">robotics</span></p>
          <p>Dataset and benchmarks for computer vision research in the context of autonomous driving. The dataset has been recorded in and around the city of Karlsruhe, Germany using the mobile platform AnnieWay (VW station wagon) which has been equipped with several RGB and monochrome cameras, a Velodyne HDL 64 laser scanner as well as an accurate RTK corrected GPS/IMU localization unit. The dataset has been created for computer vision and machine learning research on stereo, optical flow, visual odometry, semantic segmentation, semantic instance segmentation, road segmentation, single image depth predic...</p>
          <p><a href="/kitti/">Details &rarr;</a></p>
        </div>
        <div id="kepler" class="dataset">
          <h3><a href="/kepler/">Kepler Mission Data</a></h3>
          <p><span class="label label-info tag link-tag">astronomy</span></p>
          <p>The Kepler mission observed the brightness of more than 180,000 stars near the Cygnus constellation at a 30 minute cadence for 4 years in order to find transiting exoplanets, study variable stars, and find eclipsing binaries. More information about the Kepler mission is available at <a href="https://archive.stsci.edu/kepler/">MAST</a>.</p>
          <p><a href="/kepler/">Details &rarr;</a></p>
        </div>
        <div id="multimedia-commons" class="dataset">
          <h3><a href="/multimedia-commons/">Multimedia Commons</a></h3>
          <p><span class="label label-info tag link-tag">computer vision</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">multimedia</span></p>
          <p>The Multimedia Commons is a collection of audio and visual features computed for the nearly 100 million Creative Commons-licensed Flickr images and videos in the YFCC100M dataset from Yahoo! Labs, along with ground-truth annotations for selected subsets. The International Computer Science Institute (ICSI) and Lawrence Livermore National Laboratory are producing and distributing a core set of derived feature sets and annotations as part of an effort to enable large-scale video search capabilities. They have released this feature corpus into the public domain, under Creative Commons License 0, s...</p>
          <p><a href="/multimedia-commons/">Details &rarr;</a></p>
        </div>
        <div id="mevadata" class="dataset">
          <h3><a href="/mevadata/">Multiview Extended Video with Activities (MEVA)</a></h3>
          <p><span class="label label-info tag link-tag">computer vision</span><span class="label label-info tag link-tag">urban</span><span class="label label-info tag link-tag">us</span></p>
          <p>The Multiview Extended Video with Activities (MEVA) dataset consists
video data of human activity, both scripted and unscripted,
collected with roughly 100 actors over several weeks.  The data was
collected with 29 cameras with overlapping and non-overlapping
fields of view. The current release consists of about 328 hours
(516GB, 4259 clips) of video data, as well as 4.6 hours (26GB) of
UAV data. Other data includes GPS tracks of actors, camera models,
and a site map. We have also released annotations for 22 hours of
data. Further updates are planned.</p>
          <p><a href="/mevadata/">Details &rarr;</a></p>
        </div>
        <div id="fast-ai-nlp" class="dataset">
          <h3><a href="/fast-ai-nlp/">NLP - fast.ai datasets</a></h3>
          <p><span class="label label-info tag link-tag">deep learning</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">natural language processing</span></p>
          <p>Some of the most important datasets for NLP, with a focus on classification, including
IMDb, AG-News, Amazon Reviews (polarity and full), Yelp Reviews (polarity and
full), Dbpedia, Sogou News (Pinyin), Yahoo Answers, Wikitext 2 and Wikitext
103, and ACL-2010 French-English 10^9 corpus.  This is part of the
fast.ai datasets collection hosted by AWS for convenience of fast.ai
students. See documentation link for citation and license details for each
dataset.</p>
          <p><a href="/fast-ai-nlp/">Details &rarr;</a></p>
        </div>
        <div id="noaa-coastal-lidar" class="dataset">
          <h3><a href="/noaa-coastal-lidar/">NOAA Coastal Lidar Data</a></h3>
          <p><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">elevation</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">lidar</span><span class="label label-info-sustainability tag link-tag">sustainability</span></p>
          <p>Lidar (light detection and ranging) is a technology that can measure the 3-dimentional location of objects, including the solid earth surface. The data consists of a point cloud of the positions of solid objects that reflected a laser pulse, typically from an airborne platform. In addition to the position, each point may also be attributed by the type of object it reflected from, the intensity of the reflection, and other system dependent metadata. The NOAA Coastal Lidar Data is a collection of lidar projects from many different sources and agencies, geographically focused on the coastal areas...</p>
          <p><a href="/noaa-coastal-lidar/">Details &rarr;</a></p>
        </div>
        <div id="noaa-ncn" class="dataset">
          <h3><a href="/noaa-ncn/">NOAA Continuously Operating Reference Stations (CORS) Network (NCN)</a></h3>
          <p><span class="label label-info tag link-tag">broadcast ephemeris</span><span class="label label-info tag link-tag">Continuously Operating Reference Station (CORS)</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">GNSS</span><span class="label label-info tag link-tag">GPS</span><span class="label label-info tag link-tag">mapping</span><span class="label label-info tag link-tag">NOAA CORS Network (NCN)</span><span class="label label-info tag link-tag">post-processing</span><span class="label label-info tag link-tag">RINEX</span><span class="label label-info tag link-tag">survey</span></p>
          <p>The <a href="https://geodesy.noaa.gov/CORS/">NOAA Continuously Operating Reference Stations (CORS) Network (NCN)</a>, managed by NOAA/National Geodetic Survey (<a href="https://geodesy.noaa.gov/">NGS</a>), provide Global Navigation Satellite System (GNSS) data, supporting three dimensional positioning, meteorology, space weather, and geophysical applications throughout the United States. The NCN is a multi-purpose, multi-agency cooperative endeavor, combining the efforts of hundreds of government, academic, and private organizations. The stations are independently owned and operated. Each agency shares their GNSS/GPS carrier phase and code range measurements and station metadata with NGS, which are analyzed and distributed free of charge.
...</p>
          <p><a href="/noaa-ncn/">Details &rarr;</a></p>
        </div>
        <div id="noaa-eri" class="dataset">
          <h3><a href="/noaa-eri/">NOAA Emergency Response Imagery</a></h3>
          <p><span class="label label-info tag link-tag">aerial imagery</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info-sustainability tag link-tag">sustainability</span><span class="label label-info tag link-tag">weather</span></p>
          <p>In order to support NOAA&#39;s homeland security and emergency response requirements, the National Geodetic Survey Remote Sensing Division (NGS/RSD) has the capability to acquire and rapidly disseminate a variety of spatially-referenced datasets to federal, state, and local government agencies, as well as the general public. Remote sensing technologies used for these projects have included lidar, high-resolution digital cameras, a film-based RC-30 aerial camera system, and hyperspectral imagers. Examples of rapid response initiatives include acquiring high resolution images with the Emerge/App...</p>
          <p><a href="/noaa-eri/">Details &rarr;</a></p>
        </div>
        <div id="noaa-gefs" class="dataset">
          <h3><a href="/noaa-gefs/">NOAA Global Ensemble Forecast System (GEFS)</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info-sustainability tag link-tag">sustainability</span><span class="label label-info tag link-tag">weather</span></p>
          <p>The Global Ensemble Forecast System (GEFS), previously known as the GFS Global ENSemble (GENS), is a weather forecast model made up of 21 separate forecasts, or ensemble members.  The National Centers for Environmental Prediction (NCEP) started the GEFS to address the nature of uncertainty in weather observations, which is used to initialize weather forecast models. The GEFS attempts to quantify the amount of uncertainty in a forecast by generating an ensemble of multiple forecasts, each minutely different, or perturbed, from the original observations. With global coverage, GEFS is produced fo...</p>
          <p><a href="/noaa-gefs/">Details &rarr;</a></p>
        </div>
        <div id="noaa-gestofs" class="dataset">
          <h3><a href="/noaa-gestofs/">NOAA Global Extratropical Surge and Tide Operational Forecast System (Global ESTOFS)</a></h3>
          <p><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">coastal</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">global</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">oceans</span><span class="label label-info-sustainability tag link-tag">sustainability</span><span class="label label-info tag link-tag">water</span><span class="label label-info tag link-tag">weather</span></p>
          <p>NOAA&#39;s Global Extratropical Surge and Tide Operational Forecast System (Global ESTOFS) provides users with nowcasts (analyses of near present conditions) and forecast guidance of water level conditions for the entire globe. Global ESTOFS has been developed to serve the marine navigation, weather forecasting, and disaster mitigation user communities. Global ESTOFS was developed in a collaborative effort between the NOAA/National Ocean Service (NOS)/Office of Coast Survey, the NOAA/National Weather Service (NWS)/National Centers for Environmental Prediction (NCEP) Central Operations (NCO), the University of Notre Dame, the University of North Carolina, and The Water Institute of the Gulf. The model generates forecasts out to 180 hours four times per day; forecast output includes water levels caused by the combined effects of storm surge and tides, by astronomical tides alone, and by sub-tidal water levels (isolated storm surge). <br /><br />The hydrodynamic model employed by Global ESTOFS is the ADvanced CIRCulation (ADCIRC) finite element model. The model is forced by GFS winds, mean sea level pressure, and sea ice. The unstructured grid used by Global ESTOFS consists of 8,063,409 nodes and 15,478,900 triangular elements. Coastal resolution is up to 80 m for Hawaii and the U.S. West Coast; up to 90-120 m for the Pacific Islands including Guam, American Samoa, Marianas, Wake Island, Marshall Islands, and Palau; and up to 120 m for the U.S. East Coast, Puerto Rico, Micronesia, and Alaska. The flood plain extends overland to approx</b>...</p>
          <p><a href="/noaa-gestofs/">Details &rarr;</a></p>
        </div>
        <div id="noaa-ghe" class="dataset">
          <h3><a href="/noaa-ghe/">NOAA Global Hydro Estimator (GHE)</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info-sustainability tag link-tag">sustainability</span><span class="label label-info tag link-tag">water</span><span class="label label-info tag link-tag">weather</span></p>
          <p>Global Hydro-Estimator provides a global
mosaic imagery of rainfall estimates from
multi-geostationary satellites, which
currently includes GOES-16, GOES-15,
Meteosat-8, Meteosat-11 and Himawari-8.
The GHE products include: Instantaneous
rain rate, 1 hour, 3 hour, 6 hour, 24 hour
and also multi-day rainfall accumulation.</p>
          <p><a href="/noaa-ghe/">Details &rarr;</a></p>
        </div>
        <div id="noaa-isd" class="dataset">
          <h3><a href="/noaa-isd/">NOAA Integrated Surface Database (ISD)</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info-sustainability tag link-tag">sustainability</span><span class="label label-info tag link-tag">weather</span></p>
          <p>The Integrated Surface Database (ISD) consists
of global hourly and synoptic observations
compiled from numerous sources into a gzipped
fixed width format. ISD was developed as a joint
activity within Asheville&#39;s Federal Climate
Complex. The database includes over 35,000 stations
worldwide, with some having data as far back
as 1901, though the data show a substantial
increase in volume in the 1940s and again in
the early 1970s. Currently, there are over
14,000 &quot;active&quot; stations updated daily in the
database. The total uncompressed data volume is
around 600 gigabytes; however, it ...</p>
          <p><a href="/noaa-isd/">Details &rarr;</a></p>
        </div>
        <div id="noaa-nbm" class="dataset">
          <h3><a href="/noaa-nbm/">NOAA National Blend of Models (NBM)</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info-sustainability tag link-tag">sustainability</span><span class="label label-info tag link-tag">weather</span></p>
          <p>The National Blend of Models (NBM) is a nationally consistent and skillful suite of calibrated forecast guidance based on a blend of both NWS and non-NWS numerical weather prediction model data and post-processed model guidance. The goal of the NBM is to create a highly accurate, skillful and consistent starting point for the gridded forecast. </p>
          <p><a href="/noaa-nbm/">Details &rarr;</a></p>
        </div>
        <div id="noaa-nwm-pds" class="dataset">
          <h3><a href="/noaa-nwm-pds/">NOAA National Water Model Short-Range Forecast</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info-sustainability tag link-tag">sustainability</span><span class="label label-info tag link-tag">transportation</span><span class="label label-info tag link-tag">weather</span></p>
          <p>The National Water Model (NWM) is a water resources model that simulates and forecasts water
budget variables, including snowpack, evapotranspiration, soil moisture and streamflow, over
the entire continental United States (CONUS). The model, launched in August 2016, is designed
to improve the ability of NOAA to meet the needs of its stakeholders (forecasters, emergency
managers, reservoir operators, first responders, recreationists, farmers, barge operators, and
ecosystem and floodplain managers) by providing expanded accuracy, detail, and frequency of water
information. It is operated by NOA...</p>
          <p><a href="/noaa-nwm-pds/">Details &rarr;</a></p>
        </div>
        <div id="noaa-rap" class="dataset">
          <h3><a href="/noaa-rap/">NOAA Rapid Refresh (RAP)</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info-sustainability tag link-tag">sustainability</span><span class="label label-info tag link-tag">weather</span></p>
          <p>The Rapid Refresh (RAP) is a NOAA/NCEP operational weather prediction system comprised primarily of a numerical forecast model and analysis/assimilation system to initialize that model. It covers North America and is run with a horizontal resolution of 13 km and 50 vertical layers. The RAP was developed to serve users needing frequently updated short-range weather forecasts, including those in the US aviation community and US severe weather forecasting community. The model is run for every hour of the day; it is integrated to 51 hours for the 03/09/15/21 UTC cycles and to 21 hours for every ot...</p>
          <p><a href="/noaa-rap/">Details &rarr;</a></p>
        </div>
        <div id="noaa-s111" class="dataset">
          <h3><a href="/noaa-s111/">NOAA S-111 Surface Water Currents Data</a></h3>
          <p><span class="label label-info tag link-tag">oceans</span><span class="label label-info-sustainability tag link-tag">sustainability</span><span class="label label-info tag link-tag">water</span></p>
          <p>S-111 is a data and metadata encoding specification that is part of the <a href="https://iho.int/en/s100-project">S-100 Universal Hydrographic Data Model</a>, an international standard for hydrographic data. This collection of data contains surface water currents forecast guidance from <a href="https://tidesandcurrents.noaa.gov/models.html">NOAA/NOS Operational Forecast Systems</a>, a set of operational hydrodynamic nowcast and forecast modeling systems, for various U.S. coastal waters and the great lakes. The collection also contains surface current forecast guidance output from the <a href="https://polar.ncep.noaa.gov/global/">NCEP Global Real-Time Ocean Forecast System (GRTOFS)</a> for some offshore areas. These datasets are encoded as HDF-5 f...</p>
          <p><a href="/noaa-s111/">Details &rarr;</a></p>
        </div>
        <div id="noaa-swdi" class="dataset">
          <h3><a href="/noaa-swdi/">NOAA Severe Weather Data Inventory (SWDI)</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info-sustainability tag link-tag">sustainability</span><span class="label label-info tag link-tag">weather</span></p>
          <p>The Storm Events Database is an integrated database of severe weather events across the United States from 1950 to this year, with information about a storm event&#39;s location, azimuth, distance, impact, and severity, including the cost of damages to property and crops. It contains data documenting: The occurrence of storms and other significant weather phenomena having sufficient intensity to cause loss of life, injuries, significant property damage, and/or disruption to commerce. Rare, unusual, weather phenomena that generate media attention, such as snow flurries in South Florida or the S...</p>
          <p><a href="/noaa-swdi/">Details &rarr;</a></p>
        </div>
        <div id="noaa-space-weather" class="dataset">
          <h3><a href="/noaa-space-weather/">NOAA Space Weather Forecast and Observation Data</a></h3>
          <p><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">solar</span><span class="label label-info-sustainability tag link-tag">sustainability</span><span class="label label-info tag link-tag">weather</span></p>
          <p>Space weather forecast and observation data is collected and disseminated by NOAA’s Space Weather Prediction Center (SWPC) in Boulder, CO. SWPC produces forecasts for multiple space weather phenomenon types and the resulting impacts to Earth and human activities.  A variety of products are available that provide these forecast expectations, and their respective measurements, in formats that range from detailed technical forecast discussions to NOAA Scale values to simple bulletins that give information in laymen&#39;s terms. Forecasting is the prediction of future events, based on analysis and...</p>
          <p><a href="/noaa-space-weather/">Details &rarr;</a></p>
        </div>
        <div id="noaa-ufs-s2s" class="dataset">
          <h3><a href="/noaa-ufs-s2s/">NOAA Unified Forecast System Subseasonal to Seasonal Prototype 5</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">oceans</span><span class="label label-info-sustainability tag link-tag">sustainability</span><span class="label label-info tag link-tag">weather</span></p>
          <p>The Unified Forecast System Subseasonal to Seasonal prototype 5 (UFS S2Sp5) dataset is reforecast data from the UFS atmosphere-ocean coupled model experimental prototype version 5 produced by the Medium Range and Subseasonal to Seasonal Application team of the UFS-R2O project. The UFS S2Sp5 is the first dataset released to the broader weather community for analysis and feedback as part of the development of the next generation operational numerical weather prediction system from NWS. The dataset includes all the major weather variables for atmosphere, land, ocean, sea ice, and ocean waves.
<br/>
<br/>
A</br/>...</p>
          <p><a href="/noaa-ufs-s2s/">Details &rarr;</a></p>
        </div>
        <div id="nanopore" class="dataset">
          <h3><a href="/nanopore/">Nanopore Reference Human Genome</a></h3>
          <p><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span></p>
          <p>This dataset includes the sequencing and assembly of a reference standard human genome (GM12878) using the MinION nanopore sequencing instrument with the R9.4 1D chemistry.</p>
          <p><a href="/nanopore/">Details &rarr;</a></p>
        </div>
        <div id="nsd" class="dataset">
          <h3><a href="/nsd/">Natural Scenes Dataset</a></h3>
          <p><span class="label label-info tag link-tag">computer vision</span><span class="label label-info tag link-tag">image processing</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">magnetic resonance imaging</span><span class="label label-info tag link-tag">neuroimaging</span><span class="label label-info tag link-tag">neuroscience</span><span class="label label-info tag link-tag">nifti</span></p>
          <p>Here, we collected and pre-processed a massive, high-quality 7T fMRI dataset that can be used to advance our understanding of how the brain works. A unique feature of this dataset is the massive amount of data available per individual subject. The data were acquired using ultra-high-field fMRI (7T, whole-brain, 1.8-mm resolution, 1.6-s TR). We measured fMRI responses while each of 8 participants viewed 9,000–10,000 distinct, color natural scenes (22,500–30,000 trials) in 30–40 weekly scan sessions over the course of a year. Additional measures were collected including resting-state data, retin...</p>
          <p><a href="/nsd/">Details &rarr;</a></p>
        </div>
        <div id="ooni" class="dataset">
          <h3><a href="/ooni/">Open Observatory of Network Interference</a></h3>
          <p><span class="label label-info tag link-tag">internet</span></p>
          <p>A free software, global observation network for detecting censorship, surveillance and traffic manipulation on the internet.</p>
          <p><a href="/ooni/">Details &rarr;</a></p>
        </div>
        <div id="openneuro" class="dataset">
          <h3><a href="/openneuro/">OpenNeuro</a></h3>
          <p><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">imaging</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">neurobiology</span><span class="label label-info tag link-tag">neuroimaging</span></p>
          <p>OpenNeuro is a database of openly-available brain imaging data. The data are shared according to a Creative Commons CC0 license, providing a broad range of brain imaging data to researchers and citizen scientists alike. The database primarily focuses on functional magnetic resonance imaging (fMRI) data, but also includes other imaging modalities including structural and diffusion MRI, electroencephalography (EEG), and magnetoencephalograpy (MEG). OpenfMRI is a project of the <a href="http://reproducibility.stanford.edu">Center for Reproducible Neuroscience at Stanford University</a>. Development of the OpenNeuro resource has been funded by th...</p>
          <p><a href="/openneuro/">Details &rarr;</a></p>
        </div>
        <div id="osmlr" class="dataset">
          <h3><a href="/osmlr/">OpenStreetMap Linear Referencing</a></h3>
          <p><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">osm</span><span class="label label-info-sustainability tag link-tag">sustainability</span><span class="label label-info tag link-tag">traffic</span></p>
          <p>OSMLR a linear referencing system built on top of OpenStreetMap. OSM has great information about roads around the world and their interconnections, but it lacks the means to give a stable identifier to a stretch of roadway. OSMLR provides a stable set of numerical IDs for every 1 kilometer stretch of roadway around the world. In urban areas, OSMLR IDs are attached to each block of roadways between significant intersections.</p>
          <p><a href="/osmlr/">Details &rarr;</a></p>
        </div>
        <div id="proj-datum-grids" class="dataset">
          <h3><a href="/proj-datum-grids/">PROJ datum grids</a></h3>
          <p><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">mapping</span></p>
          <p>Horizontal and vertical adjustment datasets for coordinate transformation to be used by PROJ 7 or later. PROJ is a generic coordinate transformation software that transforms geospatial coordinates from one coordinate reference system (CRS) to another. This includes cartographic projections as well as geodetic transformations.</p>
          <p><a href="/proj-datum-grids/">Details &rarr;</a></p>
        </div>
        <div id="physionet" class="dataset">
          <h3><a href="/physionet/">Physionet</a></h3>
          <p><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">life sciences</span></p>
          <p>PhysioNet offers free web access to large collections of recorded physiologic signals (PhysioBank) and related open-source software (PhysioToolkit).</p>
          <p><a href="/physionet/">Details &rarr;</a></p>
        </div>
        <div id="paracrawl" class="dataset">
          <h3><a href="/paracrawl/">Provision of Web-Scale Parallel Corpora for Official European Languages (ParaCrawl)</a></h3>
          <p><span class="label label-info tag link-tag">machine translation</span><span class="label label-info tag link-tag">natural language processing</span></p>
          <p>ParaCrawl is a set of large parallel corpora to/from English for all official EU languages by a broad web crawling effort. State-of-the-art methods are applied for the entire processing chain from identifying web sites with translated text all the way to collecting, cleaning and delivering parallel corpora that are ready as training data for CEF.AT and translation memories for DG Translation.</p>
          <p><a href="/paracrawl/">Details &rarr;</a></p>
        </div>
        <div id="smithsonian-open-access" class="dataset">
          <h3><a href="/smithsonian-open-access/">Smithsonian Open Access</a></h3>
          <p><span class="label label-info tag link-tag">art</span><span class="label label-info tag link-tag">culture</span><span class="label label-info tag link-tag">encyclopedic</span><span class="label label-info tag link-tag">history</span><span class="label label-info tag link-tag">museum</span></p>
          <p>The Smithsonian’s mission is the &quot;increase and diffusion of knowledge&quot; and has been collecting since 1846. The Smithsonian, through its efforts to digitize its multidisciplinary collections, has created millions of digital assets and related metadata describing the collection objects. On February 25th, 2020, the Smithsonian released over 2.8 million CC0 interdisciplinary 2-D and 3-D images, related metadata, and additionally, research data from researches across the Smithsonian. The 2.8 million &quot;open access&quot; collections are a subset of the Smithsonian’s 155 million objects,...</p>
          <p><a href="/smithsonian-open-access/">Details &rarr;</a></p>
        </div>
        <div id="software-heritage" class="dataset">
          <h3><a href="/software-heritage/">Software Heritage Graph Dataset</a></h3>
          <p><span class="label label-info tag link-tag">digital preservation</span><span class="label label-info tag link-tag">free software</span><span class="label label-info tag link-tag">open source software</span><span class="label label-info tag link-tag">source code</span></p>
          <p><a href="https://www.softwareheritage.org/">Software Heritage</a> is the largest
existing public archive of software source code and accompanying
development history. The Software Heritage Graph Dataset is a fully
deduplicated Merkle DAG representation of the Software Heritage archive.The dataset links together file content identifiers, source code
directories, Version Control System (VCS) commits tracking evolution over
time, up to the full states of VCS repositories as observed by Software
Heritage during periodic crawls. The dataset’s contents come from major
development forges (including GitHub and GitLab), FOSS distributions (e.g.,
Deb...</p>
          <p><a href="/software-heritage/">Details &rarr;</a></p>
        </div>
        <div id="elp-nouabale-landscape" class="dataset">
          <h3><a href="/elp-nouabale-landscape/">Sounds of Central African landscapes</a></h3>
          <p><span class="label label-info tag link-tag">biodiversity</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">ecosystems</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">land</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">natural resource</span><span class="label label-info tag link-tag">survey</span></p>
          <p>Archival soundscapes recorded in the rainforest landscapes of
Central Africa, with a focus on the vocalizations of African forest
elephants (Loxodonta cyclotis).</p>
          <p><a href="/elp-nouabale-landscape/">Details &rarr;</a></p>
        </div>
        <div id="tabula-muris-senis" class="dataset">
          <h3><a href="/tabula-muris-senis/">Tabula Muris Senis</a></h3>
          <p><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">encyclopedic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">health</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">medicine</span><span class="label label-info tag link-tag">single-cell transcriptomics</span></p>
          <p>Tabula Muris Senis is a comprehensive compendium of single cell transcriptomic data from the model organism <em>Mus musculus</em> comprising more than 500,000 cells from 18 organs and tissues across the mouse lifespan. We discovered cell-specific changes occurring across multiple cell types and organs, as well as age related changes in the cellular composition of different organs. Using single-cell transcriptomic data we were able to assess cell type specific manifestations of different hallmarks of aging, such as senescence, changes in the activity of metabolic pathways, depletion of stem-cell populat...</p>
          <p><a href="/tabula-muris-senis/">Details &rarr;</a></p>
        </div>
        <div id="gmsdata" class="dataset">
          <h3><a href="/gmsdata/">The Genome Modeling System</a></h3>
          <p><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span></p>
          <p>The Genome Institute at Washington University has developed a high-throughput, fault-tolerant analysis information management system called the Genome Modeling System (GMS), capable of executing complex, interdependent, and automated genome analysis pipelines at a massive scale. The GMS framework provides detailed tracking of samples and data coupled with reliable and repeatable analysis pipelines. GMS includes a full system image with software and services, expandable from one workstation to a large compute cluster.</p>
          <p><a href="/gmsdata/">Details &rarr;</a></p>
        </div>
        <div id="hcp-openaccess" class="dataset">
          <h3><a href="/hcp-openaccess/">The Human Connectome Project</a></h3>
          <p><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">neuroimaging</span></p>
          <p>The Human Connectome Project aims to provide an unparalleled compilation of neural data, an interface to graphically navigate this data and the opportunity to achieve never before realized conclusions about the living human brain.</p>
          <p><a href="/hcp-openaccess/">Details &rarr;</a></p>
        </div>
        <div id="mmid" class="dataset">
          <h3><a href="/mmid/">The Massively Multilingual Image Dataset (MMID)</a></h3>
          <p><span class="label label-info tag link-tag">computer vision</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">machine translation</span><span class="label label-info tag link-tag">natural language processing</span></p>
          <p>MMID is a large-scale, massively multilingual dataset of images paired with the words they represent collected at the <a href="https://upenn.edu">University of Pennsylvania</a>.
The dataset is doubly parallel: for each language, words are stored parallel to images that represent the word, <em>and</em> parallel to the word&#39;s translation into English (and corresponding images.)</p>
          <p><a href="/mmid/">Details &rarr;</a></p>
        </div>
        <div id="ubc-sunflower-genome" class="dataset">
          <h3><a href="/ubc-sunflower-genome/">University of British Columbia Sunflower Genome Dataset</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">biodiversity</span><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">food security</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>This dataset captures Sunflower&#39;s genetic diversity originating
from thousands of wild, cultivated, and landrace sunflower
individuals distributed across North America.The data consists of raw sequences and associated botanical metadata,
aligned sequences (to three different reference genomes), and sets of
SNPs computed across several cohorts.</p>
          <p><a href="/ubc-sunflower-genome/">Details &rarr;</a></p>
        </div>
        <div id="stdpopsim_kern" class="dataset">
          <h3><a href="/stdpopsim_kern/">stdpopsim species resources</a></h3>
          <p><span class="label label-info tag link-tag">genetic maps</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">population genetics</span><span class="label label-info tag link-tag">recombination maps</span><span class="label label-info tag link-tag">simulations</span></p>
          <p>Contains all resources (genome specifications, recombination maps, etc.) required for species specific simulation with the stdpopsim package. These resources are originally from a variety of other consortium and published work but are consolidated here for ease of access and use. If you are interested in adding a new species to the stdpopsim resource please raise an issue on the stdpopsim GitHub page to have the necessary files added here.</p>
          <p><a href="/stdpopsim_kern/">Details &rarr;</a></p>
        </div>
        <div id="binding-db" class="dataset">
          <h3><a href="/binding-db/">Binding DB</a></h3>
          <p><span class="label label-info tag link-tag">biotech blueprint</span><span class="label label-info tag link-tag">chemistry</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">molecule</span><span class="label label-info tag link-tag">parquet</span></p>
          <p>This a parquet representation of The Binding Database&#39;s <a href="https://www.bindingdb.org/bind/chemsearch/marvin/SDFdownload.jsp?all_download=yes">Full BindingDB Database Dump</a> that you can query straight from Athena in under 60 seconds (no Oracle database required). The Binding Database projects aims to make experimental data on the noncovalent association of molecules in solution searchable via the WWW. The initial focus is on biomolecular systems, but data on host-guest and supramolecular systems are also important and being included over time. It is expected that the enhanced access to data provided by this resource will facilitate drug-discovery, the design of self-assemblin...</p>
          <p><a href="/binding-db/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://aws.amazon.com/blogs/startups/a-data-lake-as-code-featuring-chembl-and-opentargets/" class="aws-link" target="_none">Data Lake as Code, Featuring ChEMBL and Open Targets</a> by Paul Underwood</li></a>
          </li>
          <li>
            <a href="https://github.com/aws-samples/data-lake-as-code/blob/roda/docs/roda_install.md" class="aws-link" target="_none">Data Lake as Code Deployment Guide</a> by AWS Biotech Blueprints Team</li></a>
          </li>
        </ul>
        <p><a href="/binding-db/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="chembl" class="dataset">
          <h3><a href="/chembl/">ChEMBL</a></h3>
          <p><span class="label label-info tag link-tag">biotech blueprint</span><span class="label label-info tag link-tag">chemistry</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">molecule</span><span class="label label-info tag link-tag">parquet</span></p>
          <p>ChEMBL is a manually curated database of bioactive molecules with drug-like properties. It brings together chemical, bioactivity and genomic data to aid the translation of genomic information into effective new drugs. This representation of ChEMBL is stored in Parquet format and most easily utilized through Amazon Athena. Follow the documentation for install instructions (&lt; 2 minute install). New ChEMBL releases occur sporadically; the most up to date information on ChEMBL releases can be found <a href="https://chembl.gitbook.io/chembl-interface-documentation/downloads">here</a>.</p>
          <p><a href="/chembl/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/aws-samples/data-lake-as-code/blob/roda/docs/roda_install.md" class="aws-link" target="_none">Data Lake as Code Deployment Guide</a> by AWS Biotech Blueprints Team</li></a>
          </li>
          <li>
            <a href="https://aws.amazon.com/blogs/startups/a-data-lake-as-code-featuring-chembl-and-opentargets/" class="aws-link" target="_none">Data Lake as Code, Featuring ChEMBL and Open Targets</a> by Paul Underwood</li></a>
          </li>
        </ul>
        <p><a href="/chembl/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="clinvar" class="dataset">
          <h3><a href="/clinvar/">ClinVar</a></h3>
          <p><span class="label label-info tag link-tag">biotech blueprint</span><span class="label label-info tag link-tag">chemistry</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">parquet</span></p>
          <p>ClinVar is a freely accessible, public archive of reports of the relationships among human variations and phenotypes, with supporting evidence. ClinVar thus facilitates access to and communication about the relationships asserted between human variation and observed health status, and the history of that interpretation. ClinVar processes submissions reporting variants found in patient samples, assertions made regarding their clinical significance, information about the submitter, and other supporting data. The alleles described in submissions are mapped to reference sequences, and reported acc...</p>
          <p><a href="/clinvar/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://aws.amazon.com/blogs/startups/a-data-lake-as-code-featuring-chembl-and-opentargets/" class="aws-link" target="_none">Data Lake as Code, Featuring ChEMBL and Open Targets</a> by Paul Underwood</li></a>
          </li>
          <li>
            <a href="https://github.com/aws-samples/data-lake-as-code/blob/roda/docs/roda_install.md" class="aws-link" target="_none">Data Lake as Code Deployment Guide</a> by AWS Biotech Blueprints Team</li></a>
          </li>
        </ul>
        <p><a href="/clinvar/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="us-hiring-rates-pandemic" class="dataset">
          <h3><a href="/us-hiring-rates-pandemic/">Covid Job Impacts - US Hiring Data Since March 1 2020</a></h3>
          <p><span class="label label-info tag link-tag">COVID-19</span><span class="label label-info tag link-tag">economics</span><span class="label label-info tag link-tag">financial markets</span><span class="label label-info tag link-tag">hiring</span><span class="label label-info tag link-tag">market data</span></p>
          <p>This dataset provides daily updates on the volume of US job listings filtered by geography industry job family and role; normalized to pre-covid levels.These data files feed the business intelligence visuals at covidjobimpacts.greenwich.hr, a public-facing site hosted by Greenwich.HR and OneModel Inc.
Data is derived from online job listings tracked continuously, calculated daily and published nightly.  On average data from 70% of all new US jobs are captured,
and the dataset currently contains data from 3.3 million hiring organizations.Data for each filter segment is represented as the 7-day ...</p>
          <p><a href="/us-hiring-rates-pandemic/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://greenwichhr-covidjobimpacts.s3.us-east-2.amazonaws.com/ghr_data_specs_covid_public.pdf" class="" target="_none">Documentation of dataset schemas</a> by <a href="https://www.greenwich.hr" target="_none">Greenwich.HR</a></li></a>
          </li>
          <li>
            <a href="https://covidjobimpacts.greenwich.hr" class="" target="_none">CovidJobImpats.Greenwich.HR - online visualization of daily hiring data and weekly unemployment data including links to recorded discussions using the data</a> by Greenwich.HR and OneModel Inc.</li></a>
          </li>
        </ul>
        <p><a href="/us-hiring-rates-pandemic/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="opentargets" class="dataset">
          <h3><a href="/opentargets/">Open Targets</a></h3>
          <p><span class="label label-info tag link-tag">biotech blueprint</span><span class="label label-info tag link-tag">chemistry</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">molecule</span><span class="label label-info tag link-tag">parquet</span></p>
          <p>This a Parquet representation of the Open Targets Platform&#39;s <a href="https://www.targetvalidation.org/downloads/data">2020-06 export</a>. The Open Targets Platform integrates evidence from genetics, genomics, transcriptomics, drugs, animal models and scientific literature to score and rank target-disease associations for drug target identification. The Open Targets Platform (<a href="https://www.targetvalidation.org">https://www.targetvalidation.org</a>) is a freely available resource for the integration of genetics, genomics, and chemical data to aid systematic drug target identification and prioritisation.</p>
          <p><a href="/opentargets/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/aws-samples/data-lake-as-code/blob/roda/docs/roda_install.md" class="aws-link" target="_none">Data Lake as Code Deployment Guide</a> by AWS Biotech Blueprints Team</li></a>
          </li>
          <li>
            <a href="https://aws.amazon.com/blogs/startups/a-data-lake-as-code-featuring-chembl-and-opentargets/" class="aws-link" target="_none">Data Lake as Code, Featuring ChEMBL and Open Targets</a> by Paul Underwood</li></a>
          </li>
        </ul>
        <p><a href="/opentargets/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="agriculture_vision" class="dataset">
          <h3><a href="/agriculture_vision/">AgricultureVision</a></h3>
          <p><span class="label label-info tag link-tag">aerial imagery</span><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">computer vision</span><span class="label label-info tag link-tag">deep learning</span></p>
          <p>Dataset associated with the paper &quot;Agriculture-Vision: A Large Aerial Image Database for Agricultural Pattern Analysis&quot;. Agriculture-Vision aims to be a publicly available large-scale aerial agricultural image dataset that is high-resolution, multi-band, and with multiple types of patterns annotated by agronomy experts. In its current stage, we have captured 94,986 512x512images sampled from 3,432 farmlands with nine types of annotations: double plant, drydown, endrow, nutrient deficiency, planter skip, storm damage, water, waterway and weed cluster. All of these patterns have substa...</p>
          <p><a href="/agriculture_vision/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://arxiv.org/abs/2001.01306" class="" target="_none">Agriculture-Vision: A Large Aerial Image Database for Agricultural Pattern Analysis</a> by Mang Tik Chiu, et al.</li></a>
          </li>
        </ul>
        <p><a href="/agriculture_vision/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="cord-19" class="dataset">
          <h3><a href="/cord-19/">COVID-19 Open Research Dataset (CORD-19)</a></h3>
          <p><span class="label label-info tag link-tag">coronavirus</span><span class="label label-info tag link-tag">COVID-19</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">MERS</span><span class="label label-info tag link-tag">SARS</span></p>
          <p>Full-text and metadata dataset of COVID-19 and coronavirus-related research articles optimized for machine readability.</p>
          <p><a href="/cord-19/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.kaggle.com/allen-institute-for-ai/CORD-19-research-challenge" class="" target="_none">COVID-19 Open Research Dataset Challenge (CORD-19)</a> by Kaggle</li></a>
          </li>
        </ul>
        <p><a href="/cord-19/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="modis-astraea" class="dataset">
          <h3><a href="/modis-astraea/">MODIS MYD13A1, MOD13A1, MYD11A1, MOD11A1, MCD43A4</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">natural resource</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info-sustainability tag link-tag">sustainability</span></p>
          <p>Data from the Moderate Resolution Imaging Spectroradiometer (MODIS), managed by
the U.S. Geological Survey and NASA. Five products are included:
MCD43A4 (MODIS/Terra and Aqua Nadir BRDF-Adjusted Reflectance Daily L3 Global 500 m SIN Grid),
MOD11A1 (MODIS/Terra Land Surface Temperature/Emissivity Daily L3 Global 1 km SIN Grid),
MYD11A1 (MODIS/Aqua Land Surface Temperature/Emissivity Daily L3 Global 1 km SIN Grid),
MOD13A1 (MODIS/Terra Vegetation Indices 16-Day L3 Global 500 m SIN Grid),
and MYD13A1 (MODIS/Aqua Vegetation Indices 16-Day L3 Global 500 m SIN Grid).
MCD43A4 has global coverage, all...</p>
          <p><a href="/modis-astraea/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://astraea.earth/earth-ondemand/" class="" target="_none">Astraea Earth OnDemand</a> by <a href="https://astraea.earth" target="_none">Astraea, Inc.</a></li></a>
          </li>
        </ul>
        <p><a href="/modis-astraea/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="schweizer-haltestellen-oev" class="dataset">
          <h3><a href="/schweizer-haltestellen-oev/">Swiss Public Transport Stops</a></h3>
          <p><span class="label label-info tag link-tag">cities</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">infrastructure</span><span class="label label-info tag link-tag">mapping</span><span class="label label-info tag link-tag">traffic</span><span class="label label-info tag link-tag">transportation</span></p>
          <p>The basic geo-data set for public transport stops comprises public transport stops in Switzerland and additional selected geo-referenced public transport locations that are of operational or structural importance (operating points).</p>
          <p><a href="/schweizer-haltestellen-oev/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://map.geo.admin.ch/?layers&#x3D;ch.bav.haltestellen-oev" class="" target="_none">Map Viewer</a> by <a href="https://twitter.com/swiss_geoportal" target="_none">Swiss Geoportal</a></li></a>
          </li>
        </ul>
        <p><a href="/schweizer-haltestellen-oev/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="digital-globe-open-data" class="dataset">
          <h3><a href="/digital-globe-open-data/">DigitalGlobe Open Data Program</a></h3>
          <p><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info-sustainability tag link-tag">sustainability</span></p>
          <p>Pre and post event high-resolution satellite imagery in support of emergency planning, risk assessment, monitoring of staging areas and emergency response, damage assessment, and recovery. Also incudes crowdsourced damage assessments for major, sudden onset disasters.</p>
          <p><a href="/digital-globe-open-data/">Details &rarr;</a></p>
        </div>
      </div>
    </div>


    <hr/>
  </div>  </body>
  <script src="/js/index.js"></script>
</html>
