<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    
  
    
  
    <link rel="alternate" type="application/rss+xml" href="/rss.xml" />
    
    <link rel="shortcut icon" type="image/ico" href="https://a0.awsstatic.com/libra-css/images/site/fav/favicon.ico"/>
    <link rel="apple-touch-icon" sizes="57x57" href="https://a0.awsstatic.com/libra-css/images/site/touch-icon-iphone-114-smile.png"/>
    <link rel="apple-touch-icon" sizes="72x72" href="https://a0.awsstatic.com/libra-css/images/site/touch-icon-ipad-144-smile.png"/>
    <link rel="apple-touch-icon" sizes="114x114" href="https://a0.awsstatic.com/libra-css/images/site/touch-icon-iphone-114-smile.png"/>
    <link rel="apple-touch-icon" sizes="144x144" href="https://a0.awsstatic.com/libra-css/images/site/touch-icon-ipad-144-smile.png"/>  
  
    <title>Registry of Open Data on AWS</title>
  
    <!-- Latest compiled and minified CSS -->
    <link rel="stylesheet" href="https://assets.opendata.aws/css/bootstrap/3.4.1/bootstrap.min.css">
  
    <!-- Our local CSS -->
    <link rel="stylesheet" href="/css/main.css">
  
  
  </head>
  <body>
    <!-- jQuery (necessary for Bootstrap's JavaScript plugins) -->
    <script src="https://assets.opendata.aws/js/jquery/3.5.1/jquery.min.js"></script>
    <!-- Latest compiled and minified JavaScript -->
    <script src="https://assets.opendata.aws/js/bootstrap/3.4.1/bootstrap.min.js"></script>
    <nav class="navbar navbar-default navbar-fixed-top">
      <div class="container">
        <div class="roda-header">
          <h2><a href="/" alt="Home">Registry of Open Data on AWS</a></h2>
          <a href="http://aws.amazon.com/what-is-cloud-computing">
            <img src="https://assets.opendata.aws/img/AWS-Logo_White-Color_300x180.png" alt="Powered by AWS Cloud Computing" id="aws_header_logo">
          </a>
        </div>
      </div>
    </nav>
    <div class="container" >

    <div class="about col-md-5">
      <div class="row aboutbox">
            <h3>About</h3>
            <p>This registry exists to help people discover and share datasets that are available via AWS resources. <a href="https://opendata.aws">Learn more about sharing data on AWS</a>.</p>
    
            <p>See <a href="/tag/life-sciences/usage-examples">all usage examples for datasets listed in this registry</a> tagged with <strong>life sciences</strong>.</p>
    
        <hr>
    
        <h4>Search datasets (currently <span id="count-matching">13</span> matching <span id="count-matching-text">datasets</span>)</h4>
        <form>
          <div class="form-group">
            <input type="text" id="search-box" class="form-control" placeholder="Search datasets" spellcheck="false" autocorrect="off">
          </div>
        </form>
        <p>You are currently viewing a subset of data tagged with <strong>life sciences</strong>.</p>
    
        <hr>
    
        <h4>Add to this registry</h4>
        <p>If you want to add a dataset or example of how to use a dataset to this registry, please follow the instructions on the <a href="https://github.com/awslabs/open-data-registry/">Registry of Open Data on AWS GitHub repository</a>.</p>
    
        <p>Unless specifically stated in the applicable dataset documentation, datasets available through the Registry of Open Data on AWS are not provided and maintained by AWS. Datasets are provided and maintained by a variety of third parties under a variety of licenses. Please check dataset licenses and related documentation to determine if a dataset may be used for your application.</p>
      </div>
    </div>
    <div class="col-md-6 col-md-offset-1 datasets">
      <div class="row">
        <div id="tcga" class="dataset">
          <h3><a href="/tcga/">The Cancer Genome Atlas</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">STRIDES</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>The Cancer Genome Atlas (TCGA), a collaboration between the National Cancer Institute (NCI) and National Human Genome Research Institute (NHGRI), aims to generate comprehensive, multi-dimensional maps of the key genomic changes in major types and subtypes of cancer. TCGA has analyzed matched tumor and normal tissues from 11,000 patients, allowing for the comprehensive characterization of 33 cancer types and subtypes, including 10 rare cancers.
The dataset contains open Clinical Supplement, Biospecimen Supplement, RNA-Seq Gene Expression Quantification, miRNA-Seq Isoform Expression Quantificati...</p>
          <p><a href="/tcga/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1016/j.celrep.2018.03.046" class="" target="_none">Machine Learning Detects Pan-Cancer Ras Pathway Activation in The Cancer Genome Atlas</a> by Gregory P. Way, Francisco Sanchez-Vega, et al.</li></a>
          </li>
          <li>
            <a href="https://portal.gdc.cancer.gov/projects/" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></li></a>
          </li>
          <li>
            <a href="https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga/studied-cancers
" class="" target="_none">TCGA Cancers Selected for Study</a> by <a href="https://ocg.cancer.gov/" target="_none">National Cancer Institute</a></li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1016/j.cels.2018.03.002" class="" target="_none">Scalable Open Science Approach for Mutation Calling of Tumor Exomes Using Multiple Genomic
Pipelines
</a> by Kyle Ellrott, Matthew H. Bailey, et al.</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1016/j.cels.2019.06.006" class="" target="_none">&quot;Before and After: A Comparison of Legacy and Harmonized TCGA Data at the Genomic Data
Commons&quot;
</a> by Galen F. Gao, Joel S. Parker, et al.</li></a>
          </li>
        </ul>
        <p><a href="/tcga/#usageexamples">See 29 usage examples &rarr;</a></p>
        </div>
        <div id="target" class="dataset">
          <h3><a href="/target/">Therapeutically Applicable Research to Generate Effective Treatments (TARGET)</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">STRIDES</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>Therapeutically Applicable Research to Generate Effective Treatments (TARGET) is the collaborative effort of a large, diverse consortium of extramural and NCI investigators. The goal of the effort is to accelerate molecular discoveries that drive the initiation and progression of hard-to-treat childhood cancers and facilitate rapid translation of those findings into the clinic.
TARGET projects provide comprehensive molecular characterization to determine the genetic changes that drive the initiation and progression of childhood cancers.The dataset contains open Clinical Supplement, Biospecimen...</p>
          <p><a href="/target/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term&#x3D;26121087" class="" target="_none">Relapsed neuroblastomas show frequent RAS-MAPK pathway mutations</a> by Eleveld TF, Oldridge DA, Bernard V, Koster J, et al.</li></a>
          </li>
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term&#x3D;29228567" class="" target="_none">Biomarker significance of plasma and tumor miR-21, miR-221, and miR-106a in osteosarcoma
</a> by Nakka M, Allen-Rhoades W, Li Y, et al.</li></a>
          </li>
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term&#x3D;23334666" class="" target="_none">The genetic landscape of high-risk neuroblastoma</a> by Pugh TJ, Morozova O, Attiyeh EF, Asgharzadeh S, Wei JS, Auclair D, Carter SL, Cibulskis K,
Hanna M, Kiezun A, Kim J, Lawrence MS, Lichenstein L, et al.
</li></a>
          </li>
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term&#x3D;28671688" class="" target="_none">The genomic landscape of pediatric and young adult T-lineage acute lymphoblastic leukemia
</a> by Yu Liu, John Easton, Ying Shao, et al.</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1038/s41586-018-0436-0" class="" target="_none">Genomic classification and identification of the cell of origin of pediatric mixed phenotype
acute leukemia
</a> by Thomas B. Alexander, Zhaohui Gu, Ilaria Iacobucci, et al.</li></a>
          </li>
        </ul>
        <p><a href="/target/#usageexamples">See 24 usage examples &rarr;</a></p>
        </div>
        <div id="kids-first" class="dataset">
          <h3><a href="/kids-first/">Gabriella Miller Kids First Pediatric Research Program (Kids First)</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">Homo sapiens</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">pediatric</span><span class="label label-info tag link-tag">STRIDES</span><span class="label label-info tag link-tag">structural birth defect</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>The NIH Common Fund&#39;s Gabriella Miller Kids First Pediatric Research Program’s (“Kids First”) vision is to “alleviate suffering from childhood cancer and structural birth defects by fostering collaborative research to uncover the etiology of these diseases and by supporting data sharing within the pediatric research community.” The program continues to generate and share whole genome sequence data from thousands of children affected by these conditions, ranging from rare pediatric cancers, such as osteosarcoma, to more prevalent diagnoses, such as congenital heart defects. In 2018, Kids Fi...</p>
          <p><a href="/kids-first/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/kids-first/" class="" target="_none">Kids First DRC Source Code</a> by <a href="https://kidsfirstdrc.org" target="_none">Kids First DRC</a></li></a>
          </li>
          <li>
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/31562520" class="" target="_none">Germline microsatellite genotypes differentiate children with medulloblastoma.</a> by Samuel Rivero-Hinojosa, Nicholas Kinney, et al.</li></a>
          </li>
          <li>
            <a href="https://pedcbioportal.kidsfirstdrc.org" class="" target="_none">PedcBioPortal</a> by <a href="https://www.cbioportal.org/" target="_none">cBioPortal</a></li></a>
          </li>
          <li>
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/31474320" class="" target="_none">Germline 16p11.2 Microdeletion Predisposes to Neuroblastoma.</a> by Laura Egolf, Zalman Vaksman, et al.</li></a>
          </li>
          <li>
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/31211835" class="" target="_none">Decreased ACKR3 (CXCR7) function causes oculomotor synkinesis in mice and humans.</a> by Mary C Whitman, Noriko Miyake, et al.</li></a>
          </li>
        </ul>
        <p><a href="/kids-first/#usageexamples">See 19 usage examples &rarr;</a></p>
        </div>
        <div id="foldingathome-covid19" class="dataset">
          <h3><a href="/foldingathome-covid19/">Foldingathome COVID-19 Datasets</a></h3>
          <p><span class="label label-info tag link-tag">alchemical free energy calculations</span><span class="label label-info tag link-tag">biomolecular modeling</span><span class="label label-info tag link-tag">coronavirus</span><span class="label label-info tag link-tag">COVID-19</span><span class="label label-info tag link-tag">foldingathome</span><span class="label label-info tag link-tag">health</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">molecular dynamics</span><span class="label label-info tag link-tag">protein</span><span class="label label-info tag link-tag">SARS-CoV-2</span><span class="label label-info tag link-tag">simulations</span><span class="label label-info tag link-tag">structural biology</span></p>
          <p><a href="http://foldingathome.org">Folding@home</a> is a massively distributed computing project that uses biomolecular simulations to investigate the <a href="https://foldingathome.org/diseases/">molecular origins of disease</a> and accelerate the discovery of new therapies. Run by the <a href="https://foldingathome.org/about/the-foldinghome-consortium/">Folding@home Consortium</a>, a worldwide network of research laboratories focusing on a variety of different diseases, Folding@home seeks to address problems in human health on a scale that is infeasible by another other means, sharing the results of these large-scale studies with the research community through <a href="https://foldingathome.org/papers-results/">peer-reviewed publications</a> and publicly shared datasets. During the <a href="https://en.wikipedia.org/wiki/Coronavirus_disease_2019">COVID-19 epidemic</a>, Folding@home focused its resources on understanding the vulernabilities in <a href="https://en.wikipedia.org/wiki/Severe_acute_respiratory_syndrome_coronavirus_2">SARS-CoV-2</a>, the virus that causes COVID-19 disease, and working closely with a number of experimental collaborators to accelerate progress toward effective therapies for treating COVID-19 and ending the pandemic. In the process, it created the world&#39;s first <a href="https://doi.org/10.1101/2020.06.27.175430">exascale distributed computing resource</a>, enabling it to generate valuable scientific datasets of unprecedented size. More information about Folding@home&#39;s COVID-19 research activities at the <a href="https://foldingathome.org/diseases/infectious-diseases/covid-19/">Folding@home COVID-19 page</a>. In addition to working directly with experimental collaborators and rapidly sharing new research findings through preprint servers, Folding@home has joined other researchers in committing to <a href="https://doi.org/10.1021/acs.jcim.0c00319">rapidly share all COVID-19 research data</a>, and has joined forces with <a href="https://aws.amazon.com/">AWS</a> and the <a href="http://molssi.org">Molecular Sciences Software Institute (MolSSI)</a> to share datasets of unprecented side through the <a href="https://registry.opendata.aws/">AWS Open Data Registry</a>, indexing these massive datsets via the <a href="https://covid.molssi.org/">MolSSI COVID-19 Molecular Structure and Therapeutics Hub</a>. The complete index of all Folding@home datasets can be found <a href="https://covid.molssi.org//org-contributions/#folding--home">here</a>. Th...</p>
          <p><a href="/foldingathome-covid19/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://covid.molssi.org//simulations/#foldinghome-simulations-of-nsp7-nsp7-nil" class="" target="_none">SARS-CoV-2 nsp7 simulations: A 3.7 ms dataset of the SARS-CoV-2 nsp7 protein in search of cryptic pockets</a> by <a href="https://bowmanlab.biochem.wustl.edu" target="_none">The Bowman lab at Washington University in St. Louis</a></li></a>
          </li>
          <li>
            <a href="https://covid.molssi.org//simulations/#foldinghome-sars-cov-2-main-protease-apo-monomer-simulations-3clpro-3clpro-protease-activity" class="" target="_none">SARS-CoV-2 main viral protease (Mpro, 3CLPro, nsp5) monomer simulations: A 2.6 ms equilibrium dataset of the SARS-CoV-2 main viral protease (apo, monomer)</a> by <a href="https://choderalab.org" target="_none">The Chodera lab at MSKCC</a></li></a>
          </li>
          <li>
            <a href="https://covid.molssi.org//simulations/#foldinghome-expanded-ensemble-absolute-free-energy-calculations-of-potential-small-molecule-inhibitors-of-the-sars-cov-2-main-protease-from-the-covid-moonshot-3clpro-3clpro-protease-activity" class="" target="_none">SARS-CoV-2 COVID Moonshot absolute free energy calculations</a> by <a href="http://www.voelzlab.org/" target="_none">The Voelz lab at Temple University</a></li></a>
          </li>
          <li>
            <a href="https://covid.molssi.org//simulations/#foldinghome-simulations-of-nsp3-pl2pro-domain-plpro-plpro-protease-activity" class="" target="_none">SARS-CoV-2 nsp3 pl2pro domain dataset: An 731 µs dataset of the SARS-CoV-2 nsp3 pl2pro domain in search of cryptic pockets</a> by <a href="https://bowmanlab.biochem.wustl.edu" target="_none">The Bowman lab at Washington University in St. Louis</a></li></a>
          </li>
          <li>
            <a href="https://covid.molssi.org//simulations/#foldinghome-simulations-of-nsp5-3clpro-3clpro-protease-activity" class="" target="_none">SARS-CoV-2 main viral protease (Mpro, 3CLPro, nsp5) dimer simulations: A 2.9 ms dataset of the SARS-CoV-2 main viral protease (apo, dimer) in search of cryptic pockets</a> by <a href="https://bowmanlab.biochem.wustl.edu" target="_none">The Bowman lab at Washington University in St. Louis</a></li></a>
          </li>
        </ul>
        <p><a href="/foldingathome-covid19/#usageexamples">See 15 usage examples &rarr;</a></p>
        </div>
        <div id="broad-gnomad" class="dataset">
          <h3><a href="/broad-gnomad/">Genome Aggregation Database (gnomAD)</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">population</span><span class="label label-info tag link-tag">population genetics</span><span class="label label-info tag link-tag">short read sequencing</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>The Genome Aggregation Database (gnomAD) is a resource developed by an international coalition of investigators that aggregates and harmonizes both exome and genome data from a wide range of large-scale human sequencing projects. The summary data provided here are released for the benefit of the wider scientific community without restriction on use.
The v2 data set (GRCh37) spans 125,748 exome sequences and 15,708 whole-genome sequences from unrelated individuals. The v3 data set (GRCh38) spans 71,702 genomes, selected as in v2.
Sign up for the gnomAD mailing list <a href="http://broad.io/gnomad_list">here</a>.</p>
          <p><a href="/broad-gnomad/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1101/784157" class="" target="_none">Technical artifact drives apparent deviation from Hardy-Weinberg equilibrium at CCR5-∆32 and other variants in gnomAD. bioRxiv (p. 784157)</a> by Karczewski, K. J., Gauthier, L. D., Daly, M. J.</li></a>
          </li>
          <li>
            <a href="https://gnomad.broadinstitute.org/blog/2019-10-gnomad-v3-0/" class="" target="_none">gnomAD v3.0</a> by Laurent Francioli, Daniel MacArthur</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1038/s41467-019-10717-9" class="" target="_none">Characterising the loss-of-function impact of 5’ untranslated region variants in 15,708 individuals. Nature Communications 11, 2523 (2020)</a> by Whiffin, N., Karczewski, K. J., Zhang, X., Chothani, S., Smith, M. J., Gareth Evans, D., Roberts, A. M., Quaife, N. M., Schafer, S., Rackham, O., Alföldi, J., O’Donnell-Luria, A. H., Francioli, L. C., Genome Aggregation Database (gnomAD) Production Team, Genome Aggregation Database (gnomAD) Consortium, Cook, S. A., Barton, P. J. R., MacArthur, D. G., &amp; Ware, J. S.</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1038/nature19057" class="" target="_none">Analysis of protein-coding genetic variation in 60,706 humans. Nature 536, 285–291 (2016)</a> by Lek, M., Karczewski, K., Minikel, E. et al.</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1038/s41586-020-2287-8" class="" target="_none">A structural variation reference for medical and population genetics. Nature 581, 444–451 (2020)</a> by Collins, R. L., Brand, H., Karczewski, K. J., Zhao, X., Alföldi, J., Francioli, L. C., Khera, A. V., Lowther, C., Gauthier, L. D., Wang, H., Watts, N. A., Solomonson, M., O’Donnell-Luria, A., Baumann, A., Munshi, R., Walker, M., Whelan, C., Huang, Y., Brookings, T., ... Talkowski, M. E.</li></a>
          </li>
        </ul>
        <p><a href="/broad-gnomad/#usageexamples">See 15 usage examples &rarr;</a></p>
        </div>
        <div id="allen-cell-imaging-collections" class="dataset">
          <h3><a href="/allen-cell-imaging-collections/">Allen Cell Imaging Collections</a></h3>
          <p><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">cell biology</span><span class="label label-info tag link-tag">cell imaging</span><span class="label label-info tag link-tag">Homo sapiens</span><span class="label label-info tag link-tag">image processing</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">microscopy</span></p>
          <p>This bucket contains multiple datasets (as Quilt packages) created by the
Allen Institute for Cell Science (AICS). The imaging data in this bucket contains
either of the following:1) field of view images from glass plates
2) cell membrane, DNA, and structure segmentations
3) cell membrane, DNA and structure contours
4) machine learning imaging predictions of the previously listed modalities.In addition, many of the datasets include CSVs that contain feature sets
related to that data.</p>
          <p><a href="/allen-cell-imaging-collections/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.biorxiv.org/content/10.1101/2020.12.08.415562v1" class="" target="_none">Robust integrated intracellular organization of the human iPS cell: where, how much, and how variable?</a> by <a href="https://www.alleninstitute.org/what-we-do/cell-science/about/team/" target="_none">Matheus P. Viana, Jianxu Chen*, Theo A. Knijnenburg*, Ritvik Vasan*, Calysta Yan*... Allen Institute for Cell Science... Graham T. Johnson, Ruwanthi N. Gunawardane, Nathalie Gaudreault, Julie A. Theriot, Susanne M. Rafelski</a></li></a>
          </li>
          <li>
            <a href="https://github.com/AllenCellModeling/aicsimageio" class="" target="_none">AICSImageIO</a> by <a href="https://www.alleninstitute.org/what-we-do/cell-science/about/team/" target="_none">Matthew Bowden, Jackson Brown, Jamie Sherman, Dan Toloudis</a></li></a>
          </li>
          <li>
            <a href="https://imsc.allencell.org" class="" target="_none">Integrated Mitotic Stem Cell</a> by <a href="https://www.alleninstitute.org/what-we-do/cell-science/about/team/" target="_none">Allen Institute for Cell Science</a></li></a>
          </li>
          <li>
            <a href="https://www.allencell.org/visual-guide-to-human-cells.html" class="" target="_none">Visual Guide to Human Cells</a> by <a href="https://www.alleninstitute.org/what-we-do/cell-science/about/team/" target="_none">Allen Institute for Cell Science</a></li></a>
          </li>
          <li>
            <a href="https://www.allencell.org/segmenter.html" class="" target="_none">Allen Cell Structure Segmenter</a> by <a href="https://www.alleninstitute.org/what-we-do/cell-science/about/team/" target="_none">Jianxu Chen, Liya Ding, Matheus P. Viana, Melissa C. Hendershott, Ruian Yang, Irina A. Mueller, Susanne M. Rafelski</a></li></a>
          </li>
        </ul>
        <p><a href="/allen-cell-imaging-collections/#usageexamples">See 11 usage examples &rarr;</a></p>
        </div>
        <div id="fcp-indi" class="dataset">
          <h3><a href="/fcp-indi/">International Neuroimaging Data-Sharing Initiative (INDI)</a></h3>
          <p><span class="label label-info tag link-tag">Homo sapiens</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">magnetic resonance imaging</span><span class="label label-info tag link-tag">neuroimaging</span><span class="label label-info tag link-tag">neuroscience</span></p>
          <p>This bucket contains multiple neuroimaging datasets that are part of the International Neuroimaging Data-Sharing Initiative. Raw human and non-human primate neuroimaging data include 1) Structural MRI; 2) Functional MRI; 3) Diffusion Tensor Imaging; 4) Electroencephalogram (EEG)
In addition to the raw data, preprocessed data is also included for some datasets.
A complete list of the available datasets can be seen in the documentation lonk provided below. </p>
          <p><a href="/fcp-indi/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.sciencedirect.com/science/article/pii/S0896627318307682" class="" target="_none">An Open Resource for Non-human Primate Imaging</a> by M.P. Milham, L. Ai, ..., C.E. Schroeder</li></a>
          </li>
          <li>
            <a href="https://academic.oup.com/gigascience/article/6/2/giw011/2865212" class="" target="_none">The Healthy Brain Network Serial Scanning Initiative: a resource for evaluating inter-individual differences and their reliabilities across scan conditions and sessions</a> by D. O&#x27;Connor, N.V. Potler, ..., M.P. Milham</li></a>
          </li>
          <li>
            <a href="https://www.nature.com/articles/sdata201710" class="" target="_none">Enhancing studies of the connectome in autism using the autism brain imaging data exchange II.</a> by A. Di Martino, D. O&#x27;Connor, M.P. Milham</li></a>
          </li>
          <li>
            <a href="https://fcp-indi.github.io/docs/user/index.html#" class="" target="_none">Configurable Pipeline for the Analysis of Connectomes (C-PAC)</a> by [INDI C-PAC Team](https://fcp-indi.github.io/)</li></a>
          </li>
          <li>
            <a href="https://www.sciencedirect.com/science/article/abs/pii/S1053811912010671?via%3Dihub" class="" target="_none">Making data sharing work: The FCP/INDI experience</a> by M. Mennes, B.B. Biswal, F.X. Castellanos, M.P. Milham</li></a>
          </li>
        </ul>
        <p><a href="/fcp-indi/#usageexamples">See 11 usage examples &rarr;</a></p>
        </div>
        <div id="janelia-flylight" class="dataset">
          <h3><a href="/janelia-flylight/">Fly Brain Anatomy: FlyLight Gen1 and Split-GAL4 Imagery</a></h3>
          <p><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">fluorescence imaging</span><span class="label label-info tag link-tag">image processing</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">microscopy</span><span class="label label-info tag link-tag">neurobiology</span><span class="label label-info tag link-tag">neuroimaging</span><span class="label label-info tag link-tag">neuroscience</span></p>
          <p>This data set, made available by Janelia&#39;s FlyLight project, consists of fluorescence images 
of Drosophila melanogaster driver lines, aligned to standard templates, and stored in formats 
suitable for rapid searching and visualization. Additional data will be added as it is published. 
A large release of Gen1 MCFO samples is coming at the beginning of May 2020. </p>
          <p><a href="/janelia-flylight/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="http://splitgal4.janelia.org/cgi-bin/splitgal4.cgi" class="" target="_none">Fly Light Split-GAL4 Driver Collection</a> by Rob Svirskas</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1016/j.celrep.2012.09.011" class="" target="_none">A GAL4-Driver Line Resource for Drosophila Neurobiology</a> by Jenett, A., Rubin, G. M., Ngo, T., Shepherd, D., Murphy, C., Dionne, H., Pfeiffer, B. D., Cavallaro, A., Hall, D., Jeter, J., Iyer, N., Fetter, D., Hausenfluck, J. H., Peng, H., Trautman, E. T., Svirskas, R. R., Myers, E. W., Iwinski, Z. R., Aso, Y., DePasquale, G. M., Enos, A., Hulamm, P., Lam, S. C. B., Li, H., Laverty, T. R., Long, F., Qu, L., Murphy, S. D., Rokicki, K., Safford, T., Shaw, K., Simpson, J. H., Sowell, A., Tae, S., Yu, Y., &amp; Zugates, C. T.</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.7554/eLife.04577" class="" target="_none">The neuronal architecture of the mushroom body provides a logic for associative learning</a> by Aso, Y., Hattori, D., Yu, Y., Johnston, R. M., Iyer, N. A., Ngo, T., Dionne, H., Abbott, L., Axel, R., Tanimoto, H., &amp; Rubin, G. M.</li></a>
          </li>
          <li>
            <a href="https://neuronbridge.janelia.org" class="" target="_none">NeuronBridge</a> by Jody Clements, Rob Svirskas, Hideo Otsuna, Cristian Goina, Konrad Rokicki</li></a>
          </li>
          <li>
            <a href="https://github.com/JaneliaSciComp/open-data-flylight/blob/master/tutorials/Using_imagery_on_AWS_S3.ipynb" class="" target="_none">Using Imagery on AWS S3</a> by Rob Svirskas</li></a>
          </li>
        </ul>
        <p><a href="/janelia-flylight/#usageexamples">See 10 usage examples &rarr;</a></p>
        </div>
        <div id="open-neurodata" class="dataset">
          <h3><a href="/open-neurodata/">Open NeuroData</a></h3>
          <p><span class="label label-info tag link-tag">array tomography</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">electron microscopy</span><span class="label label-info tag link-tag">image processing</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">lightsheet microscopy</span><span class="label label-info tag link-tag">magnetic resonance imaging</span><span class="label label-info tag link-tag">neuroimaging</span><span class="label label-info tag link-tag">neuroscience</span></p>
          <p>This bucket contains multiple neuroimaging datasets (as Neuroglancer Precomputed Volumes) across multiple modalities and scales, ranging from nanoscale (electron microscopy), to microscale (cleared lightsheet microscopy and array tomography), and mesoscale (structural and functional magnetic resonance imaging). Additionally, many of the datasets include segmentations and meshes.</p>
          <p><a href="/open-neurodata/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://arxiv.org/abs/1306.3543" class="" target="_none">The Open Connectome Project Data Cluster: Scalable Analysis and Vision for High-Throughput Neuroscience</a> by R. Burns, W. G. Roncal, D. Kleissas, K. Lillaney, P. Manavalan, E. Perlman, D. R. Berger, D. D. Bock, K. Chung, L. Grosenick, N. Kasthuri, N. C. Weiler, K. Deisseroth, M. Kazhdan, J. Lichtman, R. C. Reid, S. J. Smith, A. S. Szalay, J. T. Vogelstein, and R. J. Vogelstein.</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1016/j.neuron.2014.08.045" class="" target="_none">From cosmos to connectomes: The evolution of data-intensive science</a> by R. Burns, J. T. Vogelstein, and A. S. Szalay</li></a>
          </li>
          <li>
            <a href="https://github.com/seung-lab/igneous" class="" target="_none">Igneous</a> by <a href="https://github.com/william-silversmith" target="_none">William Silversmith</a></li></a>
          </li>
          <li>
            <a href="https://neurodata.io/help/visualization/" class="" target="_none">Visualization using Neuroglancer</a> by <a href="https://github.com/falkben" target="_none">Benjamin Falk</a></li></a>
          </li>
          <li>
            <a href="https://www.nature.com/articles/s41592-018-0181-1" class="" target="_none">A Community-Developed Open-Source Computational Ecosystem for Big Neuro Data</a> by J. T. Vogelstein, E. Perlman, B. Falk, A. Baden, W. Gray Roncal, V. Chandrashekhar, F. Collman, S. Seshamani, J. L. Patsolic, K. Lillaney, M. Kazhdan, R. Hider, D. Pryor, J. Matelsky, T. Gion, P. Manavalan, B. Wester, M. Chevillet, E. T. Trautman, K. Khairy, E. Bridgeford, D. M. Kleissas, D. J. Tward, A. K. Crow, B. Hsueh, M. A. Wright, M. I. Miller, S. J. Smith, R. J. Vogelstein, K. Deisseroth, and R. Burns</li></a>
          </li>
        </ul>
        <p><a href="/open-neurodata/#usageexamples">See 9 usage examples &rarr;</a></p>
        </div>
        <div id="ccle" class="dataset">
          <h3><a href="/ccle/">Cancer Cell Line Encyclopedia (CCLE)</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">Homo sapiens</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">STRIDES</span><span class="label label-info tag link-tag">transcriptomics</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>The Cancer Cell Line Encyclopedia (CCLE) project is an effort to conduct a detailed genetic
characterization of a large panel of human cancer cell lines. The CCLE provides public access to
genomic data, visualization and analysis for over 1100 cancer cell lines. This dataset contains
RNA-Seq Aligned Reads, WXS Aligned Reads, and WGS Aligned Reads data.</p>
          <p><a href="/ccle/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://portal.gdc.cancer.gov/legacy-archive/" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1038/nature11003" class="" target="_none">The Cancer Cell Line Encyclopedia enables predictive modelling of anticancer drug
sensitivity
</a> by Barretina Caponigro Stransky et al.</li></a>
          </li>
          <li>
            <a href="https://console.cloud.google.com/bigquery?p&#x3D;isb-cgc&amp;d&#x3D;CCLE_bioclin_v0&amp;page&#x3D;dataset" class="" target="_none">ISB CGC BigQuery tables</a> by <a href="https://isb-cancer-genomics-cloud.readthedocs.io/en/latest/sections/data/CCLE_top.html
" target="_none">Institute for Systems Biology</a></li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1038/nature15736" class="" target="_none">Pharmacogenomic agreement between two cancer cell line data sets</a> by The Cancer Cell Line Encyclopedia Consortium &amp; The Genomics of Drug Sensitivity in Cancer
Consortium
</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1038/s41591-019-0404-8" class="" target="_none">The landscape of cancer cell line metabolism</a> by Li, H. et al.</li></a>
          </li>
        </ul>
        <p><a href="/ccle/#usageexamples">See 8 usage examples &rarr;</a></p>
        </div>
        <div id="ebirdst" class="dataset">
          <h3><a href="/ebirdst/">eBird Status and Trends Model Results</a></h3>
          <p><span class="label label-info tag link-tag">biodiversity</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">ecosystems</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info-sustainability tag link-tag">sustainability</span></p>
          <p>The eBird Status and Trends project generates estimates of bird
occurrence and abundance at a high spatiotemporal resolution.
This dataset represents the primary modeled results from the
analysis workflow and are designed for further analysis,
synthesis, visualization, and exploration.</p>
          <p><a href="/ebirdst/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://cornelllabofornithology.github.io/ebirdst/" class="" target="_none">ebirdst R Package</a> by Cornell Lab of Ornithology</li></a>
          </li>
          <li>
            <a href="https://cornelllabofornithology.github.io/ebirdst/articles/ebirdst-introduction.html" class="" target="_none">ebirdst - Background, Data Access and Data Structure</a> by Tom Auer and Daniel Fink</li></a>
          </li>
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/31837058" class="" target="_none">Modeling avian full annual cycle distribution and population trends with citizen science data</a> by Fink et al (2019)</li></a>
          </li>
          <li>
            <a href="https://cornelllabofornithology.github.io/ebirdst/articles/ebirdst-intro-mapping.html" class="" target="_none">ebirdst - Intro Mapping</a> by Tom Auer and Daniel Fink</li></a>
          </li>
          <li>
            <a href="https://cornelllabofornithology.github.io/ebirdst/articles/ebirdst-advanced-mapping.html" class="" target="_none">ebirdst - Generating Seasonal Abundance and Range Maps and Stats</a> by Matt Strimas-Mackey, Tom Auer, and Daniel Fink</li></a>
          </li>
        </ul>
        <p><a href="/ebirdst/#usageexamples">See 8 usage examples &rarr;</a></p>
        </div>
        <div id="cptac-2" class="dataset">
          <h3><a href="/cptac-2/">Clinical Proteomic Tumor Analysis Consortium 2 (CPTAC-2)</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">STRIDES</span><span class="label label-info tag link-tag">transcriptomics</span></p>
          <p>The Clinical Proteomic Tumor Analysis Consortium (CPTAC) is a national effort to accelerate the
understanding of the molecular basis of cancer through the application of large-scale proteome and
genome analysis, or proteogenomics. CPTAC-2 is the Phase II of the CPTAC Initiative (2011-2016).
Datasets contain open RNA-Seq Gene Expression Quantification, miRNA-Seq Isoform Expression
Quantification, and miRNA Expression Quantification data.</p>
          <p><a href="/cptac-2/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://pdc.esacinc.com/pdc/browse" class="" target="_none">Proteomic Data Commons</a> by <a href="https://pdc.esacinc.com/pdc" target="_none">National Cancer Institute</a></li></a>
          </li>
          <li>
            <a href="https://portal.gdc.cancer.gov/projects/CPTAC-2" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1038/sdata.2015.22" class="" target="_none">Proteomic analysis of colon and rectal carcinoma using standard and customized databases
</a> by Slebos RJ, Wang X, Wang X, Zhang B, Tabb DL, Liebler DC</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1016/j.cell.2019.03.030" class="" target="_none">Proteogenomic Analysis of Human Colon Cancer Reveals New Therapeutic Opportunities</a> by Suhas Vasaikar, Chen Huang, Xiaojing Wang. Vladislav A. Petyuk, Sara R. Savage, Bo Wen,
Yongchao Dou, Yun Zhang, Zhiao Shi, Osama A. Arshad, Marina A. Gritsenko, Lisa J. Zimmerman,
Jason E. McDermott, Therese R. Clauss, Ronald J. Moore, Rui Zhao, Matthew E. Monroe, Yi-Ting
Wang, Matthew C. Chambers, Robbert J.C. Slebos, Ken S. Lau, Qianxing Mo, Li Ding, Matthew
Ellis, Mathangi Thiagarajan, Christopher R. Kinsinger, Henry Rodriguez, Richard D. Smith,
Karin D. Rodland, Daniel C. Liebler, Tao Liu, Bing Zhang, Clinical Proteomic Tumor Analysis
Consortium
</li></a>
          </li>
          <li>
            <a href="https://cgc-accounts.sbgenomics.com/" class="" target="_none">Cancer Genomics Cloud</a> by <a href="http://www.cancergenomicscloud.org/" target="_none">Seven Bridges</a></li></a>
          </li>
        </ul>
        <p><a href="/cptac-2/#usageexamples">See 7 usage examples &rarr;</a></p>
        </div>
        <div id="icgc" class="dataset">
          <h3><a href="/icgc/">ICGC on AWS</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span></p>
          <p>The International Cancer Genome Consortium (ICGC) coordinates projects with the common aim of accelerating research into the causes and control of cancer. The PanCancer Analysis of Whole Genomes (PCAWG) study is an international collaboration to identify common patterns of mutation in whole genomes from ICGC. More than 2,400 consistently analyzed genomes corresponding to over 1,100 unique ICGC donors are now freely available on Amazon S3 to credentialed researchers subject to ICGC data sharing policies.</p>
          <p><a href="/icgc/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.nature.com/articles/s41586-020-1965-x" class="" target="_none">Analyses of non-coding somatic drivers in 2,693 cancer whole genomes</a> by <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term&#x3D;Rheinbay%20E%5BAuthor%5D&amp;cauthor&#x3D;true&amp;cauthor_uid&#x3D;28658208" target="_none">Rheinbay et al (2020)</a></li></a>
          </li>
          <li>
            <a href="https://www.nature.com/articles/s41586-019-1907-7" class="" target="_none">The evolutionary history of 2,658 cancers</a> by <a href="https://www.ncbi.nlm.nih.gov/pubmed?cmd&#x3D;search&amp;term&#x3D;Moritz%20Gerstung" target="_none">Gerstung et al (2020)</a></li></a>
          </li>
          <li>
            <a href="https://www.nature.com/articles/s41586-019-1913-9" class="" target="_none">Patterns of somatic structural variation in human cancer genomes</a> by <a href="https://www.ncbi.nlm.nih.gov/pubmed?cmd&#x3D;search&amp;term&#x3D;Yilong%20Li" target="_none">Li et al. (2020)</a></li></a>
          </li>
          <li>
            <a href="https://github.com/ICGC-TCGA-PanCancer/cli/wiki" class="" target="_none">AWS Pancancer Workflow Launcher</a> by <a href="https://github.com/solomonshorser" target="_none">Solomon Shorser</a></li></a>
          </li>
          <li>
            <a href="https://www.nature.com/articles/s41586-020-1943-3" class="" target="_none">The repertoire of mutational signatures in human cancer</a> by <a href="https://www.ncbi.nlm.nih.gov/pubmed?cmd&#x3D;search&amp;term&#x3D;Ludmil%20B.%20Alexandrov" target="_none">Alexandrov et al (2020)</a></li></a>
          </li>
        </ul>
        <p><a href="/icgc/#usageexamples">See 7 usage examples &rarr;</a></p>
        </div>
        <div id="cptac-3" class="dataset">
          <h3><a href="/cptac-3/">Clinical Proteomic Tumor Analysis Consortium 3 (CPTAC-3)</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">STRIDES</span><span class="label label-info tag link-tag">transcriptomics</span></p>
          <p>The Clinical Proteomic Tumor Analysis Consortium (CPTAC) is a national effort to accelerate the
understanding of the molecular basis of cancer through the application of large-scale proteome and
genome analysis, or proteogenomics. CPTAC-3 is the Phase III of the CPTAC Initiative. The dataset
contains open RNA-Seq Gene Expression Quantification data.</p>
          <p><a href="/cptac-3/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://cgc-accounts.sbgenomics.com/" class="" target="_none">Cancer Genomics Cloud</a> by <a href="http://www.cancergenomicscloud.org/" target="_none">Seven Bridges</a></li></a>
          </li>
          <li>
            <a href="https://portal.gdc.cancer.gov/projects/CPTAC-3" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></li></a>
          </li>
          <li>
            <a href="https://pdc.esacinc.com/pdc/browse" class="" target="_none">Proteomic Data Commons</a> by <a href="https://pdc.esacinc.com/pdc" target="_none">National Cancer Institute</a></li></a>
          </li>
          <li>
            <a href="https://proteomics.cancer.gov/data-portal" class="" target="_none">CPTAC Data Portal</a> by <a href="https://proteomics.cancer.gov/" target="_none">National Cancer Institute</a></li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1016/j.cell.2019.10.007" class="" target="_none">Integrated Proteogenomic Characterization of Clear Cell Renal Cell Carcinoma</a> by Clark DJ, Dhanasekaran SM, Petralia F, Pan J, Song X, Hu Y, da Veiga Leprevost F, Reva B,
Lih TM, Chang HY, Ma W, Huang C, Ricketts CJ, Chen L1, Krek A, Li Y, Rykunov D, Li QK, Chen
LS, Ozbek U, Vasaikar S, Wu Y, Yoo S, Chowdhury S, Wyczalkowski MA, Ji J, Schnaubelt M, Kong
A, Sethuraman S, Avtonomov DM, Ao M, Colaprico A, Cao S, Cho KC, Kalayci S, Ma S, Liu W,
Ruggles K, Calinawan A, Gümüş ZH, Geizler D, Kawaler E, Teo GC, Wen B, Zhang Y, Keegan S, Li
K, Chen F, Edwards N, Pierorazio PM, Chen XS, Pavlovich CP, Hakimi AA, Brominski G, Hsieh
JJ, Antczak A, Omelchenko T, Lubinski J, Wiznerowicz M, Linehan WM, Kinsinger CR,
Thiagarajan M, Boja ES, Mesri M, Hiltke T, Robles AI, Rodriguez H, Qian J, Fenyö D, Zhang B,
Ding L, Schadt E, Chinnaiyan AM, Zhang Z, Omenn GS, Cieslik M, Chan DW, Nesvizhskii AI, Wang
P, Zhang H; Clinical Proteomic Tumor Analysis Consortium
</li></a>
          </li>
        </ul>
        <p><a href="/cptac-3/#usageexamples">See 6 usage examples &rarr;</a></p>
        </div>
        <div id="nyu-fastmri" class="dataset">
          <h3><a href="/nyu-fastmri/">NYU Langone &amp; FAIR FastMRI Dataset</a></h3>
          <p><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">health</span><span class="label label-info tag link-tag">image processing</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">magnetic resonance imaging</span><span class="label label-info tag link-tag">neurobiology</span><span class="label label-info tag link-tag">neuroimaging</span></p>
          <p>This dataset contains deidentified raw k-space data and DICOM image files of over 1,500 knees and 6,970 brains.</p>
          <p><a href="/nyu-fastmri/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://arxiv.org/abs/2001.02518" class="" target="_none">Advancing machine learning for MR image reconstruction with an open competition:Overview of the 2019 fastMRI challenge</a> by <a href="https://arxiv.org/search/eess?searchtype&#x3D;author&amp;query&#x3D;Knoll%2C+F" target="_none">Knoll et al (2020)</a></li></a>
          </li>
          <li>
            <a href="https://arxiv.org/abs/1904.01112" class="" target="_none">Deep Learning Methods for Parallel Magnetic Resonance Image Reconstruction</a> by <a href="https://arxiv.org/search/eess?searchtype&#x3D;author&amp;query&#x3D;Knoll%2C+F" target="_none">Knoll et al (2019)</a></li></a>
          </li>
          <li>
            <a href="https://arxiv.org/abs/1811.08839" class="" target="_none">fastMRI:An Open Dataset and Benchmarks for Accelerated MRI</a> by <a href="https://arxiv.org/search/cs?searchtype&#x3D;author&amp;query&#x3D;Zbontar%2C+J" target="_none">Zbontar et al (2019)</a></li></a>
          </li>
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/29774597" class="" target="_none">Assessment of the generalization of learned image reconstruction and the potential for transfer learning.</a> by <a href="https://arxiv.org/search/eess?searchtype&#x3D;author&amp;query&#x3D;Knoll%2C+F" target="_none">Knoll et al (2019)</a></li></a>
          </li>
          <li>
            <a href="https://medium.com/@khodges42/how-to-get-started-diy-neurohacking-with-the-fastmri-dataset-and-why-you-should-6a02a6bb896e" class="" target="_none">How to Get Started Working with the FastMRI Dataset (and why you should!)</a> by Kevin Hodges</li></a>
          </li>
        </ul>
        <p><a href="/nyu-fastmri/#usageexamples">See 6 usage examples &rarr;</a></p>
        </div>
        <div id="aws-covid19-lake" class="dataset">
          <h3><a href="/aws-covid19-lake/">COVID-19 Data Lake</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">coronavirus</span><span class="label label-info tag link-tag">COVID-19</span><span class="label label-info tag link-tag">health</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">medicine</span><span class="label label-info tag link-tag">MERS</span><span class="label label-info tag link-tag">SARS</span></p>
          <p>A centralized repository of up-to-date and curated datasets on or related to the spread and characteristics of the novel corona virus (SARS-CoV-2) and its associated illness, COVID-19. Globally, there are several efforts underway to gather this data, and we are working with partners to make this crucial data freely available and keep it up-to-date. Hosted on the AWS cloud, we have seeded our curated data lake with COVID-19 case tracking data from Johns Hopkins and The New York Times, hospital bed availability from Definitive Healthcare, and over 45,000 research articles about COVID-19 and rela...</p>
          <p><a href="/aws-covid19-lake/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://us-east-2.console.aws.amazon.com/cloudformation/home?region&#x3D;us-east-2#/stacks/create/review?templateURL&#x3D;https://covid19-lake.s3.us-east-2.amazonaws.com/cfn/CovidLakeStack.template.json&amp;stackName&#x3D;CovidLakeStack" class="" target="_none">CloudFormation template for Glue Catalog table definitions</a> by <a href="https://aws.amazon.com/big-data/datalakes-and-analytics/" target="_none">AWS Data Lake Team</a></li></a>
          </li>
          <li>
            <a href="https://aws.amazon.com/blogs/big-data/exploring-the-public-aws-covid-19-data-lake/" class="aws-link" target="_none">Exploring the public AWS COVID-19 data lake</a> by <a href="https://aws.amazon.com/big-data/datalakes-and-analytics/" target="_none">AWS Data Lake Team</a></li></a>
          </li>
          <li>
            <a href="https://go.aws/covid-19-data-lake" class="aws-link" target="_none">Explore the COVID-19 data lake public S3 bucket</a> by <a href="https://aws.amazon.com/big-data/datalakes-and-analytics/" target="_none">AWS Data Lake Team</a></li></a>
          </li>
          <li>
            <a href="https://aws.amazon.com/blogs/big-data/a-public-data-lake-for-analysis-of-covid-19-data/" class="aws-link" target="_none">A public data lake for analysis of COVID-19 data</a> by <a href="https://aws.amazon.com/big-data/datalakes-and-analytics/" target="_none">AWS Data Lake Team</a></li></a>
          </li>
          <li>
            <a href="https://github.com/aws-samples/query-data-in-s3-with-amazon-athena-and-aws-sdk-for-dotnet" class="aws-link" target="_none">How to use SQL to query data in S3 Bucket with Amazon Athena and AWS SDK for .NET</a> by <a href="https://github.com/orgs/aws-samples/teams/aws-proserve-us-west-applications" target="_none">AWS ProServe US West Applications Team</a></li></a>
          </li>
        </ul>
        <p><a href="/aws-covid19-lake/#usageexamples">See 5 usage examples &rarr;</a></p>
        </div>
        <div id="ilmn-dragen-1kgp" class="dataset">
          <h3><a href="/ilmn-dragen-1kgp/">1000 Genomes Phase 3 Reanalysis with DRAGEN 3.5</a></h3>
          <p><span class="label label-info tag link-tag">bam</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">health</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">vcf</span></p>
          <p>This dataset contains alignment files and short nucleotide, copy number, and structural variant call files from the newly revised (n=3202) 1000 Genomes Project Phase 3 using Illumina DRAGEN v3.5.7b</p>
          <p><a href="/ilmn-dragen-1kgp/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://support.illumina.com/content/dam/illumina-support/documents/downloads/software/dragen/DRAGEN-3.5.7b-Customer-Release-Notes-1000000131537-00.pdf" class="" target="_none">DRAGEN 3.7.5b Release</a> by <a href="https://www.illumina.com/" target="_none">Illumina Inc.</a></li></a>
          </li>
          <li>
            <a href="https://www.biorxiv.org/content/10.1101/2020.11.13.380741v1" class="" target="_none">precisionFDA Truth Challenge V2: Calling variants from short- and long-reads in difficult-to-map regions (Preprint)</a> by Olson et al (2020)</li></a>
          </li>
          <li>
            <a href="https://github.com/aws-quickstart/quickstart-illumina-dragen" class="" target="_none">DRAGEN Quick Start on AWS</a> by <a href="https://github.com/aws-quickstart" target="_none">AWS Quick Start Team</a></li></a>
          </li>
          <li>
            <a href="https://www.illumina.com/products/by-type/informatics-products/basespace-sequence-hub.html#key-features" class="" target="_none">DRAGEN on BaseSpace Sequence Hub</a> by <a href="https://www.illumina.com/" target="_none">Illumina Inc.</a></li></a>
          </li>
        </ul>
        <p><a href="/ilmn-dragen-1kgp/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="3kricegenome" class="dataset">
          <h3><a href="/3kricegenome/">3000 Rice Genomes Project</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">food security</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span></p>
          <p>The 3000 Rice Genome Project is an international effort to sequence the genomes of 3,024 rice varieties from 89 countries.</p>
          <p><a href="/3kricegenome/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://genome.cshlp.org/content/29/5/870.short" class="" target="_none">Structural variants in 3000 rice genomes</a> by Fuentes RR et al (2019)</li></a>
          </li>
          <li>
            <a href="https://github.com/InternationalRiceResearchInstitute/RiceGalaxy" class="" target="_none">RiceGalaxy</a> by <a href="https://www.irri.org/" target="_none">International Rice Research Institute</a></li></a>
          </li>
          <li>
            <a href="https://www.nature.com/articles/s41467-019-08451-3" class="" target="_none">Tracking the origin of two genetic components associated with transposable element bursts in domesticated rice</a> by Chen J et al (2019)</li></a>
          </li>
          <li>
            <a href="https://academic.oup.com/gigascience/article/8/5/giz028/5492234" class="" target="_none">Rice Galaxy: an open resource for plant science</a> by Juanillas V et al (2019)</li></a>
          </li>
        </ul>
        <p><a href="/3kricegenome/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="encode-project" class="dataset">
          <h3><a href="/encode-project/">Encyclopedia of DNA Elements (ENCODE)</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">deep learning</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">machine learning</span></p>
          <p>The Encyclopedia of DNA Elements (ENCODE) Consortium is an international collaboration of
research groups funded by the National Human Genome Research Institute (NHGRI). The goal
of ENCODE is to build a comprehensive parts list of functional elements in the human genome,
including elements that act at the protein and RNA levels, and regulatory elements that
control cells and circumstances in which a gene is active. ENCODE investigators employ a
variety of assays and methods to identify functional elements. The discovery and annotation
of gene elements is accomplished primarily by sequencing a ...</p>
          <p><a href="/encode-project/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://academic.oup.com/nar/article/48/D1/D882/5622708#190992715" class="" target="_none">New developments on the Encyclopedia of DNA Elements (ENCODE) data portal</a> by <a href="http://orcid.org/0000-0001-6948-2042" target="_none">Luo et al 2020</a></li></a>
          </li>
          <li>
            <a href="https://github.com/ENCODE-DCC/encode-data-usage-examples/blob/master/ctcf_chip_seq_cross_cell_type_correlation.ipynb" class="" target="_none">ENCODE CTCF ChIP-seq data correlation across different cell types</a> by <a href="https://github.com/p-sud" target="_none">Paul Sud</a></li></a>
          </li>
          <li>
            <a href="https://github.com/ENCODE-DCC/encode-data-usage-examples/blob/master/mount_s3_bucket_and_run_jupyter_on_ec2.ipynb" class="" target="_none">Exploring ENCODE data from EC2 with Jupyter notebook</a> by <a href="https://github.com/keenangraham" target="_none">Keenan Graham</a></li></a>
          </li>
          <li>
            <a href="https://github.com/ENCODE-DCC/encode-data-usage-examples/blob/master/ingest_encode_data_tile_db_with_s3_backend.ipynb" class="" target="_none">Ingesting ENCODE data into TileDB with S3 backend</a> by <a href="https://github.com/ottojolanki" target="_none">Otto Jolanki</a></li></a>
          </li>
        </ul>
        <p><a href="/encode-project/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="giab" class="dataset">
          <h3><a href="/giab/">Genome in a Bottle on AWS</a></h3>
          <p><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">reference index</span><span class="label label-info tag link-tag">vcf</span></p>
          <p>Several reference genomes to enable translation of whole human genome sequencing to clinical practice. On 11/12/2020 these data were updated to reflect the most <a href="https://www.nist.gov/programs-projects/genome-bottle">up to date GIAB release</a>.</p>
          <p><a href="/giab/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.nature.com/articles/s41597-019-0098-2" class="" target="_none">High-coverage, long-read sequencing of Han Chinese trio reference samples</a> by Wang Y et al (2019)</li></a>
          </li>
          <li>
            <a href="https://github.com/ga4gh/benchmarking-tools" class="" target="_none">GA4GH Benchmarking Tools</a> by GA4GH Benchmarking Team</li></a>
          </li>
          <li>
            <a href="https://github.com/genome-in-a-bottle/about_GIAB" class="" target="_none">The Genome in a Bottle Github Project</a> by Genome In A Bottle Consortium</li></a>
          </li>
          <li>
            <a href="https://www.nature.com/articles/sdata201625" class="" target="_none">Extensive sequencing of seven human genomes to characterize benchmark reference materials</a> by Zook J et al (2016)</li></a>
          </li>
        </ul>
        <p><a href="/giab/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="ncbi-sra" class="dataset">
          <h3><a href="/ncbi-sra/">NIH NCBI Sequence Read Archive (SRA) on AWS</a></h3>
          <p><span class="label label-info tag link-tag">bam</span><span class="label label-info tag link-tag">fastq</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">STRIDES</span><span class="label label-info tag link-tag">transcriptomics</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>The Sequence Read Archive (SRA), produced by the <a href="https://www.ncbi.nlm.nih.gov/">National Center for Biotechnology Information (NCBI)</a> at the <a href="http://nlm.nih.gov/">National Library of Medicine (NLM)</a> at the <a href="http://www.nih.gov/">National Institutes of Health (NIH)</a>, stores raw DNA sequencing data and alignment information from high-throughput sequencing platforms. The SRA provides open access to these biological sequence data to support the research community&#39;s efforts to enhance reproducibility and make new discoveries by comparing data s...</p>
          <p><a href="/ncbi-sra/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/ncbi/sra-tools" class="" target="_none">SRA Toolkit</a> by <a href="https://www.ncbi.nlm.nih.gov/sra/" target="_none">NCBI SRA</a></li></a>
          </li>
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3013647/" class="" target="_none">The Sequence Read Archive</a> by <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term&#x3D;Leinonen%20R%5BAuthor%5D&amp;cauthor&#x3D;true&amp;cauthor_uid&#x3D;21062823" target="_none">Leinonen et al (2011)</a></li></a>
          </li>
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/sra/docs/sra-aws-download/" class="" target="_none">Access SRA data using Amazon Web Services (AWS)</a> by <a href="https://www.ncbi.nlm.nih.gov/sra/" target="_none">NCBI SRA</a></li></a>
          </li>
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/sra/docs/sra-cloud/" class="" target="_none">SRA in the Cloud</a> by <a href="https://www.ncbi.nlm.nih.gov/sra/" target="_none">NCBI SRA</a></li></a>
          </li>
        </ul>
        <p><a href="/ncbi-sra/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="refgenie" class="dataset">
          <h3><a href="/refgenie/">Refgenie reference genome assets</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">infrastructure</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">single-cell transcriptomics</span><span class="label label-info tag link-tag">transcriptomics</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>Pre-built refgenie reference genome data assets used for aligning and analyzing DNA sequence data.</p>
          <p><a href="/refgenie/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1093/gigascience/giz149" class="" target="_none">Refgenie: a reference genome resource manager</a> by <a href="http://www.databio.org" target="_none">Michał Stolarczyk, Vincent P Reuter, Jason P Smith, Neal E Magee, Nathan C Sheffield</a></li></a>
          </li>
          <li>
            <a href="http://pepatac.databio.org" class="" target="_none">Pipeline for ATAC-seq data analysis</a> by <a href="http://databio.org" target="_none">Jason Smith and Nathan Sheffield</a></li></a>
          </li>
          <li>
            <a href="http://peppro.databio.org" class="" target="_none">Pipeline for PRO-seq data analysis</a> by <a href="http://databio.org" target="_none">Jason Smith and Nathan Sheffield</a></li></a>
          </li>
          <li>
            <a href="http://refgenie.databio.org/en/latest/tutorial/" class="" target="_none">Basic Refgenie tutorial</a> by <a href="http://www.databio.org" target="_none">Nathan Sheffield</a></li></a>
          </li>
        </ul>
        <p><a href="/refgenie/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="broad-pan-ukb" class="dataset">
          <h3><a href="/broad-pan-ukb/">UK Biobank Pan-Ancestry Summary Statistics</a></h3>
          <p><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genome wide association study</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">population genetics</span></p>
          <p>A multi-ancestry analysis of 7,221 phenotypes using a generalized mixed model association testing framework, spanning 16,119 genome-wide association studies. We provide standard meta-analysis across all populations and with a leave-one-population-out approach for each trait. The data are provided in tsv format (per phenotype) and Hail MatrixTable (all phenotypes and variants). Metadata is provided in phenotype and variant manifests.</p>
          <p><a href="/broad-pan-ukb/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://aws.amazon.com/quickstart/architecture/hail/" class="aws-link" target="_none">Hail on AWS Quick Start</a> by Amazon Web Services and PrivoIT</li></a>
          </li>
          <li>
            <a href="https://hail.is" class="" target="_none">Hail</a> by <a href="https://hail.is/" target="_none">Hail Team</a></li></a>
          </li>
          <li>
            <a href="https://pan.ukbb.broadinstitute.org/" class="" target="_none">Pan-ancestry genetic analysis of the UK Biobank</a> by Pan UKBB Team</li></a>
          </li>
          <li>
            <a href="https://hail.is/docs/0.2/tutorials-landing.html" class="" target="_none">Hail Tutorials</a> by <a href="https://hail.is/" target="_none">Hail Team</a></li></a>
          </li>
        </ul>
        <p><a href="/broad-pan-ukb/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="allen-mouse-brain-atlas" class="dataset">
          <h3><a href="/allen-mouse-brain-atlas/">Allen Mouse Brain Atlas</a></h3>
          <p><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">gene expression</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">image processing</span><span class="label label-info tag link-tag">imaging</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">Mus musculus</span><span class="label label-info tag link-tag">neurobiology</span><span class="label label-info tag link-tag">transcriptomics</span></p>
          <p>The Allen Mouse Brain Atlas is a genome-scale collection of cellular resolution gene expression profiles using in situ hybridization (ISH). Highly methodical data production methods and comprehensive anatomical coverage via dense, uniformly spaced sampling facilitate data consistency and comparability across &gt;20,000 genes. The use of an inbred mouse strain with minimal animal-to-animal variance allows one to treat the brain essentially as a complex but highly reproducible three-dimensional tissue array. The entire Allen Mouse Brain Atlas dataset and associated tools are available through an...</p>
          <p><a href="/allen-mouse-brain-atlas/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="http://www.nature.com/articles/nature05453" class="" target="_none">Genome-wide atlas of gene expression in the adult mouse brain</a> by <a href="www.alleninstitute.org" target="_none">Ed Lein, et al.</a></li></a>
          </li>
          <li>
            <a href="https://mouse.brain-map.org" class="" target="_none">Allen Mouse Brain Atlas</a> by <a href="www.alleninstitute.org" target="_none">Allen Institute for Brain Science</a></li></a>
          </li>
          <li>
            <a href="https://github.com/AllenInstitute/open_dataset_tools/blob/master/Visualizing_Images_from_Allen_Mouse_Brain_Atlas.ipynb" class="" target="_none">Visualizing Images from the Allen Mouse Brain Atlas</a> by <a href="www.alleninstitute.org" target="_none">Allen Institute for Brain Science</a></li></a>
          </li>
        </ul>
        <p><a href="/allen-mouse-brain-atlas/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="ncbi-blast-databases" class="dataset">
          <h3><a href="/ncbi-blast-databases/">Basic Local Alignment Sequences Tool (BLAST) Databases</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">health</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">protein</span><span class="label label-info tag link-tag">reference index</span><span class="label label-info tag link-tag">transcriptomics</span></p>
          <p>A centralized repository of pre-formatted BLAST databases created by the National Center for Biotechnology Information (NCBI).</p>
          <p><a href="/ncbi-blast-databases/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://pubmed.ncbi.nlm.nih.gov/20003500" class="" target="_none">BLAST+: Architecture and Applications</a> by Christiam Camacho  1 , George Coulouris, Vahram Avagyan, Ning Ma, Jason Papadopoulos, Kevin Bealer, Thomas L Madden</li></a>
          </li>
          <li>
            <a href="https://pubmed.ncbi.nlm.nih.gov/9254694/" class="" target="_none">Gapped BLAST and PSI-BLAST: A New Generation of Protein Database Search Programs</a> by S F Altschul, T L Madden, A A Schäffer, J Zhang, Z Zhang, W Miller, D J Lipman</li></a>
          </li>
          <li>
            <a href="https://github.com/ncbi/blast_plus_docs" class="" target="_none">BLAST+ Docker</a> by NCBI BLAST</li></a>
          </li>
        </ul>
        <p><a href="/ncbi-blast-databases/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="beataml" class="dataset">
          <h3><a href="/beataml/">Beat Acute Myeloid Leukemia (AML) 1.0</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">Homo sapiens</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">STRIDES</span></p>
          <p>Beat AML 1.0 is a collaborative research program involving 11 academic medical centers who worked
collectively to better understand  drugs and drug combinations that should be prioritized for
further development within clinical and/or molecular subsets of acute myeloid leukemia (AML)
patients. Beat AML 1.0 provides the largest-to-date dataset on primary acute myeloid leukemia
samples offering genomic, clinical, and drug response.This dataset contains open Clinical Supplement and RNA-Seq Gene Expression Quantification data.This dataset also contains controlled Whole Exome Sequencing (WXS) and R...</p>
          <p><a href="/beataml/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/30651561" class="" target="_none">Clinical resistance to crenolanib in acute myeloid leukemia due to diverse molecular
mechanisms
</a> by Zhang H, Savage S, Schultz AR, Bottomly D, White L, Segerdell E, et al.</li></a>
          </li>
          <li>
            <a href="https://portal.gdc.cancer.gov/projects/" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></li></a>
          </li>
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/30333627" class="" target="_none">Functional Genomic Landscape of Acute Myeloid Leukemia</a> by Jeffrey W. Tyner, Cristina E. Tognon, Dan Bottomly et al.</li></a>
          </li>
        </ul>
        <p><a href="/beataml/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="talend-covid19" class="dataset">
          <h3><a href="/talend-covid19/">COVID-19 Harmonized Data</a></h3>
          <p><span class="label label-info tag link-tag">coronavirus</span><span class="label label-info tag link-tag">COVID-19</span><span class="label label-info tag link-tag">life sciences</span></p>
          <p>A harmonized collection of the core data pertaining to COVID-19 reported cases by geography, in a format prepared for analysis</p>
          <p><a href="/talend-covid19/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/singer-io/tap-covid-19" class="" target="_none">Tap COVID-19 Python</a> by <a href="https://github.com/jeffhuth-bytecode" target="_none">Jeff Huth</a></li></a>
          </li>
          <li>
            <a href="https://www.stitchdata.com/integrations/covid-19/" class="" target="_none">Stitch COVID-19 Integration</a> by <a href="https://github.com/jeffhuth-bytecode" target="_none">Jeff Huth</a></li></a>
          </li>
          <li>
            <a href="https://www.talend.com/blog/2020/04/05/talend-joins-fight-against-covid-19-unlocking-data-for-health-researchers/" class="" target="_none">How Talend is joining the fight against COVID-19</a> by <a href="https://www.linkedin.com/in/thomasbennettw/" target="_none">Thomas Bennett</a></li></a>
          </li>
        </ul>
        <p><a href="/talend-covid19/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="ctsp-dlbcl" class="dataset">
          <h3><a href="/ctsp-dlbcl/">Clinical Trial Sequencing Project - Diffuse Large B-Cell Lymphoma</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">STRIDES</span><span class="label label-info tag link-tag">transcriptomics</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>The goal of the project is to identify recurrent genetic alterations (mutations, deletions,
amplifications, rearrangements) and/or gene expression signatures. National Cancer Institute (NCI)
utilized whole genome sequencing and/or whole exome sequencing in conjunction with transcriptome
sequencing. The samples were processed and submitted for genomic characterization using pipelines
and procedures established within The Cancer Genome Analysis (TCGA) project.</p>
          <p><a href="/ctsp-dlbcl/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed?cmd&#x3D;DetailsSearch&amp;term&#x3D;29925955[PMID]" class="" target="_none">A multiprotein supercomplex controlling oncogenic signalling in lymphoma</a> by Phelan JD, Young RM, Webster DE, Roulland S, Wright GW, Kasbekar M, Shaffer AL 3rd,
Ceribelli M, Wang JQ, Schmitz R, Nakagawa M, Bachy E, Huang DW, Ji Y, Chen L, Yang Y, Zhao
H, Yu X, Xu W, Palisoc MM, Valadez RR, Davies-Hill T, Wilson WH, Chan WC, Jaffe ES, Gascoyne
RD, Campo E, Rosenwald A, Ott G, Delabie J, Rimsza LM, Rodriguez FJ, Estephan F, Holdhoff M,
Kruhlak MJ, Hewitt SM, Thomas CJ, Pittaluga S, Oellerich T, Staudt LM
</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1056/NEJMoa1801445" class="" target="_none">Genetics and Pathogenesis of Diffuse Large B Cell Lymphoma</a> by Roland Schmitz, Ph.D., George W. Wright, Ph.D., Da Wei Huang, M.D., Calvin A.  Johnson,
Ph.D., James D. Phelan, Ph.D., James Q. Wang, Ph.D., Sandrine Roulland, Ph.D., Monica
Kasbekar, Ph.D., Ryan M. Young, Ph.D., Arthur L. Shaffer, Ph.D., Daniel J. Hodson, M.D.,
Ph.D., Wenming Xiao, Ph.D., et al.
</li></a>
          </li>
          <li>
            <a href="https://portal.gdc.cancer.gov/projects/CTSP-DLBCL1" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></li></a>
          </li>
        </ul>
        <p><a href="/ctsp-dlbcl/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="dandiarchive" class="dataset">
          <h3><a href="/dandiarchive/">Distributed Archives for Neurophysiology Data Integration (DANDI)</a></h3>
          <p><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">cell imaging</span><span class="label label-info tag link-tag">electrophysiology</span><span class="label label-info tag link-tag">infrastructure</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">neuroimaging</span><span class="label label-info tag link-tag">neurophysiology</span><span class="label label-info tag link-tag">neuroscience</span></p>
          <p>DANDI is a public archive of neurophysiology datasets, including raw and processed data,  and associated software containers. Datasets are shared according to a Creative Commons  CC0 or CC-BY licenses. The data archive provides a broad range of cellular neurophysiology data.  This includes electrode and optical recordings, and associated imaging data using a  set of community standards: <a href="https://www.nwb.org/nwb-neurophysiology/">NWB:N - NWB:Neurophysiology</a>, <a href="https://bids.neuroimaging.io/">BIDS - Brain Imaging Data Structure</a>, and  <a href="htt...</p>
          <p><a href="/dandiarchive/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://dandiarchive.org/" class="" target="_none">DANDI Web Interface</a> by <a href="https://dandiarchive.org/" target="_none">DANDI Project</a></li></a>
          </li>
          <li>
            <a href="https://github.com/dandi/dandi-cli" class="" target="_none">DANDI Shell Interface</a> by <a href="https://dandiarchive.org/" target="_none">DANDI Project</a></li></a>
          </li>
          <li>
            <a href="https://hub.dandiarchive.org/" class="" target="_none">DANDI JupyterHub Interface</a> by <a href="https://dandiarchive.org/" target="_none">DANDI Project</a></li></a>
          </li>
        </ul>
        <p><a href="/dandiarchive/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="msd" class="dataset">
          <h3><a href="/msd/">Medical Segmentation Decathlon</a></h3>
          <p><span class="label label-info tag link-tag">computed tomography</span><span class="label label-info tag link-tag">health</span><span class="label label-info tag link-tag">imaging</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">magnetic resonance imaging</span><span class="label label-info tag link-tag">medicine</span><span class="label label-info tag link-tag">nifti</span><span class="label label-info tag link-tag">segmentation</span></p>
          <p>With recent advances in machine learning, semantic segmentation algorithms are becoming increasingly general purpose and translatable to unseen tasks. Many key algorithmic advances in the field of medical imaging are commonly validated on a small number of tasks, limiting our understanding of the generalisability of the proposed contributions. A model which works out-of-the-box on many tasks, in the spirit of AutoML, would have a tremendous impact on healthcare. The field of medical imaging is also missing a fully open source and comprehensive benchmark for general purpose algorithmic validati...</p>
          <p><a href="/msd/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://arxiv.org/pdf/1902.09063.pdf" class="" target="_none">A large annotated medical image dataset for the development and evaluation of segmentation algorithms</a> by Simpson A. L., Antonelli M., Bakas S., Bilello M., Farahana K., van Ginneken B., et al</li></a>
          </li>
          <li>
            <a href="https://github.com/Project-MONAI/tutorials" class="" target="_none">Pytorch-Integrated MSD Data Loader</a> by <a href="https://github.com/Project-MONAI/MONAI" target="_none">MONAI Development Team</a></li></a>
          </li>
          <li>
            <a href="www.monai.io" class="" target="_none">MONAI: Getting Started</a> by <a href="https://monai.io/start.html" target="_none">MONAI Development Team</a></li></a>
          </li>
        </ul>
        <p><a href="/msd/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="human-microbiome-project" class="dataset">
          <h3><a href="/human-microbiome-project/">The Human Microbiome Project</a></h3>
          <p><span class="label label-info tag link-tag">amino acid</span><span class="label label-info tag link-tag">fasta</span><span class="label label-info tag link-tag">fastq</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">metagenomics</span><span class="label label-info tag link-tag">microbiome</span></p>
          <p>The NIH-funded Human Microbiome Project (HMP) is a collaborative effort of over 300 scientists from more than 80 organizations to comprehensively characterize the microbial communities inhabiting the human body and elucidate their role in human health and disease. To accomplish this task, microbial community samples were isolated from a cohort of 300 healthy adult human subjects at 18 specific sites within five regions of the body (oral cavity, airways, urogenital track, skin, and gut). Targeted sequencing of the 16S bacterial marker gene and/or whole metagenome shotgun sequencing was performe...</p>
          <p><a href="/human-microbiome-project/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.sciencedirect.com/science/article/pii/S2452231718300289" class="" target="_none">New microbe genomic variants in patients fecal community following surgical disruption of the upper human gastrointestinal tract</a> by Ranjit Kumar, Jayleen Grams, Daniel I. Chu, David K.Crossman, Richard Stahl, Peter Eipers, et al</li></a>
          </li>
          <li>
            <a href="https://www.nature.com/articles/nature23889" class="" target="_none">Strains, functions and dynamics in the expanded Human Microbiome Project</a> by Jason Lloyd-Price, Anup Mahurkar, Gholamali Rahnavard, Jonathan Crabtree, Joshua Orvis, A. Brantley Hall, et al.</li></a>
          </li>
          <li>
            <a href="https://www.nature.com/articles/nature06244" class="" target="_none">The Human Microbiome Project</a> by Peter J. Turnbaugh, Ruth E. Ley, Micah Hamady, Claire M. Fraser-Liggett, Rob Knight &amp; Jeffrey I. Gordon</li></a>
          </li>
        </ul>
        <p><a href="/human-microbiome-project/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="hail-vep-pipeline" class="dataset">
          <h3><a href="/hail-vep-pipeline/">Variant Effect Predictor (VEP) and the Loss-Of-Function Transcript Effect Estimator (LOFTEE) Plugin</a></h3>
          <p><span class="label label-info tag link-tag">genome wide association study</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">loftee</span><span class="label label-info tag link-tag">vep</span></p>
          <p>VEP determines the effect of genetic variants (SNPs, insertions, deletions, CNVs or structural variants) on genes, transcripts, and protein sequence, as well as regulatory regions. The European Bioinformatics Institute produces the VEP tool/db and releases updates every 1 - 6 months. The latest release contains 267 genomes from 232 species containing 5567663 protein coding genes. This dataset hosts the last 5 releases for human, rat, and zebrafish. Also, it hosts the required reference files for the Loss-Of-Function Transcript Effect Estimator (LOFTEE) plugin as it is commonly used with VEP.</p>
          <p><a href="/hail-vep-pipeline/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/konradjk/loftee" class="" target="_none">Loss-Of-Function Transcript Effect Estimator (LOFTEE)</a> by <a href="https://github.com/konradjk" target="_none">Konrad Karczewski</a></li></a>
          </li>
          <li>
            <a href="https://hail.is/" class="" target="_none">Hail</a> by <a href="https://www.nealelab.is/" target="_none">Neale Lab</a></li></a>
          </li>
          <li>
            <a href="http://uswest.ensembl.org/info/docs/tools/vep/index.html" class="" target="_none">Variant Effect Predictor (VEP)</a> by <a href="http://uswest.ensembl.org/info/docs/tools/vep/index.html" target="_none">Ensembl</a></li></a>
          </li>
        </ul>
        <p><a href="/hail-vep-pipeline/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="afsis" class="dataset">
          <h3><a href="/afsis/">Africa Soil Information Service (AfSIS) Soil Chemistry</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">food security</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info-sustainability tag link-tag">sustainability</span></p>
          <p>This dataset contains soil infrared spectral data and paired soil property
reference measurements for georeferenced soil samples that were collected
through the Africa Soil Information Service (AfSIS) project, which lasted
from 2009 through 2018. In this release, we include data collected during
Phase I (2009-2013.) Georeferenced samples were collected from 19 countries
in Sub-Saharan African using a statistically sound sampling scheme,
and their soil properties were analyzed using <em>both</em> conventional soil
testing methods and spectral methods (infrared diffuse reflectance
spectroscopy). The two ...</p>
          <p><a href="/afsis/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.youtube.com/watch?v&#x3D;Fb9R0CnPMkc" class="" target="_none">Goalkeepers 2018, Soil - The Big Data Beneath Your Feet</a> by <a href="https://qed.ai" target="_none">QED</a></li></a>
          </li>
          <li>
            <a href="https://github.com/qedsoftware/afsis-soil-chem-tutorial/" class="" target="_none">AfSIS Soil Chemistry - Usage Tutorial</a> by <a href="https://qed.ai" target="_none">QED</a></li></a>
          </li>
        </ul>
        <p><a href="/afsis/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="broad-references" class="dataset">
          <h3><a href="/broad-references/">Broad Genome References</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">Homo sapiens</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">reference index</span></p>
          <p>Broad maintained human genome reference builds hg19/hg38 and decoy references.</p>
          <p><a href="/broad-references/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://docs.opendata.aws/genomics-workflows/orchestration/cromwell/cromwell-examples/#real-world-example-haplotypecaller" class="aws-link" target="_none">Genomics Workflows on AWS - Cromwell on AWS</a> by <a href="https://www.linkedin.com/in/lee-pang-a039a26/" target="_none">W. Lee Pang</a></li></a>
          </li>
          <li>
            <a href="https://aws.amazon.com/blogs/storage/using-amazon-fsx-for-lustre-for-genomics-workflows-on-aws/" class="aws-link" target="_none">Using Amazon FSx for Lustre for Genomics Workflows on AWS</a> by <a href="https://www.linkedin.com/in/lee-pang-a039a26/" target="_none">W. Lee Pang</a></li></a>
          </li>
        </ul>
        <p><a href="/broad-references/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="cgci" class="dataset">
          <h3><a href="/cgci/">Cancer Genome Characterization Initiatives - Burkitt Lymphoma, HIV+ Cervical Cancer</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">STRIDES</span><span class="label label-info tag link-tag">transcriptomics</span></p>
          <p>The Cancer Genome Characterization Initiatives (CGCI) program supports cutting-edge genomics research of adult and pediatric cancers. CGCI investigators develop and apply advanced sequencing methods that examine genomes, exomes, and transcriptomes within various types of tumors. The program includes Burkitt Lymphoma Genome Sequencing Project (BLGSP) project and HIV+ Tumor Molecular Characterization Project - Cervical Cancer (HTMCP-CC) project.
The dataset contains open Clinical Supplement, Biospecimen Supplement, RNA-Seq Gene Expression Quantification, miRNA-Seq Isoform Expression Quantificati...</p>
          <p><a href="/cgci/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1182/blood-2018-09-871418" class="" target="_none">Genome-wide discovery of somatic coding and noncoding mutations in pediatric endemic and
sporadic Burkitt lymphoma
</a> by Grande B. M., Gerhard D. S., Jiang A., Griner N. B., Abramson J. S., Alexander T. B., et al.
</li></a>
          </li>
          <li>
            <a href="https://portal.gdc.cancer.gov/projects/CGCI-BLGSP" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></li></a>
          </li>
        </ul>
        <p><a href="/cgci/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="mimiciii" class="dataset">
          <h3><a href="/mimiciii/">MIMIC-III (‘Medical Information Mart for Intensive Care’)</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">health</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">natural language processing</span><span class="label label-info tag link-tag">us</span></p>
          <p>MIMIC-III (‘Medical Information Mart for Intensive Care’) is a large, 
single-center database comprising information relating to patients 
admitted to critical care units at a large tertiary care hospital. 
Data includes vital signs, medications, laboratory measurements, 
observations and notes charted by care providers, fluid balance, 
procedure codes, diagnostic codes, imaging reports, hospital length 
of stay, survival data, and more. The database supports applications 
including academic and industrial research, quality improvement initiatives, 
and higher education coursework. The MIMIC-I...</p>
          <p><a href="/mimiciii/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://aws.amazon.com/blogs/big-data/perform-biomedical-informatics-without-a-database-using-mimic-iii-data-and-amazon-athena/" class="aws-link" target="_none">Perform biomedical informatics without a database using MIMIC-III data and Amazon Athena</a> by James Wiggins, Alistair Johnson</li></a>
          </li>
          <li>
            <a href="https://github.com/MIT-LCP/mimic-code" class="" target="_none">MIMIC-code GitHub repository</a> by <a href="https://github.com/alistairewj" target="_none">Alistair Johnson</a></li></a>
          </li>
        </ul>
        <p><a href="/mimiciii/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="ohsu-cnl" class="dataset">
          <h3><a href="/ohsu-cnl/">Oregon Health &amp; Science University Chronic Neutrophilic Leukemia Dataset</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span></p>
          <p>The OHSU-CNL study offers the whole exome and RNA-sequencing on a cohort of 100 cases with rare
hematologic malignancies such as Chronic neutrophilic leukemia (CNL), atypical chronic myeloid
leukemia (aCML), and unclassified myelodysplastic syndrome/myeloproliferative neoplasms
(MDS/MPN-U). This dataset contains open RNA-Seq Gene Expression Quantification data.</p>
          <p><a href="/ohsu-cnl/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://portal.gdc.cancer.gov/projects/" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></li></a>
          </li>
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/31366621" class="" target="_none">Genomic landscape of neutrophilic leukemias of ambiguous diagnosis</a> by Zhang H, Wilmot B, Bottomly D et al.</li></a>
          </li>
        </ul>
        <p><a href="/ohsu-cnl/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="serratus-lovelywater" class="dataset">
          <h3><a href="/serratus-lovelywater/">Serratus: Ultra-deep Search for Novel Viruses - Versioned Data Release</a></h3>
          <p><span class="label label-info tag link-tag">bam</span><span class="label label-info tag link-tag">COVID-19</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">MERS</span><span class="label label-info tag link-tag">SARS</span><span class="label label-info tag link-tag">SARS-CoV-2</span><span class="label label-info tag link-tag">virus</span></p>
          <p>Serratus is a collaborative open science project for ultra-rapid discovery of known and unknown coronaviruses in response to the COVID-19 pandemic through re-analysis of publicly available genomic data. Our resulting vertebrate viral alignment data is explorable via the <a href="https://serratus.io/explorer">Serratus Explorer</a> and directly accessible on Amazon S3.</p>
          <p><a href="/serratus-lovelywater/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.biorxiv.org/content/10.1101/2020.07.28.224584v2" class="" target="_none">coronaSPAdes. From biosynthetic gene clusters to RNA viral assemblies</a> by Meleshko D., Hajirasouliha I., and Korobeynikov A. (2021)</li></a>
          </li>
          <li>
            <a href="https://github.com/serratus-bio/tantalus" class="" target="_none">Tantalus: An R Package for exploration of Serratus data</a> by Serratus Team</li></a>
          </li>
          <li>
            <a href="https://www.pnas.org/content/118/3/e2019907118" class="" target="_none">Diversification of mammalian deltaviruses by host shifting</a> by Bergner L.M., Orton R.J., et al (2021)</li></a>
          </li>
          <li>
            <a href="https://www.biorxiv.org/content/10.1101/2020.08.07.241729v1" class="" target="_none">Petabase-scale sequence alignment catalyses viral discovery</a> by Edgar R., Taylor J., Altman T., Barbera P., Meleshko D., Lin V., et al (2020)</li></a>
          </li>
          <li>
            <a href="https://serratus.io/explorer" class="" target="_none">Serratus Explorer</a> by Serratus Team</li></a>
          </li>
        </ul>
        <p><a href="/serratus-lovelywater/#usageexamples">See 5 usage examples &rarr;</a></p>
        </div>
        <div id="1000-genomes" class="dataset">
          <h3><a href="/1000-genomes/">1000 Genomes</a></h3>
          <p><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span></p>
          <p>The 1000 Genomes Project is an international collaboration which has established the most detailed catalogue of human genetic variation, including SNPs, structural variants, and their haplotype context. The final phase of the project sequenced more than 2500 individuals from 26 different populations around the world and produced an integrated set of phased haplotypes with more than 80 million variants for these individuals.</p>
          <p><a href="/1000-genomes/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://aws.amazon.com/blogs/big-data/exploratory-data-analysis-of-genomic-datasets-using-adam-and-mango-with-apache-spark-on-amazon-emr/" class="aws-link" target="_none">Exploratory data analysis of genomic datasets using ADAM and Mango with Apache Spark on Amazon EMR</a> by <a href="https://research.eecs.berkeley.edu/~akmorrow/" target="_none">Alyssa Marrow</a></li></a>
          </li>
        </ul>
        <p><a href="/1000-genomes/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="aws-igenomes" class="dataset">
          <h3><a href="/aws-igenomes/">AWS iGenomes</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">Caenorhabditis elegans</span><span class="label label-info tag link-tag">Danio rerio</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">Homo sapiens</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">Mus musculus</span><span class="label label-info tag link-tag">Rattus norvegicus</span><span class="label label-info tag link-tag">reference index</span></p>
          <p>Common reference genomes hosted on AWS S3. Can be used when aligning and analysing raw DNA sequencing data.</p>
          <p><a href="/aws-igenomes/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="http://nf-co.re/" class="" target="_none">nf-core analysis pipelines</a> by <a href="http://phil.ewels.co.uk/" target="_none">Phil Ewels</a></li></a>
          </li>
        </ul>
        <p><a href="/aws-igenomes/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="allen-brain-observatory" class="dataset">
          <h3><a href="/allen-brain-observatory/">Allen Brain Observatory - Visual Coding AWS Public Data Set</a></h3>
          <p><span class="label label-info tag link-tag">electrophysiology</span><span class="label label-info tag link-tag">image processing</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">Mus musculus</span><span class="label label-info tag link-tag">neurobiology</span><span class="label label-info tag link-tag">neuroimaging</span><span class="label label-info tag link-tag">signal processing</span></p>
          <p>The Allen Brain Observatory – Visual Coding is a large-scale, standardized survey of physiological activity across the mouse visual cortex, hippocampus, and thalamus. It includes datasets collected with both two-photon imaging and Neuropixels probes, two complementary techniques for measuring the activity of neurons in vivo. The two-photon imaging dataset features visually evoked calcium responses from GCaMP6-expressing neurons in a range of cortical layers, visual areas, and Cre lines. The Neuropixels dataset features spiking activity from distributed cortical and subcortical brain regions, c...</p>
          <p><a href="/allen-brain-observatory/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/AllenInstitute/AllenSDK/wiki/Use-the-Allen-Brain-Observatory-%E2%80%93-Visual-Coding-on-AWS" class="" target="_none">Use the Allen Brain Observatory – Visual Coding on AWS</a> by <a href="https://twitter.com/AllenInstitute" target="_none">Nika Keller, David Feng</a></li></a>
          </li>
        </ul>
        <p><a href="/allen-brain-observatory/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="ncbi-covid-19" class="dataset">
          <h3><a href="/ncbi-covid-19/">COVID-19 Genome Sequence Dataset</a></h3>
          <p><span class="label label-info tag link-tag">bam</span><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">coronavirus</span><span class="label label-info tag link-tag">COVID-19</span><span class="label label-info tag link-tag">cram</span><span class="label label-info tag link-tag">fastq</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">health</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">MERS</span><span class="label label-info tag link-tag">SARS</span><span class="label label-info tag link-tag">STRIDES</span><span class="label label-info tag link-tag">transcriptomics</span><span class="label label-info tag link-tag">virus</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>A centralized sequence repository for all records containing sequence associated with the novel corona virus (SARS-CoV-2) submitted to the National Center for Biotechnology Information (NCBI) Sequence Read Archive (SRA). Included are both the original sequences submitted by the principal investigator as well as SRA-processed sequences that require the SRA Toolkit for analysis. Additionally, submitter provided metadata included in associated BioSample and BioProject records is available alongside NCBI calculated data, such k-mer based taxonomy analysis results, contiguous assemblies (contigs) a...</p>
          <p><a href="/ncbi-covid-19/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/sra/docs/sra-aws-download/" class="" target="_none">Download SRA sequence data using Amazon Web Services (AWS)</a> by NCBI SRA</li></a>
          </li>
        </ul>
        <p><a href="/ncbi-covid-19/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="cell-painting-image-collection" class="dataset">
          <h3><a href="/cell-painting-image-collection/">Cell Painting Image Collection</a></h3>
          <p><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">cell imaging</span><span class="label label-info tag link-tag">cell painting</span><span class="label label-info tag link-tag">fluorescence imaging</span><span class="label label-info tag link-tag">high-throughput imaging</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">microscopy</span></p>
          <p>The Cell Painting Image Collection is a collection of freely
downloadable microscopy image sets. Cell Painting is an
unbiased high throughput imaging assay used to analyze
perturbations in cell models. In addition to the images
themselves, each set includes a description of the biological
application and some type of &quot;ground truth&quot; (expected results).
Researchers are encouraged to use these image sets as reference
points when developing, testing, and publishing new image
analysis algorithms for the life sciences. We hope that the
this data set will lead to a better understanding of w...</p>
          <p><a href="/cell-painting-image-collection/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/cytodata/cytodata-hackathon-2018/tree/master/cytodata-toolkit/" class="" target="_none">Example submission for the 2018 CytoData Hackathon (in R and Python)</a> by <a href="broadinstitute.org" target="_none">Juan Caicedo, Tim Becker</a></li></a>
          </li>
        </ul>
        <p><a href="/cell-painting-image-collection/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="gatk-test-data" class="dataset">
          <h3><a href="/gatk-test-data/">GATK Test Data</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span></p>
          <p>The GATK test data resource bundle is a collection of files for resequencing human genomic data with the
Broad Institute&#39;s <a href="https://software.broadinstitute.org/gatk/">Genome Analysis Toolkit (GATK)</a>.</p>
          <p><a href="/gatk-test-data/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://docs.opendata.aws/genomics-workflows/orchestration/cromwell/cromwell-examples/#real-world-example-haplotypecaller" class="aws-link" target="_none">Genomics Workflows on AWS - Cromwell on AWS</a> by <a href="https://www.linkedin.com/in/lee-pang-a039a26/" target="_none">W. Lee Pang</a></li></a>
          </li>
        </ul>
        <p><a href="/gatk-test-data/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="hcmi-cmdc" class="dataset">
          <h3><a href="/hcmi-cmdc/">Human Cancer Models Initiative (HCMI) Cancer Model Development Center</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">STRIDES</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>The Human Cancer Models Initiative (HCMI) is an international consortium that is generating novel,
next-generation, tumor-derived culture models annotated with genomic and clinical data.
HCMI-developed models and related data are available as a community resource. The NCI is
contributing to the initiative by supporting four Cancer Model Development Centers (CMDCs).  CMDCs
are tasked with producing next-generation cancer models from clinical samples. The cancer models
include tumor types that are rare, originate from patients from underrepresented populations, lack
precision therapy, or lack ca...</p>
          <p><a href="/hcmi-cmdc/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://portal.gdc.cancer.gov/projects/HCMI-CMDC" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></li></a>
          </li>
        </ul>
        <p><a href="/hcmi-cmdc/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="hpgp-data" class="dataset">
          <h3><a href="/hpgp-data/">Human PanGenomics Project</a></h3>
          <p><span class="label label-info tag link-tag">cram</span><span class="label label-info tag link-tag">fast5</span><span class="label label-info tag link-tag">fastq</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span></p>
          <p>This dataset includes sequencing data, assemblies, and analyses for the offspring of ten parent-offspring trios.</p>
          <p><a href="/hpgp-data/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.nature.com/articles/s41587-020-0503-6" class="" target="_none">Nanopore sequencing and the Shasta toolkit enable efficient de novo assembly of eleven human genomes</a> by Shafin et al (2020)</li></a>
          </li>
        </ul>
        <p><a href="/hpgp-data/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="ocmr_data" class="dataset">
          <h3><a href="/ocmr_data/">Ohio State Cardiac MRI Raw Data (OCMR)</a></h3>
          <p><span class="label label-info tag link-tag">Homo sapiens</span><span class="label label-info tag link-tag">image processing</span><span class="label label-info tag link-tag">imaging</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">magnetic resonance imaging</span><span class="label label-info tag link-tag">signal processing</span></p>
          <p>OCMR is an open-access repository that provides multi-coil k-space data for cardiac cine.  The fully sampled MRI datasets are intended for quantitative comparison and evaluation of image reconstruction methods. The free-breathing, prospectively undersampled datasets are intended to evaluate their performance and generalizability qualitatively.</p>
          <p><a href="/ocmr_data/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/MRIOSU/OCMR/blob/master/Python/example_ocmr.ipynb" class="" target="_none">OCMR Tutorial</a> by <a href="https://cmr.engineering.osu.edu/people/trainees" target="_none">Chong Chen</a></li></a>
          </li>
        </ul>
        <p><a href="/ocmr_data/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="ont-open-data" class="dataset">
          <h3><a href="/ont-open-data/">Oxford Nanopore Technologies Benchmark Datasets</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">fast5</span><span class="label label-info tag link-tag">fastq</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">Homo sapiens</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>The ont-open-data registry provides reference sequencing data from Oxford Nanopore Technologies to support, 1) Exploration of the characteristics of nanopore sequence data. 2) Assessment and reproduction of performance benchmarks 3) Development of tools and methods. The data deposited showcases DNA sequences from a representative subset of sequencing chemistries. The datasets correspond to publicly-available reference samples (e.g. GM24385 as reference human). Raw data are provided with metadata and scripts to describe sample and data provenance.</p>
          <p><a href="/ont-open-data/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://nanoporetech.github.io/ont-open-datasets/tutorials" class="" target="_none">ONT Dataset Tutorials</a> by <a href="https://labs.epi2me.io/" target="_none">EPI2MELabs</a></li></a>
          </li>
        </ul>
        <p><a href="/ont-open-data/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="tabula-muris" class="dataset">
          <h3><a href="/tabula-muris/">Tabula Muris</a></h3>
          <p><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">encyclopedic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">health</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">medicine</span></p>
          <p>Tabula Muris is a compendium of single cell transcriptomic data from the model organism <em>Mus musculus</em> comprising more than 100,000 cells from 20 organs and tissues. These data represent a new resource for cell biology, reveal gene expression in poorly characterized cell populations, and allow for direct and controlled comparison of gene expression in cell types shared between tissues, such as T-lymphocytes and endothelial cells from different anatomical locations. Two distinct technical approaches were used for most organs: one approach, microfluidic droplet-based 3’-end counting, enabled the s...</p>
          <p><a href="/tabula-muris/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/30283141" class="" target="_none">Single-cell transcriptomics of 20 mouse organs creates a Tabula Muris.</a> by <a href="https://tabula-muris.ds.czbiohub.org/" target="_none">Tabula Muris Consortium (2019)</a></li></a>
          </li>
        </ul>
        <p><a href="/tabula-muris/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="zinc15" class="dataset">
          <h3><a href="/zinc15/">ZINC Database</a></h3>
          <p><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">chemical biology</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">molecular docking</span><span class="label label-info tag link-tag">pharmaceutical</span><span class="label label-info tag link-tag">protein</span></p>
          <p>3D models for molecular docking screens.</p>
          <p><a href="/zinc15/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="http://zinc15.docking.org" class="" target="_none">ZINC Database</a> by <a href="http://irwinlab.compbio.ucsf.edu" target="_none">John Irwin</a></li></a>
          </li>
        </ul>
        <p><a href="/zinc15/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="ihart" class="dataset">
          <h3><a href="/ihart/">iHART Whole Genome Sequencing Data Set</a></h3>
          <p><span class="label label-info tag link-tag">autism spectrum disorder</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>iHART is the <a href="http://www.thehartwellfoundation.org/">Hartwell Foundation</a>’s Autism Research and Technology Initiative. This release contains whole genome data from over 1000 families with 2 or more children with autism, of which biomaterials were provided by the Autism Genetic Resource Exchange (<a href="http://research.agre.org/">AGRE</a>).</p>
          <p><a href="/ihart/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.cell.com/cell/fulltext/S0092-8674(19)30780-9" class="" target="_none">Inherited and De Novo Genetic Risk for Autism Impacts Shared Networks</a> by Ruzzo et al. (2020)</li></a>
          </li>
        </ul>
        <p><a href="/ihart/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="molssi-covid19-hub" class="dataset">
          <h3><a href="/molssi-covid19-hub/">COVID-19 Molecular Structure and Therapeutics Hub</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">coronavirus</span><span class="label label-info tag link-tag">COVID-19</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">molecular docking</span><span class="label label-info tag link-tag">pharmaceutical</span></p>
          <p>Aggregating critical information to accelerate drug discovery for the molecular modeling and simulation community.
A community-driven data repository and curation service for molecular structures, models, therapeutics, and
simulations related to computational research related to therapeutic opportunities for COVID-19
(caused by the SARS-CoV-2 coronavirus).</p>
          <p><a href="/molssi-covid19-hub/">Details &rarr;</a></p>
        </div>
        <div id="genomeark" class="dataset">
          <h3><a href="/genomeark/">Genome Ark</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span></p>
          <p>The Genome Ark hosts genomic information for the Vertebrate Genomes Project (VGP) and other related projects. The VGP is an international collaboration that aims to generate complete and near error-free reference genomes for all extant vertebrate species. These genomes will be used to address fundamental questions in biology and disease, to identify species most genetically at risk for extinction, and to preserve genetic information of life.</p>
          <p><a href="/genomeark/">Details &rarr;</a></p>
        </div>
        <div id="nanopore" class="dataset">
          <h3><a href="/nanopore/">Nanopore Reference Human Genome</a></h3>
          <p><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span></p>
          <p>This dataset includes the sequencing and assembly of a reference standard human genome (GM12878) using the MinION nanopore sequencing instrument with the R9.4 1D chemistry.</p>
          <p><a href="/nanopore/">Details &rarr;</a></p>
        </div>
        <div id="openneuro" class="dataset">
          <h3><a href="/openneuro/">OpenNeuro</a></h3>
          <p><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">imaging</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">neurobiology</span><span class="label label-info tag link-tag">neuroimaging</span></p>
          <p>OpenNeuro is a database of openly-available brain imaging data. The data are shared according to a Creative Commons CC0 license, providing a broad range of brain imaging data to researchers and citizen scientists alike. The database primarily focuses on functional magnetic resonance imaging (fMRI) data, but also includes other imaging modalities including structural and diffusion MRI, electroencephalography (EEG), and magnetoencephalograpy (MEG). OpenfMRI is a project of the <a href="http://reproducibility.stanford.edu">Center for Reproducible Neuroscience at Stanford University</a>. Development of the OpenNeuro resource has been funded by th...</p>
          <p><a href="/openneuro/">Details &rarr;</a></p>
        </div>
        <div id="physionet" class="dataset">
          <h3><a href="/physionet/">Physionet</a></h3>
          <p><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">life sciences</span></p>
          <p>PhysioNet offers free web access to large collections of recorded physiologic signals (PhysioBank) and related open-source software (PhysioToolkit).</p>
          <p><a href="/physionet/">Details &rarr;</a></p>
        </div>
        <div id="elp-nouabale-landscape" class="dataset">
          <h3><a href="/elp-nouabale-landscape/">Sounds of Central African landscapes</a></h3>
          <p><span class="label label-info tag link-tag">biodiversity</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">ecosystems</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">land</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">natural resource</span><span class="label label-info tag link-tag">survey</span></p>
          <p>Archival soundscapes recorded in the rainforest landscapes of
Central Africa, with a focus on the vocalizations of African forest
elephants (Loxodonta cyclotis).</p>
          <p><a href="/elp-nouabale-landscape/">Details &rarr;</a></p>
        </div>
        <div id="tabula-muris-senis" class="dataset">
          <h3><a href="/tabula-muris-senis/">Tabula Muris Senis</a></h3>
          <p><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">encyclopedic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">health</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">medicine</span><span class="label label-info tag link-tag">single-cell transcriptomics</span></p>
          <p>Tabula Muris Senis is a comprehensive compendium of single cell transcriptomic data from the model organism <em>Mus musculus</em> comprising more than 500,000 cells from 18 organs and tissues across the mouse lifespan. We discovered cell-specific changes occurring across multiple cell types and organs, as well as age related changes in the cellular composition of different organs. Using single-cell transcriptomic data we were able to assess cell type specific manifestations of different hallmarks of aging, such as senescence, changes in the activity of metabolic pathways, depletion of stem-cell populat...</p>
          <p><a href="/tabula-muris-senis/">Details &rarr;</a></p>
        </div>
        <div id="gmsdata" class="dataset">
          <h3><a href="/gmsdata/">The Genome Modeling System</a></h3>
          <p><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span></p>
          <p>The Genome Institute at Washington University has developed a high-throughput, fault-tolerant analysis information management system called the Genome Modeling System (GMS), capable of executing complex, interdependent, and automated genome analysis pipelines at a massive scale. The GMS framework provides detailed tracking of samples and data coupled with reliable and repeatable analysis pipelines. GMS includes a full system image with software and services, expandable from one workstation to a large compute cluster.</p>
          <p><a href="/gmsdata/">Details &rarr;</a></p>
        </div>
        <div id="hcp-openaccess" class="dataset">
          <h3><a href="/hcp-openaccess/">The Human Connectome Project</a></h3>
          <p><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">neuroimaging</span></p>
          <p>The Human Connectome Project aims to provide an unparalleled compilation of neural data, an interface to graphically navigate this data and the opportunity to achieve never before realized conclusions about the living human brain.</p>
          <p><a href="/hcp-openaccess/">Details &rarr;</a></p>
        </div>
        <div id="ubc-sunflower-genome" class="dataset">
          <h3><a href="/ubc-sunflower-genome/">University of British Columbia Sunflower Genome Dataset</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">biodiversity</span><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">food security</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>This dataset captures Sunflower&#39;s genetic diversity originating
from thousands of wild, cultivated, and landrace sunflower
individuals distributed across North America.The data consists of raw sequences and associated botanical metadata,
aligned sequences (to three different reference genomes), and sets of
SNPs computed across several cohorts.</p>
          <p><a href="/ubc-sunflower-genome/">Details &rarr;</a></p>
        </div>
        <div id="stdpopsim_kern" class="dataset">
          <h3><a href="/stdpopsim_kern/">stdpopsim species resources</a></h3>
          <p><span class="label label-info tag link-tag">genetic maps</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">population genetics</span><span class="label label-info tag link-tag">recombination maps</span><span class="label label-info tag link-tag">simulations</span></p>
          <p>Contains all resources (genome specifications, recombination maps, etc.) required for species specific simulation with the stdpopsim package. These resources are originally from a variety of other consortium and published work but are consolidated here for ease of access and use. If you are interested in adding a new species to the stdpopsim resource please raise an issue on the stdpopsim GitHub page to have the necessary files added here.</p>
          <p><a href="/stdpopsim_kern/">Details &rarr;</a></p>
        </div>
        <div id="binding-db" class="dataset">
          <h3><a href="/binding-db/">Binding DB</a></h3>
          <p><span class="label label-info tag link-tag">biotech blueprint</span><span class="label label-info tag link-tag">chemistry</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">molecule</span><span class="label label-info tag link-tag">parquet</span></p>
          <p>This a parquet representation of The Binding Database&#39;s <a href="https://www.bindingdb.org/bind/chemsearch/marvin/SDFdownload.jsp?all_download=yes">Full BindingDB Database Dump</a> that you can query straight from Athena in under 60 seconds (no Oracle database required). The Binding Database projects aims to make experimental data on the noncovalent association of molecules in solution searchable via the WWW. The initial focus is on biomolecular systems, but data on host-guest and supramolecular systems are also important and being included over time. It is expected that the enhanced access to data provided by this resource will facilitate drug-discovery, the design of self-assemblin...</p>
          <p><a href="/binding-db/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/aws-samples/data-lake-as-code/blob/roda/docs/roda_install.md" class="aws-link" target="_none">Data Lake as Code Deployment Guide</a> by AWS Biotech Blueprints Team</li></a>
          </li>
          <li>
            <a href="https://aws.amazon.com/blogs/startups/a-data-lake-as-code-featuring-chembl-and-opentargets/" class="aws-link" target="_none">Data Lake as Code, Featuring ChEMBL and Open Targets</a> by Paul Underwood</li></a>
          </li>
        </ul>
        <p><a href="/binding-db/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="chembl" class="dataset">
          <h3><a href="/chembl/">ChEMBL</a></h3>
          <p><span class="label label-info tag link-tag">biotech blueprint</span><span class="label label-info tag link-tag">chemistry</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">molecule</span><span class="label label-info tag link-tag">parquet</span></p>
          <p>ChEMBL is a manually curated database of bioactive molecules with drug-like properties. It brings together chemical, bioactivity and genomic data to aid the translation of genomic information into effective new drugs. This representation of ChEMBL is stored in Parquet format and most easily utilized through Amazon Athena. Follow the documentation for install instructions (&lt; 2 minute install). New ChEMBL releases occur sporadically; the most up to date information on ChEMBL releases can be found <a href="https://chembl.gitbook.io/chembl-interface-documentation/downloads">here</a>.</p>
          <p><a href="/chembl/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/aws-samples/data-lake-as-code/blob/roda/docs/roda_install.md" class="aws-link" target="_none">Data Lake as Code Deployment Guide</a> by AWS Biotech Blueprints Team</li></a>
          </li>
          <li>
            <a href="https://aws.amazon.com/blogs/startups/a-data-lake-as-code-featuring-chembl-and-opentargets/" class="aws-link" target="_none">Data Lake as Code, Featuring ChEMBL and Open Targets</a> by Paul Underwood</li></a>
          </li>
        </ul>
        <p><a href="/chembl/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="clinvar" class="dataset">
          <h3><a href="/clinvar/">ClinVar</a></h3>
          <p><span class="label label-info tag link-tag">biotech blueprint</span><span class="label label-info tag link-tag">chemistry</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">parquet</span></p>
          <p>ClinVar is a freely accessible, public archive of reports of the relationships among human variations and phenotypes, with supporting evidence. ClinVar thus facilitates access to and communication about the relationships asserted between human variation and observed health status, and the history of that interpretation. ClinVar processes submissions reporting variants found in patient samples, assertions made regarding their clinical significance, information about the submitter, and other supporting data. The alleles described in submissions are mapped to reference sequences, and reported acc...</p>
          <p><a href="/clinvar/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://aws.amazon.com/blogs/startups/a-data-lake-as-code-featuring-chembl-and-opentargets/" class="aws-link" target="_none">Data Lake as Code, Featuring ChEMBL and Open Targets</a> by Paul Underwood</li></a>
          </li>
          <li>
            <a href="https://github.com/aws-samples/data-lake-as-code/blob/roda/docs/roda_install.md" class="aws-link" target="_none">Data Lake as Code Deployment Guide</a> by AWS Biotech Blueprints Team</li></a>
          </li>
        </ul>
        <p><a href="/clinvar/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="opentargets" class="dataset">
          <h3><a href="/opentargets/">Open Targets</a></h3>
          <p><span class="label label-info tag link-tag">biotech blueprint</span><span class="label label-info tag link-tag">chemistry</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">molecule</span><span class="label label-info tag link-tag">parquet</span></p>
          <p>This a Parquet representation of the Open Targets Platform&#39;s <a href="https://www.targetvalidation.org/downloads/data">2020-06 export</a>. The Open Targets Platform integrates evidence from genetics, genomics, transcriptomics, drugs, animal models and scientific literature to score and rank target-disease associations for drug target identification. The Open Targets Platform (<a href="https://www.targetvalidation.org">https://www.targetvalidation.org</a>) is a freely available resource for the integration of genetics, genomics, and chemical data to aid systematic drug target identification and prioritisation.</p>
          <p><a href="/opentargets/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/aws-samples/data-lake-as-code/blob/roda/docs/roda_install.md" class="aws-link" target="_none">Data Lake as Code Deployment Guide</a> by AWS Biotech Blueprints Team</li></a>
          </li>
          <li>
            <a href="https://aws.amazon.com/blogs/startups/a-data-lake-as-code-featuring-chembl-and-opentargets/" class="aws-link" target="_none">Data Lake as Code, Featuring ChEMBL and Open Targets</a> by Paul Underwood</li></a>
          </li>
        </ul>
        <p><a href="/opentargets/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="cord-19" class="dataset">
          <h3><a href="/cord-19/">COVID-19 Open Research Dataset (CORD-19)</a></h3>
          <p><span class="label label-info tag link-tag">coronavirus</span><span class="label label-info tag link-tag">COVID-19</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">MERS</span><span class="label label-info tag link-tag">SARS</span></p>
          <p>Full-text and metadata dataset of COVID-19 and coronavirus-related research articles optimized for machine readability.</p>
          <p><a href="/cord-19/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.kaggle.com/allen-institute-for-ai/CORD-19-research-challenge" class="" target="_none">COVID-19 Open Research Dataset Challenge (CORD-19)</a> by Kaggle</li></a>
          </li>
        </ul>
        <p><a href="/cord-19/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
      </div>
    </div>


    <hr/>
  </div>  </body>
  <script src="/js/index.js"></script>
</html>
