<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    
  
    
  
    <link rel="alternate" type="application/rss+xml" href="/rss.xml" />
    
    <link rel="shortcut icon" type="image/ico" href="https://a0.awsstatic.com/libra-css/images/site/fav/favicon.ico"/>
    <link rel="apple-touch-icon" sizes="57x57" href="https://a0.awsstatic.com/libra-css/images/site/touch-icon-iphone-114-smile.png"/>
    <link rel="apple-touch-icon" sizes="72x72" href="https://a0.awsstatic.com/libra-css/images/site/touch-icon-ipad-144-smile.png"/>
    <link rel="apple-touch-icon" sizes="114x114" href="https://a0.awsstatic.com/libra-css/images/site/touch-icon-iphone-114-smile.png"/>
    <link rel="apple-touch-icon" sizes="144x144" href="https://a0.awsstatic.com/libra-css/images/site/touch-icon-ipad-144-smile.png"/>  
  
    <title>Registry of Open Data on AWS</title>
  
    <!-- Latest compiled and minified CSS -->
    <link rel="stylesheet" href="https://assets.opendata.aws/css/bootstrap/3.4.1/bootstrap.min.css">
  
    <!-- Our local CSS -->
    <link rel="stylesheet" href="/css/main.css">
  
  
  </head>
  <body>
    <!-- jQuery (necessary for Bootstrap's JavaScript plugins) -->
    <script src="https://assets.opendata.aws/js/jquery/3.5.1/jquery.min.js"></script>
    <!-- Latest compiled and minified JavaScript -->
    <script src="https://assets.opendata.aws/js/bootstrap/3.4.1/bootstrap.min.js"></script>
    <nav class="navbar navbar-default navbar-fixed-top">
      <div class="container">
        <div class="roda-header">
          <h2><a href="/" alt="Home">Registry of Open Data on AWS</a></h2>
          <a href="http://aws.amazon.com/what-is-cloud-computing">
            <img src="https://assets.opendata.aws/img/AWS-Logo_White-Color_300x180.png" alt="Powered by AWS Cloud Computing" id="aws_header_logo">
          </a>
        </div>
      </div>
    </nav>
    <div class="container" >

    <div class="about col-md-5">
      <div class="row aboutbox">
            <h3>About</h3>
            <p>This registry exists to help people discover and share datasets that are available via AWS resources. <a href="https://opendata.aws">Learn more about sharing data on AWS</a>.</p>
    
            <p>See <a href="/tag/biology/usage-examples">all usage examples for datasets listed in this registry</a> tagged with <strong>biology</strong>.</p>
    
        <hr>
    
        <h4>Search datasets (currently <span id="count-matching">13</span> matching <span id="count-matching-text">datasets</span>)</h4>
        <form>
          <div class="form-group">
            <input type="text" id="search-box" class="form-control" placeholder="Search datasets" spellcheck="false" autocorrect="off">
          </div>
        </form>
        <p>You are currently viewing a subset of data tagged with <strong>biology</strong>.</p>
    
        <hr>
    
        <h4>Add to this registry</h4>
        <p>If you want to add a dataset or example of how to use a dataset to this registry, please follow the instructions on the <a href="https://github.com/awslabs/open-data-registry/">Registry of Open Data on AWS GitHub repository</a>.</p>
    
        <p>Unless specifically stated in the applicable dataset documentation, datasets available through the Registry of Open Data on AWS are not provided and maintained by AWS. Datasets are provided and maintained by a variety of third parties under a variety of licenses. Please check dataset licenses and related documentation to determine if a dataset may be used for your application.</p>
      </div>
    </div>
    <div class="col-md-6 col-md-offset-1 datasets">
      <div class="row">
        <div id="allen-cell-imaging-collections" class="dataset">
          <h3><a href="/allen-cell-imaging-collections/">Allen Cell Imaging Collections</a></h3>
          <p><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">cell biology</span><span class="label label-info tag link-tag">cell imaging</span><span class="label label-info tag link-tag">Homo sapiens</span><span class="label label-info tag link-tag">image processing</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">microscopy</span></p>
          <p>This bucket contains multiple datasets (as Quilt packages) created by the
Allen Institute for Cell Science (AICS). The imaging data in this bucket contains
either of the following:1) field of view images from glass plates
2) cell membrane, DNA, and structure segmentations
3) cell membrane, DNA and structure contours
4) machine learning imaging predictions of the previously listed modalities.In addition, many of the datasets include CSVs that contain feature sets
related to that data.</p>
          <p><a href="/allen-cell-imaging-collections/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://drive.google.com/file/d/1nuiYOurmbic7rZeHPen_M3fDbsN7n5Ws/view?usp&#x3D;sharing" class="" target="_none">Integrating single-cell sequencing and nuclear imaging data</a> by <a href="https://noble.gs.washington.edu/" target="_none">Chengxiang Qiu and William Noble</a></li></a>
          </li>
          <li>
            <a href="https://www.allencell.org/segmenter.html" class="" target="_none">Allen Cell Structure Segmenter</a> by <a href="https://www.alleninstitute.org/what-we-do/cell-science/about/team/" target="_none">Jianxu Chen, Liya Ding, Matheus P. Viana, Melissa C. Hendershott, Ruian Yang, Irina A. Mueller, Susanne M. Rafelski</a></li></a>
          </li>
          <li>
            <a href="https://cfe.allencell.org/" class="" target="_none">Allen Cell Feature Explorer</a> by <a href="https://www.alleninstitute.org/what-we-do/cell-science/about/team/" target="_none">Allen Institute for Cell Science</a></li></a>
          </li>
          <li>
            <a href="https://www.biorxiv.org/content/10.1101/2020.12.08.415562v1" class="" target="_none">Robust integrated intracellular organization of the human iPS cell: where, how much, and how variable?</a> by <a href="https://www.alleninstitute.org/what-we-do/cell-science/about/team/" target="_none">Matheus P. Viana, Jianxu Chen*, Theo A. Knijnenburg*, Ritvik Vasan*, Calysta Yan*... Allen Institute for Cell Science... Graham T. Johnson, Ruwanthi N. Gunawardane, Nathalie Gaudreault, Julie A. Theriot, Susanne M. Rafelski</a></li></a>
          </li>
          <li>
            <a href="https://www.nature.com/articles/s41592-018-0111-2" class="" target="_none">Label-free prediction of three-dimensional fluorescence images from transmitted-light microscopy</a> by <a href="https://alleninstitute.org/about/team/" target="_none">Chawin Ounkomol, Sharmishtaa Seshamani, Mary M. Maleckar, Forrest Collman &amp; Gregory R. Johnson</a></li></a>
          </li>
        </ul>
        <p><a href="/allen-cell-imaging-collections/#usageexamples">See 11 usage examples &rarr;</a></p>
        </div>
        <div id="janelia-flylight" class="dataset">
          <h3><a href="/janelia-flylight/">Fly Brain Anatomy: FlyLight Gen1 and Split-GAL4 Imagery</a></h3>
          <p><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">fluorescence imaging</span><span class="label label-info tag link-tag">image processing</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">microscopy</span><span class="label label-info tag link-tag">neurobiology</span><span class="label label-info tag link-tag">neuroimaging</span><span class="label label-info tag link-tag">neuroscience</span></p>
          <p>This data set, made available by Janelia&#39;s FlyLight project, consists of fluorescence images 
of Drosophila melanogaster driver lines, aligned to standard templates, and stored in formats 
suitable for rapid searching and visualization. Additional data will be added as it is published. 
A large release of Gen1 MCFO samples is coming at the beginning of May 2020. </p>
          <p><a href="/janelia-flylight/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1016/j.celrep.2012.09.011" class="" target="_none">A GAL4-Driver Line Resource for Drosophila Neurobiology</a> by Jenett, A., Rubin, G. M., Ngo, T., Shepherd, D., Murphy, C., Dionne, H., Pfeiffer, B. D., Cavallaro, A., Hall, D., Jeter, J., Iyer, N., Fetter, D., Hausenfluck, J. H., Peng, H., Trautman, E. T., Svirskas, R. R., Myers, E. W., Iwinski, Z. R., Aso, Y., DePasquale, G. M., Enos, A., Hulamm, P., Lam, S. C. B., Li, H., Laverty, T. R., Long, F., Qu, L., Murphy, S. D., Rokicki, K., Safford, T., Shaw, K., Simpson, J. H., Sowell, A., Tae, S., Yu, Y., &amp; Zugates, C. T.</li></a>
          </li>
          <li>
            <a href="https://neuronbridge.janelia.org" class="" target="_none">NeuronBridge</a> by Jody Clements, Rob Svirskas, Hideo Otsuna, Cristian Goina, Konrad Rokicki</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1101/376384" class="" target="_none">An unbiased template of the Drosophila brain and ventral nerve cord</a> by Bogovic, J.A., Otsuna, H., Heinrich, L., Ito, M., Jeter, J., Meissner, G., Nern, A., Colonell, J., Malkesman, O., Ito, K., &amp; Saalfeld, S.</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1101/318006" class="" target="_none">Color depth MIP mask search: a new tool to expedite Split-GAL4 creation</a> by Otsuna, H., Ito, M., &amp; Kawase, T.</li></a>
          </li>
          <li>
            <a href="https://github.com/JaneliaSciComp/open-data-flylight/blob/master/tutorials/Using_imagery_on_AWS_S3.ipynb" class="" target="_none">Using Imagery on AWS S3</a> by Rob Svirskas</li></a>
          </li>
        </ul>
        <p><a href="/janelia-flylight/#usageexamples">See 10 usage examples &rarr;</a></p>
        </div>
        <div id="open-neurodata" class="dataset">
          <h3><a href="/open-neurodata/">Open NeuroData</a></h3>
          <p><span class="label label-info tag link-tag">array tomography</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">electron microscopy</span><span class="label label-info tag link-tag">image processing</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">lightsheet microscopy</span><span class="label label-info tag link-tag">magnetic resonance imaging</span><span class="label label-info tag link-tag">neuroimaging</span><span class="label label-info tag link-tag">neuroscience</span></p>
          <p>This bucket contains multiple neuroimaging datasets (as Neuroglancer Precomputed Volumes) across multiple modalities and scales, ranging from nanoscale (electron microscopy), to microscale (cleared lightsheet microscopy and array tomography), and mesoscale (structural and functional magnetic resonance imaging). Additionally, many of the datasets include segmentations and meshes.</p>
          <p><a href="/open-neurodata/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://arxiv.org/abs/1306.3543" class="" target="_none">The Open Connectome Project Data Cluster: Scalable Analysis and Vision for High-Throughput Neuroscience</a> by R. Burns, W. G. Roncal, D. Kleissas, K. Lillaney, P. Manavalan, E. Perlman, D. R. Berger, D. D. Bock, K. Chung, L. Grosenick, N. Kasthuri, N. C. Weiler, K. Deisseroth, M. Kazhdan, J. Lichtman, R. C. Reid, S. J. Smith, A. S. Szalay, J. T. Vogelstein, and R. J. Vogelstein.</li></a>
          </li>
          <li>
            <a href="https://github.com/google/neuroglancer" class="" target="_none">Neuroglancer</a> by <a href="https://github.com/jbms" target="_none">Jeremy Maitin-Shepard</a></li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1016/j.neuron.2014.08.045" class="" target="_none">From cosmos to connectomes: The evolution of data-intensive science</a> by R. Burns, J. T. Vogelstein, and A. S. Szalay</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1016/j.neuron.2016.10.033" class="" target="_none">To the Cloud! A Grassroots Proposal to Accelerate Brain Science Discovery</a> by J. T. Vogelstein, B. Mensh, M. Häusser, N. Spruston, A. C. Evans, K. Kording, K. Amunts, C. Ebell, J. Muller, M. Telefont, S. Hill, S. P. Koushika, C. Calì, P. A. Valdés-Sosa, P. B. Littlewood, C. Koch, S. Saalfeld, A. Kepecs, H. Peng, Y. O. Halchenko, G. Kiar, M. M. Poo, J. B. Poline, M. P. Milham, A. P. Schaffer, R. Gidron, H. Okano, V. D. Calhoun, M. Chun, D. M. Kleissas, R. J. Vogelstein, E. Perlman, R. Burns, R. Huganir, and M. I. Miller</li></a>
          </li>
          <li>
            <a href="https://github.com/seung-lab/igneous" class="" target="_none">Igneous</a> by <a href="https://github.com/william-silversmith" target="_none">William Silversmith</a></li></a>
          </li>
        </ul>
        <p><a href="/open-neurodata/#usageexamples">See 9 usage examples &rarr;</a></p>
        </div>
        <div id="ebirdst" class="dataset">
          <h3><a href="/ebirdst/">eBird Status and Trends Model Results</a></h3>
          <p><span class="label label-info tag link-tag">biodiversity</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">ecosystems</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info-sustainability tag link-tag">sustainability</span></p>
          <p>The eBird Status and Trends project generates estimates of bird
occurrence and abundance at a high spatiotemporal resolution.
This dataset represents the primary modeled results from the
analysis workflow and are designed for further analysis,
synthesis, visualization, and exploration.</p>
          <p><a href="/ebirdst/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://cornelllabofornithology.github.io/ebirdst/articles/ebirdst-advanced-mapping.html" class="" target="_none">ebirdst - Generating Seasonal Abundance and Range Maps and Stats</a> by Matt Strimas-Mackey, Tom Auer, and Daniel Fink</li></a>
          </li>
          <li>
            <a href="https://ebird.org/science/status-and-trends/faq" class="" target="_none">eBird Status and Trends FAQ</a> by Cornell Lab of Ornithology</li></a>
          </li>
          <li>
            <a href="https://cornelllabofornithology.github.io/ebirdst/articles/ebirdst-intro-mapping.html" class="" target="_none">ebirdst - Intro Mapping</a> by Tom Auer and Daniel Fink</li></a>
          </li>
          <li>
            <a href="https://cornelllabofornithology.github.io/ebirdst/articles/ebirdst-non-raster.html" class="" target="_none">ebirdst - Non-raster Data</a> by Tom Auer and Daniel Fink</li></a>
          </li>
          <li>
            <a href="https://ebird.org/science/status-and-trends" class="" target="_none">eBird Status and Trends Visualizations</a> by Cornell Lab of Ornithology</li></a>
          </li>
        </ul>
        <p><a href="/ebirdst/#usageexamples">See 8 usage examples &rarr;</a></p>
        </div>
        <div id="nyu-fastmri" class="dataset">
          <h3><a href="/nyu-fastmri/">NYU Langone &amp; FAIR FastMRI Dataset</a></h3>
          <p><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">health</span><span class="label label-info tag link-tag">image processing</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">magnetic resonance imaging</span><span class="label label-info tag link-tag">neurobiology</span><span class="label label-info tag link-tag">neuroimaging</span></p>
          <p>This dataset contains deidentified raw k-space data and DICOM image files of over 1,500 knees and 6,970 brains.</p>
          <p><a href="/nyu-fastmri/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/facebookresearch/fastMRI/blob/master/fastMRI_tutorial.ipynb" class="" target="_none">FastMRI Tutorial (Jupyter Notebook)</a> by <a href="https://scholar.google.com/citations?user&#x3D;LjjLn2YAAAAJ&amp;hl&#x3D;en" target="_none">Tullie Murrell</a></li></a>
          </li>
          <li>
            <a href="https://arxiv.org/abs/1811.08839" class="" target="_none">fastMRI:An Open Dataset and Benchmarks for Accelerated MRI</a> by <a href="https://arxiv.org/search/cs?searchtype&#x3D;author&amp;query&#x3D;Zbontar%2C+J" target="_none">Zbontar et al (2019)</a></li></a>
          </li>
          <li>
            <a href="https://medium.com/@khodges42/how-to-get-started-diy-neurohacking-with-the-fastmri-dataset-and-why-you-should-6a02a6bb896e" class="" target="_none">How to Get Started Working with the FastMRI Dataset (and why you should!)</a> by Kevin Hodges</li></a>
          </li>
          <li>
            <a href="https://arxiv.org/abs/2001.02518" class="" target="_none">Advancing machine learning for MR image reconstruction with an open competition:Overview of the 2019 fastMRI challenge</a> by <a href="https://arxiv.org/search/eess?searchtype&#x3D;author&amp;query&#x3D;Knoll%2C+F" target="_none">Knoll et al (2020)</a></li></a>
          </li>
          <li>
            <a href="https://arxiv.org/abs/1904.01112" class="" target="_none">Deep Learning Methods for Parallel Magnetic Resonance Image Reconstruction</a> by <a href="https://arxiv.org/search/eess?searchtype&#x3D;author&amp;query&#x3D;Knoll%2C+F" target="_none">Knoll et al (2019)</a></li></a>
          </li>
        </ul>
        <p><a href="/nyu-fastmri/#usageexamples">See 6 usage examples &rarr;</a></p>
        </div>
        <div id="aws-covid19-lake" class="dataset">
          <h3><a href="/aws-covid19-lake/">COVID-19 Data Lake</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">coronavirus</span><span class="label label-info tag link-tag">COVID-19</span><span class="label label-info tag link-tag">health</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">medicine</span><span class="label label-info tag link-tag">MERS</span><span class="label label-info tag link-tag">SARS</span></p>
          <p>A centralized repository of up-to-date and curated datasets on or related to the spread and characteristics of the novel corona virus (SARS-CoV-2) and its associated illness, COVID-19. Globally, there are several efforts underway to gather this data, and we are working with partners to make this crucial data freely available and keep it up-to-date. Hosted on the AWS cloud, we have seeded our curated data lake with COVID-19 case tracking data from Johns Hopkins and The New York Times, hospital bed availability from Definitive Healthcare, and over 45,000 research articles about COVID-19 and rela...</p>
          <p><a href="/aws-covid19-lake/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://aws.amazon.com/blogs/big-data/exploring-the-public-aws-covid-19-data-lake/" class="aws-link" target="_none">Exploring the public AWS COVID-19 data lake</a> by <a href="https://aws.amazon.com/big-data/datalakes-and-analytics/" target="_none">AWS Data Lake Team</a></li></a>
          </li>
          <li>
            <a href="https://us-east-2.console.aws.amazon.com/cloudformation/home?region&#x3D;us-east-2#/stacks/create/review?templateURL&#x3D;https://covid19-lake.s3.us-east-2.amazonaws.com/cfn/CovidLakeStack.template.json&amp;stackName&#x3D;CovidLakeStack" class="" target="_none">CloudFormation template for Glue Catalog table definitions</a> by <a href="https://aws.amazon.com/big-data/datalakes-and-analytics/" target="_none">AWS Data Lake Team</a></li></a>
          </li>
          <li>
            <a href="https://aws.amazon.com/blogs/big-data/a-public-data-lake-for-analysis-of-covid-19-data/" class="aws-link" target="_none">A public data lake for analysis of COVID-19 data</a> by <a href="https://aws.amazon.com/big-data/datalakes-and-analytics/" target="_none">AWS Data Lake Team</a></li></a>
          </li>
          <li>
            <a href="https://github.com/aws-samples/query-data-in-s3-with-amazon-athena-and-aws-sdk-for-dotnet" class="aws-link" target="_none">How to use SQL to query data in S3 Bucket with Amazon Athena and AWS SDK for .NET</a> by <a href="https://github.com/orgs/aws-samples/teams/aws-proserve-us-west-applications" target="_none">AWS ProServe US West Applications Team</a></li></a>
          </li>
          <li>
            <a href="https://go.aws/covid-19-data-lake" class="aws-link" target="_none">Explore the COVID-19 data lake public S3 bucket</a> by <a href="https://aws.amazon.com/big-data/datalakes-and-analytics/" target="_none">AWS Data Lake Team</a></li></a>
          </li>
        </ul>
        <p><a href="/aws-covid19-lake/#usageexamples">See 5 usage examples &rarr;</a></p>
        </div>
        <div id="ilmn-dragen-1kgp" class="dataset">
          <h3><a href="/ilmn-dragen-1kgp/">1000 Genomes Phase 3 Reanalysis with DRAGEN 3.5</a></h3>
          <p><span class="label label-info tag link-tag">bam</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">health</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">vcf</span></p>
          <p>This dataset contains alignment files and short nucleotide, copy number, and structural variant call files from the newly revised (n=3202) 1000 Genomes Project Phase 3 using Illumina DRAGEN v3.5.7b</p>
          <p><a href="/ilmn-dragen-1kgp/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://support.illumina.com/content/dam/illumina-support/documents/downloads/software/dragen/DRAGEN-3.5.7b-Customer-Release-Notes-1000000131537-00.pdf" class="" target="_none">DRAGEN 3.7.5b Release</a> by <a href="https://www.illumina.com/" target="_none">Illumina Inc.</a></li></a>
          </li>
          <li>
            <a href="https://www.illumina.com/products/by-type/informatics-products/basespace-sequence-hub.html#key-features" class="" target="_none">DRAGEN on BaseSpace Sequence Hub</a> by <a href="https://www.illumina.com/" target="_none">Illumina Inc.</a></li></a>
          </li>
          <li>
            <a href="https://github.com/aws-quickstart/quickstart-illumina-dragen" class="" target="_none">DRAGEN Quick Start on AWS</a> by <a href="https://github.com/aws-quickstart" target="_none">AWS Quick Start Team</a></li></a>
          </li>
          <li>
            <a href="https://www.biorxiv.org/content/10.1101/2020.11.13.380741v1" class="" target="_none">precisionFDA Truth Challenge V2: Calling variants from short- and long-reads in difficult-to-map regions (Preprint)</a> by Olson et al (2020)</li></a>
          </li>
        </ul>
        <p><a href="/ilmn-dragen-1kgp/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="encode-project" class="dataset">
          <h3><a href="/encode-project/">Encyclopedia of DNA Elements (ENCODE)</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">deep learning</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">machine learning</span></p>
          <p>The Encyclopedia of DNA Elements (ENCODE) Consortium is an international collaboration of
research groups funded by the National Human Genome Research Institute (NHGRI). The goal
of ENCODE is to build a comprehensive parts list of functional elements in the human genome,
including elements that act at the protein and RNA levels, and regulatory elements that
control cells and circumstances in which a gene is active. ENCODE investigators employ a
variety of assays and methods to identify functional elements. The discovery and annotation
of gene elements is accomplished primarily by sequencing a ...</p>
          <p><a href="/encode-project/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/ENCODE-DCC/encode-data-usage-examples/blob/master/mount_s3_bucket_and_run_jupyter_on_ec2.ipynb" class="" target="_none">Exploring ENCODE data from EC2 with Jupyter notebook</a> by <a href="https://github.com/keenangraham" target="_none">Keenan Graham</a></li></a>
          </li>
          <li>
            <a href="https://academic.oup.com/nar/article/48/D1/D882/5622708#190992715" class="" target="_none">New developments on the Encyclopedia of DNA Elements (ENCODE) data portal</a> by <a href="http://orcid.org/0000-0001-6948-2042" target="_none">Luo et al 2020</a></li></a>
          </li>
          <li>
            <a href="https://github.com/ENCODE-DCC/encode-data-usage-examples/blob/master/ingest_encode_data_tile_db_with_s3_backend.ipynb" class="" target="_none">Ingesting ENCODE data into TileDB with S3 backend</a> by <a href="https://github.com/ottojolanki" target="_none">Otto Jolanki</a></li></a>
          </li>
          <li>
            <a href="https://github.com/ENCODE-DCC/encode-data-usage-examples/blob/master/ctcf_chip_seq_cross_cell_type_correlation.ipynb" class="" target="_none">ENCODE CTCF ChIP-seq data correlation across different cell types</a> by <a href="https://github.com/p-sud" target="_none">Paul Sud</a></li></a>
          </li>
        </ul>
        <p><a href="/encode-project/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="refgenie" class="dataset">
          <h3><a href="/refgenie/">Refgenie reference genome assets</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">infrastructure</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">single-cell transcriptomics</span><span class="label label-info tag link-tag">transcriptomics</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>Pre-built refgenie reference genome data assets used for aligning and analyzing DNA sequence data.</p>
          <p><a href="/refgenie/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="http://refgenie.databio.org/en/latest/tutorial/" class="" target="_none">Basic Refgenie tutorial</a> by <a href="http://www.databio.org" target="_none">Nathan Sheffield</a></li></a>
          </li>
          <li>
            <a href="http://peppro.databio.org" class="" target="_none">Pipeline for PRO-seq data analysis</a> by <a href="http://databio.org" target="_none">Jason Smith and Nathan Sheffield</a></li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1093/gigascience/giz149" class="" target="_none">Refgenie: a reference genome resource manager</a> by <a href="http://www.databio.org" target="_none">Michał Stolarczyk, Vincent P Reuter, Jason P Smith, Neal E Magee, Nathan C Sheffield</a></li></a>
          </li>
          <li>
            <a href="http://pepatac.databio.org" class="" target="_none">Pipeline for ATAC-seq data analysis</a> by <a href="http://databio.org" target="_none">Jason Smith and Nathan Sheffield</a></li></a>
          </li>
        </ul>
        <p><a href="/refgenie/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="allen-mouse-brain-atlas" class="dataset">
          <h3><a href="/allen-mouse-brain-atlas/">Allen Mouse Brain Atlas</a></h3>
          <p><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">gene expression</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">image processing</span><span class="label label-info tag link-tag">imaging</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">Mus musculus</span><span class="label label-info tag link-tag">neurobiology</span><span class="label label-info tag link-tag">transcriptomics</span></p>
          <p>The Allen Mouse Brain Atlas is a genome-scale collection of cellular resolution gene expression profiles using in situ hybridization (ISH). Highly methodical data production methods and comprehensive anatomical coverage via dense, uniformly spaced sampling facilitate data consistency and comparability across &gt;20,000 genes. The use of an inbred mouse strain with minimal animal-to-animal variance allows one to treat the brain essentially as a complex but highly reproducible three-dimensional tissue array. The entire Allen Mouse Brain Atlas dataset and associated tools are available through an...</p>
          <p><a href="/allen-mouse-brain-atlas/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="http://www.nature.com/articles/nature05453" class="" target="_none">Genome-wide atlas of gene expression in the adult mouse brain</a> by <a href="www.alleninstitute.org" target="_none">Ed Lein, et al.</a></li></a>
          </li>
          <li>
            <a href="https://mouse.brain-map.org" class="" target="_none">Allen Mouse Brain Atlas</a> by <a href="www.alleninstitute.org" target="_none">Allen Institute for Brain Science</a></li></a>
          </li>
          <li>
            <a href="https://github.com/AllenInstitute/open_dataset_tools/blob/master/Visualizing_Images_from_Allen_Mouse_Brain_Atlas.ipynb" class="" target="_none">Visualizing Images from the Allen Mouse Brain Atlas</a> by <a href="www.alleninstitute.org" target="_none">Allen Institute for Brain Science</a></li></a>
          </li>
        </ul>
        <p><a href="/allen-mouse-brain-atlas/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="ncbi-blast-databases" class="dataset">
          <h3><a href="/ncbi-blast-databases/">Basic Local Alignment Sequences Tool (BLAST) Databases</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">health</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">protein</span><span class="label label-info tag link-tag">reference index</span><span class="label label-info tag link-tag">transcriptomics</span></p>
          <p>A centralized repository of pre-formatted BLAST databases created by the National Center for Biotechnology Information (NCBI).</p>
          <p><a href="/ncbi-blast-databases/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://pubmed.ncbi.nlm.nih.gov/20003500" class="" target="_none">BLAST+: Architecture and Applications</a> by Christiam Camacho  1 , George Coulouris, Vahram Avagyan, Ning Ma, Jason Papadopoulos, Kevin Bealer, Thomas L Madden</li></a>
          </li>
          <li>
            <a href="https://github.com/ncbi/blast_plus_docs" class="" target="_none">BLAST+ Docker</a> by NCBI BLAST</li></a>
          </li>
          <li>
            <a href="https://pubmed.ncbi.nlm.nih.gov/9254694/" class="" target="_none">Gapped BLAST and PSI-BLAST: A New Generation of Protein Database Search Programs</a> by S F Altschul, T L Madden, A A Schäffer, J Zhang, Z Zhang, W Miller, D J Lipman</li></a>
          </li>
        </ul>
        <p><a href="/ncbi-blast-databases/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="dandiarchive" class="dataset">
          <h3><a href="/dandiarchive/">Distributed Archives for Neurophysiology Data Integration (DANDI)</a></h3>
          <p><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">cell imaging</span><span class="label label-info tag link-tag">electrophysiology</span><span class="label label-info tag link-tag">infrastructure</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">neuroimaging</span><span class="label label-info tag link-tag">neurophysiology</span><span class="label label-info tag link-tag">neuroscience</span></p>
          <p>DANDI is a public archive of neurophysiology datasets, including raw and processed data,  and associated software containers. Datasets are shared according to a Creative Commons  CC0 or CC-BY licenses. The data archive provides a broad range of cellular neurophysiology data.  This includes electrode and optical recordings, and associated imaging data using a  set of community standards: <a href="https://www.nwb.org/nwb-neurophysiology/">NWB:N - NWB:Neurophysiology</a>, <a href="https://bids.neuroimaging.io/">BIDS - Brain Imaging Data Structure</a>, and  <a href="htt...</p>
          <p><a href="/dandiarchive/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://hub.dandiarchive.org/" class="" target="_none">DANDI JupyterHub Interface</a> by <a href="https://dandiarchive.org/" target="_none">DANDI Project</a></li></a>
          </li>
          <li>
            <a href="https://github.com/dandi/dandi-cli" class="" target="_none">DANDI Shell Interface</a> by <a href="https://dandiarchive.org/" target="_none">DANDI Project</a></li></a>
          </li>
          <li>
            <a href="https://dandiarchive.org/" class="" target="_none">DANDI Web Interface</a> by <a href="https://dandiarchive.org/" target="_none">DANDI Project</a></li></a>
          </li>
        </ul>
        <p><a href="/dandiarchive/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="nsw-herbarium" class="dataset">
          <h3><a href="/nsw-herbarium/">National Herbarium of NSW</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">biodiversity</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">digital preservation</span><span class="label label-info tag link-tag">ecosystems</span><span class="label label-info tag link-tag">environmental</span></p>
          <p>The National Herbarium of New South Wales is one of the most significant scientific, cultural and historical botanical resources in the Southern hemisphere. The 1.43 million preserved plant specimens have been captured as high-resolution images and the biodiversity metadata associated with each of the images captured in digital form. Botanical specimens date from year 1770 to today, and form voucher collections that document the distribution and diversity of the world&#39;s flora through time, particularly that of NSW, Austalia and the Pacific.The data is used in biodiversity assessment, syste...</p>
          <p><a href="/nsw-herbarium/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.ala.org.au/" class="" target="_none">Atlas of Living Australia</a> by <a href="https://www.ala.org.au/about-ala/" target="_none">CSIRO</a></li></a>
          </li>
          <li>
            <a href="https://www.rbgsyd.nsw.gov.au/science/national-herbarium-of-new-south-wales" class="" target="_none">Accessing the National Herbarium of NSW on AWS</a> by <a href="https://www.rbgsyd.nsw.gov.au/Science/Our-Science-Staff" target="_none">Dr Shelley James</a></li></a>
          </li>
          <li>
            <a href="https://avh.ala.org.au/occurrences/search?q&#x3D;collection_uid%3Aco54#tab_mapView" class="" target="_none">The Australasian Virtual Herbarium</a> by <a href="https://avh.chah.org.au/index.php/about/" target="_none">avh@chah.org.au</a></li></a>
          </li>
        </ul>
        <p><a href="/nsw-herbarium/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="broad-references" class="dataset">
          <h3><a href="/broad-references/">Broad Genome References</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">Homo sapiens</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">reference index</span></p>
          <p>Broad maintained human genome reference builds hg19/hg38 and decoy references.</p>
          <p><a href="/broad-references/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://aws.amazon.com/blogs/storage/using-amazon-fsx-for-lustre-for-genomics-workflows-on-aws/" class="aws-link" target="_none">Using Amazon FSx for Lustre for Genomics Workflows on AWS</a> by <a href="https://www.linkedin.com/in/lee-pang-a039a26/" target="_none">W. Lee Pang</a></li></a>
          </li>
          <li>
            <a href="https://docs.opendata.aws/genomics-workflows/orchestration/cromwell/cromwell-examples/#real-world-example-haplotypecaller" class="aws-link" target="_none">Genomics Workflows on AWS - Cromwell on AWS</a> by <a href="https://www.linkedin.com/in/lee-pang-a039a26/" target="_none">W. Lee Pang</a></li></a>
          </li>
        </ul>
        <p><a href="/broad-references/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="jhu-indexes" class="dataset">
          <h3><a href="/jhu-indexes/">Cloud Indexes for Genomic Analyses</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">mapping</span><span class="label label-info tag link-tag">medicine</span><span class="label label-info tag link-tag">reference index</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>Genomic tools use reference databases as indexes to operate quickly and efficiently, analogous to how web search engines use indexes for fast querying. Here, we aggregate genomic, pan-genomic and metagenomic indexes for analysis of sequencing data.</p>
          <p><a href="/jhu-indexes/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.biorxiv.org/content/10.1101/2020.03.03.975219v4.full.pdf" class="" target="_none">Reducing reference bias using multiple population reference genomes</a> by Chen et al (2020)</li></a>
          </li>
          <li>
            <a href="https://benlangmead.github.io/aws-indexes/tutorial" class="" target="_none">Table of contents for tutorials for constituent tools</a> by <a href="https://benlangmead.github.io/aws-indexes" target="_none">Ben Langmead</a></li></a>
          </li>
        </ul>
        <p><a href="/jhu-indexes/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="aws-igenomes" class="dataset">
          <h3><a href="/aws-igenomes/">AWS iGenomes</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">Caenorhabditis elegans</span><span class="label label-info tag link-tag">Danio rerio</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">Homo sapiens</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">Mus musculus</span><span class="label label-info tag link-tag">Rattus norvegicus</span><span class="label label-info tag link-tag">reference index</span></p>
          <p>Common reference genomes hosted on AWS S3. Can be used when aligning and analysing raw DNA sequencing data.</p>
          <p><a href="/aws-igenomes/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="http://nf-co.re/" class="" target="_none">nf-core analysis pipelines</a> by <a href="http://phil.ewels.co.uk/" target="_none">Phil Ewels</a></li></a>
          </li>
        </ul>
        <p><a href="/aws-igenomes/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="ncbi-covid-19" class="dataset">
          <h3><a href="/ncbi-covid-19/">COVID-19 Genome Sequence Dataset</a></h3>
          <p><span class="label label-info tag link-tag">bam</span><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">coronavirus</span><span class="label label-info tag link-tag">COVID-19</span><span class="label label-info tag link-tag">cram</span><span class="label label-info tag link-tag">fastq</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">health</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">MERS</span><span class="label label-info tag link-tag">SARS</span><span class="label label-info tag link-tag">STRIDES</span><span class="label label-info tag link-tag">transcriptomics</span><span class="label label-info tag link-tag">virus</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>A centralized sequence repository for all records containing sequence associated with the novel corona virus (SARS-CoV-2) submitted to the National Center for Biotechnology Information (NCBI) Sequence Read Archive (SRA). Included are both the original sequences submitted by the principal investigator as well as SRA-processed sequences that require the SRA Toolkit for analysis. Additionally, submitter provided metadata included in associated BioSample and BioProject records is available alongside NCBI calculated data, such k-mer based taxonomy analysis results, contiguous assemblies (contigs) a...</p>
          <p><a href="/ncbi-covid-19/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/sra/docs/sra-aws-download/" class="" target="_none">Download SRA sequence data using Amazon Web Services (AWS)</a> by NCBI SRA</li></a>
          </li>
        </ul>
        <p><a href="/ncbi-covid-19/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="cell-painting-image-collection" class="dataset">
          <h3><a href="/cell-painting-image-collection/">Cell Painting Image Collection</a></h3>
          <p><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">cell imaging</span><span class="label label-info tag link-tag">cell painting</span><span class="label label-info tag link-tag">fluorescence imaging</span><span class="label label-info tag link-tag">high-throughput imaging</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">microscopy</span></p>
          <p>The Cell Painting Image Collection is a collection of freely
downloadable microscopy image sets. Cell Painting is an
unbiased high throughput imaging assay used to analyze
perturbations in cell models. In addition to the images
themselves, each set includes a description of the biological
application and some type of &quot;ground truth&quot; (expected results).
Researchers are encouraged to use these image sets as reference
points when developing, testing, and publishing new image
analysis algorithms for the life sciences. We hope that the
this data set will lead to a better understanding of w...</p>
          <p><a href="/cell-painting-image-collection/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/cytodata/cytodata-hackathon-2018/tree/master/cytodata-toolkit/" class="" target="_none">Example submission for the 2018 CytoData Hackathon (in R and Python)</a> by <a href="broadinstitute.org" target="_none">Juan Caicedo, Tim Becker</a></li></a>
          </li>
        </ul>
        <p><a href="/cell-painting-image-collection/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="gatk-test-data" class="dataset">
          <h3><a href="/gatk-test-data/">GATK Test Data</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span></p>
          <p>The GATK test data resource bundle is a collection of files for resequencing human genomic data with the
Broad Institute&#39;s <a href="https://software.broadinstitute.org/gatk/">Genome Analysis Toolkit (GATK)</a>.</p>
          <p><a href="/gatk-test-data/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://docs.opendata.aws/genomics-workflows/orchestration/cromwell/cromwell-examples/#real-world-example-haplotypecaller" class="aws-link" target="_none">Genomics Workflows on AWS - Cromwell on AWS</a> by <a href="https://www.linkedin.com/in/lee-pang-a039a26/" target="_none">W. Lee Pang</a></li></a>
          </li>
        </ul>
        <p><a href="/gatk-test-data/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="ont-open-data" class="dataset">
          <h3><a href="/ont-open-data/">Oxford Nanopore Technologies Benchmark Datasets</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">fast5</span><span class="label label-info tag link-tag">fastq</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">Homo sapiens</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>The ont-open-data registry provides reference sequencing data from Oxford Nanopore Technologies to support, 1) Exploration of the characteristics of nanopore sequence data. 2) Assessment and reproduction of performance benchmarks 3) Development of tools and methods. The data deposited showcases DNA sequences from a representative subset of sequencing chemistries. The datasets correspond to publicly-available reference samples (e.g. GM24385 as reference human). Raw data are provided with metadata and scripts to describe sample and data provenance.</p>
          <p><a href="/ont-open-data/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://nanoporetech.github.io/ont-open-datasets/tutorials" class="" target="_none">ONT Dataset Tutorials</a> by <a href="https://labs.epi2me.io/" target="_none">EPI2MELabs</a></li></a>
          </li>
        </ul>
        <p><a href="/ont-open-data/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="qiime2" class="dataset">
          <h3><a href="/qiime2/">QIIME 2 User Tutorial Datasets</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">denoising</span><span class="label label-info tag link-tag">ecosystems</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">health</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">microbiome</span><span class="label label-info tag link-tag">statistics</span></p>
          <p>QIIME 2 is a powerful, extensible, and decentralized microbiome analysis package with a focus on data and analysis transparency. QIIME 2 enables researchers to start an analysis with raw DNA sequence data and finish with publication-quality figures and statistical results. This dataset contains the user docs (and related datasets) for QIIME 2.</p>
          <p><a href="/qiime2/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://docs.qiime2.org/2018.11/install/virtual/aws/" class="" target="_none">Installing QIIME 2 using Amazon Web Services</a> by <a href="https://github.com/qiime2/docs/graphs/contributors" target="_none">The QIIME 2 Development Team</a></li></a>
          </li>
        </ul>
        <p><a href="/qiime2/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="tabula-muris" class="dataset">
          <h3><a href="/tabula-muris/">Tabula Muris</a></h3>
          <p><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">encyclopedic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">health</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">medicine</span></p>
          <p>Tabula Muris is a compendium of single cell transcriptomic data from the model organism <em>Mus musculus</em> comprising more than 100,000 cells from 20 organs and tissues. These data represent a new resource for cell biology, reveal gene expression in poorly characterized cell populations, and allow for direct and controlled comparison of gene expression in cell types shared between tissues, such as T-lymphocytes and endothelial cells from different anatomical locations. Two distinct technical approaches were used for most organs: one approach, microfluidic droplet-based 3’-end counting, enabled the s...</p>
          <p><a href="/tabula-muris/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/30283141" class="" target="_none">Single-cell transcriptomics of 20 mouse organs creates a Tabula Muris.</a> by <a href="https://tabula-muris.ds.czbiohub.org/" target="_none">Tabula Muris Consortium (2019)</a></li></a>
          </li>
        </ul>
        <p><a href="/tabula-muris/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="zinc15" class="dataset">
          <h3><a href="/zinc15/">ZINC Database</a></h3>
          <p><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">chemical biology</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">molecular docking</span><span class="label label-info tag link-tag">pharmaceutical</span><span class="label label-info tag link-tag">protein</span></p>
          <p>3D models for molecular docking screens.</p>
          <p><a href="/zinc15/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="http://zinc15.docking.org" class="" target="_none">ZINC Database</a> by <a href="http://irwinlab.compbio.ucsf.edu" target="_none">John Irwin</a></li></a>
          </li>
        </ul>
        <p><a href="/zinc15/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="molssi-covid19-hub" class="dataset">
          <h3><a href="/molssi-covid19-hub/">COVID-19 Molecular Structure and Therapeutics Hub</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">coronavirus</span><span class="label label-info tag link-tag">COVID-19</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">molecular docking</span><span class="label label-info tag link-tag">pharmaceutical</span></p>
          <p>Aggregating critical information to accelerate drug discovery for the molecular modeling and simulation community.
A community-driven data repository and curation service for molecular structures, models, therapeutics, and
simulations related to computational research related to therapeutic opportunities for COVID-19
(caused by the SARS-CoV-2 coronavirus).</p>
          <p><a href="/molssi-covid19-hub/">Details &rarr;</a></p>
        </div>
        <div id="genomeark" class="dataset">
          <h3><a href="/genomeark/">Genome Ark</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span></p>
          <p>The Genome Ark hosts genomic information for the Vertebrate Genomes Project (VGP) and other related projects. The VGP is an international collaboration that aims to generate complete and near error-free reference genomes for all extant vertebrate species. These genomes will be used to address fundamental questions in biology and disease, to identify species most genetically at risk for extinction, and to preserve genetic information of life.</p>
          <p><a href="/genomeark/">Details &rarr;</a></p>
        </div>
        <div id="openneuro" class="dataset">
          <h3><a href="/openneuro/">OpenNeuro</a></h3>
          <p><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">imaging</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">neurobiology</span><span class="label label-info tag link-tag">neuroimaging</span></p>
          <p>OpenNeuro is a database of openly-available brain imaging data. The data are shared according to a Creative Commons CC0 license, providing a broad range of brain imaging data to researchers and citizen scientists alike. The database primarily focuses on functional magnetic resonance imaging (fMRI) data, but also includes other imaging modalities including structural and diffusion MRI, electroencephalography (EEG), and magnetoencephalograpy (MEG). OpenfMRI is a project of the <a href="http://reproducibility.stanford.edu">Center for Reproducible Neuroscience at Stanford University</a>. Development of the OpenNeuro resource has been funded by th...</p>
          <p><a href="/openneuro/">Details &rarr;</a></p>
        </div>
        <div id="physionet" class="dataset">
          <h3><a href="/physionet/">Physionet</a></h3>
          <p><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">life sciences</span></p>
          <p>PhysioNet offers free web access to large collections of recorded physiologic signals (PhysioBank) and related open-source software (PhysioToolkit).</p>
          <p><a href="/physionet/">Details &rarr;</a></p>
        </div>
        <div id="elp-nouabale-landscape" class="dataset">
          <h3><a href="/elp-nouabale-landscape/">Sounds of Central African landscapes</a></h3>
          <p><span class="label label-info tag link-tag">biodiversity</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">ecosystems</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">land</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">natural resource</span><span class="label label-info tag link-tag">survey</span></p>
          <p>Archival soundscapes recorded in the rainforest landscapes of
Central Africa, with a focus on the vocalizations of African forest
elephants (Loxodonta cyclotis).</p>
          <p><a href="/elp-nouabale-landscape/">Details &rarr;</a></p>
        </div>
        <div id="tabula-muris-senis" class="dataset">
          <h3><a href="/tabula-muris-senis/">Tabula Muris Senis</a></h3>
          <p><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">encyclopedic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">health</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">medicine</span><span class="label label-info tag link-tag">single-cell transcriptomics</span></p>
          <p>Tabula Muris Senis is a comprehensive compendium of single cell transcriptomic data from the model organism <em>Mus musculus</em> comprising more than 500,000 cells from 18 organs and tissues across the mouse lifespan. We discovered cell-specific changes occurring across multiple cell types and organs, as well as age related changes in the cellular composition of different organs. Using single-cell transcriptomic data we were able to assess cell type specific manifestations of different hallmarks of aging, such as senescence, changes in the activity of metabolic pathways, depletion of stem-cell populat...</p>
          <p><a href="/tabula-muris-senis/">Details &rarr;</a></p>
        </div>
        <div id="ubc-sunflower-genome" class="dataset">
          <h3><a href="/ubc-sunflower-genome/">University of British Columbia Sunflower Genome Dataset</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">biodiversity</span><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">food security</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>This dataset captures Sunflower&#39;s genetic diversity originating
from thousands of wild, cultivated, and landrace sunflower
individuals distributed across North America.The data consists of raw sequences and associated botanical metadata,
aligned sequences (to three different reference genomes), and sets of
SNPs computed across several cohorts.</p>
          <p><a href="/ubc-sunflower-genome/">Details &rarr;</a></p>
        </div>
      </div>
    </div>


    <hr/>
  </div>  </body>
  <script src="/js/index.js"></script>
</html>
