<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    
  
    
  
    <link rel="alternate" type="application/rss+xml" href="/rss.xml" />
    
    <link rel="shortcut icon" type="image/ico" href="https://a0.awsstatic.com/libra-css/images/site/fav/favicon.ico"/>
    <link rel="apple-touch-icon" sizes="57x57" href="https://a0.awsstatic.com/libra-css/images/site/touch-icon-iphone-114-smile.png"/>
    <link rel="apple-touch-icon" sizes="72x72" href="https://a0.awsstatic.com/libra-css/images/site/touch-icon-ipad-144-smile.png"/>
    <link rel="apple-touch-icon" sizes="114x114" href="https://a0.awsstatic.com/libra-css/images/site/touch-icon-iphone-114-smile.png"/>
    <link rel="apple-touch-icon" sizes="144x144" href="https://a0.awsstatic.com/libra-css/images/site/touch-icon-ipad-144-smile.png"/>  
  
    <title>Registry of Open Data on AWS</title>
  
    <!-- Latest compiled and minified CSS -->
    <link rel="stylesheet" href="https://assets.opendata.aws/css/bootstrap/3.4.1/bootstrap.min.css">
  
    <!-- Our local CSS -->
    <link rel="stylesheet" href="/css/main.css">
  
  
  </head>
  <body>
    <!-- jQuery (necessary for Bootstrap's JavaScript plugins) -->
    <script src="https://assets.opendata.aws/js/jquery/3.5.1/jquery.min.js"></script>
    <!-- Latest compiled and minified JavaScript -->
    <script src="https://assets.opendata.aws/js/bootstrap/3.4.1/bootstrap.min.js"></script>
    <nav class="navbar navbar-default navbar-fixed-top">
      <div class="container">
        <div class="roda-header">
          <h2><a href="/" alt="Home">Registry of Open Data on AWS</a></h2>
          <a href="http://aws.amazon.com/what-is-cloud-computing">
            <img src="https://assets.opendata.aws/img/AWS-Logo_White-Color_300x180.png" alt="Powered by AWS Cloud Computing" id="aws_header_logo">
          </a>
        </div>
      </div>
    </nav>
    <div class="container" >

    <div class="about col-md-5">
      <div class="row aboutbox">
            <h3>About</h3>
            <p>This registry exists to help people discover and share datasets that are available via AWS resources. <a href="https://opendata.aws">Learn more about sharing data on AWS</a>.</p>
    
            <p>See <a href="/tag/cancer/usage-examples">all usage examples for datasets listed in this registry</a> tagged with <strong>cancer</strong>.</p>
    
        <hr>
    
        <h4>Search datasets (currently <span id="count-matching">13</span> matching <span id="count-matching-text">datasets</span>)</h4>
        <form>
          <div class="form-group">
            <input type="text" id="search-box" class="form-control" placeholder="Search datasets" spellcheck="false" autocorrect="off">
          </div>
        </form>
        <p>You are currently viewing a subset of data tagged with <strong>cancer</strong>.</p>
    
        <hr>
    
        <h4>Add to this registry</h4>
        <p>If you want to add a dataset or example of how to use a dataset to this registry, please follow the instructions on the <a href="https://github.com/awslabs/open-data-registry/">Registry of Open Data on AWS GitHub repository</a>.</p>
    
        <p>Unless specifically stated in the applicable dataset documentation, datasets available through the Registry of Open Data on AWS are not provided and maintained by AWS. Datasets are provided and maintained by a variety of third parties under a variety of licenses. Please check dataset licenses and related documentation to determine if a dataset may be used for your application.</p>
      </div>
    </div>
    <div class="col-md-6 col-md-offset-1 datasets">
      <div class="row">
        <div id="tcga" class="dataset">
          <h3><a href="/tcga/">The Cancer Genome Atlas</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">STRIDES</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>The Cancer Genome Atlas (TCGA), a collaboration between the National Cancer Institute (NCI) and National Human Genome Research Institute (NHGRI), aims to generate comprehensive, multi-dimensional maps of the key genomic changes in major types and subtypes of cancer. TCGA has analyzed matched tumor and normal tissues from 11,000 patients, allowing for the comprehensive characterization of 33 cancer types and subtypes, including 10 rare cancers.
The dataset contains open Clinical Supplement, Biospecimen Supplement, RNA-Seq Gene Expression Quantification, miRNA-Seq Isoform Expression Quantificati...</p>
          <p><a href="/tcga/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1016/j.celrep.2018.03.046" class="" target="_none">Machine Learning Detects Pan-Cancer Ras Pathway Activation in The Cancer Genome Atlas</a> by Gregory P. Way, Francisco Sanchez-Vega, et al.</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1016/j.cell.2018.03.034" class="" target="_none">Machine Learning Identifies Stemness Features Associated with Oncogenic Dedifferentiation
</a> by Tathiane M. Malta, Artem Sokolov, et al.</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1016/j.immuni.2018.03.023" class="" target="_none">The Immune Landscape of Cancer</a> by Vésteinn Thorsson, David L. Gibbs, et al.</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1016/j.celrep.2018.03.047" class="" target="_none">Integrated Genomic Analysis of the Ubiquitin Pathway across Cancer Types</a> by Zhongqi Ge, Jake S. Leighton, et al.</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1016/j.cels.2019.06.006" class="" target="_none">&quot;Before and After: A Comparison of Legacy and Harmonized TCGA Data at the Genomic Data
Commons&quot;
</a> by Galen F. Gao, Joel S. Parker, et al.</li></a>
          </li>
        </ul>
        <p><a href="/tcga/#usageexamples">See 29 usage examples &rarr;</a></p>
        </div>
        <div id="target" class="dataset">
          <h3><a href="/target/">Therapeutically Applicable Research to Generate Effective Treatments (TARGET)</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">STRIDES</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>Therapeutically Applicable Research to Generate Effective Treatments (TARGET) is the collaborative effort of a large, diverse consortium of extramural and NCI investigators. The goal of the effort is to accelerate molecular discoveries that drive the initiation and progression of hard-to-treat childhood cancers and facilitate rapid translation of those findings into the clinic.
TARGET projects provide comprehensive molecular characterization to determine the genetic changes that drive the initiation and progression of childhood cancers.The dataset contains open Clinical Supplement, Biospecimen...</p>
          <p><a href="/target/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term&#x3D;28671688" class="" target="_none">The genomic landscape of pediatric and young adult T-lineage acute lymphoblastic leukemia
</a> by Yu Liu, John Easton, Ying Shao, et al.</li></a>
          </li>
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term&#x3D;29068783" class="" target="_none">MicroRNA Expression-Based Model Indicates Event-Free Survival in Pediatric Acute Myeloid
Leukemia
</a> by Lim EL, Trinh DL, Ries RE, et al.</li></a>
          </li>
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term&#x3D;27143256" class="" target="_none">CSF3R mutations have a high degree of overlap with CEBPA mutations in pediatric AM</a> by Maxson JE, Ries RE, Wang YC, et al.</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1038/ng.3940" class="" target="_none">A Children&#x27;s Oncology Group and TARGET initiative exploring the genetic landscape of Wilms
tumor
</a> by Gadd S, Huff V, Walz AL, et al.</li></a>
          </li>
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term&#x3D;29228567" class="" target="_none">Biomarker significance of plasma and tumor miR-21, miR-221, and miR-106a in osteosarcoma
</a> by Nakka M, Allen-Rhoades W, Li Y, et al.</li></a>
          </li>
        </ul>
        <p><a href="/target/#usageexamples">See 24 usage examples &rarr;</a></p>
        </div>
        <div id="kids-first" class="dataset">
          <h3><a href="/kids-first/">Gabriella Miller Kids First Pediatric Research Program (Kids First)</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">Homo sapiens</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">pediatric</span><span class="label label-info tag link-tag">STRIDES</span><span class="label label-info tag link-tag">structural birth defect</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>The NIH Common Fund&#39;s Gabriella Miller Kids First Pediatric Research Program’s (“Kids First”) vision is to “alleviate suffering from childhood cancer and structural birth defects by fostering collaborative research to uncover the etiology of these diseases and by supporting data sharing within the pediatric research community.” The program continues to generate and share whole genome sequence data from thousands of children affected by these conditions, ranging from rare pediatric cancers, such as osteosarcoma, to more prevalent diagnoses, such as congenital heart defects. In 2018, Kids Fi...</p>
          <p><a href="/kids-first/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/kids-first/" class="" target="_none">Kids First DRC Source Code</a> by <a href="https://kidsfirstdrc.org" target="_none">Kids First DRC</a></li></a>
          </li>
          <li>
            <a href="https://pubmed.ncbi.nlm.nih.gov/30322880/" class="" target="_none">Clinically Relevant and Minimally Invasive Tumor Surveillance of Pediatric Diffuse Midline Gliomas Using Patient-Derived Liquid Biopsy</a> by Eshini Panditharatna, Lindsay B Kilburn, et al.</li></a>
          </li>
          <li>
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/31361404" class="" target="_none">Phenotype delineation of ZNF462 related syndrome.</a> by Paul Kruszka, Tommy Hu, et al.</li></a>
          </li>
          <li>
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/30930166" class="" target="_none">The Pediatric Cell Atlas: Defining the Growth Phase of Human Development at Single-Cell Resolution.</a> by Deanne M Taylor, Bruce J Aronow, et al.</li></a>
          </li>
          <li>
            <a href="https://pubmed.ncbi.nlm.nih.gov/32601476/" class="" target="_none">Genomic Analyses Implicate Noncoding De Novo Variants in Congenital Heart Disease.</a> by Felix Richter, Sarah U Morton, et al.</li></a>
          </li>
        </ul>
        <p><a href="/kids-first/#usageexamples">See 19 usage examples &rarr;</a></p>
        </div>
        <div id="ccle" class="dataset">
          <h3><a href="/ccle/">Cancer Cell Line Encyclopedia (CCLE)</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">Homo sapiens</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">STRIDES</span><span class="label label-info tag link-tag">transcriptomics</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>The Cancer Cell Line Encyclopedia (CCLE) project is an effort to conduct a detailed genetic
characterization of a large panel of human cancer cell lines. The CCLE provides public access to
genomic data, visualization and analysis for over 1100 cancer cell lines. This dataset contains
RNA-Seq Aligned Reads, WXS Aligned Reads, and WGS Aligned Reads data.</p>
          <p><a href="/ccle/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://cgc-accounts.sbgenomics.com/" class="" target="_none">Cancer Genomics Cloud</a> by <a href="https://docs.sevenbridges.com/docs/ccle-data" target="_none">Seven Bridges</a></li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1038/nature11003" class="" target="_none">The Cancer Cell Line Encyclopedia enables predictive modelling of anticancer drug
sensitivity
</a> by Barretina Caponigro Stransky et al.</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1038/nature15736" class="" target="_none">Pharmacogenomic agreement between two cancer cell line data sets</a> by The Cancer Cell Line Encyclopedia Consortium &amp; The Genomics of Drug Sensitivity in Cancer
Consortium
</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1038/s41586-019-1186-3" class="" target="_none">Next-generation characterization of the Cancer Cell Line Encyclopedia</a> by Ghandi, M., Huang F. et al.</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1038/s41591-019-0404-8" class="" target="_none">The landscape of cancer cell line metabolism</a> by Li, H. et al.</li></a>
          </li>
        </ul>
        <p><a href="/ccle/#usageexamples">See 8 usage examples &rarr;</a></p>
        </div>
        <div id="cptac-2" class="dataset">
          <h3><a href="/cptac-2/">Clinical Proteomic Tumor Analysis Consortium 2 (CPTAC-2)</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">STRIDES</span><span class="label label-info tag link-tag">transcriptomics</span></p>
          <p>The Clinical Proteomic Tumor Analysis Consortium (CPTAC) is a national effort to accelerate the
understanding of the molecular basis of cancer through the application of large-scale proteome and
genome analysis, or proteogenomics. CPTAC-2 is the Phase II of the CPTAC Initiative (2011-2016).
Datasets contain open RNA-Seq Gene Expression Quantification, miRNA-Seq Isoform Expression
Quantification, and miRNA Expression Quantification data.</p>
          <p><a href="/cptac-2/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://proteomics.cancer.gov/data-portal" class="" target="_none">CPTAC Data Portal</a> by <a href="https://proteomics.cancer.gov/" target="_none">National Cancer Institute</a></li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1038/sdata.2015.22" class="" target="_none">Proteomic analysis of colon and rectal carcinoma using standard and customized databases
</a> by Slebos RJ, Wang X, Wang X, Zhang B, Tabb DL, Liebler DC</li></a>
          </li>
          <li>
            <a href="https://cgc-accounts.sbgenomics.com/" class="" target="_none">Cancer Genomics Cloud</a> by <a href="http://www.cancergenomicscloud.org/" target="_none">Seven Bridges</a></li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1016/j.cell.2016.05.069" class="" target="_none">Integrated Proteogenomic Characterization of Human High-Grade Serous Ovarian Cancer</a> by Hui Zhang, Tao Liu, Zhen Zhang, Samuel H. Payne, Bai Zhang, Jason E. McDermott, Jian-Ying
Zhou, Vladislav A. Petyuk, Li Chen, Debjit Ray, Shisheng Sun, Feng Yang, Lijun Chen, Jing
Wang, Punit Shah, Seong Won Cha, Paul Aiyetan, Sunghee Woo, Yuan Tian, Marina A. Gritsenko,
Therese R. Clauss, Caitlin Choi, Matthew E. Monroe, Stefani Thomas, Song Nie, Chaochao Wu,
Ronald J. Moore, Kun-Hsing Yu, David L. Tabb, David Fenyö, Vineet Bafna, Yue Wang, Henry
Rodriguez, Emily S. Boja, Tara Hiltke, Robert C. Rivers, Lori Sokoll, Heng Zhu, Ie-Ming
Shih, Leslie Cope, Akhilesh Pandey, Bing Zhang, Michael P. Snyder, Douglas A. Levine,
Richard D. Smith, Daniel W. Chan, Karin D. Rodland, the CPTAC Investigators
</li></a>
          </li>
          <li>
            <a href="https://pdc.esacinc.com/pdc/browse" class="" target="_none">Proteomic Data Commons</a> by <a href="https://pdc.esacinc.com/pdc" target="_none">National Cancer Institute</a></li></a>
          </li>
        </ul>
        <p><a href="/cptac-2/#usageexamples">See 7 usage examples &rarr;</a></p>
        </div>
        <div id="icgc" class="dataset">
          <h3><a href="/icgc/">ICGC on AWS</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span></p>
          <p>The International Cancer Genome Consortium (ICGC) coordinates projects with the common aim of accelerating research into the causes and control of cancer. The PanCancer Analysis of Whole Genomes (PCAWG) study is an international collaboration to identify common patterns of mutation in whole genomes from ICGC. More than 2,400 consistently analyzed genomes corresponding to over 1,100 unique ICGC donors are now freely available on Amazon S3 to credentialed researchers subject to ICGC data sharing policies.</p>
          <p><a href="/icgc/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.nature.com/articles/s41586-020-1969-6" class="" target="_none">Pan-cancer analysis of whole genomes</a> by <a href="https://dcc.icgc.org/pcawg" target="_none">The ICGC/TCGA Pan-Cancer Analysis of Whole Genomes Consortium (2020)</a></li></a>
          </li>
          <li>
            <a href="https://github.com/ICGC-TCGA-PanCancer/cli/wiki" class="" target="_none">AWS Pancancer Workflow Launcher</a> by <a href="https://github.com/solomonshorser" target="_none">Solomon Shorser</a></li></a>
          </li>
          <li>
            <a href="https://www.nature.com/articles/s41586-020-1965-x" class="" target="_none">Analyses of non-coding somatic drivers in 2,693 cancer whole genomes</a> by <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term&#x3D;Rheinbay%20E%5BAuthor%5D&amp;cauthor&#x3D;true&amp;cauthor_uid&#x3D;28658208" target="_none">Rheinbay et al (2020)</a></li></a>
          </li>
          <li>
            <a href="https://www.nature.com/articles/s41586-020-1970-0" class="" target="_none">Genomic basis of RNA alterations in cancer</a> by <a href="https://icgc.org/committees-and-working-groups#71954" target="_none">PCAWG Transcriptome Core Group et al. (2020)</a></li></a>
          </li>
          <li>
            <a href="https://www.nature.com/articles/s41586-019-1907-7" class="" target="_none">The evolutionary history of 2,658 cancers</a> by <a href="https://www.ncbi.nlm.nih.gov/pubmed?cmd&#x3D;search&amp;term&#x3D;Moritz%20Gerstung" target="_none">Gerstung et al (2020)</a></li></a>
          </li>
        </ul>
        <p><a href="/icgc/#usageexamples">See 7 usage examples &rarr;</a></p>
        </div>
        <div id="cptac-3" class="dataset">
          <h3><a href="/cptac-3/">Clinical Proteomic Tumor Analysis Consortium 3 (CPTAC-3)</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">STRIDES</span><span class="label label-info tag link-tag">transcriptomics</span></p>
          <p>The Clinical Proteomic Tumor Analysis Consortium (CPTAC) is a national effort to accelerate the
understanding of the molecular basis of cancer through the application of large-scale proteome and
genome analysis, or proteogenomics. CPTAC-3 is the Phase III of the CPTAC Initiative. The dataset
contains open RNA-Seq Gene Expression Quantification data.</p>
          <p><a href="/cptac-3/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1016/j.cell.2019.10.007" class="" target="_none">Integrated Proteogenomic Characterization of Clear Cell Renal Cell Carcinoma</a> by Clark DJ, Dhanasekaran SM, Petralia F, Pan J, Song X, Hu Y, da Veiga Leprevost F, Reva B,
Lih TM, Chang HY, Ma W, Huang C, Ricketts CJ, Chen L1, Krek A, Li Y, Rykunov D, Li QK, Chen
LS, Ozbek U, Vasaikar S, Wu Y, Yoo S, Chowdhury S, Wyczalkowski MA, Ji J, Schnaubelt M, Kong
A, Sethuraman S, Avtonomov DM, Ao M, Colaprico A, Cao S, Cho KC, Kalayci S, Ma S, Liu W,
Ruggles K, Calinawan A, Gümüş ZH, Geizler D, Kawaler E, Teo GC, Wen B, Zhang Y, Keegan S, Li
K, Chen F, Edwards N, Pierorazio PM, Chen XS, Pavlovich CP, Hakimi AA, Brominski G, Hsieh
JJ, Antczak A, Omelchenko T, Lubinski J, Wiznerowicz M, Linehan WM, Kinsinger CR,
Thiagarajan M, Boja ES, Mesri M, Hiltke T, Robles AI, Rodriguez H, Qian J, Fenyö D, Zhang B,
Ding L, Schadt E, Chinnaiyan AM, Zhang Z, Omenn GS, Cieslik M, Chan DW, Nesvizhskii AI, Wang
P, Zhang H; Clinical Proteomic Tumor Analysis Consortium
</li></a>
          </li>
          <li>
            <a href="https://cgc-accounts.sbgenomics.com/" class="" target="_none">Cancer Genomics Cloud</a> by <a href="http://www.cancergenomicscloud.org/" target="_none">Seven Bridges</a></li></a>
          </li>
          <li>
            <a href="https://pdc.esacinc.com/pdc/browse" class="" target="_none">Proteomic Data Commons</a> by <a href="https://pdc.esacinc.com/pdc" target="_none">National Cancer Institute</a></li></a>
          </li>
          <li>
            <a href="https://portal.gdc.cancer.gov/projects/CPTAC-3" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></li></a>
          </li>
          <li>
            <a href="https://proteomics.cancer.gov/data-portal" class="" target="_none">CPTAC Data Portal</a> by <a href="https://proteomics.cancer.gov/" target="_none">National Cancer Institute</a></li></a>
          </li>
        </ul>
        <p><a href="/cptac-3/#usageexamples">See 6 usage examples &rarr;</a></p>
        </div>
        <div id="mmrf-commpass" class="dataset">
          <h3><a href="/mmrf-commpass/">CoMMpass from the Multiple Myeloma Research Foundation</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">STRIDES</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>The Relating Clinical Outcomes in Multiple Myeloma to Personal Assessment of Genetic Profile study is the Multiple Myeloma Research Foundation (MMRF)’s landmark personalized medicine initiative.  CoMMpass is a
longitudinal observation study of around 1000 newly diagnosed myeloma patients receiving various
standard approved treatments. The MMRF’s vision is to track the treatment and results for each
CoMMpass patient so that someday the information can be used to guide decisions for newly
diagnosed patients. CoMMpass checked on patients every 6 months for 8 years, collecting tissue
samples, gene...</p>
          <p><a href="/mmrf-commpass/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1182/blood.V126.23.722.722" class="" target="_none">&quot;Identification of Initiating Trunk Mutations and Distinct Molecular Subtypes: An Interim
Analysis of the Mmrf Commpass Study&quot;
</a> by Jonathan J Keats, PhD, Gil Speyer, Legendre Christophe, Christofferson Austin, Kristi
Stephenson, BS, Ahmet Kurdoglu, Megan Russell, Aldrich Jessica, Cuyugan Lori, Jonathan
Adkins, Jackie McDonald, Adrienne Helland, Alex Blanski, Meghan Hodges, Dan Rohrer, Sundar
Jagannath, MD, David Siegel, MD PhD, Ravi Vij, MD MBA, Gregory Orloff, MD, Todd Zimmerman,
MD, Ruben Niesvizky, MD, Darla Liles, MD, Joseph W. Fay, Jeffrey L. Wolf, MD PhD, Robert M.
Rifkin, Norma C Gutierrez, The MMRF CoMMpass Network, Jen Toups, Mary Derome, MS, Winnie
Liang, PhD, Seunchan Kim, Daniel Auclair, PhD, Pamela G. Kidd, MD, Scott Jewell, PhD, John
David Carpten, PhD, Sagar Lonial, MD
</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1182/blood.V122.21.532.532" class="" target="_none">&quot;Interim Analysis Of The MMRF CoMMpass Trial: a Longitudinal Study In Multiple Myeloma
Relating Clinical Outcomes To Genomic and Immunophenotypic Profiles&quot;
</a> by Keats JJ, Craig DW, Liang W, Venkata Y, Kurdoglu A, Aldrich J, Auclair D, Allen K, Harrison
B, Jewell S, Kidd PG, Correll M, Jagannath S, Siegel DS, Vij R, Orloff G, Zimmerman TM, MMRF
CoMMpass Network, Capone W, Carpten J, Lonial S.
</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1182/blood.V128.22.194.194" class="" target="_none">&quot;Molecular Predictors of Outcome and Drug Response in Multiple Myeloma: An Interim Analysis
of the Mmrf CoMMpass Study&quot;
</a> by Jonathan J Keats, PhD, Gil Speyer, Austin Christofferson, Christophe Legendre, PhD, Jessica
Aldrich, Megan Russell, Lori Cuyugan, Jonathan Adkins, Alex Blanski, Meghan Hodges, Dan
Rohrer, Sundar Jagannath, MD, Ravi Vij, MD, Gregory Orloff, MD, Todd Zimmerman, MD, Ruben
Niesvizky, MD, Darla Liles, MD, Joseph W. Fay, Jeffrey L. Wolf, MD, Robert M Rifkin, Norma C
Gutierrez, MD PhD, Mmrf CoMMpass Network, Jennifer Yesil, MS, Mary Derome, MS, Seungchan
Kim, PhD, Winnie Liang, PhD, Pamela G. Kidd, MD, Scott Jewell, PhD, John David Carpten, PhD,
Daniel Auclair, PhD, Sagar Lonial, MD FACP
</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1182/blood.V124.21.722.722" class="" target="_none">&quot;Interim Analysis of the Mmrf Commpass Trial: Identification of Novel Rearrangements
Potentially Associated with Disease Initiation and Progression&quot;
</a> by Sagar Lonial, MD, Venkata D Yellapantula, Winnie Liang, PhD, Ahmet Kurdoglu, BS, Jessica
Aldrich, MSc, Christophe M. Legendre, MD, Kristi Stephenson, Jonathan Adkins, Jackie
McDonald, Adrienne Helland, Megan Russell, Austin Christofferson, Lori Cuyugan, Dan Rohrer,
Alex Blanski, Meghan Hodges, Mmrf CoMMpass Network, Mary Derome, Daniel Auclair, PhD, Pamela
G. Kidd, MD, Scott Jewell, PhD, David Craig, PhD, John Carpten, PhD, Jonathan J. Keats, PhD
</li></a>
          </li>
          <li>
            <a href="https://portal.gdc.cancer.gov/projects/MMRF-COMMPASS" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></li></a>
          </li>
        </ul>
        <p><a href="/mmrf-commpass/#usageexamples">See 5 usage examples &rarr;</a></p>
        </div>
        <div id="beataml" class="dataset">
          <h3><a href="/beataml/">Beat Acute Myeloid Leukemia (AML) 1.0</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">Homo sapiens</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">STRIDES</span></p>
          <p>Beat AML 1.0 is a collaborative research program involving 11 academic medical centers who worked
collectively to better understand  drugs and drug combinations that should be prioritized for
further development within clinical and/or molecular subsets of acute myeloid leukemia (AML)
patients. Beat AML 1.0 provides the largest-to-date dataset on primary acute myeloid leukemia
samples offering genomic, clinical, and drug response.This dataset contains open Clinical Supplement and RNA-Seq Gene Expression Quantification data.This dataset also contains controlled Whole Exome Sequencing (WXS) and R...</p>
          <p><a href="/beataml/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/30651561" class="" target="_none">Clinical resistance to crenolanib in acute myeloid leukemia due to diverse molecular
mechanisms
</a> by Zhang H, Savage S, Schultz AR, Bottomly D, White L, Segerdell E, et al.</li></a>
          </li>
          <li>
            <a href="https://portal.gdc.cancer.gov/projects/" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></li></a>
          </li>
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/30333627" class="" target="_none">Functional Genomic Landscape of Acute Myeloid Leukemia</a> by Jeffrey W. Tyner, Cristina E. Tognon, Dan Bottomly et al.</li></a>
          </li>
        </ul>
        <p><a href="/beataml/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="ctsp-dlbcl" class="dataset">
          <h3><a href="/ctsp-dlbcl/">Clinical Trial Sequencing Project - Diffuse Large B-Cell Lymphoma</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">STRIDES</span><span class="label label-info tag link-tag">transcriptomics</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>The goal of the project is to identify recurrent genetic alterations (mutations, deletions,
amplifications, rearrangements) and/or gene expression signatures. National Cancer Institute (NCI)
utilized whole genome sequencing and/or whole exome sequencing in conjunction with transcriptome
sequencing. The samples were processed and submitted for genomic characterization using pipelines
and procedures established within The Cancer Genome Analysis (TCGA) project.</p>
          <p><a href="/ctsp-dlbcl/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1056/NEJMoa1801445" class="" target="_none">Genetics and Pathogenesis of Diffuse Large B Cell Lymphoma</a> by Roland Schmitz, Ph.D., George W. Wright, Ph.D., Da Wei Huang, M.D., Calvin A.  Johnson,
Ph.D., James D. Phelan, Ph.D., James Q. Wang, Ph.D., Sandrine Roulland, Ph.D., Monica
Kasbekar, Ph.D., Ryan M. Young, Ph.D., Arthur L. Shaffer, Ph.D., Daniel J. Hodson, M.D.,
Ph.D., Wenming Xiao, Ph.D., et al.
</li></a>
          </li>
          <li>
            <a href="https://portal.gdc.cancer.gov/projects/CTSP-DLBCL1" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></li></a>
          </li>
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed?cmd&#x3D;DetailsSearch&amp;term&#x3D;29925955[PMID]" class="" target="_none">A multiprotein supercomplex controlling oncogenic signalling in lymphoma</a> by Phelan JD, Young RM, Webster DE, Roulland S, Wright GW, Kasbekar M, Shaffer AL 3rd,
Ceribelli M, Wang JQ, Schmitz R, Nakagawa M, Bachy E, Huang DW, Ji Y, Chen L, Yang Y, Zhao
H, Yu X, Xu W, Palisoc MM, Valadez RR, Davies-Hill T, Wilson WH, Chan WC, Jaffe ES, Gascoyne
RD, Campo E, Rosenwald A, Ott G, Delabie J, Rimsza LM, Rodriguez FJ, Estephan F, Holdhoff M,
Kruhlak MJ, Hewitt SM, Thomas CJ, Pittaluga S, Oellerich T, Staudt LM
</li></a>
          </li>
        </ul>
        <p><a href="/ctsp-dlbcl/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="fm-ad" class="dataset">
          <h3><a href="/fm-ad/">Foundation Medicine Adult Cancer Clinical Dataset (FM-AD)</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genomic</span></p>
          <p>The Foundation Medicine Adult Cancer Clinical Dataset (FM-AD) is a study conducted by Foundation
Medicine Inc (FMI). Genomic profiling data for approximately 18,000 adult patients with a diverse
array of cancers was generated using FoundationeOne, FMI&#39;s commercially available, comprehensive
genomic profiling assay. This dataset contains open Clinical and Biospecimen data.</p>
          <p><a href="/fm-ad/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://portal.gdc.cancer.gov/projects/FM-AD" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></li></a>
          </li>
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed?cmd&#x3D;DetailsSearch&amp;term&#x3D;28235761[PMID]" class="" target="_none">High-Throughput Genomic Profiling of Adult Solid Tumors Reveals Novel Insights into Cancer
Pathogenesis
</a> by Ryan J. Hartmaier, Lee A. Albacker, Juliann Chmielecki, Mark Bailey, Jie He, Michael E.
Goldberg, Shakti Ramkissoon, James Suh, Julia A. Elvin, Samuel Chiacchia, Garrett M.
Frampton, Jeffrey S. Ross, Vincent Miller, Philip J. Stephens and Doron Lipson
</li></a>
          </li>
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed?cmd&#x3D;DetailsSearch&amp;term&#x3D;22981675[PMID]" class="" target="_none">Targeted next-generation sequencing of advanced prostate cancer identifies potential
therapeutic targets and disease heterogeneity.
</a> by Beltran H, Yelensky R, Frampton GM, Park K, Downing SR, MacDonald TY, Jarosz M, Lipson D,
Tagawa ST, Nanus DM, Stephens PJ, Mosquera JM, Cronin MT, Rubin MA
</li></a>
          </li>
        </ul>
        <p><a href="/fm-ad/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="broad-references" class="dataset">
          <h3><a href="/broad-references/">Broad Genome References</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">Homo sapiens</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">reference index</span></p>
          <p>Broad maintained human genome reference builds hg19/hg38 and decoy references.</p>
          <p><a href="/broad-references/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://aws.amazon.com/blogs/storage/using-amazon-fsx-for-lustre-for-genomics-workflows-on-aws/" class="aws-link" target="_none">Using Amazon FSx for Lustre for Genomics Workflows on AWS</a> by <a href="https://www.linkedin.com/in/lee-pang-a039a26/" target="_none">W. Lee Pang</a></li></a>
          </li>
          <li>
            <a href="https://docs.opendata.aws/genomics-workflows/orchestration/cromwell/cromwell-examples/#real-world-example-haplotypecaller" class="aws-link" target="_none">Genomics Workflows on AWS - Cromwell on AWS</a> by <a href="https://www.linkedin.com/in/lee-pang-a039a26/" target="_none">W. Lee Pang</a></li></a>
          </li>
        </ul>
        <p><a href="/broad-references/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="cgci" class="dataset">
          <h3><a href="/cgci/">Cancer Genome Characterization Initiatives - Burkitt Lymphoma, HIV+ Cervical Cancer</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">STRIDES</span><span class="label label-info tag link-tag">transcriptomics</span></p>
          <p>The Cancer Genome Characterization Initiatives (CGCI) program supports cutting-edge genomics research of adult and pediatric cancers. CGCI investigators develop and apply advanced sequencing methods that examine genomes, exomes, and transcriptomes within various types of tumors. The program includes Burkitt Lymphoma Genome Sequencing Project (BLGSP) project and HIV+ Tumor Molecular Characterization Project - Cervical Cancer (HTMCP-CC) project.
The dataset contains open Clinical Supplement, Biospecimen Supplement, RNA-Seq Gene Expression Quantification, miRNA-Seq Isoform Expression Quantificati...</p>
          <p><a href="/cgci/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1182/blood-2018-09-871418" class="" target="_none">Genome-wide discovery of somatic coding and noncoding mutations in pediatric endemic and
sporadic Burkitt lymphoma
</a> by Grande B. M., Gerhard D. S., Jiang A., Griner N. B., Abramson J. S., Alexander T. B., et al.
</li></a>
          </li>
          <li>
            <a href="https://portal.gdc.cancer.gov/projects/CGCI-BLGSP" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></li></a>
          </li>
        </ul>
        <p><a href="/cgci/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="nciccr-dlbcl" class="dataset">
          <h3><a href="/nciccr-dlbcl/">National Cancer Institute Center for Cancer Research - Diffuse Large B Cell Lymphoma (DLBCL) Genomics and Expression</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genomic</span></p>
          <p>The study describes integrative analysis of genetic lesions in 574 diffuse large B cell lymphomas
(DLBCL) involving exome and transcriptome sequencing, array-based DNA copy number analysis and
targeted amplicon resequencing. The dataset contains open RNA-Seq Gene Expression Quantification
data.</p>
          <p><a href="/nciccr-dlbcl/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1056/NEJMoa1801445" class="" target="_none">Genetics and Pathogenesis of Diffuse Large B Cell Lymphoma</a> by Roland Schmitz, Ph.D., George W. Wright, Ph.D., Da Wei Huang, M.D., et al.</li></a>
          </li>
          <li>
            <a href="https://portal.gdc.cancer.gov/projects/NCICCR-DLBCL" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></li></a>
          </li>
        </ul>
        <p><a href="/nciccr-dlbcl/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="ohsu-cnl" class="dataset">
          <h3><a href="/ohsu-cnl/">Oregon Health &amp; Science University Chronic Neutrophilic Leukemia Dataset</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span></p>
          <p>The OHSU-CNL study offers the whole exome and RNA-sequencing on a cohort of 100 cases with rare
hematologic malignancies such as Chronic neutrophilic leukemia (CNL), atypical chronic myeloid
leukemia (aCML), and unclassified myelodysplastic syndrome/myeloproliferative neoplasms
(MDS/MPN-U). This dataset contains open RNA-Seq Gene Expression Quantification data.</p>
          <p><a href="/ohsu-cnl/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://portal.gdc.cancer.gov/projects/" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></li></a>
          </li>
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/31366621" class="" target="_none">Genomic landscape of neutrophilic leukemias of ambiguous diagnosis</a> by Zhang H, Wilmot B, Bottomly D et al.</li></a>
          </li>
        </ul>
        <p><a href="/ohsu-cnl/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="organoid-pancreatic" class="dataset">
          <h3><a href="/organoid-pancreatic/">Pancreatic Cancer Organoid Profiling</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">STRIDES</span><span class="label label-info tag link-tag">transcriptomics</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>This study generated a collection of patient-derived pancreatic normal and cancer organoids and it was sequenced using Whole Genome Sequencing (WGS), Whole Exome Sequencing (WXS) and RNA-Seq as well as matched tumor and normal tissue if available. The study provides a valuable resource for pancreatic cancer researchers.
The dataset contains open RNA-Seq Gene Expression Quantification data and controlled WGS/WXS/RNA-Seq Aligned Reads, WXS Annotated Somatic Mutation, WXS Raw Somatic Mutation, and RNA-Seq Splice Junction Quantification.</p>
          <p><a href="/organoid-pancreatic/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed?cmd&#x3D;DetailsSearch&amp;term&#x3D;29853643[PMID]" class="" target="_none">Organoid Profiling Identifies Common Responders to Chemotherapy in Pancreatic Cancer</a> by Tiriac H, Belleau P, Engle DD, Plenker D, Deschênes A, Somerville TD, et al.</li></a>
          </li>
          <li>
            <a href="https://portal.gdc.cancer.gov/projects/ORGANOID-PANCREATIC" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></li></a>
          </li>
        </ul>
        <p><a href="/organoid-pancreatic/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="gatk-test-data" class="dataset">
          <h3><a href="/gatk-test-data/">GATK Test Data</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span></p>
          <p>The GATK test data resource bundle is a collection of files for resequencing human genomic data with the
Broad Institute&#39;s <a href="https://software.broadinstitute.org/gatk/">Genome Analysis Toolkit (GATK)</a>.</p>
          <p><a href="/gatk-test-data/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://docs.opendata.aws/genomics-workflows/orchestration/cromwell/cromwell-examples/#real-world-example-haplotypecaller" class="aws-link" target="_none">Genomics Workflows on AWS - Cromwell on AWS</a> by <a href="https://www.linkedin.com/in/lee-pang-a039a26/" target="_none">W. Lee Pang</a></li></a>
          </li>
        </ul>
        <p><a href="/gatk-test-data/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="hcmi-cmdc" class="dataset">
          <h3><a href="/hcmi-cmdc/">Human Cancer Models Initiative (HCMI) Cancer Model Development Center</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">STRIDES</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>The Human Cancer Models Initiative (HCMI) is an international consortium that is generating novel,
next-generation, tumor-derived culture models annotated with genomic and clinical data.
HCMI-developed models and related data are available as a community resource. The NCI is
contributing to the initiative by supporting four Cancer Model Development Centers (CMDCs).  CMDCs
are tasked with producing next-generation cancer models from clinical samples. The cancer models
include tumor types that are rare, originate from patients from underrepresented populations, lack
precision therapy, or lack ca...</p>
          <p><a href="/hcmi-cmdc/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://portal.gdc.cancer.gov/projects/HCMI-CMDC" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></li></a>
          </li>
        </ul>
        <p><a href="/hcmi-cmdc/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
      </div>
    </div>


    <hr/>
  </div>  </body>
  <script src="/js/index.js"></script>
</html>
