<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    
  
    
  
    <link rel="alternate" type="application/rss+xml" href="/rss.xml" />
    
    <link rel="shortcut icon" type="image/ico" href="https://a0.awsstatic.com/libra-css/images/site/fav/favicon.ico"/>
    <link rel="apple-touch-icon" sizes="57x57" href="https://a0.awsstatic.com/libra-css/images/site/touch-icon-iphone-114-smile.png"/>
    <link rel="apple-touch-icon" sizes="72x72" href="https://a0.awsstatic.com/libra-css/images/site/touch-icon-ipad-144-smile.png"/>
    <link rel="apple-touch-icon" sizes="114x114" href="https://a0.awsstatic.com/libra-css/images/site/touch-icon-iphone-114-smile.png"/>
    <link rel="apple-touch-icon" sizes="144x144" href="https://a0.awsstatic.com/libra-css/images/site/touch-icon-ipad-144-smile.png"/>  
  
    <title>Registry of Open Data on AWS</title>
  
    <!-- Latest compiled and minified CSS -->
    <link rel="stylesheet" href="https://assets.opendata.aws/css/bootstrap/3.4.1/bootstrap.min.css">
  
    <!-- Our local CSS -->
    <link rel="stylesheet" href="/css/main.css">
  
  
  </head>
  <body>
    <!-- jQuery (necessary for Bootstrap's JavaScript plugins) -->
    <script src="https://assets.opendata.aws/js/jquery/3.5.1/jquery.min.js"></script>
    <!-- Latest compiled and minified JavaScript -->
    <script src="https://assets.opendata.aws/js/bootstrap/3.4.1/bootstrap.min.js"></script>
    <nav class="navbar navbar-default navbar-fixed-top">
      <div class="container">
        <div class="roda-header">
          <h2><a href="/" alt="Home">Registry of Open Data on AWS</a></h2>
          <a href="http://aws.amazon.com/what-is-cloud-computing">
            <img src="https://assets.opendata.aws/img/AWS-Logo_White-Color_300x180.png" alt="Powered by AWS Cloud Computing" id="aws_header_logo">
          </a>
        </div>
      </div>
    </nav>
    <div class="container" id="dataatwork">
  <p class="lede">Usage examples for all datasets listed in the <a href="/">Registry of Open Data on AWS</a> tagged with <strong>aws-pds</strong>.</p>
  <hr>

      <div id="tcga" class="dataset">
        <h4><a href="../tcga/">The Cancer Genome Atlas</a></h4>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://firecloud.terra.bio/" class="" target="_none">Broad Institute FireCloud</a> by <a href="https://www.broadinstitute.org/" target="_none">The Broad Institute of MIT &amp; Harvard</a></a>
            <br>
            
          </li>          <li>
            <a href="https://cgc-accounts.sbgenomics.com/" class="" target="_none">Cancer Genomics Cloud</a> by <a href="https://docs.sevenbridges.com/docs/tcga-data" target="_none">Seven Bridges</a></a>
            <br>
            
          </li>          <li>
            <a href="https://portal.gdc.cancer.gov/legacy-archive/" class="" target="_none">GDC Legacy Archive</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></a>
            <br>
            
          </li>          <li>
            <a href="https://portal.gdc.cancer.gov/projects/" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></a>
            <br>
            
          </li>          <li>
            <a href="https://isb-cgc.appspot.com/" class="" target="_none">ISB Cancer Genomics Cloud</a> by <a href="https://isb-cancer-genomics-cloud.readthedocs.io/en/latest/sections/data/TCGA_top.html
          " target="_none">Institute for Systems Biology</a></a>
            <br>
            
          </li>          <li>
            <a href="https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga/studied-cancers
          " class="" target="_none">TCGA Cancers Selected for Study</a> by <a href="https://ocg.cancer.gov/" target="_none">National Cancer Institute</a></a>
            <br>
            
          </li>          <li>
            <a href="https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga/using-tcga
          " class="" target="_none">Using TCGA Data, Resources, and Materials</a> by <a href="https://ocg.cancer.gov/" target="_none">National Cancer Institute</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1016/j.cels.2019.06.006" class="" target="_none">&quot;Before and After: A Comparison of Legacy and Harmonized TCGA Data at the Genomic Data
          Commons&quot;
          </a> by Galen F. Gao, Joel S. Parker, et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1016/j.cell.2018.03.027" class="" target="_none">A Pan-Cancer Analysis of Enhancer Expression in Nearly 9000 Patient Samples</a> by Han Chen, Chunyan Li, et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1016/j.cell.2018.02.052" class="" target="_none">An Integrated TCGA Pan-Cancer Clinical Data Resource to Drive High-Quality Survival Outcome
          Analytics
          </a> by Jianfang Liu, Tara Lichtenberg, et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1016/j.cell.2018.03.022" class="" target="_none">Cell-of-Origin Patterns Dominate the Molecular Classification of 10,000 Tumors from 33 Types
          of Cancer
          </a> by Katherine A. Hoadley, Christina Yau, et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1016/j.ccell.2018.03.010" class="" target="_none">Comparative Molecular Analysis of Gastrointestinal Adenocarcinomas</a> by Yang Liu, Nilay S. Sethi, et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1016/j.ccell.2018.07.001" class="" target="_none">Comprehensive Analysis of Alternative Splicing Across Tumors from 8,705 Patients</a> by André Kahles, Kjong-Van Lehmann, et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1016/j.cell.2018.02.060" class="" target="_none">Comprehensive Characterization of Cancer Driver Genes and Mutations</a> by Matthew H. Bailey, Collin Tokheim, et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1016/j.ccell.2018.03.007" class="" target="_none">Genomic and Functional Approaches to Understanding Cancer Aneuploidy</a> by Alison M. Taylor, Juliann Shih, et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1016/j.celrep.2018.03.076" class="" target="_none">Genomic and Molecular Landscape of DNA Damage Repair Deficiency across The Cancer Genome
          Atlas
          </a> by Theo A. Knijnenburg, Linghua Wang, et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1016/j.celrep.2018.03.063" class="" target="_none">Genomic, Pathway Network, and Immunologic Features Distinguishing Squamous Carcinomas</a> by Joshua D. Campbell, Christina Yau, et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1016/j.celrep.2018.03.047" class="" target="_none">Integrated Genomic Analysis of the Ubiquitin Pathway across Cancer Types</a> by Zhongqi Ge, Jake S. Leighton, et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1016/j.celrep.2018.03.046" class="" target="_none">Machine Learning Detects Pan-Cancer Ras Pathway Activation in The Cancer Genome Atlas</a> by Gregory P. Way, Francisco Sanchez-Vega, et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1016/j.cell.2018.03.034" class="" target="_none">Machine Learning Identifies Stemness Features Associated with Oncogenic Dedifferentiation
          </a> by Tathiane M. Malta, Artem Sokolov, et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1016/j.celrep.2018.03.077" class="" target="_none">Molecular Characterization and Clinical Relevance of Metabolic Expression Subtypes in Human
          Cancers
          </a> by Xinxin Peng, Zhongyuan Chen, et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1016/j.cell.2018.03.035" class="" target="_none">Oncogenic Signaling Pathways in The Cancer Genome Atlas</a> by Francisco Sanchez-Vega, Marco Mina, et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1016/j.cels.2018.03.003" class="" target="_none">Pan-Cancer Alterations of the MYC Oncogene and its Proximal Network Across The Cancer Genome
          Atlas
          </a> by Franz X. Schaub, Varsha Dhankani, et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1016/j.celrep.2018.03.064" class="" target="_none">Pan-Cancer Analysis of lncRNA Regulation Supports Their Targeting of Cancer Genes in Each
          Tumor Context
          </a> by Hua-Sheng Chiu, Sonal Somvanshi, et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1016/j.cell.2018.03.039" class="" target="_none">Pathogenic Germline Variants in 10,389 Adult Cancers</a> by Kuan-lin Huang, R. Jay Mashl, et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1016/j.cels.2018.03.002" class="" target="_none">Scalable Open Science Approach for Mutation Calling of Tumor Exomes Using Multiple Genomic
          Pipelines
          </a> by Kyle Ellrott, Matthew H. Bailey, et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1016/j.celrep.2018.03.086" class="" target="_none">Spatial Organization And Molecular Correlation Of Tumor-Infiltrating Lymphocytes Using Deep
          Learning On Pathology Images
          </a> by Joel Saltz, Rajarsi Gupta, et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1126/science.aav1898" class="" target="_none">The chromatin accessibility landscape of primary human cancers</a> by M. Ryan Corces, Jeffrey M. Granja, et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1016/j.immuni.2018.03.023" class="" target="_none">The Immune Landscape of Cancer</a> by Vésteinn Thorsson, David L. Gibbs, et al.</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="target" class="dataset">
        <h4><a href="../target/">Therapeutically Applicable Research to Generate Effective Treatments (TARGET)</a></h4>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://firecloud.terra.bio/" class="" target="_none">Broad Institute FireCloud</a> by <a href="https://www.broadinstitute.org/" target="_none">The Broad Institute of MIT &amp; Harvard</a></a>
            <br>
            
          </li>          <li>
            <a href="https://cgc-accounts.sbgenomics.com/" class="" target="_none">Cancer Genomics Cloud</a> by <a href="https://docs.sevenbridges.com/docs/target-data" target="_none">Seven Bridges</a></a>
            <br>
            
          </li>          <li>
            <a href="https://portal.gdc.cancer.gov/legacy-archive/" class="" target="_none">GDC Legacy Archive</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></a>
            <br>
            
          </li>          <li>
            <a href="https://portal.gdc.cancer.gov/projects/" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></a>
            <br>
            
          </li>          <li>
            <a href="https://isb-cgc.appspot.com/" class="" target="_none">ISB Cancer Genomics Cloud</a> by <a href="https://isb-cancer-genomics-cloud.readthedocs.io/en/latest/sections/data/TARGET_top.html
          " target="_none">Institute for Systems Biology</a></a>
            <br>
            
          </li>          <li>
            <a href="https://ocg.cancer.gov/programs/target/data-matrix" class="" target="_none">TARGET data matrix</a> by <a href="https://ocg.cancer.gov/" target="_none">National Cancer Institute</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1038/ng.3940" class="" target="_none">A Children&#x27;s Oncology Group and TARGET initiative exploring the genetic landscape of Wilms
          tumor
          </a> by Gadd S, Huff V, Walz AL, et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/21297632" class="" target="_none">Ancestry and pharmacogenomics of relapse in acute lymphoblastic leukemia</a> by Yang JJ, Cheng C, Devidas M, et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term&#x3D;29228567" class="" target="_none">Biomarker significance of plasma and tumor miR-21, miR-221, and miR-106a in osteosarcoma
          </a> by Nakka M, Allen-Rhoades W, Li Y, et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term&#x3D;27143256" class="" target="_none">CSF3R mutations have a high degree of overlap with CEBPA mutations in pediatric AM</a> by Maxson JE, Ries RE, Wang YC, et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term&#x3D;26560027" class="" target="_none">Genetic predisposition to neuroblastoma mediated by a LMO1 super-enhancer polymorphism</a> by Oldridge DA, Wood AC, Weichert-Leahey N, Crimmins I, Sussman R, Winter C, McDaniel LD,
          Diamond M, Hart LS, Zhu S, Durbin AD, Abraham BJ, et al.
          </a>
            <br>
            
          </li>          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term&#x3D;26977886" class="" target="_none">Genome-Wide Profiles of Extra-cranial Malignant Rhabdoid Tumors Reveal Heterogeneity and
          Dysregulated Developmental Pathways
          </a> by Chun HJ, Lim EL, Heravi-Moussavi A, et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1038/s41586-018-0436-0" class="" target="_none">Genomic classification and identification of the cell of origin of pediatric mixed phenotype
          acute leukemia
          </a> by Thomas B. Alexander, Zhaohui Gu, Ilaria Iacobucci, et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term&#x3D;29315296" class="" target="_none">Genomics in childhood acute myeloid leukemia comes of age</a> by Brunner AM, Graubert TA.</a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1016/j.celrep.2019.10.013" class="" target="_none">Identification and analyses of extra-cranial and cranial rhabdoid tumor molecular subgroups
          reveal tumors with cytotoxic T cell infiltration
          </a> by Hye-Jung E. Chun, Pascal D. Johann, Katy Milne et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term&#x3D;29068783" class="" target="_none">MicroRNA Expression-Based Model Indicates Event-Free Survival in Pediatric Acute Myeloid
          Leukemia
          </a> by Lim EL, Trinh DL, Ries RE, et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term&#x3D;26635203" class="" target="_none">MLLT1 YEATS domain mutations in clinically distinctive Favourable Histology Wilms tumours
          </a> by Perlman EJ, Gadd S, Arold ST, et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term&#x3D;25670082" class="" target="_none">Recurrent DGCR8, DROSHA, and SIX homeodomain mutations in favorable histology Wilms tumors
          </a> by Walz AL, Ooms A, Gadd S, et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term&#x3D;26121087" class="" target="_none">Relapsed neuroblastomas show frequent RAS-MAPK pathway mutations</a> by Eleveld TF, Oldridge DA, Bernard V, Koster J, et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term&#x3D;27702824" class="" target="_none">Significance of TP53 Mutation in Wilms Tumors with Diffuse Anaplasia: A Report from the
          Children&#x27;s Oncology Group
          </a> by Ooms AH, Gadd S, Gerhard DS, et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term&#x3D;26158413" class="" target="_none">TCF21 hypermethylation in genetically quiescent clear cell sarcoma of the kidney</a> by Gooskens SL, Gadd S, Guidry Auvil JM, et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term&#x3D;23334666" class="" target="_none">The genetic landscape of high-risk neuroblastoma</a> by Pugh TJ, Morozova O, Attiyeh EF, Asgharzadeh S, Wei JS, Auclair D, Carter SL, Cibulskis K,
          Hanna M, Kiezun A, Kim J, Lawrence MS, Lichenstein L, et al.
          </a>
            <br>
            
          </li>          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term&#x3D;28671688" class="" target="_none">The genomic landscape of pediatric and young adult T-lineage acute lymphoblastic leukemia
          </a> by Yu Liu, John Easton, Ying Shao, et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term&#x3D;29227476" class="" target="_none">The molecular landscape of pediatric acute myeloid leukemia reveals recurrent structural
          alterations and age-specific mutational interactions
          </a> by Bolouri H, Farrar JE, Triche T Jr, et al.</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="commoncrawl" class="dataset">
        <h4><a href="../commoncrawl/">Common Crawl</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="http://tech.marksblogg.com/petabytes-of-website-data-spark-emr.html" class="" target="_none">Analysing Petabytes of Websites</a> by Mark Litwintschik</a>
            <br>
            <span class="label label-info tag service-tag">EMR</span>
          </li>          <li>
            <a href="https://skeptric.com/common-crawl-index-athena/" class="" target="_none">Common Crawl Index Athena</a> by Edward Ross</a>
            <br>
            <span class="label label-info tag service-tag">Athena</span>
          </li>          <li>
            <a href="https://commoncrawl.org/2018/03/index-to-warc-files-and-urls-in-columnar-format/" class="" target="_none">Index to WARC Files and URLs in Columnar Format</a> by Sebastian Nagel</a>
            <br>
            <span class="label label-info tag service-tag">Athena</span>
          </li>          <li>
            <a href="https://towardsdatascience.com/large-scale-graph-mining-with-spark-750995050656" class="" target="_none">Large-scale graph mining with Spark</a> by <a href="https://github.com/wsuen/pygotham2018_graphmining" target="_none">Win Suen</a></a>
            <br>
            
          </li>          <li>
            <a href="https://github.com/andresriancho/cc-lambda" class="" target="_none">Search the Common Crawl Using Lambda Functions</a> by Andres Riancho</a>
            <br>
            <span class="label label-info tag service-tag">Lambda</span>
          </li>        </ul>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://arxiv.org/abs/1911.00359" class="" target="_none">CCNet: Extracting high quality monolingual datasets from web crawl data</a> by <a href="https://github.com/facebookresearch/cc_net" target="_none">Facebook AI Research</a></a>
            <br>
            
          </li>          <li>
            <a href="https://wwwdb.inf.tu-dresden.de/research-projects/dresden-web-table-corpus/" class="" target="_none">Dresden Web Table Corpus (DWTC)</a> by <a href="https://wwwdb.inf.tu-dresden.de/" target="_none">Database Systems Group Dresden</a></a>
            <br>
            
          </li>          <li>
            <a href="https://www.aclweb.org/anthology/L18-1550" class="" target="_none">Learning word vectors for 157 languages</a> by <a href="https://fasttext.cc/docs/en/crawl-vectors.html" target="_none">Facebook AI Research</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://hal.inria.fr/hal-02148693" class="" target="_none">Asynchronous pipeline for processing huge corpora on medium to low resource infrastructures</a> by <a href="https://oscar-corpus.com/" target="_none">Pedro Javier Ortiz Suárez, Benoît Sagot, Laurent Romary</a></a>
            <br>
            
          </li>          <li>
            <a href="https://arxiv.org/pdf/1710.01779.pdf" class="" target="_none">Building a Web-Scale Dependency-Parsed Corpus from CommonCrawl</a> by Alexander Panchenko, Eugen Ruppert, Stefano Faralli, Simone Paolo Ponzetto, Chris Biemann</a>
            <br>
            
          </li>          <li>
            <a href="http://www.lrec-conf.org/proceedings/lrec2016/pdf/388_Paper.pdf" class="" target="_none">C4Corpus: Multilingual Web-Size Corpus with Free License</a> by <a href="https://dkpro.github.io/dkpro-c4corpus/" target="_none">Ivan Habernal, Omnia Zayed, Iryna Gurevych</a></a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1145/3340531.3412762" class="" target="_none">CC-News-En: A large English news corpus</a> by Joel Mackenzie, Rodger Benham, Matthias Petri, Johanne R. Trippas, J. Shane Culpepper, Alistair Moffat</a>
            <br>
            
          </li>          <li>
            <a href="https://www.aclweb.org/anthology/2020.emnlp-main.480" class="" target="_none">CCAligned: A Massive collection of cross-lingual web-document pairs</a> by <a href="http://www.statmt.org/cc-aligned/" target="_none">Ahmed El-Kishky, Vishrav Chaudhary, Francisco Guzmán, Philipp Koehn</a></a>
            <br>
            
          </li>          <li>
            <a href="http://papers.nips.cc/paper/9106-defending-against-neural-fake-news.pdf" class="" target="_none">Defending against neural fake news</a> by <a href="https://rowanzellers.com/grover/" target="_none">Rowan Zellers, Ari Holtzman, Hannah Rashkin, Yonatan Bisk, Ali Farhadi, Franziska Roesner, et al</a></a>
            <br>
            
          </li>          <li>
            <a href="https://psuter.net/2019/07/07/z-index" class="" target="_none">Index fun</a> by Philippe Suter</a>
            <br>
            
          </li>          <li>
            <a href="https://arxiv.org/abs/2005.14165" class="" target="_none">Language models are few-shot learners</a> by Tom B. Brown, Benjamin Mann, Nick Ryder, Melanie Subbiah, Jared Kaplan, Prafulla Dhariwal, et al</a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1145/3178876.3186090" class="" target="_none">Large-scale analysis of style injection by relative path overwrite</a> by Sajjad Arshad, Seyed Ali Mirheidari, Tobias Lauinger, Bruno Crispo, Engin Kirda, William Robertson</a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1007/s10579-020-09489-2" class="" target="_none">Mapping languages: The Corpus of Global Language Use</a> by <a href="https://www.earthlings.io/" target="_none">Jonathan Dunn</a></a>
            <br>
            
          </li>          <li>
            <a href="http://www.lrec-conf.org/proceedings/lrec2014/pdf/1097_Paper.pdf" class="" target="_none">N-gram counts and language models from the Common Crawl</a> by <a href="http://statmt.org/ngrams/" target="_none">Christian Buck, Kenneth Heafield, Bas van Ooyen</a></a>
            <br>
            
          </li>          <li>
            <a href="https://fulmicoton.com/posts/commoncrawl/" class="" target="_none">Of using Common Crawl to play Family Feud</a> by Paul Masurel</a>
            <br>
            
          </li>          <li>
            <a href="https://arxiv.org/abs/2010.12002" class="" target="_none">On the impact of publicly available news and information transfer to financial markets</a> by Metod Jazbec, Barna Pásztor, Felix Faltings, Nino Antulov-Fantulin, Petter N. Kolm</a>
            <br>
            
          </li>          <li>
            <a href="https://education.emc.com/content/dam/dell-emc/documents/en-us/2017KS_Ravinder-Using_Open_Data_to_Predict_Market_Movements.pdf" class="" target="_none">Using open data to predict market movements</a> by DELL EMC</a>
            <br>
            
          </li>          <li>
            <a href="http://webdatacommons.org/structureddata/" class="" target="_none">Web Data Commons - RDFa, microdata, and microformat data sets</a> by Christian Bizer, Robert Meusel, Anna Primpeli</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="sentinel-2" class="dataset">
        <h4><a href="../sentinel-2/">Sentinel-2</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://www.gearthblog.com/blog/archives/2017/02/exploring-chile-wildfires-landsat-sentinel-2-imagery.html" class="" target="_none">Exploring the Chile wildfires with Landsat and Sentinel-2 imagery</a> by Timothy Whitehead</a>
            <br>
            
          </li>          <li>
            <a href="https://www.esri.com/arcgis-blog/products/arcgis-living-atlas/imagery/sentinel-2-more-than-meets-the-eye/" class="" target="_none">Integrate imagery from the Sentinel-2 archive into your own apps, maps, and analysis with the Sentinel-2 image service</a> by <a href="https://www.esri.com/en-us/home" target="_none">Esri</a></a>
            <br>
            
          </li>          <li>
            <a href="https://medium.com/sentinel-hub/create-useful-and-beautiful-satellite-images-with-custom-scripts-8ef0e6a474c6" class="" target="_none">Learning Custom Scripts to Make Useful and Beautiful Satellite Images</a> by Monja Šebela</a>
            <br>
            
          </li>          <li>
            <a href="https://medium.com/@anttilip/seeing-earth-from-space-from-raw-satellite-data-to-beautiful-high-resolution-images-feb522adfa3f" class="" target="_none">Tutorial for using Sentinel-2 data</a> by <a href="https://medium.com/@anttilip" target="_none">Antti Lipponen</a></a>
            <br>
            
          </li>          <li>
            <a href="https://remotepixel.ca/blog/searchapi_20171211.html" class="" target="_none">Using Vector tiles and AWS Lambda, we can build a really simple API to get Landsat and Sentinel images</a> by <a href="https://remotepixel.ca/" target="_none">Remote Pixel</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="http://apps.sentinel-hub.com/eo-browser/" class="" target="_none">EO Browser</a> by <a href="http://www.sinergise.com/" target="_none">Sinergise</a></a>
            <br>
            
          </li>          <li>
            <a href="https://eos.com/landviewer/" class="" target="_none">EOS Land Viewer</a> by <a href="https://eos.com/" target="_none">Earth Observing System</a></a>
            <br>
            
          </li>          <li>
            <a href="http://demos.fmeserver.com/landsat-sentinel-selector/" class="" target="_none">FME Landsat-8/Sentinel-2 File Selector</a> by <a href="https://www.safe.com/" target="_none">Safe Software</a></a>
            <br>
            
          </li>          <li>
            <a href="https://map.onesoil.ai/" class="" target="_none">Interactive map with AI detected fields and crops based on Sentinel-2 imagery</a> by <a href="https://onesoil.ai/" target="_none">OneSoil</a></a>
            <br>
            
          </li>          <li>
            <a href="https://github.com/sentinel-hub/sentinelhub-py" class="" target="_none">Python package for working with Sentinel-2 AWS data</a> by <a href="http://www.sinergise.com/" target="_none">Sinergise</a></a>
            <br>
            
          </li>          <li>
            <a href="https://medium.com/sentinel-hub/control-sentinel-hub-from-within-qgis-2a83eb7f13db" class="" target="_none">QGIS plugin for Sentinel-2 data</a> by <a href="http://www.sinergise.com/" target="_none">Sinergise</a></a>
            <br>
            
          </li>          <li>
            <a href="https://remotepixel.ca/projects/satellitesearch.html" class="" target="_none">Satellite Search by Remote Pixel</a> by <a href="https://remotepixel.ca/" target="_none">Remote Pixel</a></a>
            <br>
            
          </li>          <li>
            <a href="http://www.sentinel-hub.com/apps/wms" class="" target="_none">Sentinel Hub WMS/WMTS/WCS Service</a> by <a href="http://www.sinergise.com/" target="_none">Sinergise</a></a>
            <br>
            
          </li>          <li>
            <a href="http://apps.sentinel-hub.com/sentinel-playground/" class="" target="_none">Sentinel Playground</a> by <a href="http://www.sinergise.com/" target="_none">Sinergise</a></a>
            <br>
            
          </li>          <li>
            <a href="https://s2maps.eu/" class="" target="_none">Sentinel-2 Cloudless Atlas</a> by <a href="https://eox.at" target="_none">EOX</a></a>
            <br>
            
          </li>          <li>
            <a href="https://spectator.earth/" class="" target="_none">Spectator - tracking Sentinel 2, accessing the data and quick preview</a> by <a href="https://spectator.earth/" target="_none">Spectator</a></a>
            <br>
            
          </li>          <li>
            <a href="https://www.youtube.com/watch?v&#x3D;3tIIUr-keps" class="" target="_none">Sterling Geo Using Sentinel-2 on Amazon Web Services to Create NDVI</a> by Sterling Geo</a>
            <br>
            
          </li>          <li>
            <a href="https://sentinel2explorer.esri.com/" class="" target="_none">Use the Sentinel Explorer app to explore, visualize, and analyze the entire Sentinel-2 archive</a> by <a href="https://www.esri.com/en-us/home" target="_none">Esri</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://www.sciencedirect.com/science/article/pii/S0034425720304284" class="" target="_none">Coral-spawn slicks: Reflectance spectra and detection using optical satellite data</a> by Hiroya Yamano, Asahi Sakuma, Saki Harii</a>
            <br>
            
          </li>          <li>
            <a href="https://ieeexplore.ieee.org/abstract/document/9165649" class="" target="_none">Using Remote Sensing Images and Cloud Services on AWS to Improve Land Use and Cover Monitoring</a> by K. R. Ferreira, et al.</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="kids-first" class="dataset">
        <h4><a href="../kids-first/">Gabriella Miller Kids First Pediatric Research Program (Kids First)</a></h4>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="http://cavatica.org" class="" target="_none">CAVATICA</a> by <a href="http://www.sevenbridges.com" target="_none">Seven Bridges Genomics</a></a>
            <br>
            
          </li>          <li>
            <a href="https://portal.kidsfirstdrc.org" class="" target="_none">Kids First DRC Portal</a> by <a href="https://kidsfirstdrc.org" target="_none">Kids First DRC</a></a>
            <br>
            
          </li>          <li>
            <a href="https://github.com/kids-first/" class="" target="_none">Kids First DRC Source Code</a> by <a href="https://kidsfirstdrc.org" target="_none">Kids First DRC</a></a>
            <br>
            
          </li>          <li>
            <a href="https://pedcbioportal.kidsfirstdrc.org" class="" target="_none">PedcBioPortal</a> by <a href="https://www.cbioportal.org/" target="_none">cBioPortal</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://pubmed.ncbi.nlm.nih.gov/30322880/" class="" target="_none">Clinically Relevant and Minimally Invasive Tumor Surveillance of Pediatric Diffuse Midline Gliomas Using Patient-Derived Liquid Biopsy</a> by Eshini Panditharatna, Lindsay B Kilburn, et al.</a>
            <br>
            
          </li>          <li>
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/31211835" class="" target="_none">Decreased ACKR3 (CXCR7) function causes oculomotor synkinesis in mice and humans.</a> by Mary C Whitman, Noriko Miyake, et al.</a>
            <br>
            
          </li>          <li>
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/31206972" class="" target="_none">Deleterious de novo variants of X-linked ZC4H2 in females cause a variable phenotype with neurogenic arthrogryposis multiplex congenita.</a> by Suzanna G M Frints, Friederike Hennig, et al.</a>
            <br>
            
          </li>          <li>
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/31255796" class="" target="_none">Development and Clinical Validation of a Large Fusion Gene Panel for Pediatric Cancers.</a> by Fengqi Chang, Fumin Lin, et al.</a>
            <br>
            
          </li>          <li>
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/31898844" class="" target="_none">Elucidation of de novo small insertion/deletion biology with parent-of-origin phasing.</a> by Allison H Seiden, Felix Richter, et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://pubmed.ncbi.nlm.nih.gov/30098192/" class="" target="_none">Genome-Wide Association Study Identifies a Susceptibility Locus for Comitant Esotropia and Suggests a Parent-of-Origin Effect</a> by Sherin Shaaban, Sarah MacKinnon et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://pubmed.ncbi.nlm.nih.gov/32574564/" class="" target="_none">Genome-wide Enrichment of De Novo Coding Mutations in Orofacial Cleft Trios.</a> by Madison R Bishop, Kimberly K Diaz Perez, et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://pubmed.ncbi.nlm.nih.gov/32601476/" class="" target="_none">Genomic Analyses Implicate Noncoding De Novo Variants in Congenital Heart Disease.</a> by Felix Richter, Sarah U Morton, et al.</a>
            <br>
            
          </li>          <li>
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/31474320" class="" target="_none">Germline 16p11.2 Microdeletion Predisposes to Neuroblastoma.</a> by Laura Egolf, Zalman Vaksman, et al.</a>
            <br>
            
          </li>          <li>
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/31562520" class="" target="_none">Germline microsatellite genotypes differentiate children with medulloblastoma.</a> by Samuel Rivero-Hinojosa, Nicholas Kinney, et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://pubmed.ncbi.nlm.nih.gov/30471716/" class="" target="_none">MACF1 Mutations Encoding Highly Conserved Zinc-Binding Residues of the GAR Domain Cause Defects in Neuronal Migration and Axon Guidance</a> by William B Dobyns, Kimberly A Aldinger, et al.</a>
            <br>
            
          </li>          <li>
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/31397880" class="" target="_none">MAGEL2-Related Disorders: A study and case series.</a> by Jameson Patak, James Gilfert, et al.</a>
            <br>
            
          </li>          <li>
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/31361404" class="" target="_none">Phenotype delineation of ZNF462 related syndrome.</a> by Paul Kruszka, Tommy Hu, et al.</a>
            <br>
            
          </li>          <li>
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/30930166" class="" target="_none">The Pediatric Cell Atlas: Defining the Growth Phase of Human Development at Single-Cell Resolution.</a> by Deanne M Taylor, Bruce J Aronow, et al.</a>
            <br>
            
          </li>          <li>
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/31848685" class="" target="_none">Whole genome sequencing of orofacial cleft trios from the Gabriella Miller Kids First Pediatric Research Consortium identifies a new locus on chromosome 21.</a> by Nandita Mukhopadhyay, Madison Bishop, et al.</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="landsat-8" class="dataset">
        <h4><a href="../landsat-8/">Landsat 8</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://medium.com/planet-stories/a-gentle-introduction-to-gdal-part-4-working-with-satellite-data-d3835b5e2971" class="" target="_none">A Gentle Introduction to GDAL Part 4: Working with Satellite Data</a> by <a href="https://www.planet.com/" target="_none">Planet</a></a>
            <br>
            
          </li>          <li>
            <a href="https://www.gearthblog.com/blog/archives/2017/02/exploring-chile-wildfires-landsat-sentinel-2-imagery.html" class="" target="_none">Exploring the Chile wildfires with Landsat and Sentinel-2 imagery</a> by Timothy Whitehead</a>
            <br>
            
          </li>          <li>
            <a href="http://www.arcgis.com/home/group.html?id&#x3D;a74dff13f1be4b2ba7264c3315c57077#overview" class="" target="_none">Integrate imagery from the full Landsat archive into your own apps, maps, and analysis with Landsat image services</a> by <a href="https://www.esri.com/en-us/home" target="_none">Esri</a></a>
            <br>
            
          </li>          <li>
            <a href="http://www.acgeospatial.co.uk/cog-part-2-python/" class="" target="_none">Using Cloud-Optimized GeoTIFF</a> by <a href="http://www.acgeospatial.co.uk/" target="_none">acgeospatial</a></a>
            <br>
            
          </li>          <li>
            <a href="https://remotepixel.ca/blog/searchapi_20171211.html" class="" target="_none">Using Vector tiles and AWS Lambda, we can build a really simple API to get Landsat and Sentinel images</a> by <a href="https://remotepixel.ca/" target="_none">Remote Pixel</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://twitter.com/aggibot" class="" target="_none">Aggi: the fire detecting Twitter bot</a> by <a href="https://twitter.com/aggibot" target="_none">Aggi</a></a>
            <br>
            
          </li>          <li>
            <a href="http://www.esri.com/landing-pages/software/landsat/unlock-earths-secrets" class="" target="_none">Apps for exploring and analyzing Landsat imagery on the fly</a> by <a href="https://www.esri.com/en-us/home" target="_none">Esri</a></a>
            <br>
            
          </li>          <li>
            <a href="https://geotiffjs.github.io/cog-explorer/" class="" target="_none">COG-Explorer - View Cloud Optimized GeoTIFF images in the browser directly from object storage</a> by <a href="https://eox.at/" target="_none">EOX</a></a>
            <br>
            
          </li>          <li>
            <a href="https://developmentseed.org/blog/2017/08/17/geolambdas/" class="" target="_none">Development Seed Geolambda</a> by <a href="https://developmentseed.org/team/matt-hanson/" target="_none">Matthew Hanson</a></a>
            <br>
            
          </li>          <li>
            <a href="https://kylebarron.dev/blog/cog-mosaic/overview" class="" target="_none">Dynamic map tiling with Cloud-Optimized GeoTIFFs</a> by Kyle Barron</a>
            <br>
            
          </li>          <li>
            <a href="https://eos.com/landviewer/" class="" target="_none">EOS Land Viewer</a> by <a href="https://eos.com/" target="_none">Earth Observing System</a></a>
            <br>
            
          </li>          <li>
            <a href="https://docs.safe.com/fme/html/FME_Desktop_Documentation/FME_ReadersWriters/landsat8aws/landsat8aws.htm" class="" target="_none">FME Landsat-8 on AWS Reader</a> by <a href="https://www.safe.com/" target="_none">Safe Software</a></a>
            <br>
            
          </li>          <li>
            <a href="http://demos.fmeserver.com/landsat-sentinel-selector/" class="" target="_none">FME Landsat-8/Sentinel-2 File Selector</a> by <a href="https://www.safe.com/" target="_none">Safe Software</a></a>
            <br>
            
          </li>          <li>
            <a href="https://landsatlive.live" class="" target="_none">landsatlive.live</a> by <a href="https://developmentseed.org/" target="_none">Development Seed</a></a>
            <br>
            
          </li>          <li>
            <a href="http://www.sentinel-hub.com/apps/wms" class="" target="_none">Sentinel Hub WMS/WMTS/WCS Service for Landsat</a> by <a href="http://www.sinergise.com/" target="_none">Sinergise</a></a>
            <br>
            
          </li>          <li>
            <a href="https://apps.sentinel-hub.com/sentinel-playground/?source&#x3D;L8" class="" target="_none">Sentinel Playground for Landsat</a> by <a href="http://www.sinergise.com/" target="_none">Sinergise</a></a>
            <br>
            
          </li>          <li>
            <a href="https://spectator.earth/" class="" target="_none">Spectator - tracking Landsat 8, accessing the data and quick preview</a> by <a href="https://spectator.earth/" target="_none">Spectator</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://ieeexplore.ieee.org/abstract/document/9165649" class="" target="_none">Using Remote Sensing Images and Cloud Services on AWS to Improve Land Use and Cover Monitoring</a> by K. R. Ferreira, et al.</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="sudachi" class="dataset">
        <h4><a href="../sudachi/">Sudachi Language Resources</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/WorksApplications/elasticsearch-sudachi/blob/develop/docs/tutorial.md" class="" target="_none">analysis-sudachi Tutorial</a> by Works Applications</a>
            <br>
            
          </li>          <li>
            <a href="https://github.com/WorksApplications/chiVe/blob/master/docs/tutorial.md" class="" target="_none">chiVe Tutorial</a> by Works Applications</a>
            <br>
            
          </li>          <li>
            <a href="https://github.com/WorksApplications/Sudachi/blob/develop/docs/tutorial.md" class="" target="_none">Sudachi Tutorial</a> by Works Applications</a>
            <br>
            
          </li>          <li>
            <a href="https://github.com/WorksApplications/SudachiPy/blob/develop/docs/tutorial.md" class="" target="_none">SudachiPy Tutorial</a> by Works Applications</a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/WorksApplications/elasticsearch-sudachi" class="" target="_none">analysis-sudachi: Sudachi pluglin for Elasticsearch</a> by Works Applications</a>
            <br>
            
          </li>          <li>
            <a href="https://github.com/WorksApplications/jdartsclone" class="" target="_none">jdartsclone: TRIE Data Structure using Double-Array</a> by Works Applications</a>
            <br>
            
          </li>          <li>
            <a href="https://github.com/WorksApplications/kintoki" class="" target="_none">Kintoki: Dependency Parser</a> by Works Applications</a>
            <br>
            
          </li>          <li>
            <a href="https://github.com/WorksApplications/Sudachi" class="" target="_none">Sudachi: Japanese Tokenizer for Business</a> by Works Applications</a>
            <br>
            
          </li>          <li>
            <a href="https://pypi.org/project/SudachiDict-core/" class="" target="_none">sudachidict_core on pypi.python.org - a Python module to download and install SudachiDict for the python tokenizer</a> by Works Applications</a>
            <br>
            
          </li>          <li>
            <a href="https://pypi.org/project/SudachiDict-full/" class="" target="_none">sudachidict_full on pypi.python.org - a Python module to download and install SudachiDict for the python tokenizer</a> by Works Applications</a>
            <br>
            
          </li>          <li>
            <a href="https://pypi.org/project/SudachiDict-small/" class="" target="_none">sudachidict_small on pypi.python.org - a Python module to download and install SudachiDict for the python tokenizer</a> by Works Applications</a>
            <br>
            
          </li>          <li>
            <a href="https://github.com/WorksApplications/Sudachi" class="" target="_none">SudachiPy: Python version of Sudachi</a> by Works Applications</a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://www.anlp.jp/proceedings/annual_meeting/2020/pdf_dir/P6-16.pdf" class="" target="_none">chiVe 2.0: SudachiとNWJCを用いた実用的な日本語単語ベクトルの実現に向けて</a> by 河村宗一郎, 久本空海, 真鍋陽俊, 髙岡一馬, 内田佳孝, 岡照晃, 浅原正幸</a>
            <br>
            
          </li>          <li>
            <a href="https://www.ieice.org/ken/paper/20200910U1zQ/" class="" target="_none">chiVe: 製品利用可能な日本語単語ベクトル資源の実現へ向けて ～形態素解析器Sudachiと超大規模ウェブコーパスNWJCによる分散表現の獲得と改良～</a> by 久本空海, 山村崇, 勝田哲弘, 竹林佑斗, 髙岡一馬, 内田佳孝, 岡照晃, 浅原正幸</a>
            <br>
            
          </li>          <li>
            <a href="http://www.lrec-conf.org/proceedings/lrec2018/summaries/8884.html" class="" target="_none">Sudachi: a Japanese Tokenizer for Business</a> by Kazuma Takaoka, Sorami Hisamoto, Noriko Kawahara, Miho Sakamoto, Yoshitaka Uchida, Yuji Matsumoto</a>
            <br>
            
          </li>          <li>
            <a href="https://pj.ninjal.ac.jp/corpus_center/lrw/lrw2018/P-1-08.pdf" class="" target="_none">形態素解析器『Sudachi』のための大規模辞書開発</a> by 坂本美保, 川原典子, 久本空海, 髙岡一馬, 内田佳孝</a>
            <br>
            
          </li>          <li>
            <a href="https://www.anlp.jp/proceedings/annual_meeting/2019/pdf_dir/P8-5.pdf" class="" target="_none">複数粒度の分割結果に基づく日本語単語分散表現</a> by 真鍋陽俊, 岡照晃, 海川祥毅, 髙岡一馬, 内田佳孝, 浅原正幸</a>
            <br>
            
          </li>          <li>
            <a href="https://www.anlp.jp/proceedings/annual_meeting/2020/pdf_dir/P4-14.pdf" class="" target="_none">詳細化した同義関係をもつ同義語辞書の作成</a> by 高岡一馬, 岡部裕子, 川原典子, 坂本美保, 内田佳孝</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="foldingathome-covid19" class="dataset">
        <h4><a href="../foldingathome-covid19/">Foldingathome COVID-19 Datasets</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/FoldingAtHome/coronavirus/blob/master/README.md" class="" target="_none">Folding@home COVID-19 efforts</a> by Folding@home Consortium</a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://covid.molssi.org//simulations/#foldinghome-expanded-ensemble-absolute-free-energy-calculations-of-potential-small-molecule-inhibitors-of-the-sars-cov-2-main-protease-from-the-covid-moonshot-3clpro-3clpro-protease-activity" class="" target="_none">SARS-CoV-2 COVID Moonshot absolute free energy calculations</a> by <a href="http://www.voelzlab.org/" target="_none">The Voelz lab at Temple University</a></a>
            <br>
            
          </li>          <li>
            <a href="https://covid.molssi.org//simulations/#foldinghome-simulations-of-nsp5-3clpro-3clpro-protease-activity" class="" target="_none">SARS-CoV-2 main viral protease (Mpro, 3CLPro, nsp5) dimer simulations: A 2.9 ms dataset of the SARS-CoV-2 main viral protease (apo, dimer) in search of cryptic pockets</a> by <a href="https://bowmanlab.biochem.wustl.edu" target="_none">The Bowman lab at Washington University in St. Louis</a></a>
            <br>
            
          </li>          <li>
            <a href="https://covid.molssi.org//simulations/#foldinghome-sars-cov-2-main-protease-apo-monomer-simulations-3clpro-3clpro-protease-activity" class="" target="_none">SARS-CoV-2 main viral protease (Mpro, 3CLPro, nsp5) monomer simulations: A 2.6 ms equilibrium dataset of the SARS-CoV-2 main viral protease (apo, monomer)</a> by <a href="https://choderalab.org" target="_none">The Chodera lab at MSKCC</a></a>
            <br>
            
          </li>          <li>
            <a href="https://covid.molssi.org//simulations/#foldinghome-simulations-of-nsp5-3clpro-3clpro-protease-activity" class="" target="_none">SARS-CoV-2 main viral protease (Mpro, 3CLPro, nsp5) monomer simulations: A 6.4 ms dataset of the SARS-CoV-2 main viral protease (apo, monomer) in search of cryptic pockets</a> by <a href="https://bowmanlab.biochem.wustl.edu" target="_none">The Bowman lab at Washington University in St. Louis</a></a>
            <br>
            
          </li>          <li>
            <a href="https://covid.molssi.org//simulations/#foldinghome-simulations-of-nsp10-nsp10-nil" class="" target="_none">SARS-CoV-2 nsp10 dataset: A 6.1 ms dataset of the SARS-CoV-2 nsp10 protein in search of cryptic pockets</a> by <a href="https://bowmanlab.biochem.wustl.edu" target="_none">The Bowman lab at Washington University in St. Louis</a></a>
            <br>
            
          </li>          <li>
            <a href="https://covid.molssi.org//simulations/#foldinghome-simulations-of-nsp3-macrodomain-macrodomain-host-immune-response" class="" target="_none">SARS-CoV-2 nsp3 macrodomain dataset: An 11 ms dataset of the SARS-CoV-2 nsp3 macrodomain in search of cryptic pockets</a> by <a href="https://bowmanlab.biochem.wustl.edu" target="_none">The Bowman lab at Washington University in St. Louis</a></a>
            <br>
            
          </li>          <li>
            <a href="https://covid.molssi.org//simulations/#foldinghome-simulations-of-nsp3-pl2pro-domain-plpro-plpro-protease-activity" class="" target="_none">SARS-CoV-2 nsp3 pl2pro domain dataset: An 731 µs dataset of the SARS-CoV-2 nsp3 pl2pro domain in search of cryptic pockets</a> by <a href="https://bowmanlab.biochem.wustl.edu" target="_none">The Bowman lab at Washington University in St. Louis</a></a>
            <br>
            
          </li>          <li>
            <a href="https://covid.molssi.org//simulations/#foldinghome-simulations-of-nsp7-nsp7-nil" class="" target="_none">SARS-CoV-2 nsp7 simulations: A 3.7 ms dataset of the SARS-CoV-2 nsp7 protein in search of cryptic pockets</a> by <a href="https://bowmanlab.biochem.wustl.edu" target="_none">The Bowman lab at Washington University in St. Louis</a></a>
            <br>
            
          </li>          <li>
            <a href="https://covid.molssi.org//simulations/#foldinghome-simulations-of-nsp8-nsp8-nil" class="" target="_none">SARS-CoV-2 nsp8 simulations: A 1.8 ms dataset of the SARS-CoV-2 nsp8 protein in search of cryptic pockets</a> by <a href="https://bowmanlab.biochem.wustl.edu" target="_none">The Bowman lab at Washington University in St. Louis</a></a>
            <br>
            
          </li>          <li>
            <a href="https://covid.molssi.org//simulations/#foldinghome-simulations-of-nsp9-nsp9-nil" class="" target="_none">SARS-CoV-2 nsp9 simulations: A 9 ms dataset of the SARS-CoV-2 nsp9 protein in search of cryptic pockets</a> by <a href="https://bowmanlab.biochem.wustl.edu" target="_none">The Bowman lab at Washington University in St. Louis</a></a>
            <br>
            
          </li>          <li>
            <a href="https://covid.molssi.org//simulations/#foldinghome-simulations-of-nsp12-rdrp-nil" class="" target="_none">SARS-CoV-2 RNA polymerase (nsp12, RdRP) dataset: A 3.4 ms dataset of the SARS-CoV-2 nsp12 protein in search of cryptic pockets</a> by <a href="https://bowmanlab.biochem.wustl.edu" target="_none">The Bowman lab at Washington University in St. Louis</a></a>
            <br>
            
          </li>          <li>
            <a href="https://covid.molssi.org//simulations/#foldinghome-simulations-of-the-sars-cov-2-spike-protein-spike-spike-binding" class="" target="_none">SARS-CoV-2 spike protein dataset: A 1.2 ms dataset of the SARS-CoV-2 spike protein in search of cryptic pockets</a> by <a href="https://bowmanlab.biochem.wustl.edu" target="_none">The Bowman lab at Washington University in St. Louis</a></a>
            <br>
            
          </li>          <li>
            <a href="https://covid.molssi.org//simulations/#foldinghome-simulations-of-the-sars-cov-2-spike-rbd-bound-to-human-ace2" class="" target="_none">SARS-CoV-2 spike RBD bound to human ACE2 receptor (173.8 us): Wild-type and mutant simulations</a> by <a href="https://choderalab.org" target="_none">The Chodera lab at the Memorial Sloan Kettering Cancer Center</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1101/2020.06.27.175430" class="" target="_none">SARS-CoV-2 Simulations Go Exascale to Capture Spike Opening and Reveal Cryptic Pockets Across the Proteome</a> by Maxwell I. Zimmerman, Justin R. Porter, Michael D. Ward, Sukrit Singh, Neha Vithani, Artur Meller, Upasana L. Mallimadugula, Catherine E. Kuhn, Jonathan H. Borowsky,  View ORCID ProfileRafal P. Wiewiora, Matthew F. D. Hurley, Aoife M Harbison, Carl A Fogarty, Joseph E. Coffland, Elisa Fadda, Vincent A. Voelz, John D. Chodera, Gregory R. Bowman</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="broad-gnomad" class="dataset">
        <h4><a href="../broad-gnomad/">Genome Aggregation Database (gnomAD)</a></h4>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/broadinstitute/gnomad_qc" class="" target="_none">gnomAD quality control GitHub repository</a> by <a href="https://github.com/orgs/broadinstitute/teams/gnomad/members" target="_none">gnomAD Production Team</a></a>
            <br>
            
          </li>          <li>
            <a href="https://hail.is/" class="" target="_none">Hail</a> by <a href="https://hail.is/" target="_none">Hail Team</a></a>
            <br>
            
          </li>          <li>
            <a href="https://aws.amazon.com/quickstart/architecture/hail/" class="aws-link" target="_none">Hail on AWS Quick Start</a> by Amazon Web Services and PrivoIT</a>
            <br>
            
          </li>          <li>
            <a href="https://pypi.org/project/gnomad/" class="" target="_none">Hail utilities for gnomAD</a> by <a href="https://github.com/orgs/broadinstitute/teams/gnomad/members" target="_none">gnomAD Production Team</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1038/s41586-020-2287-8" class="" target="_none">A structural variation reference for medical and population genetics. Nature 581, 444–451 (2020)</a> by Collins, R. L., Brand, H., Karczewski, K. J., Zhao, X., Alföldi, J., Francioli, L. C., Khera, A. V., Lowther, C., Gauthier, L. D., Wang, H., Watts, N. A., Solomonson, M., O’Donnell-Luria, A., Baumann, A., Munshi, R., Walker, M., Whelan, C., Huang, Y., Brookings, T., ... Talkowski, M. E.</a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1038/nature19057" class="" target="_none">Analysis of protein-coding genetic variation in 60,706 humans. Nature 536, 285–291 (2016)</a> by Lek, M., Karczewski, K., Minikel, E. et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1038/s41467-019-10717-9" class="" target="_none">Characterising the loss-of-function impact of 5’ untranslated region variants in 15,708 individuals. Nature Communications 11, 2523 (2020)</a> by Whiffin, N., Karczewski, K. J., Zhang, X., Chothani, S., Smith, M. J., Gareth Evans, D., Roberts, A. M., Quaife, N. M., Schafer, S., Rackham, O., Alföldi, J., O’Donnell-Luria, A. H., Francioli, L. C., Genome Aggregation Database (gnomAD) Production Team, Genome Aggregation Database (gnomAD) Consortium, Cook, S. A., Barton, P. J. R., MacArthur, D. G., &amp; Ware, J. S.</a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1038/s41586-020-2267-z" class="" target="_none">Evaluating potential drug targets through human loss-of-function genetic variation. Nature 581, 459–464 (2020)</a> by Minikel, E. V., Karczewski, K. J., Martin, H. C., Cummings, B. B., Whiffin, N., Rhodes, D., Alföldi, J., Trembath, R. C., van Heel, D. A., Daly, M. J., Genome Aggregation Database Production Team, Genome Aggregation Database Consortium, Schreiber, S. L., &amp; MacArthur, D. G.</a>
            <br>
            
          </li>          <li>
            <a href="https://gnomad.broadinstitute.org/blog/2018-10-gnomad-v2-1/" class="" target="_none">gnomAD v2.1</a> by Laurent Francioli, Grace Tiao, Konrad Karczewski, Matthew Solomonson, Nick Watts</a>
            <br>
            
          </li>          <li>
            <a href="https://gnomad.broadinstitute.org/blog/2019-10-gnomad-v3-0/" class="" target="_none">gnomAD v3.0</a> by Laurent Francioli, Daniel MacArthur</a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1038/s41467-019-12438-5" class="" target="_none">Landscape of multi-nucleotide variants in 125,748 human exomes and 15,708 genomes. Nature Communications 11, 2539 (2020)</a> by Wang, Q., Pierce-Hoffman, E., Cummings, B. B., Karczewski, K. J., Alföldi, J., Francioli, L. C., Gauthier, L. D., Hill, A. J., O’Donnell-Luria, A. H., Genome Aggregation Database (gnomAD) Production Team, Genome Aggregation Database (gnomAD) Consortium, &amp; MacArthur, D. G.</a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1101/784157" class="" target="_none">Technical artifact drives apparent deviation from Hardy-Weinberg equilibrium at CCR5-∆32 and other variants in gnomAD. bioRxiv (p. 784157)</a> by Karczewski, K. J., Gauthier, L. D., Daly, M. J.</a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1038/s41591-020-0893-5" class="" target="_none">The effect of LRRK2 loss-of-function variants in humans. Nature Medicine (2020)</a> by Whiffin, N., Armean, I. M., Kleinman, A., Marshall, J. L., Minikel, E. V., Goodrich, J. K., Quaife, N. M., Cole, J. B., Wang, Q., Karczewski, K. J., Cummings, B. B., Francioli, L., Laricchia, K., Guan, A., Alipanahi, B., Morrison, P., Baptista, M. A. S., Merchant, K. M., Genome Aggregation Database Production Team, ... MacArthur, D. G.</a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1038/s41586-020-2308-7" class="" target="_none">The mutational constraint spectrum quantified from variation in 141,456 humans. Nature 581, 434–443 (2020)</a> by Karczewski, K. J., Francioli, L. C., Tiao, G., Cummings, B. B., Alföldi, J., Wang, Q., Collins, R. L., Laricchia, K. M., Ganna, A., Birnbaum, D. P., Gauthier, L. D., Brand, H., Solomonson, M., Watts, N. A., Rhodes, D., Singer-Berk, M., England, E. M., Seaby, E. G., Kosmicki, J. A., ... MacArthur, D. G.</a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1038/s41586-020-2329-2" class="" target="_none">Transcript expression-aware annotation improves rare variant interpretation. Nature 581, 452–458 (2020)</a> by Cummings, B. B., Karczewski, K. J., Kosmicki, J. A., Seaby, E. G., Watts, N. A., Singer-Berk, M., Mudge, J. M., Karjalainen, J., Kyle Satterstrom, F., O’Donnell-Luria, A., Poterba, T., Seed, C., Solomonson, M., Alföldi, J., The Genome Aggregation Database Production Team, The Genome Aggregation Database Consortium, Daly, M. J., &amp; MacArthur, D. G.</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="noaa-nexrad" class="dataset">
        <h4><a href="../noaa-nexrad/">NEXRAD on AWS</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://carto.com/blog/mapping-nexrad-radar-data/" class="" target="_none">Mapping Noaa Nexrad Radar Data With CARTO</a> by <a href="https://carto.com/blog/author/stuart-lynn/" target="_none">Stuart Lynn</a></a>
            <br>
            
          </li>          <li>
            <a href="https://github.com/openradar/AMS_radar_in_the_cloud" class="" target="_none">NEXRAD on EC2 tutorial</a> by <a href="https://github.com/openradar" target="_none">openradar</a></a>
            <br>
            
          </li>          <li>
            <a href="http://nbviewer.jupyter.org/gist/dopplershift/356f2e14832e9b676207" class="" target="_none">Using Python to Access NCEI Archived NEXRAD Level 2 Data (Jupyter notebook)</a> by <a href="http://dopplershift.github.io" target="_none">Ryan May</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://blog.mapbox.com/3d-animated-hurricane-map-from-radar-data-affaed8a3d70" class="" target="_none">Into the eye of the storm: NEXRAD Level II open data</a> by Jonni Walker</a>
            <br>
            
          </li>          <li>
            <a href="https://pypi.org/project/nexradaws/" class="" target="_none">nexradaws on pypi.python.org - python module to query and download Nexrad data from Amazon S3</a> by <a href="https://github.com/aarande" target="_none">Aaron Anderson</a></a>
            <br>
            
          </li>          <li>
            <a href="https://github.com/zflamig/ondas-pds" class="" target="_none">Open NEXRAD Data Access for Level 3 with Lambda@Edge</a> by Zac Flamig</a>
            <br>
            
          </li>          <li>
            <a href="https://github.com/stephenlienharrell/WeatherPipe" class="" target="_none">WeatherPipe - Amazon EMR based analysis tool for NEXRAD data stored on Amazon S3</a> by <a href="https://github.com/stephenlienharrell" target="_none">Stephen Lien Harrell</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://www.pnas.org/content/117/6/2987" class="" target="_none">Declines in an abundant aquatic insect, the burrowing mayfly, across major North American waterways</a> by Phillip M. Stepanian, Sally A. Entrekin, Charlotte E. Wainwright, Djordje Mirkovic, Jennifer L. Tank, &amp; Jeffrey F. Kelly</a>
            <br>
            
          </li>          <li>
            <a href="https://agupubs.onlinelibrary.wiley.com/doi/abs/10.1029/2020GL089001" class="" target="_none">Extreme Pyroconvective Updrafts During a Megafire</a> by B. Rodriguez, N. P. Lareau, D. E. Kingsmill, &amp; C. B. Clements</a>
            <br>
            
          </li>          <li>
            <a href="https://www.roc.noaa.gov/wsr88d/PublicDocs/ICDs/2620002R.pdf" class="" target="_none">Level 2 Interface Control Document for Message Data Formats: Build 18</a> by NOAA ROC</a>
            <br>
            
          </li>          <li>
            <a href="https://www.roc.noaa.gov/wsr88d/PublicDocs/ICDs/2620010G.pdf" class="" target="_none">Level 2 Interface Control Document for Transfer: Build 18</a> by NOAA ROC</a>
            <br>
            
          </li>          <li>
            <a href="https://www.roc.noaa.gov/wsr88d/PublicDocs/ICDs/2620001X.pdf" class="" target="_none">Level 3 Interface Control Document for Message Data Formats: Build 18</a> by NOAA ROC</a>
            <br>
            
          </li>          <li>
            <a href="https://www.nature.com/articles/s41559-018-0666-4" class="" target="_none">Seasonal abundance and survival of North America’s migratory avifauna determined by weather radar</a> by Adriaan M. Dokter, Andrew Farnsworth, Daniel Fink, Viviana Ruiz-Gutierrez, Wesley M. Hochachka, Frank A. La Sorte, Orin J. Robinson, Kenneth V. Rosenberg &amp; Steve Kelling</a>
            <br>
            
          </li>          <li>
            <a href="https://journals.ametsoc.org/doi/full/10.1175/BAMS-D-16-0021.1" class="" target="_none">Unlocking the Potential of NEXRAD Data through NOAA’s Big Data Partnership</a> by Steve Ansari and Stephen Del Greco</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="terrain-tiles" class="dataset">
        <h4><a href="../terrain-tiles/">Terrain Tiles</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://gis.stackexchange.com/questions/201831/how-to-efficiently-access-files-with-gdal-from-an-s3-bucket-using-vsis3/202608#202608" class="" target="_none">Using GDAL to Produce a Geotiff for an Area</a> by Chris Henrick</a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://eos.com/landviewer/" class="" target="_none">EOS Land Viewer</a> by <a href="https://eos.com/" target="_none">Earth Observing System</a></a>
            <br>
            
          </li>          <li>
            <a href="https://docs.safe.com/fme/html/FME_Desktop_Documentation/FME_ReadersWriters/terraintilesaws/terraintilesaws.htm" class="" target="_none">Mapzen Terrain Tiles on AWS Reader</a> by <a href="https://www.safe.com/" target="_none">Safe Software</a></a>
            <br>
            
          </li>          <li>
            <a href="https://onthegomap.com/" class="" target="_none">On The Go Map</a> by <a href="https://onthegomap.com/" target="_none">On The Go Map</a></a>
            <br>
            
          </li>          <li>
            <a href="https://elevation.racemap.com" class="" target="_none">Open-Source Elevation Service</a> by <a href="https://racemap.com/" target="_none">Racemap</a></a>
            <br>
            
          </li>          <li>
            <a href="https://github.com/interline-io/planetutils" class="" target="_none">PlanetUtils (GitHub): Scripts and a Docker container to download, merge, and resample terrain tiles</a> by <a href="https://www.interline.io/" target="_none">Interline Technologies</a></a>
            <br>
            
          </li>          <li>
            <a href="https://podpac.org/datasets.html#terraintiles" class="" target="_none">PODPAC: Python Library supporting TerrainTiles for analysis</a> by <a href="http://www.creare.com/" target="_none">Creare</a></a>
            <br>
            
          </li>          <li>
            <a href="https://cran.r-project.org/web/packages/elevatr/vignettes/introduction_to_elevatr.html" class="" target="_none">R package for accessing Terrain Tiles</a> by <a href="https://github.com/jhollist" target="_none">Jeffrey W. Hollister</a></a>
            <br>
            
          </li>          <li>
            <a href="http://www.sentinel-hub.com/apps/wms" class="" target="_none">Sentinel Hub WMS/WMTS/WCS Service for DEM</a> by <a href="http://www.sinergise.com/" target="_none">Sinergise</a></a>
            <br>
            
          </li>          <li>
            <a href="https://apps.sentinel-hub.com/sentinel-playground/?source&#x3D;DEM&amp;lat&#x3D;41.902277040963696&amp;lng&#x3D;-102.83203125&amp;zoom&#x3D;4" class="" target="_none">Sentinel Playground for DEM</a> by <a href="http://www.sinergise.com/" target="_none">Sinergise</a></a>
            <br>
            
          </li>          <li>
            <a href="https://terradactile.sparkgeo.com/" class="" target="_none">Terradactile: Generate DEM and hillshade Cloud Optimized GeoTIFFs for visualization and download based on terrain tiles</a> by <a href="https://sparkgeo.com/" target="_none">Sparkgeo</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://ieeexplore.ieee.org/abstract/document/9298063" class="" target="_none">Interactive Visualization of 3D Terrain Data Stored in the Cloud</a> by Gregory Larrick, Yun Tian, Uri Rogers, Halim Acosta, and Fangyang Shen</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="allen-cell-imaging-collections" class="dataset">
        <h4><a href="../allen-cell-imaging-collections/">Allen Cell Imaging Collections</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/AllenCellModeling/pytorch_fnet/blob/master/examples/download_and_train.py" class="" target="_none">Download and train label-free models</a> by <a href="https://www.alleninstitute.org/what-we-do/cell-science/about/team/staff-profiles/greg-johnson/" target="_none">Greg Johnson</a></a>
            <br>
            
          </li>          <li>
            <a href="https://drive.google.com/file/d/1nuiYOurmbic7rZeHPen_M3fDbsN7n5Ws/view?usp&#x3D;sharing" class="" target="_none">Integrating single-cell sequencing and nuclear imaging data</a> by <a href="https://noble.gs.washington.edu/" target="_none">Chengxiang Qiu and William Noble</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/AllenInstitute/volume-viewer" class="" target="_none">AICS Volume Viewer</a> by <a href="https://alleninstitute.org/what-we-do/cell-science/about/team/staff-profiles/daniel-toloudis/" target="_none">Dan Toloudis</a></a>
            <br>
            
          </li>          <li>
            <a href="https://github.com/AllenCellModeling/aicsimageio" class="" target="_none">AICSImageIO</a> by <a href="https://www.alleninstitute.org/what-we-do/cell-science/about/team/" target="_none">Matthew Bowden, Jackson Brown, Jamie Sherman, Dan Toloudis</a></a>
            <br>
            
          </li>          <li>
            <a href="https://cfe.allencell.org/" class="" target="_none">Allen Cell Feature Explorer</a> by <a href="https://www.alleninstitute.org/what-we-do/cell-science/about/team/" target="_none">Allen Institute for Cell Science</a></a>
            <br>
            
          </li>          <li>
            <a href="https://www.allencell.org/segmenter.html" class="" target="_none">Allen Cell Structure Segmenter</a> by <a href="https://www.alleninstitute.org/what-we-do/cell-science/about/team/" target="_none">Jianxu Chen, Liya Ding, Matheus P. Viana, Melissa C. Hendershott, Ruian Yang, Irina A. Mueller, Susanne M. Rafelski</a></a>
            <br>
            
          </li>          <li>
            <a href="https://imsc.allencell.org" class="" target="_none">Integrated Mitotic Stem Cell</a> by <a href="https://www.alleninstitute.org/what-we-do/cell-science/about/team/" target="_none">Allen Institute for Cell Science</a></a>
            <br>
            
          </li>          <li>
            <a href="github.com/AllenCellModeling/pytorch_integrated_cell" class="" target="_none">Pytorch 3D Integrated Cell</a> by <a href="https://www.alleninstitute.org/what-we-do/cell-science/about/team/" target="_none">Gregory R. Johnson, Rory M. Donovan-Maiye, Mary M. Maleckar</a></a>
            <br>
            
          </li>          <li>
            <a href="https://www.allencell.org/visual-guide-to-human-cells.html" class="" target="_none">Visual Guide to Human Cells</a> by <a href="https://www.alleninstitute.org/what-we-do/cell-science/about/team/" target="_none">Allen Institute for Cell Science</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://www.nature.com/articles/s41592-018-0111-2" class="" target="_none">Label-free prediction of three-dimensional fluorescence images from transmitted-light microscopy</a> by <a href="https://alleninstitute.org/about/team/" target="_none">Chawin Ounkomol, Sharmishtaa Seshamani, Mary M. Maleckar, Forrest Collman &amp; Gregory R. Johnson</a></a>
            <br>
            
          </li>          <li>
            <a href="https://www.biorxiv.org/content/10.1101/2020.12.08.415562v1" class="" target="_none">Robust integrated intracellular organization of the human iPS cell: where, how much, and how variable?</a> by <a href="https://www.alleninstitute.org/what-we-do/cell-science/about/team/" target="_none">Matheus P. Viana, Jianxu Chen*, Theo A. Knijnenburg*, Ritvik Vasan*, Calysta Yan*... Allen Institute for Cell Science... Graham T. Johnson, Ruwanthi N. Gunawardane, Nathalie Gaudreault, Julie A. Theriot, Susanne M. Rafelski</a></a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="cbers" class="dataset">
        <h4><a href="../cbers/">CBERS on AWS</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://aws.amazon.com/blogs/publicsector/keeping-a-spatiotemporal-asset-catalog-stac-up-to-date-with-sns-sqs/" class="aws-link" target="_none">Keeping a SpatioTemporal Asset Catalog (STAC) Up To Date with SNS/SQS</a> by Frederico Liporace</a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/RemotePixel/aws-sat-api-py" class="" target="_none">aws-sat-api-py</a> by <a href="http://remotepixel.ca/" target="_none">Remote Pixel</a></a>
            <br>
            
          </li>          <li>
            <a href="https://cbers.stac.cloud" class="" target="_none">CBERS static STAC catalog served by stac-browser</a> by <a href="https://github.com/radiantearth/stac-browser" target="_none">Radiant Earth</a></a>
            <br>
            
          </li>          <li>
            <a href="https://github.com/fredliporace/cbersgif" class="" target="_none">CBERS timelapse GIF generator</a> by <a href="https://github.com/fredliporace" target="_none">Frederico Liporace</a></a>
            <br>
            
          </li>          <li>
            <a href="https://github.com/mapbox/cbers-tiler" class="" target="_none">cbers-tiler</a> by <a href="https://www.mapbox.com/" target="_none">Mapbox</a></a>
            <br>
            
          </li>          <li>
            <a href="https://eos.com/landviewer/" class="" target="_none">EOS Land Viewer</a> by <a href="https://eos.com/" target="_none">Earth Observing System</a></a>
            <br>
            
          </li>          <li>
            <a href="http://brazildatacube.dpi.inpe.br/forest-monitor/" class="" target="_none">Forest Monitor</a> by <a href="http://brazildatacube.org/" target="_none">Brazil Datacube, INPE</a></a>
            <br>
            
          </li>          <li>
            <a href="https://viewer.remotepixel.ca" class="" target="_none">Remote Pixel Viewer</a> by <a href="http://remotepixel.ca/" target="_none">Remote Pixel</a></a>
            <br>
            
          </li>          <li>
            <a href="https://github.com/mapbox/rio-tiler" class="" target="_none">rio-tiler</a> by <a href="https://www.mapbox.com/" target="_none">Mapbox</a></a>
            <br>
            
          </li>          <li>
            <a href="https://stac.amskepler.com/v10/stac/search" class="" target="_none">STAC V1.0 search endpoint for archive</a> by <a href="https://github.com/fredliporace/cbers-2-stac" target="_none">AMS Kepler</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://ieeexplore.ieee.org/abstract/document/9165649" class="" target="_none">Using Remote Sensing Images and Cloud Services on AWS to Improve Land Use and Cover Monitoring</a> by K. R. Ferreira, et al.</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="fcp-indi" class="dataset">
        <h4><a href="../fcp-indi/">International Neuroimaging Data-Sharing Initiative (INDI)</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://fcon_1000.projects.nitrc.org/indi/s3/index.html" class="" target="_none">Downloading FCP-INDI Neuroimaging Data from Amazon S3</a> by INDI</a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://fcp-indi.github.io/docs/user/index.html#" class="" target="_none">Configurable Pipeline for the Analysis of Connectomes (C-PAC)</a> by [INDI C-PAC Team](https://fcp-indi.github.io/)</a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://www.sciencedirect.com/science/article/pii/S089662731931089X" class="" target="_none">Accelerating the Evolution of Nonhuman Primate Neuroimaging</a> by M.P. Milham, C. Petkov</a>
            <br>
            
          </li>          <li>
            <a href="https://www.sciencedirect.com/science/article/pii/S0896627318307682" class="" target="_none">An Open Resource for Non-human Primate Imaging</a> by M.P. Milham, L. Ai, ..., C.E. Schroeder</a>
            <br>
            
          </li>          <li>
            <a href="https://www.nature.com/articles/sdata2017181" class="" target="_none">An open resource for transdiagnostic research in pediatric mental health and learning disorders</a> by L.M. Alexander, J. Escalera, ..., M.P. Milham</a>
            <br>
            
          </li>          <li>
            <a href="https://www.nature.com/articles/s41467-018-04976-1" class="" target="_none">Assessment of the impact of shared brain imaging data on the scientific literature</a> by M.P. Milham, R.C. Craddock, ..., A. Klein</a>
            <br>
            
          </li>          <li>
            <a href="https://www.nature.com/articles/sdata201710" class="" target="_none">Enhancing studies of the connectome in autism using the autism brain imaging data exchange II.</a> by A. Di Martino, D. O&#x27;Connor, M.P. Milham</a>
            <br>
            
          </li>          <li>
            <a href="https://www.sciencedirect.com/science/article/abs/pii/S1053811912010671?via%3Dihub" class="" target="_none">Making data sharing work: The FCP/INDI experience</a> by M. Mennes, B.B. Biswal, F.X. Castellanos, M.P. Milham</a>
            <br>
            
          </li>          <li>
            <a href="https://www.nature.com/articles/mp201378" class="" target="_none">The autism brain imaging data exchange: towards a large-scale evaluation of the intrinsic brain architecture in autism.</a> by A. Di Martino, C-G Yan, ..., M.P. Milham</a>
            <br>
            
          </li>          <li>
            <a href="https://academic.oup.com/gigascience/article/6/2/giw011/2865212" class="" target="_none">The Healthy Brain Network Serial Scanning Initiative: a resource for evaluating inter-individual differences and their reliabilities across scan conditions and sessions</a> by D. O&#x27;Connor, N.V. Potler, ..., M.P. Milham</a>
            <br>
            
          </li>          <li>
            <a href="https://www.frontiersin.org/articles/10.3389/fnins.2012.00152/full" class="" target="_none">The NKI-Rockland sample: a model for accelerating the pace of discovery science in psychiatry</a> by K.B. Nooner, S.J. Colcombe, ..., M.P. Milham</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="spacenet" class="dataset">
        <h4><a href="../spacenet/">SpaceNet</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://aws.amazon.com/blogs/machine-learning/extracting-buildings-and-roads-from-aws-open-data-using-amazon-sagemaker/" class="aws-link" target="_none">Extracting buildings and roads from AWS Open Data using Amazon SageMaker</a> by Yunzhi Shi, Tianyu Zhang, and Xin Chen</a>
            <br>
            <span class="label label-info tag service-tag">SageMaker</span>
          </li>          <li>
            <a href="https://medium.com/the-downlinq/getting-started-with-spacenet-data-827fd2ec9f53" class="" target="_none">Getting Started with SpaceNet Data</a> by <a href="https://medium.com/@avanetten" target="_none">Adam Van Etten</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://medium.com/the-downlinq/announcing-solaris-an-open-source-python-library-for-analyzing-overhead-imagery-with-machine-48c1489c29f7" class="" target="_none">Solaris: an open source Python library for analyzing overhead imagery with machine learning</a> by Nick Weir</a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://medium.com/the-downlinq/creating-training-datasets-for-the-spacenet-road-detection-and-routing-challenge-6f970d413e2f" class="" target="_none">Creating Training Datasets for the SpaceNet Road Detection and Routing Challenge</a> by <a href="https://medium.com/@avanetten" target="_none">Adam Van Etten and Jake Shermeyer</a></a>
            <br>
            
          </li>          <li>
            <a href="https://medium.com/the-downlinq/deploying-the-spacenet-6-baseline-on-aws-c811ad82da1" class="" target="_none">Deploying the SpaceNet 6 Baseline on AWS</a> by <a href="https://medium.com/@avanetten" target="_none">Adam Van Etten and Nick Weir</a></a>
            <br>
            
          </li>          <li>
            <a href="https://medium.com/the-downlinq/introducing-the-spacenet-off-nadir-imagery-and-buildings-dataset-e4a3c1cb4ce3" class="" target="_none">Introducing the SpaceNet Off-Nadir Imagery Dataset</a> by <a href="https://medium.com/@david.lindenbaum" target="_none">David Lindenbaum</a></a>
            <br>
            
          </li>          <li>
            <a href="https://medium.com/the-downlinq/introducing-the-spacenet-road-detection-and-routing-challenge-and-dataset-7604de39b779" class="" target="_none">Introducing the SpaceNet Road Detection and Routing Challenge and Dataset</a> by <a href="https://medium.com/@david.lindenbaum" target="_none">David Lindenbaum</a></a>
            <br>
            
          </li>          <li>
            <a href="https://medium.com/the-downlinq/spacenet-5-dataset-release-80bd82d7c528" class="" target="_none">SpaceNet 5 Dataset Release</a> by <a href="https://medium.com/@avanetten" target="_none">Adam Van Etten and Ryan Lewis</a></a>
            <br>
            
          </li>          <li>
            <a href="https://medium.com/the-downlinq/spacenet-6-dataset-release-66076c8fb79b" class="" target="_none">SpaceNet 6: Dataset Release</a> by <a href="https://medium.com/@jshermeyer" target="_none">Jake Shermeyer</a></a>
            <br>
            
          </li>          <li>
            <a href="https://medium.com/the-downlinq/spacenet-winning-implementations-and-new-imagery-release-55f738e14253" class="" target="_none">SpaceNet: Winning Implementations and New Imagery Release</a> by <a href="https://medium.com/@toddstavish" target="_none">Todd Stavish</a></a>
            <br>
            
          </li>          <li>
            <a href="https://medium.com/the-downlinq/the-spacenet-7-multi-temporal-urban-development-challenge-dataset-release-9e6e5f65c8d5" class="" target="_none">The SpaceNet 7 Multi-Temporal Urban Development Challenge: Dataset Release</a> by <a href="https://medium.com/@avanetten" target="_none">Adam Van Etten</a></a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="janelia-flylight" class="dataset">
        <h4><a href="../janelia-flylight/">Fly Brain Anatomy: FlyLight Gen1 and Split-GAL4 Imagery</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/JaneliaSciComp/open-data-flylight/blob/master/tutorials/File_operations_on_AWS_S3.ipynb" class="" target="_none">File Operations on AWS S3</a> by Rob Svirskas</a>
            <br>
            
          </li>          <li>
            <a href="https://github.com/JaneliaSciComp/open-data-flylight/blob/master/tutorials/Using_imagery_on_AWS_S3.ipynb" class="" target="_none">Using Imagery on AWS S3</a> by Rob Svirskas</a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/JaneliaSciComp/ColorMIP_Mask_Search" class="" target="_none">Color Depth Search Fiji Plugin</a> by Hideo Otsuna</a>
            <br>
            
          </li>          <li>
            <a href="http://splitgal4.janelia.org/cgi-bin/splitgal4.cgi" class="" target="_none">Fly Light Split-GAL4 Driver Collection</a> by Rob Svirskas</a>
            <br>
            
          </li>          <li>
            <a href="https://www.janelia.org/project-team/flylight" class="" target="_none">FlyLight Project Website</a> by Geoffrey Meissner</a>
            <br>
            
          </li>          <li>
            <a href="https://neuronbridge.janelia.org" class="" target="_none">NeuronBridge</a> by Jody Clements, Rob Svirskas, Hideo Otsuna, Cristian Goina, Konrad Rokicki</a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1016/j.celrep.2012.09.011" class="" target="_none">A GAL4-Driver Line Resource for Drosophila Neurobiology</a> by Jenett, A., Rubin, G. M., Ngo, T., Shepherd, D., Murphy, C., Dionne, H., Pfeiffer, B. D., Cavallaro, A., Hall, D., Jeter, J., Iyer, N., Fetter, D., Hausenfluck, J. H., Peng, H., Trautman, E. T., Svirskas, R. R., Myers, E. W., Iwinski, Z. R., Aso, Y., DePasquale, G. M., Enos, A., Hulamm, P., Lam, S. C. B., Li, H., Laverty, T. R., Long, F., Qu, L., Murphy, S. D., Rokicki, K., Safford, T., Shaw, K., Simpson, J. H., Sowell, A., Tae, S., Yu, Y., &amp; Zugates, C. T.</a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1101/376384" class="" target="_none">An unbiased template of the Drosophila brain and ventral nerve cord</a> by Bogovic, J.A., Otsuna, H., Heinrich, L., Ito, M., Jeter, J., Meissner, G., Nern, A., Colonell, J., Malkesman, O., Ito, K., &amp; Saalfeld, S.</a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1101/318006" class="" target="_none">Color depth MIP mask search: a new tool to expedite Split-GAL4 creation</a> by Otsuna, H., Ito, M., &amp; Kawase, T.</a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.7554/eLife.04577" class="" target="_none">The neuronal architecture of the mushroom body provides a logic for associative learning</a> by Aso, Y., Hattori, D., Yu, Y., Johnston, R. M., Iyer, N. A., Ngo, T., Dionne, H., Abbott, L., Axel, R., Tanimoto, H., &amp; Rubin, G. M.</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="irs990" class="dataset">
        <h4><a href="../irs990/">IRS 990 Filings</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://medium.com/@open990/the-irs-990-e-file-dataset-getting-to-the-chocolatey-center-of-data-deliciousness-90f66097a600" class="" target="_none">Tutorial on using the IRS 990 e-file dataset</a> by <a href="https://www.990consulting.com/" target="_none">990 Consulting, LLC</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/CharityNavigator/990_long" class="" target="_none">990_long</a> by <a href="https://www.charitynavigator.org/" target="_none">Charity Navigator</a></a>
            <br>
            
          </li>          <li>
            <a href="http://irs-990-explorer.chrisgherbert.com" class="" target="_none">aws-irs-990-explorer</a> by <a href="http://chrisgherbert.com/" target="_none">Chris Herbert</a></a>
            <br>
            
          </li>          <li>
            <a href="https://www.grantmakers.io" class="" target="_none">Grantmakers.io</a> by <a href="https://www.chadkruse.com/" target="_none">Chad Kruse</a></a>
            <br>
            
          </li>          <li>
            <a href="https://lecy.github.io/Open-Data-for-Nonprofit-Research/" class="" target="_none">Guide to Open Data for Nonprofit Research</a> by <a href="https://github.com/lecy" target="_none">lecy</a></a>
            <br>
            
          </li>          <li>
            <a href="http://nonprofitlight.com" class="" target="_none">Non Profit Light</a> by Non Profit Light</a>
            <br>
            
          </li>          <li>
            <a href="https://projects.propublica.org/nonprofits/" class="" target="_none">Nonprofit Explorer</a> by <a href="https://propublica.org" target="_none">ProPublica</a></a>
            <br>
            
          </li>          <li>
            <a href="https://github.com/ma-ji/npo_classifier" class="" target="_none">npo_classifier: Automated coding using machine-learning and remapping the U.S. nonprofit sector</a> by <a href="https://jima.me/" target="_none">Ji Ma</a></a>
            <br>
            
          </li>          <li>
            <a href="https://www.open990.com/" class="" target="_none">Open990</a> by <a href="https://www.990consulting.com/" target="_none">990 Consulting, LLC</a></a>
            <br>
            
          </li>          <li>
            <a href="https://github.com/jsfenfen/990-xml-reader" class="" target="_none">Parse 990 XML using IRSx</a> by <a href="http://jacobfenton.com/" target="_none">Jacob Fenton</a></a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="noaa-goes" class="dataset">
        <h4><a href="../noaa-goes/">NOAA Geostationary Operational Environmental Satellites (GOES) 16 &amp; 17</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://nbviewer.jupyter.org/github/oceanhackweek/ohw20-tutorials/blob/master/10-satellite-data-access/goes-cmp-netcdf-zarr.ipynb" class="" target="_none">Comparison reading GOES-R data from AWS S3 in netCDF versus zarr</a> by <a href="http://www.faralloninstitute.org/chelle" target="_none">Chelle Gentemann</a></a>
            <br>
            
          </li>          <li>
            <a href="https://github.com/awslabs/amazon-asdi/tree/master/examples/goes16-precip-julia" class="aws-link" target="_none">NOAA GOES16 Julia Jupyter Notebook Example</a> by Peter Schmiedeskamp</a>
            <br>
            
          </li>          <li>
            <a href="https://nbviewer.jupyter.org/github/awslabs/amazon-asdi/blob/main/examples/dask/notebooks/goes16_dask.ipynb" class="aws-link" target="_none">Processing GOES-16 data with Dask &amp; AWS Fargate</a> by Zac Flamig</a>
            <br>
            
          </li>          <li>
            <a href="https://github.com/HamedAlemo/visualize-goes16" class="" target="_none">Visualize GOES-16 in Python using Xarray</a> by <a href="https://www.linkedin.com/in/hamedalemohammad/" target="_none">Hamed Alemohammad</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/blaylockbk/goes2go" class="" target="_none">GOES-2-go is a python package for downloading and plotting plotting GOES data.</a> by <a href="https://www.linkedin.com/in/blaylockbk/" target="_none">Brian K. Blaylock</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://www.nationalgeographic.com/magazine/2018/03/bird-migration-interactive-maps/" class="" target="_none">Billions of Birds Migrate. Where Do They Go?</a> by National Geographic</a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1175/WAF-D-19-0141.1" class="" target="_none">Comparison of Lightning Forecasts from the High-Resolution Rapid Refresh Model to Geostationary Lightning Mapper Observations</a> by Brian K. Blaylock, and John D. Horel</a>
            <br>
            
          </li>          <li>
            <a href="https://aws.amazon.com/blogs/publicsector/embracing-the-cloud-for-climate-research/" class="aws-link" target="_none">Embracing the cloud for climate research</a> by North Carolina State University’s North Carolina Institute for Climate Studies</a>
            <br>
            
          </li>          <li>
            <a href="https://www.nature.com/articles/s41598-020-74576-x" class="" target="_none">Observations of lightning in relation to transitions in volcanic activity during the 3 June 2018 Fuego Eruption</a> by Christopher J. Schultz, Virginia P. Andrews, Kimberly D. Genareau, and Aaron R. Naeger</a>
            <br>
            
          </li>          <li>
            <a href="https://aws.amazon.com/blogs/publicsector/solcast-solar-irradiance-forecasting-for-the-solar-powered-future/" class="aws-link" target="_none">Solar irradiance forecasting for the solar powered future</a> by Solcast</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="mur" class="dataset">
        <h4><a href="../mur/">Multi-Scale Ultra High Resolution (MUR) Sea Surface Temperature (SST)</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/pangeo-gallery/osm2020tutorial" class="" target="_none">Python Jupyter Notebooks</a> by <a href="https://faralloninstitute.org" target="_none">Chelle Gentemann, Rich Signell</a></a>
            <br>
            
          </li>          <li>
            <a href="https://podaac-tools.jpl.nasa.gov/drive/files/allData/ghrsst/sw/generic_nc_readers/python/" class="" target="_none">Python Reader Software</a> by <a href="https://podaac.jpl.nasa.gov" target="_none">PO.DAAC</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://podaac-tools.jpl.nasa.gov/drive/files/allData/ghrsst/data/GDS2/L4/GLOB/JPL/MUR/v4.1" class="" target="_none">HTTPS server</a> by <a href="https://podaac.jpl.nasa.gov" target="_none">PO.DAAC</a></a>
            <br>
            
          </li>          <li>
            <a href="https://opendap.jpl.nasa.gov/opendap/OceanTemperature/ghrsst/data/GDS2/L4/GLOB/JPL/MUR/v4.1/contents.html" class="" target="_none">OPeNDAP server</a> by <a href="https://podaac.jpl.nasa.gov" target="_none">Physical Oceanography Distributed Active Archive Center (PO.DAAC)</a></a>
            <br>
            
          </li>          <li>
            <a href="https://podaac-tools.jpl.nasa.gov/soto" class="" target="_none">State of the Ocean (SOTO) server</a> by <a href="https://podaac.jpl.nasa.gov" target="_none">PO.DAAC</a></a>
            <br>
            
          </li>          <li>
            <a href="http://thredds.jpl.nasa.gov/thredds/catalog_ghrsst_gds2.html?dataset&#x3D;MUR-JPL-L4-GLOB-v4.1" class="" target="_none">THREDDS server</a> by <a href="https://podaac.jpl.nasa.gov" target="_none">PO.DAAC</a></a>
            <br>
            
          </li>          <li>
            <a href="https://podaac.jpl.nasa.gov/ws/search/granule/?datasetId&#x3D;PODAAC-GHGMR-4FJ04" class="" target="_none">Web discovery service</a> by <a href="https://podaac.jpl.nasa.gov" target="_none">PO.DAAC</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1016/j.rse.2017.07.029" class="" target="_none">A multi-scale high-resolution analysis of global sea surface temperature</a> by Chin, Toshio Michael, Jorge Vazquez-Cuervo, and Edward M. Armstrong</a>
            <br>
            
          </li>          <li>
            <a href="https://podaac-tools.jpl.nasa.gov/drive/files/OceanTemperature/ghrsst/docs/GDS20r5.pdf" class="" target="_none">GHRSST Data Processing Specification</a> by GHRSST Project</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="open-neurodata" class="dataset">
        <h4><a href="../open-neurodata/">Open NeuroData</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://neurodata.io/help/download/" class="" target="_none">Download</a> by <a href="https://github.com/falkben" target="_none">Benjamin Falk</a></a>
            <br>
            
          </li>          <li>
            <a href="https://neurodata.io/help/visualization/" class="" target="_none">Visualization using Neuroglancer</a> by <a href="https://github.com/falkben" target="_none">Benjamin Falk</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/seung-lab/cloud-volume" class="" target="_none">CloudVolume</a> by <a href="https://github.com/william-silversmith" target="_none">William Silversmith</a></a>
            <br>
            
          </li>          <li>
            <a href="https://github.com/seung-lab/igneous" class="" target="_none">Igneous</a> by <a href="https://github.com/william-silversmith" target="_none">William Silversmith</a></a>
            <br>
            
          </li>          <li>
            <a href="https://github.com/google/neuroglancer" class="" target="_none">Neuroglancer</a> by <a href="https://github.com/jbms" target="_none">Jeremy Maitin-Shepard</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://www.nature.com/articles/s41592-018-0181-1" class="" target="_none">A Community-Developed Open-Source Computational Ecosystem for Big Neuro Data</a> by J. T. Vogelstein, E. Perlman, B. Falk, A. Baden, W. Gray Roncal, V. Chandrashekhar, F. Collman, S. Seshamani, J. L. Patsolic, K. Lillaney, M. Kazhdan, R. Hider, D. Pryor, J. Matelsky, T. Gion, P. Manavalan, B. Wester, M. Chevillet, E. T. Trautman, K. Khairy, E. Bridgeford, D. M. Kleissas, D. J. Tward, A. K. Crow, B. Hsueh, M. A. Wright, M. I. Miller, S. J. Smith, R. J. Vogelstein, K. Deisseroth, and R. Burns</a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1016/j.neuron.2014.08.045" class="" target="_none">From cosmos to connectomes: The evolution of data-intensive science</a> by R. Burns, J. T. Vogelstein, and A. S. Szalay</a>
            <br>
            
          </li>          <li>
            <a href="https://arxiv.org/abs/1306.3543" class="" target="_none">The Open Connectome Project Data Cluster: Scalable Analysis and Vision for High-Throughput Neuroscience</a> by R. Burns, W. G. Roncal, D. Kleissas, K. Lillaney, P. Manavalan, E. Perlman, D. R. Berger, D. D. Bock, K. Chung, L. Grosenick, N. Kasthuri, N. C. Weiler, K. Deisseroth, M. Kazhdan, J. Lichtman, R. C. Reid, S. J. Smith, A. S. Szalay, J. T. Vogelstein, and R. J. Vogelstein.</a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1016/j.neuron.2016.10.033" class="" target="_none">To the Cloud! A Grassroots Proposal to Accelerate Brain Science Discovery</a> by J. T. Vogelstein, B. Mensh, M. Häusser, N. Spruston, A. C. Evans, K. Kording, K. Amunts, C. Ebell, J. Muller, M. Telefont, S. Hill, S. P. Koushika, C. Calì, P. A. Valdés-Sosa, P. B. Littlewood, C. Koch, S. Saalfeld, A. Kepecs, H. Peng, Y. O. Halchenko, G. Kiar, M. M. Poo, J. B. Poline, M. P. Milham, A. P. Schaffer, R. Gidron, H. Okano, V. D. Calhoun, M. Chun, D. M. Kleissas, R. J. Vogelstein, E. Perlman, R. Burns, R. Huganir, and M. I. Miller</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="ccle" class="dataset">
        <h4><a href="../ccle/">Cancer Cell Line Encyclopedia (CCLE)</a></h4>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://portals.broadinstitute.org/ccle" class="" target="_none">Broad Institute Cancer Cell Line Encyclopedia</a> by <a href="https://www.broadinstitute.org/" target="_none">The Broad Institute of MIT &amp; Harvard</a></a>
            <br>
            
          </li>          <li>
            <a href="https://cgc-accounts.sbgenomics.com/" class="" target="_none">Cancer Genomics Cloud</a> by <a href="https://docs.sevenbridges.com/docs/ccle-data" target="_none">Seven Bridges</a></a>
            <br>
            
          </li>          <li>
            <a href="https://portal.gdc.cancer.gov/legacy-archive/" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></a>
            <br>
            
          </li>          <li>
            <a href="https://console.cloud.google.com/bigquery?p&#x3D;isb-cgc&amp;d&#x3D;CCLE_bioclin_v0&amp;page&#x3D;dataset" class="" target="_none">ISB CGC BigQuery tables</a> by <a href="https://isb-cancer-genomics-cloud.readthedocs.io/en/latest/sections/data/CCLE_top.html
          " target="_none">Institute for Systems Biology</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1038/s41586-019-1186-3" class="" target="_none">Next-generation characterization of the Cancer Cell Line Encyclopedia</a> by Ghandi, M., Huang F. et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1038/nature15736" class="" target="_none">Pharmacogenomic agreement between two cancer cell line data sets</a> by The Cancer Cell Line Encyclopedia Consortium &amp; The Genomics of Drug Sensitivity in Cancer
          Consortium
          </a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1038/nature11003" class="" target="_none">The Cancer Cell Line Encyclopedia enables predictive modelling of anticancer drug
          sensitivity
          </a> by Barretina Caponigro Stransky et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1038/s41591-019-0404-8" class="" target="_none">The landscape of cancer cell line metabolism</a> by Li, H. et al.</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="ncei-wcsd-archive" class="dataset">
        <h4><a href="../ncei-wcsd-archive/">NOAA Water-Column Sonar Data Archive</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/CI-CMG/pyEcholab/tree/master/examples/cloud%20tutorials/Frequency_Differencing_With_Raw_Data.ipynb" class="" target="_none">Frequency Differencing with Raw Data</a> by Carrie Wall</a>
            <br>
            
          </li>          <li>
            <a href="https://github.com/CI-CMG/pyEcholab/tree/master/examples/cloud%20tutorials/Plotting_Raw_EK60_Data.ipynb" class="" target="_none">Plotting Raw EK60 Data</a> by Carrie Wall</a>
            <br>
            
          </li>          <li>
            <a href="https://github.com/CI-CMG/pyEcholab/tree/master/examples/cloud%20tutorials/Reading_And_Plotting_Bottom_Data.ipynb" class="" target="_none">Reading and Plotting Bottom Data</a> by Carrie Wall</a>
            <br>
            
          </li>          <li>
            <a href="https://github.com/CI-CMG/pyEcholab/tree/master/examples/cloud%20tutorials/Reading_And_Plotting_Processed_CSV_Data.ipynb" class="" target="_none">Reading and Plotting Processed CSV Data</a> by Carrie Wall</a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/CI-CMG/pyEcholab" class="" target="_none">pyEcholab - an open-source, python-based toolkit for reading, processing, plotting, and exporting fisheries acoustic echosounder data</a> by Rick Towler, Chuck Anderson, Veronica Martinez, Pamme Crandell</a>
            <br>
            
          </li>          <li>
            <a href="https://www.ngdc.noaa.gov/maps/water_column_sonar/index.html" class="" target="_none">Water Column Sonar Data Explorer - Map viewer for exploring sonar water column data</a> by NCEI, NOAA</a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://eos.org/science-updates/building-an-accessible-archive-for-water-column-sonar-data" class="" target="_none">Building an Accessible Archive for Water Column Sonar Data</a> by Carrie Wall</a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1093/icesjms/fsw014" class="" target="_none">Increasing the accessibility of acoustic data through global access and imagery</a> by Carrie Wall, Michael Jech, and Susan McLean</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="sentinel-2-l2a-cogs" class="dataset">
        <h4><a href="../sentinel-2-l2a-cogs/">Sentinel-2 Cloud-Optimized GeoTIFFs</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/developmentseed/titiler/blob/master/docs/examples/Create_CustomSentinel2Tiler.ipynb" class="" target="_none">Create a Dynamic Tiler with TiTiler</a> by Vincent Sarago</a>
            <br>
            
          </li>          <li>
            <a href="https://github.com/intake/intake-stac/blob/master/examples/aws-earth-search.ipynb" class="" target="_none">Intake-STAC with sat-search</a> by Scott Henderson</a>
            <br>
            
          </li>          <li>
            <a href="http://www.acgeospatial.co.uk/stac-cog-python-and-qgis/" class="" target="_none">STAC, COG, Python and QGIS</a> by Andrew Cutts</a>
            <br>
            
          </li>          <li>
            <a href="https://github.com/opendatacube/cube-in-a-box" class="" target="_none">Use Sentinel-2 data in the Open Data Cube</a> by Alex Leith</a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/cogeotiff/rio-tiler-pds" class="" target="_none">rio-tiler-pds</a> by Vincent Sarago, et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://github.com/sat-utils/sat-search" class="" target="_none">Sat-search</a> by Matthew Hanson</a>
            <br>
            
          </li>          <li>
            <a href="https://stacindex.org/" class="" target="_none">SpatioTemporal Asset Catalogs</a> by STAC Contriubutors</a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://docs.google.com/presentation/d/14NsKFZ3UF2Swwx_9L7sPMX9ccFUK1ruQyZXWK9Cz4L4/edit?usp&#x3D;sharing" class="" target="_none">STAC and Sentinel-2 COGs (ESIP Summer Meeting 2020)</a> by Matthew Hanson</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="wb-light-every-night" class="dataset">
        <h4><a href="../wb-light-every-night/">World Bank - Light Every Night</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://worldbank.github.io/OpenNightLights/welcome.html" class="" target="_none">Open Nighttime Lights</a> by <a href="https://worldbank.github.io/OpenNightLights/welcome.html" target="_none">Daynan Crull, Trevor Monroe</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="http://www-personal.umich.edu/~brianmin/HREA/" class="" target="_none">High Resolution Electricity Access Indicators (HREA) - Settlement-level measures of electricity access, reliability, and usage.</a> by <a href="http://www-personal.umich.edu/~brianmin/" target="_none">Brian Min, Zachary O&#x27;Keeffe</a></a>
            <br>
            
          </li>          <li>
            <a href="http://nightlights.io" class="" target="_none">Twenty Years of India Lights</a> by <a href="http://nightlights.io" target="_none">Kwawu Mensan Gaba, Brian Min, Anand Thakker, Christopher Elvidge</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://openknowledge.worldbank.org/handle/10986/16192" class="" target="_none">Detection of Rural Electrification in Africa using DMSP-OLS Night Lights Imagery. International Journal of Remote Sensing</a> by Brian Min, Kwawu Mensan Gaba, Ousmane Fall Sarr, Alassane Agalassou.</a>
            <br>
            
          </li>          <li>
            <a href="http://documents.worldbank.org/curated/en/305771562750007469/Mainstreaming-Disruptive-Technologies-in-Energy" class="" target="_none">Mainstreaming Disruptive Technologies in Energy. World Bank Report. 2019</a> by Kwawu Mensan Gaba, Brian Min, Olaf Veerman, Kimberly Baugh</a>
            <br>
            
          </li>          <li>
            <a href="https://www.asprs.org/wp-content/uploads/pers/97journal/june/1997_jun_727-734.pdf" class="" target="_none">Mapping city lights with nighttime data from the DMSP Operational Linescan System. Photogrammetric Engineering and Remote Sensing, 63(6)727-734.</a> by Elvidge, C.D., Baugh, K.E., Kihn, E.A., Kroehl, H.W. and Davis, E.R.</a>
            <br>
            
          </li>          <li>
            <a href="https://journals.sfu.ca/apan/index.php/apan/article/view/8/0" class="" target="_none">Nighttime lights compositing using the VIIRS day-night band - Preliminary results. Proceedings of the Asia-Pacific Advanced Network 35 (2013)70-86.</a> by Kimberly Baugh, Feng-Chi Hsu, Christopher D. Elvidge, and Mikhail Zhizhin.</a>
            <br>
            
          </li>          <li>
            <a href="https://www.cambridge.org/core/books/power-and-the-vote/8091820171410B4FA557B6EBC3A5DD06" class="" target="_none">Power and the Vote - Elections and Electricity in the Developing World. Cambridge. 2015.</a> by Brian Min</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="ebirdst" class="dataset">
        <h4><a href="../ebirdst/">eBird Status and Trends Model Results</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://ebird.org/science/status-and-trends/faq" class="" target="_none">eBird Status and Trends FAQ</a> by Cornell Lab of Ornithology</a>
            <br>
            
          </li>          <li>
            <a href="https://ebird.org/science/status-and-trends" class="" target="_none">eBird Status and Trends Visualizations</a> by Cornell Lab of Ornithology</a>
            <br>
            
          </li>          <li>
            <a href="https://cornelllabofornithology.github.io/ebirdst/articles/ebirdst-introduction.html" class="" target="_none">ebirdst - Background, Data Access and Data Structure</a> by Tom Auer and Daniel Fink</a>
            <br>
            
          </li>          <li>
            <a href="https://cornelllabofornithology.github.io/ebirdst/articles/ebirdst-advanced-mapping.html" class="" target="_none">ebirdst - Generating Seasonal Abundance and Range Maps and Stats</a> by Matt Strimas-Mackey, Tom Auer, and Daniel Fink</a>
            <br>
            
          </li>          <li>
            <a href="https://cornelllabofornithology.github.io/ebirdst/articles/ebirdst-intro-mapping.html" class="" target="_none">ebirdst - Intro Mapping</a> by Tom Auer and Daniel Fink</a>
            <br>
            
          </li>          <li>
            <a href="https://cornelllabofornithology.github.io/ebirdst/articles/ebirdst-non-raster.html" class="" target="_none">ebirdst - Non-raster Data</a> by Tom Auer and Daniel Fink</a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://cornelllabofornithology.github.io/ebirdst/" class="" target="_none">ebirdst R Package</a> by Cornell Lab of Ornithology</a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/31837058" class="" target="_none">Modeling avian full annual cycle distribution and population trends with citizen science data</a> by Fink et al (2019)</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="cptac-2" class="dataset">
        <h4><a href="../cptac-2/">Clinical Proteomic Tumor Analysis Consortium 2 (CPTAC-2)</a></h4>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://cgc-accounts.sbgenomics.com/" class="" target="_none">Cancer Genomics Cloud</a> by <a href="http://www.cancergenomicscloud.org/" target="_none">Seven Bridges</a></a>
            <br>
            
          </li>          <li>
            <a href="https://proteomics.cancer.gov/data-portal" class="" target="_none">CPTAC Data Portal</a> by <a href="https://proteomics.cancer.gov/" target="_none">National Cancer Institute</a></a>
            <br>
            
          </li>          <li>
            <a href="https://portal.gdc.cancer.gov/projects/CPTAC-2" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></a>
            <br>
            
          </li>          <li>
            <a href="https://pdc.esacinc.com/pdc/browse" class="" target="_none">Proteomic Data Commons</a> by <a href="https://pdc.esacinc.com/pdc" target="_none">National Cancer Institute</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1016/j.cell.2016.05.069" class="" target="_none">Integrated Proteogenomic Characterization of Human High-Grade Serous Ovarian Cancer</a> by Hui Zhang, Tao Liu, Zhen Zhang, Samuel H. Payne, Bai Zhang, Jason E. McDermott, Jian-Ying
          Zhou, Vladislav A. Petyuk, Li Chen, Debjit Ray, Shisheng Sun, Feng Yang, Lijun Chen, Jing
          Wang, Punit Shah, Seong Won Cha, Paul Aiyetan, Sunghee Woo, Yuan Tian, Marina A. Gritsenko,
          Therese R. Clauss, Caitlin Choi, Matthew E. Monroe, Stefani Thomas, Song Nie, Chaochao Wu,
          Ronald J. Moore, Kun-Hsing Yu, David L. Tabb, David Fenyö, Vineet Bafna, Yue Wang, Henry
          Rodriguez, Emily S. Boja, Tara Hiltke, Robert C. Rivers, Lori Sokoll, Heng Zhu, Ie-Ming
          Shih, Leslie Cope, Akhilesh Pandey, Bing Zhang, Michael P. Snyder, Douglas A. Levine,
          Richard D. Smith, Daniel W. Chan, Karin D. Rodland, the CPTAC Investigators
          </a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1016/j.cell.2019.03.030" class="" target="_none">Proteogenomic Analysis of Human Colon Cancer Reveals New Therapeutic Opportunities</a> by Suhas Vasaikar, Chen Huang, Xiaojing Wang. Vladislav A. Petyuk, Sara R. Savage, Bo Wen,
          Yongchao Dou, Yun Zhang, Zhiao Shi, Osama A. Arshad, Marina A. Gritsenko, Lisa J. Zimmerman,
          Jason E. McDermott, Therese R. Clauss, Ronald J. Moore, Rui Zhao, Matthew E. Monroe, Yi-Ting
          Wang, Matthew C. Chambers, Robbert J.C. Slebos, Ken S. Lau, Qianxing Mo, Li Ding, Matthew
          Ellis, Mathangi Thiagarajan, Christopher R. Kinsinger, Henry Rodriguez, Richard D. Smith,
          Karin D. Rodland, Daniel C. Liebler, Tao Liu, Bing Zhang, Clinical Proteomic Tumor Analysis
          Consortium
          </a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1038/sdata.2015.22" class="" target="_none">Proteomic analysis of colon and rectal carcinoma using standard and customized databases
          </a> by Slebos RJ, Wang X, Wang X, Zhang B, Tabb DL, Liebler DC</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="icgc" class="dataset">
        <h4><a href="../icgc/">ICGC on AWS</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/ICGC-TCGA-PanCancer/cli/wiki" class="" target="_none">AWS Pancancer Workflow Launcher</a> by <a href="https://github.com/solomonshorser" target="_none">Solomon Shorser</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://www.nature.com/articles/s41586-020-1965-x" class="" target="_none">Analyses of non-coding somatic drivers in 2,693 cancer whole genomes</a> by <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term&#x3D;Rheinbay%20E%5BAuthor%5D&amp;cauthor&#x3D;true&amp;cauthor_uid&#x3D;28658208" target="_none">Rheinbay et al (2020)</a></a>
            <br>
            
          </li>          <li>
            <a href="https://www.nature.com/articles/s41586-020-1970-0" class="" target="_none">Genomic basis of RNA alterations in cancer</a> by <a href="https://icgc.org/committees-and-working-groups#71954" target="_none">PCAWG Transcriptome Core Group et al. (2020)</a></a>
            <br>
            
          </li>          <li>
            <a href="https://www.nature.com/articles/s41586-020-1969-6" class="" target="_none">Pan-cancer analysis of whole genomes</a> by <a href="https://dcc.icgc.org/pcawg" target="_none">The ICGC/TCGA Pan-Cancer Analysis of Whole Genomes Consortium (2020)</a></a>
            <br>
            
          </li>          <li>
            <a href="https://www.nature.com/articles/s41586-019-1913-9" class="" target="_none">Patterns of somatic structural variation in human cancer genomes</a> by <a href="https://www.ncbi.nlm.nih.gov/pubmed?cmd&#x3D;search&amp;term&#x3D;Yilong%20Li" target="_none">Li et al. (2020)</a></a>
            <br>
            
          </li>          <li>
            <a href="https://www.nature.com/articles/s41586-019-1907-7" class="" target="_none">The evolutionary history of 2,658 cancers</a> by <a href="https://www.ncbi.nlm.nih.gov/pubmed?cmd&#x3D;search&amp;term&#x3D;Moritz%20Gerstung" target="_none">Gerstung et al (2020)</a></a>
            <br>
            
          </li>          <li>
            <a href="https://www.nature.com/articles/s41586-020-1943-3" class="" target="_none">The repertoire of mutational signatures in human cancer</a> by <a href="https://www.ncbi.nlm.nih.gov/pubmed?cmd&#x3D;search&amp;term&#x3D;Ludmil%20B.%20Alexandrov" target="_none">Alexandrov et al (2020)</a></a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="nrel-pds-wtk" class="dataset">
        <h4><a href="../nrel-pds-wtk/">NREL Wind Integration National Dataset</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/NREL/hsds-examples" class="" target="_none">HSDS Examples</a> by <a href="https://www.nrel.gov/research/caleb-phillips.html" target="_none">Caleb Phillips, Caroline Draxl, John Readey, Jordan Perr-Sauer, Michael Rossol</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://maps.nrel.gov/wind-prospector/" class="" target="_none">Wind Prospector</a> by Paul Edwards</a>
            <br>
            
          </li>          <li>
            <a href="https://github.com/NREL/hsds-viz" class="" target="_none">Wind Visualization</a> by <a href="https://www.nrel.gov/research/jordan-perr-sauer.html" target="_none">Jordan Perr-Sauer</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://www.nrel.gov/docs/fy15osti/61740.pdf" class="" target="_none">Overview and Meteorological Validation of the Wind Integration National Dataset Toolkit</a> by Caroline Draxl, Bri-Mathias Hodge, Andrew Clifton, Jim McCaa</a>
            <br>
            
          </li>          <li>
            <a href="https://aws.amazon.com/blogs/big-data/power-from-wind-open-data-on-aws/" class="aws-link" target="_none">Power from wind: Open data on AWS</a> by <a href="https://www.nrel.gov/research/caleb-phillips.html" target="_none">Caleb Phillips, Caroline Draxl, John Readey, Jordan Perr-Sauer</a></a>
            <br>
            
          </li>          <li>
            <a href="https://www.sciencedirect.com/science/article/pii/S0306261915004237?via%3Dihub/" class="" target="_none">The Wind Integration National Dataset (WIND) Toolkit</a> by Caroline Draxl, Andrew Clifton, Bri-Mathias Hodge, Jim McCaa</a>
            <br>
            
          </li>          <li>
            <a href="https://www.nrel.gov/docs/fy14osti/61714.pdf" class="" target="_none">Validation of Power Output for the WIND Toolkit</a> by J. King, Andrew Clifton, Bri-Mathias Hodge</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="cptac-3" class="dataset">
        <h4><a href="../cptac-3/">Clinical Proteomic Tumor Analysis Consortium 3 (CPTAC-3)</a></h4>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://cgc-accounts.sbgenomics.com/" class="" target="_none">Cancer Genomics Cloud</a> by <a href="http://www.cancergenomicscloud.org/" target="_none">Seven Bridges</a></a>
            <br>
            
          </li>          <li>
            <a href="https://proteomics.cancer.gov/data-portal" class="" target="_none">CPTAC Data Portal</a> by <a href="https://proteomics.cancer.gov/" target="_none">National Cancer Institute</a></a>
            <br>
            
          </li>          <li>
            <a href="https://portal.gdc.cancer.gov/projects/CPTAC-3" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></a>
            <br>
            
          </li>          <li>
            <a href="https://pdc.esacinc.com/pdc/browse" class="" target="_none">Proteomic Data Commons</a> by <a href="https://pdc.esacinc.com/pdc" target="_none">National Cancer Institute</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/29718670" class="" target="_none">Evaluation of NCI-7 Cell Line Panel as a Reference Material for Clinical Proteomics</a> by Clark DJ, Hu Y, Bocik W, Chen L, Schnaubelt M, Roberts R, Shah P, Whiteley G, Zhang H
          </a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1016/j.cell.2019.10.007" class="" target="_none">Integrated Proteogenomic Characterization of Clear Cell Renal Cell Carcinoma</a> by Clark DJ, Dhanasekaran SM, Petralia F, Pan J, Song X, Hu Y, da Veiga Leprevost F, Reva B,
          Lih TM, Chang HY, Ma W, Huang C, Ricketts CJ, Chen L1, Krek A, Li Y, Rykunov D, Li QK, Chen
          LS, Ozbek U, Vasaikar S, Wu Y, Yoo S, Chowdhury S, Wyczalkowski MA, Ji J, Schnaubelt M, Kong
          A, Sethuraman S, Avtonomov DM, Ao M, Colaprico A, Cao S, Cho KC, Kalayci S, Ma S, Liu W,
          Ruggles K, Calinawan A, Gümüş ZH, Geizler D, Kawaler E, Teo GC, Wen B, Zhang Y, Keegan S, Li
          K, Chen F, Edwards N, Pierorazio PM, Chen XS, Pavlovich CP, Hakimi AA, Brominski G, Hsieh
          JJ, Antczak A, Omelchenko T, Lubinski J, Wiznerowicz M, Linehan WM, Kinsinger CR,
          Thiagarajan M, Boja ES, Mesri M, Hiltke T, Robles AI, Rodriguez H, Qian J, Fenyö D, Zhang B,
          Ding L, Schadt E, Chinnaiyan AM, Zhang Z, Omenn GS, Cieslik M, Chan DW, Nesvizhskii AI, Wang
          P, Zhang H; Clinical Proteomic Tumor Analysis Consortium
          </a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="gdelt" class="dataset">
        <h4><a href="../gdelt/">Global Database of Events, Language and Tone (GDELT)</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://www.linkedin.com/pulse/media-brexit-story-so-far-may-corbyn-less-shambles-than-mark-chopping/" class="" target="_none">Analysing Brexit Coverage In The Media Over Time</a> by <a href="https://www.kognitio.com" target="_none">Mark Chopping</a></a>
            <br>
            
          </li>          <li>
            <a href="http://www.geomesa.org/documentation/tutorials/geomesa-hbase-s3-on-aws.html" class="" target="_none">Bootstrapping GeoMesa HBase on AWS S3</a> by <a href="https://www.ccri.com" target="_none">Commonwealth Computer Research, Inc.</a></a>
            <br>
            
          </li>          <li>
            <a href="https://gist.github.com/jakechen/6955f2de51212163312b6430555b8e0b" class="" target="_none">Creating PySpark DataFrame from CSV in AWS S3 in EMR</a> by <a href="https://github.com/jakechen" target="_none">Jake Chen</a></a>
            <br>
            
          </li>          <li>
            <a href="http://blog.julien.org/2017/03/exploring-gdelt-data-set-with-amazon.html" class="" target="_none">Exploring GDELT with Athena</a> by <a href="https://twitter.com/julsimon" target="_none">Julien Simon</a></a>
            <br>
            
          </li>          <li>
            <a href="https://aws.amazon.com/blogs/big-data/running-r-on-amazon-athena/" class="aws-link" target="_none">Running R on Amazon Athena</a> by <a href="https://www.linkedin.com/in/gopal-wunnava-b11a77/" target="_none">Gopal Wunnava</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="http://live.thermobook.net/globe-events/" class="" target="_none">Globe Events</a> by <a href="http://www.thermobook.net" target="_none">thermobook</a></a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="ladi" class="dataset">
        <h4><a href="../ladi/">Low Altitude Disaster Imagery (LADI) Dataset</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/LADI-Dataset/ladi-tutorial" class="" target="_none">LADI Tutorials</a> by <a href="https://github.com/LADI-Dataset" target="_none">Andrew Weinert, Jianyu Mao, Kiana Harris, Nae-Rong Chang, Caleb Pennell, Yiming Ren, Ryan Earley, Nadia Dimitrova</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://www-nlpir.nist.gov/projects/tv2020/dsdi.html" class="" target="_none">NIST TRECVID 2020 - Disaster Scene Description and Indexing (DSDI)</a> by TREC Video Retrieval Evaluation (TRECVID)</a>
            <br>
            
          </li>          <li>
            <a href="https://github.com/bwsi-hadr" class="" target="_none">Remote Sensing for Disaster Response Course</a> by <a href="https://beaverworks.ll.mit.edu/CMS/bw/bwsi" target="_none">Beaver Works Summer Institute</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1109/HPEC.2019.8916437" class="" target="_none">Large Scale Organization and Inference of an Imagery Dataset for Public Safety</a> by Jeffrey Liu, David Strohschein, Siddharth Samsi, Andrew Weinert</a>
            <br>
            
          </li>          <li>
            <a href="https://arxiv.org/abs/2005.05495" class="" target="_none">Train and Deploy an Image Classifier for Disaster Response</a> by Jianyu Mao, Kiana Harris, Nae-Rong Chang, Caleb Pennell, Yiming Ren</a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1109/LNET.2019.2955833" class="" target="_none">Video Testing at the FirstNet Innovation and Test Lab Using a Public Safety Dataset</a> by Chris Budny, Jeffrey Liu, Andrew Weinert</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="nyu-fastmri" class="dataset">
        <h4><a href="../nyu-fastmri/">NYU Langone &amp; FAIR FastMRI Dataset</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/facebookresearch/fastMRI/blob/master/fastMRI_tutorial.ipynb" class="" target="_none">FastMRI Tutorial (Jupyter Notebook)</a> by <a href="https://scholar.google.com/citations?user&#x3D;LjjLn2YAAAAJ&amp;hl&#x3D;en" target="_none">Tullie Murrell</a></a>
            <br>
            
          </li>          <li>
            <a href="https://medium.com/@khodges42/how-to-get-started-diy-neurohacking-with-the-fastmri-dataset-and-why-you-should-6a02a6bb896e" class="" target="_none">How to Get Started Working with the FastMRI Dataset (and why you should!)</a> by Kevin Hodges</a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://arxiv.org/abs/2001.02518" class="" target="_none">Advancing machine learning for MR image reconstruction with an open competition:Overview of the 2019 fastMRI challenge</a> by <a href="https://arxiv.org/search/eess?searchtype&#x3D;author&amp;query&#x3D;Knoll%2C+F" target="_none">Knoll et al (2020)</a></a>
            <br>
            
          </li>          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/29774597" class="" target="_none">Assessment of the generalization of learned image reconstruction and the potential for transfer learning.</a> by <a href="https://arxiv.org/search/eess?searchtype&#x3D;author&amp;query&#x3D;Knoll%2C+F" target="_none">Knoll et al (2019)</a></a>
            <br>
            
          </li>          <li>
            <a href="https://arxiv.org/abs/1904.01112" class="" target="_none">Deep Learning Methods for Parallel Magnetic Resonance Image Reconstruction</a> by <a href="https://arxiv.org/search/eess?searchtype&#x3D;author&amp;query&#x3D;Knoll%2C+F" target="_none">Knoll et al (2019)</a></a>
            <br>
            
          </li>          <li>
            <a href="https://arxiv.org/abs/1811.08839" class="" target="_none">fastMRI:An Open Dataset and Benchmarks for Accelerated MRI</a> by <a href="https://arxiv.org/search/cs?searchtype&#x3D;author&amp;query&#x3D;Zbontar%2C+J" target="_none">Zbontar et al (2019)</a></a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="radiant-mlhub" class="dataset">
        <h4><a href="../radiant-mlhub/">Radiant MLHub</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/radiantearth/mlhub-tutorials/blob/master/RadiantMLHub-intro.pdf" class="" target="_none">How to access Radiant MLHub Data</a> by <a href="https://www.radiant.earth/" target="_none">Radiant Earth</a></a>
            <br>
            
          </li>          <li>
            <a href="https://github.com/radiantearth/mlhub-tutorials/tree/master/notebooks" class="" target="_none">Radiant MLHub Tutorials with Jupyter Notebooks</a> by <a href="https://www.linkedin.com/in/kbgg/" target="_none">Kevin Booth</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://zindi.africa/competitions/iclr-workshop-challenge-2-radiant-earth-computer-vision-for-crop-recognition" class="" target="_none">Challenge on Computer Vision for Crop Detection from Satellite Imagery</a> by <a href="https://www.radiant.earth/" target="_none">Radiant Earth</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://medium.com/radiant-earth-insights/a-guide-for-collecting-and-sharing-ground-reference-data-for-machine-learning-applications-90664930925e" class="" target="_none">A Guide for Collecting and Sharing Ground Reference Data for Machine Learning Applications</a> by <a href="https://www.linkedin.com/in/yonahbg/" target="_none">Yonah Bromberg Gaber</a></a>
            <br>
            
          </li>          <li>
            <a href="https://medium.com/radiant-earth-insights/creating-a-machine-learning-commons-for-global-development-256ef3dd46aa" class="" target="_none">Creating a Machine Learning Commons for Global Development</a> by <a href="https://www.linkedin.com/in/hamedalemohammad/" target="_none">Hamed Alemohammad</a></a>
            <br>
            
          </li>          <li>
            <a href="https://aws.amazon.com/blogs/publicsector/geo-diverse-open-training-data-as-a-global-public-good/" class="aws-link" target="_none">Geo-Diverse Open Training Data as a Global Public Good</a> by <a href="https://www.linkedin.com/in/hamedalemohammad/" target="_none">Hamed Alemohammad</a></a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="usgs-lidar" class="dataset">
        <h4><a href="../usgs-lidar/">USGS 3DEP LiDAR Point Clouds</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://aws.amazon.com/blogs/machine-learning/extracting-buildings-and-roads-from-aws-open-data-using-amazon-sagemaker/" class="aws-link" target="_none">Extracting buildings and roads from AWS Open Data using Amazon SageMaker</a> by Yunzhi Shi, Tianyu Zhang, and Xin Chen</a>
            <br>
            <span class="label label-info tag service-tag">SageMaker</span>
          </li>          <li>
            <a href="https://github.com/hobu/usgs-lidar/tree/master/lambda" class="" target="_none">Using Lambda Layers with USGS 3DEP LiDAR Point Clouds</a> by <a href="https://twitter.com/howardbutler" target="_none">Howard Butler</a></a>
            <br>
            <span class="label label-info tag service-tag">Lambda</span>
          </li>          <li>
            <a href="https://usgs.entwine.io/" class="" target="_none">WebGL Visualization of USGS 3DEP Lidar Point Clouds with Potree and Plasio.js</a> by <a href="https://twitter.com/csmannin" target="_none">Connor Manning</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://www.facebook.com/isptoolbox/line-of-sight-check/" class="" target="_none">Facebook Line of Sight Check</a> by <a href="https://www.facebook.com/isptoolbox/" target="_none">Facebook</a></a>
            <br>
            
          </li>          <li>
            <a href="https://portal.opentopography.org/datasets" class="" target="_none">OpenTopography access to 3DEP lidar point cloud data</a> by <a href="https://opentopography.org/" target="_none">OpenTopography</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://www.usgs.gov/news/usgs-3dep-lidar-point-cloud-now-available-amazon-public-dataset" class="" target="_none">USGS 3DEP Lidar Point Cloud Now Available as Amazon Public Dataset</a> by <a href="https://www.usgs.gov" target="_none">Department of the Interior, U.S. Geological Survey</a></a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="aws-covid19-lake" class="dataset">
        <h4><a href="../aws-covid19-lake/">COVID-19 Data Lake</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://aws.amazon.com/blogs/big-data/a-public-data-lake-for-analysis-of-covid-19-data/" class="aws-link" target="_none">A public data lake for analysis of COVID-19 data</a> by <a href="https://aws.amazon.com/big-data/datalakes-and-analytics/" target="_none">AWS Data Lake Team</a></a>
            <br>
            <span class="label label-info tag service-tag">Glue</span><span class="label label-info tag service-tag">Athena</span><span class="label label-info tag service-tag">Comprehend</span><span class="label label-info tag service-tag">Kendra</span><span class="label label-info tag service-tag">QuickSight</span>
          </li>          <li>
            <a href="https://aws.amazon.com/blogs/big-data/exploring-the-public-aws-covid-19-data-lake/" class="aws-link" target="_none">Exploring the public AWS COVID-19 data lake</a> by <a href="https://aws.amazon.com/big-data/datalakes-and-analytics/" target="_none">AWS Data Lake Team</a></a>
            <br>
            <span class="label label-info tag service-tag">Athena</span><span class="label label-info tag service-tag">Glue</span><span class="label label-info tag service-tag">SageMaker</span>
          </li>          <li>
            <a href="https://github.com/aws-samples/query-data-in-s3-with-amazon-athena-and-aws-sdk-for-dotnet" class="aws-link" target="_none">How to use SQL to query data in S3 Bucket with Amazon Athena and AWS SDK for .NET</a> by <a href="https://github.com/orgs/aws-samples/teams/aws-proserve-us-west-applications" target="_none">AWS ProServe US West Applications Team</a></a>
            <br>
            <span class="label label-info tag service-tag">Athena</span><span class="label label-info tag service-tag">Glue</span><span class="label label-info tag service-tag">S3</span><span class="label label-info tag service-tag">Cloud​Formation</span>
          </li>        </ul>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://us-east-2.console.aws.amazon.com/cloudformation/home?region&#x3D;us-east-2#/stacks/create/review?templateURL&#x3D;https://covid19-lake.s3.us-east-2.amazonaws.com/cfn/CovidLakeStack.template.json&amp;stackName&#x3D;CovidLakeStack" class="" target="_none">CloudFormation template for Glue Catalog table definitions</a> by <a href="https://aws.amazon.com/big-data/datalakes-and-analytics/" target="_none">AWS Data Lake Team</a></a>
            <br>
            
          </li>          <li>
            <a href="https://go.aws/covid-19-data-lake" class="aws-link" target="_none">Explore the COVID-19 data lake public S3 bucket</a> by <a href="https://aws.amazon.com/big-data/datalakes-and-analytics/" target="_none">AWS Data Lake Team</a></a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="mmrf-commpass" class="dataset">
        <h4><a href="../mmrf-commpass/">CoMMpass from the Multiple Myeloma Research Foundation</a></h4>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://portal.gdc.cancer.gov/projects/MMRF-COMMPASS" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1182/blood.V126.23.722.722" class="" target="_none">&quot;Identification of Initiating Trunk Mutations and Distinct Molecular Subtypes: An Interim
          Analysis of the Mmrf Commpass Study&quot;
          </a> by Jonathan J Keats, PhD, Gil Speyer, Legendre Christophe, Christofferson Austin, Kristi
          Stephenson, BS, Ahmet Kurdoglu, Megan Russell, Aldrich Jessica, Cuyugan Lori, Jonathan
          Adkins, Jackie McDonald, Adrienne Helland, Alex Blanski, Meghan Hodges, Dan Rohrer, Sundar
          Jagannath, MD, David Siegel, MD PhD, Ravi Vij, MD MBA, Gregory Orloff, MD, Todd Zimmerman,
          MD, Ruben Niesvizky, MD, Darla Liles, MD, Joseph W. Fay, Jeffrey L. Wolf, MD PhD, Robert M.
          Rifkin, Norma C Gutierrez, The MMRF CoMMpass Network, Jen Toups, Mary Derome, MS, Winnie
          Liang, PhD, Seunchan Kim, Daniel Auclair, PhD, Pamela G. Kidd, MD, Scott Jewell, PhD, John
          David Carpten, PhD, Sagar Lonial, MD
          </a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1182/blood.V122.21.532.532" class="" target="_none">&quot;Interim Analysis Of The MMRF CoMMpass Trial: a Longitudinal Study In Multiple Myeloma
          Relating Clinical Outcomes To Genomic and Immunophenotypic Profiles&quot;
          </a> by Keats JJ, Craig DW, Liang W, Venkata Y, Kurdoglu A, Aldrich J, Auclair D, Allen K, Harrison
          B, Jewell S, Kidd PG, Correll M, Jagannath S, Siegel DS, Vij R, Orloff G, Zimmerman TM, MMRF
          CoMMpass Network, Capone W, Carpten J, Lonial S.
          </a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1182/blood.V124.21.722.722" class="" target="_none">&quot;Interim Analysis of the Mmrf Commpass Trial: Identification of Novel Rearrangements
          Potentially Associated with Disease Initiation and Progression&quot;
          </a> by Sagar Lonial, MD, Venkata D Yellapantula, Winnie Liang, PhD, Ahmet Kurdoglu, BS, Jessica
          Aldrich, MSc, Christophe M. Legendre, MD, Kristi Stephenson, Jonathan Adkins, Jackie
          McDonald, Adrienne Helland, Megan Russell, Austin Christofferson, Lori Cuyugan, Dan Rohrer,
          Alex Blanski, Meghan Hodges, Mmrf CoMMpass Network, Mary Derome, Daniel Auclair, PhD, Pamela
          G. Kidd, MD, Scott Jewell, PhD, David Craig, PhD, John Carpten, PhD, Jonathan J. Keats, PhD
          </a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1182/blood.V128.22.194.194" class="" target="_none">&quot;Molecular Predictors of Outcome and Drug Response in Multiple Myeloma: An Interim Analysis
          of the Mmrf CoMMpass Study&quot;
          </a> by Jonathan J Keats, PhD, Gil Speyer, Austin Christofferson, Christophe Legendre, PhD, Jessica
          Aldrich, Megan Russell, Lori Cuyugan, Jonathan Adkins, Alex Blanski, Meghan Hodges, Dan
          Rohrer, Sundar Jagannath, MD, Ravi Vij, MD, Gregory Orloff, MD, Todd Zimmerman, MD, Ruben
          Niesvizky, MD, Darla Liles, MD, Joseph W. Fay, Jeffrey L. Wolf, MD, Robert M Rifkin, Norma C
          Gutierrez, MD PhD, Mmrf CoMMpass Network, Jennifer Yesil, MS, Mary Derome, MS, Seungchan
          Kim, PhD, Winnie Liang, PhD, Pamela G. Kidd, MD, Scott Jewell, PhD, John David Carpten, PhD,
          Daniel Auclair, PhD, Sagar Lonial, MD FACP
          </a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="noaa-hrrr-pds" class="dataset">
        <h4><a href="../noaa-hrrr-pds/">NOAA High-Resolution Rapid Refresh (HRRR) Model</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://mesowest.utah.edu/html/hrrr/zarr_documentation/conda_environment.html" class="" target="_none">Conda Enironment Setup Guide</a> by Zach Rieck</a>
            <br>
            
          </li>          <li>
            <a href="https://github.com/blaylockbk/HRRR_archive_download" class="" target="_none">HRRR-B Python package: download and read HRRR grib2 files</a> by Brian Blaylock</a>
            <br>
            
          </li>          <li>
            <a href="https://mesowest.utah.edu/html/hrrr/zarr_documentation/What_is_Zarr.html" class="" target="_none">What is Zarr?</a> by Taylor Gowan</a>
            <br>
            
          </li>          <li>
            <a href="https://mesowest.utah.edu/html/hrrr/zarr_documentation/zarrFileVariables.html" class="" target="_none">Zarr File Varable Definitions</a> by Taylor Gowan</a>
            <br>
            
          </li>          <li>
            <a href="https://mesowest.utah.edu/html/hrrr/zarr_documentation/zarr_examples.html" class="" target="_none">Zarr Visualization Example</a> by Taylor Gowan, James Powell, Zach Rieck</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="nrel-oedi-pv-rooftops" class="dataset">
        <h4><a href="../nrel-oedi-pv-rooftops/">National Renewable Energy Laboratory&#x27;s (NREL) PV Rooftop Database</a></h4>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://maps.nrel.gov/nsrdb-viewer/?aL&#x3D;0&amp;bL&#x3D;groad&amp;cE&#x3D;0&amp;lR&#x3D;0&amp;mC&#x3D;31.653381399664%2C-3.1640625&amp;zL&#x3D;2" class="" target="_none">NSRDB Viewer</a> by <a href="https://www.nrel.gov/" target="_none">NREL</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://iopscience.iop.org/article/10.1088/1748-9326/aaa554/pdf" class="" target="_none">Estimating rooftop solar technical potential across the US using a combination of GIS-based methods, lidar data, and statistical modeling</a> by Pieter Gagnon et al 2018 Environ. Res. Lett. 13 024027</a>
            <br>
            
          </li>          <li>
            <a href="https://www.nrel.gov/docs/fy16osti/65298.pdf" class="" target="_none">Rooftop Solar Photovoltaic Technical Potential in the United States: A Detailed Assessment</a> by <a href="https://www.nrel.gov/" target="_none">Pieter Gagnon, Robert Margolis, Jennifer Melius, Caleb Phillips, and Ryan Elmore</a></a>
            <br>
            
          </li>          <li>
            <a href="https://www.nrel.gov/docs/fy18osti/70901.pdf" class="" target="_none">Rooftop Solar Technical Potential for Low-to-Moderate Income Households in the United States</a> by <a href="https://www.nrel.gov/" target="_none">Benjamin Sigrin and Meghan Mooney</a></a>
            <br>
            
          </li>          <li>
            <a href="https://iopscience.iop.org/article/10.1088/1748-9326/aa7225/pdf" class="" target="_none">Using GIS-based methods and lidar data to estimate rooftop solar technical potential in US cities</a> by Robert Margolis et al 2017 Environ. Res. Lett. 12 074013</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="nyc-tlc-trip-records-pds" class="dataset">
        <h4><a href="../nyc-tlc-trip-records-pds/">New York City Taxi and Limousine Commission (TLC) Trip Record Data</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://aws.amazon.com/blogs/big-data/build-a-real-time-stream-processing-pipeline-with-apache-flink-on-aws/" class="aws-link" target="_none">Build a Real-time Stream Processing Pipeline with Apache Flink on AWS</a> by Steffen Hausmann</a>
            <br>
            
          </li>          <li>
            <a href="https://aws.amazon.com/blogs/big-data/build-and-run-streaming-applications-with-apache-flink-and-amazon-kinesis-data-analytics-for-java-applications/" class="aws-link" target="_none">Build and run streaming applications with Apache Flink and Amazon Kinesis Data Analytics for Java Applications</a> by Steffen Hausmann</a>
            <br>
            
          </li>          <li>
            <a href="https://www.youtube.com/watch?v&#x3D;MzTZp47Jy7E" class="" target="_none">Deep Dive on Flink &amp; Spark on Amazon EMR</a> by Keith Steward</a>
            <br>
            
          </li>          <li>
            <a href="https://github.com/aws-samples/aws-open-data-analytics-notebooks/tree/master/exploring-data" class="aws-link" target="_none">Exploring data with Python and Amazon S3 Select</a> by Manav Sehgal</a>
            <br>
            
          </li>          <li>
            <a href="https://github.com/aws-samples/aws-open-data-analytics-notebooks/tree/master/optimizing-data" class="aws-link" target="_none">Optimizing data for analysis with Amazon Athena and AWS Glue</a> by Manav Sehgal</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="omi-no2-nasa" class="dataset">
        <h4><a href="../omi-no2-nasa/">Ozone Monitoring Instrument (OMI) / Aura NO2 Tropospheric Column Density</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://medium.com/devseed/cog-talk-part-1-whats-new-941facbcd3d1" class="" target="_none">Medium Post: COG Talk — Part 1: What’s new?</a> by <a href="https://medium.com/@_VincentS_" target="_none">Vincent Sarago</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://pfg2k5mjei.execute-api.us-west-2.amazonaws.com/docs" class="" target="_none">COG Application Programming Interface (API)</a> by <a href="https://developmentseed.org/" target="_none">Development Seed</a></a>
            <br>
            
          </li>          <li>
            <a href="https://pfg2k5mjei.execute-api.us-west-2.amazonaws.com/v1/metadata?url&#x3D;https%3A%2F%2Fomi-no2-nasa.s3-us-west-2.amazonaws.com%2FOMI-Aura_L3-OMNO2d_2018m0101_v003-2019m1123t055315.tif&amp;pmin&#x3D;2&amp;pmax&#x3D;98&amp;max_size&#x3D;1024&amp;histogram_bins&#x3D;20" class="" target="_none">COG metadata example </a> by <a href="https://developmentseed.org/" target="_none">Development Seed</a></a>
            <br>
            
          </li>          <li>
            <a href="https://cogeo.xyz/?url&#x3D;s3://omi-no2-nasa/OMI-Aura_L3-OMNO2d_2020m0101_v003-2020m0330t173100.tif" class="" target="_none">COG Viewer</a> by <a href="https://developmentseed.org/" target="_none">Development Seed</a></a>
            <br>
            
          </li>          <li>
            <a href="https://rasterio.readthedocs.io/" class="" target="_none">Rasterio (Python Library) for access to geospatial raster data</a> by <a href="https://www.mapbox.com/" target="_none">Mapbox</a></a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="pmsp-lidar" class="dataset">
        <h4><a href="../pmsp-lidar/">Prefeitura Municipal de São Paulo (PMSP) LiDAR Point Cloud</a></h4>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://entwine.io/" class="" target="_none">Entwine</a> by <a href="https://hobu.co/" target="_none">Hobu, Inc.</a></a>
            <br>
            
          </li>          <li>
            <a href="http://forsys.sefs.uw.edu/fusion/fusionlatest.html" class="" target="_none">Fusion</a> by <a href="https://www.usda.gov/" target="_none">US Department of Agriculture - Forest Service</a></a>
            <br>
            
          </li>          <li>
            <a href="https://rapidlasso.com/lastools/" class="" target="_none">LAStools</a> by <a href="https://rapidlasso.com" target="_none">rapidlasso GmbH, GERMANY</a></a>
            <br>
            
          </li>          <li>
            <a href="https://pdal.io/" class="" target="_none">PDAL - Point Data Abstraction Library</a> by <a href="https://doi.org/10.5281/zenodo.2556738" target="_none">PDAL Contributors</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.5194/isprs-annals-IV-2-W5-437-2019" class="" target="_none">Describing the Vertical Structure of Informal Settlements on the Basis of LiDAR Data – A Case Study for Favelas (Slums) in Sao Paulo City</a> by S. C. L. Ribeiro, M. Jarzabek-Rychard, J. P. Cintra, H.-G. Maas</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="sondehub-telemetry" class="dataset">
        <h4><a href="../sondehub-telemetry/">SondeHub Radiosonde Telemetry</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/projecthorus/pysondehub/blob/main/examples/README.md" class="" target="_none">Loading example notebooks into SageMaker</a> by Michaela Wheeler</a>
            <br>
            <span class="label label-info tag service-tag">SageMaker</span>
          </li>          <li>
            <a href="https://github.com/projecthorus/pysondehub/blob/main/examples/sondehub-athena.ipynb" class="" target="_none">Using Athena to read radiosonde data</a> by Michaela Wheeler</a>
            <br>
            <span class="label label-info tag service-tag">Athena</span>
          </li>          <li>
            <a href="https://github.com/projecthorus/pysondehub/blob/main/examples/sondehub.ipynb" class="" target="_none">Using pysondehub to read radiosonde data</a> by Michaela Wheeler</a>
            <br>
            <span class="label label-info tag service-tag">S3</span>
          </li>        </ul>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://pypi.org/project/sondehub/" class="" target="_none">pysondehub</a> by <a href="https://github.com/projecthorus/pysondehub" target="_none">Sondehub</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://www.youtube.com/watch?v&#x3D;YBy-bXEWZeM" class="" target="_none">STM32 Development Boards (literally) Falling From The Sky (How to submit data)</a> by Mark Jessop &amp; Michaela Wheeler</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="southern-california-earthquakes" class="dataset">
        <h4><a href="../southern-california-earthquakes/">Southern California Earthquake Data</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://nextjournal.com/thclements/cactus-to-clouds/#accessing-the-scedc-dataset-on-aws" class="" target="_none">Cactus to Clouds: Processing The SCEDC Open Data Set on AWS</a> by <a href="https://tclements.github.io/" target="_none">Tim Clements</a></a>
            <br>
            
          </li>          <li>
            <a href="https://scedc.caltech.edu/data/getstarted-pds.html" class="" target="_none">Getting Started with SCEDC AWS Public Dataset</a> by <a href="https://scedc.caltech.edu" target="_none">Ellen Yu</a></a>
            <br>
            
          </li>          <li>
            <a href="https://github.com/SCEDC/cloud/tree/master/pds-fetch-continuous-data" class="" target="_none">Script to Download Seismic Waveforms from the SCEDC AWS Public Dataset</a> by <a href="https://scedc.caltech.edu" target="_none">Aparna Bhaskaran</a></a>
            <br>
            
          </li>          <li>
            <a href="https://nextjournal.com/thclements/seisnoisejl-gpu-computing-tutorial" class="" target="_none">SeisNoise.jl GPU Computing Tutorial - Another example of accessing data s3://scedc-pds for ambient noise cross-correlation</a> by <a href="https://tclements.github.io/" target="_none">Tim Clements</a></a>
            <br>
            
          </li>          <li>
            <a href="https://github.com/SCEDC/cloud/tree/master/pds-lambda-example" class="" target="_none">Using Lambda to Process Seismograms</a> by <a href="https://scedc.caltech.edu" target="_none">Shang-Lin Chen</a></a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="ilmn-dragen-1kgp" class="dataset">
        <h4><a href="../ilmn-dragen-1kgp/">1000 Genomes Phase 3 Reanalysis with DRAGEN 3.5</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/aws-quickstart/quickstart-illumina-dragen" class="" target="_none">DRAGEN Quick Start on AWS</a> by <a href="https://github.com/aws-quickstart" target="_none">AWS Quick Start Team</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://support.illumina.com/content/dam/illumina-support/documents/downloads/software/dragen/DRAGEN-3.5.7b-Customer-Release-Notes-1000000131537-00.pdf" class="" target="_none">DRAGEN 3.7.5b Release</a> by <a href="https://www.illumina.com/" target="_none">Illumina Inc.</a></a>
            <br>
            
          </li>          <li>
            <a href="https://www.illumina.com/products/by-type/informatics-products/basespace-sequence-hub.html#key-features" class="" target="_none">DRAGEN on BaseSpace Sequence Hub</a> by <a href="https://www.illumina.com/" target="_none">Illumina Inc.</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://www.biorxiv.org/content/10.1101/2020.11.13.380741v1" class="" target="_none">precisionFDA Truth Challenge V2: Calling variants from short- and long-reads in difficult-to-map regions (Preprint)</a> by Olson et al (2020)</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="3kricegenome" class="dataset">
        <h4><a href="../3kricegenome/">3000 Rice Genomes Project</a></h4>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/InternationalRiceResearchInstitute/RiceGalaxy" class="" target="_none">RiceGalaxy</a> by <a href="https://www.irri.org/" target="_none">International Rice Research Institute</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://academic.oup.com/gigascience/article/8/5/giz028/5492234" class="" target="_none">Rice Galaxy: an open resource for plant science</a> by Juanillas V et al (2019)</a>
            <br>
            
          </li>          <li>
            <a href="https://genome.cshlp.org/content/29/5/870.short" class="" target="_none">Structural variants in 3000 rice genomes</a> by Fuentes RR et al (2019)</a>
            <br>
            
          </li>          <li>
            <a href="https://www.nature.com/articles/s41467-019-08451-3" class="" target="_none">Tracking the origin of two genetic components associated with transposable element bursts in domesticated rice</a> by Chen J et al (2019)</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="ncar-cesm-lens" class="dataset">
        <h4><a href="../ncar-cesm-lens/">Community Earth System Model Large Ensemble (CESM LENS)</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://medium.com/pangeo/cesm-lens-on-aws-4e2a996397a1" class="" target="_none">Analyzing large climate model ensembles in the cloud</a> by Joe Hamman, NCAR</a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/NCAR/cesm-lens-aws" class="" target="_none">Jupyter Notebook and other documentation and tools for CESM LENS on AWS</a> by NCAR Science at Scale team</a>
            <br>
            
          </li>          <li>
            <a href="http://ncar-aws-www.s3-website-us-west-2.amazonaws.com/kay-et-al-2015.v2.html" class="" target="_none">Rendered (static) version of Jupyter Notebook</a> by Anderson Banihirwe, NCAR</a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.5065/d6j101d1" class="" target="_none">The Community Earth System Model (CESM) Large Ensemble Project: A Community Resource for Studying Climate Change in the Presence of Internal Climate Variability</a> by Kay et al. (2015), Bull. AMS, 96, 1333-1349</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="wpto-pds-us-wave" class="dataset">
        <h4><a href="../wpto-pds-us-wave/">DOE&#x27;s Water Power Technology Office&#x27;s (WPTO) US Wave dataset</a></h4>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/NREL/hsds-examples" class="" target="_none">HSDS Examples</a> by Caleb Phillips, Caroline Draxl, John Readey, Jordan Perr-Sauer, Michael Rossol</a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://www.osti.gov/biblio/1599105" class="" target="_none">Development and validation of a high-resolution regional wave hindcast model for U.S. West Coast wave   resource characterization</a> by Wu, Wei-Cheng; Wang, Taiping; Yang, Zhaoqing; Garcia Medina, Gabriel</a>
            <br>
            
          </li>          <li>
            <a href="https://www.osti.gov/biblio/1573061/" class="" target="_none">High-Resolution Regional Wave Hindcast for the U.S. West Coast</a> by Yang, Zhaoqing; Wu, Wei-Cheng; Wang, Taiping; Castrucci, Luca</a>
            <br>
            
          </li>          <li>
            <a href="http://swanmodel.sourceforge.net/download/zip/swanuse.pdf" class="" target="_none">SWAN Cycle III version 41.31A</a> by The SWAN team</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="oedi-data-lake" class="dataset">
        <h4><a href="../oedi-data-lake/">Department of Energy&#x27;s Open Energy Data Initiative (OEDI)</a></h4>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://emp.lbl.gov/tracking-sun-tool" class="" target="_none">Tracking the Sun Tool</a> by <a href="https://emp.lbl.gov/tracking-the-sun" target="_none">Lawrence Berkeley National Laboratory (LBNL)</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://www.nrel.gov/docs/fy16osti/65283.pdf" class="" target="_none">On the Use of Coupled Wind, Wave, and Current Fields in the Simulation of Loads on BottomSupported Offshore Wind Turbines during Hurricanes</a> by E. Kim, L. Manuel, M. Curcic, S. S. Chen, C. Phillips, P. Veers</a>
            <br>
            
          </li>          <li>
            <a href="https://www.nrel.gov/docs/fy16osti/65231.pdf" class="" target="_none">The Distributed Generation Market Demand Model (dGen):Documentation</a> by B. Sigrin, M. Gleason, R. Preus, I. Baring-Gould, R. Margolis</a>
            <br>
            
          </li>          <li>
            <a href="https://emp.lbl.gov/sites/default/files/tracking_the_sun_2019_report.pdf" class="" target="_none">Tracking the Sun Pricing and Design Trends for Distributed Photovoltaic Systems in the United States: 2019 Edition</a> by G. Barbose, N. Darghouth</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="ecmwf-era5" class="dataset">
        <h4><a href="../ecmwf-era5/">ECMWF ERA5 Reanalysis</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/planet-os/notebooks/blob/master/aws/era5-s3-via-boto.ipynb" class="" target="_none">Accessing ERA5 Data on S3 Using Boto</a> by <a href="https://www.intertrust.com/" target="_none">Intertrust Technologies Corporation</a></a>
            <br>
            
          </li>          <li>
            <a href="https://github.com/planet-os/notebooks/blob/master/api-examples/ERA5_tutorial.ipynb" class="" target="_none">ERA5 tutorial using the Planet OS API</a> by <a href="https://www.intertrust.com/" target="_none">Intertrust Technologies Corporation</a></a>
            <br>
            
          </li>          <li>
            <a href="https://nbviewer.jupyter.org/github/awslabs/amazon-asdi/blob/main/examples/dask/notebooks/era5.ipynb" class="aws-link" target="_none">Processing ERA5 data in NetCDF Format</a> by Zac Flamig</a>
            <br>
            
          </li>          <li>
            <a href="https://nbviewer.jupyter.org/github/awslabs/amazon-asdi/blob/main/examples/dask/notebooks/era5_zarr.ipynb" class="aws-link" target="_none">Processing ERA5 data in Zarr Format</a> by Zac Flamig</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="encode-project" class="dataset">
        <h4><a href="../encode-project/">Encyclopedia of DNA Elements (ENCODE)</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/ENCODE-DCC/encode-data-usage-examples/blob/master/ctcf_chip_seq_cross_cell_type_correlation.ipynb" class="" target="_none">ENCODE CTCF ChIP-seq data correlation across different cell types</a> by <a href="https://github.com/p-sud" target="_none">Paul Sud</a></a>
            <br>
            
          </li>          <li>
            <a href="https://github.com/ENCODE-DCC/encode-data-usage-examples/blob/master/mount_s3_bucket_and_run_jupyter_on_ec2.ipynb" class="" target="_none">Exploring ENCODE data from EC2 with Jupyter notebook</a> by <a href="https://github.com/keenangraham" target="_none">Keenan Graham</a></a>
            <br>
            
          </li>          <li>
            <a href="https://github.com/ENCODE-DCC/encode-data-usage-examples/blob/master/ingest_encode_data_tile_db_with_s3_backend.ipynb" class="" target="_none">Ingesting ENCODE data into TileDB with S3 backend</a> by <a href="https://github.com/ottojolanki" target="_none">Otto Jolanki</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://academic.oup.com/nar/article/48/D1/D882/5622708#190992715" class="" target="_none">New developments on the Encyclopedia of DNA Elements (ENCODE) data portal</a> by <a href="http://orcid.org/0000-0001-6948-2042" target="_none">Luo et al 2020</a></a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="fsf-flood-risk" class="dataset">
        <h4><a href="../fsf-flood-risk/">First Street Foundation (FSF) Flood Risk Summary Statistics</a></h4>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://firststreet.org/flood-lab/" class="" target="_none">First Street Foundation Flood Lab</a> by First Street Foundation</a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://eos.org/opinions/do-you-know-your-homes-flood-risk" class="" target="_none">Do You Know Your Home’s Flood Risk?</a> by Edward Kearns, Jeremy Porter, Michael Amodeo</a>
            <br>
            
          </li>          <li>
            <a href="https://link.springer.com/article/10.1007/s11113-018-9473-5" class="" target="_none">Estimating Recent Local Impacts of Sea-Level Rise on Current Real-Estate Losses: A Housing Market Case Study in Miami-Dade, Florida</a> by Steven A. McAlpine, Jeremy R. Porter</a>
            <br>
            
          </li>          <li>
            <a href="https://agupubs.onlinelibrary.wiley.com/doi/full/10.1002/2017WR020917" class="" target="_none">Validation of a 30 m resolution flood hazard model of the conterminous United States</a> by Oliver E. J. Wing, Paul D. Bates, Christopher C. Sampson, Andrew M. Smith, Kris A. Johnson, Tyler A. Erickson</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="geoschem-input-data" class="dataset">
        <h4><a href="../geoschem-input-data/">GEOS-Chem Input Data</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="http://cloud-gc.readthedocs.io/en/stable/chapter02_beginner-tutorial/use-s3.html#access-geos-chem-input-data-repository-in-s3" class="" target="_none">Tutorial on accessing GEOS-Chem data bucket in S3</a> by <a href="https://github.com/JiaweiZhuang" target="_none">Jiawei Zhuang</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1175/BAMS-D-18-0243.1" class="" target="_none">Enabling Immediate Access to Earth Science Models through Cloud Computing: Application to the GEOS-Chem Model</a> by Jiawei Zhuang, et al.</a>
            <br>
            
          </li>          <li>
            <a href="http://acmg.seas.harvard.edu/research.html#cloud" class="" target="_none">Overview of the GEOSChem-on-cloud project</a> by <a href="http://acmg.seas.harvard.edu/index.html" target="_none">Atmospheric Chemistry Modeling Group, Harvard University</a></a>
            <br>
            
          </li>          <li>
            <a href="http://acmg.seas.harvard.edu/presentations/IGC8/posters/Mon_Posters_Zhuang_Jiawei.pdf" class="" target="_none">Running GEOS-Chem on Cloud Computing Platforms, presented at the 8th International GEOS-Chem Meeting</a> by <a href="https://github.com/JiaweiZhuang" target="_none">Jiawei Zhuang, et al.</a></a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="giab" class="dataset">
        <h4><a href="../giab/">Genome in a Bottle on AWS</a></h4>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/ga4gh/benchmarking-tools" class="" target="_none">GA4GH Benchmarking Tools</a> by GA4GH Benchmarking Team</a>
            <br>
            
          </li>          <li>
            <a href="https://github.com/genome-in-a-bottle/about_GIAB" class="" target="_none">The Genome in a Bottle Github Project</a> by Genome In A Bottle Consortium</a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://www.nature.com/articles/sdata201625" class="" target="_none">Extensive sequencing of seven human genomes to characterize benchmark reference materials</a> by Zook J et al (2016)</a>
            <br>
            
          </li>          <li>
            <a href="https://www.nature.com/articles/s41597-019-0098-2" class="" target="_none">High-coverage, long-read sequencing of Han Chinese trio reference samples</a> by Wang Y et al (2019)</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="ncbi-sra" class="dataset">
        <h4><a href="../ncbi-sra/">NIH NCBI Sequence Read Archive (SRA) on AWS</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/sra/docs/sra-aws-download/" class="" target="_none">Access SRA data using Amazon Web Services (AWS)</a> by <a href="https://www.ncbi.nlm.nih.gov/sra/" target="_none">NCBI SRA</a></a>
            <br>
            
          </li>          <li>
            <a href="https://www.ncbi.nlm.nih.gov/sra/docs/sra-cloud/" class="" target="_none">SRA in the Cloud</a> by <a href="https://www.ncbi.nlm.nih.gov/sra/" target="_none">NCBI SRA</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/ncbi/sra-tools" class="" target="_none">SRA Toolkit</a> by <a href="https://www.ncbi.nlm.nih.gov/sra/" target="_none">NCBI SRA</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3013647/" class="" target="_none">The Sequence Read Archive</a> by <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term&#x3D;Leinonen%20R%5BAuthor%5D&amp;cauthor&#x3D;true&amp;cauthor_uid&#x3D;21062823" target="_none">Leinonen et al (2011)</a></a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="osm" class="dataset">
        <h4><a href="../osm/">OpenStreetMap on AWS</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://aws.amazon.com/blogs/publicsector/develop-and-extract-value-from-open-data/" class="aws-link" target="_none">Develop and Extract Value from Open Data</a> by Daniel Bernao</a>
            <br>
            
          </li>          <li>
            <a href="https://aws.amazon.com/blogs/big-data/querying-openstreetmap-with-amazon-athena/" class="aws-link" target="_none">Querying OpenStreetMap with Amazon Athena</a> by <a href="https://github.com/mojodna" target="_none">Seth Fitzsimmons</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/developmentseed/mrworldwide/" class="" target="_none">OSM+Athena (GitHub)</a> by <a href="https://developmentseed.org/" target="_none">Development Seed</a></a>
            <br>
            
          </li>          <li>
            <a href="https://github.com/interline-io/planetutils" class="" target="_none">PlanetUtils (GitHub): Scripts and a Docker container to maintain your own OpenStreetMap planet</a> by <a href="https://www.interline.io/" target="_none">Interline Technologies</a></a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="refgenie" class="dataset">
        <h4><a href="../refgenie/">Refgenie reference genome assets</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="http://refgenie.databio.org/en/latest/tutorial/" class="" target="_none">Basic Refgenie tutorial</a> by <a href="http://www.databio.org" target="_none">Nathan Sheffield</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="http://pepatac.databio.org" class="" target="_none">Pipeline for ATAC-seq data analysis</a> by <a href="http://databio.org" target="_none">Jason Smith and Nathan Sheffield</a></a>
            <br>
            
          </li>          <li>
            <a href="http://peppro.databio.org" class="" target="_none">Pipeline for PRO-seq data analysis</a> by <a href="http://databio.org" target="_none">Jason Smith and Nathan Sheffield</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1093/gigascience/giz149" class="" target="_none">Refgenie: a reference genome resource manager</a> by <a href="http://www.databio.org" target="_none">Michał Stolarczyk, Vincent P Reuter, Jason P Smith, Neal E Magee, Nathan C Sheffield</a></a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="sentinel-1" class="dataset">
        <h4><a href="../sentinel-1/">Sentinel-1</a></h4>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="http://apps.sentinel-hub.com/eo-browser/" class="" target="_none">EO Browser</a> by <a href="http://www.sinergise.com/" target="_none">Sinergise</a></a>
            <br>
            
          </li>          <li>
            <a href="https://eos.com/landviewer/" class="" target="_none">EOS Land Viewer</a> by <a href="https://eos.com/" target="_none">Earth Observing System</a></a>
            <br>
            
          </li>          <li>
            <a href="http://www.sentinel-hub.com/apps/wms" class="" target="_none">Sentinel Hub WMS/WMTS/WCS Service by Sinergise</a> by <a href="http://www.sinergise.com/" target="_none">Sinergise</a></a>
            <br>
            
          </li>          <li>
            <a href="https://apps.sentinel-hub.com/sentinel-playground/?baseWmsUrl&#x3D;services.sentinel-hub.com&amp;instanceID&#x3D;a7c04225-b861-452d-ac89-7765352cd855&amp;source&#x3D;S2&amp;lat&#x3D;40.98676619345964&amp;lng&#x3D;28.89301300048828&amp;zoom&#x3D;13&amp;preset&#x3D;IW_VV&amp;layers&#x3D;B04,B03,B02&amp;maxcc&#x3D;20&amp;gain&#x3D;1.0&amp;gamma&#x3D;1.0&amp;time&#x3D;2015-01-01%7C2018-01-16&amp;atmFilter&#x3D;&amp;showDates&#x3D;false" class="" target="_none">Sentinel Playground</a> by <a href="http://www.sinergise.com/" target="_none">Sinergise</a></a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="sentinel-3" class="dataset">
        <h4><a href="../sentinel-3/">Sentinel-3</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/Sentinel-5P/data-on-s3/blob/master/notebooks/Sentinel3_Tutorial.ipynb" class="" target="_none">Accessing Sentinel-3 Data on S3 by MEEO</a> by <a href="http://www.meeo.it/" target="_none">Meteorological Envionmental Earth Observation</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://meeo-s3.s3.amazonaws.com/index.html#/?t&#x3D;catalogs" class="" target="_none">Catalogue of data set</a> by <a href="https://www.meeo.it/" target="_none">Meteorological Envionmental Earth Observation</a></a>
            <br>
            
          </li>          <li>
            <a href="https://step.esa.int/main/toolboxes/sentinel-3-toolbox/" class="" target="_none">Sentinel-3 Toolbox</a> by <a href="https://www.esa.int/" target="_none">European Space Agency</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://sentinel.esa.int/web/sentinel/user-guides" class="" target="_none">Sentinel-3 Document Library</a> by <a href="https://www.esa.int/" target="_none">European Space Agency</a></a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="sentinel5p" class="dataset">
        <h4><a href="../sentinel5p/">Sentinel-5P Level 2</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/Sentinel-5P/data-on-s3/blob/master/notebooks/Sentinel5P_Tutorial.ipynb" class="" target="_none">Accessing Sentinel-5P Data on S3 by MEEO</a> by <a href="http://www.meeo.it/" target="_none">Meteorological Envionmental Earth Observation</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://meeo-s5p.s3.amazonaws.com/index.html#/?t&#x3D;catalogs" class="" target="_none">Catalogue of data set</a> by <a href="https://www.meeo.it/" target="_none">Meteorological Envionmental Earth Observation</a></a>
            <br>
            
          </li>          <li>
            <a href="https://atmospherictoolbox.org/" class="" target="_none">The Atmospheric Toolbox</a> by <a href="https://www.esa.int/" target="_none">European Space Agency</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://sentinel.esa.int/web/sentinel/user-guides/sentinel-5p-tropomi/document-library" class="" target="_none">Sentinel-5P TROPOMI Document Library</a> by <a href="https://www.esa.int/" target="_none">European Space Agency</a></a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="broad-pan-ukb" class="dataset">
        <h4><a href="../broad-pan-ukb/">UK Biobank Pan-Ancestry Summary Statistics</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://hail.is/docs/0.2/tutorials-landing.html" class="" target="_none">Hail Tutorials</a> by <a href="https://hail.is/" target="_none">Hail Team</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://hail.is" class="" target="_none">Hail</a> by <a href="https://hail.is/" target="_none">Hail Team</a></a>
            <br>
            
          </li>          <li>
            <a href="https://aws.amazon.com/quickstart/architecture/hail/" class="aws-link" target="_none">Hail on AWS Quick Start</a> by Amazon Web Services and PrivoIT</a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://pan.ukbb.broadinstitute.org/" class="" target="_none">Pan-ancestry genetic analysis of the UK Biobank</a> by Pan UKBB Team</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="ycb-benchmarks" class="dataset">
        <h4><a href="../ycb-benchmarks/">Yale-CMU-Berkeley (YCB) Object and Model Set</a></h4>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://ieeexplore.ieee.org/document/7254318" class="" target="_none">Benchmarking in Manipulation Research: Using the Yale-CMU-Berkeley Object and Model Set</a> by <a href="https://www.wpi.edu/people/faculty/bcalli" target="_none">Berk Calli, Aaron Walsman, Arjun Singh, Siddhartha Srinivasa, Pieter Abbeel, Aaron M Dollar</a></a>
            <br>
            
          </li>          <li>
            <a href="https://ieeexplore.ieee.org/abstract/document/8460950" class="" target="_none">Label Fusion: A Pipeline for Generating Ground Truth Labels for Real RGBD Data of Cluttered Scenes</a> by <a href="https://groups.csail.mit.edu/locomotion/russt.html" target="_none">Pat Marion, Peter R. Florence, Lucas Manuelli, Russ Tedrake</a></a>
            <br>
            
          </li>          <li>
            <a href="https://ieeexplore.ieee.org/abstract/document/7989594" class="" target="_none">Pre-touch sensing for sequential manipulation</a> by <a href="https://sensor.cs.washington.edu/jrs.html" target="_none">Boling Yang, Patrick Lancaster, Joshua R. Smith</a></a>
            <br>
            
          </li>          <li>
            <a href="https://ieeexplore.ieee.org/abstract/document/8304743" class="" target="_none">The Closure Signature: A Functional Approach to Model Underactuated Compliant Robotic Hands</a> by <a href="http://www.dii.unisi.it/~domenico/" target="_none">Maria Pozzi, Gionata Salvietti, João Bimbo, Monica Malvezzi, Domenico Prattichizzo</a></a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="allen-mouse-brain-atlas" class="dataset">
        <h4><a href="../allen-mouse-brain-atlas/">Allen Mouse Brain Atlas</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/AllenInstitute/open_dataset_tools/blob/master/Visualizing_Images_from_Allen_Mouse_Brain_Atlas.ipynb" class="" target="_none">Visualizing Images from the Allen Mouse Brain Atlas</a> by <a href="www.alleninstitute.org" target="_none">Allen Institute for Brain Science</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://mouse.brain-map.org" class="" target="_none">Allen Mouse Brain Atlas</a> by <a href="www.alleninstitute.org" target="_none">Allen Institute for Brain Science</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="http://www.nature.com/articles/nature05453" class="" target="_none">Genome-wide atlas of gene expression in the adult mouse brain</a> by <a href="www.alleninstitute.org" target="_none">Ed Lein, et al.</a></a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="ncbi-blast-databases" class="dataset">
        <h4><a href="../ncbi-blast-databases/">Basic Local Alignment Sequences Tool (BLAST) Databases</a></h4>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/ncbi/blast_plus_docs" class="" target="_none">BLAST+ Docker</a> by NCBI BLAST</a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://pubmed.ncbi.nlm.nih.gov/20003500" class="" target="_none">BLAST+: Architecture and Applications</a> by Christiam Camacho  1 , George Coulouris, Vahram Avagyan, Ning Ma, Jason Papadopoulos, Kevin Bealer, Thomas L Madden</a>
            <br>
            
          </li>          <li>
            <a href="https://pubmed.ncbi.nlm.nih.gov/9254694/" class="" target="_none">Gapped BLAST and PSI-BLAST: A New Generation of Protein Database Search Programs</a> by S F Altschul, T L Madden, A A Schäffer, J Zhang, Z Zhang, W Miller, D J Lipman</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="beataml" class="dataset">
        <h4><a href="../beataml/">Beat Acute Myeloid Leukemia (AML) 1.0</a></h4>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://portal.gdc.cancer.gov/projects/" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/30651561" class="" target="_none">Clinical resistance to crenolanib in acute myeloid leukemia due to diverse molecular
          mechanisms
          </a> by Zhang H, Savage S, Schultz AR, Bottomly D, White L, Segerdell E, et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/30333627" class="" target="_none">Functional Genomic Landscape of Acute Myeloid Leukemia</a> by Jeffrey W. Tyner, Cristina E. Tognon, Dan Bottomly et al.</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="talend-covid19" class="dataset">
        <h4><a href="../talend-covid19/">COVID-19 Harmonized Data</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://www.talend.com/blog/2020/04/05/talend-joins-fight-against-covid-19-unlocking-data-for-health-researchers/" class="" target="_none">How Talend is joining the fight against COVID-19</a> by <a href="https://www.linkedin.com/in/thomasbennettw/" target="_none">Thomas Bennett</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://www.stitchdata.com/integrations/covid-19/" class="" target="_none">Stitch COVID-19 Integration</a> by <a href="https://github.com/jeffhuth-bytecode" target="_none">Jeff Huth</a></a>
            <br>
            
          </li>          <li>
            <a href="https://github.com/singer-io/tap-covid-19" class="" target="_none">Tap COVID-19 Python</a> by <a href="https://github.com/jeffhuth-bytecode" target="_none">Jeff Huth</a></a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="ctsp-dlbcl" class="dataset">
        <h4><a href="../ctsp-dlbcl/">Clinical Trial Sequencing Project - Diffuse Large B-Cell Lymphoma</a></h4>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://portal.gdc.cancer.gov/projects/CTSP-DLBCL1" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed?cmd&#x3D;DetailsSearch&amp;term&#x3D;29925955[PMID]" class="" target="_none">A multiprotein supercomplex controlling oncogenic signalling in lymphoma</a> by Phelan JD, Young RM, Webster DE, Roulland S, Wright GW, Kasbekar M, Shaffer AL 3rd,
          Ceribelli M, Wang JQ, Schmitz R, Nakagawa M, Bachy E, Huang DW, Ji Y, Chen L, Yang Y, Zhao
          H, Yu X, Xu W, Palisoc MM, Valadez RR, Davies-Hill T, Wilson WH, Chan WC, Jaffe ES, Gascoyne
          RD, Campo E, Rosenwald A, Ott G, Delabie J, Rimsza LM, Rodriguez FJ, Estephan F, Holdhoff M,
          Kruhlak MJ, Hewitt SM, Thomas CJ, Pittaluga S, Oellerich T, Staudt LM
          </a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1056/NEJMoa1801445" class="" target="_none">Genetics and Pathogenesis of Diffuse Large B Cell Lymphoma</a> by Roland Schmitz, Ph.D., George W. Wright, Ph.D., Da Wei Huang, M.D., Calvin A.  Johnson,
          Ph.D., James D. Phelan, Ph.D., James Q. Wang, Ph.D., Sandrine Roulland, Ph.D., Monica
          Kasbekar, Ph.D., Ryan M. Young, Ph.D., Arthur L. Shaffer, Ph.D., Daniel J. Hodson, M.D.,
          Ph.D., Wenming Xiao, Ph.D., et al.
          </a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="deutsche-boerse-pds" class="dataset">
        <h4><a href="../deutsche-boerse-pds/">Deutsche Börse Public Dataset</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://aws.amazon.com/blogs/big-data/10-visualizations-to-try-in-amazon-quicksight-with-sample-data/" class="aws-link" target="_none">10 visualizations to try in Amazon QuickSight with sample data</a> by <a href="https://aws.amazon.com/blogs/big-data/" target="_none">AWS Big Data Blog</a></a>
            <br>
            
          </li>          <li>
            <a href="https://github.com/Originate/dbg-pds-tensorflow-demo" class="" target="_none">Stock Price Movement Prediction Using The Deutsche Börse Public Dataset &amp; Machine Learning</a> by <a href="https://www.originate.com" target="_none">Originate</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="http://live.thermobook.net/dataStreamingWeb" class="" target="_none">Streaming XETRA Data Using Apache Spark</a> by <a href="http://www.thermobook.net" target="_none">Thermobook</a></a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="dandiarchive" class="dataset">
        <h4><a href="../dandiarchive/">Distributed Archives for Neurophysiology Data Integration (DANDI)</a></h4>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://hub.dandiarchive.org/" class="" target="_none">DANDI JupyterHub Interface</a> by <a href="https://dandiarchive.org/" target="_none">DANDI Project</a></a>
            <br>
            
          </li>          <li>
            <a href="https://github.com/dandi/dandi-cli" class="" target="_none">DANDI Shell Interface</a> by <a href="https://dandiarchive.org/" target="_none">DANDI Project</a></a>
            <br>
            
          </li>          <li>
            <a href="https://dandiarchive.org/" class="" target="_none">DANDI Web Interface</a> by <a href="https://dandiarchive.org/" target="_none">DANDI Project</a></a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="fmi-radar" class="dataset">
        <h4><a href="../fmi-radar/">Finnish Meteorological Institute Weather Radar Data</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/fmidev/opendata-resources/tree/master/examples/qgis/radar-composite" class="" target="_none">Handling data with QGIS</a> by Markus Peura</a>
            <br>
            
          </li>          <li>
            <a href="https://github.com/fmidev/opendata-resources/blob/master/examples/python/Accumulated_prec_amount_from_radar_GeoTiff.ipynb" class="" target="_none">Processing GeoTIFF data with python</a> by Roope Tervo</a>
            <br>
            
          </li>          <li>
            <a href="https://github.com/fmidev/opendata-resources/blob/master/examples/python/radar-rhi-from-hdf5.ipynb" class="" target="_none">Processing HDF5 data with python</a> by Roope Tervo</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="fm-ad" class="dataset">
        <h4><a href="../fm-ad/">Foundation Medicine Adult Cancer Clinical Dataset (FM-AD)</a></h4>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://portal.gdc.cancer.gov/projects/FM-AD" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed?cmd&#x3D;DetailsSearch&amp;term&#x3D;28235761[PMID]" class="" target="_none">High-Throughput Genomic Profiling of Adult Solid Tumors Reveals Novel Insights into Cancer
          Pathogenesis
          </a> by Ryan J. Hartmaier, Lee A. Albacker, Juliann Chmielecki, Mark Bailey, Jie He, Michael E.
          Goldberg, Shakti Ramkissoon, James Suh, Julia A. Elvin, Samuel Chiacchia, Garrett M.
          Frampton, Jeffrey S. Ross, Vincent Miller, Philip J. Stephens and Doron Lipson
          </a>
            <br>
            
          </li>          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed?cmd&#x3D;DetailsSearch&amp;term&#x3D;22981675[PMID]" class="" target="_none">Targeted next-generation sequencing of advanced prostate cancer identifies potential
          therapeutic targets and disease heterogeneity.
          </a> by Beltran H, Yelensky R, Frampton GM, Park K, Downing SR, MacDonald TY, Jarosz M, Lipson D,
          Tagawa ST, Nanus DM, Stephens PJ, Mosquera JM, Cronin MT, Rubin MA
          </a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="noaa-himawari" class="dataset">
        <h4><a href="../noaa-himawari/">JMA Himawari-8</a></h4>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://docs.opendata.aws/noaa-himawari/2020July08_NOAA_Himawari.pdf" class="aws-link" target="_none">Himawari-8 Advanced Himawari Imager Data on AWS (pdf file)</a> by NOAA NESDIS</a>
            <br>
            
          </li>          <li>
            <a href="https://docs.opendata.aws/noaa-himawari/2020July08_Amazon_Himawari.pdf" class="aws-link" target="_none">Himawari-8 on AWS (pdf file)</a> by ASDI</a>
            <br>
            
          </li>          <li>
            <a href="https://docs.opendata.aws/noaa-himawari/2020July07_JMA_Himawari.pdf" class="aws-link" target="_none">Introduction of Himawari-8/9 (pdf file)</a> by JMA</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="cotonoha-dic" class="dataset">
        <h4><a href="../cotonoha-dic/">Japanese Tokenizer Dictionaries</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/polm/fugashi-sagemaker-demo/blob/master/fugashi%20wordcount.ipynb" class="" target="_none">Fugashi Word Count Tutorial</a> by <a href="https://cotonoha.io" target="_none">Paul O&#x27;Leary McCann</a></a>
            <br>
            <span class="label label-info tag service-tag">SageMaker</span>
          </li>        </ul>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/polm/unidic-py" class="" target="_none">unidic-py</a> by <a href="https://cotonoha.io" target="_none">Paul O&#x27;Leary McCann</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://www.dampfkraft.com/nlp/how-to-tokenize-japanese.html" class="" target="_none">How to Tokenize Japanese in Python</a> by <a href="https://dampfkraft.com" target="_none">Paul O&#x27;Leary McCann</a></a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="msd" class="dataset">
        <h4><a href="../msd/">Medical Segmentation Decathlon</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/Project-MONAI/tutorials" class="" target="_none">Pytorch-Integrated MSD Data Loader</a> by <a href="https://github.com/Project-MONAI/MONAI" target="_none">MONAI Development Team</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="www.monai.io" class="" target="_none">MONAI: Getting Started</a> by <a href="https://monai.io/start.html" target="_none">MONAI Development Team</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://arxiv.org/pdf/1902.09063.pdf" class="" target="_none">A large annotated medical image dataset for the development and evaluation of segmentation algorithms</a> by Simpson A. L., Antonelli M., Bakas S., Bilello M., Farahana K., van Ginneken B., et al</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="nasanex" class="dataset">
        <h4><a href="../nasanex/">NASA NEX</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="http://cloud-gc.readthedocs.io/en/stable/chapter02_beginner-tutorial/use-s3.html#access-nasa-nex-data-in-s3-optional-but-recommended" class="" target="_none">Accessing and plotting NASA-NEX data, from GEOSChem-on-cloud tutorial.</a> by <a href="https://github.com/JiaweiZhuang" target="_none">Jiawei Zhuang</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://climate.azavea.com" class="" target="_none">Azavea Climate API</a> by <a href="https://www.azavea.com/" target="_none">Azavea</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://www.kdd.org/kdd2020/accepted-papers/view/climate-downscaling-using-ynet-a-deep-convolutional-network-with-skip-conne" class="" target="_none">Climate Downscaling Using YNet: a Deep Convolutional Network with Skip Connections and Fusion</a> by Yumin Liu, Auroop Ganguly, and Jennifer Dy</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="noaa-gefs-reforecast" class="dataset">
        <h4><a href="../noaa-gefs-reforecast/">NOAA Global Ensemble Forecast System (GEFS) Re-forecast</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/vgensini/gefs_v12_example/blob/master/GEFS_v12_eample.ipynb" class="" target="_none">Example using GEFS Reforecast Data from AWS</a> by <a href="https://atlas.niu.edu" target="_none">Victor Gensini</a></a>
            <br>
            
          </li>          <li>
            <a href="https://mogismog.github.io/retro-diss-part-3.html#retro-diss-part-3" class="" target="_none">Retrospective Dissertation - Part 3: Reforecast Data</a> by <a href="https://github.com/mogismog" target="_none">Francisco Alvarez</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://drive.google.com/file/d/1lCIB3qz9S1nu2qAaetnLq0N5ReyTXuHd/view" class="" target="_none">The GEFS v12 and its reanalyses and reforecasts (Slides)</a> by Tom Hamill</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="noaa-ghcn" class="dataset">
        <h4><a href="../noaa-ghcn/">NOAA Global Historical Climatology Network Daily (GHCN-D)</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://aws.amazon.com/blogs/industries/calculating-growing-degree-days-using-aws-registry-of-open-data/" class="aws-link" target="_none">Calculating growing degree days using AWS Registry of Open Data</a> by Karen Hildebrand and Zac Flamig</a>
            <br>
            
          </li>          <li>
            <a href="https://medium.com/swlh/explore-visualize-200-years-of-global-temperature-using-apache-spark-bigquery-and-google-data-699ed8b3c67?sk&#x3D;3721ca3220a75af80293ac3e4bf3f4d1" class="" target="_none">Explore &amp; Visualize 200+ Years of Global Temperature</a> by Kapil Sreedharan</a>
            <br>
            
          </li>          <li>
            <a href="https://aws.amazon.com/blogs/big-data/visualize-over-200-years-of-global-climate-data-using-amazon-athena-and-amazon-quicksight/" class="aws-link" target="_none">Visualize over 200 years of global climate data using Amazon Athena and Amazon QuickSight</a> by Conor Delaney</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="nrel-pds-nsrdb" class="dataset">
        <h4><a href="../nrel-pds-nsrdb/">NREL National Solar Radiation Database</a></h4>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/NREL/hsds-examples" class="" target="_none">HSDS Examples</a> by Caleb Phillips, Caroline Draxl, John Readey, Jordan Perr-Sauer, Michael Rossol</a>
            <br>
            
          </li>          <li>
            <a href="https://nsrdb.nrel.gov/nsrdb-viewer" class="" target="_none">NSRDB Viewer</a> by Manajit Sengupta, Yu Xe, Anthony Lopez, Aron Habte, Galen Maclaurin, James Shelby, Paul Edwards</a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://www.sciencedirect.com/science/article/pii/S136403211830087X" class="" target="_none">The National Solar Radiation Data Base (NSRDB)</a> by Manajit Sengupta, Yu Xe, Anthony Lopez, Aron Habte, Galen Maclaurin, James Shelby</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="nsw-herbarium" class="dataset">
        <h4><a href="../nsw-herbarium/">National Herbarium of NSW</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://www.rbgsyd.nsw.gov.au/science/national-herbarium-of-new-south-wales" class="" target="_none">Accessing the National Herbarium of NSW on AWS</a> by <a href="https://www.rbgsyd.nsw.gov.au/Science/Our-Science-Staff" target="_none">Dr Shelley James</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://www.ala.org.au/" class="" target="_none">Atlas of Living Australia</a> by <a href="https://www.ala.org.au/about-ala/" target="_none">CSIRO</a></a>
            <br>
            
          </li>          <li>
            <a href="https://avh.ala.org.au/occurrences/search?q&#x3D;collection_uid%3Aco54#tab_mapView" class="" target="_none">The Australasian Virtual Herbarium</a> by <a href="https://avh.chah.org.au/index.php/about/" target="_none">avh@chah.org.au</a></a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="grillo-openeew" class="dataset">
        <h4><a href="../grillo-openeew/">OpenEEW</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://openeew.com/docs/analyze-historic" class="" target="_none">Analyzing a magnitude 7.2 earthquake in Mexico using Python</a> by <a href="https://grillo.io/" target="_none">Grillo</a></a>
            <br>
            
          </li>          <li>
            <a href="https://openeew.com/docs/machine-learning" class="" target="_none">Developing a machine learning model for better earthquake detection</a> by <a href="https://grillo.io/" target="_none">Grillo</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/openeew/openeew-python" class="" target="_none">OpenEEW library for Python</a> by <a href="https://grillo.io/" target="_none">Grillo</a></a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="nrel-pds-porotomo" class="dataset">
        <h4><a href="../nrel-pds-porotomo/">PoroTomo</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/openEDI/documentation/blob/master/PoroTomo_Distributed_Acoustic_Sensing_(DAS)_Data.ipynb" class="" target="_none">PoroTomo DAS Data Processing Tutorial</a> by <a href="https://orcid.org/0000-0001-7776-2028" target="_none">Nicole Taverna and Ross Ring-Jarvi</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://pangea.stanford.edu/ERE/pdf/IGAstandard/SGW/2018/Miller.pdf" class="" target="_none">DAS and DTS at Brady Hot Springs: Observations about Coupling and Coupled Interpretations</a> by Douglas E. Miller, Thomas Coleman, Xiangfang Zeng, Jeremy R. Patterson , Elena C. Reinnisch, Michael A. Cardiff, Herbert F. Wang, Dante Fratta, Whitney Trainor-Guitton, Clifford H. Thurber, Michelle ROBERTSON, Kurt FEIGL, and The PoroTomo Team</a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1093/gji/ggy182" class="" target="_none">Ground motion response to an ML 4.3 earthquake using co-located distributed acoustic sensing and seismometer arrays</a> by Herbert F Wang, Xiangfang Zeng, Douglas E Miller, Dante Fratta, Kurt L Feigl, Clifford H Thurber, Robert J Mellors</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="silo" class="dataset">
        <h4><a href="../silo/">SILO climate data on AWS</a></h4>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://www.longpaddock.qld.gov.au/silo/gridded-data/conversion" class="" target="_none">Convert NetCDF to ESRI ArcASCII or GeoTIFF</a> by SILO</a>
            <br>
            
          </li>          <li>
            <a href="https://www.longpaddock.qld.gov.au/silo/gridded-data/nco" class="" target="_none">NetCDF Operators to calculate seasonal means</a> by SILO</a>
            <br>
            
          </li>          <li>
            <a href="https://www.longpaddock.qld.gov.au/silo/gridded-data/python" class="" target="_none">Python script to calculate a regional mean</a> by SILO</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="sorel-20m" class="dataset">
        <h4><a href="../sorel-20m/">Sophos/ReversingLabs 20 Million malware detection dataset</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/sophos-ai/SOREL-20M#quickstart" class="" target="_none">SOREL-20M quickstart</a> by Richard Harang</a>
            <br>
            <span class="label label-info tag service-tag">S3</span><span class="label label-info tag service-tag">EC2</span>
          </li>        </ul>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/sophos-ai/SOREL-20M" class="" target="_none">SOREL-20M dataset interface code</a> by Richard Harang and Ethan M Rudd</a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://arxiv.org/abs/2012.07634" class="" target="_none">SOREL-20M: A Large Scale Benchmark Dataset for Malicious PE Detection
          </a> by Richard Harang and Ethan M Rudd</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="sevir" class="dataset">
        <h4><a href="../sevir/">Storm EVent ImageRy (SEVIR)</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://nbviewer.jupyter.org/github/MIT-AI-Accelerator/eie-sevir/blob/master/examples/SEVIR_Tutorial.ipynb" class="" target="_none">Introduction to SEVIR</a> by <a href="https://github.com/markveillette" target="_none">Mark Veillette</a></a>
            <br>
            
          </li>          <li>
            <a href="https://nbviewer.jupyter.org/github/MIT-AI-Accelerator/eie-sevir/blob/master/examples/Generators.ipynb" class="" target="_none">Using Generators for SEVIR data</a> by <a href="https://github.com/markveillette" target="_none">Mark Veillette</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/MIT-AI-Accelerator/eie-sevir" class="" target="_none">sevir -- python utilities for working with SEVIR dataset</a> by <a href="https://github.com/markveillette" target="_none">Mark Veillette</a></a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="human-microbiome-project" class="dataset">
        <h4><a href="../human-microbiome-project/">The Human Microbiome Project</a></h4>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://www.sciencedirect.com/science/article/pii/S2452231718300289" class="" target="_none">New microbe genomic variants in patients fecal community following surgical disruption of the upper human gastrointestinal tract</a> by Ranjit Kumar, Jayleen Grams, Daniel I. Chu, David K.Crossman, Richard Stahl, Peter Eipers, et al</a>
            <br>
            
          </li>          <li>
            <a href="https://www.nature.com/articles/nature23889" class="" target="_none">Strains, functions and dynamics in the expanded Human Microbiome Project</a> by Jason Lloyd-Price, Anup Mahurkar, Gholamali Rahnavard, Jonathan Crabtree, Joshua Orvis, A. Brantley Hall, et al.</a>
            <br>
            
          </li>          <li>
            <a href="https://www.nature.com/articles/nature06244" class="" target="_none">The Human Microbiome Project</a> by Peter J. Turnbaugh, Ruth E. Ley, Micah Hamady, Claire M. Fraser-Liggett, Rob Knight &amp; Jeffrey I. Gordon</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="hail-vep-pipeline" class="dataset">
        <h4><a href="../hail-vep-pipeline/">Variant Effect Predictor (VEP) and the Loss-Of-Function Transcript Effect Estimator (LOFTEE) Plugin</a></h4>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://hail.is/" class="" target="_none">Hail</a> by <a href="https://www.nealelab.is/" target="_none">Neale Lab</a></a>
            <br>
            
          </li>          <li>
            <a href="https://github.com/konradjk/loftee" class="" target="_none">Loss-Of-Function Transcript Effect Estimator (LOFTEE)</a> by <a href="https://github.com/konradjk" target="_none">Konrad Karczewski</a></a>
            <br>
            
          </li>          <li>
            <a href="http://uswest.ensembl.org/info/docs/tools/vep/index.html" class="" target="_none">Variant Effect Predictor (VEP)</a> by <a href="http://uswest.ensembl.org/info/docs/tools/vep/index.html" target="_none">Ensembl</a></a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="afsis" class="dataset">
        <h4><a href="../afsis/">Africa Soil Information Service (AfSIS) Soil Chemistry</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/qedsoftware/afsis-soil-chem-tutorial/" class="" target="_none">AfSIS Soil Chemistry - Usage Tutorial</a> by <a href="https://qed.ai" target="_none">QED</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://www.youtube.com/watch?v&#x3D;Fb9R0CnPMkc" class="" target="_none">Goalkeepers 2018, Soil - The Big Data Beneath Your Feet</a> by <a href="https://qed.ai" target="_none">QED</a></a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="amazon-bin-imagery" class="dataset">
        <h4><a href="../amazon-bin-imagery/">Amazon Bin Image Dataset</a></h4>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/silverbottlep/abid_challenge" class="" target="_none">Amazon Bin Image Dataset Challenge</a> by <a href="https://github.com/silverbottlep" target="_none">silverbottlep</a></a>
            <br>
            
          </li>          <li>
            <a href="https://github.com/OneNow/AI-Inventory-Reconciliation" class="" target="_none">Amazon Inventory Reconciliation using AI</a> by <a href="https://github.com/pablo-tech" target="_none">Pablo Rodriguez Bertorello, Sravan Sripada, Nutchapol Dendumrongsup</a></a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="meteo-france-models" class="dataset">
        <h4><a href="../meteo-france-models/">Atmospheric Models from Météo-France</a></h4>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://windguru.cz" class="" target="_none">Windguru.cz</a> by <a href="https://windguru.cz" target="_none">Windguru</a></a>
            <br>
            
          </li>          <li>
            <a href="https://windy.com" class="" target="_none">Windy.com</a> by <a href="https://windy.com" target="_none">Windy</a></a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="broad-references" class="dataset">
        <h4><a href="../broad-references/">Broad Genome References</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://aws.amazon.com/blogs/storage/using-amazon-fsx-for-lustre-for-genomics-workflows-on-aws/" class="aws-link" target="_none">Using Amazon FSx for Lustre for Genomics Workflows on AWS</a> by <a href="https://www.linkedin.com/in/lee-pang-a039a26/" target="_none">W. Lee Pang</a></a>
            <br>
            <span class="label label-info tag service-tag">Batch</span><span class="label label-info tag service-tag">FSx</span>
          </li>        </ul>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://docs.opendata.aws/genomics-workflows/orchestration/cromwell/cromwell-examples/#real-world-example-haplotypecaller" class="aws-link" target="_none">Genomics Workflows on AWS - Cromwell on AWS</a> by <a href="https://www.linkedin.com/in/lee-pang-a039a26/" target="_none">W. Lee Pang</a></a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="cgci" class="dataset">
        <h4><a href="../cgci/">Cancer Genome Characterization Initiatives - Burkitt Lymphoma, HIV+ Cervical Cancer</a></h4>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://portal.gdc.cancer.gov/projects/CGCI-BLGSP" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1182/blood-2018-09-871418" class="" target="_none">Genome-wide discovery of somatic coding and noncoding mutations in pediatric endemic and
          sporadic Burkitt lymphoma
          </a> by Grande B. M., Gerhard D. S., Jiang A., Griner N. B., Abramson J. S., Alexander T. B., et al.
          </a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="janelia-cosem" class="dataset">
        <h4><a href="../janelia-cosem/">Cell Organelle Segmentation in Electron Microscopy (COSEM) on AWS</a></h4>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://science.sciencemag.org/content/367/6475/eaaz5357" class="" target="_none">Correlative three-dimensional super-resolution and block-face electron microscopy of whole vitreously frozen cells.</a> by David P. Hoffman1, Gleb Shtengel, C. Shan Xu, Kirby R. Campbell, Melanie Freeman, Lei Wang, Daniel E. Milkie, H. Amalia Pasolli, Nirmala Iyer, John A. Bogovic, Daniel R. Stabley, Abbas Shirinifard, Song Pang, David Peale, Kathy Schaefer, Wim Pomp, Chi-Lun Chang, Jennifer Lippincott-Schwartz, Tom Kirchhausen1, David J. Solecki, Eric Betzig, Harald F. Hess.</a>
            <br>
            
          </li>          <li>
            <a href="https://elifesciences.org/articles/25916" class="" target="_none">Enhanced FIB-SEM systems for large-volume 3D imaging</a> by C. Shan Xu, Kenneth J. Hayworth, Zhiyuan Lu, Patricia Grob, Ahmed M. Hassan, José G. García-Cerdán, Krishna K. Niyogi, Eva Nogales, Richard J. Weinberg, Harald F. Hess.</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="jhu-indexes" class="dataset">
        <h4><a href="../jhu-indexes/">Cloud Indexes for Genomic Analyses</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://benlangmead.github.io/aws-indexes/tutorial" class="" target="_none">Table of contents for tutorials for constituent tools</a> by <a href="https://benlangmead.github.io/aws-indexes" target="_none">Ben Langmead</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://www.biorxiv.org/content/10.1101/2020.03.03.975219v4.full.pdf" class="" target="_none">Reducing reference bias using multiple population reference genomes</a> by Chen et al (2020)</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="copernicus-dem" class="dataset">
        <h4><a href="../copernicus-dem/">Copernicus Digital Elevation Model (DEM)</a></h4>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="http://apps.sentinel-hub.com/eo-browser/" class="" target="_none">EO Browser</a> by <a href="http://www.sinergise.com/" target="_none">Sinergise</a></a>
            <br>
            
          </li>          <li>
            <a href="http://www.sentinel-hub.com/" class="" target="_none">Sentinel Hub WMS/WMTS/WCS Service and Process API</a> by <a href="http://www.sinergise.com/" target="_none">Sinergise</a></a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="cmip6" class="dataset">
        <h4><a href="../cmip6/">Coupled Model Intercomparison Project 6</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://nbviewer.jupyter.org/github/awslabs/amazon-asdi/blob/main/examples/dask/notebooks/cmip6_zarr.ipynb" class="aws-link" target="_none">Processing CMIP6 data in Zarr format with Dask &amp; AWS Fargate</a> by Zac Flamig</a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://gmd.copernicus.org/articles/special_issue590.html" class="" target="_none">Special issue | Coupled Model Intercomparison Project Phase 6 (CMIP6) Experimental Design and Organization</a> by V. Eyring</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="digitalcorpora" class="dataset">
        <h4><a href="../digitalcorpora/">DigitalCorpora</a></h4>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="http://www.simson.net/clips/academic/2009.DFRWS.Corpora.pdf" class="" target="_none">Bringing Science to Digital Forensics with Standardized Forensic Corpora</a> by <a href="https://simson.net/" target="_none">Garfinkel, Farrell, Roussev and Dinolt</a></a>
            <br>
            
          </li>          <li>
            <a href="http://simson.net/clips/academic/2011.ADFSL.Corpora.pdf" class="" target="_none">Creating Realistic Corpora for Forensic and Security Education</a> by <a href="https://simson.net/" target="_none">Woods, K., Christopher Lee, Simson Garfinkel, David Dittrich, Adam Russel, Kris Kearton</a></a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="hst" class="dataset">
        <h4><a href="../hst/">Hubble Space Telescope Public Data</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://mast-labs.stsci.io/2018/06/exploring-aws-lambda-with-hst-public-data" class="" target="_none">Exploring AWS Lambda with cloud-hosted Hubble public data</a> by <a href="https://twitter.com/arfon" target="_none">Arfon Smith</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://mast-labs.stsci.io/2018/06/hst-public-data-on-aws" class="" target="_none">Making HST Public Data Available on AWS</a> by <a href="https://twitter.com/arfon" target="_none">Arfon Smith</a></a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="mimiciii" class="dataset">
        <h4><a href="../mimiciii/">MIMIC-III (‘Medical Information Mart for Intensive Care’)</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://aws.amazon.com/blogs/big-data/perform-biomedical-informatics-without-a-database-using-mimic-iii-data-and-amazon-athena/" class="aws-link" target="_none">Perform biomedical informatics without a database using MIMIC-III data and Amazon Athena</a> by James Wiggins, Alistair Johnson</a>
            <br>
            <span class="label label-info tag service-tag">Athena</span><span class="label label-info tag service-tag">Glue</span>
          </li>        </ul>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/MIT-LCP/mimic-code" class="" target="_none">MIMIC-code GitHub repository</a> by <a href="https://github.com/alistairewj" target="_none">Alistair Johnson</a></a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="naip" class="dataset">
        <h4><a href="../naip/">NAIP on AWS</a></h4>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://eos.com/landviewer/" class="" target="_none">EOS Land Viewer</a> by <a href="https://eos.com/" target="_none">Earth Observing System</a></a>
            <br>
            
          </li>          <li>
            <a href="https://www.voyagersearch.com/using-aws-batch-to-generate-image-thumbnails-for-voyager" class="" target="_none">VoyagerSearch showing off Batch + NAIP</a> by <a href="https://www.voyagersearch.com/" target="_none">Voyager</a></a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="noaa-cfs" class="dataset">
        <h4><a href="../noaa-cfs/">NOAA Climate Forecast System (CFS)</a></h4>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://journals.ametsoc.org/view/journals/bams/91/8/2010bams3001_1.xml" class="" target="_none">The NCEP Climate Forecast System Reanalysis</a> by Saha, Suranjana, and Coauthors</a>
            <br>
            
          </li>          <li>
            <a href="https://journals.ametsoc.org/view/journals/clim/27/6/jcli-d-12-00823.1.xml" class="" target="_none">The NCEP Climate Forecast System Version 2</a> by Saha, Suranjana, and Coauthors</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="nwm-archive" class="dataset">
        <h4><a href="../nwm-archive/">NOAA National Water Model Reanalysis</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://nbviewer.jupyter.org/gist/rsignell-usgs/d3dfaf3cd3d8b39894a69b22127dfe38" class="" target="_none">Explore the National Water Model Reanalysis in Zarr</a> by <a href="https://www.usgs.gov/staff-profiles/richard-p-signell" target="_none">Rich Signell</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1016/j.ocemod.2019.101526" class="" target="_none">Simulating storm surge and compound flooding events with a creek-to-ocean model: Importance of baroclinic effects</a> by Fei Ye, et al.</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="noaa-ofs" class="dataset">
        <h4><a href="../noaa-ofs/">NOAA Operational Forecast System (OFS)</a></h4>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://opendap.co-ops.nos.noaa.gov/ofsdataagg/index.html" class="" target="_none">OFS Data Aggregation and Sub-Setting</a> by NOAA</a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://www.weather.gov/media/notification/pdf2/scn21-14nos_wcofs_ngofs2aaa.pdf" class="" target="_none">Implementation of new Oceanographic Forecast Modeling System for the U.S. West Coast (WCOFS) and the Upgraded Northern Gulf of Mexico (NGOFS2)</a> by NOAA</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="noaa-wod" class="dataset">
        <h4><a href="../noaa-wod/">NOAA World Ocean Database (WOD)</a></h4>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://data.nodc.noaa.gov/woa/WOD/DOC/wod_intro.pdf" class="" target="_none">The World Ocean Database Introduction</a> by Tim P. Boyer, Olga K. Baranova, Carla Coleman, Hernan E. Garcia, Alexandra Grodsky, Ricardo A. Locarnini, Alexey V. Mishonov, Christopher R. Paver, James R. Reagan, Dan Seidov, Igor V. Smolyar, Katharine W. Weathers, Melissa M. Zweng</a>
            <br>
            
          </li>          <li>
            <a href="https://data.nodc.noaa.gov/woa/WOD/DOC/wodreadme.pdf" class="" target="_none">The World Ocean Database User&#x27;s Manual</a> by Hernan E. Garcia, Tim P. Boyer, Ricardo A. Locarnini, Olga K. Baranova, Melissa M. Zweng</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="nciccr-dlbcl" class="dataset">
        <h4><a href="../nciccr-dlbcl/">National Cancer Institute Center for Cancer Research - Diffuse Large B Cell Lymphoma (DLBCL) Genomics and Expression</a></h4>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://portal.gdc.cancer.gov/projects/NCICCR-DLBCL" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1056/NEJMoa1801445" class="" target="_none">Genetics and Pathogenesis of Diffuse Large B Cell Lymphoma</a> by Roland Schmitz, Ph.D., George W. Wright, Ph.D., Da Wei Huang, M.D., et al.</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="opencitymodel" class="dataset">
        <h4><a href="../opencitymodel/">Open City Model (OCM)</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/opencitymodel/opencitymodel/blob/master/examples/Query-OpenCityModel-using-AWS-Athena.md" class="" target="_none">Running queries on Open City Model using AWS Athena</a> by <a href="https://github.com/agilliland" target="_none">Allen Gilliland</a></a>
            <br>
            
          </li>          <li>
            <a href="https://github.com/opencitymodel/opencitymodel/blob/master/examples/3dCityDB-to-GoogleEarth.md" class="" target="_none">Using Open City Model with the 3dCityDB</a> by <a href="https://github.com/agilliland" target="_none">Allen Gilliland</a></a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="ohsu-cnl" class="dataset">
        <h4><a href="../ohsu-cnl/">Oregon Health &amp; Science University Chronic Neutrophilic Leukemia Dataset</a></h4>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://portal.gdc.cancer.gov/projects/" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/31366621" class="" target="_none">Genomic landscape of neutrophilic leukemias of ambiguous diagnosis</a> by Zhang H, Wilmot B, Bottomly D et al.</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="organoid-pancreatic" class="dataset">
        <h4><a href="../organoid-pancreatic/">Pancreatic Cancer Organoid Profiling</a></h4>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://portal.gdc.cancer.gov/projects/ORGANOID-PANCREATIC" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed?cmd&#x3D;DetailsSearch&amp;term&#x3D;29853643[PMID]" class="" target="_none">Organoid Profiling Identifies Common Responders to Chemotherapy in Pancreatic Cancer</a> by Tiriac H, Belleau P, Engle DD, Plenker D, Deschênes A, Somerville TD, et al.</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="rapid-nrt-flood-maps" class="dataset">
        <h4><a href="../rapid-nrt-flood-maps/">RAPID NRT Flood Maps</a></h4>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.3390/rs11070879" class="" target="_none">Inundation Extent Mapping by Synthetic Aperture Radar: A Review</a> by Xinyi Shen, Dacheng Wang, Kebiao Mao, Emmanouil Anagnostou, and Yang Hong</a>
            <br>
            
          </li>          <li>
            <a href="https://doi.org/10.1016/j.rse.2018.11.008" class="" target="_none">Near Real-Time Nonobstructed Flood Inundation Mapping by Synthetic Aperture Radar</a> by Xinyi Shen, Emmanouil N. Anagnostou, George H. Allen, G. Robert Brakenridge, Albert J. Kettner</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="rapid7-fdns-any" class="dataset">
        <h4><a href="../rapid7-fdns-any/">Rapid7 FDNS ANY Dataset</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="http://securityriskadvisors.com/blog/creating-a-project-sonar-fdns-api-with-aws/" class="" target="_none">Creating a Project Sonar FDNS API with AWS</a> by <a href="https://twitter.com/2xxeformyshirt" target="_none">Evan Perotti at SecurityRiskAdvisors</a></a>
            <br>
            
          </li>          <li>
            <a href="https://blog.rapid7.com/2018/10/16/how-to-conduct-dns-reconnaissance-for-02-using-rapid7-open-data-and-aws/" class="" target="_none">How to Conduct DNS Reconnaissance for $.02 Using Rapid7 Open Data and AWS</a> by <a href="https://blog.rapid7.com/author/shan/" target="_none">Shan Sikdar at Rapid7</a></a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="rareplanes" class="dataset">
        <h4><a href="../rareplanes/">RarePlanes</a></h4>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/aireveries/RarePlanes" class="" target="_none">RarePlanes Codebase</a> by Thomas Hossler and Jacob Shermeyer</a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://arxiv.org/abs/2006.02963" class="" target="_none">RarePlanes: Synthetic Data Takes Flight</a> by Jacob Shermeyer, Thomas Hossler, Adam Van Etten, Daniel Hogan, Ryan Lewis, Daeil Kim</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="sentinel1-slc-seasia-pds" class="dataset">
        <h4><a href="../sentinel1-slc-seasia-pds/">Sentinel-1 SLC dataset for South and Southeast Asia, Taiwan, Korea and Japan</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/earthobservatory/sentinel1-opds/wiki" class="" target="_none">Sentinel-1 Opendataset Wiki and Tutorials</a> by <a href="https://earthobservatory.sg/research-group/geodesy-group-emma-hill/people" target="_none">Earth Observatory of Singapore</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://www.nature.com/articles/s41597-020-0443-5" class="" target="_none">Rapid flood and damage mapping using synthetic aperture radar in response to Typhoon Hagibis, Japan</a> by Cheryl W. J. Tay, Sang-Ho Yun, Shi Tong Chin, Alok Bhardwaj, Jungkyo Jung &amp; Emma M. Hill</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="terrafusion" class="dataset">
        <h4><a href="../terrafusion/">Terra Fusion Data Sampler</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://go.illinois.edu/terra-fusion-doc" class="" target="_none">Basic Terra fusion product algorithm theoretical basis and data specifications</a> by <a href="http://hdl.handle.net/2142/105967" target="_none">Zhao, Guangu; Yang, Muqun; Clipp, Landon; Gao, Yizhao; Lee, Joe H.</a></a>
            <br>
            
          </li>        </ul>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/TerraFusion" class="" target="_none">TerraFusion GitHub</a> by University of Illinois</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="uk-met-office" class="dataset">
        <h4><a href="../uk-met-office/">UK Met Office Atmospheric Deterministic and Probabilistic Forecasts</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/MetOffice/aws-earth-examples/blob/master/examples/1.%20Getting%20Started.ipynb" class="" target="_none">Met Office AWS Earth data - Getting Started</a> by Jacob Tomlinson</a>
            <br>
            
          </li>          <li>
            <a href="https://github.com/MetOffice/aws-earth-examples/blob/master/examples/2.%20Subscribing%20to%20data.ipynb" class="" target="_none">Met Office AWS Earth data - Subscribing to data</a> by Jacob Tomlinson</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="1000-genomes" class="dataset">
        <h4><a href="../1000-genomes/">1000 Genomes</a></h4>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://aws.amazon.com/blogs/big-data/exploratory-data-analysis-of-genomic-datasets-using-adam-and-mango-with-apache-spark-on-amazon-emr/" class="aws-link" target="_none">Exploratory data analysis of genomic datasets using ADAM and Mango with Apache Spark on Amazon EMR</a> by <a href="https://research.eecs.berkeley.edu/~akmorrow/" target="_none">Alyssa Marrow</a></a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="aws-igenomes" class="dataset">
        <h4><a href="../aws-igenomes/">AWS iGenomes</a></h4>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="http://nf-co.re/" class="" target="_none">nf-core analysis pipelines</a> by <a href="http://phil.ewels.co.uk/" target="_none">Phil Ewels</a></a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="allen-brain-observatory" class="dataset">
        <h4><a href="../allen-brain-observatory/">Allen Brain Observatory - Visual Coding AWS Public Data Set</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/AllenInstitute/AllenSDK/wiki/Use-the-Allen-Brain-Observatory-%E2%80%93-Visual-Coding-on-AWS" class="" target="_none">Use the Allen Brain Observatory – Visual Coding on AWS</a> by <a href="https://twitter.com/AllenInstitute" target="_none">Nika Keller, David Feng</a></a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="answer-reformulation" class="dataset">
        <h4><a href="../answer-reformulation/">Answer Reformulation</a></h4>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://www2020.thewebconf.org/" class="" target="_none">Voice-based Reformulation of Community Answers</a> by Simone Filice, Nachshon Cohen &amp; David Carmel</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="asr-error-robustness" class="dataset">
        <h4><a href="../asr-error-robustness/">Automatic Speech Recognition (ASR) Error Robustness</a></h4>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://assets.amazon.science/64/94/639ae0c44890837b0f1fbf11ef77/using-phoneme-representations-to-build-predictive-models-robust-to-asr-errors.pdf" class="" target="_none">Using Phoneme Representations to Build Predictive Models Robust to ASR Errors</a> by <a href="https://anjiefang.github.io" target="_none">Anjie Fang, Simone Filice, Nut Limsopatham and Oleg Rokhlenko</a></a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="wrf-cmip6" class="dataset">
        <h4><a href="../wrf-cmip6/">CMIP6 GCMs downscaled using WRF</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://nbviewer.jupyter.org/urls/wrf-cmip6-noversioning.s3.amazonaws.com/Example_notebook/Notebook_example.ipynb" class="" target="_none">Jupyter Notebook Example</a> by Stefan Rahimi</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="ncbi-covid-19" class="dataset">
        <h4><a href="../ncbi-covid-19/">COVID-19 Genome Sequence Dataset</a></h4>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/sra/docs/sra-aws-download/" class="" target="_none">Download SRA sequence data using Amazon Web Services (AWS)</a> by NCBI SRA</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="cell-painting-image-collection" class="dataset">
        <h4><a href="../cell-painting-image-collection/">Cell Painting Image Collection</a></h4>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/cytodata/cytodata-hackathon-2018/tree/master/cytodata-toolkit/" class="" target="_none">Example submission for the 2018 CytoData Hackathon (in R and Python)</a> by <a href="broadinstitute.org" target="_none">Juan Caicedo, Tim Becker</a></a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="covers-br" class="dataset">
        <h4><a href="../covers-br/">CoversBR</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/SPLab-IT/CoversBR#using-the-dataset---tutorial" class="" target="_none">Using the (CoversBR) dataset</a> by <a href="https://github.com/SPLab-IT/CoversBR" target="_none">Dirceu Silva, Atila Xavier, Edgard Moraes, Marco Grivet and Fernando Perdigão</a></a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="odp-noaa-nesdis-ncei-csb" class="dataset">
        <h4><a href="../odp-noaa-nesdis-ncei-csb/">Crowdsourced Bathymetry</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/dneufeldcu/notebooks/blob/main/esipCSBfinal.ipynb" class="" target="_none">Crowdsourced Bathymetry Data (CSB) Visualization</a> by David Neufeld</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="dialoglue" class="dataset">
        <h4><a href="../dialoglue/">DialoGLUE: A Natural Language Understanding Benchmark for Task-Oriented Dialogue</a></h4>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://arxiv.org/abs/2009.13570" class="" target="_none">DialoGLUE: A Natural Language Understanding Benchmark for Task-Oriented Dialogue</a> by Shikib Mehri, Mihail Eric, Dilek Hakkani-Tur</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="topical-chat-enriched" class="dataset">
        <h4><a href="../topical-chat-enriched/">Enriched Topical-Chat Dataset for Knowledge-Grounded Dialogue Systems</a></h4>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://arxiv.org/abs/2005.12529?context&#x3D;cs.CL" class="" target="_none">Policy-Driven Neural Response Generation for Knowledge-Grounded Dialogue Systems</a> by Behnam Hedayatnia, Karthik Gopalakrishnan, Seokhwan Kim, Yang Liu, Mihail Eric &amp; Dilek Hakkani-Tur</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="ford-multi-av-seasonal" class="dataset">
        <h4><a href="../ford-multi-av-seasonal/">Ford Multi-AV Seasonal Dataset</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/Ford/AVData" class="" target="_none">Ford AV Dataset Tutorial</a> by Ford Motor Company</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="gatk-test-data" class="dataset">
        <h4><a href="../gatk-test-data/">GATK Test Data</a></h4>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://docs.opendata.aws/genomics-workflows/orchestration/cromwell/cromwell-examples/#real-world-example-haplotypecaller" class="aws-link" target="_none">Genomics Workflows on AWS - Cromwell on AWS</a> by <a href="https://www.linkedin.com/in/lee-pang-a039a26/" target="_none">W. Lee Pang</a></a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="spatial-ucr" class="dataset">
        <h4><a href="../spatial-ucr/">Geosnap Data, Center for Geospatial Sciences</a></h4>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://spatialucr.github.io/geosnap-guide/home" class="" target="_none">Geosnap User Guide</a> by <a href="https://knaaptime.com" target="_none">Eli Knaap</a></a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="hcmi-cmdc" class="dataset">
        <h4><a href="../hcmi-cmdc/">Human Cancer Models Initiative (HCMI) Cancer Model Development Center</a></h4>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://portal.gdc.cancer.gov/projects/HCMI-CMDC" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="hpgp-data" class="dataset">
        <h4><a href="../hpgp-data/">Human PanGenomics Project</a></h4>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://www.nature.com/articles/s41587-020-0503-6" class="" target="_none">Nanopore sequencing and the Shasta toolkit enable efficient de novo assembly of eleven human genomes</a> by Shafin et al (2020)</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="humor-detection" class="dataset">
        <h4><a href="../humor-detection/">Humor Detection from Product Question Answering Systems</a></h4>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://sigir.org/sigir2020/" class="" target="_none">Humor Detection in Product Question Answering Systems.</a> by Yftah Ziser, Elad Kravi &amp; David Carmel</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="ideam-radares" class="dataset">
        <h4><a href="../ideam-radares/">IDEAM - Colombian Radar Network</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="http://www.pronosticosyalertas.gov.co/documents/78690/112207077/AWS_RADARESCOL.pdf/aafc181d-0b6a-47e0-97e9-9b6290ac84d6" class="" target="_none">Guia de como explorar y plotear los archivos de radar utilizando el lenguaje de programación Python</a> by IDEAM</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="lofar-elais-n1" class="dataset">
        <h4><a href="../lofar-elais-n1/">LOFAR ELAIS-N1 cycle 2 observations on AWS</a></h4>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://www.lofarcloud.uk" class="" target="_none">Calibration of LOFAR ELAIS-N1 data in the Amazon cloud</a> by <a href="https://github.com/nudomarinero" target="_none">J. Sabater</a></a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="noaa-gfs-bdp-pds" class="dataset">
        <h4><a href="../noaa-gfs-bdp-pds/">NOAA Global Forecast System (GFS)</a></h4>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://noaa-gfs-warmstart-pds.s3.amazonaws.com/GFS+Warm+Restart+Files.v1.pdf" class="" target="_none">GFS Warm Restart Files Additional Information</a> by Fanglin Yang</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="noaa-ndfd" class="dataset">
        <h4><a href="../noaa-ndfd/">NOAA National Digital Forecast Database (NDFD)</a></h4>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://noaa-ndfd-pds.s3.amazonaws.com/NDFDelem_fullres_201906.xls" class="" target="_none">NDFD Product Spreadsheet (excel file)</a> by NOAA MDL</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="nj-imagery" class="dataset">
        <h4><a href="../nj-imagery/">New Jersey Statewide Digital Aerial Imagery Catalog</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://njogis-imagery.s3-us-west-2.amazonaws.com/NJ+Imagery+Change.pdf" class="" target="_none">Visualize Imagery Changes</a> by <a href="https://njgin.nj.gov/njgin/edata/imagery/index.html" target="_none">stephanie.bosits@tech.nj.gov</a></a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="nj-lidar" class="dataset">
        <h4><a href="../nj-lidar/">New Jersey Statewide LiDAR</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://njogis-elevation.s3-us-west-2.amazonaws.com/NJ+3D+Visualization.pdf" class="" target="_none">3D Visualization</a> by <a href="https://njgin.nj.gov/njgin/edata/elevation/index.html" target="_none">stephanie.bosits@tech.nj.us</a></a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="ocmr_data" class="dataset">
        <h4><a href="../ocmr_data/">Ohio State Cardiac MRI Raw Data (OCMR)</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/MRIOSU/OCMR/blob/master/Python/example_ocmr.ipynb" class="" target="_none">OCMR Tutorial</a> by <a href="https://cmr.engineering.osu.edu/people/trainees" target="_none">Chong Chen</a></a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="ont-open-data" class="dataset">
        <h4><a href="../ont-open-data/">Oxford Nanopore Technologies Benchmark Datasets</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://nanoporetech.github.io/ont-open-datasets/tutorials" class="" target="_none">ONT Dataset Tutorials</a> by <a href="https://labs.epi2me.io/" target="_none">EPI2MELabs</a></a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="qiime2" class="dataset">
        <h4><a href="../qiime2/">QIIME 2 User Tutorial Datasets</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://docs.qiime2.org/2018.11/install/virtual/aws/" class="" target="_none">Installing QIIME 2 using Amazon Web Services</a> by <a href="https://github.com/qiime2/docs/graphs/contributors" target="_none">The QIIME 2 Development Team</a></a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="silam" class="dataset">
        <h4><a href="../silam/">SILAM Air Quality</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/fmidev/opendata-resources/tree/master/examples/python" class="" target="_none">Simple examples</a> by Roope Tervo</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="safecast" class="dataset">
        <h4><a href="../safecast/">Safecast</a></h4>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="http://safecast.org/tilemap/" class="" target="_none">Safecast Map</a> by <a href="https://github.com/frangible" target="_none">Nick Dolezal</a></a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="sentinel-s2-l2a-mosaic-120" class="dataset">
        <h4><a href="../sentinel-s2-l2a-mosaic-120/">Sentinel-2 L2A 120m Mosaic</a></h4>
        <h5 class="dataatwork-category">Tools & Applications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/sentinel-hub/public-collections/blob/main/collections/sentinel-s2-l2a-mosaic-120.md" class="" target="_none">Sentinel Hub WMS/WMTS/WCS Service and Process API</a> by <a href="http://www.sinergise.com/" target="_none">Sinergise</a></a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="speedtest-global-performance" class="dataset">
        <h4><a href="../speedtest-global-performance/">Speedtest by Ookla Global Fixed and Mobile Network Performance Maps</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://www.speedtest.net/insights/blog/best-ookla-open-data-projects-2021/" class="" target="_none">New Year, Great Data: The Best Ookla Open Data Projects We’ve Seen So Far</a> by Katie Jolly</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="tabula-muris" class="dataset">
        <h4><a href="../tabula-muris/">Tabula Muris</a></h4>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/30283141" class="" target="_none">Single-cell transcriptomics of 20 mouse organs creates a Tabula Muris.</a> by <a href="https://tabula-muris.ds.czbiohub.org/" target="_none">Tabula Muris Consortium (2019)</a></a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="amazon-reviews-ml" class="dataset">
        <h4><a href="../amazon-reviews-ml/">The Multilingual Amazon Reviews Corpus</a></h4>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://arxiv.org/abs/2010.02573" class="" target="_none">The Multilingual Amazon Reviews Corpus</a> by Phillip Keung, Yichao Lu, György Szarvas, Noah A. Smith</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="tess" class="dataset">
        <h4><a href="../tess/">Transiting Exoplanet Survey Satellite (TESS)</a></h4>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://mast-labs.stsci.io/2018/12/tess-data-available-on-aws" class="" target="_none">TESS data available on AWS</a> by <a href="https://twitter.com/arfon" target="_none">Arfon Smith</a></a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="census-dataworld-pums" class="dataset">
        <h4><a href="../census-dataworld-pums/">U.S. Census ACS PUMS</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://docs.data.world/uscensus/#50---getting-started" class="" target="_none">Setting up Blazegraph on EC2</a> by <a href="https://data.world/" target="_none">data.world</a></a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="lab41-sri-voices" class="dataset">
        <h4><a href="../lab41-sri-voices/">Voices Obscured in Complex Environmental Settings (VOiCES)</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/voices18/utilities/blob/master/Using%20VOiCES%20corpus%20tutorial.ipynb" class="" target="_none">Getting started with VOiCES data</a> by M.A. Barrios</a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="xiph-media" class="dataset">
        <h4><a href="../xiph-media/">Xiph.Org Test Media</a></h4>
        <h5 class="dataatwork-category">Tutorials</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://media.xiph.org/aws.html" class="" target="_none">Encoding video with AV1 on EC2</a> by <a href="https://www.xiph.org/" target="_none">Thomas Daede</a></a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="zinc15" class="dataset">
        <h4><a href="../zinc15/">ZINC Database</a></h4>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="http://zinc15.docking.org" class="" target="_none">ZINC Database</a> by <a href="http://irwinlab.compbio.ucsf.edu" target="_none">John Irwin</a></a>
            <br>
            
          </li>        </ul>
      </div>
      <div id="ihart" class="dataset">
        <h4><a href="../ihart/">iHART Whole Genome Sequencing Data Set</a></h4>
        <h5 class="dataatwork-category">Publications</h5>
        <ul class="dataatwork-list">
          <li>
            <a href="https://www.cell.com/cell/fulltext/S0092-8674(19)30780-9" class="" target="_none">Inherited and De Novo Genetic Risk for Autism Impacts Shared Networks</a> by Ruzzo et al. (2020)</a>
            <br>
            
          </li>        </ul>
      </div>

    <hr/>
    <p>If you want to add a dataset or usage example to this registry, please follow the instructions on the <a href="https://github.com/awslabs/open-data-registry/">Registry of Open Data on AWS GitHub repository</a>.</p>
    <p><a href="/"><b>Home</b></a></p>
  </div>