<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    
  
    
  
    <link rel="alternate" type="application/rss+xml" href="/rss.xml" />
    
    <link rel="shortcut icon" type="image/ico" href="https://a0.awsstatic.com/libra-css/images/site/fav/favicon.ico"/>
    <link rel="apple-touch-icon" sizes="57x57" href="https://a0.awsstatic.com/libra-css/images/site/touch-icon-iphone-114-smile.png"/>
    <link rel="apple-touch-icon" sizes="72x72" href="https://a0.awsstatic.com/libra-css/images/site/touch-icon-ipad-144-smile.png"/>
    <link rel="apple-touch-icon" sizes="114x114" href="https://a0.awsstatic.com/libra-css/images/site/touch-icon-iphone-114-smile.png"/>
    <link rel="apple-touch-icon" sizes="144x144" href="https://a0.awsstatic.com/libra-css/images/site/touch-icon-ipad-144-smile.png"/>  
  
    <title>Registry of Open Data on AWS</title>
  
    <!-- Latest compiled and minified CSS -->
    <link rel="stylesheet" href="https://assets.opendata.aws/css/bootstrap/3.4.1/bootstrap.min.css">
  
    <!-- Our local CSS -->
    <link rel="stylesheet" href="/css/main.css">
  
  
  </head>
  <body>
    <!-- jQuery (necessary for Bootstrap's JavaScript plugins) -->
    <script src="https://assets.opendata.aws/js/jquery/3.5.1/jquery.min.js"></script>
    <!-- Latest compiled and minified JavaScript -->
    <script src="https://assets.opendata.aws/js/bootstrap/3.4.1/bootstrap.min.js"></script>
    <nav class="navbar navbar-default navbar-fixed-top">
      <div class="container">
        <div class="roda-header">
          <h2><a href="/" alt="Home">Registry of Open Data on AWS</a></h2>
          <a href="http://aws.amazon.com/what-is-cloud-computing">
            <img src="https://assets.opendata.aws/img/AWS-Logo_White-Color_300x180.png" alt="Powered by AWS Cloud Computing" id="aws_header_logo">
          </a>
        </div>
      </div>
    </nav>
    <div class="container" >

    <div class="about col-md-5">
      <div class="row aboutbox">
            <h3>About</h3>
            <p>This registry exists to help people discover and share datasets that are available via AWS resources. <a href="https://opendata.aws">Learn more about sharing data on AWS</a>.</p>
    
            <p>See <a href="/tag/transcriptomics/usage-examples">all usage examples for datasets listed in this registry</a> tagged with <strong>transcriptomics</strong>.</p>
    
        <hr>
    
        <h4>Search datasets (currently <span id="count-matching">13</span> matching <span id="count-matching-text">datasets</span>)</h4>
        <form>
          <div class="form-group">
            <input type="text" id="search-box" class="form-control" placeholder="Search datasets" spellcheck="false" autocorrect="off">
          </div>
        </form>
        <p>You are currently viewing a subset of data tagged with <strong>transcriptomics</strong>.</p>
    
        <hr>
    
        <h4>Add to this registry</h4>
        <p>If you want to add a dataset or example of how to use a dataset to this registry, please follow the instructions on the <a href="https://github.com/awslabs/open-data-registry/">Registry of Open Data on AWS GitHub repository</a>.</p>
    
        <p>Unless specifically stated in the applicable dataset documentation, datasets available through the Registry of Open Data on AWS are not provided and maintained by AWS. Datasets are provided and maintained by a variety of third parties under a variety of licenses. Please check dataset licenses and related documentation to determine if a dataset may be used for your application.</p>
      </div>
    </div>
    <div class="col-md-6 col-md-offset-1 datasets">
      <div class="row">
        <div id="ccle" class="dataset">
          <h3><a href="/ccle/">Cancer Cell Line Encyclopedia (CCLE)</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">Homo sapiens</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">STRIDES</span><span class="label label-info tag link-tag">transcriptomics</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>The Cancer Cell Line Encyclopedia (CCLE) project is an effort to conduct a detailed genetic
characterization of a large panel of human cancer cell lines. The CCLE provides public access to
genomic data, visualization and analysis for over 1100 cancer cell lines. This dataset contains
RNA-Seq Aligned Reads, WXS Aligned Reads, and WGS Aligned Reads data.</p>
          <p><a href="/ccle/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1038/s41586-019-1186-3" class="" target="_none">Next-generation characterization of the Cancer Cell Line Encyclopedia</a> by Ghandi, M., Huang F. et al.</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1038/s41591-019-0404-8" class="" target="_none">The landscape of cancer cell line metabolism</a> by Li, H. et al.</li></a>
          </li>
          <li>
            <a href="https://console.cloud.google.com/bigquery?p&#x3D;isb-cgc&amp;d&#x3D;CCLE_bioclin_v0&amp;page&#x3D;dataset" class="" target="_none">ISB CGC BigQuery tables</a> by <a href="https://isb-cancer-genomics-cloud.readthedocs.io/en/latest/sections/data/CCLE_top.html
" target="_none">Institute for Systems Biology</a></li></a>
          </li>
          <li>
            <a href="https://cgc-accounts.sbgenomics.com/" class="" target="_none">Cancer Genomics Cloud</a> by <a href="https://docs.sevenbridges.com/docs/ccle-data" target="_none">Seven Bridges</a></li></a>
          </li>
          <li>
            <a href="https://portals.broadinstitute.org/ccle" class="" target="_none">Broad Institute Cancer Cell Line Encyclopedia</a> by <a href="https://www.broadinstitute.org/" target="_none">The Broad Institute of MIT &amp; Harvard</a></li></a>
          </li>
        </ul>
        <p><a href="/ccle/#usageexamples">See 8 usage examples &rarr;</a></p>
        </div>
        <div id="cptac-2" class="dataset">
          <h3><a href="/cptac-2/">Clinical Proteomic Tumor Analysis Consortium 2 (CPTAC-2)</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">STRIDES</span><span class="label label-info tag link-tag">transcriptomics</span></p>
          <p>The Clinical Proteomic Tumor Analysis Consortium (CPTAC) is a national effort to accelerate the
understanding of the molecular basis of cancer through the application of large-scale proteome and
genome analysis, or proteogenomics. CPTAC-2 is the Phase II of the CPTAC Initiative (2011-2016).
Datasets contain open RNA-Seq Gene Expression Quantification, miRNA-Seq Isoform Expression
Quantification, and miRNA Expression Quantification data.</p>
          <p><a href="/cptac-2/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1016/j.cell.2016.05.069" class="" target="_none">Integrated Proteogenomic Characterization of Human High-Grade Serous Ovarian Cancer</a> by Hui Zhang, Tao Liu, Zhen Zhang, Samuel H. Payne, Bai Zhang, Jason E. McDermott, Jian-Ying
Zhou, Vladislav A. Petyuk, Li Chen, Debjit Ray, Shisheng Sun, Feng Yang, Lijun Chen, Jing
Wang, Punit Shah, Seong Won Cha, Paul Aiyetan, Sunghee Woo, Yuan Tian, Marina A. Gritsenko,
Therese R. Clauss, Caitlin Choi, Matthew E. Monroe, Stefani Thomas, Song Nie, Chaochao Wu,
Ronald J. Moore, Kun-Hsing Yu, David L. Tabb, David Feny√∂, Vineet Bafna, Yue Wang, Henry
Rodriguez, Emily S. Boja, Tara Hiltke, Robert C. Rivers, Lori Sokoll, Heng Zhu, Ie-Ming
Shih, Leslie Cope, Akhilesh Pandey, Bing Zhang, Michael P. Snyder, Douglas A. Levine,
Richard D. Smith, Daniel W. Chan, Karin D. Rodland, the CPTAC Investigators
</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1038/sdata.2015.22" class="" target="_none">Proteomic analysis of colon and rectal carcinoma using standard and customized databases
</a> by Slebos RJ, Wang X, Wang X, Zhang B, Tabb DL, Liebler DC</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1016/j.cell.2019.03.030" class="" target="_none">Proteogenomic Analysis of Human Colon Cancer Reveals New Therapeutic Opportunities</a> by Suhas Vasaikar, Chen Huang, Xiaojing Wang. Vladislav A. Petyuk, Sara R. Savage, Bo Wen,
Yongchao Dou, Yun Zhang, Zhiao Shi, Osama A. Arshad, Marina A. Gritsenko, Lisa J. Zimmerman,
Jason E. McDermott, Therese R. Clauss, Ronald J. Moore, Rui Zhao, Matthew E. Monroe, Yi-Ting
Wang, Matthew C. Chambers, Robbert J.C. Slebos, Ken S. Lau, Qianxing Mo, Li Ding, Matthew
Ellis, Mathangi Thiagarajan, Christopher R. Kinsinger, Henry Rodriguez, Richard D. Smith,
Karin D. Rodland, Daniel C. Liebler, Tao Liu, Bing Zhang, Clinical Proteomic Tumor Analysis
Consortium
</li></a>
          </li>
          <li>
            <a href="https://portal.gdc.cancer.gov/projects/CPTAC-2" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></li></a>
          </li>
          <li>
            <a href="https://proteomics.cancer.gov/data-portal" class="" target="_none">CPTAC Data Portal</a> by <a href="https://proteomics.cancer.gov/" target="_none">National Cancer Institute</a></li></a>
          </li>
        </ul>
        <p><a href="/cptac-2/#usageexamples">See 7 usage examples &rarr;</a></p>
        </div>
        <div id="cptac-3" class="dataset">
          <h3><a href="/cptac-3/">Clinical Proteomic Tumor Analysis Consortium 3 (CPTAC-3)</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">STRIDES</span><span class="label label-info tag link-tag">transcriptomics</span></p>
          <p>The Clinical Proteomic Tumor Analysis Consortium (CPTAC) is a national effort to accelerate the
understanding of the molecular basis of cancer through the application of large-scale proteome and
genome analysis, or proteogenomics. CPTAC-3 is the Phase III of the CPTAC Initiative. The dataset
contains open RNA-Seq Gene Expression Quantification data.</p>
          <p><a href="/cptac-3/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1016/j.cell.2019.10.007" class="" target="_none">Integrated Proteogenomic Characterization of Clear Cell Renal Cell Carcinoma</a> by Clark DJ, Dhanasekaran SM, Petralia F, Pan J, Song X, Hu Y, da Veiga Leprevost F, Reva B,
Lih TM, Chang HY, Ma W, Huang C, Ricketts CJ, Chen L1, Krek A, Li Y, Rykunov D, Li QK, Chen
LS, Ozbek U, Vasaikar S, Wu Y, Yoo S, Chowdhury S, Wyczalkowski MA, Ji J, Schnaubelt M, Kong
A, Sethuraman S, Avtonomov DM, Ao M, Colaprico A, Cao S, Cho KC, Kalayci S, Ma S, Liu W,
Ruggles K, Calinawan A, G√ºm√º≈ü ZH, Geizler D, Kawaler E, Teo GC, Wen B, Zhang Y, Keegan S, Li
K, Chen F, Edwards N, Pierorazio PM, Chen XS, Pavlovich CP, Hakimi AA, Brominski G, Hsieh
JJ, Antczak A, Omelchenko T, Lubinski J, Wiznerowicz M, Linehan WM, Kinsinger CR,
Thiagarajan M, Boja ES, Mesri M, Hiltke T, Robles AI, Rodriguez H, Qian J, Feny√∂ D, Zhang B,
Ding L, Schadt E, Chinnaiyan AM, Zhang Z, Omenn GS, Cieslik M, Chan DW, Nesvizhskii AI, Wang
P, Zhang H; Clinical Proteomic Tumor Analysis Consortium
</li></a>
          </li>
          <li>
            <a href="https://pdc.esacinc.com/pdc/browse" class="" target="_none">Proteomic Data Commons</a> by <a href="https://pdc.esacinc.com/pdc" target="_none">National Cancer Institute</a></li></a>
          </li>
          <li>
            <a href="https://portal.gdc.cancer.gov/projects/CPTAC-3" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></li></a>
          </li>
          <li>
            <a href="https://proteomics.cancer.gov/data-portal" class="" target="_none">CPTAC Data Portal</a> by <a href="https://proteomics.cancer.gov/" target="_none">National Cancer Institute</a></li></a>
          </li>
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/29718670" class="" target="_none">Evaluation of NCI-7 Cell Line Panel as a Reference Material for Clinical Proteomics</a> by Clark DJ, Hu Y, Bocik W, Chen L, Schnaubelt M, Roberts R, Shah P, Whiteley G, Zhang H
</li></a>
          </li>
        </ul>
        <p><a href="/cptac-3/#usageexamples">See 6 usage examples &rarr;</a></p>
        </div>
        <div id="ncbi-sra" class="dataset">
          <h3><a href="/ncbi-sra/">NIH NCBI Sequence Read Archive (SRA) on AWS</a></h3>
          <p><span class="label label-info tag link-tag">bam</span><span class="label label-info tag link-tag">fastq</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">STRIDES</span><span class="label label-info tag link-tag">transcriptomics</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>The Sequence Read Archive (SRA), produced by the <a href="https://www.ncbi.nlm.nih.gov/">National Center for Biotechnology Information (NCBI)</a> at the <a href="http://nlm.nih.gov/">National Library of Medicine (NLM)</a> at the <a href="http://www.nih.gov/">National Institutes of Health (NIH)</a>, stores raw DNA sequencing data and alignment information from high-throughput sequencing platforms. The SRA provides open access to these biological sequence data to support the research community&#39;s efforts to enhance reproducibility and make new discoveries by comparing data s...</p>
          <p><a href="/ncbi-sra/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/sra/docs/sra-cloud/" class="" target="_none">SRA in the Cloud</a> by <a href="https://www.ncbi.nlm.nih.gov/sra/" target="_none">NCBI SRA</a></li></a>
          </li>
          <li>
            <a href="https://github.com/ncbi/sra-tools" class="" target="_none">SRA Toolkit</a> by <a href="https://www.ncbi.nlm.nih.gov/sra/" target="_none">NCBI SRA</a></li></a>
          </li>
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3013647/" class="" target="_none">The Sequence Read Archive</a> by <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term&#x3D;Leinonen%20R%5BAuthor%5D&amp;cauthor&#x3D;true&amp;cauthor_uid&#x3D;21062823" target="_none">Leinonen et al (2011)</a></li></a>
          </li>
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/sra/docs/sra-aws-download/" class="" target="_none">Access SRA data using Amazon Web Services (AWS)</a> by <a href="https://www.ncbi.nlm.nih.gov/sra/" target="_none">NCBI SRA</a></li></a>
          </li>
        </ul>
        <p><a href="/ncbi-sra/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="refgenie" class="dataset">
          <h3><a href="/refgenie/">Refgenie reference genome assets</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">infrastructure</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">single-cell transcriptomics</span><span class="label label-info tag link-tag">transcriptomics</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>Pre-built refgenie reference genome data assets used for aligning and analyzing DNA sequence data.</p>
          <p><a href="/refgenie/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1093/gigascience/giz149" class="" target="_none">Refgenie: a reference genome resource manager</a> by <a href="http://www.databio.org" target="_none">Micha≈Ç Stolarczyk, Vincent P Reuter, Jason P Smith, Neal E Magee, Nathan C Sheffield</a></li></a>
          </li>
          <li>
            <a href="http://peppro.databio.org" class="" target="_none">Pipeline for PRO-seq data analysis</a> by <a href="http://databio.org" target="_none">Jason Smith and Nathan Sheffield</a></li></a>
          </li>
          <li>
            <a href="http://refgenie.databio.org/en/latest/tutorial/" class="" target="_none">Basic Refgenie tutorial</a> by <a href="http://www.databio.org" target="_none">Nathan Sheffield</a></li></a>
          </li>
          <li>
            <a href="http://pepatac.databio.org" class="" target="_none">Pipeline for ATAC-seq data analysis</a> by <a href="http://databio.org" target="_none">Jason Smith and Nathan Sheffield</a></li></a>
          </li>
        </ul>
        <p><a href="/refgenie/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="allen-mouse-brain-atlas" class="dataset">
          <h3><a href="/allen-mouse-brain-atlas/">Allen Mouse Brain Atlas</a></h3>
          <p><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">gene expression</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">image processing</span><span class="label label-info tag link-tag">imaging</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">Mus musculus</span><span class="label label-info tag link-tag">neurobiology</span><span class="label label-info tag link-tag">transcriptomics</span></p>
          <p>The Allen Mouse Brain Atlas is a genome-scale collection of cellular resolution gene expression profiles using in situ hybridization (ISH). Highly methodical data production methods and comprehensive anatomical coverage via dense, uniformly spaced sampling facilitate data consistency and comparability across &gt;20,000 genes. The use of an inbred mouse strain with minimal animal-to-animal variance allows one to treat the brain essentially as a complex but highly reproducible three-dimensional tissue array. The entire Allen Mouse Brain Atlas dataset and associated tools are available through an...</p>
          <p><a href="/allen-mouse-brain-atlas/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/AllenInstitute/open_dataset_tools/blob/master/Visualizing_Images_from_Allen_Mouse_Brain_Atlas.ipynb" class="" target="_none">Visualizing Images from the Allen Mouse Brain Atlas</a> by <a href="www.alleninstitute.org" target="_none">Allen Institute for Brain Science</a></li></a>
          </li>
          <li>
            <a href="http://www.nature.com/articles/nature05453" class="" target="_none">Genome-wide atlas of gene expression in the adult mouse brain</a> by <a href="www.alleninstitute.org" target="_none">Ed Lein, et al.</a></li></a>
          </li>
          <li>
            <a href="https://mouse.brain-map.org" class="" target="_none">Allen Mouse Brain Atlas</a> by <a href="www.alleninstitute.org" target="_none">Allen Institute for Brain Science</a></li></a>
          </li>
        </ul>
        <p><a href="/allen-mouse-brain-atlas/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="ncbi-blast-databases" class="dataset">
          <h3><a href="/ncbi-blast-databases/">Basic Local Alignment Sequences Tool (BLAST) Databases</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">health</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">protein</span><span class="label label-info tag link-tag">reference index</span><span class="label label-info tag link-tag">transcriptomics</span></p>
          <p>A centralized repository of pre-formatted BLAST databases created by the National Center for Biotechnology Information (NCBI).</p>
          <p><a href="/ncbi-blast-databases/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://pubmed.ncbi.nlm.nih.gov/9254694/" class="" target="_none">Gapped BLAST and PSI-BLAST: A New Generation of Protein Database Search Programs</a> by S F Altschul, T L Madden, A A Sch√§ffer, J Zhang, Z Zhang, W Miller, D J Lipman</li></a>
          </li>
          <li>
            <a href="https://pubmed.ncbi.nlm.nih.gov/20003500" class="" target="_none">BLAST+: Architecture and Applications</a> by Christiam Camacho  1 , George Coulouris, Vahram Avagyan, Ning Ma, Jason Papadopoulos, Kevin Bealer, Thomas L Madden</li></a>
          </li>
          <li>
            <a href="https://github.com/ncbi/blast_plus_docs" class="" target="_none">BLAST+ Docker</a> by NCBI BLAST</li></a>
          </li>
        </ul>
        <p><a href="/ncbi-blast-databases/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="ctsp-dlbcl" class="dataset">
          <h3><a href="/ctsp-dlbcl/">Clinical Trial Sequencing Project - Diffuse Large B-Cell Lymphoma</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">STRIDES</span><span class="label label-info tag link-tag">transcriptomics</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>The goal of the project is to identify recurrent genetic alterations (mutations, deletions,
amplifications, rearrangements) and/or gene expression signatures. National Cancer Institute (NCI)
utilized whole genome sequencing and/or whole exome sequencing in conjunction with transcriptome
sequencing. The samples were processed and submitted for genomic characterization using pipelines
and procedures established within The Cancer Genome Analysis (TCGA) project.</p>
          <p><a href="/ctsp-dlbcl/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1056/NEJMoa1801445" class="" target="_none">Genetics and Pathogenesis of Diffuse Large B Cell Lymphoma</a> by Roland Schmitz, Ph.D., George W. Wright, Ph.D., Da Wei Huang, M.D., Calvin A.  Johnson,
Ph.D., James D. Phelan, Ph.D., James Q. Wang, Ph.D., Sandrine Roulland, Ph.D., Monica
Kasbekar, Ph.D., Ryan M. Young, Ph.D., Arthur L. Shaffer, Ph.D., Daniel J. Hodson, M.D.,
Ph.D., Wenming Xiao, Ph.D., et al.
</li></a>
          </li>
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed?cmd&#x3D;DetailsSearch&amp;term&#x3D;29925955[PMID]" class="" target="_none">A multiprotein supercomplex controlling oncogenic signalling in lymphoma</a> by Phelan JD, Young RM, Webster DE, Roulland S, Wright GW, Kasbekar M, Shaffer AL 3rd,
Ceribelli M, Wang JQ, Schmitz R, Nakagawa M, Bachy E, Huang DW, Ji Y, Chen L, Yang Y, Zhao
H, Yu X, Xu W, Palisoc MM, Valadez RR, Davies-Hill T, Wilson WH, Chan WC, Jaffe ES, Gascoyne
RD, Campo E, Rosenwald A, Ott G, Delabie J, Rimsza LM, Rodriguez FJ, Estephan F, Holdhoff M,
Kruhlak MJ, Hewitt SM, Thomas CJ, Pittaluga S, Oellerich T, Staudt LM
</li></a>
          </li>
          <li>
            <a href="https://portal.gdc.cancer.gov/projects/CTSP-DLBCL1" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></li></a>
          </li>
        </ul>
        <p><a href="/ctsp-dlbcl/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="cgci" class="dataset">
          <h3><a href="/cgci/">Cancer Genome Characterization Initiatives - Burkitt Lymphoma, HIV+ Cervical Cancer</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">STRIDES</span><span class="label label-info tag link-tag">transcriptomics</span></p>
          <p>The Cancer Genome Characterization Initiatives (CGCI) program supports cutting-edge genomics research of adult and pediatric cancers. CGCI investigators develop and apply advanced sequencing methods that examine genomes, exomes, and transcriptomes within various types of tumors. The program includes Burkitt Lymphoma Genome Sequencing Project (BLGSP) project and HIV+ Tumor Molecular Characterization Project - Cervical Cancer (HTMCP-CC) project.
The dataset contains open Clinical Supplement, Biospecimen Supplement, RNA-Seq Gene Expression Quantification, miRNA-Seq Isoform Expression Quantificati...</p>
          <p><a href="/cgci/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1182/blood-2018-09-871418" class="" target="_none">Genome-wide discovery of somatic coding and noncoding mutations in pediatric endemic and
sporadic Burkitt lymphoma
</a> by Grande B. M., Gerhard D. S., Jiang A., Griner N. B., Abramson J. S., Alexander T. B., et al.
</li></a>
          </li>
          <li>
            <a href="https://portal.gdc.cancer.gov/projects/CGCI-BLGSP" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></li></a>
          </li>
        </ul>
        <p><a href="/cgci/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="organoid-pancreatic" class="dataset">
          <h3><a href="/organoid-pancreatic/">Pancreatic Cancer Organoid Profiling</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">STRIDES</span><span class="label label-info tag link-tag">transcriptomics</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>This study generated a collection of patient-derived pancreatic normal and cancer organoids and it was sequenced using Whole Genome Sequencing (WGS), Whole Exome Sequencing (WXS) and RNA-Seq as well as matched tumor and normal tissue if available. The study provides a valuable resource for pancreatic cancer researchers.
The dataset contains open RNA-Seq Gene Expression Quantification data and controlled WGS/WXS/RNA-Seq Aligned Reads, WXS Annotated Somatic Mutation, WXS Raw Somatic Mutation, and RNA-Seq Splice Junction Quantification.</p>
          <p><a href="/organoid-pancreatic/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://portal.gdc.cancer.gov/projects/ORGANOID-PANCREATIC" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></li></a>
          </li>
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed?cmd&#x3D;DetailsSearch&amp;term&#x3D;29853643[PMID]" class="" target="_none">Organoid Profiling Identifies Common Responders to Chemotherapy in Pancreatic Cancer</a> by Tiriac H, Belleau P, Engle DD, Plenker D, Desch√™nes A, Somerville TD, et al.</li></a>
          </li>
        </ul>
        <p><a href="/organoid-pancreatic/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="ncbi-covid-19" class="dataset">
          <h3><a href="/ncbi-covid-19/">COVID-19 Genome Sequence Dataset</a></h3>
          <p><span class="label label-info tag link-tag">bam</span><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">coronavirus</span><span class="label label-info tag link-tag">COVID-19</span><span class="label label-info tag link-tag">cram</span><span class="label label-info tag link-tag">fastq</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">health</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">MERS</span><span class="label label-info tag link-tag">SARS</span><span class="label label-info tag link-tag">STRIDES</span><span class="label label-info tag link-tag">transcriptomics</span><span class="label label-info tag link-tag">virus</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>A centralized sequence repository for all records containing sequence associated with the novel corona virus (SARS-CoV-2) submitted to the National Center for Biotechnology Information (NCBI) Sequence Read Archive (SRA). Included are both the original sequences submitted by the principal investigator as well as SRA-processed sequences that require the SRA Toolkit for analysis. Additionally, submitter provided metadata included in associated BioSample and BioProject records is available alongside NCBI calculated data, such k-mer based taxonomy analysis results, contiguous assemblies (contigs) a...</p>
          <p><a href="/ncbi-covid-19/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/sra/docs/sra-aws-download/" class="" target="_none">Download SRA sequence data using Amazon Web Services (AWS)</a> by NCBI SRA</li></a>
          </li>
        </ul>
        <p><a href="/ncbi-covid-19/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
      </div>
    </div>


    <hr/>
  </div>  </body>
  <script src="/js/index.js"></script>
</html>
