<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    
  
    
  
    <link rel="alternate" type="application/rss+xml" href="/rss.xml" />
    
    <link rel="shortcut icon" type="image/ico" href="https://a0.awsstatic.com/libra-css/images/site/fav/favicon.ico"/>
    <link rel="apple-touch-icon" sizes="57x57" href="https://a0.awsstatic.com/libra-css/images/site/touch-icon-iphone-114-smile.png"/>
    <link rel="apple-touch-icon" sizes="72x72" href="https://a0.awsstatic.com/libra-css/images/site/touch-icon-ipad-144-smile.png"/>
    <link rel="apple-touch-icon" sizes="114x114" href="https://a0.awsstatic.com/libra-css/images/site/touch-icon-iphone-114-smile.png"/>
    <link rel="apple-touch-icon" sizes="144x144" href="https://a0.awsstatic.com/libra-css/images/site/touch-icon-ipad-144-smile.png"/>  
  
    <title>Registry of Open Data on AWS</title>
  
    <!-- Latest compiled and minified CSS -->
    <link rel="stylesheet" href="https://assets.opendata.aws/css/bootstrap/3.4.1/bootstrap.min.css">
  
    <!-- Our local CSS -->
    <link rel="stylesheet" href="/css/main.css">
  
  
  </head>
  <body>
    <!-- jQuery (necessary for Bootstrap's JavaScript plugins) -->
    <script src="https://assets.opendata.aws/js/jquery/3.5.1/jquery.min.js"></script>
    <!-- Latest compiled and minified JavaScript -->
    <script src="https://assets.opendata.aws/js/bootstrap/3.4.1/bootstrap.min.js"></script>
    <nav class="navbar navbar-default navbar-fixed-top">
      <div class="container">
        <div class="roda-header">
          <h2><a href="/" alt="Home">Registry of Open Data on AWS</a></h2>
          <a href="http://aws.amazon.com/what-is-cloud-computing">
            <img src="https://assets.opendata.aws/img/AWS-Logo_White-Color_300x180.png" alt="Powered by AWS Cloud Computing" id="aws_header_logo">
          </a>
        </div>
      </div>
    </nav>
    <div class="container" >

    <div class="about col-md-5">
      <div class="row aboutbox">
            <h3>About</h3>
            <p>This registry exists to help people discover and share datasets that are available via AWS resources. <a href="https://opendata.aws">Learn more about sharing data on AWS</a>.</p>
    
            <p>See <a href="/tag/bioinformatics/usage-examples">all usage examples for datasets listed in this registry</a> tagged with <strong>bioinformatics</strong>.</p>
    
        <hr>
    
        <h4>Search datasets (currently <span id="count-matching">13</span> matching <span id="count-matching-text">datasets</span>)</h4>
        <form>
          <div class="form-group">
            <input type="text" id="search-box" class="form-control" placeholder="Search datasets" spellcheck="false" autocorrect="off">
          </div>
        </form>
        <p>You are currently viewing a subset of data tagged with <strong>bioinformatics</strong>.</p>
    
        <hr>
    
        <h4>Add to this registry</h4>
        <p>If you want to add a dataset or example of how to use a dataset to this registry, please follow the instructions on the <a href="https://github.com/awslabs/open-data-registry/">Registry of Open Data on AWS GitHub repository</a>.</p>
    
        <p>Unless specifically stated in the applicable dataset documentation, datasets available through the Registry of Open Data on AWS are not provided and maintained by AWS. Datasets are provided and maintained by a variety of third parties under a variety of licenses. Please check dataset licenses and related documentation to determine if a dataset may be used for your application.</p>
      </div>
    </div>
    <div class="col-md-6 col-md-offset-1 datasets">
      <div class="row">
        <div id="broad-gnomad" class="dataset">
          <h3><a href="/broad-gnomad/">Genome Aggregation Database (gnomAD)</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">population</span><span class="label label-info tag link-tag">population genetics</span><span class="label label-info tag link-tag">short read sequencing</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>The Genome Aggregation Database (gnomAD) is a resource developed by an international coalition of investigators that aggregates and harmonizes both exome and genome data from a wide range of large-scale human sequencing projects. The summary data provided here are released for the benefit of the wider scientific community without restriction on use.
The v2 data set (GRCh37) spans 125,748 exome sequences and 15,708 whole-genome sequences from unrelated individuals. The v3 data set (GRCh38) spans 71,702 genomes, selected as in v2.
Sign up for the gnomAD mailing list <a href="http://broad.io/gnomad_list">here</a>.</p>
          <p><a href="/broad-gnomad/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1038/s41586-020-2329-2" class="" target="_none">Transcript expression-aware annotation improves rare variant interpretation. Nature 581, 452–458 (2020)</a> by Cummings, B. B., Karczewski, K. J., Kosmicki, J. A., Seaby, E. G., Watts, N. A., Singer-Berk, M., Mudge, J. M., Karjalainen, J., Kyle Satterstrom, F., O’Donnell-Luria, A., Poterba, T., Seed, C., Solomonson, M., Alföldi, J., The Genome Aggregation Database Production Team, The Genome Aggregation Database Consortium, Daly, M. J., &amp; MacArthur, D. G.</li></a>
          </li>
          <li>
            <a href="https://aws.amazon.com/quickstart/architecture/hail/" class="aws-link" target="_none">Hail on AWS Quick Start</a> by Amazon Web Services and PrivoIT</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1038/s41467-019-12438-5" class="" target="_none">Landscape of multi-nucleotide variants in 125,748 human exomes and 15,708 genomes. Nature Communications 11, 2539 (2020)</a> by Wang, Q., Pierce-Hoffman, E., Cummings, B. B., Karczewski, K. J., Alföldi, J., Francioli, L. C., Gauthier, L. D., Hill, A. J., O’Donnell-Luria, A. H., Genome Aggregation Database (gnomAD) Production Team, Genome Aggregation Database (gnomAD) Consortium, &amp; MacArthur, D. G.</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1038/s41591-020-0893-5" class="" target="_none">The effect of LRRK2 loss-of-function variants in humans. Nature Medicine (2020)</a> by Whiffin, N., Armean, I. M., Kleinman, A., Marshall, J. L., Minikel, E. V., Goodrich, J. K., Quaife, N. M., Cole, J. B., Wang, Q., Karczewski, K. J., Cummings, B. B., Francioli, L., Laricchia, K., Guan, A., Alipanahi, B., Morrison, P., Baptista, M. A. S., Merchant, K. M., Genome Aggregation Database Production Team, ... MacArthur, D. G.</li></a>
          </li>
          <li>
            <a href="https://pypi.org/project/gnomad/" class="" target="_none">Hail utilities for gnomAD</a> by <a href="https://github.com/orgs/broadinstitute/teams/gnomad/members" target="_none">gnomAD Production Team</a></li></a>
          </li>
        </ul>
        <p><a href="/broad-gnomad/#usageexamples">See 15 usage examples &rarr;</a></p>
        </div>
        <div id="aws-covid19-lake" class="dataset">
          <h3><a href="/aws-covid19-lake/">COVID-19 Data Lake</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">coronavirus</span><span class="label label-info tag link-tag">COVID-19</span><span class="label label-info tag link-tag">health</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">medicine</span><span class="label label-info tag link-tag">MERS</span><span class="label label-info tag link-tag">SARS</span></p>
          <p>A centralized repository of up-to-date and curated datasets on or related to the spread and characteristics of the novel corona virus (SARS-CoV-2) and its associated illness, COVID-19. Globally, there are several efforts underway to gather this data, and we are working with partners to make this crucial data freely available and keep it up-to-date. Hosted on the AWS cloud, we have seeded our curated data lake with COVID-19 case tracking data from Johns Hopkins and The New York Times, hospital bed availability from Definitive Healthcare, and over 45,000 research articles about COVID-19 and rela...</p>
          <p><a href="/aws-covid19-lake/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://us-east-2.console.aws.amazon.com/cloudformation/home?region&#x3D;us-east-2#/stacks/create/review?templateURL&#x3D;https://covid19-lake.s3.us-east-2.amazonaws.com/cfn/CovidLakeStack.template.json&amp;stackName&#x3D;CovidLakeStack" class="" target="_none">CloudFormation template for Glue Catalog table definitions</a> by <a href="https://aws.amazon.com/big-data/datalakes-and-analytics/" target="_none">AWS Data Lake Team</a></li></a>
          </li>
          <li>
            <a href="https://github.com/aws-samples/query-data-in-s3-with-amazon-athena-and-aws-sdk-for-dotnet" class="aws-link" target="_none">How to use SQL to query data in S3 Bucket with Amazon Athena and AWS SDK for .NET</a> by <a href="https://github.com/orgs/aws-samples/teams/aws-proserve-us-west-applications" target="_none">AWS ProServe US West Applications Team</a></li></a>
          </li>
          <li>
            <a href="https://aws.amazon.com/blogs/big-data/exploring-the-public-aws-covid-19-data-lake/" class="aws-link" target="_none">Exploring the public AWS COVID-19 data lake</a> by <a href="https://aws.amazon.com/big-data/datalakes-and-analytics/" target="_none">AWS Data Lake Team</a></li></a>
          </li>
          <li>
            <a href="https://go.aws/covid-19-data-lake" class="aws-link" target="_none">Explore the COVID-19 data lake public S3 bucket</a> by <a href="https://aws.amazon.com/big-data/datalakes-and-analytics/" target="_none">AWS Data Lake Team</a></li></a>
          </li>
          <li>
            <a href="https://aws.amazon.com/blogs/big-data/a-public-data-lake-for-analysis-of-covid-19-data/" class="aws-link" target="_none">A public data lake for analysis of COVID-19 data</a> by <a href="https://aws.amazon.com/big-data/datalakes-and-analytics/" target="_none">AWS Data Lake Team</a></li></a>
          </li>
        </ul>
        <p><a href="/aws-covid19-lake/#usageexamples">See 5 usage examples &rarr;</a></p>
        </div>
        <div id="encode-project" class="dataset">
          <h3><a href="/encode-project/">Encyclopedia of DNA Elements (ENCODE)</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">deep learning</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">machine learning</span></p>
          <p>The Encyclopedia of DNA Elements (ENCODE) Consortium is an international collaboration of
research groups funded by the National Human Genome Research Institute (NHGRI). The goal
of ENCODE is to build a comprehensive parts list of functional elements in the human genome,
including elements that act at the protein and RNA levels, and regulatory elements that
control cells and circumstances in which a gene is active. ENCODE investigators employ a
variety of assays and methods to identify functional elements. The discovery and annotation
of gene elements is accomplished primarily by sequencing a ...</p>
          <p><a href="/encode-project/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/ENCODE-DCC/encode-data-usage-examples/blob/master/ctcf_chip_seq_cross_cell_type_correlation.ipynb" class="" target="_none">ENCODE CTCF ChIP-seq data correlation across different cell types</a> by <a href="https://github.com/p-sud" target="_none">Paul Sud</a></li></a>
          </li>
          <li>
            <a href="https://academic.oup.com/nar/article/48/D1/D882/5622708#190992715" class="" target="_none">New developments on the Encyclopedia of DNA Elements (ENCODE) data portal</a> by <a href="http://orcid.org/0000-0001-6948-2042" target="_none">Luo et al 2020</a></li></a>
          </li>
          <li>
            <a href="https://github.com/ENCODE-DCC/encode-data-usage-examples/blob/master/mount_s3_bucket_and_run_jupyter_on_ec2.ipynb" class="" target="_none">Exploring ENCODE data from EC2 with Jupyter notebook</a> by <a href="https://github.com/keenangraham" target="_none">Keenan Graham</a></li></a>
          </li>
          <li>
            <a href="https://github.com/ENCODE-DCC/encode-data-usage-examples/blob/master/ingest_encode_data_tile_db_with_s3_backend.ipynb" class="" target="_none">Ingesting ENCODE data into TileDB with S3 backend</a> by <a href="https://github.com/ottojolanki" target="_none">Otto Jolanki</a></li></a>
          </li>
        </ul>
        <p><a href="/encode-project/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="refgenie" class="dataset">
          <h3><a href="/refgenie/">Refgenie reference genome assets</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">infrastructure</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">single-cell transcriptomics</span><span class="label label-info tag link-tag">transcriptomics</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>Pre-built refgenie reference genome data assets used for aligning and analyzing DNA sequence data.</p>
          <p><a href="/refgenie/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="http://refgenie.databio.org/en/latest/tutorial/" class="" target="_none">Basic Refgenie tutorial</a> by <a href="http://www.databio.org" target="_none">Nathan Sheffield</a></li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1093/gigascience/giz149" class="" target="_none">Refgenie: a reference genome resource manager</a> by <a href="http://www.databio.org" target="_none">Michał Stolarczyk, Vincent P Reuter, Jason P Smith, Neal E Magee, Nathan C Sheffield</a></li></a>
          </li>
          <li>
            <a href="http://peppro.databio.org" class="" target="_none">Pipeline for PRO-seq data analysis</a> by <a href="http://databio.org" target="_none">Jason Smith and Nathan Sheffield</a></li></a>
          </li>
          <li>
            <a href="http://pepatac.databio.org" class="" target="_none">Pipeline for ATAC-seq data analysis</a> by <a href="http://databio.org" target="_none">Jason Smith and Nathan Sheffield</a></li></a>
          </li>
        </ul>
        <p><a href="/refgenie/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="ncbi-blast-databases" class="dataset">
          <h3><a href="/ncbi-blast-databases/">Basic Local Alignment Sequences Tool (BLAST) Databases</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">health</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">protein</span><span class="label label-info tag link-tag">reference index</span><span class="label label-info tag link-tag">transcriptomics</span></p>
          <p>A centralized repository of pre-formatted BLAST databases created by the National Center for Biotechnology Information (NCBI).</p>
          <p><a href="/ncbi-blast-databases/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://pubmed.ncbi.nlm.nih.gov/20003500" class="" target="_none">BLAST+: Architecture and Applications</a> by Christiam Camacho  1 , George Coulouris, Vahram Avagyan, Ning Ma, Jason Papadopoulos, Kevin Bealer, Thomas L Madden</li></a>
          </li>
          <li>
            <a href="https://github.com/ncbi/blast_plus_docs" class="" target="_none">BLAST+ Docker</a> by NCBI BLAST</li></a>
          </li>
          <li>
            <a href="https://pubmed.ncbi.nlm.nih.gov/9254694/" class="" target="_none">Gapped BLAST and PSI-BLAST: A New Generation of Protein Database Search Programs</a> by S F Altschul, T L Madden, A A Schäffer, J Zhang, Z Zhang, W Miller, D J Lipman</li></a>
          </li>
        </ul>
        <p><a href="/ncbi-blast-databases/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="broad-references" class="dataset">
          <h3><a href="/broad-references/">Broad Genome References</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">Homo sapiens</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">reference index</span></p>
          <p>Broad maintained human genome reference builds hg19/hg38 and decoy references.</p>
          <p><a href="/broad-references/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://docs.opendata.aws/genomics-workflows/orchestration/cromwell/cromwell-examples/#real-world-example-haplotypecaller" class="aws-link" target="_none">Genomics Workflows on AWS - Cromwell on AWS</a> by <a href="https://www.linkedin.com/in/lee-pang-a039a26/" target="_none">W. Lee Pang</a></li></a>
          </li>
          <li>
            <a href="https://aws.amazon.com/blogs/storage/using-amazon-fsx-for-lustre-for-genomics-workflows-on-aws/" class="aws-link" target="_none">Using Amazon FSx for Lustre for Genomics Workflows on AWS</a> by <a href="https://www.linkedin.com/in/lee-pang-a039a26/" target="_none">W. Lee Pang</a></li></a>
          </li>
        </ul>
        <p><a href="/broad-references/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="jhu-indexes" class="dataset">
          <h3><a href="/jhu-indexes/">Cloud Indexes for Genomic Analyses</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">mapping</span><span class="label label-info tag link-tag">medicine</span><span class="label label-info tag link-tag">reference index</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>Genomic tools use reference databases as indexes to operate quickly and efficiently, analogous to how web search engines use indexes for fast querying. Here, we aggregate genomic, pan-genomic and metagenomic indexes for analysis of sequencing data.</p>
          <p><a href="/jhu-indexes/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://benlangmead.github.io/aws-indexes/tutorial" class="" target="_none">Table of contents for tutorials for constituent tools</a> by <a href="https://benlangmead.github.io/aws-indexes" target="_none">Ben Langmead</a></li></a>
          </li>
          <li>
            <a href="https://www.biorxiv.org/content/10.1101/2020.03.03.975219v4.full.pdf" class="" target="_none">Reducing reference bias using multiple population reference genomes</a> by Chen et al (2020)</li></a>
          </li>
        </ul>
        <p><a href="/jhu-indexes/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="mimiciii" class="dataset">
          <h3><a href="/mimiciii/">MIMIC-III (‘Medical Information Mart for Intensive Care’)</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">health</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">natural language processing</span><span class="label label-info tag link-tag">us</span></p>
          <p>MIMIC-III (‘Medical Information Mart for Intensive Care’) is a large, 
single-center database comprising information relating to patients 
admitted to critical care units at a large tertiary care hospital. 
Data includes vital signs, medications, laboratory measurements, 
observations and notes charted by care providers, fluid balance, 
procedure codes, diagnostic codes, imaging reports, hospital length 
of stay, survival data, and more. The database supports applications 
including academic and industrial research, quality improvement initiatives, 
and higher education coursework. The MIMIC-I...</p>
          <p><a href="/mimiciii/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://aws.amazon.com/blogs/big-data/perform-biomedical-informatics-without-a-database-using-mimic-iii-data-and-amazon-athena/" class="aws-link" target="_none">Perform biomedical informatics without a database using MIMIC-III data and Amazon Athena</a> by James Wiggins, Alistair Johnson</li></a>
          </li>
          <li>
            <a href="https://github.com/MIT-LCP/mimic-code" class="" target="_none">MIMIC-code GitHub repository</a> by <a href="https://github.com/alistairewj" target="_none">Alistair Johnson</a></li></a>
          </li>
        </ul>
        <p><a href="/mimiciii/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="ncbi-covid-19" class="dataset">
          <h3><a href="/ncbi-covid-19/">COVID-19 Genome Sequence Dataset</a></h3>
          <p><span class="label label-info tag link-tag">bam</span><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">coronavirus</span><span class="label label-info tag link-tag">COVID-19</span><span class="label label-info tag link-tag">cram</span><span class="label label-info tag link-tag">fastq</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">health</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">MERS</span><span class="label label-info tag link-tag">SARS</span><span class="label label-info tag link-tag">STRIDES</span><span class="label label-info tag link-tag">transcriptomics</span><span class="label label-info tag link-tag">virus</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>A centralized sequence repository for all records containing sequence associated with the novel corona virus (SARS-CoV-2) submitted to the National Center for Biotechnology Information (NCBI) Sequence Read Archive (SRA). Included are both the original sequences submitted by the principal investigator as well as SRA-processed sequences that require the SRA Toolkit for analysis. Additionally, submitter provided metadata included in associated BioSample and BioProject records is available alongside NCBI calculated data, such k-mer based taxonomy analysis results, contiguous assemblies (contigs) a...</p>
          <p><a href="/ncbi-covid-19/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/sra/docs/sra-aws-download/" class="" target="_none">Download SRA sequence data using Amazon Web Services (AWS)</a> by NCBI SRA</li></a>
          </li>
        </ul>
        <p><a href="/ncbi-covid-19/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="gatk-test-data" class="dataset">
          <h3><a href="/gatk-test-data/">GATK Test Data</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span></p>
          <p>The GATK test data resource bundle is a collection of files for resequencing human genomic data with the
Broad Institute&#39;s <a href="https://software.broadinstitute.org/gatk/">Genome Analysis Toolkit (GATK)</a>.</p>
          <p><a href="/gatk-test-data/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://docs.opendata.aws/genomics-workflows/orchestration/cromwell/cromwell-examples/#real-world-example-haplotypecaller" class="aws-link" target="_none">Genomics Workflows on AWS - Cromwell on AWS</a> by <a href="https://www.linkedin.com/in/lee-pang-a039a26/" target="_none">W. Lee Pang</a></li></a>
          </li>
        </ul>
        <p><a href="/gatk-test-data/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="ont-open-data" class="dataset">
          <h3><a href="/ont-open-data/">Oxford Nanopore Technologies Benchmark Datasets</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">fast5</span><span class="label label-info tag link-tag">fastq</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">Homo sapiens</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>The ont-open-data registry provides reference sequencing data from Oxford Nanopore Technologies to support, 1) Exploration of the characteristics of nanopore sequence data. 2) Assessment and reproduction of performance benchmarks 3) Development of tools and methods. The data deposited showcases DNA sequences from a representative subset of sequencing chemistries. The datasets correspond to publicly-available reference samples (e.g. GM24385 as reference human). Raw data are provided with metadata and scripts to describe sample and data provenance.</p>
          <p><a href="/ont-open-data/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://nanoporetech.github.io/ont-open-datasets/tutorials" class="" target="_none">ONT Dataset Tutorials</a> by <a href="https://labs.epi2me.io/" target="_none">EPI2MELabs</a></li></a>
          </li>
        </ul>
        <p><a href="/ont-open-data/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="qiime2" class="dataset">
          <h3><a href="/qiime2/">QIIME 2 User Tutorial Datasets</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">denoising</span><span class="label label-info tag link-tag">ecosystems</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">health</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">microbiome</span><span class="label label-info tag link-tag">statistics</span></p>
          <p>QIIME 2 is a powerful, extensible, and decentralized microbiome analysis package with a focus on data and analysis transparency. QIIME 2 enables researchers to start an analysis with raw DNA sequence data and finish with publication-quality figures and statistical results. This dataset contains the user docs (and related datasets) for QIIME 2.</p>
          <p><a href="/qiime2/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://docs.qiime2.org/2018.11/install/virtual/aws/" class="" target="_none">Installing QIIME 2 using Amazon Web Services</a> by <a href="https://github.com/qiime2/docs/graphs/contributors" target="_none">The QIIME 2 Development Team</a></li></a>
          </li>
        </ul>
        <p><a href="/qiime2/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="molssi-covid19-hub" class="dataset">
          <h3><a href="/molssi-covid19-hub/">COVID-19 Molecular Structure and Therapeutics Hub</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">coronavirus</span><span class="label label-info tag link-tag">COVID-19</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">molecular docking</span><span class="label label-info tag link-tag">pharmaceutical</span></p>
          <p>Aggregating critical information to accelerate drug discovery for the molecular modeling and simulation community.
A community-driven data repository and curation service for molecular structures, models, therapeutics, and
simulations related to computational research related to therapeutic opportunities for COVID-19
(caused by the SARS-CoV-2 coronavirus).</p>
          <p><a href="/molssi-covid19-hub/">Details &rarr;</a></p>
        </div>
        <div id="genomeark" class="dataset">
          <h3><a href="/genomeark/">Genome Ark</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span></p>
          <p>The Genome Ark hosts genomic information for the Vertebrate Genomes Project (VGP) and other related projects. The VGP is an international collaboration that aims to generate complete and near error-free reference genomes for all extant vertebrate species. These genomes will be used to address fundamental questions in biology and disease, to identify species most genetically at risk for extinction, and to preserve genetic information of life.</p>
          <p><a href="/genomeark/">Details &rarr;</a></p>
        </div>
        <div id="ubc-sunflower-genome" class="dataset">
          <h3><a href="/ubc-sunflower-genome/">University of British Columbia Sunflower Genome Dataset</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">biodiversity</span><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">food security</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>This dataset captures Sunflower&#39;s genetic diversity originating
from thousands of wild, cultivated, and landrace sunflower
individuals distributed across North America.The data consists of raw sequences and associated botanical metadata,
aligned sequences (to three different reference genomes), and sets of
SNPs computed across several cohorts.</p>
          <p><a href="/ubc-sunflower-genome/">Details &rarr;</a></p>
        </div>
      </div>
    </div>


    <hr/>
  </div>  </body>
  <script src="/js/index.js"></script>
</html>
